<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Slack Export - #no-stupid-questions</title>
    
    <link rel=stylesheet type=text/css
          href="../../static/viewer.css">
    
</head>
<body>
<div id="slack-archive-viewer">
    
    <div id="sidebar">
        <h3 id="channel-title">Public Channels</h3>
        <ul class="list" id="channel-list">
            
                <li class="channel">
                    <a href="../2022-summerschool-introtometagenomics/index.html">
                        # 2022-summerschool-introtometagenomics
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../2023-summerschool-introtometagenomics/index.html">
                        # 2023-summerschool-introtometagenomics
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../2024-acad-aedna-workshop/index.html">
                        # 2024-acad-aedna-workshop
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../2024-summerschool-introtometagenomics/index.html">
                        # 2024-summerschool-introtometagenomics
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../amdirt-dev/index.html">
                        # amdirt-dev
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../analysis-comparison-challenge/index.html">
                        # analysis-comparison-challenge
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../analysis-reproducibility/index.html">
                        # analysis-reproducibility
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../ancient-metagenomics-labs/index.html">
                        # ancient-metagenomics-labs
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../ancient-microbial-genomics/index.html">
                        # ancient-microbial-genomics
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../ancient-microbiomes/index.html">
                        # ancient-microbiomes
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../ancientmetagenomedir/index.html">
                        # ancientmetagenomedir
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../ancientmetagenomedir-c14-extension/index.html">
                        # ancientmetagenomedir-c14-extension
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../authentication-standards/index.html">
                        # authentication-standards
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../benchmark-datasets/index.html">
                        # benchmark-datasets
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../classifier-committee/index.html">
                        # classifier-committee
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../datasharing/index.html">
                        # datasharing
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../de-novo-assembly/index.html">
                        # de-novo-assembly
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../dir-environmental/index.html">
                        # dir-environmental
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../dir-host-metagenome/index.html">
                        # dir-host-metagenome
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../dir-single-genome/index.html">
                        # dir-single-genome
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../eaa-2024-rome/index.html">
                        # eaa-2024-rome
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../early-career-funding-opportunities/index.html">
                        # early-career-funding-opportunities
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../espaam%C3%B1ol/index.html">
                        # espaam√±ol
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../events/index.html">
                        # events
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../general/index.html">
                        # general
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../it-crowd/index.html">
                        # it-crowd
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../jobs/index.html">
                        # jobs
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../lab-community/index.html">
                        # lab-community
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../lactobacillaceae-spaam4/index.html">
                        # lactobacillaceae-spaam4
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../little-book-smiley-plots/index.html">
                        # little-book-smiley-plots
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../microbial-genomics/index.html">
                        # microbial-genomics
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../minas-environmental/index.html">
                        # minas-environmental
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../minas-metadata-standards/index.html">
                        # minas-metadata-standards
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../minas-microbiome/index.html">
                        # minas-microbiome
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../minas-pathogen/index.html">
                        # minas-pathogen
                    </a>
                </li>
            
                <li class="channel active">
                    <a href="index.html">
                        # no-stupid-questions
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../papers/index.html">
                        # papers
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../random/index.html">
                        # random
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../sampling/index.html">
                        # sampling
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../scr-protocol/index.html">
                        # scr-protocol
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../seda-dna/index.html">
                        # seda-dna
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-across-the-pond/index.html">
                        # spaam-across-the-pond
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-bingo/index.html">
                        # spaam-bingo
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-blog/index.html">
                        # spaam-blog
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-ethics/index.html">
                        # spaam-ethics
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-pets/index.html">
                        # spaam-pets
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-turkish/index.html">
                        # spaam-turkish
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam-tv/index.html">
                        # spaam-tv
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam2-open/index.html">
                        # spaam2-open
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam3-open/index.html">
                        # spaam3-open
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam4-open/index.html">
                        # spaam4-open
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam5-open/index.html">
                        # spaam5-open
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaam5-organizers/index.html">
                        # spaam5-organizers
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaamfic/index.html">
                        # spaamfic
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaamghetti/index.html">
                        # spaamghetti
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../spaamtisch/index.html">
                        # spaamtisch
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../twitter/index.html">
                        # twitter
                    </a>
                </li>
            
                <li class="channel">
                    <a href="../wetlab_protocols/index.html">
                        # wetlab_protocols
                    </a>
                </li>
            
        </ul>
        <h3 id="group-title">Private Channels</h3>
        <ul class="list" id="group-list">
            
        </ul>
        <h3 id="dm-title">Direct Messages</h3>
        <ul class="list" id="dms-list">
            
        </ul>
        <h3 id="mpim-title">Group Direct Messages</h3>
        <ul class="list" id="mpims-list">
            
        </ul>
    </div>
    
    <div class="messages">
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:52:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-03 17:52:48"><div class="time">2021-09-03 17:52:48</div></a>
                <div class="msg">
                    <p>@James Fellows Yates has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:13">
            
                <img src="https://avatars.slack-edge.com/2022-06-06/3632216693492_e57e679aa24ba9eeb74d_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nasreen Broomand
                     <span class="print-only user-email">(nbrooman@ucsc.edu)</span>
                </div>
                <a href="#2021-09-03 17:53:13"><div class="time">2021-09-03 17:53:13</div></a>
                <div class="msg">
                    <p>@Nasreen Broomand has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:13">
            
                <img src="https://secure.gravatar.com/avatar/f04fc021b2d7b781c87e96d95b007cca.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Rita M Austin
                     <span class="print-only user-email">(rita.austin@nhm.uio.no)</span>
                </div>
                <a href="#2021-09-03 17:53:13"><div class="time">2021-09-03 17:53:13</div></a>
                <div class="msg">
                    <p>@Rita M Austin has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:14">
            
                <img src="https://secure.gravatar.com/avatar/0ddbf34c2ef897adc3514ff7e19ef5ea.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0008-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Jessica Hider
                     <span class="print-only user-email">(hiderj@mcmaster.ca)</span>
                </div>
                <a href="#2021-09-03 17:53:14"><div class="time">2021-09-03 17:53:14</div></a>
                <div class="msg">
                    <p>@Jessica Hider has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:14">
            
                <img src="https://avatars.slack-edge.com/2022-06-22/3704142247219_830f57556b4d56af3f0e_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Allie Mann
                     <span class="print-only user-email">(amann3@clemson.edu)</span>
                </div>
                <a href="#2021-09-03 17:53:14"><div class="time">2021-09-03 17:53:14</div></a>
                <div class="msg">
                    <p>@Allie Mann has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:14">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-09-03 17:53:14"><div class="time">2021-09-03 17:53:14</div></a>
                <div class="msg">
                    <p>@Ele has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:14">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-09-03 17:53:14"><div class="time">2021-09-03 17:53:14</div></a>
                <div class="msg">
                    <p>@Shreya has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:53:14">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2021-09-03 17:53:14"><div class="time">2021-09-03 17:53:14</div></a>
                <div class="msg">
                    <p>@Maria Zicos has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 17:57:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-09-03 17:57:51"><div class="time">2021-09-03 17:57:51</div></a>
                <div class="msg">
                    <p>@Kelly Blevins has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-03 17:58:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-03 17:58:33"><div class="time">2021-09-03 17:58:33</div></a>
                <div class="msg">
                    <p><a href="https://software.broadinstitute.org/software/igv/">https://software.broadinstitute.org/software/igv/</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:01:08">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-09-03 18:01:08"><div class="time">2021-09-03 18:01:08</div></a>
                <div class="msg">
                    <p>@aidanva has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:01:08">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-09-03 18:01:08"><div class="time">2021-09-03 18:01:08</div></a>
                <div class="msg">
                    <p>@aidanva has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:02:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2021-09-03 18:02:58"><div class="time">2021-09-03 18:02:58</div></a>
                <div class="msg">
                    <p>@Meriam Guellil has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:09:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465355512771_d76bd23c72a4c5f61546_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Gunnar Neumann
                     <span class="print-only user-email">(gunnar_neumann@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-03 18:09:33"><div class="time">2021-09-03 18:09:33</div></a>
                <div class="msg">
                    <p>@Gunnar Neumann has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:12:50">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2021-09-03 18:12:50"><div class="time">2021-09-03 18:12:50</div></a>
                <div class="msg">
                    <p>@Iseult has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:17:32">
            
                <img src="https://avatars.slack-edge.com/2020-12-09/1568237202500_7267766f139ec989f942_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Arumi Mendoza
                     <span class="print-only user-email">(arumidelapascua@gmail.com)</span>
                </div>
                <a href="#2021-09-03 18:17:32"><div class="time">2021-09-03 18:17:32</div></a>
                <div class="msg">
                    <p>@Arumi Mendoza has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 18:23:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-09-03 18:23:42"><div class="time">2021-09-03 18:23:42</div></a>
                <div class="msg">
                    <p>@Nico Rascovan has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-03 19:17:12">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2021-09-03 19:17:12"><div class="time">2021-09-03 19:17:12</div></a>
                <div class="msg">
                    <p>@Bjorn Bartholdy has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-04 00:24:20">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2021-09-04 00:24:20"><div class="time">2021-09-04 00:24:20</div></a>
                <div class="msg">
                    <p>@Maria Lopopolo has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-04 20:36:38">
            
                <img src="https://avatars.slack-edge.com/2021-08-31/2454735149025_a5707b0e71a756c6bb38_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ashley Brennaman
                     <span class="print-only user-email">(abrenn31@gmail.com)</span>
                </div>
                <a href="#2021-09-04 20:36:38"><div class="time">2021-09-04 20:36:38</div></a>
                <div class="msg">
                    <p>@Ashley Brennaman has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-06 09:27:19">
            
                <img src="https://secure.gravatar.com/avatar/b6d82a8d6c1082eefeea8152795b0bb6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Claudio Ottoni
                     <span class="print-only user-email">(claudio.ottoni@uniroma2.it)</span>
                </div>
                <a href="#2021-09-06 09:27:19"><div class="time">2021-09-06 09:27:19</div></a>
                <div class="msg">
                    <p>@Claudio Ottoni has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:05:56">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-06 11:05:56"><div class="time">2021-09-06 11:05:56</div></a>
                <div class="msg">
                    <p>@channel what sort of (no stupid) questions would you be interested in covering if there was an ancient metageomics summer school?</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:54:02">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-09-06 11:54:02"><div class="time">2021-09-06 11:54:02</div></a>
                <div class="msg">
                    <p>Databases! How to choose, download, build</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëè:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:56:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-06 11:56:34"><div class="time">2021-09-06 11:56:34</div></a>
                <div class="msg">
                    <p>Anything else (because noone has an answer for <code>choose</code> üòÜ) ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:57:33">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-09-06 11:57:33"><div class="time">2021-09-06 11:57:33</div></a>
                <div class="msg">
                    <p>For sure, how quick do you need them or can I just post them here as they arrive in my brain</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:57:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-06 11:57:46"><div class="time">2021-09-06 11:57:46</div></a>
                <div class="msg">
                    <p>As they arrive!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:58:23">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-09-06 11:58:23"><div class="time">2021-09-06 11:58:23</div></a>
                <div class="msg">
                    <p>How to authenticate your species hits for dummies. Is a match really a match?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëè:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:58:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-06 11:58:54"><div class="time">2021-09-06 11:58:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* authenticate what?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:59:33">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-09-06 11:59:33"><div class="time">2021-09-06 11:59:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Species hits, i‚Äôll edit</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 11:59:10">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-09-06 11:59:10"><div class="time">2021-09-06 11:59:10</div></a>
                <div class="msg">
                    <p>I am working on trying to pull together information I learnt last week, planning for just personal use, but something more ‚Äúformal‚Äù might be more helpful for a wider group</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 12:29:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-06 12:29:39"><div class="time">2021-09-06 12:29:39</div></a>
                <div class="msg">
                    <p>The reason why I ask is in my new position I have to set up a summer school (part of the grant its funded by)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëè Arumi Mendoza
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele, Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 12:29:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-06 12:29:52"><div class="time">2021-09-06 12:29:52</div></a>
                <div class="msg">
                    <p>So this will actually be a thing at some point, but need to work out content</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-06 13:14:13">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2021-09-06 13:14:13"><div class="time">2021-09-06 13:14:13</div></a>
                <div class="msg">
                    <p>Once I get results from EAGER and have no idea what they mean, I'll also have some "not stupid" questions üò¨</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-09-08 16:30:49">
            
                <img src="https://secure.gravatar.com/avatar/4b8f0795e46428bce5fe9524f3434a4f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Pooja Swali
                     <span class="print-only user-email">(swalipooja@gmail.com)</span>
                </div>
                <a href="#2021-09-08 16:30:49"><div class="time">2021-09-08 16:30:49</div></a>
                <div class="msg">
                    <p>@Pooja Swali has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-16 15:37:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-09-16 15:37:47"><div class="time">2021-09-16 15:37:47</div></a>
                <div class="msg">
                    <p>Hi everyone! I‚Äôm wondering if people have preferences for variant calling tools for ancient pathogens. Never done this before!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-16 15:59:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-16 15:59:34"><div class="time">2021-09-16 15:59:34</div></a>
                <div class="msg">
                    <p>1) If you want to use MultiVCFAnalyzer (e.g., to visualise cross-mapping) you <em>must</em> use UnifiedGenotyper 3.5 with ploidy set 2
2) Otherwise, AFAIK there isn't one in particular. FreeBayes is popular though</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è aidanva
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-16 16:01:19">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-09-16 16:01:19"><div class="time">2021-09-16 16:01:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thank you so much!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-16 18:45:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-09-16 18:45:09"><div class="time">2021-09-16 18:45:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve been using freebayes. It‚Äôs very easy to use. I‚Äôve played with the parameters to try to mimic the emit all sites option of UnifiedGenotyper (ie force it to be a genotype caller) so that I could use the vcfs with multivcfanalyzer or MUSIAL, but I haven‚Äôt been successful. I‚Äôm happy to share all of my scripts if you want. </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-16 19:57:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-16 19:57:42"><div class="time">2021-09-16 19:57:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah, you need a very highly specific format of your VCF files exactly as it comes out of GATK3.5 üôÑ</p>

<p>@Kelly Blevins you could also post on the MVA github page to put pressure on Alexander to find and properly release the VCF format agnostic version üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-23 00:38:03">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-09-23 00:38:03"><div class="time">2021-09-23 00:38:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Kelly Blevins, that would be incredible! Thank you so much! I‚Äôve never used a variant caller before so I‚Äôm starting from complete scratch here</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:30:24">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 10:30:24"><div class="time">2021-09-24 10:30:24</div></a>
                <div class="msg">
                    <p>Hi everybody, would be great if someone could help me. I am trying to classify test reads (generated from RefSeq viral sequences) with MALT (latest version) using as db the RefSeq viruses. Unexpectedly, I get some misclassification outside of the superkingdom Viruses, namely within Bacteria, Archaea and Eukaryota. Does someone can give me a hint, why I get a classification ‚Äòoutside‚Äô of the database?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:32:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:32:44"><div class="time">2021-09-24 10:32:44</div></a>
                <div class="msg">
                    <p>What exactly do you mean by 'refseq viruses'?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:32:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:32:52"><div class="time">2021-09-24 10:32:52</div></a>
                <div class="msg">
                    <p>What has gone into your database?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:39:36">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 10:39:36"><div class="time">2021-09-24 10:39:36</div></a>
                <div class="msg">
                    <p>Hi James, thank you for the quick reply. That is how I build the db:
<code>wget &lt;ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/viral.1.1.genomic.fna.gz&gt;
wget &lt;ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/viral.2.1.genomic.fna.gz&gt;
wget &lt;ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/viral.3.1.genomic.fna.gz&gt;
wget &lt;a href="http://ftp.ncbi.nlm.nih.gov/pub/taxonomy/accession2taxid/nucl_wgs.accession2taxid.gz"&gt;ftp.ncbi.nlm.nih.gov/pub/taxonomy/accession2taxid/nucl_wgs.accession2taxid.gz&lt;/a&gt;
malt-build -i **.fna.gz -s DNA --index viral_db --acc2taxa nucl_wgs.accession2taxid.gz</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:41:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:41:31"><div class="time">2021-09-24 10:41:31</div></a>
                <div class="msg">
                    <p>Hm ok. Have you looked at the list of non-virsus to see what they are, whether it's consistent across all samples?</p>

<p>I am slightly suspicious you're using <code>nucl_wgs</code> accession IDs, as that is just raw sequencing data and I guess could contain contamination? On the otherhand that's just the taxonomy, so you shouldn't have those sort of refeference sequences that they could align to, in your DB ü§î</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:52:11">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 10:52:11"><div class="time">2021-09-24 10:52:11</div></a>
                <div class="msg">
                    <ol><li>Not yet, that is the next step on my agenda‚Ä¶</li><li>You are right that this accession ID‚Äôs may not be the best choice. But on the other hand since the input sequences are ‚Äòjust‚Äô RefSeq sequences, I would have thought that this should be rather stable. Which accession ID‚Äôs would you recommend?</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:52:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:52:37"><div class="time">2021-09-24 10:52:37</div></a>
                <div class="msg">
                    <p>What version of MALT are you using?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:53:47">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 10:53:47"><div class="time">2021-09-24 10:53:47</div></a>
                <div class="msg">
                    <p>MALT (version 0.5.3, built 4 Aug 2021)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:54:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:54:26"><div class="time">2021-09-24 10:54:26</div></a>
                <div class="msg">
                    <p><a href="https://software-ab.informatik.uni-tuebingen.de/download/megan6/welcome.html">https://software-ab.informatik.uni-tuebingen.de/download/megan6/welcome.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:54:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:54:37"><div class="time">2021-09-24 10:54:37</div></a>
                <div class="msg">
                    
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F02G7PH4BR6/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F02G7PH4BR6/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F02G7PH4BR6-638db45cab/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="38" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:54:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:54:46"><div class="time">2021-09-24 10:54:46</div></a>
                <div class="msg">
                    <p>I <em>think</em> that should still work</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:55:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:55:22"><div class="time">2021-09-24 10:55:22</div></a>
                <div class="msg">
                    <p>I can't remembe off the top of my head the minimum assembly build level a 'genome' needs to be to get to RefSeq ...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:55:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 10:55:37"><div class="time">2021-09-24 10:55:37</div></a>
                <div class="msg">
                    <p>Maybe there is some messy WGS stuff in there? I don't know...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 10:59:17">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 10:59:17"><div class="time">2021-09-24 10:59:17</div></a>
                <div class="msg">
                    <p>Thank you James, I will try it out with the indicated accession table and see if I get more convincing results‚Ä¶ Having a messy accession table would explain the observed pattern. Let‚Äôs see. I will report it here later. James, thank you for the quick help!!!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:00:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 11:00:03"><div class="time">2021-09-24 11:00:03</div></a>
                <div class="msg">
                    <p>It would be nice if someone else could chime in though, I might be talking out of my arse üò¨ (@Maxime Borry?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:04:26">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-09-24 11:04:26"><div class="time">2021-09-24 11:04:26</div></a>
                <div class="msg">
                    <p>Hey @Samuel Neuenschwander,
At what taxonomic level do you get your assignations outside of the virus clade ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:09:53">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 11:09:53"><div class="time">2021-09-24 11:09:53</div></a>
                <div class="msg">
                    <p>Hi @Maxime Borry, indeed from the incorrectly classified reads ~6% are outside of the db scope, e.g. not Virsues. The classified rank varies, but is often around the species level.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:11:28">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-09-24 11:11:28"><div class="time">2021-09-24 11:11:28</div></a>
                <div class="msg">
                    <p>Interesting, I see two possible explanations:
‚Ä¢ the accessions IDs of some of these viruses match to their host (eg. human or whatever)
‚Ä¢ The RefSeq viruses database doesn‚Äôt only contain viruses</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:14:15">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 11:14:15"><div class="time">2021-09-24 11:14:15</div></a>
                <div class="msg">
                    <p>Yes, I agree. Both points are not so nice‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:14:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2021-09-24 11:14:33"><div class="time">2021-09-24 11:14:33</div></a>
                <div class="msg">
                    <p>Does it contain phages? That could explain some cross hits depending on the database building</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:16:12">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 11:16:12"><div class="time">2021-09-24 11:16:12</div></a>
                <div class="msg">
                    <p>That is a good point, I have to check. I don‚Äôt have the data in front of me, but I also get insects‚Ä¶ Thank you @Meriam Guellil &amp; @Maxime Borry!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:20:15">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-09-24 11:20:15"><div class="time">2021-09-24 11:20:15</div></a>
                <div class="msg">
                    <p>There could be a lot of insect viruses in the DB, because they‚Äôre commonly used in molecular biology as vectors for eukaryotic cells, the so called baculoviruses</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:23:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 11:23:58"><div class="time">2021-09-24 11:23:58</div></a>
                <div class="msg">
                    <p>Maybe contigs are taxonomic assigned in some cases in the assembly pipeline used by some researchers, and so those contigs pop up, even if the vast majortiy of contigs are from teh virus itself?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:24:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 11:24:08"><div class="time">2021-09-24 11:24:08</div></a>
                <div class="msg">
                    <p>(so point 1 of what Maxime said above)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:34:23">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 11:34:23"><div class="time">2021-09-24 11:34:23</div></a>
                <div class="msg">
                    <p>Yes, that makes absolutely sense, if an ‚Äòentire‚Äô db with all taxa would be used. With a virus only db however this should not be possible, if not one or the other input is messy.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 11:54:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-09-24 11:54:22"><div class="time">2021-09-24 11:54:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* üëç</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-09-24 19:18:49">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2021-09-24 19:18:49"><div class="time">2021-09-24 19:18:49</div></a>
                <div class="msg">
                    <p>I have replaced the accession file (with the one pointed at by James) and then also used other viral sequences to create the MALT db, but more or less with the same result. I am trying to nail down the problem, but first I will go on vacation üòâ Thank you fore the great help @James Fellows Yates@Meriam Guellil@Maxime Borry!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-06 22:40:28">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-10-06 22:40:28"><div class="time">2021-10-06 22:40:28</div></a>
                <div class="msg">
                    <p>Hi everyone! There‚Äôs a discussion going on in my lab about whether or not to sample from a tooth with a pretty large, visible cavity. I‚Äôm wondering if people have any thoughts!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 06:20:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-07 06:20:49"><div class="time">2021-10-07 06:20:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can you share a photo or sketch of what you mean?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 12:05:58">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-07 12:05:58"><div class="time">2021-10-07 12:05:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We have an ongoing project on bear oral health (@Adrian Forsythe can chip in) and see that sampling from within and closely around cavities produces a very distinct microbial signature, whereas samples from healthy teeth in the mouth with cavities look almost ‚Äúnormal‚Äù (e.g. they hardly differ from samples from healthy individuals).</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Adrian Forsythe
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 05:06:31">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2021-10-07 05:06:31"><div class="time">2021-10-07 05:06:31</div></a>
                <div class="msg">
                    <p><strong>Opinion question for the pathogen peeps:</strong> I'm screening samples for pathogens, currently particularly focusing on TB and Leprosy. If you see in MALT 100% identity reads mapping to either M. leprae or M.tuberculosis, and those same reads also map to for example M. avium with one mismatch (~97%), would you trust those reads as M. leprae/M. tuberculosis? üôÉ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 06:24:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-07 06:24:20"><div class="time">2021-10-07 06:24:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Calling: @aidanva @√Öshild (Ash) @Meriam Guellil @Marcel Keller @Maria Spyrou (off the top of my head)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 09:39:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-07 09:39:03"><div class="time">2021-10-07 09:39:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Betsy Nelson for TB as well</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-10-07 06:24:25">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2021-10-07 06:24:25"><div class="time">2021-10-07 06:24:25</div></a>
                <div class="msg">
                    <p>@√Öshild (Ash) has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-10-07 06:24:25">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2021-10-07 06:24:25"><div class="time">2021-10-07 06:24:25</div></a>
                <div class="msg">
                    <p>@Marcel Keller has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 08:48:49">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-10-07 08:48:49"><div class="time">2021-10-07 08:48:49</div></a>
                <div class="msg">
                    <p>I would say it depends on different factors. How many reads do you have? Are those reads distributed evenly on your reference? Are those reads coming from a low diversity region?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 08:49:11">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-10-07 08:49:11"><div class="time">2021-10-07 08:49:11</div></a>
                <div class="msg">
                    <p>So I guess my point being, it is very hard to do an assessment on a single read</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 08:50:15">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-10-07 08:50:15"><div class="time">2021-10-07 08:50:15</div></a>
                <div class="msg">
                    <p>one need to look at different lines to see if what you are seeing is actually the species of interest.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 08:51:59">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-10-07 08:51:59"><div class="time">2021-10-07 08:51:59</div></a>
                <div class="msg">
                    <p>but maybe others can chip in since I have not worked on mycobacterium identification so far</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 09:22:20">
            
                <img src="https://avatars.slack-edge.com/2023-08-11/5727013151811_5866d2031f65a72cd9a6_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ian Light
                     <span class="print-only user-email">(light@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2021-10-07 09:22:20"><div class="time">2021-10-07 09:22:20</div></a>
                <div class="msg">
                    <p>i would map your reads to the references of the various hits, compare the number of hits normalized over the reference length (even in Mycobacterium, genome lengths can vary by quite a bit) and also investigate edit distances to try to get a handle on where the reads ''''truly'''' are coming from. as with most things we do, nothing will be "definitive" but the above should give you a better idea of the affinity of the reads to each of the reference genomes, the coverage across the difference reference genomes and the edit distance plots which look the best (ie steadily declining rather than increasing edit distances to your reference)</p>

<p>Others might suggest doing a competitive mapping within the mycobacterium diversity, but i have not done this myself.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-10-07 09:39:09">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1356145892630_974a3d903a96540defa2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Betsy Nelson
                     <span class="print-only user-email">(nelson@shh.mpg.de)</span>
                </div>
                <a href="#2021-10-07 09:39:09"><div class="time">2021-10-07 09:39:09</div></a>
                <div class="msg">
                    <p>@Betsy Nelson has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 09:52:23">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2021-10-07 09:52:23"><div class="time">2021-10-07 09:52:23</div></a>
                <div class="msg">
                    <p>Thanks @aidanva and @Ian Light for your answers! Yes, I've also been looking at the evenness of coverage and edit distance (although I still need to create plots). I think my main problem (and probably everyone else's looking into Mycobacterium species), is trying to see through all the background in the hope there are some very low levels of a true signal... I've been using BWA mapping and MALT so far, and now working on extracting all the reads mapping to the MTBC clade in the MALT results as a way to filter out background noise... Any thoughts on this approach??</p>

<p>I also want to try stricter BWA mapping parameters. I've been using an edit distance (-n) of 0.03 for the screening, but that still picks up very deep clusters/conserved regions with lots of multiallelic sites. But I wonder if there might still be a more random distribution buried underneath these clusters. ü§î</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 09:55:32">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2021-10-07 09:55:32"><div class="time">2021-10-07 09:55:32</div></a>
                <div class="msg">
                    <p>Yeah, Mycobacteria are a harsh one. I think in this regard @Betsy Nelson @√Öshild (Ash) @Kelly Blevins can share their experience working with TB</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 10:41:38">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2021-10-07 10:41:38"><div class="time">2021-10-07 10:41:38</div></a>
                <div class="msg">
                    <p>It‚Äôs great to have this channel, and I‚Äôm happy to contribute where I‚Äôm able to. But can we establish as etiquette that questions are only answered with the thread function? This way it‚Äôs easier to keep track on what‚Äôs going on and one doesn‚Äôt get flooded with notifications.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 11:11:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-07 11:11:55"><div class="time">2021-10-07 11:11:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you are getting bothered a lot, I would highly recommend setting your notifications to mention only:</p>

<p>(right click on the channel name &gt; change notifications)</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F02GJQKEJQP/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F02GJQKEJQP/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F02GJQKEJQP-4629524c51/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="357" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 11:12:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-07 11:12:10"><div class="time">2021-10-07 11:12:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Then you will be notified when someone wants your specific feedback, but otherwise you can just look in there when you're interested</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 11:16:17">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2021-10-07 11:16:17"><div class="time">2021-10-07 11:16:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* not bothered a lot, but I would still say for a channel meant for questions and answers (rather than general discussions or announcements) it would help</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 12:19:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2021-10-07 12:19:25"><div class="time">2021-10-07 12:19:25</div></a>
                <div class="msg">
                    <p>Most species have their own threshold of how much sequence coverage you need to be able to identify them for sure which depends on a lot of factors and this can sadly vary a lot. If you don't have enough coverage there are a couple of other rabbit holes you can go down (specific regions, competitive mappings, masking of conserved regions, etc) but sometimes you will need to increase the coverage to be sure. Without seeing the data its hard to tell, but I am sure the MTB people have some tips up their sleeves :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 12:20:03">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2021-10-07 12:20:03"><div class="time">2021-10-07 12:20:03</div></a>
                <div class="msg">
                    <p>(sorry @Marcel Keller ship has sailed for this question lets do better on the next one üòâ)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates, Nico Rascovan
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Meriam van Os
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 13:12:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-07 13:12:29"><div class="time">2021-10-07 13:12:29</div></a>
                <div class="msg">
                    <p>@Meriam van Os I echo everyone else‚Äôs advice - breadth of coverage and edit distance are really important. If you‚Äôre picking positives from MALT for downstream investigation though, I have found that the frequency of MTBC summed reads/mycobacteria summed reads (at least .25) predicts successful capture enrichment pretty well. If you‚Äôre seeing something that looks like the attached, I would say don‚Äôt get your hopes up. How deep are these libraries sequenced? And how old? Most of my experience is from screening ~500 year old remains from central Mexico for MTBC, but I‚Äôve worked with some more recent (200-300 years old) samples from Belgium and Spain that followed the same pattern (intermittent qPCR assay positivity (IS1081, IS610); pileup at mycobacteria node in MALT after shotgun; less than 5% of the MTBC ref genome covered after MTBC capture, where the 5% is pileup at conserved regions).</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F02GZTXTLTU/screen_shot_2021-10-07_at_12.03.00_pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2021-10-07 at 12.03.00 PM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F02GZTXTLTU/screen_shot_2021-10-07_at_12.03.00_pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F02GZTXTLTU-14011b60d1/screen_shot_2021-10-07_at_12.03.00_pm_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="354" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 13:15:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-07 13:15:09"><div class="time">2021-10-07 13:15:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For example of a weak positive, attached is a screenshot after a MALT screening of a shotgun library. It doesn‚Äôt look great, right? A solid chunk of reads could be assigned to the MTBC node, but the majority could not be resolved further. I was able to recover a partial genome (~70% of the genome covered at 1x) from this library. So not enough for analyses, but enough to confirm MTBC and  justify making another extract or library.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F02GZUHT34K/screen_shot_2021-10-07_at_12.01.54_pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2021-10-07 at 12.01.54 PM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F02GZUHT34K/screen_shot_2021-10-07_at_12.01.54_pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F02GZUHT34K-3439304d3d/screen_shot_2021-10-07_at_12.01.54_pm_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="344"
                 height="360" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 13:16:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-07 13:16:56"><div class="time">2021-10-07 13:16:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What database are you using for MALT? I made (a small) one for myobacteria and friends that I‚Äôm happy to share.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-07 23:17:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2021-10-07 23:17:42"><div class="time">2021-10-07 23:17:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Kelly, this is super helpful, thank you! We've built a MALT database with ~600 Mycobacterium genomes (MTBC and non-MTBC species). Do you think this should be sufficient? And, so you're saying that in MALT, if about 25% of Mycobacterium reads map to the MTBC, that's a pretty good indicator?
I've done capture as well, samples have about 12 -20 million collapsed reads. All of them are showing over 700 MTBC reads with a 100% identity threshold. These samples were selected for their lesions and have tested positive for at least IS1081. Looking at the distribution today, so finger crossed something real is in there!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-12 11:59:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-12 11:59:25"><div class="time">2021-10-12 11:59:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes of course no problem! Yeah, I think that‚Äôs plenty. My myco database only has around 500 genomes.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-12 12:03:15">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-12 12:03:15"><div class="time">2021-10-12 12:03:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yep, that‚Äôs been a good indicator for me. If at least 25% of the myco summed reads can be assigned to the MTBC, then I have been able to get at least a partial genome from the library after capture. So I would say it‚Äôs not a way to authenticate MTBC positivity in a sample but a way to predict a positive capture, if that makes sense. I don‚Äôt think we‚Äôve a good enough grasp of myco diversity to authenticate MTBC with just a few reads, even if they‚Äôre evenly distributed.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-12 12:13:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-12 12:13:47"><div class="time">2021-10-12 12:13:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Depending on your capture efficiency, you should be able to get away with sequencing at half that depth. I‚Äôve found that I can sequence at a depth of ~2-3 million reads after Daicel Arbor Biosciences myBaits capture (we send them MTBC genomic DNA and they make the baits and send us the kit <a href="https://arborbiosci.com/genomics/targeted-sequencing/mybaits/mybaits-expert/mybaits-expert-wge-whole-genome-enrichment/">https://arborbiosci.com/genomics/targeted-sequencing/mybaits/mybaits-expert/mybaits-expert-wge-whole-genome-enrichment/</a>) and expect to sequence the captured library to saturation.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Daicel Arbor Biosciences</div>
                            
                            
                                
                                <div class="link-title"><a href="https://arborbiosci.com/genomics/targeted-sequencing/mybaits/mybaits-expert/mybaits-expert-wge-whole-genome-enrichment/">myBaits Expert Whole Genome Enrichment (WGE)</a></div>
                                <div class="link-text">
                                    Daicel Arbor Biosciences‚Äô WGE bait manufacturing technology can cost-effectively produce biotinylated RNA baits representative of an entire nuclear genome. This allows for bulk enrichment of genome-wide endogenous DNA from complex metagenomic samples, such as environmental or ancient DNA. To manufacture custom WGE baits, all that is required is a sample of high-quality genomic DNA from [‚Ä¶]
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Est. reading time</div>
                                        1 minute
                                    </div>
                                
                                
    
        <a href="https://arborbiosci.com/genomics/targeted-sequencing/mybaits/mybaits-expert/mybaits-expert-wge-whole-genome-enrichment/">
            <img class="preview" src="https://arborbiosci.com/wp-content/uploads/2017/10/banner-Paleogenetics.jpg"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://arborbiosci.com/genomics/targeted-sequencing/mybaits/mybaits-expert/mybaits-expert-wge-whole-genome-enrichment/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-12 12:15:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-12 12:15:42"><div class="time">2021-10-12 12:15:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good luck! You‚Äôre working in the Pacific, right? I REALLY hope you find something üôè:skin<em>tone</em>2:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2021-10-07 13:59:37">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-07 13:59:37"><div class="time">2021-10-07 13:59:37</div></a>
                <div class="msg">
                    <p>@Adrian Forsythe has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 11:54:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-08 11:54:37"><div class="time">2021-10-08 11:54:37</div></a>
                <div class="msg">
                    <p>Hello, I have a question. If you want to re-sequence a certain library to reach the desired genome coverage (e.g. 10X), where the targeted genome could be human, microbial, etc., which calculation do you use to estimate how many additional reads/Gbp you will need?</p>

<p>Of course the simple formula would be something like this:
Reads to generate = <em>[ GENOME SIZE ** (proportion of unique mapping reads from total raw reads) ** Expected read depth ]</em> <strong>/</strong> <em>(average read size of uniquely mapping reads)</em></p>

<p>A second option is to Run preseq, and propagate the curve it estimates. This solution is certainly better than the previous one, as the % of duplicate reads is not linear.</p>

<p>Do you have some other method that outperforms preseq for this? If using preseq, do you use any particular tuning ? Do you have a script to share? (other than the outputs we can get from nf-core/eager)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:06:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:06:20"><div class="time">2021-10-08 16:06:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Felix Key has a nice little script that parses PreSeq output to give you more precise information: <a href="https://github.com/keyfm/shh">https://github.com/keyfm/shh</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/keyfm/shh">GitHub - keyfm/shh: Code developed for SHH projects</a></div>
                                <div class="link-text">
                                    Code developed for SHH projects. Contribute to keyfm/shh development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/keyfm/shh">
            <img class="preview" src="https://opengraph.githubassets.com/1563220519f087a43389f473de3b1616c2e55ae299f0217ae66e205e9779b712/keyfm/shh"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/keyfm/shh</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:06:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:06:41"><div class="time">2021-10-08 16:06:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* However looking at the README it might customised for eager1 üòï</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:06:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:06:50"><div class="time">2021-10-08 16:06:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So you might have to do some reconstruction</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:07:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:07:27"><div class="time">2021-10-08 16:07:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Looks like this:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:07:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:07:30"><div class="time">2021-10-08 16:07:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F02H5LXSTT5/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F02H5LXSTT5/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F02H5LXSTT5-8ceb1beee5/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="353" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:07:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:07:43"><div class="time">2021-10-08 16:07:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (CF == cluster factor == duplication ratio)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:09:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:09:51"><div class="time">2021-10-08 16:09:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Still PreSeq though</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 16:12:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-08 16:12:23"><div class="time">2021-10-08 16:12:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It looks liek a lot ofnf-core pipelines still use PreSeq actually, so I dunno if there is  abetter mehtod</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-08 20:36:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-08 20:36:31"><div class="time">2021-10-08 20:36:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok, great. Thanks a lot James. I think that building a plot like this will be enough for a first try. We were just discussing today with @Pierre Luisi if in addition to the expected distinct reads curve we could also estimate if the genomic positions covered are evenly distributed along the genome or if there is also some sort of asymptotic curve of the chromosomes/regions/breadth of coverage while increasing the sequencing depth, that does not fully follow the same trend than the preseq curve‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-10 23:48:13">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-10-10 23:48:13"><div class="time">2021-10-10 23:48:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* you can also have a look at nonpareil <a href="https://github.com/lmrodriguezr/nonpareil">https://github.com/lmrodriguezr/nonpareil</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/lmrodriguezr/nonpareil">GitHub - lmrodriguezr/nonpareil: Estimate metagenomic coverage and sequence diversity</a></div>
                                <div class="link-text">
                                    Estimate metagenomic coverage and sequence diversity - GitHub - lmrodriguezr/nonpareil: Estimate metagenomic coverage and sequence diversity
                                </div>
                                
                                
    
        <a href="https://github.com/lmrodriguezr/nonpareil">
            <img class="preview" src="https://opengraph.githubassets.com/071b211095d30ed7aaa70ca9f4248306440fc44f9e31a9e6f0977f9235e7680c/lmrodriguezr/nonpareil"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/lmrodriguezr/nonpareil</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nico Rascovan, James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 08:31:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-11 08:31:20"><div class="time">2021-10-11 08:31:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Two caveats about nonpareil:</p>

<p>1) It's more for metagenomics rather than genome coverage for a target genome
2) Only supports uncompressed FASTQ files, which sucks baaaaad</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 09:53:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-11 09:53:59"><div class="time">2021-10-11 09:53:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see. Incredible that there are still tools using uncompressed fastq files out there‚Ä¶ I got the impression it was more metagenomics oriented</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 09:54:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-11 09:54:05"><div class="time">2021-10-11 09:54:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* but it‚Äôs good to know it exists</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 10:09:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-11 10:09:44"><div class="time">2021-10-11 10:09:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* A LOT of metagenomic tools still only accept/produce uncompressed files</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 10:10:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-11 10:10:01"><div class="time">2021-10-11 10:10:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It was crazy, Maxime and I went through loads recently. But I think with modern data, you don't need to sequence much...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 10:12:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-11 10:12:54"><div class="time">2021-10-11 10:12:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* still, disk space is way too expensive to spare it in something like this</p>
                    
                    
                    
                        <div class="message-reaction">
                        üíØ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 10:26:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-11 10:26:37"><div class="time">2021-10-11 10:26:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Either the devs have üí∞</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-11 10:26:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-11 10:26:42"><div class="time">2021-10-11 10:26:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* or just tiny assemblies</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-12 10:47:51">
            
                <img src="https://secure.gravatar.com/avatar/3f0f432081691561ae770131929daaf6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Markella Moraitou
                     <span class="print-only user-email">(Markella.Moraitou.0437@student.uu.se)</span>
                </div>
                <a href="#2021-10-12 10:47:51"><div class="time">2021-10-12 10:47:51</div></a>
                <div class="msg">
                    <p>Hello everyone :)</p>

<p>I am wondering what the SPAAM community thinks of the different releases of Kraken2. I did my master's project on dental calculus and the first time I run Kraken2 (v. 2.0.8 beta) / Bracken I got almost 10000 taxa identified in the dataset. I recently analyzed (almost) the same dataset again, using Kraken2 v.2.1.1 and only got about 2500 taxa identified. From what I understand, this most recent release of the software works a bit like KrakenUniq in reducing the false positive rate. Do you think this change could explain such a big difference in the number of taxa identified? In your experience, do the most recent versions of Kraken2 indeed perform better? Would there be any acceptable reason to keep using an old version?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-12 10:51:42">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-12 10:51:42"><div class="time">2021-10-12 10:51:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To add to that: With the new version of Kraken2, we recover considerably fewer oral taxa from our community (40 vs 120).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-21 11:30:57">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2021-10-21 11:30:57"><div class="time">2021-10-21 11:30:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi everyone! I've been thinking about this question a bit since I'm currently using Kraken2 but have seen how Kraken1+KrakenUniq was a preferred alternative during SPAAM3. I found the github page with the changes in each version released (<a href="https://github.com/DerrickWood/kraken2/blob/master/CHANGELOG.md">https://github.com/DerrickWood/kraken2/blob/master/CHANGELOG.md</a>) but I don't understand it well enough so I don't know if these changes are more likely to affect false positive detection.
Would it be possible to get advice on this? A friend and I in our group are the only ones doing metagenomics and we're kind of towards the end of our PhD so now is a good time to decide whether it's ok to stay with Kraken2 or if we should just reconfigure our reference database for Kraken1 and KrakenUniq. If Kraken2 now behaves a bit less wildly with false positives that would be a faster solution for us since all that's needed is a software update!</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/DerrickWood/kraken2/blob/master/CHANGELOG.md">kraken2/CHANGELOG.md at master ¬∑ DerrickWood/kraken2</a></div>
                                <div class="link-text">
                                    The second version of the Kraken taxonomic sequence classification system - kraken2/CHANGELOG.md at master ¬∑ DerrickWood/kraken2
                                </div>
                                
                                
    
        <a href="https://github.com/DerrickWood/kraken2/blob/master/CHANGELOG.md">
            <img class="preview" src="https://opengraph.githubassets.com/9992763d5599de320f80b3b50082ff0d802398637fb00526d7844fdcc7dc15df/DerrickWood/kraken2"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/DerrickWood/kraken2/blob/master/CHANGELOG.md</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 10:40:38">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-10-25 10:40:38"><div class="time">2021-10-25 10:40:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates if you get a second could you please do your tagging people who might know on this post? Obvs no rush!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 10:42:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-25 10:42:07"><div class="time">2021-10-25 10:42:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oooff...</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§Ø Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 10:42:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-25 10:42:22"><div class="time">2021-10-25 10:42:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'm not really sure. Generally I would use the latests tools, but I've really used kraken2...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 10:42:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-25 10:42:29"><div class="time">2021-10-25 10:42:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov or @Maxime Borry maybe?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 10:56:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-10-25 10:56:33"><div class="time">2021-10-25 10:56:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov was doing some comparisons between KrakenUniq and Kraken2 if I remember correctly.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 11:35:22">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-10-25 11:35:22"><div class="time">2021-10-25 11:35:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But the official word is that Kraken2 (since version 2.1.0) with the <code>--report-minimizer-data</code> flag should produce the same results as KrakenUniq</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        ü§© Ele, Maria Zicos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 11:56:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-25 11:56:37"><div class="time">2021-10-25 11:56:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Maxime üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 12:05:03">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-25 12:05:03"><div class="time">2021-10-25 12:05:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks everyone. Any first hand experience comparing the new version of Kraken2 to the older one? I am happy about the reduction of the overall number of taxa detected (as obviously the previous version was recovering lots of spurious hits) but the loss of 2/3 of oral taxa is scary‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 12:22:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-25 12:22:11"><div class="time">2021-10-25 12:22:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Did you use the same database &amp; database version?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 12:22:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-25 12:22:25"><div class="time">2021-10-25 12:22:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maybe just many more genomes added and pushed hits further the tree?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 12:23:51">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-25 12:23:51"><div class="time">2021-10-25 12:23:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No, new database, so we are not sure what is causing the effect. Have not tested systematically, as simply not enough time for that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-25 12:25:22">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-25 12:25:22"><div class="time">2021-10-25 12:25:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Am hoping someone has the answer ü§™</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-06 11:26:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2021-11-06 11:26:28"><div class="time">2021-11-06 11:26:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi guys, a lot of things to say here, sorry for being late to the discussion. First, I do not think that newer versions of Kraken2 are more accurate than the older versions of Kraken2 @Katerina Guschanski. Instead, the growth of databases (as @James Fellows Yates asked about) with time usually results in fewer detected taxa (at the same depth of coverage threshold). Second, newer versions of Kraken2 themselves do not reduce false-positive rate @Markella Moraitou if you do not filter your output with respect to the breadth of coverage, which is provided ONLY if you use the --report-minimizer-data flag as @Maxime Borry mentioned, please note it is not a default flag. Third, even with --report-minimizer-data the breadth of coverage delivered by Kraken2 is very different from the one from KrakenUniq (if you just look at the number of unique kmers stats), however, they are correlated. In my case, after I checked ~10 human + non-human samples, the correlation slope was ~0.7 meaning that if one used 1000 unique kmers as a threshold for KrakenUniq, this should corespond to 700 unique kmers threshold for Kraken2 in order to get more or less comparable final lists of detected taxa. Fourth, if you carefully examine Kraken1 / KrakenUniq and Kraken2 papers, they clearly write (in the Kraken2 paper) that the database search they have implemented in Kraken2 (approximate minimizer search instead of exact kmer search in Kraken1 / KrakenUniq) reduces specificity of classification (how much - not clear) but offers instead superior speed and memory advantages. Moreover, the authors also mention the effect of "collision" of minimizers corresponding to different kmers (a minimizer is just a substring of a kmer), which introduces some "randomness" in the taxonomic classification from one database to another. In other words, no two Kraken2 databases are identical (obviously not good for reproducibility), so even if you use the same ref genomes, two databases built at two different time points might result in different classification results. In summary, after extensive testing Kraken2 vs. KrakenUniq I am still not convinced that the speed and memory advantages of Kraken2 can compensate for the reduction in specificity</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§© Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-06 11:56:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-11-06 11:56:50"><div class="time">2021-11-06 11:56:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Brah... That would be a perfect blog post, literally just that... @Shreya @Ele üëÄ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-06 20:30:00">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-11-06 20:30:00"><div class="time">2021-11-06 20:30:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Wow, so cool and helpful @Nikolay Oskolkov! Thank you so much for the explainer!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-07 13:05:14">
            
                <img src="https://secure.gravatar.com/avatar/3f0f432081691561ae770131929daaf6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella Moraitou
                     <span class="print-only user-email">(Markella.Moraitou.0437@student.uu.se)</span>
                </div>
                <a href="#2021-11-07 13:05:14"><div class="time">2021-11-07 13:05:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Nikolay Oskolkov (and everyone else who responded)! I was not aware that the increase of database size could have this effect!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-08 13:17:41">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2021-11-08 13:17:41"><div class="time">2021-11-08 13:17:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maria Zicos think this will be of interest to you. @Nikolay Oskolkov as always you explain things so nicely, thank you! And yes if you fancy a writing a blog post we would love to host your expertise! Let me know and I can send you some details üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-15 15:54:14">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-11-15 15:54:14"><div class="time">2021-11-15 15:54:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov comment prompted me to have a look at the Kraken2 paper again: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1891-0">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1891-0</a>
One additional point that I should point in favour of Kraken2: it uses spaced-seeds, meaning that it allows for mistmatches (7 out of 31 by default) in the minimizer, which AFAIK, is not the case for KrakenUniq.
In KrakenUniq, no exact kmer match = no detection.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-15 16:05:14">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-11-15 16:05:14"><div class="time">2021-11-15 16:05:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's really interesting, @Maxime Borry. So this could be one of the reasons for higher number of taxa detected by Kraken2</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-15 17:13:16">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-11-15 17:13:16"><div class="time">2021-11-15 17:13:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could be one of the reason, yes. You can also adjust this parameter to allow the number of mismatches that you want (from 0 to 0.25**minimizer length).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-15 21:41:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2021-11-15 21:41:21"><div class="time">2021-11-15 21:41:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maxime Borry for KrakenUniq / Kraken1 I would say "no exact kmer match = no assignment of the kmer", which does not necessarily mean that a read is not assigned since a read can have a number of kmers and they all "vote" for a particular taxon. But generally agree that Kraken2 is more permissive and KrakenUniq / Kraken1 is more conservative.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-13 18:06:37">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-13 18:06:37"><div class="time">2021-10-13 18:06:37</div></a>
                <div class="msg">
                    <p>Hi, all! What is everyone doing to decontaminate skeletal elements before powdering? Does everyone still UV bone samples for 15-30 minutes per side? Is there any concern that doing so does more harm than good?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-13 18:24:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-13 18:24:30"><div class="time">2021-10-13 18:24:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Uving isn't actually AS bad as some people thingk.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-13 18:24:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-13 18:24:40"><div class="time">2021-10-13 18:24:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Christina Warinner often refers to this paper: <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0013042">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0013042</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">journals.plos.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0013042">An Efficient Multistrategy DNA Decontamination Procedure of PCR Reagents for Hypersensitive PCR Applications</a></div>
                                <div class="link-text">
                                    Background PCR amplification of minute quantities of degraded DNA for ancient DNA research, forensic analyses, wildlife studies and ultrasensitive diagnostics is often hampered by contamination problems. The extent of these problems is inversely related to DNA concentration and target fragment size and concern (i) sample contamination, (ii) laboratory surface contamination, (iii) carry-over contamination, and (iv) contamination of reagents. Methodology/Principal Findings Here we performed a quantitative evaluation of current decontamination methods for these last three sources of contamination, and developed a new procedure to eliminate contaminating DNA contained in PCR reagents. We observed that most current decontamination methods are either not efficient enough to degrade short contaminating DNA molecules, rendered inefficient by the reagents themselves, or interfere with the PCR when used at doses high enough to eliminate these molecules. We also show that efficient reagent decontamination can be achieved by using a combination of treatments adapted to different reagent categories. Our procedure involves Œ≥- and UV-irradiation and treatment with a mutant recombinant heat-labile double-strand specific DNase from the Antarctic shrimp Pandalus borealis. Optimal performance of these treatments is achieved in narrow experimental conditions that have been precisely analyzed and defined herein. Conclusions/Significance There is not a single decontamination method valid for all possible contamination sources occurring in PCR reagents and in the molecular biology laboratory and most common decontamination methods are not efficient enough to decontaminate short DNA fragments of low concentration. We developed a versatile multistrategy decontamination procedure for PCR reagents. We demonstrate that this procedure allows efficient reagent decontamination while preserving the efficiency of PCR amplification of minute quantities of DNA.
                                </div>
                                
                                
    
        <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0013042">
            <img class="preview" src="https://journals.plos.org/plosone/article/figure/image?id=10.1371/journal.pone.0013042.g004&amp;size=inline"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0013042</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-13 18:25:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-13 18:25:21"><div class="time">2021-10-13 18:25:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It can be bad if you have wet samples + UV is strong and right next to the sample (IIRC)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-14 10:20:36">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-14 10:20:36"><div class="time">2021-10-14 10:20:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We tested UV only vs. EDTA wash only on a small subset of samples and ended up doing both, as both were removing some parts of contaminants but having little effect on endogenous (oral, using DC samples). Not very systematic results of this not very systematic experiment are presented here: <a href="https://academic.oup.com/mbe/article/37/10/3003/5848415?login=true">https://academic.oup.com/mbe/article/37/10/3003/5848415?login=true</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-14 10:22:17">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-14 10:22:17"><div class="time">2021-10-14 10:22:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We also forgot to put away our tooth samples from the lab bench once and they ended up under ceiling UV for 2 h, without any advert effects on the host endogenous content. We never published this faux pas (surprise...)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üòÖ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-14 10:22:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-14 10:22:42"><div class="time">2021-10-14 10:22:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* sounds like a good anonymous twitter post üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-14 10:22:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-14 10:22:47"><div class="time">2021-10-14 10:22:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <b>#HonestMethods</b></p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§™ Katerina Guschanski
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-14 11:54:41">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-10-14 11:54:41"><div class="time">2021-10-14 11:54:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you both so much! This is very helpful üôå:skin<em>tone</em>2:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-14 15:18:50">
            
                <img src="https://avatars.slack-edge.com/2021-11-06/2696551938660_c79c1f6ef245c5ff1040_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christina Warinner
                     <span class="print-only user-email">(warinner@shh.mpg.de)</span>
                </div>
                <a href="#2021-10-14 15:18:50"><div class="time">2021-10-14 15:18:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* With UV, it matters a LOT how close your sample is to the UV bulb. The strength really drops quickly off with distance. The main advantage of using UV in a room is that it kills all the microbes on the surfaces and so keeps the room low biomass. However, overhead UV does very little to short aDNA fragments, especially if the sample is dry and not close to the UV bulb. I think it can probably mostly be skipped. I think the EDTA wash is more useful</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-22 17:27:05">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2021-10-22 17:27:05"><div class="time">2021-10-22 17:27:05</div></a>
                <div class="msg">
                    <p>I have a question regarding reference sequences: doing my first steps in microbiome analysis, I na√Øvely thought that you could go on <a href="https://www.ncbi.nlm.nih.gov/genome/">https://www.ncbi.nlm.nih.gov/genome/</a>, search for your species you are interested, and download the fasta under ‚ÄúReference genome‚Äù. It turns out this was wrong, e.g., for Streptococcus sanguinis, where the reference genome is SK36 but ncbi directs to SK405, which is not even a full genome. So if downloading bigger numbers of different reference sequences, where do people find them without going through the literature?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-26 09:19:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-26 09:19:40"><div class="time">2021-10-26 09:19:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Mmmm, I would have done the same thing as you did‚Ä¶. You can also check all the assemblies of a species and their sizes in bp and chose one in the median size o something like that‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-26 10:10:09">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2021-10-26 10:10:09"><div class="time">2021-10-26 10:10:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks, in the meantime I also noticed that even for Yersinia pestis the ‚Äúreference genome‚Äù is not CO92 but A1122 (assembly), so I‚Äôm wondering if this should be used at all</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-26 10:32:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-10-26 10:32:02"><div class="time">2021-10-26 10:32:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That‚Äôs strange‚Ä¶ It was CO92 not long ago‚Ä¶
Nevertheless, CO92 was a very bad reference to use anyway‚Ä¶ haha</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:30:53">
            
                <img src="https://secure.gravatar.com/avatar/c6c81f36dfff58739a14554171aedb1b.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Christina Balentine
                     <span class="print-only user-email">(cmbalentine@utexas.edu)</span>
                </div>
                <a href="#2021-10-27 14:30:53"><div class="time">2021-10-27 14:30:53</div></a>
                <div class="msg">
                    <p>Hi all! I‚Äôm sequencing aDNA libraries prepped following a protocol derived from Rohland et al 2015's partial UDG protocol on an Illumina NovaSeq platform. The samples aren‚Äôt pooled; each library has its own Illumina adapter. I‚Äôm finding that all of my R2 reads contain 20-60% single-G repeat sequences (basically 40 Gs in a row repeated dozens to thousands of times). Has anyone seen this before, and have any idea on how to remedy it for future sequencing attempts? Thanks in advanced!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:33:18">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-27 14:33:18"><div class="time">2021-10-27 14:33:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, poly-G tails are common on Illumina 2-colour chemistry machines (NextSeq/NovaSeq)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:38:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-27 14:38:26"><div class="time">2021-10-27 14:38:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://github.com/OpenGene/fastp">https://github.com/OpenGene/fastp</a></p>

<p>More specifically: <a href="https://github.com/OpenGene/fastp#polyg-tail-trimming">https://github.com/OpenGene/fastp#polyg-tail-trimming</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/OpenGene/fastp">GitHub - OpenGene/fastp: An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)</a></div>
                                <div class="link-text">
                                    An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...) - GitHub - OpenGene/fastp: An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filter...
                                </div>
                                
                                
    
        <a href="https://github.com/OpenGene/fastp">
            <img class="preview" src="https://opengraph.githubassets.com/6b4f415437539855ae58e7a5d54e8497359307e4127d80bffad763a27320db3a/OpenGene/fastp"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/OpenGene/fastp</div>
                                
                                
                            
                        </div>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/OpenGene/fastp#polyg-tail-trimming">GitHub - OpenGene/fastp: An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)</a></div>
                                <div class="link-text">
                                    An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...) - GitHub - OpenGene/fastp: An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filter...
                                </div>
                                
                                
    
        <a href="https://github.com/OpenGene/fastp#polyg-tail-trimming">
            <img class="preview" src="https://opengraph.githubassets.com/6b4f415437539855ae58e7a5d54e8497359307e4127d80bffad763a27320db3a/OpenGene/fastp"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/OpenGene/fastp#polyg-tail-trimming</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:38:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-27 14:38:40"><div class="time">2021-10-27 14:38:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* fastP has a specific command to clip them off (we use that in nf-core/eager)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:39:36">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-27 14:39:36"><div class="time">2021-10-27 14:39:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* More info: <a href="https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/">https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">sequencing.qcfail.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/">QC Fail Sequencing ¬ª Illumina 2 colour chemistry can overcall high confidence G bases</a></div>
                                <div class="link-text">
                                    With the introduction of the NextSeq system Illumina changed the way their image data was acquired so that instead of capturing 4 images per cycle they needed only 2. This speeds up image acquisition significantly but also introduces a problem where high quality calls for G bases can be made where there is actually no signal on the flowcell.
                                </div>
                                
                                
    
        <a href="https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/">
            <img class="preview" src="https://sequencing.qcfail.com/wp-content/uploads/sites/2/2016/05/single_seq_quality.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:47:54">
            
                <img src="https://secure.gravatar.com/avatar/c6c81f36dfff58739a14554171aedb1b.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christina Balentine
                     <span class="print-only user-email">(cmbalentine@utexas.edu)</span>
                </div>
                <a href="#2021-10-27 14:47:54"><div class="time">2021-10-27 14:47:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much for the info, @James Fellows Yates! Good to know it‚Äôs not just my libraries.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 14:51:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-27 14:51:06"><div class="time">2021-10-27 14:51:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Nope, perfectly normal üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Christina Balentine
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-27 20:58:32">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2021-10-27 20:58:32"><div class="time">2021-10-27 20:58:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Wondering why this would affect predominantly R2 reads. Any idea?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-29 06:05:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-10-29 06:05:52"><div class="time">2021-10-29 06:05:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* End of the chemistry reagents, more out of sync cluster, meaning the camera can't work out what colour it is, resulting in 'nothing' to be detected</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-28 04:33:56">
            
                <img src="https://avatars.slack-edge.com/2021-08-31/2454735149025_a5707b0e71a756c6bb38_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ashley Brennaman
                     <span class="print-only user-email">(abrenn31@gmail.com)</span>
                </div>
                <a href="#2021-10-28 04:33:56"><div class="time">2021-10-28 04:33:56</div></a>
                <div class="msg">
                    <p>Hello all, I am an archaeologist doing research with aDNA for my dissertation. This is my first experience with this type of research and I am focusing on microbiome analysis from historic calculus samples. I have just received my sequenced data from the lab and I wanted to ask if either 1) someone might be willing to help me get started with analysis, and/or 2) might know of anyone/anywhere that might offer analytical services for this type of research (ideally someone who would work with me collaboratively so I could learn as well)? I am working with a microbiologist at my university, but there is no one on campus with aDNA experience, and my efforts to find external help have not been successful. I've discussed some particulars of my project with some other graduate students I met at SPAAM, but many of the programs are already setup for analysis on their lab network. Using my research questions as a guide, I have loaded all of the necessary programs for analysis to my computer, I'm just not sure where to go from there in getting things setup. I am a beginner at this, but I am eager to learn and I appreciate any guidance that anyone is willing to provide. Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-28 18:09:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-10-28 18:09:09"><div class="time">2021-10-28 18:09:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You could check the summer-school we made this year to get you started: <a href="https://github.com/mpi-sHH-SummerSchool/dag-material">https://github.com/mpi-sHH-SummerSchool/dag-material</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/mpi-sHH-SummerSchool/dag-material">GitHub - MPI-SHH-SummerSchool/dag-material</a></div>
                                <div class="link-text">
                                    Contribute to MPI-SHH-SummerSchool/dag-material development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/mpi-sHH-SummerSchool/dag-material">
            <img class="preview" src="https://opengraph.githubassets.com/b4f78b6d57ee58578f5ec5f05147d2168f6e09f66caf6d56650efbfdcd7123fa/MPI-SHH-SummerSchool/dag-material"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/mpi-sHH-SummerSchool/dag-material</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-28 18:09:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-10-28 18:09:58"><div class="time">2021-10-28 18:09:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and <a href="https://youtube.com/channel/UC4ieuUEHUqYQGQF_DKIwcKA">https://youtube.com/channel/UC4ieuUEHUqYQGQF_DKIwcKA</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">YouTube</div>
                            
                            
                                
                                <div class="link-title"><a href="https://youtube.com/channel/UC4ieuUEHUqYQGQF_DKIwcKA">MPI-SHH Summer School</a></div>
                                <div class="link-text">
                                    Share your videos with friends, family, and the world
                                </div>
                                
                                
    
        <a href="https://youtube.com/channel/UC4ieuUEHUqYQGQF_DKIwcKA">
            <img class="preview" src="https://yt3.ggpht.com/WglB4ixXHFinTGwbxlS1akZjoILl16u3iLW75-Bxak2IREO6Wb4-4LV-jCUFOJiPvtfLbxsj=s900-c-k-c0x00ffffff-no-rj"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://youtube.com/channel/UC4ieuUEHUqYQGQF_DKIwcKA</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-10-29 00:36:56">
            
                <img src="https://avatars.slack-edge.com/2021-08-31/2454735149025_a5707b0e71a756c6bb38_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ashley Brennaman
                     <span class="print-only user-email">(abrenn31@gmail.com)</span>
                </div>
                <a href="#2021-10-29 00:36:56"><div class="time">2021-10-29 00:36:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you, I will definitely check out those resources.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-03 16:25:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-11-03 16:25:29"><div class="time">2021-11-03 16:25:29</div></a>
                <div class="msg">
                    <p>I feel like there are lots of Krakenuniq questions, do we need a whole channel just for it? I‚Äôm interested in building as comprehensive a database as possible. The krakenuniq default ‚Äúnt‚Äù option is only the microbes in nt‚Äî should I be manually downloading the entire ‚Äúnt‚Äù from the blast server, running dustmasker on it, and kraken-building it from there? </p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates, Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-03 16:29:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-11-03 16:29:33"><div class="time">2021-11-03 16:29:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Nah 9 domt think it's necessary, would be overkill. What would be good is a blog post or community contribution to the krakenuniq docs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-06 11:46:38">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2021-11-06 11:46:38"><div class="time">2021-11-06 11:46:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya and @James Fellows Yates, yes, this is exactly what I do. I download the whole blast nt database and build a KrakenUniq database out of it. This however would only be beneficial (compared to the microbial NT from KrakenUniq) if you want to also screen for eukaryotes in addition to microbes. However, with sediment aDNA becoming very common (next will be catching aDNA from the air of Denosova cave via filters üôÇ ), adding eukaryotes to a database seems unavoidable to me. The problem here however is the very poor quality (or unequal quality) of eukaryotic (more specifically, mammalian) reference genomes in the blast nt database, this might lead to such mammals as horse, wolf, mouse and pig always present if you classify or align reads from another mammal (which is a pure artifact of unequal quality of mammalian ref genomes in the blast nt database). However, the poor quality of mammalian ref genomes in the blast nt database makes it technically possible to run classification or alignment (since the whole blast nt is only ~300 GB so not so much RAM is needed). If one needs to use good quality ref genomes from ~300 mammals, that would be close to impossible to use for alignment or classification (in terms of memory resources, because it would require a few TB of RAM)</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates, Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-15 17:16:31">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-11-15 17:16:31"><div class="time">2021-11-15 17:16:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you Nikolay!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-16 22:51:18">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-11-16 22:51:18"><div class="time">2021-11-16 22:51:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Well, I am back with more questions to annoy you with, apologies and thank you in advance @Nikolay Oskolkov! I do indeed want to include eukaryotes--humans in particular and at least being able to detect a potential mammal! I have downloaded nt.fna.gz and run dustmasker on it and now realizing I need to make the seqid to taxid mapping file in order to build the database. If I download nucl<em>gb.accession2taxid and nucl</em>wgs.accession2taxid, concatenate them, and pull the accession and taxid columns, would that work? Seems like the krakenuniq-download command would handle all of this for me but leave out the eukaryotes‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-16 22:55:03">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-11-16 22:55:03"><div class="time">2021-11-16 22:55:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh dear and I have to add a third column if i want to use the --taxids-for-sequences and --taxids-for-genomes! ü§Ø</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-17 14:04:43">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2021-11-17 14:04:43"><div class="time">2021-11-17 14:04:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya yes, you need nucl<em>gb.accession2taxid and nucl</em>wgs.accession2taxid files but also names.dmp and nodes.dmp files, if I remember correctly. You know what, the simplest would be if you use
<code>kraken2-build --download-library nt --db FULL_NT</code>
this will download the full NT (including eukaryotes, invertebrates etc.) with all the mapping files needed. Despite you use Kraken2 for this purpose, you will build the KrakenUniq database using something like this:</p>

<p><code>krakenuniq-build --db FULL_NT --kmer-len 31 --threads 80 --taxids-for-genomes --taxids-for-sequences</code>
@Abby Gancz you might also find this thread useful because you are also building a KrakenUniq database</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-17 16:31:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2021-11-17 16:31:32"><div class="time">2021-11-17 16:31:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* üòÆ I hadn‚Äôt even thought of using kraken2 to download!! Brilliant!! Thank you so much Nikolay!!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-16 11:39:36">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2021-11-16 11:39:36"><div class="time">2021-11-16 11:39:36</div></a>
                <div class="msg">
                    <p>Hi Friends! I have a question about <em><strong>library method choice</strong></em> for coprolite metagenomics, which I feel is applicable to general experimental design.
<strong>My background/setup:</strong> I am doing dietary/paleoenvironmental metagenomics on ancient sloth coprolites for my PhD. I started the work with 7 samples, experimenting with extraction methods, and doing double-stranded libraries, which I screened. I determined the best extraction method from those (PowerSoil kit &gt; Plant mini kit or Dabney adapted for tissue, which is the standard my lab uses for non-bone).
<strong>My conundrum</strong>: I have now got five more coprolites, which I extracted only with the PowerSoil kit (because there's not too much point with the other methods). My group is looking into changing from Meyer-Kircher libs, potentially towards single-stranded protocols, likely SCR. I'm wondering whether I should still do these five libraries with Meyer-Kircher to keep consistency with the rest of the study, or whether it's better to just go with whatever is likely to yield more data. I feel the latter is a more responsible a way to use up a paleontological sample (always maximise outputs), but I'm worried that maybe using a different library protocol, especially moving from double-stranded to single-stranded, will impact the DNA community in the sample quite a bit and make these new samples not comparable with the previous ones...
<strong>My questions to you all</strong>: Are some of you working with single-stranded libraries on metagenomics? How has that impacted your results? Does recovering ssDNA do cool things for detecting more taxa? Have you had trouble with mixed datasets of double and single-stranded libraries? What would you advise me to do? Do you think my choice could impact "publishability" of the work?
In an ideal world, I think my instinct would be to try these new libs with the ss protocol, and if it improves yields etc. as expected, re-do the previous 7 samples with this new protocol too, and re-screen them. Only this chapter is currently not a spending priority (I'm doing a lot of popgen stuff too) so I don't know if I can afford re-building and re-sequencing the old 7 samples.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-18 06:46:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-11-18 06:46:20"><div class="time">2021-11-18 06:46:20</div></a>
                <div class="msg">
                    <p>Hi @Maria Zicos </p>

<p>It's a very good question. No one has done a systematic study for ancient Metagenomics (low hanging fruit for a good publication I think ;))</p>

<p>For calculus I did a tiny bit of experimentation and it didn't make much difference when doing microbial genome construction (both very old and more recent).</p>

<p>It depends on your question ultimately I think. Do you want to do Metagenomic de novo assembly? In this case having DS libraries are probably better as you get more longer (still ancient!) reads, whereas SS will increase the proportion of very short but unusable reads for assembly.</p>

<p>However if you're looking for dietary DNA or host DNA, it would make more sense maybe to Indeed switch to SS as you will pick up a greater proportion of true endogenous DNA, which you <em>can</em> use for reference based mapping approaches.</p>

<p>I don't know of any study that estimates the amount of increase you may get for SS. I would maybe refer you to my normal poop question person @Alex H√ºbner, but he often is dealing with samples with unusual preservation. I don't know if he has SS libraries</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-18 08:17:23">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2021-11-18 08:17:23"><div class="time">2021-11-18 08:17:23</div></a>
                <div class="msg">
                    <p>Hi @Maria Zicos,
That‚Äôs indeed a very interesting question and I guess no one has answered yet. I agree with James‚Äô classification of the use of DS and SS libraries. If you are mainly interested in doing reference-based analysis, you should try SS libraries, however, they won‚Äôt be off much help for assemblies. Many of the palaeofaeces samples, we have processed in our lab that relatively good preservation but also were excavated at archaeological sites that favour preservation (low-water exposure environments) and we always only did DS libraries.
Regarding the design of your experiment, in case you figure out that SS libraries are far superior over DS libraries, you should go back to re-process the old coprolites, too, if possible. You have a relatively low number of samples in your study and consistency in the preparation would increase the power for statistical analyses.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-18 14:25:45">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-11-18 14:25:45"><div class="time">2021-11-18 14:25:45</div></a>
                <div class="msg">
                    <p>Is there any reason you could not perform Kircher et al. 2012 style dual-indexing with indexing primers of different lengths? For example, could I use a forward indexing primer with an 8 nucleotide index and a reverse indexing primer with a 7 nucleotide index?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-18 17:28:50">
            
                <img src="https://avatars.slack-edge.com/2022-06-06/3632216693492_e57e679aa24ba9eeb74d_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nasreen Broomand
                     <span class="print-only user-email">(nbrooman@ucsc.edu)</span>
                </div>
                <a href="#2021-11-18 17:28:50"><div class="time">2021-11-18 17:28:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't think there would be an issue with building it (although I also might be wrong there, idk), but I think it might make data processing more difficult? I could be wrong, but I would imagine you would need to only use the first 7 nucleotides of the 8-nt index when demultiplexing (which could be an issue if that last base is the only difference between two indices) and then you'd have to trim an additional base</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-11-19 11:31:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2021-11-19 11:31:58"><div class="time">2021-11-19 11:31:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This is what I am not sure about, because I have no experience with demultiplexing. I thought maybe as long as you specify the index seqs in the sample sheet it wouldn‚Äôt matter if they were different lengths.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-02 20:31:54">
            
                <img src="https://avatars.slack-edge.com/2022-06-06/3632216693492_e57e679aa24ba9eeb74d_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nasreen Broomand
                     <span class="print-only user-email">(nbrooman@ucsc.edu)</span>
                </div>
                <a href="#2021-12-02 20:31:54"><div class="time">2021-12-02 20:31:54</div></a>
                <div class="msg">
                    <p>Does anyone know how the spreadsheets that come out of the HOPS pipeline are created? We just noticed that the species listed in these spreadsheets aren't always the same when comparing different runs with different individuals, and read numbers and species listed in these spreadsheets are also not always the same as the read numbers and species present in the rma6 file for the same library (which I'm guessing has to do with that not being filtered through yet?). At first I thought it was maybe just whatever species had hits were listed in the spreadsheet, but a lot of the time there are 0 reads listed for some of these species, so I'm not sure. Is it that there are some species hard coded into the spreadsheet and others that just make it into the spreadsheet when there are hits in the rma6 file? Any insight on how the program decides which species and reads make it into these output spreadsheets is appreciated, thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-02 20:50:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-12-02 20:50:39"><div class="time">2021-12-02 20:50:39</div></a>
                <div class="msg">
                    <p>Answer from @aidanva and I (she's sitting next to me)</p>

<p>1) rma6 contains everything - consider this raw
2) hops will only report hits of stuff your taxon list that you specify. 
3) However it also looks <em>below</em> that node (I think two levels) and report those as well (depends on the discuss). With the MALT LCA it may not push reads higher up the tree if it's unique at that node
4) Megan may not agree with hops because hops does additional filtering by default such as destacking and deduplication that will reduce the numbers of reads</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Betsy Nelson, Nasreen Broomand
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-02 20:51:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-12-02 20:51:33"><div class="time">2021-12-02 20:51:33</div></a>
                <div class="msg">
                    <p>Also Aida says in a given HOPS table, it will show all taxa that has been found in the run across all samples (even if that particular sample doesn't have his to it)</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Nasreen Broomand
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-07 00:13:25">
            
                <img src="https://avatars.slack-edge.com/2022-06-06/3632216693492_e57e679aa24ba9eeb74d_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nasreen Broomand
                     <span class="print-only user-email">(nbrooman@ucsc.edu)</span>
                </div>
                <a href="#2021-12-07 00:13:25"><div class="time">2021-12-07 00:13:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you both!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-14 14:17:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-12-14 14:17:06"><div class="time">2021-12-14 14:17:06</div></a>
                <div class="msg">
                    <p>Someone remembers the explanation of why ‚Äúporcine type-c oncovirus‚Äù is a typical taxonomic hit in metagenomics?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-14 15:58:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2021-12-14 15:58:25"><div class="time">2021-12-14 15:58:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If it‚Äôs a guessing game, I would place my bets on the endogenous retrovirus explanation (apparently it‚Äôs a virus that can be integrated in mammalian genomes), and/or a genome contamination (since <em>Sus scrofa</em> appears quite a few times in <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1">Conterminator</a>)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1">Terminating contamination: large-scale search identifies more than 2,000,000 contaminated entries in GenBank - Genome Biology</a></div>
                                <div class="link-text">
                                    Genomic analyses are sensitive to contamination in public databases caused by incorrectly labeled reference sequences. Here, we describe Conterminator, an efficient method to detect and remove incorrectly labeled sequences by an exhaustive all-against-all sequence comparison. Our analysis reports contamination of 2,161,746, 114,035, and 14,148 sequences in the RefSeq, GenBank, and NR databases, respectively, spanning the whole range from draft to ‚Äúcomplete‚Äù model organism genomes. Our method scales linearly with input size and can process 3.3 TB in 12 days on a 32-core computer. Conterminator can help ensure the quality of reference databases. Source code (GPLv3): &lt;https://github.com/martin-steinegger/conterminator&gt;
                                </div>
                                
                                
    
        <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/13059.jpg"
                width="177"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üíØ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-14 16:15:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-12-14 16:15:02"><div class="time">2021-12-14 16:15:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My response too! I also wonder if it could be a model organism or something...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-14 16:15:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2021-12-14 16:15:04"><div class="time">2021-12-14 16:15:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But more likely the retro virus</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-14 18:01:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-12-14 18:01:46"><div class="time">2021-12-14 18:01:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, cool</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2021-12-14 18:03:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2021-12-14 18:03:42"><div class="time">2021-12-14 18:03:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* it is just that I have a group of saliva samples, all taken and processed in the same way, but I only get the hit in some of them‚Ä¶ so that‚Äôs the tricky part‚Ä¶ Once would expect to find them in all or most of them‚Ä¶ But well, of course there are many technical biases that could explain this as well‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 10:56:38">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 10:56:38"><div class="time">2022-01-17 10:56:38</div></a>
                <div class="msg">
                    <p>Hello everyoneI'm writing to you because I'm doing my master thesis on a permafrost analysis. I am working on samples of  a very old permafrost soil from Greenland. I have extracted  some DNA from it. However, I had a very small biomass and I don't think I have much DNA. After extracting the DNA I did PCRs on the 16s of the bacteria to check if there was any in my sample and I had bacterial DNA in my samples. After that, I wanted to know the DNA concentration of my extractions.</p>

<p>And I could get these results:¬†</p>

<p>&gt; Sample¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†Concentration (ng/¬µL)               A260                       260/280            260/230
&gt; 226¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†             4.8227¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†¬†                  0.096                      2.119                 0.088
&gt; 227                                     6.063                                          0.1213                    2.073                 0.044</p>

<p>All the numbers correspond to what we expect (very low absorbance at 260nm, but i think it is normal the biomass is very low and even more the DNA)
But the 260/230 ratio is very strange, it should be around 2, and here it is really tiny. Have you ever had this kind of case with ancient DNA? What is also possible is the presence of many soil phenols in my sample. But could the presence of degraded ancient DNA have any influence on this ratio?</p>

<p>Thank you in advance for your answers!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 11:50:29">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 11:50:29"><div class="time">2022-01-17 11:50:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* **My project consists in sequencing the metagenome of the organisms still living in this soil.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:28:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:28:19"><div class="time">2022-01-17 14:28:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Louis Lhote that's somewhat to say. A first caveat is that the vast majority of people here so shotgun sequencing not (16S) PCR</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:31:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:31:41"><div class="time">2022-01-17 14:31:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For the ratios I know @Zandra Fagern√§s @ivelsko might have some knowledge as they've talked about it in our group meetings in the past.</p>

<p>More specifically for sediment in general: @Pete Heintzman @Vilma P√©rez @Barbara @Anan Ibrahim @Linda Armbrecht might have some experience</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:33:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:33:25"><div class="time">2022-01-17 14:33:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Louis, the low 260/230 suggests one of 2 kinds of contamination: guanidium salts, or protein</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:33:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:33:58"><div class="time">2022-01-17 14:33:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Both can be cleaned up some by washing, but with such low DNA concentrations you risk losing most of it</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:34:36">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 14:34:36"><div class="time">2022-01-17 14:34:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey, I plan on doing some shotgun sequencing afterwards. But the PCR was only to verify the presence of live bacterial DNA (the pcr gave fragments of about 500 bp which excludes ancient degraded dna)
I want to be sure of the quality of my DNA before sequencing it
Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:36:10">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:36:10"><div class="time">2022-01-17 14:36:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's really low for post-PCR values though. How did you clean up the PCR product before you took the 260/280/230 readings? And what did you take the readings with? NanoDrop? Qubit?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:40:15">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 14:40:15"><div class="time">2022-01-17 14:40:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok thanks! @ivelsko  do you think it is still possible to do a sequencing with this kind of contamination ? This is only the raw result after extraction of the dna from the soil. The PCR was only used to ensure the presence of large dna in my sample. I calculated this value with a nanodrop</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:44:36">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:44:36"><div class="time">2022-01-17 14:44:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah, ok. With values this low, the NanoDrop isn't very accurate and it's better to take readings with a Qubit. Do you have access to one? If so, re-run your extracts through that and you'll have much more accurate values</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:44:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:44:51"><div class="time">2022-01-17 14:44:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you can't use a Qubit, I think you can still proceed with library building with this little DNA</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 14:45:23">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 14:45:23"><div class="time">2022-01-17 14:45:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The subsequent clean-up steps and amplification should be enough to remove the contamination and bring up the DNA concentration</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 15:21:48">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 15:21:48"><div class="time">2022-01-17 15:21:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @ivelsko Thank you for this information! I normally have access to a Qbit. I will try to see what I can get as a value with it.  What is the risk of doing a sequencing without cleaning my sample? Can it create bias?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 15:27:34">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 15:27:34"><div class="time">2022-01-17 15:27:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The main issue will probably be poor library construction, so your library might be biased and then your sequencing results will be too. It sort of depends on the method you use for library construction, some will likely be more affected than others</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 15:28:59">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 15:28:59"><div class="time">2022-01-17 15:28:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Since you're looking at the aDNA you'll probably be using the Meyer&amp;Kircher protocol? I think that will be less affected than say the Nextera kits that use an enzyme to shear the DNA</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 15:35:47">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 15:35:47"><div class="time">2022-01-17 15:35:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @ivelsko I'm actually going to do a Pacbio metagenome sequencing (I've never done one so I'll check with my lab what they use). The goal is to have the dna of organisms still living in the soil.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 16:47:01">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459627217282_61a755f605bb00ddb48a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">ivelsko
                     <span class="print-only user-email">(velsko@shh.mpg.de)</span>
                </div>
                <a href="#2022-01-17 16:47:01"><div class="time">2022-01-17 16:47:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've never done any PacBio sequencing, so I'm not familiar with what they want for input quality. It's definitely good to ask the sequencing center what they expect of sample quality for high-quality sequencing data</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 16:54:00">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-17 16:54:00"><div class="time">2022-01-17 16:54:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @ivelsko ok thanks! I'll ask</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç ivelsko
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-17 19:00:19">
            
                <img src="https://secure.gravatar.com/avatar/84c0abe3d5fd77dc5667f9fbbe12294c.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anan Ibrahim
                     <span class="print-only user-email">(ananhamido@hotmail.com)</span>
                </div>
                <a href="#2022-01-17 19:00:19"><div class="time">2022-01-17 19:00:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Louis Lhote What type of material is your sediment? i had a similar problem with sediment samples from the deep layers (~20m). They had a particularly high level of clay content which i presume always affected the ratio readings of my nanodrop.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-18 11:31:39">
            
                <img src="https://secure.gravatar.com/avatar/71dff709e64f44dee922b7975cd2cd6a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Lhote
                     <span class="print-only user-email">(louis.lhote@outlook.fr)</span>
                </div>
                <a href="#2022-01-18 11:31:39"><div class="time">2022-01-18 11:31:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Anan Ibrahim it is permafrost core so a very dry and cold soil. Thanks for your response! I think it is the same problem</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:11:59">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 14:11:59"><div class="time">2022-01-19 14:11:59</div></a>
                <div class="msg">
                    <p>Hi!
in the lab we advanced quite a lot in processing our aDNA data with nf-core/eager. It is going quite well, but we see some bias and some metrics (with fastQC after clipping and damaeprfiler) that disturb us. We are now in the great moment when you need to undestand the effect of parameters at the different steps. I have a couple of specific question about parameters for Adapter removal and damaprofiler. Better to ask here or in nf-core/eager slack channel?
Thanks</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:13:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:13:44"><div class="time">2022-01-19 14:13:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If it's specifically questions about Adapter removal and damage profile, this is not eager specific so here should be fine</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:19:00">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 14:19:00"><div class="time">2022-01-19 14:19:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Great!
so, as a starter...</p>

<ol><li><a href="http://1.in">in</a> nf-core/eager the  minimum adapter overlap required for clipping is set to 1 by default, when usually it is preferred the Adapter Removal default value (0). Why did you choose such a default parameter value??</li><li>Also the Specify minimum base quality required for clipping is by default 20. Some people prefer to lower down that threshold down to even 2, so they keep potentially informative reads here and non informative will be removed when mapping because of the mapping quality. What are you thoughts on that?
Thanks üôÇ</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:32:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:32:29"><div class="time">2022-01-19 14:32:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Uhh, can you clarify number 2 - do you mean quality clipping?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:32:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:32:39"><div class="time">2022-01-19 14:32:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You say adapter overlap twice</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:34:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:34:21"><div class="time">2022-01-19 14:34:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But 1) is a left over from eager 1. There was in house testing that suggested 1 worked best. I personally don't think it's a very good idea but no one has complained so far. It's more aggressive but cleans up reads more, I guess.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:35:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:35:32"><div class="time">2022-01-19 14:35:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For quality trimming 20 refers to base quality score. I've never heard people going down to 2, that doesn't make sense to me because that means you would keep extremely low confidence base calls in your reads. I've only heard people go higher than 20...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:38:08">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 14:38:08"><div class="time">2022-01-19 14:38:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You were right for 2... base quality üò¨</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:38:36">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 14:38:36"><div class="time">2022-01-19 14:38:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok for 1). Good to know</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:40:10">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 14:40:10"><div class="time">2022-01-19 14:40:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* for 2). I totally agree with you, and I will stick to this.
but here is  a quote from a Science paper (as usually digging into the Supplementary Material to get those details)...
<em>"AdapterRemoval v1.5.3</em>¬†(<em>60</em>) was used to trim Illumina adapter sequences, leading Ns (-- trimns) and trailing quality 2 runs (--trimqualities --minquality 2) from both single- and paired-end reads."</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:41:03">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 14:41:03"><div class="time">2022-01-19 14:41:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks James, so I will definitely increase the min adapter overlap and I'll stick to the min base quality of 20üôå</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:49:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:49:57"><div class="time">2022-01-19 14:49:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which paper is that? That looked like it could be a typo</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:50:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:50:14"><div class="time">2022-01-19 14:50:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Quality 2 runs sounds weird too</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 14:51:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 14:51:01"><div class="time">2022-01-19 14:51:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But yeah no harm in increasing the number of overlap at all.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 15:04:09">
            
                <img src="https://secure.gravatar.com/avatar/bdd3b29adc6689f9b072edf8403588fb.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">pierrespc
                     <span class="print-only user-email">(pierrespc@gmail.com)</span>
                </div>
                <a href="#2022-01-19 15:04:09"><div class="time">2022-01-19 15:04:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you want some names? haha
doi:10.1126/science.aav2621</p>

<p>Yes I was wondering too if it could be a typo. But Nico then told me it could be to follow the strategy I mentioned before of not be so stringent at that step and remove later when mapping</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 15:07:16">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-01-19 15:07:16"><div class="time">2022-01-19 15:07:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes, as far as I remember some people in CPH where using --minquality 2 for AdapterRemoval, I guess maximize number of reads that get mapped, despite the quality score‚Ä¶ But I agree on keeping quality filter at 20 for our analyses</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 21:06:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-19 21:06:13"><div class="time">2022-01-19 21:06:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I guess it depends how much coverage you have.</p>

<p>If you have low coverage it massively risks getting false SNP calls. If you have higher coverage is not as bad, maybe it's worth the trade off to get ultimately higher confidence</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-19 21:08:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-01-19 21:08:50"><div class="time">2022-01-19 21:08:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I agree‚Ä¶ It would depend on the case and how many supporting reads there are by position.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-20 10:01:10">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-01-20 10:01:10"><div class="time">2022-01-20 10:01:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But 2 is very low, it's a 63% base miscalling probability.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-20 10:34:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-01-20 10:34:23"><div class="time">2022-01-20 10:34:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I know, but if you check, several papers from CPH have used this parameter</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-20 11:54:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-01-20 11:54:30"><div class="time">2022-01-20 11:54:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @√Öshild (Ash) do you happen to have any insight on that (I know you don't really work in those groups, but maybe you've heard/been in discussions?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-20 12:02:24">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-01-20 12:02:24"><div class="time">2022-01-20 12:02:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think it is used because of something related to the error probability of base calling with the older Illumina HiSeqs, and that that was a parameter used by Illumina. But I‚Äôm not entirely sure.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-21 16:37:54">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-01-21 16:37:54"><div class="time">2022-01-21 16:37:54</div></a>
                <div class="msg">
                    <p>Have any UK-based people had a hard time getting KAPA HiFi HotStart ready mix? Do any UK-based suppliers sell it?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-21 17:15:51">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-01-21 17:15:51"><div class="time">2022-01-21 17:15:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is there any lab-based (or otherwise) product that UK is not short on currently?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-01-21 20:58:00">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-01-21 20:58:00"><div class="time">2022-01-21 20:58:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* lol fair. but I can‚Äôt even find a provider that‚Äôs ‚Äúout of stock‚Äù</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-26 17:25:24">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-02-26 17:25:24"><div class="time">2022-02-26 17:25:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If anyone ever comes across this post with the same question, I could not find a way to order it without directly contacting Roche and getting a quote and placing a sales order. Farzad is the man and can help you meet all of your KAPA needs: <a href="mailto:farzad.javad@roche.com">farzad.javad@roche.com</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 10:48:52">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-02-10 10:48:52"><div class="time">2022-02-10 10:48:52</div></a>
                <div class="msg">
                    <p>Hello experts! Would it theoretically be possible to identify the geographic origin of plant remains recovered from dental calculus (let's say Medieval/post-medieval) from their DNA? And what would the practical challenges be? (I'm assuming fragmented DNA and the overwhelming oral signal would cause issues?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 13:07:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-10 13:07:10"><div class="time">2022-02-10 13:07:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Theoretically yes.</p>

<p>You would need sufficient DNA reads/coverage on loci on the plant genome that you can do populatio ngenetics - assuming the given species has suffcient structure diverstiy that it can be linked with location</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 13:08:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-10 13:08:04"><div class="time">2022-02-10 13:08:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Primary issue would indeed be overwhelming oral signal meaning it's difficult to retrieve the plant reads</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 13:17:00">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-02-10 13:17:00"><div class="time">2022-02-10 13:17:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Theoretically is all I need for a grant application üòõ well, some grants...</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 15:19:51">
            
                <img src="https://avatars.slack-edge.com/2020-02-05/937654706020_e9e8576298b89a1ec13c_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zandra Fagern√§s
                     <span class="print-only user-email">(fagernaes@shh.mpg.de)</span>
                </div>
                <a href="#2022-02-10 15:19:51"><div class="time">2022-02-10 15:19:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I mean theoretically yes, but a lot of plants have enormous genomes with lots of repetitive elements, so without a huge amount of plant reads/a very successful capture, it will be... Challenging. To say the least.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üíØ James Fellows Yates, Nasreen Broomand
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 15:37:33">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-02-10 15:37:33"><div class="time">2022-02-10 15:37:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeeeeeaa thought so... Any insight on protein-markers that could be useful @Zandra Fagern√§s?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 16:35:16">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-02-10 16:35:16"><div class="time">2022-02-10 16:35:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Wouldn't you need a "genetic atlas" of plants to project your sample onto? I mean, in human genetics you can predict ancestry of an individual e.g. based on a PCA on genotypes from thousands of other human individuals with known ancestry</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 16:35:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-10 16:35:54"><div class="time">2022-02-10 16:35:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yup</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-10 16:36:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-10 16:36:13"><div class="time">2022-02-10 16:36:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's what I mean 'on loci on the plant genome that you can do pop-gen'</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-26 17:26:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-02-26 17:26:25"><div class="time">2022-02-26 17:26:25</div></a>
                <div class="msg">
                    <p>Hi, all. Does anyone know why the Meyer and Kircher 2010 blunt end direct ligation protocol is used preferentially with aDNA? Does anyone do blunt end lib preps with a-tailing and Y-shaped adapters with aDNA? Is blunt end direct ligation more efficient?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-26 17:49:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-26 17:49:14"><div class="time">2022-02-26 17:49:14</div></a>
                <div class="msg">
                    <p>I think the Orlando group may have experimented with this a bit: <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0078575">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0078575</a> (I don't think that directly addresses your question though)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">journals.plos.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0078575">Ligation Bias in Illumina Next-Generation DNA Libraries: Implications for Sequencing Ancient Genomes</a></div>
                                <div class="link-text">
                                    Ancient DNA extracts consist of a mixture of endogenous molecules and contaminant DNA templates, often originating from environmental microbes. These two populations of templates exhibit different chemical characteristics, with the former showing depurination and cytosine deamination by-products, resulting from post-mortem DNA damage. Such chemical modifications can interfere with the molecular tools used for building second-generation DNA libraries, and limit our ability to fully characterize the true complexity of ancient DNA extracts. In this study, we first use fresh DNA extracts to demonstrate that library preparation based on adapter ligation at AT-overhangs are biased against DNA templates starting with thymine residues, contrarily to blunt-end adapter ligation. We observe the same bias on fresh DNA extracts sheared on Bioruptor, Covaris and nebulizers. This contradicts previous reports suggesting that this bias could originate from the methods used for shearing DNA. This also suggests that AT-overhang adapter ligation efficiency is affected in a sequence-dependent manner and results in an uneven representation of different genomic contexts. We then show how this bias could affect the base composition of ancient DNA libraries prepared following AT-overhang ligation, mainly by limiting the ability to ligate DNA templates starting with thymines and therefore deaminated cytosines. This results in particular nucleotide misincorporation damage patterns, deviating from the signature generally expected for authenticating ancient sequence data. Consequently, we show that models adequate for estimating post-mortem DNA damage levels must be robust to the molecular tools used for building ancient DNA libraries.
                                </div>
                                
                                
    
        <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0078575">
            <img class="preview" src="https://journals.plos.org/plosone/article/figure/image?id=10.1371/journal.pone.0078575.g004&amp;size=inline"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0078575</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Christina Warinner
                        </div>
                    
                        <div class="message-reaction">
                        üëÄ Kelly Blevins
                        </div>
                    
                        <div class="message-reaction">
                        üôè:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 15:13:34">
            
                <img src="https://avatars.slack-edge.com/2021-11-06/2696551938660_c79c1f6ef245c5ff1040_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Christina Warinner
                     <span class="print-only user-email">(warinner@shh.mpg.de)</span>
                </div>
                <a href="#2022-02-28 15:13:34"><div class="time">2022-02-28 15:13:34</div></a>
                <div class="msg">
                    <p>Yes, many groups had serious problems with aDNA and TA ligation. Blunt end ligation was found to work better empirically</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÆ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 15:19:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-28 15:19:49"><div class="time">2022-02-28 15:19:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you know of any other papers that describe that?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 15:19:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-28 15:19:58"><div class="time">2022-02-28 15:19:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or is the Orlando one the main one?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 15:37:09">
            
                <img src="https://avatars.slack-edge.com/2021-11-06/2696551938660_c79c1f6ef245c5ff1040_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christina Warinner
                     <span class="print-only user-email">(warinner@shh.mpg.de)</span>
                </div>
                <a href="#2022-02-28 15:37:09"><div class="time">2022-02-28 15:37:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thats the published one</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 15:43:12">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 15:43:12"><div class="time">2022-02-28 15:43:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Has somebody tried the Santa Cruz Reaction Protocol?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 15:53:56">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-02-28 15:53:56"><div class="time">2022-02-28 15:53:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Katerina Guschanski maybe ask on <b>#general</b>?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 16:28:43">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-02-28 16:28:43"><div class="time">2022-02-28 16:28:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks so much, @James Fellows Yates and @Christina Warinner! This is good to know. I imagined there must be a reason why blunt end ligation is used almost exclusively in aDNA despite AT ligation being the mainstream method in genomics.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 17:59:42">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-02-28 17:59:42"><div class="time">2022-02-28 17:59:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Katerina Guschanski Yes, we (Tom Gilbert‚Äôs group) switched to using SCR for all aDNA work in 2020, it works really well in my experience üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:00:48">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 18:00:48"><div class="time">2022-02-28 18:00:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Brilliant! Any chance you have a protocol you'd be open to share? We'd like to test it in comparison to our standard double-barcode double-index protocol</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:07:17">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-02-28 18:07:17"><div class="time">2022-02-28 18:07:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There is a full protocol supplied by the authors in their supplementary material <a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jhered/112/3/10.1093_jhered_esab012/1/esab012_suppl_supplementary_material.pdf?Expires=1649091871&amp;Signature=QTyxLLQOb1QgZ-YTCdBng8xd9rC3rZul8qX7-dUD18KIFRN~GjkQyemPD0mRfv11fcwJQTjfxl9su8u8XvdC-xUKDf5C5fv8X5xi~D7tY0pW04NZNcZh5dhrEmo46Oc0Oc3nMQ0hvm0VqvfGAq9qmPkHccrQKtgzdtBPghfMBHcfsWYToYRAOq2OilKaQ8dCUJ0gm9ewLL9wh4vrrperSEKxdTaCe~amI-K-Vo0fD0HFN8EN32K7ecx1qD8M~ngq68dm73tAI9rt6ZzMPZzZ6nNeJC-xDI5o~ZbtlJ2DGPIyEjSoOhwNkIeGjBnb0gpWls-rtdg6NdyCNHg77r-X4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA|https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jhered/112/3/10.1093[‚Ä¶]ls-rtdg6NdyCNHg77r-X4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">https://oup.silverchair-cdn.com/oup/backfile/Content<em>public/Journal/jhered/112/3/10.1093</em>jhered<em>esab012/1/esab012</em>suppl<em>supplementary</em>material.pdf?Expires=1649091871&amp;Signature=QTyxLLQOb1QgZ-YTCdBng8xd9rC3rZul8qX7-dUD18KIFRN~GjkQyemPD0mRfv11fcwJQTjfxl9su8u8XvdC-xUKDf5C5fv8X5xi~D7tY0pW04NZNcZh5dhrEmo46Oc0Oc3nMQ0hvm0VqvfGAq9qmPkHccrQKtgzdtBPghfMBHcfsWYToYRAOq2OilKaQ8dCUJ0gm9ewLL9wh4vrrperSEKxdTaCe~amI-K-Vo0fD0HFN8EN32K7ecx1qD8M~ngq68dm73tAI9rt6ZzMPZzZ6nNeJC-xDI5o~ZbtlJ2DGPIyEjSoOhwNkIeGjBnb0gpWls-rtdg6NdyCNHg77r-X4A<strong>&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA|https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/jhered/112/3/10.1093[‚Ä¶]ls-rtdg6NdyCNHg77r-X4A</strong>&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA</a>  which is the same as the one we use üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:08:23">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 18:08:23"><div class="time">2022-02-28 18:08:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's rare that a protocol from the paper can be taken and applied without tweaks ü§™</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:08:35">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 18:08:35"><div class="time">2022-02-28 18:08:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks a lot, @√Öshild (Ash)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:09:26">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-02-28 18:09:26"><div class="time">2022-02-28 18:09:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The only major difference is that we do a SPRI bead clean-up after the library build instead of MinElute cleanup.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:09:47">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 18:09:47"><div class="time">2022-02-28 18:09:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Any specific reason why?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:10:03">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-02-28 18:10:03"><div class="time">2022-02-28 18:10:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* it‚Äôs faster and less pipetting</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:10:18">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 18:10:18"><div class="time">2022-02-28 18:10:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hahaha, true that!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:16:44">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-02-28 18:16:44"><div class="time">2022-02-28 18:16:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The protocol they provide in the paper is really good and explains the steps well , it‚Äôs true that many of the other protocol papers out there are often lacking in detailed explanations.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:17:43">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-02-28 18:17:43"><div class="time">2022-02-28 18:17:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's really comforting to know</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-02-28 18:44:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-02-28 18:44:12"><div class="time">2022-02-28 18:44:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We had issues with the Santa Cruz protocol. We got a lot of primer dimmers. @Maria Lopopolo or @Miren Iraeta Orbegozo (@Iraeta Miren) can give more details about the issues we had, as it was them that did it. @Betsy Nelson on the other hand has just spent some time in California learning the protocol and may be able to give some details on the tricks to make it work. But for us, our first shot was not very good.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-01 00:18:12">
            
                <img src="https://avatars.slack-edge.com/2022-06-06/3632216693492_e57e679aa24ba9eeb74d_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nasreen Broomand
                     <span class="print-only user-email">(nbrooman@ucsc.edu)</span>
                </div>
                <a href="#2022-03-01 00:18:12"><div class="time">2022-03-01 00:18:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nico Rascovan did you QC the adapter and splint oligos before using them?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-01 11:24:59">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-03-01 11:24:59"><div class="time">2022-03-01 11:24:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* we QC‚Äô d the splits and adapters. I think this library prep success it‚Äôs really dependant upon the type of sample and how you can fine tune beads clean-up and the c1-c5 dilutions e.g. more stringent beads to DNA ration post adapter libration or adding less adapters (e.g. one step below suggestion) at ligation step. I think Miren is now testing a few of these adjustments and was seeing less dimers  üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2022-02-28 18:44:20">
            
                <img src="https://avatars.slack-edge.com/2021-05-06/2044838635828_cfa0d915e91e7de3f161_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Miren Iraeta Orbegozo
                     <span class="print-only user-email">(miren.iraeta@palaeome.org)</span>
                </div>
                <a href="#2022-02-28 18:44:20"><div class="time">2022-02-28 18:44:20</div></a>
                <div class="msg">
                    <p>@Miren Iraeta Orbegozo has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-03 18:55:29">
            
                <img src="https://avatars.slack-edge.com/2022-08-23/3993422039521_96350c959aa22f50119c_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Olivia
                     <span class="print-only user-email">(smithsmitholivia@gmail.com)</span>
                </div>
                <a href="#2022-03-03 18:55:29"><div class="time">2022-03-03 18:55:29</div></a>
                <div class="msg">
                    <p>Hi folks ‚Äî I‚Äôm wondering if anyone else has been having trouble (or has solved the issue of) downloading ENA files which give FTP directory errors recently? Trying both ENA‚Äôs enaDataGet scripts and wget directly (and across different projects), I sometimes get <code>Error with FTP transfer: &amp;lt;urlopen error ftp error: error_perm('550 Failed to change directory.')&amp;gt;</code>. Googling suggests that this often happens when the directory doesn‚Äôt exist or perhaps in the case of permissions issues, which is not something I could resolve on my own ofc üòï</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-04 07:40:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-03-04 07:40:14"><div class="time">2022-03-04 07:40:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Olivia, can you post a few example commands to see if we can replicate this?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-04 07:40:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-03-04 07:40:39"><div class="time">2022-03-04 07:40:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The ENA servers are sometimes a bit shaky, so it could just be a connection issue - particularly if you're trying to download from the US.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-04 07:42:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-03-04 07:42:40"><div class="time">2022-03-04 07:42:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You could consider trying to download stuff with pipelines such as <code>nf-core/fetchNGS</code> which does some retrying mechanisms for you.</p>

<p>Or even try: <a href="https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump">https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump</a></p>

<p>which should be able to download ENA data but from the SRA mirror of the ENA (as SRA/ENA basically mirror each other and has the same data)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump">HowTo: fasterq dump ¬∑ ncbi/sra-tools Wiki</a></div>
                                <div class="link-text">
                                    SRA Tools. Contribute to ncbi/sra-tools development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump">
            <img class="preview" src="https://opengraph.githubassets.com/575106097ba23dd3806cf3c4e5e5a78d6f39c44567332e6533c73c7e83ce6403/ncbi/sra-tools"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-03-04 22:47:27">
            
                <img src="https://avatars.slack-edge.com/2022-08-23/3993422039521_96350c959aa22f50119c_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Olivia
                     <span class="print-only user-email">(smithsmitholivia@gmail.com)</span>
                </div>
                <a href="#2022-03-04 22:47:27"><div class="time">2022-03-04 22:47:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ooh thank you for the tool recommendations, those are good to have and I‚Äôll check them out! The good news is that I had a labmate test it and I also tried it locally, both of which succeeded, so ENA isn‚Äôt the issue. The bad news is that my profile on our advanced computing cluster is still being rejected! But that‚Äôs a problem for our computing team, so I‚Äôll leave it at that. /endthread</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-04 09:27:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2022-04-04 09:27:35"><div class="time">2022-04-04 09:27:35</div></a>
                <div class="msg">
                    <p>Hey all, wondering if anyone has got experience working with sourcepredict, and might be able to help me? Just keep getting an error and I don't know what I'm doing wrong.</p>

<p>It seems like step 1 runs smoothly and it tell me something like:</p><ul><li>Sample: 7309-02<em>minikraken</em>All_report
known:98.61%
unknown:1.39%</li>
</ul>

<p>But then it tries to run step 2, and it keeps saying:
Traceback (most recent call last):
  File "/home/vanme090/.local/bin/sourcepredict", line 8, in &lt;module&gt;
   sys.exit(main())
  File "/home/vanme090/.local/lib/python3.8/site-packages/sourcepredict/<strong>main</strong>.py", line 172, in main
   sm.compute<em>distance(distance</em>method=distance<em>method, rank=RANK)
  File "/home/vanme090/.local/lib/python3.8/site-packages/sourcepredict/sourcepredictlib/ml.py", line 260, in compute</em>distance
   tree = ncbi.get<em>topology(
  File "/home/vanme090/.local/lib/python3.8/site-packages/ete3/ncbi</em>taxonomy/ncbiquery.py", line 463, in get_topology
   root = elem2node[1]
KeyError: 1</p>

<p>I've tried different Python packages, but they all give me the same error. To get the taxid sample files, I run kraken with (tried both the standard and the minikraken<em>v2</em>8GB databases). It looks like ete3 can't get the ncbi taxonomy, and therefore it also crashes further down the line?  Anyone got any ideas what I'm missing/doing wrong?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-04 09:28:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-04-04 09:28:00"><div class="time">2022-04-04 09:28:00</div></a>
                <div class="msg">
                    <p>@Maxime Borry ‚òùÔ∏è</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-10 02:22:58">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2022-04-10 02:22:58"><div class="time">2022-04-10 02:22:58</div></a>
                <div class="msg">
                    <p>Hello all,
I am looking to test for positive selection using the branch-site model in codeml (PAML), but I am a bit confused with some parameters in the control file (I've understood the main ones including NSsites and model but I am unsure about some of the others). I was wondering if anyone had experience with this, and if they would be happy to have a quick chat or message exchange to confirm my understanding?
Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-20 14:24:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-04-20 14:24:10"><div class="time">2022-04-20 14:24:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ooofff, not heard of that tool unfortunately...</p>

<p>@aidanva @Arthur Kocher @Meriam Guellil? I guess you guys work with phylogenies relatively regularly... any ideas?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-20 14:35:45">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-04-20 14:35:45"><div class="time">2022-04-20 14:35:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sorry have never used that tool either üò∂</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-20 17:34:06">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2022-04-20 17:34:06"><div class="time">2022-04-20 17:34:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Me neither </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-21 19:28:51">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2022-04-21 19:28:51"><div class="time">2022-04-21 19:28:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all for you replies, and no worries üôÇ I will go by trial and error and see where it leads üòä</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-04-29 15:34:26">
            
                <img src="https://secure.gravatar.com/avatar/631e0c1a4a0b8aaa3f55b50da5a07bb4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Audrey Lin
                     <span class="print-only user-email">(linat@si.edu)</span>
                </div>
                <a href="#2022-04-29 15:34:26"><div class="time">2022-04-29 15:34:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Ophelie! I think that Antony had worked with PAML before</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Oph√©lie Lebrasseur
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 10:43:56">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2022-05-03 10:43:56"><div class="time">2022-05-03 10:43:56</div></a>
                <div class="msg">
                    <p>Hi all! has anyone here worked with Metaphlan3? I just analysed a sample but got no classification at all:
```#SampleID       Metaphlan_Analysis</p>

<h1>clade<em>name     NCBI</em>tax<em>id     relative</em>abundance      additional_species</h1>

<p>UNKNOWN -1      100.0```
but I got classification results with Metaphlan2, has anyone had this same issue?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 10:52:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-03 10:52:52"><div class="time">2022-05-03 10:52:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I know @Alex H√ºbner</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 11:12:19">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-03 11:12:19"><div class="time">2022-05-03 11:12:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, I had some sediment samples that still had something like five species for MP2 but no species in MP3. They completely changed the marker-gene database and there are some species that were dropped during this process.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 11:28:39">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2022-05-03 11:28:39"><div class="time">2022-05-03 11:28:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* oh really‚Ä¶. that doesn‚Äôt sound good at all‚Ä¶ It‚Äôs weird though that in my case M2 reported some kind of common fungi but M3 didn‚Äôt detect anything‚Ä¶ let me show you a snippet:
<code>#SampleID       Metaphlan2_Analysis
k__Eukaryota    78.741
k__Bacteria     21.259
k__Eukaryota|p__Ascomycota      78.5878
k__Bacteria|p__Proteobacteria   19.33983
k__Bacteria|p__Actinobacteria   1.91918
k__Eukaryota|p__Apicomplexa     0.1532
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes    67.16866
k__Bacteria|p__Proteobacteria|c__Alphaproteobacteria    19.33983
k__Eukaryota|p__Ascomycota|c__Schizosaccharomycetes     7.90379
k__Eukaryota|p__Ascomycota|c__Sordariomycetes   3.51535
k__Bacteria|p__Actinobacteria|c__Actinobacteria 1.91918
k__Eukaryota|p__Apicomplexa|c__Coccidia 0.1532
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Eurotiales      66.18799
k__Bacteria|p__Proteobacteria|c__Alphaproteobacteria|o__Rhizobiales     19.33983
k__Eukaryota|p__Ascomycota|c__Schizosaccharomycetes|o__Schizosaccharomycetales  7.90379
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Hypocreales    2.03864
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales      1.91918
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Sordariales    1.4767
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Onygenales      0.98067
k__Eukaryota|p__Apicomplexa|c__Coccidia|o__Eucoccidiorida       0.1532
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Eurotiales|f__Aspergillaceae    66.10824
k__Bacteria|p__Proteobacteria|c__Alphaproteobacteria|o__Rhizobiales|f__Bradyrhizobiaceae        19.33983
k__Eukaryota|p__Ascomycota|c__Schizosaccharomycetes|o__Schizosaccharomycetales|f__Schizosaccharomycetaceae      7.90379
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Hypocreales|f__Nectriaceae     2.03864
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Sordariales|f__Chaetomiaceae   1.4767
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Pseudonocardiaceae        1.25179
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Onygenales|f__Ajellomycetaceae  0.74471
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Propionibacteriaceae      0.66739
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Onygenales|f__Onygenales_noname 0.23595
k__Eukaryota|p__Apicomplexa|c__Coccidia|o__Eucoccidiorida|f__Eimeriidae 0.1532
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Eurotiales|f__Trichocomaceae    0.07975
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Eurotiales|f__Aspergillaceae|g__Aspergillaceae_unclassified     66.10824
k__Bacteria|p__Proteobacteria|c__Alphaproteobacteria|o__Rhizobiales|f__Bradyrhizobiaceae|g__Bradyrhizobium      19.33983
k__Eukaryota|p__Ascomycota|c__Schizosaccharomycetes|o__Schizosaccharomycetales|f__Schizosaccharomycetaceae|g__Schizosaccharomyces       7.90379
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Hypocreales|f__Nectriaceae|g__Fusarium 2.03864
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Sordariales|f__Chaetomiaceae|g__Chaetomiaceae_unclassified     1.4767
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Pseudonocardiaceae|g__Saccharopolyspora   1.25179
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Onygenales|f__Ajellomycetaceae|g__Ajellomyces   0.74471
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Propionibacteriaceae|g__Propionibacterium 0.66739
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Onygenales|f__Onygenales_noname|g__Onygenales_noname_unclassified       0.23595
k__Eukaryota|p__Apicomplexa|c__Coccidia|o__Eucoccidiorida|f__Eimeriidae|g__Eimeria      0.1532
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Eurotiales|f__Trichocomaceae|g__Talaromyces     0.07975
k__Bacteria|p__Proteobacteria|c__Alphaproteobacteria|o__Rhizobiales|f__Bradyrhizobiaceae|g__Bradyrhizobium|s__Bradyrhizobium_sp_DFCI_1  19.33983
k__Eukaryota|p__Ascomycota|c__Schizosaccharomycetes|o__Schizosaccharomycetales|f__Schizosaccharomycetaceae|g__Schizosaccharomyces|s__Schizosaccharomyces_unclassified   7.90379
k__Eukaryota|p__Ascomycota|c__Sordariomycetes|o__Hypocreales|f__Nectriaceae|g__Fusarium|s__Fusarium_unclassified        2.03864
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Pseudonocardiaceae|g__Saccharopolyspora|s__Saccharopolyspora_unclassified 1.25179
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Onygenales|f__Ajellomycetaceae|g__Ajellomyces|s__Ajellomyces_unclassified       0.74471
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Propionibacteriaceae|g__Propionibacterium|s__Propionibacterium_acnes      0.66739
k__Eukaryota|p__Apicomplexa|c__Coccidia|o__Eucoccidiorida|f__Eimeriidae|g__Eimeria|s__Eimeria_tenella   0.1532
k__Eukaryota|p__Ascomycota|c__Eurotiomycetes|o__Eurotiales|f__Trichocomaceae|g__Talaromyces|s__Talaromyces_unclassified 0.07975
k__Bacteria|p__Proteobacteria|c__Alphaproteobacteria|o__Rhizobiales|f__Bradyrhizobiaceae|g__Bradyrhizobium|s__Bradyrhizobium_sp_DFCI_1|t__GCF_000465325 19.33983
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Propionibacteriaceae|g__Propionibacterium|s__Propionibacterium_acnes|t__Propionibacterium_acnes_unclassified      0.66739
k__Eukaryota|p__Apicomplexa|c__Coccidia|o__Eucoccidiorida|f__Eimeriidae|g__Eimeria|s__Eimeria_tenella|t__GCA_000002835  0.1532</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 11:42:52">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-03 11:42:52"><div class="time">2022-05-03 11:42:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am not familiar about the changes with respect to the eukaryotes because I don‚Äôt use MetaPhlAn for this purpose. However, you have a lot of <code>unclassified</code> assignments and I also saw that I more or less lost all species in MP3 that were named <code>unclassified</code> in MP2.
In the end, it‚Äôs up to you to decide what you want to gain from this analysis. MP3 has a higher number of species-specific marker genes, but this might lead to be less sensitive of picking up traces of species. Kraken2 might be a good alternative for this.
There has been recently a new preprint that evaluates the databases for both programs. I haven‚Äôt read it myself yet, but it might give some inspiration: <a href="https://twitter.com/RobynJWright/status/1519826469610041344?t=e14DGzJS87jjE6EWY7fpOQ&amp;s=09">https://twitter.com/RobynJWright/status/1519826469610041344?t=e14DGzJS87jjE6EWY7fpOQ&amp;s=09</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">twitter</div>
                            
                                <div class="attachment-author">
                                    
                                        <img src="https://pbs.twimg.com/profile_images/1224144748681629696/YmECnAjl_normal.jpg" class="icon">
                                    }
                                    <a href="https://twitter.com/RobynJWright/status/1519826469610041344">
                                    Robyn Wright
                                    </a><span class="print-only">(https://twitter.com/RobynJWright/status/1519826469610041344)</span>
                                </div>
                            
                            
                                
                                <div class="link-title"><a href=""></a></div>
                                <div class="link-text">
                                    Q: Which is better for taxonomic classification of &lt;b&gt;#metagenomics&lt;/b&gt; samples - Kraken2 or MetaPhlAn 3?

A: It really depends! 

Read the very short story in this thread, or the full story in my preprint w/ &lt;a href=&#34;https://twitter.com/BetaScience&#34;&gt;@BetaScience&lt;/a&gt; and Andr√© Comeau: &lt;https://bit.ly/3EWkYJf&gt;

1/
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://twitter.com/RobynJWright/status/1519826469610041344?t=e14DGzJS87jjE6EWY7fpOQ&amp;amp;s=09</div>
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://a.slack-edge.com/80588/img/services/twitter_pixel_snapped_32.png" class="icon" />
                                        Twitter
                                    </div>
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 11:45:30">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2022-05-03 11:45:30"><div class="time">2022-05-03 11:45:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* awesome, thanks for sharing @Alex H√ºbner üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-03 11:49:27">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-03 11:49:27"><div class="time">2022-05-03 11:49:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No problem!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 13:39:50">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-05 13:39:50"><div class="time">2022-05-05 13:39:50</div></a>
                <div class="msg">
                    <p>Can anyone recommend shotgun metagenomes for lab air and/or lab surfaces (not necessarily an ultra-clean lab) that I can use to run through sourcetracker?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 14:57:12">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-05-05 14:57:12"><div class="time">2022-05-05 14:57:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Bjorn, we used Salter et al. 2014 for this purpose (Accession numbers ERR584320
ERR584333
ERR584341
ERR584348)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 15:35:22">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-05 15:35:22"><div class="time">2022-05-05 15:35:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks! üòÅ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:02:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:02:31"><div class="time">2022-05-05 16:02:31</div></a>
                <div class="msg">
                    <p>Hi everyone ! I will perform a metagenomic analysis on several individuals already sequenced for demographic studies in my lab. What threshold should I use to create the sample list? We thought about only keeping the samples that were 1% endogenous human because this is the threshold we use to decide to sequence more or not in general. I used a threshold of 1X depth human for former projects and we realised that that threshold was too high but I fear that 1% endogenous human might be too low to be able to detect the actual presence of, say, a pathogen.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Diana Ivette Cruz D√°valos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:12:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:12:47"><div class="time">2022-05-05 16:12:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What type of metagenomic analysis do you want to do? For pathogen detection, generally speaking I don't orient myself on human DNA content. You can have horrible human genome coverage but still get a good pathogen hit. Depends on the sample. But maybe others have used similar thresholds before?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üíØ James Fellows Yates, √Öshild (Ash)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:15:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:15:01"><div class="time">2022-05-05 16:15:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Meriam! I think that for the moment I will focus on pathogens because I don‚Äôt really know how to analyse a microbiota yet</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:18:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:18:32"><div class="time">2022-05-05 16:18:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Does it mean that when you look for pathogens, you analyse even the individuals that were just screened ? May I ask, what threshold do you use then to know when to capture or sequence more of the pathogen you believe is in there ? Or what would be a good pathogen hit ? (Sorry for the basic questions)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:18:48">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:18:48"><div class="time">2022-05-05 16:18:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you just want to look for pathogen I would just have a look at all the samples you have frankly. As for the likelihood of detection it depends on the sequencing depth and the organism</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Diana Ivette Cruz D√°valos, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:20:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:20:40"><div class="time">2022-05-05 16:20:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So far we have used KrakenUniq as a first step in our pipeline with a threshold of 200 taxReads and 1000 kmers, so I fear that the pathogens in the individuals that were just screened would fall under this threshold because of the low depth</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:23:04">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:23:04"><div class="time">2022-05-05 16:23:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So with KU you actually get a bunch of stats which can inform you on the hit quality. Imposing a set threshold for reads and kmers like this could result in missing lower hits such as viruses.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò∞ Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:23:49">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:23:49"><div class="time">2022-05-05 16:23:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* in the original KU paper they use the kmer/reads ratio and I also normalise by coverage</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:26:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:26:23"><div class="time">2022-05-05 16:26:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Kmer/reads ratio sounds good because it accounts for the fact that you don‚Äôt have many reads sometimes. And then you normalise by coverage. Sounds good, thank you very much ! I will try to do that üòä</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:26:52">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-05 16:26:52"><div class="time">2022-05-05 16:26:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* you can check my haemophilus paper from this year if you want to do that</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëå Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:59:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-05 16:59:46"><div class="time">2022-05-05 16:59:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Like @Meriam Guellil said:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-05 16:59:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-05 16:59:49"><div class="time">2022-05-05 16:59:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03E88R83GB/6f1a2q.jpg?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">6f1a2q.jpg</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03E88R83GB/6f1a2q.jpg?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03E88R83GB-8fb372bd8d/6f1a2q_360.jpg?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="256" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üòÇ Meriam Guellil, Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        üëÜ:skin_tone_2: Meriam Guellil
                        </div>
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon, √Öshild (Ash)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 11:45:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 11:45:59"><div class="time">2022-05-06 11:45:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Meriam Guellil I could see from your presentation at SPAAM last year and your paper that you used E-value = (kmers / reads) ** cov = 0.001 threshold for filtering your KrakenUniq output. Could you please elaborate on why this kind of combination of KU stats and why this E-value threshold?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 12:03:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 12:03:32"><div class="time">2022-05-06 12:03:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Nikolay Oskolkov ! So the kmer/reads ratio is from the original paper of KU and you can expect the reason why it is used is that for a good hit you should have n times more unque kmers than reads. So for example if what you have is mostly overtiling it should technically be represented here. I started testing around in late 2019 (i think...) because I was switching from K2 to KU and I wasn't too happy in using reads or kmers for colouring heatmaps for example because depending on the genome size/structure etc these values or cutoffs for them could be widely different. From the tests I did back then this combination of stats ended up working best for me. Now keep in mind I use it almost exclusively for pathogen detection. So this gives me a nice cut-off value from when hits might be worth investigating even if they are low coverage and also eliminates noise in my heatmaps. The thresholds I have used are variable. 0.001 is usually as low as I go, mainly in order to still catch very low viral hits. There might be better combinations out there though, this is just what I ended up using and for my purposes it has done a great job.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 13:28:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 13:28:11"><div class="time">2022-05-06 13:28:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see, thank you @Meriam Guellil! I can understand the intuition behind constructing a kmers / reads variable for filtering because "kmers" and "reads" are pretty correlated so perhaps one of them can be treated as a redundant, so one could replace the two by a new one which is their ratio. However, I do not fully understand why to multiply this ratio by "cov"? Why multiply and not sum or divide? Also, why "cov" and not "dup"? I just want to understand the intuition behind this combination of KU stats that you used</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 13:34:50">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 13:34:50"><div class="time">2022-05-06 13:34:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Like I said I did try a couple of variant including dup but dup is already taken into account with the unique kmer count. The reason i use cov is because while the ration kmer / reads gives you a good indication it doesn't account for how much of the kmer dictionary for the taxon is actually covered, since the kmer numbers can differ and that way I can visualize and filter them irrespective of size</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 13:37:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 13:37:35"><div class="time">2022-05-06 13:37:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And like I said this is what I came up with but I am sure there a multiple and maybe better ways to pair the stats</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 13:56:22">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 13:56:22"><div class="time">2022-05-06 13:56:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Are you using the total kmer count for each taxon, or the unique kmer count for each taxon @Meriam Guellil?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 13:58:10">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 13:58:10"><div class="time">2022-05-06 13:58:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* unique kmer count. With total kmer count do you mean for the lower ranks as well?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:04:41">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:04:41"><div class="time">2022-05-06 14:04:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* kmer (including duplicated ones) assigned directly to a taxon</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:06:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:06:29"><div class="time">2022-05-06 14:06:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* KU lists number of distinct or unique kmers</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:08:28">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:08:28"><div class="time">2022-05-06 14:08:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So unique kmers follow read counts (at least according to KU paper fig 3. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0/figures/3">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0/figures/3</a>)
So shouldn‚Äôt K/R be more or less a constant ?</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0/figures/3">KrakenUniq: confident and fast metagenomics classification using unique k-mer counts - Genome Biology</a></div>
                                <div class="link-text">
                                    False-positive identifications are a significant problem in metagenomics classification. We present KrakenUniq, a novel metagenomics classifier that combines the fast k-mer-based classification of Kraken with an efficient algorithm for assessing the coverage of unique k-mers found in each species in a dataset. On various test datasets, KrakenUniq gives better recall and precision than other methods and effectively classifies and distinguishes pathogens with low abundance from false positives in infectious disease samples. By using the probabilistic cardinality estimator HyperLogLog, KrakenUniq runs as fast as Kraken and requires little additional memory. KrakenUniq is freely available at &lt;https://github.com/fbreitwieser/krakenuniq&gt; .
                                </div>
                                
                                
    
        <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0/figures/3">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/13059.jpg"
                width="177"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0/figures/3</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:12:10">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 14:12:10"><div class="time">2022-05-06 14:12:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maxime Borry this is what I mean when I say that those two are correlated, so if you know one you will find the other one, so no need (perhaps) to filter with respect to both of them, one of them (or their combination such as ratio) is good enough for filtering</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:13:23">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:13:23"><div class="time">2022-05-06 14:13:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But that made me check the definition of ‚Äúcoverage‚Äù for K2 vs KU @Meriam Guellil, and that‚Äôs actually a different one:
‚Ä¢ in KU, they use <code>unique_kmers_for_clade / genome_size</code> (<a href="https://github.com/fbreitwieser/krakenuniq/blob/2ac22bf7681223efa17ffba221231c7faac9da05/src/taxdb.hpp#L1107">https://github.com/fbreitwieser/krakenuniq/blob/2ac22bf7681223efa17ffba221231c7faac9da05/src/taxdb.hpp#L1107</a>)
‚Ä¢ in K2, they don‚Äôt provide it, but imply that it would be <code>uniq_kmers_for_clade/total_kmers_for_clade</code> <a href="https://github.com/DerrickWood/kraken2/pull/249#issuecomment-638311769">https://github.com/DerrickWood/kraken2/pull/249#issuecomment-638311769</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:14:41">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:14:41"><div class="time">2022-05-06 14:14:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* but you can have a high read count while still having a low unique kmer count which is what they point out in the paper, so the ratio for sure makes a difference. you can just use the base ratio it one its own too</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:17:24">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:17:24"><div class="time">2022-05-06 14:17:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes I agree with Meriam, normally the good hits have way more kmers than reads and when it‚Äôs not the case I doubt more the validity of the hit</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:19:57">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:19:57"><div class="time">2022-05-06 14:19:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* what can happen though is that you have a super high coverage genome in a utopic world and you have the whole genome covered multiple times in which case the unque kmer count wouldn't go up anymore but the read count will. So if the cov is 1, then there is no point.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:21:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:21:33"><div class="time">2022-05-06 14:21:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Fortunately, we are not used to work with utopic data üòÇ but yes you are right üëç</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 14:21:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 14:21:42"><div class="time">2022-05-06 14:21:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* exactly üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:21:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 15:21:09"><div class="time">2022-05-06 15:21:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see. But in that case, would it make sense to give more emphasis to the coverage ?
Maybe something like <code>E = K/R ** double_exp(C)</code>
This way for:
‚Ä¢ <strong>low coverage, high duplication:</strong> <code>K&amp;lt;&amp;lt;R</code> so <code>K/R -&amp;gt; 0</code>,  <code>C~=0</code> so do<code>double_exp(C) ~ 1</code>, then <code>E-&amp;gt;0</code> 
‚Ä¢ l<strong>ow coverage, low duplication:</strong> <code>K/R -&amp;gt; ~(read_len - kmer_len),</code> <code>double_exp(C) ~ 1</code> , <code>E-&amp;gt;~(read_len-kmer_len)</code>  because you have up to <code>read_len - kmer_len</code> kmers of length <code>kmer_len</code> in a read of length <code>read_len</code>  (<code>read_len - kmer_len + 1</code> to be precise)
‚Ä¢ <strong>high coverage</strong> (up to 1, because KU uses uniq kmer for ‚Äúcoverage‚Äù computation), <strong>low duplication:</strong> <code>K/R -&amp;gt; 1</code> ,<code>double_exp(C) -&amp;gt; inf</code> , <code>E-&amp;gt; inf</code> 
The assumption being that for low coverage, the K/R ratio is what matters, for high coverage, the coverage is what matters</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:28:07">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-05-06 15:28:07"><div class="time">2022-05-06 15:28:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sounds good! I am/was lacking the mathematical background to go deeper into the rabbit hole I am afraid  so happy to see this improved. I will have a look üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:30:08">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 15:30:08"><div class="time">2022-05-06 15:30:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Interesting suggestion @Maxime Borry, but why to collapse them into a single variable, why not to filter with respect to each of them separately? The thing is that 2 out of the three variables seem to be orthogonal / independent. Then constructing a single variable out of two orthogonal might reduce the quality of filtering</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:31:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 15:31:14"><div class="time">2022-05-06 15:31:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I mean to filter in 3-dimensional space is probably more intuitive than in 1-dimensional</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:33:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 15:33:25"><div class="time">2022-05-06 15:33:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We can construct different functional combinations out of the 3 KU stats, with different asymptotics, but do we really need to have one single threshold for filtering?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:37:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 15:37:25"><div class="time">2022-05-06 15:37:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For example the way GATK filters good from poor quality variants, it uses 17 QC metrics and implements different thresholds for each metric. In our case, for example, we can compute distributions of kmers, cov, reads, taxReads and dup and exclude organisms in the tails of the distributions </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 15:47:49">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-06 15:47:49"><div class="time">2022-05-06 15:47:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Next, one can e.g. compute a PCA on the kmers, reads, taxReads, cov and dup stats, where data points in the PCA plot are the oganisms, and one could perhaps detect clusters of organisms with good QC metrics and poor QC metrics. That would be a multivariate filtering approach. Fir example, a PC1 could be a linear combination of the 4-5 KU stats that can be used in case one definitely wants just one single threshold. But why not to filter in 4-5 dimensional space? It is not too many dimensions to tackle</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 16:44:31">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 16:44:31"><div class="time">2022-05-06 16:44:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, that‚Äôs another interesting approach</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 16:48:02">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 16:48:02"><div class="time">2022-05-06 16:48:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Acutally, just realised that it would need a ‚Äúdouble exponential‚Äù function for the coverage, because KU cov will be a max of 1 (since it‚Äôs using unique kmers).
<a href="https://en.wikipedia.org/wiki/Double_exponential_function">https://en.wikipedia.org/wiki/Double_exponential_function</a></p>
                    <div class="message-attachment"
                            >
                            
                            
                                <div class="attachment-author">
                                    
                                        <img src="" class="icon">
                                    }
                                    <a href="https://en.wikipedia.org/">
                                    Wikipedia
                                    </a><span class="print-only">(https://en.wikipedia.org/)</span>
                                </div>
                            
                            
                                
                                <div class="link-title"><a href="https://en.wikipedia.org/wiki/Double_exponential_function">Double exponential function</a></div>
                                <div class="link-text">
                                    A double exponential function is a constant raised to the power of an exponential function.  The general formula is 
  
    
      
        f
        (
        x
        )
        =
        
          a
          
            
              b
              
                x
              
            
          
        
        =
        
          a
          
            (
            
              b
              
                x
              
            
            )
          
        
      
    
    {\displaystyle f(x)=a^{b^{x}}=a^{(b^{x})}}
   (where a&amp;gt;1 and b&amp;gt;1), which grows much more quickly than an exponential function. For example, if a = b = 10:

f(0) = 10
f(1) = 1010
f(2) = 10100 = googol
f(3) = 101000
f(100) = 1010100 = googolplex.Factorials grow faster than exponential functions, but much more slowly than doubly exponential functions. However, tetration and the Ackermann function grow faster. See Big O notation for a comparison of the rate of growth of various functions.
The inverse of the double exponential function is the double logarithm log(log(x)).
                                </div>
                                
                                
    
        <a href="https://en.wikipedia.org/wiki/Double_exponential_function">
            <img class="preview" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/52/Double_Exponential_Function.svg/1200px-Double_Exponential_Function.svg.png"
                width="1200"
                 height="943" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://en.wikipedia.org/wiki/Double_exponential_function</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç:skin_tone_2: Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-06 16:57:14">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-06 16:57:14"><div class="time">2022-05-06 16:57:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I made a small notebook to compare the effect of adding the double-exp to the coverage</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03EGB8V3BN/comparing_kraken_evalues.html?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">comparing_kraken_evalues.html</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üôå:skin_tone_2: Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-10 15:52:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-05-10 15:52:54"><div class="time">2022-05-10 15:52:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sorry but I am not sure to understand the final plots @Maxime Borry. Does it mean that Meriam‚Äôs e-value is better than the double exponential or the opposite ? I don‚Äôt see the lines for case c and d on the penultimate plot and for the last plots, two cases are called b so I‚Äôm confused üôàüòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-10 17:07:23">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-10 17:07:23"><div class="time">2022-05-10 17:07:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I made a blog post out of it üòâ
Hopefully it‚Äôs clearer
<a href="https://maximeborry.com/post/kraken-uniq/">https://maximeborry.com/post/kraken-uniq/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Maxime Borry</div>
                            
                            
                                
                                <div class="link-title"><a href="https://maximeborry.com/post/kraken-uniq/">A new E-score for KrakenUniq | Maxime Borry</a></div>
                                <div class="link-text">
                                    How to distinguish between true and false positive assignations with KrakenUniq
                                </div>
                                
                                
    
        <a href="https://maximeborry.com/post/kraken-uniq/">
            <img class="preview" src="https://maximeborry.com/post/kraken-uniq/featured.png"
                width="376"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://maximeborry.com/post/kraken-uniq/</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        üëç:skin_tone_2: Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-19 21:42:45">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2022-05-19 21:42:45"><div class="time">2022-05-19 21:42:45</div></a>
                <div class="msg">
                    <p>Hello all,
I was wondering if anyone has had experience with setting up Path Sampling for model selection with Beast2? I'm completely lost in how you're meant to alter the xml file (<a href="http://www.beast2.org/path-sampling/">http://www.beast2.org/path-sampling/</a>). I did manage to run a Nested Sampling analysis, but it's quite unstable and keeps crashing, hence why I am thinking of trying the Path Sampling approach. If anyone has any experience, I'm game! Thanks üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:52:31">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 10:52:31"><div class="time">2022-05-25 10:52:31</div></a>
                <div class="msg">
                    <p>Hi all! Is there a list somewhere of bacterial species and whether they are aerobes, anaerobes, etc?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:54:16">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 10:54:16"><div class="time">2022-05-25 10:54:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I use the bacdive API to ask for this, but it's a bit fiddly.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:54:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 10:54:32"><div class="time">2022-05-25 10:54:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or the bacdive website itself if you only want to check a few manually</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:55:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 10:55:10"><div class="time">2022-05-25 10:55:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://bacdive.dsmz.de/">https://bacdive.dsmz.de/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">bacdive.dsmz.de</div>
                            
                            
                                
                                <div class="link-title"><a href="https://bacdive.dsmz.de/">BacDive | The Bacterial Diversity Metadatabase</a></div>
                                <div class="link-text">
                                    The Bacterial Diversity Metadatabase provides strain-linked information about bacterial and archaeal biodiversity.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://bacdive.dsmz.de/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:55:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 10:55:35"><div class="time">2022-05-25 10:55:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There is also a R package called bacdiveR but it may not work anymore (even though it was very good)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:55:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 10:55:57"><div class="time">2022-05-25 10:55:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You will need to make a consensus across multiple strains of a species in some cases</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:59:31">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 10:59:31"><div class="time">2022-05-25 10:59:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks! I'll try the API and R package and see how it goes</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 10:59:58">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2022-05-25 10:59:58"><div class="time">2022-05-25 10:59:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can confirm that bacdiveR is currently broken</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò≠ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:00:54">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 11:00:54"><div class="time">2022-05-25 11:00:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, that'll save me some time</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:01:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:01:15"><div class="time">2022-05-25 11:01:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, you have to do it more manually then but it is possible still within R, there are various packages for dealing with rest APIs. You just have to wrangle list vectors a lot into tables manually</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Bjorn Bartholdy
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:12:55">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 11:12:55"><div class="time">2022-05-25 11:12:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Any experience with BacDive? It looks like it's only on <a href="https://r-forge.r-project.org/R/?group_id=1573">https://r-forge.r-project.org/R/?group_id=1573</a> and there's not much other info I can find</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">r-forge.r-project.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://r-forge.r-project.org/R/?group_id=1573">R-Forge: opm and opmdata: R Development Page</a></div>
                                <div class="link-text">
                                    Analysing phenotype microarrays and growth curves: plots, curve parameters, discretization, metadata, simultaneous tests, pathways, genome annotation, feature selection, textual reports, YAML, database I/O; see also &lt;a href=&#34;http://opm.dsmz.de&#34;&gt;opm.dsmz.de&lt;/a&gt; and &lt;a href=&#34;http://www.goeker.org/opm&#34;&gt;www.goeker.org/opm&lt;/a&gt;
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://r-forge.r-project.org/R/?group_id=1573</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:13:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:13:46"><div class="time">2022-05-25 11:13:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://api.bacdive.dsmz.de/">https://api.bacdive.dsmz.de/</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:15:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:15:02"><div class="time">2022-05-25 11:15:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh that's the same thing sorry</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:15:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:15:08"><div class="time">2022-05-25 11:15:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's new, not used it before sorry</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:15:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:15:12"><div class="time">2022-05-25 11:15:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But looks good!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:15:36">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 11:15:36"><div class="time">2022-05-25 11:15:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* At least it looks like it's being actively maintained</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:15:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:15:46"><div class="time">2022-05-25 11:15:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:15:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:15:57"><div class="time">2022-05-25 11:15:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maybe you could write a little tutorial if you get it to work</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:16:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:16:06"><div class="time">2022-05-25 11:16:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would be curious to know how it works</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:17:07">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 11:17:07"><div class="time">2022-05-25 11:17:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sure, that'll force me to document my steps properly üòÖ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:17:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:17:31"><div class="time">2022-05-25 11:17:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could even put it on the &lt;#C02D3DJP3MY|spaam-blog></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:17:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:17:43"><div class="time">2022-05-25 11:17:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think others would be interested to know how it works</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:18:08">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 11:18:08"><div class="time">2022-05-25 11:18:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I assumed that was the reason you asked? üòâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò¨ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:18:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:18:39"><div class="time">2022-05-25 11:18:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Ele @Shreya ‚òùÔ∏è</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:19:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:19:05"><div class="time">2022-05-25 11:19:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I'm genuinely curious, there is a little project I want to do when I have time (ahahahahaha) which getting such info would be really important</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:19:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:19:25"><div class="time">2022-05-25 11:19:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So if there is a good way of doing that it would be really useful to know</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:23:19">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-05-25 11:23:19"><div class="time">2022-05-25 11:23:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What's this 'time' you talk about? üòÜ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:23:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:23:44"><div class="time">2022-05-25 11:23:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think you can find it at a place called Atlantis</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-25 11:24:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-25 11:24:00"><div class="time">2022-05-25 11:24:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's made by a people called mermaids</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§£ Bjorn Bartholdy
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-01 11:48:06">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-06-01 11:48:06"><div class="time">2022-06-01 11:48:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There‚Äôs also a working BacDive API client for Python üôÇ
<a href="https://pypi.org/project/bacdive/">https://pypi.org/project/bacdive/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PyPI</div>
                            
                            
                                
                                <div class="link-title"><a href="https://pypi.org/project/bacdive/">bacdive</a></div>
                                <div class="link-text">
                                    BacDive-API - Programmatic Access to the BacDive Database
                                </div>
                                
                                
    
        <a href="https://pypi.org/project/bacdive/">
            <img class="preview" src="https://pypi.org/static/images/twitter.6fecba6f.jpg"
                width="250"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://pypi.org/project/bacdive/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-01 12:17:45">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-06-01 12:17:45"><div class="time">2022-06-01 12:17:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Looks pretty similar to the R package, although I like the ability to filter! If only I knew python... üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-29 01:42:27">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2022-05-29 01:42:27"><div class="time">2022-05-29 01:42:27</div></a>
                <div class="msg">
                    <p>Hey all! I've been doing some work on ancient pathogen detection and have been using several authenticity criteria (fragment length, damage pattern, evenness of coverage, &amp; edit distance). I'm now looking at some oral microbiome species, and it looks like I'm getting some complete genomes (very excited!). I just wanted to check with you all what you use as authenticity criteria. I kind of figured that they would be the same?</p>

<p>But for example, I don't see edit distance and evenness of coverage mentioned much in the oral microbiome literature, but this is still important I guess? I've got a couple of perhaps ambiguous ones that I'm not sure about whether this is real. I've attached some examples. MS10211<em>A.bacterium looks real to me. But what about 7401-03</em>A.oral? Edit distance looks good, but evenness of coverage not so much. And MS10211_A.oral?
I'm using bwa aln for the mapping with seeding disabled, -n (error-rate) 0.1, and gaps open 2.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03HG70LZT6/ms10211_a.oral.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">MS10211_A.oral.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03HG70LZT6/ms10211_a.oral.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03HG70LZT6-b7c845e341/ms10211_a.oral_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="2736"
                 height="385" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03GZ8A3MJB/7401-03_a.oral.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">7401-03_A.oral.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03GZ8A3MJB/7401-03_a.oral.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03GZ8A3MJB-f4f58c6bbb/7401-03_a.oral_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="2736"
                 height="385" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03J3JRMBEC/ms10211_a.bacterium__real_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">MS10211_A.bacterium_[real].pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03J3JRMBEC/ms10211_a.bacterium__real_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03J3JRMBEC-5e68a713ff/ms10211_a.bacterium__real__thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="2736"
                 height="385" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üëÄ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 08:25:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-30 08:25:40"><div class="time">2022-05-30 08:25:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Generally it's the same concept yes, but with some additional things (I apologise if you know some/all of this already, but I'm addressing common misconceptions by aDNA researchers)</p>

<p>However you have to be careful: you can't really equate pathogens to standard commensal/pathobiont species, particularly when it comes to the oral microbiome.</p>

<p>General concepts are: what you see reported as 'disease' causing oral taxa are often from clinical studies that only look specifically at that taxa. In fact modern oral microbiology is showing very much that most of these 'disease-causing' taxa are always present in the oral cavity even in health individuals, it's just when the equilibrium is broken disease happens (i.e. something disrupts the relationships causing some taxa to 'take advantage' and over-grow or do other things). So you have to be EXTREMELY careful if you want to report this from a disease aspect (just saying as you're coming from a pathogen direction in your earlier stuff).</p>

<p>Secondly, you must be extremely careful how you are reporting your hits if they are ACTUALLY the species you've mapped against. The oral microbiome is extremely rich (&gt;= 700 species detected, and estimated 100-300 taxa at any one point in western individuals [who have reduced diversity somewhat]). You will have many strains or very closely-related taxa of species. So this means when you're mapping against a reference genome, you could be picking up lots of reads from closely related strains etc, which means if you do variant calling you may accidently incorporate multi-allelic positions and you'll call up the incorrect variant (edit distance doesn't help here as you're dealing with closely related strains so you don't necessarily expect multiple variants on a read). You may also pick up genes present in the reference genome (which are actually often weird clinical variants that don't represent a 'typical' representative of a species). So you if you want to do phylogenomic analysis you must be very careful to check you're not making a wierd 'chimeria' genomes that may bias your mutation rates/reference bias etc. I tried to address this in my PNAS paper from last year but didn't find a good solution (although it was only a small part of the paper).</p>

<p>If you use a genotyper you can sometimes pull multi-allelic SNPs out of your VCF file and display this. IIRC Warinner 2017 (Annu Rev. Human. Gen) describes this theortically, and Vagene 2018 (Nature Eco Evo) has an example of this using MultiVCFAnalyzer output (but you don't have to use that).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 08:25:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-30 08:25:47"><div class="time">2022-05-30 08:25:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* \lecture (sorry)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 09:44:53">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-05-30 09:44:53"><div class="time">2022-05-30 09:44:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I very much agree with James, but can add that I would highly value evenness of coverage and edit distance despite they are indeed sort of under-emphasized in literature. Your plots look like true hits to me although as James said I would be careful interpreting them as pathogens. And evenness of coverage and edit distance are handy to filter out majority of false-positive hits but sometimes they are not enough either. A typical situation when detecting microbes is that reads from a species A can happily map to species' B reference  genome even if those two are not close enough. Then, evenness of coverage and edit distance (and also deamination pattern to some extent) can help you to figure out that you are probably mapping your reads to a very wrong reference genome. However, when there is some resemblance between A and B, and you work with deeply sequenced samples, you might get a descent coverage of species' B reference genome by reads originating from species A. In this case you probably need some phylogenomic insights. Namely, if you happen to know species-specific alleles that are present only for your organism of interest and absent in all other species from the corresponding genus / family, this would be a way to separate the reads that truly originate from your organism of interest from the reads originating from closely related species. However, to establish species-specific alleles for your organism of interest, you need population level allele frequencies for your organism of interest as well as closely related species, which is a tricky, expensive and rarely feasible thing üòû On a positive side, it looks to me that you have done a great job with the first detection screen where you found a few good candidates for further validation! üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 09:48:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-30 09:48:28"><div class="time">2022-05-30 09:48:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, agreed evenness of coverage also very useful here (but look reasonable from your plot)! And I also agree first screen looks good!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 09:51:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-30 09:51:59"><div class="time">2022-05-30 09:51:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; . However, to establish species-specific alleles for your organism of interest, you need population level allele frequencies for your organism of interest as well as closely related species, which is a tricky, expensive and rarely feasible thing 
Yes this is very important. The problem (Which I forgot in my essay) is that most oral microbita are quite undercharacterised. For example for Tannerella, up until 2018 had a single published genome, and then this was expanded to 8 and now something like 15 in the space of of 4 years. Very few oral taxa have more than a single genome and often are not even named! So phylogenetic analyses are difficult to do.</p>

<p>So you just have to be take a lot of care :)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 16:23:47">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2022-05-30 16:23:47"><div class="time">2022-05-30 16:23:47</div></a>
                <div class="msg">
                    <p>Hi All!</p>

<p>We were wondering today if anyone out there could provide some feedback, or point to specific papers, on performing functional enrichment tests on sets of genes within a meta-community?</p>

<p>Starting with abundance estimates for microbial metabolic pathways and other molecular functions from our shotgun sequencing data (HUMAnN), one would typically perform some kind of association test on these functions to determine which ones are more strongly associated with one group of samples over another.</p>

<p>But we are considering if enrichment tests would require any additional considerations when performed on a set of genes within a metacommunity. The hierarchical structure of these tests could provide a broader look at functional enrichment across the community compared to just considering abundances under a single pathway.</p>

<p>Any thoughts?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 17:42:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-05-30 17:42:57"><div class="time">2022-05-30 17:42:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @irinavelsko any ideas? ü§î Or someone from OU?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:10:12">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:10:12"><div class="time">2022-05-30 19:10:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don‚Äôt quite understand the question, but here‚Äôs how I think about functional data for my analyses-</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:11:10">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:11:10"><div class="time">2022-05-30 19:11:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For HUMAnN analysis, I don‚Äôt consider the species-level assignments for the first step. Instead of looking at a metataxonomic community, I think of it as a metagenetic community (a community of genes/pathways, independent of the organisms they come from)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:12:09">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:12:09"><div class="time">2022-05-30 19:12:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I perform the same kind of enrichment tests and statistical tests on a gene or pathway matrix as I would on an OTU table, b/c just substitute your-favorite-taxonomic-level for genes or pathways</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:12:40">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:12:40"><div class="time">2022-05-30 19:12:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* then once I have an idea of what‚Äôs enriched in one group or another, I look at the species that the genes/pathways are coming from</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:13:28">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:13:28"><div class="time">2022-05-30 19:13:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don‚Äôt think with aDNA we can get the level of resolution on species contributions to pathways/genes that are needed to dig into species-specific contributions</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:14:41">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:14:41"><div class="time">2022-05-30 19:14:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also all bacterial genomes are full of unannotated proteins, so we‚Äôre missing a lot of information simply b/c we can‚Äôt say what all genes are or what pathways they‚Äôre involved in and what the products are</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:16:11">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:16:11"><div class="time">2022-05-30 19:16:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think the HUMAnN paper actually uses simple statistical tests of enrichment, which are built in to the program as separate scripts, which was acceptable at the time</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:16:15">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:16:15"><div class="time">2022-05-30 19:16:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you have a specific question though?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-30 19:17:22">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-05-30 19:17:22"><div class="time">2022-05-30 19:17:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would look for papers that have used HUMAnN to get an idea of what people in modern metagenomics are doing, if you‚Äôre looking for creative and interesting approaches</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-31 09:37:33">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2022-05-31 09:37:33"><div class="time">2022-05-31 09:37:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sorry just returning to this thread now. Thanks for the feedback so far @irinavelsko!</p>

<p>My question was more specifically tied to gene set enrichment tests which would be more commonly applied in differential expression or  population genomics studies. Sure, the methods commonly used with HUMAnN output with give you what pathways/genes are "enriched", but when it comes to the enrichment of a function in this type of data (you could say the combined enrichment of multiple genes/pathways under a GO term) we were thinking of applying  methods used in packages like topGO, which will do something like a Fisher's exact test on a set of genes that are mapped to GO terms, while keeping the hierarchy of GO terms.</p>

<p>Hopefully the question is clear, it is a bit off the wall though, and I haven't found papers that address this specifically ü§™</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-31 21:20:07">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-31 21:20:07"><div class="time">2022-05-31 21:20:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Haven't tried it yet on aDNA, but from my time working on transcriptomics, I had a good experience with eGSEA</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-31 21:20:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-31 21:20:11"><div class="time">2022-05-31 21:20:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://bioconductor.org/packages/release/bioc/html/EGSEA.html">https://bioconductor.org/packages/release/bioc/html/EGSEA.html</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Bioconductor</div>
                            
                            
                                
                                <div class="link-title"><a href="https://bioconductor.org/packages/release/bioc/html/EGSEA.html">EGSEA</a></div>
                                <div class="link-text">
                                    This package implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing.
                                </div>
                                
                                
    
        <a href="https://bioconductor.org/packages/release/bioc/html/EGSEA.html">
            <img class="preview" src="http://bioconductor.org/images/logo/jpg/bioconductor_logo_rgb.jpg"
                width="868"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://bioconductor.org/packages/release/bioc/html/EGSEA.html</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-31 21:23:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-31 21:23:47"><div class="time">2022-05-31 21:23:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If I understood correctly, that's GSEA (Gene Set Enrichment Analysis) that you're looking for</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-31 21:29:04">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-31 21:29:04"><div class="time">2022-05-31 21:29:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* All these GSEA methods usually start with a list of differentialy expressed genes (DEGs) that you get from one of the differential abundance method (edgeR, DeSeq2, limma, ...) and then look for these in pathways/gene-sets.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-05-31 21:29:37">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-05-31 21:29:37"><div class="time">2022-05-31 21:29:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* eGSEA is nice because it combines different GSEA methods and gives you a consensus of their results</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-01 07:58:28">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-06-01 07:58:28"><div class="time">2022-06-01 07:58:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That sounds neat! I've used ReviGO (<a href="http://revigo.irb.hr/">http://revigo.irb.hr/</a>) but I don't find GO terms useful for bacterial functional analysis and prefer other gene classification systems</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 12:02:05">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-06-13 12:02:05"><div class="time">2022-06-13 12:02:05</div></a>
                <div class="msg">
                    <p>Hi All,
For everyone using Kraken2, have you tried building databases with differing k-mer lengths and tested how this changes the number of reads assigned taxonomy, and how accurate the assignments are?</p>

<p>I have some libraries with lots of very short reads that had very few reads assigned taxonomy using a Kraken2 database built with the default k-mer length. I‚Äôm considering trying to build a database with a shorter k-mer length, but thinking that it might not be worth it if the assignments are too non-specific.</p>

<p>Does anyone have any experience with this they can share?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 12:05:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-06-13 12:05:51"><div class="time">2022-06-13 12:05:51</div></a>
                <div class="msg">
                    <p>Hello Irina. Are your reads shorter than 31bp?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 12:09:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-06-13 12:09:59"><div class="time">2022-06-13 12:09:59</div></a>
                <div class="msg">
                    <p>Hi @irinavelsko, I never tried to use kmers shorter than ~30 bp since they are too non-specific as you say. However, once I tried to do alignments of ultra-short reads (the vast majority were shorter than 30 bp) against a large (NT) database, and got approximately 1-2% of reads aligned with MAPQ&gt;0. So I believe it might make sense to build Kraken db with kmer length shorter than 30 and run classification of ultra-short reads but my guess would be that it will not help you much, i.e. I believe very few reads will be classified and even fewer will be accurately classified</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 12:53:31">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-06-13 12:53:31"><div class="time">2022-06-13 12:53:31</div></a>
                <div class="msg">
                    <p>Hi Nico and Nikolay, here‚Äôs an example of the read length distribution from a Damageprofiler run against a highly abundant species in one of the samples. Most of the reads are &gt;30pb, and the average length is ~45bp. Many of the samples look like this for each of the species mapped against for Damageprofiler, and they had ~1-2% of reads assigned taxonomy by Kraken2</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03KF3XDG9J/length_plot_2.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Length_plot_2.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03KF3XDG9J/length_plot_2.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03KF3XDG9J-176c09e434/length_plot_2_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="1286"
                 height="909" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 12:57:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-06-13 12:57:50"><div class="time">2022-06-13 12:57:50</div></a>
                <div class="msg">
                    <p>ok, super. Correct me if I am wrong, but all reads with length &gt;31 should be classified with the default k-mer size of kraken2. Maybe you can just run some tests only of the reads below 30bp, but I agree with Nikolay that such reads are more prone to misclassification. Depending on how many they are, you can probably Blast only those against NCBI nt, then import to MEGAN and see how they look like</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 12:58:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-06-13 12:58:33"><div class="time">2022-06-13 12:58:33</div></a>
                <div class="msg">
                    <p>based on your distributions it seems that there are just some thousand reads with length 25-30bp</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 13:14:14">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-06-13 13:14:14"><div class="time">2022-06-13 13:14:14</div></a>
                <div class="msg">
                    <p>Maybe centrifuge would be interesting for you  since it has extending kmers but I agree that I wouldn't trust reads shorter than 30bp too much</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 13:59:36">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-06-13 13:59:36"><div class="time">2022-06-13 13:59:36</div></a>
                <div class="msg">
                    <p>The mode of the length distributions seems to be around 30-35 bp, so I would not bother making a new Kraken DB (which takes a lot of time if your database is large enough) to classify / rescue  additional a few thousand reads</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 14:08:40">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-13 14:08:40"><div class="time">2022-06-13 14:08:40</div></a>
                <div class="msg">
                    <p>I think is not so much about whether you can assign very short reads. The question would be rather whether you have a different sensitivity of assigning reads to taxonomic units if you use a k-mer length of 30, 35, or 40 or any other number. The longer the k-mers, the more specific they are, but due to the shorter reads we cannot use very longer k-mers. So if you have a read of 45 bp length, then you will be in theory able to assign them with all three k-mer lengths. Did someone ever did test the difference in specificity per k-mer length?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 14:56:58">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-06-13 14:56:58"><div class="time">2022-06-13 14:56:58</div></a>
                <div class="msg">
                    <p>@Alex H√ºbner I agree, sensitivity vs. specificity in terms of e.g. ROC-curve is an interesting question, which can be computed on simulated data where one knows the ground truth. We are soon going to publish our ancient metagenomics workflow that I briefly presented last SPAAM, and for this manuscript I have done some benchmarking of available workflows (classification + alignment) on simulated data in terms of ROC-curves. However, I have not tested specifically the effect of kmer size on ROC AUC. There can be such benchmarking already done in papers that compare classifiers (CLARK, centrifuge, Kraken, MetaPhlan etc.), there are a few papers on this topic, I believe @irinavelsko knows classifier comparison studies better than me. My intuition here is the following. Since our reads are relatively short, I would indeed not select very long kmers (like 40) when building a database because then we will loose lots of reads, so we might be loosing sensitivity. On the other hand, kmers shorter than 30 might be very non-specific, so we jeopardize specificity. So for a rough sensitivity-specificity balance, I would select the shortest "specific-enough" kmer size, which seems to be around 30, therefore I believe this is the default in Kraken. But this is just my intuition, I haven't properly tested this</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 15:02:43">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-13 15:02:43"><div class="time">2022-06-13 15:02:43</div></a>
                <div class="msg">
                    <p>Thanks for the detailed answer @Nikolay Oskolkov!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 15:12:42">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-06-13 15:12:42"><div class="time">2022-06-13 15:12:42</div></a>
                <div class="msg">
                    <p>Yes thanks all!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 15:24:27">
            
                <img src="https://secure.gravatar.com/avatar/f0dba247752695631aaf2d1e7682c7aa.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Samuel Neuenschwander
                     <span class="print-only user-email">(samuel.neuenschwander@unil.ch)</span>
                </div>
                <a href="#2022-06-13 15:24:27"><div class="time">2022-06-13 15:24:27</div></a>
                <div class="msg">
                    <p>Last march we have published a benchmark study ‚ÄúBenchmarking metagenomics classifiers on ancient viral DNA: a simulation study‚Äù (<a href="https://peerj.com/articles/12784/">https://peerj.com/articles/12784/</a>) where in a supplement note <a href="https://doi.org/10.7717/peerj.12784/supp-9">https://doi.org/10.7717/peerj.12784/supp-9</a> we have investigated the outcome of a Kraken2 database with a smaller kmer. You can classify more viruses, but the error rate increases also significantly.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PeerJ</div>
                            
                            
                                
                                <div class="link-title"><a href="https://peerj.com/articles/12784/">Benchmarking metagenomics classifiers on ancient viral DNA: a simulation study</a></div>
                                <div class="link-text">
                                    Owing to technological advances in ancient DNA, it is now possible to sequence viruses from the past to track down their origin and evolution. However, ancient DNA data is considerably more degraded and contaminated than modern data making the identification of ancient viral genomes particularly challenging. Several methods to characterise the modern microbiome (and, within this, the virome) have been developed; in particular, tools that assign sequenced reads to specific taxa in order to characterise the organisms present in a sample of interest. While these existing tools are routinely used in modern data, their performance when applied to ancient microbiome data to screen for ancient viruses remains unknown. In this work, we conducted an extensive simulation study using public viral sequences to establish which tool is the most suitable to screen ancient samples for human DNA viruses. We compared the performance of four widely used classifiers, namely Centrifuge, Kraken2, DIAMOND and MetaPhlAn2, in correctly assigning sequencing reads to the corresponding viruses. To do so, we simulated reads by adding noise typical of ancient DNA to a set of publicly available human DNA viral sequences and to the human genome. We fragmented the DNA into different lengths, added sequencing error and C to T and G to A deamination substitutions at the read termini. Then we measured the resulting sensitivity and precision for all classifiers. Across most simulations, more than 228 out of the 233 simulated viruses were recovered by Centrifuge, Kraken2 and DIAMOND, in contrast to MetaPhlAn2 which recovered only around one third. Overall, Centrifuge and Kraken2 had the best performance with the highest values of sensitivity and precision. We found that deamination damage had little impact on the performance of the classifiers, less than the sequencing error and the length of the reads. Since Centrifuge can handle short reads (in contrast to DIAMOND and Kraken2 with default settings) and since it achieve the highest sensitivity and precision at the species level across all the simulations performed, it is our recommended tool. Regardless of the tool used, our simulations indicate that, for ancient human studies, users should use strict filters to remove all reads of potential human origin. Finally, we recommend that users verify which species are present in the database used, as it might happen that default databases lack sequences for viruses of interest.
                                </div>
                                
                                
    
        <a href="https://peerj.com/articles/12784/">
            <img class="preview" src="https://dfzljdn9uc3pi.cloudfront.net/2022/12784/1/graphical-abstract.jpg"
                width="63"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://peerj.com/articles/12784/</div>
                                
                                
                            
                        </div>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PeerJ</div>
                            
                            
                                
                                <div class="link-title"><a href="https://doi.org/10.7717/peerj.12784/supp-9">Benchmarking metagenomics classifiers on ancient viral DNA: a simulation study</a></div>
                                <div class="link-text">
                                    Owing to technological advances in ancient DNA, it is now possible to sequence viruses from the past to track down their origin and evolution. However, ancient DNA data is considerably more degraded and contaminated than modern data making the identification of ancient viral genomes particularly challenging. Several methods to characterise the modern microbiome (and, within this, the virome) have been developed; in particular, tools that assign sequenced reads to specific taxa in order to characterise the organisms present in a sample of interest. While these existing tools are routinely used in modern data, their performance when applied to ancient microbiome data to screen for ancient viruses remains unknown. In this work, we conducted an extensive simulation study using public viral sequences to establish which tool is the most suitable to screen ancient samples for human DNA viruses. We compared the performance of four widely used classifiers, namely Centrifuge, Kraken2, DIAMOND and MetaPhlAn2, in correctly assigning sequencing reads to the corresponding viruses. To do so, we simulated reads by adding noise typical of ancient DNA to a set of publicly available human DNA viral sequences and to the human genome. We fragmented the DNA into different lengths, added sequencing error and C to T and G to A deamination substitutions at the read termini. Then we measured the resulting sensitivity and precision for all classifiers. Across most simulations, more than 228 out of the 233 simulated viruses were recovered by Centrifuge, Kraken2 and DIAMOND, in contrast to MetaPhlAn2 which recovered only around one third. Overall, Centrifuge and Kraken2 had the best performance with the highest values of sensitivity and precision. We found that deamination damage had little impact on the performance of the classifiers, less than the sequencing error and the length of the reads. Since Centrifuge can handle short reads (in contrast to DIAMOND and Kraken2 with default settings) and since it achieve the highest sensitivity and precision at the species level across all the simulations performed, it is our recommended tool. Regardless of the tool used, our simulations indicate that, for ancient human studies, users should use strict filters to remove all reads of potential human origin. Finally, we recommend that users verify which species are present in the database used, as it might happen that default databases lack sequences for viruses of interest.
                                </div>
                                
                                
    
        <a href="https://doi.org/10.7717/peerj.12784/supp-9">
            <img class="preview" src="https://dfzljdn9uc3pi.cloudfront.net/2022/12784/1/graphical-abstract.jpg"
                width="63"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://doi.org/10.7717/peerj.12784/supp-9</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        üëç:skin_tone_3: Katerina Guschanski
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 15:30:49">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-13 15:30:49"><div class="time">2022-06-13 15:30:49</div></a>
                <div class="msg">
                    <p>Thanks @Samuel Neuenschwander, that‚Äôs what we were looking for. It‚Äôs impressive to see how many more spurious taxa you got when dropping the k-mer lengths from 35 to 29 bp in Kraken2.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Samuel Neuenschwander, Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-13 15:36:22">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2022-06-13 15:36:22"><div class="time">2022-06-13 15:36:22</div></a>
                <div class="msg">
                    <p>Great, thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-15 15:34:58">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-15 15:34:58"><div class="time">2022-06-15 15:34:58</div></a>
                <div class="msg">
                    <p>Just to chime in here, @irinavelsko and @Alex H√ºbner. We have explored different kmer sizes with Kraken (original) and our ancient mammalian mtDNA captures. We've essentially found that a kmer of 21 is a good compromise of true to false positives, <em><strong>with the caveat</strong></em> that we are then mapping to mtDNA genomes, and taking only reads that have a good MQ and are at least 35bp long. The mapping process itself removes a lot of the false assignments you get with Kraken, and at higher kmer sizes we fail to assign a lot of reads. We've tested this with simulated ancient DNA, but none of this is published yet.</p>

<p>On the subject of the correct kmer size - I'm curious about the thought "my average read length is 40, so a kmer of 30 will be ok." With those kmer/read sizes, each read has only a handful of kmers - and any mutation/damage/error in the middle of a read will disrupt all possible kmers in that read, right?</p>

<p>And thanks for the paper link, that indeed looks like a good reference!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 21:46:51">
            
                <img src="https://secure.gravatar.com/avatar/631e0c1a4a0b8aaa3f55b50da5a07bb4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Audrey Lin
                     <span class="print-only user-email">(linat@si.edu)</span>
                </div>
                <a href="#2022-06-28 21:46:51"><div class="time">2022-06-28 21:46:51</div></a>
                <div class="msg">
                    <p>Hi! I am trying to learn how to do qpWave/qpAdm analysis through Admixtools command line (I am having a hell of a time trying to get admixr/admixtools2 to install on R to no success...) and want to work my way through this tutorial: <a href="https://comppopgenworkshop2019.readthedocs.io/en/latest/contents/05_qpwave_qpadm/qpwave_qpadm.html">https://comppopgenworkshop2019.readthedocs.io/en/latest/contents/05_qpwave_qpadm/qpwave_qpadm.html</a>. Does anyone know where the data comes from? It does not appear to be the same data as that used in the admixtools example data/Patterson 2012. Thanks a lot!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Diana Ivette Cruz D√°valos, Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        :partyparrot: Diana Ivette Cruz D√°valos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 21:48:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-28 21:48:09"><div class="time">2022-06-28 21:48:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Audrey, are you applying this to microbial data or something? I thought qpwave stuff was designed for eukaryotic genomes and pop-gen</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 21:51:44">
            
                <img src="https://secure.gravatar.com/avatar/631e0c1a4a0b8aaa3f55b50da5a07bb4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Audrey Lin
                     <span class="print-only user-email">(linat@si.edu)</span>
                </div>
                <a href="#2022-06-28 21:51:44"><div class="time">2022-06-28 21:51:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah I am trying to apply it to eukaryotic data, I apologise that it is not appropriate for this channel but was hoping someone might know and one of my colleagues had suggested that I reach out here</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 21:59:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-28 21:59:52"><div class="time">2022-06-28 21:59:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see. Well there might be a couple of people who know but as the purpose of spaam is for metagenomics, and not pop-gen, it's unlikely we have the expertise here sadly :/.</p>

<p>Of course we can keep the question up in case someone does know, but I just wanted to say don't expect much on such a topic here unfortunately. (I keep telling my colleagues in pop-gen to set up their own thing like spaam for your discipline, as we frequently get similar questions etc. but no one has done so yet :/)</p>

<p>Sorry for the slightly negative answer :(</p>

<p>Thinking about it @Marcel Keller has been drifting into pop-gen, maybe he has an idea?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 22:02:31">
            
                <img src="https://secure.gravatar.com/avatar/631e0c1a4a0b8aaa3f55b50da5a07bb4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Audrey Lin
                     <span class="print-only user-email">(linat@si.edu)</span>
                </div>
                <a href="#2022-06-28 22:02:31"><div class="time">2022-06-28 22:02:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No worries, thank you for your input anyway, and sorry again for the intrusion! I actually am working on a metagenomics project as well and will definitely be back soon with more relevant questions ^_^</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 23:15:43">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-06-28 23:15:43"><div class="time">2022-06-28 23:15:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @Audrey Lin! You can find all the populations they use in the tutorial in the AARD 1240K + HO dataset here - <a href="https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data">https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data</a> , so you could download these and use them in place of <code>popgen_qpAdm_test_190120.geno</code> , .snp, .ind. The population names might have changed a bit, so you could either download the version from 2019 to match the individual/population names from the tutorial or edit the tutorial scripts to match the currently used population names.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Audrey Lin, Marcel Keller
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 23:18:14">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-06-28 23:18:14"><div class="time">2022-06-28 23:18:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For instance, the individuals for "Yamnaya<em>Samara" in the tutorial are now labeled as  "Russia</em>Samara<em>EBA</em>Yamnaya"</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Audrey Lin
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-28 23:19:26">
            
                <img src="https://secure.gravatar.com/avatar/631e0c1a4a0b8aaa3f55b50da5a07bb4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Audrey Lin
                     <span class="print-only user-email">(linat@si.edu)</span>
                </div>
                <a href="#2022-06-28 23:19:26"><div class="time">2022-06-28 23:19:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* THANK YOU @Hannah Moots!!!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-29 07:25:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-06-29 07:25:55"><div class="time">2022-06-29 07:25:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you @Hannah Moots! I guess there is more diversity of people here than I expected üò¨</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-29 10:46:31">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2022-06-29 10:46:31"><div class="time">2022-06-29 10:46:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for stepping in @Hannah Moots ‚ò∫Ô∏è</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò¨ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-29 23:14:20">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-06-29 23:14:20"><div class="time">2022-06-29 23:14:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Happy to! I have worked through the same tutorial myself and am happy if that experience is useful to others :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-06-29 23:15:34">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-06-29 23:15:34"><div class="time">2022-06-29 23:15:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And excited to be part of this slack group (just joined recently) and to learn more about ancient metagenomics from you all üéâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 10:15:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 10:15:11"><div class="time">2022-07-19 10:15:11</div></a>
                <div class="msg">
                    <p>Hi, all. Has anyone had persistent dimer problems with the Meyer and Kircher 2010 DS libraries with dual indexing? If so, what do you think caused it? I haven't before, but I've been working in a new lab and many of my libraries have dimer peaks. I know low starting template can lead to this, but I haven't really had dimer problems with this protocol before even in blanks. Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 11:08:33">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2022-07-19 11:08:33"><div class="time">2022-07-19 11:08:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes! Occasionally with low input samples, but mostly with material that tends to cross-linking/PCR inhibition like fixed tissues or some plant species. Trying a bead cleanup with different library to beads ratios sometimes gets rid of it</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 11:10:12">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2022-07-19 11:10:12"><div class="time">2022-07-19 11:10:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also worth checking if the adapter and index stocks are at the same concentration as your previous lab üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 11:14:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 11:14:11"><div class="time">2022-07-19 11:14:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you! These libraries are from Neolithic and Mesolithic skeletal remains, so they're quite old and preservation is not great. But I'm just curious why I've never had issues with dimers even in blanks before, but now they're all flooded. Yes - all stocks and working dilution concentrations are what they should be. </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 11:17:00">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2022-07-19 11:17:00"><div class="time">2022-07-19 11:17:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I get dimers in most of my blanks ü§î
Do the libraries look good if not for the dimers?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 12:13:06">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 12:13:06"><div class="time">2022-07-19 12:13:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Here is how I am used to my library blanks (LB) and libraries performing after indexing PCR with no size selection, only column cleanup after amplification. For these I used Pfu Turbo Cx for indexing.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03PYU7H4QM/2020-10-20_-_16.39.08__dragged_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2020-10-20 - 16.39.08 (dragged).pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03PYU7H4QM/2020-10-20_-_16.39.08__dragged_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03PYU7H4QM-1b8d7f746d/2020-10-20_-_16.39.08__dragged__thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="909"
                 height="1286" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03Q4A349DJ/2020-10-20_-_16.39.08__dragged__2.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2020-10-20 - 16.39.08 (dragged) 2.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03Q4A349DJ/2020-10-20_-_16.39.08__dragged__2.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03Q4A349DJ-83066fbdc4/2020-10-20_-_16.39.08__dragged__2_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="909"
                 height="1286" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 12:16:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 12:16:51"><div class="time">2022-07-19 12:16:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And here are examples of what I am generating now, extraction blank and sample library (this tapestation is old and has some systemic issue which is why the runs are so noisy‚Ä¶they‚Äôre all like that). These were indexed with KAPA HiFi Uracil+, but other than that and the oligos being from different suppliers, there are no differences in the protocols or reagents.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03PMA2SATH/2022-july-18-halula-and-belgium-libs__dragged_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2022-July-18-Halula-and-Belgium-libs (dragged).pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03PMA2SATH/2022-july-18-halula-and-belgium-libs__dragged_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03PMA2SATH-fe73b78653/2022-july-18-halula-and-belgium-libs__dragged__thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="909"
                 height="1286" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03QEGYR48H/2022-july-18-halula-libs__dragged_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2022-July-18-Halula-libs (dragged).pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03QEGYR48H/2022-july-18-halula-libs__dragged_.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03QEGYR48H-6fc0c34d7e/2022-july-18-halula-libs__dragged__thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="909"
                 height="1286" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 12:54:55">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2022-07-19 12:54:55"><div class="time">2022-07-19 12:54:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think this would be a good candidate for a 1:1 bead clean up</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 12:57:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 12:57:56"><div class="time">2022-07-19 12:57:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have heard that people use 1.3:1 and 1.5:1 ratios as well for ancient DNA. Do you have good success with 1:1? I am worried about losing short fragments in the 170-200 range.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 14:11:21">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2022-07-19 14:11:21"><div class="time">2022-07-19 14:11:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have excellent results with cleaning up single-stranded libraries with 1:1 and I'm going to try it soon on some M&amp;K libraries that came out with lots of dimers (at 1.7:1). You have a good amount of library compared to the dimer peak and a good size difference. If you are worried about losing short fragments you could try starting at 1:5 and decrease if you don't get rid of it?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üéØ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 15:35:16">
            
                <img src="https://secure.gravatar.com/avatar/8d2b7aa455bae0781102ec1845f71b0b.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0019-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Spyrou
                     <span class="print-only user-email">(spyrou@shh.mpg.de)</span>
                </div>
                <a href="#2022-07-19 15:35:16"><div class="time">2022-07-19 15:35:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Kelly, we used to have a similar effect in post-indexing amplifications of samples or captures with low input. We used to call this effect the ‚Äúiguana‚Äù üôÇ. Not sure if what you are seeing here is the exact same thing, but at the time (back in 2014) we were indexing with Pfu Turbo Cx and amplifying (post-indexing) with AccuPrime. Our solution was to switch the AccuPrime with Herculase. I believe the effect was completely gone after the switch. Hope this somehow helps!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÆ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 20:00:12">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 20:00:12"><div class="time">2022-07-19 20:00:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you both so much! This has been immensely helpful. I did a 1.3:1 size selection using NucleoMag beads, and it worked so well. The dimer signatures are completely gone, and there wasn‚Äôt that much DNA loss or change to fragment size distribution in the 150bp+ range.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-19 20:04:12">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-07-19 20:04:12"><div class="time">2022-07-19 20:04:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maria, I think that is exactly what is happening, since I haven‚Äôt used this enzyme before for indexing. This never happened with Pfu Turbo Cx. I usually don‚Äôt re-amplify after indexing unless I have to. When I have re-amplified, I used Pfu Turbo Cx and had no dimers even on very low yield libraries. I didn‚Äôt realize there was such variability in enzyme and dimer formation. Thank you!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Maria Spyrou
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-07-20 11:49:04">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2022-07-20 11:49:04"><div class="time">2022-07-20 11:49:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good to know! Maybe I'll try 1.3:1 as well üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:50:44">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 13:50:44"><div class="time">2022-08-08 13:50:44</div></a>
                <div class="msg">
                    <p>Hi all! Does anyone have experience using pydamage v. 0.7? I am using it now, however most of the options for parameters listed on the readthedocs website are not actually available <a href="https://pydamage.readthedocs.io/en/0.7/CLI.html">https://pydamage.readthedocs.io/en/0.7/CLI.html</a>. For example, I wanted to <code>filter</code>  with <code>-t 0</code> , however it was not possible. I also could not use the <code>--plot</code>  or <code>--group</code> functions with the <code>analyze</code> command. Does anyone know if these options are actually available or am I using an outdated version? or maybe I‚Äôm missing some components that need to be installed in my conda environment?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:51:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-08 13:51:46"><div class="time">2022-08-08 13:51:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* How did you install pyDamage? Was it via conda or pip?</p>

<p>@Maxime Borry @Alex H√ºbner?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:52:21">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 13:52:21"><div class="time">2022-08-08 13:52:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Installed with conda inside a conda env</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:54:58">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-08 13:54:58"><div class="time">2022-08-08 13:54:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which version are you using?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:55:20">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 13:55:20"><div class="time">2022-08-08 13:55:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* pydamage v. 0.70</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:56:58">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-08 13:56:58"><div class="time">2022-08-08 13:56:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ‚Äîplot and ‚Äîgroup worked for me last time I used pyDamage analyze. But Maxime knows for sure üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 13:57:22">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-08 13:57:22"><div class="time">2022-08-08 13:57:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maxime has updated it last week but not the functions that you mentioned.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:00:18">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-08 14:00:18"><div class="time">2022-08-08 14:00:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I just checked and I have also version 0.7 installed and <code>pydamage filter -t 0 pydamage_results.csv</code> works for me.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:00:45">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 14:00:45"><div class="time">2022-08-08 14:00:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* maybe it is the order in which I am listing the parameters in my command</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:02:00">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 14:02:00"><div class="time">2022-08-08 14:02:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It works now ü§¶ üòÖ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:02:21">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-08 14:02:21"><div class="time">2022-08-08 14:02:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Great!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:03:21">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 14:03:21"><div class="time">2022-08-08 14:03:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks all!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:11:06">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-08-08 14:11:06"><div class="time">2022-08-08 14:11:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Ashild, just tested it again, it works with version 0.70.
Also, side note: 0.7 != 0.70 üòâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç √Öshild (Ash)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 14:11:44">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-08-08 14:11:44"><div class="time">2022-08-08 14:11:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* the order of the parameters doesn‚Äôt matter</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç √Öshild (Ash)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 15:48:18">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-08-08 15:48:18"><div class="time">2022-08-08 15:48:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, but, the parameters have to be after the  <code>analyze/filter commands</code> , because <code>pydamage filter -t 0 pydamage_results.csv</code> works, but <code>pydamage -t 0 filter pydamage_results.csv</code>  doesn‚Äôt. I misunderstood the instructions üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-08 16:42:05">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-08-08 16:42:05"><div class="time">2022-08-08 16:42:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You're right @√Öshild (Ash), this is because they are parameters of the subfunctions (analyze or filter), but not parameters of the main program.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 05:14:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 05:14:35"><div class="time">2022-08-16 05:14:35</div></a>
                <div class="msg">
                    <p>Hi all! For anyone who‚Äôs successfully built a KrakenUniq ‚Äònt‚Äô database, have you seen this message? <code>0 sequences mapped to taxa.</code> .. accompanied with some 5GB of errors of the sort <code>Didn't find taxonomy ID mapping for sequence XXXX !!</code> ! I‚Äôm restarting the build from the previous step in case some file got corrupted but each step takes so long, I thought I‚Äôd try here alongside my own troubleshooting attempts just in case someone has a fix.
For reference I‚Äôm using a strategy suggested by @Nikolay Oskolkov ages ago, of using Kraken2 to download the full nt database with all necessary files, and then KrakenUniq to build it. Would love any tips! Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 09:32:09">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-16 09:32:09"><div class="time">2022-08-16 09:32:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya the error seems to have something to do with the NCBI taxonomy that does not match the actual sequences. Yes, when I built full NT KrakenUniq DB I used all the same input files from Kraken2, but just started a building process using krakenuniq-build utility. If this way did not work for you could you perhaps take a screenshot of what files are in your directory where you start the building process? I would like to see what files you are using as input, then perhaps I could be more specific</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 16:20:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 16:20:32"><div class="time">2022-08-16 16:20:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks so much Nikolay! Here‚Äôs the output right now, after I restarted it at the sort step last night:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03TRRNHYR3/screen_shot_2022-08-16_at_9.17.44_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2022-08-16 at 9.17.44 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03TRRNHYR3/screen_shot_2022-08-16_at_9.17.44_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03TRRNHYR3-9a0eca6af7/screen_shot_2022-08-16_at_9.17.44_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="139" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 16:20:46">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 16:20:46"><div class="time">2022-08-16 16:20:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Here is my database directory:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03TUQC3LP4/screen_shot_2022-08-16_at_9.18.05_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2022-08-16 at 9.18.05 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03TUQC3LP4/screen_shot_2022-08-16_at_9.18.05_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03TUQC3LP4-99669df26b/screen_shot_2022-08-16_at_9.18.05_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="24" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 16:21:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 16:21:11"><div class="time">2022-08-16 16:21:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* In <code>library/nt</code> :</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03TX8L7KS8/screen_shot_2022-08-16_at_9.18.22_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2022-08-16 at 9.18.22 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03TX8L7KS8/screen_shot_2022-08-16_at_9.18.22_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03TX8L7KS8-343df92334/screen_shot_2022-08-16_at_9.18.22_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="33" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 16:21:26">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 16:21:26"><div class="time">2022-08-16 16:21:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And in <code>taxonomy</code>:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03TN6HSSNS/screen_shot_2022-08-16_at_9.18.35_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2022-08-16 at 9.18.35 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03TN6HSSNS/screen_shot_2022-08-16_at_9.18.35_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03TN6HSSNS-93466f5c70/screen_shot_2022-08-16_at_9.18.35_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="20" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 16:22:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 16:22:35"><div class="time">2022-08-16 16:22:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* One thing I‚Äôm wondering about is if something might be going awry at the shrink stage-- the full database was some 4TB which seemed extremely unwieldy, but I haven‚Äôt found data on how to pick a good size to shrink it to. Thank you in advance for all your help (and for all the advice leading up to this!!)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 18:41:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-16 18:41:22"><div class="time">2022-08-16 18:41:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Shreya, thank you! What version of KrakenUniq are you using? I am quite sure that the database size reduction was broken in versions 0.6, it never worked for me. In the newewest version 0.7 they say this issue was fixed (if I remember correctly), but I did not use the database reduction with 0.7, so can‚Äôt confirm. However, your issue does not seem to be related to database size reduction, step 4 failed since taxonomy did not match the sequences in the library-directory. I can see taxdump in your taxonomy folder, did you download it separately from NCBI or you followed Kraken2 instructions? Important that the ids present in seqid2taxid match the headers present in library.fna in the library-folder. Could you check a few?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 18:42:27">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-16 18:42:27"><div class="time">2022-08-16 18:42:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Abby Gancz you also had a similar issue of taxonomy not matching seqids, did you manage to solve it and how?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 18:58:13">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 18:58:13"><div class="time">2022-08-16 18:58:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am using the newest version, 0.7.3, and it does seem like the database reduction worked, or at least it‚Äôs not reporting any errors at that stage!</p>

<p>I downloaded taxdump via the Kraken2 instructions:
<code>/gpfs/data/raghavan-lab/bin/kraken2/kraken2-build --download-library nt --db /scratch/shreya23/KrakenUniq_DBs/FULL_NT_06_23_2022 --use-ftp
/gpfs/data/raghavan-lab/bin/kraken2/kraken2-build --download-taxonomy --db /scratch/shreya23/KrakenUniq_DBs/FULL_NT_06_23_2022 --use-ftp</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 19:00:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 19:00:33"><div class="time">2022-08-16 19:00:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But‚Ä¶ the seqid2taxid.map file is completely empty! I will go check into which stage it is supposed to be filled. The library.fna file is definitely not empty!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 19:08:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-16 19:08:22"><div class="time">2022-08-16 19:08:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, is taxDB also empty?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 19:13:46">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-16 19:13:46"><div class="time">2022-08-16 19:13:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could you also please check that ids from accession2taxid in the taxonomy-folder match the headers from library.fna? Also ids from names.dmp I believe should match the headers</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 21:27:50">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 21:27:50"><div class="time">2022-08-16 21:27:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* taxDB is not empty! Cross-checking IDs now!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 23:19:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 23:19:29"><div class="time">2022-08-16 23:19:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It seems like not everything in accession2taxid is in library.fna. But I think most things in library.fna are in accession2taxid? I haven‚Äôt checked this rigorously, just pulled a random selection to check.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 23:20:12">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 23:20:12"><div class="time">2022-08-16 23:20:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don‚Äôt think the names.dmp ID numbers match the headers in library.fna though.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-16 23:20:16">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-16 23:20:16"><div class="time">2022-08-16 23:20:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks again Nikolay!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-17 17:39:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-17 17:39:01"><div class="time">2022-08-17 17:39:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya let me execute the command lines you posted above to figure out what is going wrong and get back to you</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-17 17:41:40">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-17 17:41:40"><div class="time">2022-08-17 17:41:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* THANK YOU Nikolay!!!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-19 15:21:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-19 15:21:22"><div class="time">2022-08-19 15:21:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Shreya I checked how I built the NT krakenuniq database and I am quite sure the seqid2taxid.map should not be empty. Do you remember whether this file was downloaded when you ran</p>

<p><code>kraken2-build --download-library nt --db DBDIR</code>
or you built this file as a step of krakenuniq-build process?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-19 15:23:50">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-19 15:23:50"><div class="time">2022-08-19 15:23:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I do not remember clearly where this file came from, and am currently waiting for some disk space to empty in order to start the downloading and building process in order to reproduce the files in your directory</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-19 15:28:39">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-08-19 15:28:39"><div class="time">2022-08-19 15:28:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* One thing I did in addition to what you have already done, I actually ran the krakenuniq database building on the top of the Kraken2 DB, so I used</p>

<p><code>kraken2-build --build --fast-build</code>
first, and then proceeded with krakenuniq-build. So perhaps the seqid2taxid.map was built as a step of kraken2-build. Very complicated. But as soon as I can clean up some space I will start the krakenuniq-build and see whether I get an empty seqid2taxid.map file as you got</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-22 18:52:55">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-22 18:52:55"><div class="time">2022-08-22 18:52:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I feel bad that you would have to clean up space to help me with this! Cleaning out space is miserable!
I don‚Äôt remember seeing the creation of the seqid2taxid.map file, but based on the timestamps, I think it was created as part of the build, not downloaded with <code>--download-library</code></p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F03UJPDS17F/screen_shot_2022-08-22_at_11.51.41_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screen Shot 2022-08-22 at 11.51.41 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F03UJPDS17F/screen_shot_2022-08-22_at_11.51.41_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F03UJPDS17F-50d000d87f/screen_shot_2022-08-22_at_11.51.41_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="225" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-22 18:53:54">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-08-22 18:53:54"><div class="time">2022-08-22 18:53:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Might make sense for me to delete these and start from scratch with the downloads if you think that would help! I can try it with the <code>kraken2-build --build --fast-build</code> step this time, can‚Äôt hurt to have a kraken2 database as well!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-23 08:55:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-09-23 08:55:40"><div class="time">2022-09-23 08:55:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Shreya, my sincere apologies for the very long silence, I had some updates on this but never had time to write a reply to you. I could reproduce your issue and it indeed looks like the seq2taxid.map file is built incorrectly by KrakenUniq when using the taxonomy downloaded by Kraken2. I have not fully understood the difference but it looks like Kraken2 and KrakenUniq use slightly different taxonomy formats, therefore a taxonomy file downloaded via Kraken2 is not compartible with KrakenUniq way of database building. A workaround, as we discussed, is to construct the seq2taxid.map via Kraken2 database building procedure, and then use the seq2taxid.map for building a KrakenUniq database. I am afraid, I do not have better suggestions right now, but I am going to carefully compare the taxonomies from Kraken2 and KrakenUniq and hope to figure out the difference. Will keep you updated!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2022-08-16 18:42:36">
            
                <img src="https://secure.gravatar.com/avatar/db913ec843c9b9d1de24902beed515f5.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0023-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Abby Gancz
                     <span class="print-only user-email">(agancz@gmail.com)</span>
                </div>
                <a href="#2022-08-16 18:42:36"><div class="time">2022-08-16 18:42:36</div></a>
                <div class="msg">
                    <p>@Abby Gancz has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-29 19:30:04">
            
                <img src="https://secure.gravatar.com/avatar/1288c6ac16d16550dba9213d45d145c3.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Tre Blohm
                     <span class="print-only user-email">(tre.blohm@umontana.edu)</span>
                </div>
                <a href="#2022-08-29 19:30:04"><div class="time">2022-08-29 19:30:04</div></a>
                <div class="msg">
                    <p>Hey all,
I'm kind of running into a last-resort problem. I was hoping to use my University's compute cluster to analyze a few hundred genomes in EAGER but it is continually down and broken and even randomly deleting users, and we have no other compute infrastructure.</p>

<p>Does anyone rent out or allow external access to some computing infrastructure I could use to complete my dissertation project? Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-29 19:35:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-29 19:35:05"><div class="time">2022-08-29 19:35:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you have any budget?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-29 19:42:15">
            
                <img src="https://secure.gravatar.com/avatar/1288c6ac16d16550dba9213d45d145c3.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tre Blohm
                     <span class="print-only user-email">(tre.blohm@umontana.edu)</span>
                </div>
                <a href="#2022-08-29 19:42:15"><div class="time">2022-08-29 19:42:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Not a lot to almost none, but I can maybe scrounge up some funds, or assuming it doesn't cost a lot of money I can probably sacrifice out-of-pocket money.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-29 20:17:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-29 20:17:28"><div class="time">2022-08-29 20:17:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You could maybe consider the cheaper ends of AWS/GCP or something if you don't store your data there too long, but steep learning curve and you can spend a lot if you're not careful (they might have educational offers).</p>

<p>Otherwise I don't really know for the US, but in Germany at least there are a few multi-institution (e.g. gwdg or national initiatives (deNBI is one) to provide 'free' computing to German academics. You just need a German PI. So if you could find a collaborator in Germany that may be one route but maybe there are similar schemes in the US?</p>

<p>You could also maybe see if there are other clusters at your uni in other departments, you might be able to slip into one of those via your supervisor</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-31 17:10:43">
            
                <img src="https://secure.gravatar.com/avatar/1288c6ac16d16550dba9213d45d145c3.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tre Blohm
                     <span class="print-only user-email">(tre.blohm@umontana.edu)</span>
                </div>
                <a href="#2022-08-31 17:10:43"><div class="time">2022-08-31 17:10:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, James. I'll look into what I can do with AWS then. We have no other clusters or bioinformatics support on campus</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-31 17:11:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-31 17:11:27"><div class="time">2022-08-31 17:11:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Not even the physicists or something like that?!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-31 17:11:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-31 17:11:45"><div class="time">2022-08-31 17:11:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There are also other companies that offer more normal large VMs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-31 17:12:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-31 17:12:07"><div class="time">2022-08-31 17:12:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I don't know why by name. It may give you a smaller learning curve</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-31 17:13:11">
            
                <img src="https://secure.gravatar.com/avatar/1288c6ac16d16550dba9213d45d145c3.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tre Blohm
                     <span class="print-only user-email">(tre.blohm@umontana.edu)</span>
                </div>
                <a href="#2022-08-31 17:13:11"><div class="time">2022-08-31 17:13:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Nope, as far as I know the cluster that we recently got funding for and installed that has been breaking down constantly was the first cluster our campus got. Ok I can look into that option as well, thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-08-31 17:15:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-08-31 17:15:57"><div class="time">2022-08-31 17:15:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh wow... Ok!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-02 15:47:51">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-09-02 15:47:51"><div class="time">2022-09-02 15:47:51</div></a>
                <div class="msg">
                    <p>Has anyone looked into the extent of microbe degeneration through mineralisation that would cause us to be unable to detect certain bacteria consistently in dental calculus? In other words, are there bacteria that don't show up in dental calculus simply because they are more prone to mineralisation, and are therefore too degraded to be detected?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-02 15:55:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-09-02 15:55:43"><div class="time">2022-09-02 15:55:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Btw, a question like this is better on <b>#general</b>, many more eyes and it's a good question ;)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üßê Bjorn Bartholdy
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-02 19:15:58">
            
                <img src="https://avatars.slack-edge.com/2021-11-06/2696551938660_c79c1f6ef245c5ff1040_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christina Warinner
                     <span class="print-only user-email">(warinner@shh.mpg.de)</span>
                </div>
                <a href="#2022-09-02 19:15:58"><div class="time">2022-09-02 19:15:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think mineralization should help improve their preservation</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-02 20:22:46">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-09-02 20:22:46"><div class="time">2022-09-02 20:22:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yea over the long term I agree, I was just curious about any immediate damage caused by the mineralisation process</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-02 20:25:32">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2022-09-02 20:25:32"><div class="time">2022-09-02 20:25:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Trying to understand why my model calculus is distinct from the growth medium samples I took. The likely explanation is maturation (as in @irinavelsko's 2019 paper), just looking for possible alternate explanations</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-13 12:11:23">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-09-13 12:11:23"><div class="time">2022-09-13 12:11:23</div></a>
                <div class="msg">
                    <p>Could someone jog my memory about studies that link presence/absence of dental calculus deposits in different species with species physiology, tooth morphology, diet, etc.? I know I've seen all this somewhere but struggle to dig up the relevant papers</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-13 12:34:01">
            
                <img src="https://secure.gravatar.com/avatar/d7f9ede974c0dacc69882b20d469dbb1.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Piotr Rozwalak
                     <span class="print-only user-email">(pioroz4@amu.edu.pl)</span>
                </div>
                <a href="#2022-09-13 12:34:01"><div class="time">2022-09-13 12:34:01</div></a>
                <div class="msg">
                    <p>Hi,
I have questions about using Pydamage (@Maxime Borry).</p>

<ol><li>I know that damage patterns in contigs depend on preservation and environmental conditions. But, from your experience, can you recommend any threshold for C-&gt;T transitions below, which is very unlikely to be ancient? e.g. I have a contig with frequency at the first three positions: 0.031, 0.024, 0.01. Values are really low, but conditions are also very good for DNA preservation. Should I believe that it is an ancient fragment? </li><li>I used pydamage filter -t 0 pydamage<em>results.csv, but still, I observed some obvious lab contamination with C-&gt;T frequency at the first three positions: 0.002, 0.001, 0.001. I don't understand why this evident contamination with a very low damage pattern was filtered as an ancient and why predicted</em>accuracy for them was 1? </li><li>Why coverage from metaspades is different to coverage in pydamage results, even though I used the same dataset for assembly and mapping?
Thanks a lot for your help!</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-13 12:35:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-09-13 12:35:06"><div class="time">2022-09-13 12:35:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Note Maxime is on a conference this week so might be slow to reply</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-15 11:50:49">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-09-15 11:50:49"><div class="time">2022-09-15 11:50:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* 1. No, and that's part of the reason why we developed PyDamage üòâ</p>

<ol start="2"><li>The predicted accuracy is very high because you most likely have a lot of reads mapping on a relatively long contig. So in a way, it tells you that even though the difference is very slight, it is consistently there. But again, there's no hard threshold that would for sure tell ancient from modern DNA, it's all about concurring lines of evidence</li><li>Because you get the coverage from a bam file of the reads realigned to your contigs using a short-read aligner. The way metaSpades computes this coverages (computed on the asembly graph) might differ slightly to the coverage that you get from the bam file (computed on the linear sequence of the contigs)</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 12:17:23">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-09-18 12:17:23"><div class="time">2022-09-18 12:17:23</div></a>
                <div class="msg">
                    <p>Hey hive mind, we have just established a clean room to process archeological specimen for aDNA extraction. After intense cleaning we swapped every part of the lab as a sanity check that no DNA traces are left. Our work focuses on ancient microbes, but modern microbes have been handled before in that lab. We processed all swabs using the single-tube library prep (Car√∏e 2017). All libraries have as expected very low DNA concentration (‚â§0.2 ng/uL) and long fragments (~3kb). We would like to know if those fragments are of concern for our planned experiments. We wonder to what extent those long fragments will provide any results on a miseq run? Will a library with such long fragments provide any results? How do people infer modern contamination (long DNA fragments) in blanks? Sorry for this naive question but my wet lab aDNA xp is still at its infancy üòâ Suggestions are highly welcomed.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 12:27:12">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-09-18 12:27:12"><div class="time">2022-09-18 12:27:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You‚Äôll have a sequencing bias towards shorter fragments (at least on Illumina MiSeq), so you will ‚Äúunder-sequence‚Äù the long ones. We have also attempted to use long vs short fragments in our bioinfo decontamination pipelines but since most of our samples are rather historical (only a few decades to a few hundred years old), this approach has not been too successful for us, as long genuine fragments are also present in our data. I would think that ‚Äúdiscriminating‚Äù against long fragments might be more efficient when working with truly old samples</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 14:08:49">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-09-18 14:08:49"><div class="time">2022-09-18 14:08:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Katerina. Our main concern is to what extent 3kb fragments will be sequenced at all given no shorter fragments are present? All we want to know is which organism (DNA) entered our blanks.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 16:10:41">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-09-18 16:10:41"><div class="time">2022-09-18 16:10:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Are these in your blanks as well as your libraries extracted swabs? (I.e. do you have a blank created from just the reagents, with no swab, not even a clean one). Just asking in case these very long fragments are coming from the swabs themselves (and not from the surfaces you have swabbed). I've never seen fragments of that size in blanks (although I work mostly with human DNA)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 16:19:08">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-09-18 16:19:08"><div class="time">2022-09-18 16:19:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* These libraries are made from swabs used to sweep different surfaces in the newly established clean room. Only few of the libraries have fragments of that size, making it unlikely to originate from the actual swab.</p>

<p>We did not process any sample. Those blanks are a sanity check to make sure the clean room is clean indeed.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 16:21:05">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-09-18 16:21:05"><div class="time">2022-09-18 16:21:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My q is if we will receive reads from a miseq after loading a library with 3kb fragments.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 16:27:23">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-09-18 16:27:23"><div class="time">2022-09-18 16:27:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh, I see! Well, good that the swabs can be ruled out. This paper seems to suggest that 3kb amplicons from E. coli can be sequenced on a MiSeq (figure 5d), but I don't have any direct experience with this, so I'll defer to others who do and will keep my fingers crossed for you for figuring this out!<do.mhttps://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5096-9/figures/5| &gt;<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5096-9/figures/5">https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5096-9/figures/5</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5096-9/figures/5">Bead-linked transposomes enable a normalization-free workflow for NGS library preparation - BMC Genomics</a></div>
                                <div class="link-text">
                                    Background Transposome-based technologies have enabled the streamlined production of sequencer-ready DNA libraries; however, current methods are highly sensitive to the amount and quality of input nucleic acid. Results We describe a new library preparation technology (Nextera DNA Flex) that utilizes a known concentration of transposomes conjugated directly to beads to bind a fixed amount of DNA, and enables direct input of blood and saliva using an integrated extraction protocol. We further report results from libraries generated outside the standard parameters of the workflow, highlighting novel applications for Nextera DNA Flex, including human genome builds and variant calling from below 1 ng DNA input, customization of insert size, and preparation of libraries from short fragments and severely degraded FFPE samples. Using this bead-linked library preparation method, library yield saturation was observed at an input amount of 100 ng. Preparation of libraries from a range of species with varying GC levels demonstrated uniform coverage of small genomes. For large and complex genomes, coverage across the genome, including difficult regions, was improved compared with other library preparation methods. Libraries were successfully generated from amplicons of varying sizes (from 50 bp to 11 kb), however, a decrease in efficiency was observed for amplicons smaller than 250 bp. This library preparation method was also compatible with poor-quality DNA samples, with sequenceable libraries prepared from formalin-fixed paraffin-embedded samples with varying levels of degradation. Conclusions In contrast to solution-based library preparation, this bead-based technology produces a normalized, sequencing-ready library for a wide range of DNA input types and amounts, largely obviating the need for DNA quantitation. The robustness of this bead-based library preparation kit and flexibility of input DNA facilitates application across a wide range of fields.
                                </div>
                                
                                
    
        <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5096-9/figures/5">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/12864.jpg"
                width="177"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5096-9/figures/5</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-18 16:49:30">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-09-18 16:49:30"><div class="time">2022-09-18 16:49:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thx again Hannah. the figure refers to input fragments sizes for bead-bound tranpoases but not fragment sizes in the actual library used for sequencing (that is shown in 5c with most fragments &lt; 1000b regardless of input size). in such protocols the transpoase fragments the DNA and adds seq adapter in one step. But I will dig again a bit deeper in the literature and see what I find.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-09-26 17:16:50">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-09-26 17:16:50"><div class="time">2022-09-26 17:16:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I approached the seq center and they were optimistic it will work on a miseq. apparently long fragments makes cluster localization fuzzy (leading to low base qualities), but the density on a miseq is not that high (instead of for example a NovaSeq). Will drop here what comes back.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 13:15:20">
            
                <img src="https://secure.gravatar.com/avatar/67ce71e70b31deeae5d0c9f9d064dd7a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Felix Key
                     <span class="print-only user-email">(key@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-11-01 13:15:20"><div class="time">2022-11-01 13:15:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Just to close this topic. Seq on Miseq of our long fragments was no problem.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 13:01:33">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-10-11 13:01:33"><div class="time">2022-10-11 13:01:33</div></a>
                <div class="msg">
                    <p>Hey all üôÇ I have some questions about reporting the breath and eveness of coverage. 1) Do you use a breath of coverage cut off in your work, ie. only be happy if it's over a certain percentage? 2) is there a metric you report for eveness of coverage? Right now I am visualising with samtools coverage, which does provide metrics but they are separated by accession within the genome. Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 13:06:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-11 13:06:47"><div class="time">2022-10-11 13:06:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For 1) I wouldn't do a breadth of coverage <em>cutoff</em> per se, but rather use it as a guide.</p>

<p>In some cases reference genomes will be of 'odd' pathogenic isolates that have lots of extra genes which aren't in most natural species, so by setting such a cut off could make you lose actual true positives</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 13:07:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-11 13:07:23"><div class="time">2022-10-11 13:07:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So you do sort of have to look at both X coverage and % coverage in tandem, as in if you have low X coverage you ver ylikely would have low % coverage (But it's not a 1:1 ratio...)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 13:45:15">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-11 13:45:15"><div class="time">2022-10-11 13:45:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Ele,</p>

<p>First, I typically do not use a breadth of coverage (percent of bases covered at least once) cutoff because even if say ~3% of ref genome covered (which seems low), it might still be worth following this hit up because what really matters is that the few reads all come from unique places of ref genome and do not form piles of reads at a few conserved across species regions. So one might get ~3% breadth of coverage just because it was a shallow sequenced sample. The number is less important than a visual inspection.</p>

<p>Second, evenness of coverage (much more informative compared to breadth of coverage) is difficult to summarize in a single metric. As well as a deamination pattern. Therefore, I firmly believe, an evenness of coverage plot should be equally "a must" when reporting your findings together with a deamination plot. Currently, it is common to demonstrate deamination plots but unfortunately not coverage plots. I personally tried to create heuristics summarizing the shape of a coverage plot (I bin a ref genome and count in how many bins the coverage is equal to zero, there should be just a few such bins but not too many), but they all do not seem optimal, so I would vote for always presenting a coverage plot. For example the way @Meriam Guellil did, she shared here a link to github with her codes once</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 13:51:55">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-10-11 13:51:55"><div class="time">2022-10-11 13:51:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Here you go: <a href="https://github.com/MeriamGuellil/aDNA-BAMPlotter">https://github.com/MeriamGuellil/aDNA-BAMPlotter</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/MeriamGuellil/aDNA-BAMPlotter">GitHub - MeriamGuellil/aDNA-BAMPlotter: aDNA BAM plotter script for microbial genomes</a></div>
                                <div class="link-text">
                                    aDNA BAM plotter script for microbial genomes. Contribute to MeriamGuellil/aDNA-BAMPlotter development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/MeriamGuellil/aDNA-BAMPlotter">
            <img class="preview" src="https://opengraph.githubassets.com/618ba61fe78503c536ece679cae397e5756ca4554e9d73532fb6615026e1ff9c/MeriamGuellil/aDNA-BAMPlotter"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/MeriamGuellil/aDNA-BAMPlotter</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        ü§© Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 15:33:41">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-10-11 15:33:41"><div class="time">2022-10-11 15:33:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all that's very helpful üòÑ I'll have a go with the bam plotter üòÑ Does anyone have tips for dealing with doing a similar thing for larger (non-microbial) genomes?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 15:37:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-11 15:37:59"><div class="time">2022-10-11 15:37:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Ele working on sedaDNA samples where a goal might be to detect e.g. mammalian organisms, I typically just plot the number of reads mapping to each chromosome as a barplot. This gives some indication that there is no excess of reads mapped to a particular chromosome (unless it is the longest chromosome) but the reads should be spread more or less uniformly across the chromosomes</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-11 15:38:50">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-10-11 15:38:50"><div class="time">2022-10-11 15:38:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That sounds like something I could do! Thanks so much Nikolay üòÑ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-24 14:58:20">
            
                <img src="https://secure.gravatar.com/avatar/9cc944effa4d07ca713e0495b26cec0a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ivany Argueta
                     <span class="print-only user-email">(ivanyjocelyne.arguetamejia.6320@student.uu.se)</span>
                </div>
                <a href="#2022-10-24 14:58:20"><div class="time">2022-10-24 14:58:20</div></a>
                <div class="msg">
                    <p>Hello, I am trying to run eager, but trying to skip adapterremoval since my samples have already been trimmed with cutadapt and merged with FLASH.
The files are now ending in .all.fastq.gz. My samples were initially paird-end, double stranded, 2 colour_chemistry but now essentially single-end fastq file.</p>

<p>I tried running it as double end and double stranded and it gave me the following error.
No TSV file provided - creating TSV from supplied directory.
Reading path(s): /crex/proj/snic2021-23-584/private/Ivany/dna/03-preprocessing/cutadapt/<strong>fastq.gz</strong></p>

<p>Generating paired-end FASTQ data TSV
[nf-core/eager] error: Files could not be found. Do the specified FASTQ read files end in: '.fastq.gz', '.fq.gz', '.fastq', or '.fq'?
Did you forget --single_end?</p>

<p>Should I ran eager with "single_end": true, ???</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-24 15:02:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-24 15:02:19"><div class="time">2022-10-24 15:02:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If they are already merged, then yes üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-24 15:03:13">
            
                <img src="https://secure.gravatar.com/avatar/9cc944effa4d07ca713e0495b26cec0a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ivany Argueta
                     <span class="print-only user-email">(ivanyjocelyne.arguetamejia.6320@student.uu.se)</span>
                </div>
                <a href="#2022-10-24 15:03:13"><div class="time">2022-10-24 15:03:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 15:23:37">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2022-10-26 15:23:37"><div class="time">2022-10-26 15:23:37</div></a>
                <div class="msg">
                    <p>Hi everyone üôÇ What is the (minimum) # of TOTAL reads people generally target for screening with the hope of picking up pathogens/specific microbes?
obv this is very context dependent, but a rough idea would be helpful. 1M reads vs 5M vs 100k</p>

<p>is there a minimum sequencing amount to even bother pathogen screening e.g. 100k reads / MiSeq data?
any thoughts or experience would be incredible!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Louis L&#39;H√¥te, Hannah Moots
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 16:03:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 16:03:52"><div class="time">2022-10-26 16:03:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I believe at EVA at least screening is either 5m or 10m? (Def. 10m for microbiome)</p>

<p>@aidanva @Alina Hiss @Megan Michel is it 5m still for pathogens?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 16:04:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 16:04:03"><div class="time">2022-10-26 16:04:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Gunnar Neumann too</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 16:08:23">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465355512771_d76bd23c72a4c5f61546_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Gunnar Neumann
                     <span class="print-only user-email">(gunnar_neumann@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 16:08:23"><div class="time">2022-10-26 16:08:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes, we still go for 5M if it‚Äôs from ShotGun data.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 16:45:18">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2022-10-26 16:45:18"><div class="time">2022-10-26 16:45:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes shotgun. 5M is definitely more than I had envisioned so can plan for more. Excellent, thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 16:52:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 16:52:40"><div class="time">2022-10-26 16:52:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Note thats coming from us who have cheap/routine sequencing with an internal core facility, it might be good to get input from others</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 16:53:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 16:53:19"><div class="time">2022-10-26 16:53:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'm going to askkk...</p>

<p>@Meriam Guellil @Miriam Bravo @Kelly Blevins @√Öshild (Ash) @Shreya @Pooja Swali</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 17:09:44">
            
                <img src="https://secure.gravatar.com/avatar/4b8f0795e46428bce5fe9524f3434a4f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pooja Swali
                     <span class="print-only user-email">(swalipooja@gmail.com)</span>
                </div>
                <a href="#2022-10-26 17:09:44"><div class="time">2022-10-26 17:09:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We're screening 5 million - I find that if it doesn't show up in the 5M (through kraken), then it tends not to even be good enough for capture (like very poor coverage/noisy damage etc) but this is just me and others may think different</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 17:14:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-10-26 17:14:58"><div class="time">2022-10-26 17:14:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I agree for bacterial species but for viral ones it can be a bit trickier so we usually do more. Plus it's also a matter of if you are only interested in knowing what you can get full genomes of easily or what is/could be in the sample.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 17:16:16">
            
                <img src="https://secure.gravatar.com/avatar/4b8f0795e46428bce5fe9524f3434a4f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pooja Swali
                     <span class="print-only user-email">(swalipooja@gmail.com)</span>
                </div>
                <a href="#2022-10-26 17:16:16"><div class="time">2022-10-26 17:16:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good point ‚òùÔ∏è I'm not looking for viral stuff</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:15:38">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2022-10-26 18:15:38"><div class="time">2022-10-26 18:15:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* 5M is generally a good number, although, echoing what Meriam said, if you want to screen for viruses then the more SG data you have to screen the better.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:30:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:30:23"><div class="time">2022-10-26 18:30:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I feel a collaborative micro blog post coming along (ping @Ele @Shreya)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üí° Kevin Daly
                        </div>
                    
                        <div class="message-reaction">
                        üôå Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:32:13">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2022-10-26 18:32:13"><div class="time">2022-10-26 18:32:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* hmmm viral reads are not a priority at the moment enough to push screening higher than 5M</p>

<p>these perspectives were super useful, thanks everyone üôè</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:33:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:33:19"><div class="time">2022-10-26 18:33:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Kevin Daly you might have volunteered yourself to write the micro blog post (basically summarise this thread ;))</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò∞ Kevin Daly
                        </div>
                    
                        <div class="message-reaction">
                        üòÇ Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:33:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-10-26 18:33:35"><div class="time">2022-10-26 18:33:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes I love it, popping a link to this thread in the blog channel to keep tabs on it!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-27 18:32:54">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2022-10-27 18:32:54"><div class="time">2022-10-27 18:32:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Late to the party, but I sequenced my SG libs to 8-10 million just because of how many samples I had and the HiSeq lane capacity. I was able to predict libraries that performed well for capture using pretty crude competitive mapping and breadth of ref coverage measures. I was only searching for M. tuberculosis though, so haven‚Äôt run the data through a pathogen database to see what all hits. Although I‚Äôm investigating a lib that seems positive for T. pallidum that was sequenced to 10 million but only has 31 reads mapping across the T. pallidum ref, so unsure if they would have been picked up if it were sequenced any shallower. Don‚Äôt know how/if a capture will perform though. I imagine your positive flags are heavily influenced by how deeply you sequence when you have a rich background of contaminant DNA hogging your clusters. When your target pathogen is &lt;.001%, stochasticity is bound to affects things.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container channel_join ">
        <div id="2022-10-26 16:03:55">
            
                <img src="https://secure.gravatar.com/avatar/0d057b6aaf83ddafcbe6723e21dd94d6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Megan Michel
                     <span class="print-only user-email">(megan_michel@g.harvard.edu)</span>
                </div>
                <a href="#2022-10-26 16:03:55"><div class="time">2022-10-26 16:03:55</div></a>
                <div class="msg">
                    <p>@Megan Michel has joined the channel</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:25:09">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-26 18:25:09"><div class="time">2022-10-26 18:25:09</div></a>
                <div class="msg">
                    <p>Hello people üôÇ do you know if we must do or are you doing normalisation of read counts in your OTU tables as a standard procedure? if so any recommendation on how you do it? Thanks üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòç Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:30:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:30:48"><div class="time">2022-10-26 18:30:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What sort of data and what purpose?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:34:06">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-26 18:34:06"><div class="time">2022-10-26 18:34:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Data are just kraken reports merged in a unique table so e.g. counts of reads in each species per each sample. Purpose for source tracking, metagenome composition..etc</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:34:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:34:34"><div class="time">2022-10-26 18:34:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is this microbiome data? Or just general pathogen screening for example?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:35:01">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-26 18:35:01"><div class="time">2022-10-26 18:35:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* hmmm ideally both</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:36:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:36:20"><div class="time">2022-10-26 18:36:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Source tracker you don't want to normalise</p>

<p>But for many other analyses (like classic stats) yes, would recommend it. @irinavelsko and I are strong proponents of CLR/ILR transformations due the CoDa principle (if you check my PNAS paper from last year I waffle on about that a bit in the SI)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:37:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:37:57"><div class="time">2022-10-26 18:37:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Pathogen screening I wouldn't normalise either</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:38:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:38:07"><div class="time">2022-10-26 18:38:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It doesn't help for anything really (that I can think of anyway?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:38:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 18:38:09"><div class="time">2022-10-26 18:38:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* )**</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:41:24">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-26 18:41:24"><div class="time">2022-10-26 18:41:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok I see it makes sense for the stats üôÇ thanks! I found that section in your paper üôÇ  I will have a look!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 18:52:41">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 18:52:41"><div class="time">2022-10-26 18:52:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maria Lopopolo and @James Fellows Yates CLR / ILR transform is used on compositional data, i.e. when you for example normalize your count matrix by library sizes, then your counts per library sum up to a certain number (like 1), so this is a compositional data, then indeed you should run a CLR / ILR on the top if you want to proceed with traditional stats like PCA, differential abundance, linear model etc. But Kraken counts that @Maria Lopopolo mentioned are not compositional data unless you normalize the raw counts by library sizes. In contrast, QIIME or MetaPhlan by default give you an abundance / OTU which is compositional, i.e. already normalized by library sizes, so-called TSS - total sum scaling normalization</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 19:00:47">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 19:00:47"><div class="time">2022-10-26 19:00:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Basically, my recommendation would be to normalize raw Kraken counts by TSS (library sizes) or CSS (more intelligent, cumulative sun scaling) and preferably by genome length (if you also want to compare abundances across species). Then you need to do CLR / ILR only on the top of TSS counts (compositional data), and can skip it at all if you use CSS normalization or genome length normalization (with either TSS or CSS). Sorry for very complicated explanation üòû </p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§Ø Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 19:08:15">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-26 19:08:15"><div class="time">2022-10-26 19:08:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Nikolay Oskolkov does this apply to krakenuniq too?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 19:11:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 19:11:18"><div class="time">2022-10-26 19:11:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes! Short summary: some library size normalization should be used, then you data may or may not become compositional (when abundances per library sum up to 1 for all libraries). If your abundance is compositional, then use CLR / ILR on the top, otherwise skip it</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 19:23:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 19:23:50"><div class="time">2022-10-26 19:23:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hm, if I'm following correctly, but raw read counts <em>are</em> compositional, as you're fixed by the capacity of the sequencing lane (which is a 'synthetic' cap)? I thought the CLR transformation is basically converting the variablty in the totals across different seq depth samples to make them compatible, i.e. I thought the CLR was doing the normalisation for you?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 19:34:19">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 19:34:19"><div class="time">2022-10-26 19:34:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates as far as I understand it, CLR is just a trick to transform your data from simplex space (compositional data) to euclidean space. Otherwise you e.g. have to run a PCA with aitchison (and not euclidean) distance. I agree that in theory many types of sequencing data are compositional, however as long as counts per library / sample do not strictly sum up to 1 across all libraries / samples, you should not worry about CLR. Otherwise, CLR should also be used e.g. in RNAseq, but I do not think people in this field heard about CLR üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:17:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 20:17:50"><div class="time">2022-10-26 20:17:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm ok... Im too tired to discuss it further now, but from what I remember and what I checked in these two reviews, they seem to imply you just need to give raw counts, each sample does not need to sum up to one: </p>

<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6755255/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6755255/</a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444|https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444|https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444</a></p>

<p>But it would be cool to talk about this in the future, as I imagine you understand this stuff just better than I do (I suck at maths and stats)</p>

<p>RNASeq people do know about CLR stuff but at that point they had equal or better performing methods such as DESeq</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed Central (PMC)</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6755255/">A field guide for the compositional analysis of any-omics data</a></div>
                                <div class="link-text">
                                    Next-generation sequencing (NGS) has made it possible to determine the sequence and relative abundance of all nucleotides in a biological or environmental sample. A cornerstone of NGS is the quantification of RNA or DNA presence as counts. However, these ...
                                </div>
                                
                                
    
        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6755255/">
            <img class="preview" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png?_=0"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6755255/</div>
                                
                                
                            
                        </div>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed Central (PMC)</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444">Microbiome Datasets Are Compositional: And This Is Not Optional</a></div>
                                <div class="link-text">
                                    Datasets collected by high-throughput sequencing (HTS) of 16S rRNA gene amplimers, metagenomes or metatranscriptomes are commonplace and being used to study human disease states, ecological differences between sites, and the built environment. There is ...
                                </div>
                                
                                
    
        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444">
            <img class="preview" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png?_=0"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134/#!po=6.94444</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:19:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 20:19:01"><div class="time">2022-10-26 20:19:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (sorry to hijack the thread @Maria Lopopolo)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:20:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 20:20:32"><div class="time">2022-10-26 20:20:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I also remember TSS being an alternative to CLR not applying it on top of (as also implied here: <a href="https://www.frontiersin.org/articles/10.3389/fams.2022.884810/full">https://www.frontiersin.org/articles/10.3389/fams.2022.884810/full</a>)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Frontiers</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.frontiersin.org/articles/10.3389/fams.2022.884810/full">A Survey of Statistical Methods for Microbiome Data Analysis</a></div>
                                <div class="link-text">
                                    In the last decade, numerous statistical methods have been developed for analyzing microbiome data generated from high-throughput next-generation sequencing technology. Microbiome data are typically characterized by zero inflation, overdispersion, high dimensionality, and sample heterogeneity. Three popular areas of interest in microbiome research requiring statistical methods that can account for the characterizations of microbiome data include detecting differentially abundant taxa across phenotype groups, identifying associations between the microbiome and covariates, and constructing microbiome networks to characterize ecological associations of microbes. These three areas are referred to as differential abundance analysis, integrative analysis, and network analysis, respectively. In this review, we highlight available statistical methods for differential abundance analysis, integrative analysis, and network analysis that have greatly advanced microbiome research. In addition, we discuss each method&#39;s motivation, modeling framework, and application.
                                </div>
                                
                                
    
        <a href="https://www.frontiersin.org/articles/10.3389/fams.2022.884810/full">
            <img class="preview" src="https://www.frontiersin.org/files/MyHome%20Article%20Library/884810/884810_Thumb_400.jpg"
                width="400"
                 height="111" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.frontiersin.org/articles/10.3389/fams.2022.884810/full</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:25:00">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-26 20:25:00"><div class="time">2022-10-26 20:25:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No worries @James Fellows Yates and @Nikolay Oskolkov these are the conversations that probably will be useful to many and that I need to understand too (even if you are speaking chinese rn). I guess one more topic to discuss for common mortals in SPAAMtisch :mask_parrot:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:26:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 20:26:43"><div class="time">2022-10-26 20:26:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Note I'm generally agreeing with @Nikolay Oskolkov otherwise ;)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:27:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 20:27:49"><div class="time">2022-10-26 20:27:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Just whether CLR/ILR is an alternative or on top of TSS for differential abundance of raw counts.</p>

<p>Otherwise like he said MetaPhlAn has their own normalisation</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:27:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-26 20:27:53"><div class="time">2022-10-26 20:27:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Etc etc</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:52:02">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 20:52:02"><div class="time">2022-10-26 20:52:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @James Fellows Yates, I certainly do not have a good literature overview in this field, and therefore appreciate to know your point of view. However, I have my own experience of trying to understand why compositional data are generally considered to be ‚Äúdangerous‚Äù in mathematical statistics. It has something to do with features (microbes, genes etc.) becoming artificially correlated for compositional data. In this post I tried to demonstrate how this ‚Äúspurious‚Äù correlation (discovered by Karl Pearson) can arise when a random count matrix is library size (TSS) normalized: <a href="https://towardsdatascience.com/pitfalls-of-data-normalization-bf05d65f1f4c">https://towardsdatascience.com/pitfalls-of-data-normalization-bf05d65f1f4c</a> (sorry for self-promotion). It is a super-simple code that everyone can reproduce and see how uncorrelated features become ‚Äúspuriously‚Äù correlated after TSS. Basically TSS should never be used because of that reason, but it is still widely used in metagenomics / 16S. Therefore to fix it people came up with the CLR-transform which is (if you check) a combination of log-functions and thus resembles the aitchison distance, which is a more proper distance metric when a constraint has been applied to a data (and summing up counts to 1 is a constraint). In RNAseq people used library size normalization in the early days but quickly gave up because of the danger of compositional data, and in addition equalizing sample / cell libraries seemed like an over-normalization that destroyed all the interesting heterogeneity across samples / cells</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Medium</div>
                            
                            
                                
                                <div class="link-title"><a href="https://towardsdatascience.com/pitfalls-of-data-normalization-bf05d65f1f4c">Pitfalls of Data Normalization</a></div>
                                <div class="link-text">
                                    How normalization leads to spurious correlations in Simplex
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Reading time</div>
                                        6 min read
                                    </div>
                                
                                
    
        <a href="https://towardsdatascience.com/pitfalls-of-data-normalization-bf05d65f1f4c">
            <img class="preview" src="https://miro.medium.com/max/1200/1*TC4HsDUv9V3AetnDVHa9dw.jpeg"
                width="444"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://towardsdatascience.com/pitfalls-of-data-normalization-bf05d65f1f4c</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:53:55">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2022-10-26 20:53:55"><div class="time">2022-10-26 20:53:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov I am bit confused on how library size normalization would not make your microbiome compositional. Based on my understanding, I agree with James that sequencing capacity (along with other non-biological factors) will always make your data compositional. For instance, we can never 'know' whether the zeros in our datasets are due to sequencing capabilities or because they are biologically not present there for a reason. Could you clarify or elaborate more on how the data is not compositional.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 20:55:55">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 20:55:55"><div class="time">2022-10-26 20:55:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Applying CLR on raw counts looks good to me too. It seems similar to using a simple log-transform on raw counts that makes count data more ‚Äúeuclidean‚Äù. In scRNAseq for example this simple log-transform is considered as a weak normalization that brings counts from different libraries on the same scale which is often sufficient. Although I do not think that log-transform actually properly equalizes library sizes, I believe it is better than nothing</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:09:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 21:09:25"><div class="time">2022-10-26 21:09:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Sterling Wright compositional data is data where rowSums or colSums (in R language) results in the same fixed value (often 1) for all raws / columns. You can take Kraken counts from multiple samples and check whether summing up counts (per sample) across all microbes results in the same fixed value for all samples. I bet it will not. So your Kraken data are not compositional (at least in mathematical sense). Now you can take MetaPhlan (default settings) abundances and run rowSums / colSums, I bet you will get same fixed value for each of your samples. So your MetaPhlan data are compositional and you should not naively run PCA on compositional data. Now, here comes the question: both Kraken and MetaPhlan were run on the same fastq-files. How come the same data become compositional for MetaPhlan and do not for Kaken? The thing is that, MetaPhlan was normalizing the raw abundance counts with library sizes (without telling you), while Kraken did not do it. So irrespective of sequencing capacity constraint, one tool gives you compositional and the other one gives non-compositional data. Again, I am talking about mathematical definition, not the biological natural constraints </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:15:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 21:15:22"><div class="time">2022-10-26 21:15:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Library size normalization will always give you compositional data while sequencing (lane capacity) does not result per se in compositional data (at least the raw counts do not sum up to one unless they have been bioinformatically library size normalized) üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:28:47">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2022-10-26 21:28:47"><div class="time">2022-10-26 21:28:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay I see how that would make mathematical sense. I have to admit that I have not dealt with Kraken datasets, only MALT. And so this maybe one of the reasons on why I am confused. However, out of curiosity, could you explain whether this library size normalization procedure would be impacted if you changed the sequencing depth on the same dataset. I have a clearer understanding for this the log transformation procedures. CLR/ILR/ALR transformations are applied to feature/OTU tables because if you stuck with count data you are going to get spurious correlations as soon as you change the sequencing depth. If you were to remove one read or count of a feature, then the proportion of all other features go up and the proportion of that one feature decreases. On the surface, you would think that all the other features are positively correlated with each other even though their counts did not change and only the selected feature did.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:32:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:32:11"><div class="time">2022-10-26 21:32:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Very nice Thread here! And very relevant and important for SPAAM</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:34:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:34:05"><div class="time">2022-10-26 21:34:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am totally with @Nikolay Oskolkov in all that he explained. This is what I have always done when analyzing metagenomic data (i.e., some sort of normalization by sequencing effort per sample, to make data compositional) and then some sort of transformation</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:35:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:35:44"><div class="time">2022-10-26 21:35:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* the simplest I was doing was to normalize by the total number of reads per sample and then using the VST transformation in DESseq2</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:36:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:36:03"><div class="time">2022-10-26 21:36:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* but I think that what Nikolay proposed is probably better</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:37:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:37:07"><div class="time">2022-10-26 21:37:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* in any case, in my opinion comparing datasets with very different sequencing depths is always risky, because of the 0s that Sterling was mentioning. You are not able to see the same richness in highly and lowly sequenced samples</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:37:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:37:23"><div class="time">2022-10-26 21:37:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* so some considerations on the taxa to be measured should be considered.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:38:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:38:13"><div class="time">2022-10-26 21:38:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Some people downsample reads in samples to the same number of read counts per dataset (QIIME was doing that when I used to use it)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:44:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-26 21:44:28"><div class="time">2022-10-26 21:44:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Sterling Wright I believe you see your abundance data as proportions / fractions, not absolute counts. This is similar to 16S / OTU / QIIME terminology. If your abundances are fractions then they naturally sum up to 1 and the microbes are correlated by that constraint. But I think the constraint that resulted in the proportions is a computational thing, i.e. if one wouldn‚Äôt have applied library size normalization, the data would not have been strictly compositional</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:54:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:54:33"><div class="time">2022-10-26 21:54:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think that it would be great that the SPAAM community converge to a recommended practice around this topic, for instance, to compare ancient microbiome samples for beta and alpha diversity, to identify taxa that are significantly changing between conditions, etc.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:55:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-26 21:55:03"><div class="time">2022-10-26 21:55:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If the input data is not properly treated, then results are just meaningless</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-26 21:59:12">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2022-10-26 21:59:12"><div class="time">2022-10-26 21:59:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I agree very much. If we want our results to be reproducible and try to minimize the chance of results being inconsistent due to pipeline differences, I think there is a lot of work that needs to be done in this space.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-27 11:19:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-27 11:19:01"><div class="time">2022-10-27 11:19:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov I'm catching up now with more sleep - so technically you're saying if you do TSS scaling you <em>must</em> use something like CLR.</p>

<p>However, I guess my question is <em>must</em> you do TSS scaling to do CLR, or can you apply CLR directrly on raw counts (which you appear to imply above, and is what most of the tutorials/microbiome papers on the topic also seem to suggest)?</p>

<p>also don't feel shy to self-promote, your blog posts are very good</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 19:03:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-28 19:03:25"><div class="time">2022-10-28 19:03:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @James Fellows Yates, apologies for the delay, I am also trying to find time to sleep (sometimes) with my three kids üôÇ I agree, you must always use CLR after TSS, and you must not do TSS in order to be able to run CLR üôÇ However, a simple CLR alone does not remove the dependence of microbial abundance on sequencing depth, therefore you should equalize somehow library sizes via TSS or CSS to correct for the technical variation in sequencing depth between your samples.</p>

<p>I took a Kraken abundance matrix from one of my project and compared how a mean abundance count changes vs sequencing depth (it should not depend on sequencing depth) for unnormalized, CLR normalized, library size normalized (TSS), and library size+CLR normalized counts, please see below. I am sorry, I can not share the abundance matrix, but I am sharing the code I used below so that you could understand how the plots were computed, and you could also check how the plots look like for a Kraken abundance matrix (columns are samples and rows are microbes) from your favorite project.</p>

<p>```library("compositions")</p>

<h1>Reading, adding offset +1 (to get rid of zeroes), displaying and storing library size information</h1>

<p>df&lt;-read.delim("abundance<em>matrix.txt",header=TRUE,row.names=1,check.names=FALSE,sep="\t")
df&lt;-df+1
head(df)
lib</em>size&lt;-colSums(df)</p>

<p>par(mfrow=c(2,2))</p>

<h1>Unnormalized data</h1>

<p>plot(log10(colMeans(df)+1)~log10(lib<em>size+1),main="Unnormalized data",ylab="log10 ( Mean Count )",xlab="log10 ( Sequencing Depth )")
abline(lm(log10(colMeans(df)+1)~log10(lib</em>size+1)))</p>

<h1>CLR-transformed data</h1>

<p>df<em>clr&lt;-clr(df)+20
head(df</em>clr)
plot(log10(colMeans(df<em>clr)+1)~log10(lib</em>size+1),main="CLR normalized data",ylab="log10 ( Mean Count )",xlab="log10 ( Sequencing Depth )",
     ylim=c(1.25,1.45))
abline(lm(log10(colMeans(df<em>clr)+1)~log10(lib</em>size+1)))</p>

<h1>Library-size normalized data</h1>

<p>df<em>lib</em>size&lt;-matrix(nrow=dim(df)[1],ncol=dim(df)[2])
for(i in 1:dim(df)[2])
{
  df<em>lib</em>size[,i]&lt;-(df[,i]/sum(df[,i]))
}
head(df<em>lib</em>size)
plot(log10(colMeans(df<em>lib</em>size)+1)~log10(lib<em>size+1),main="Library size normalized data",
     ylab="log10 ( Mean Count )",xlab="log10 ( Sequencing Depth )")
abline(lm(log10(colMeans(df</em>lib<em>size)+1)~log10(lib</em>size+1)))</p>

<h1>Library-size normalized + CLR-transformed data</h1>

<p>df<em>lib</em>size&lt;-df<em>lib</em>size+1
df<em>lib</em>size<em>clr&lt;-clr(df</em>lib<em>size)+20
head(df</em>lib<em>size</em>clr)
plot(log10(colMeans(df<em>lib</em>size<em>clr)+1)~log10(lib</em>size+1),main="Library size + CLR normalized data",
     ylab="log10 ( Mean Count )",xlab="log10 ( Sequencing Depth )",ylim=c(1.25,1.45))
abline(lm(log10(colMeans(df<em>lib</em>size<em>clr)+1)~log10(lib</em>size+1)))```</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F0485M22SGP/norm_kraken_counts.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">norm_kraken_counts.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F0485M22SGP/norm_kraken_counts.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F0485M22SGP-4b215dfd6b/norm_kraken_counts_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="277" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 19:05:03">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-28 19:05:03"><div class="time">2022-10-28 19:05:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* From the figure above, my conclusion is that, CLR alone is not sufficient to correct for technical variation in sequencing depth</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Sterling Wright
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 19:32:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-28 19:32:52"><div class="time">2022-10-28 19:32:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Nikolay Oskolkov (alwasy prioritise sleep over SPAAM btw ;)), but those plots are really helpful to demonstrate. This would be another nice blog post I think.</p>

<p>So to conclude for @Maria Lopopolo (Nikolay correct me if I'm wrong) -</p>

<p>Pathogen screening - no normalisation needed</p>

<p>Microbiome work: normalisation already happens for some tools for you (MetaPhlAn3, etc). But if you have raw read counts (Kraken2/MALT etc.)  it is possible for a quick and dirty normalisation to just run a CLR/PhILR transformation to do <em>basic</em> exploratory analyses (maybe PCA, for example). However to do it <em>properly</em>, i.e. before doing full statistical tests for differential abundance to make biological inference from, you <em>should</em> normalise by TSS/CSS <em>then</em> perform a log ratio transform (CLR/PhILR) etc.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 19:34:05">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-10-28 19:34:05"><div class="time">2022-10-28 19:34:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, good summary @James Fellows Yates!</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 19:34:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-28 19:34:48"><div class="time">2022-10-28 19:34:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (now we wait for @Shreya and @Ele to bug us üòâ but maybe we could do a little collab. one between the three of us?),</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        ü§© Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 20:11:26">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-10-28 20:11:26"><div class="time">2022-10-28 20:11:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates and @Nikolay Oskolkov thank you so much for this! I appreciate you both taking the time to disentangle this from my no-stupid-question üòä</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 20:27:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-10-28 20:27:17"><div class="time">2022-10-28 20:27:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Clearly not a stupid question!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-28 20:31:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-10-28 20:31:52"><div class="time">2022-10-28 20:31:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It was an awesome thread! Very thanks to @Nikolay Oskolkov!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-31 15:19:55">
            
                <img src="https://secure.gravatar.com/avatar/b6d82a8d6c1082eefeea8152795b0bb6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Claudio Ottoni
                     <span class="print-only user-email">(claudio.ottoni@uniroma2.it)</span>
                </div>
                <a href="#2022-10-31 15:19:55"><div class="time">2022-10-31 15:19:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi everyone thanks for the fantastic thread! (I managed to read through only now). We also took as routine to do TCC+CLR for microbiome data. Quick question @Nikolay Oskolkov (and the others), so after clr-normalization which distance can be used (e.g for cluster analysis)? Bray-Curtis? Or better go for Aitchinson distance on the non-clr-normalized data? (is it eventually equivalent? I never really tested that systematically but it seems not).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 12:01:52">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-01 12:01:52"><div class="time">2022-11-01 12:01:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Claudio Ottoni I personally have not tested Bray-Curtis (on TSS+CLR normalized) vs Aitchison on TSS counts for clustering, so do not unfortunately have a good answer üòû Many things to test but very little time...</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Claudio Ottoni
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-16 15:45:06">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2022-12-16 15:45:06"><div class="time">2022-12-16 15:45:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates and @Nikolay Oskolkov going back to this much discussed topic eheheh what to do when your geometric mean it‚Äôs 0 bc many species in the table have ‚Äú0‚Äù  for many samples? Is it here that you consider filtering out species that are 0 for e.g &gt; 50% of the samples?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-16 21:17:49">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-16 21:17:49"><div class="time">2022-12-16 21:17:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maria Lopopolo geometric mean on zero-inflated matrix of counts is problematic to compute (as you said it easily becomes zero if at least one element in the vector is zero). Therefore, for example, people do not use DESeq / TMM normalization (which are based on computing geometric means) for single cell RNAseq (zero-inflated) while DESeq / TMM are gold standard for bulk RNAseq (non-zero-inflated). Yes, for zero-inflated data (and I would say metegenomic data are zero-inflated) I would either harshly filter lowly abundant species (to justify the use of geometric mean normalization) or use some normalization that is not based on geometric means</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-30 16:31:28">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2022-10-30 16:31:28"><div class="time">2022-10-30 16:31:28</div></a>
                <div class="msg">
                    <p>Hi people:) how much it would cost for per sample from extraction to sequencing in an aDNA lab in your case? I was late to know and start a grant application due on tomorrow night, and I need to quote the spending, so it would be great if you can share the info with me. Thanks!! </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-30 19:59:34">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2022-10-30 19:59:34"><div class="time">2022-10-30 19:59:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Yuti, Could you provide a little more information? Do you plan on using shotgun sequencing or capture? What type of samples will you be looking at? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-30 20:24:31">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2022-10-30 20:24:31"><div class="time">2022-10-30 20:24:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Sterling, I plan to use shotgun, and it‚Äôs dental calculus</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-30 22:48:47">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2022-10-30 22:48:47"><div class="time">2022-10-30 22:48:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Got it. We budget $20 per sample for dna extraction and $120 per sample for double stranded libraries. But a big question will be how deep you want to sequence your samples and how many samples you have. Sequencing deeper depths will be better but will be more expensive. Do you have an idea which sequencer you will use? Also do you have an idea about the preservation of the samples </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-31 02:52:47">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2022-10-31 02:52:47"><div class="time">2022-10-31 02:52:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for the info Sterling!! I have 29 samples, and to be honest I have no idea yet how deep I am going to sequence, I only worked with live host samples before,  the dental calculus samples I have now are from primate collection in museum. I think I will say around 100~150 for the grant purpose so far </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-31 16:00:42">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2022-10-31 16:00:42"><div class="time">2022-10-31 16:00:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Got it. With that many samples and that much money, you should have enough for a hiseq run, maybe even a novaseq run. This does depend on where you send your samples and what kind of contracts your institution has with sequencing facilities. But hopefully this helps </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-31 17:42:59">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2022-10-31 17:42:59"><div class="time">2022-10-31 17:42:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I second these estimates! We budget $150 for extractions plus double stranded libraries per samples and then the major cost is sequencing, which will vary based on depth of coverage desired and prices at the sequencing center you work with </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 01:21:59">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2022-11-01 01:21:59"><div class="time">2022-11-01 01:21:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Hannah! </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-31 22:16:53">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-10-31 22:16:53"><div class="time">2022-10-31 22:16:53</div></a>
                <div class="msg">
                    <p>Hi all! Maybe I've just missed this in the documentation, but for users of the new aMeta pipeline, which databases are required and which are optional? Followup simple unix question - Is there an efficient way to "wget"/download these via command line? I'd prefer not to download GBs of data locally just to move them to a remote cluster later.  Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-10-31 23:20:19">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-10-31 23:20:19"><div class="time">2022-10-31 23:20:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Scratch part 2 - figured it out with a clever renaming on command line!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 05:53:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-11-01 05:53:35"><div class="time">2022-11-01 05:53:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Zo√© Pochon @Nora Bergfeldt @Nikolay Oskolkov üëÜ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 12:10:38">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-01 12:10:38"><div class="time">2022-11-01 12:10:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Carly Scott, sorry for the delayed reply! If you follow a simple installation via:</p>

<p><code>1) git clone &lt;https://github.com/NBISweden/aMeta&gt;
2) cd aMeta
3) mamba env create -f workflow/envs/environment.yaml
4) conda activate aMeta (alternatively if this fails, try conda activate ancient_microbiome_workflow)
5) cd .test
6) ./runtest.sh -j 4</code>
there is a toy KrakenUniq database and Bowtie index built for testing purposes. Please make sure that everything works for you on the testing step. For real world projects you should use one of the following databases here:</p>

<p><code>KrakenUniq database based on full NCBI NT: &lt;https://doi.org/10.17044/scilifelab.20205504&gt;
KrakenUniq database based on microbial part of NCBI NT: &lt;https://doi.org/10.17044/scilifelab.20518251&gt;
KrakenUniq database based on microbial part of NCBI RefSeq: &lt;https://doi.org/10.17044/scilifelab.21299541&gt;
Bowtie2 index for full NCBI NT database: &lt;https://doi.org/10.17044/scilifelab.21070063&gt;
Bowtie2 index for pathogenic microbial species of NCBI NT: &lt;https://doi.org/10.17044/scilifelab.21185887&gt;</code>
I personally would recommend to use the microbial NCBI NT (if you are after microbes only and less after eukaryotes) or full NCBI NT (if you are after both eukaryotes and prokaryotes) for KrakenUniq part of the workflow, and full NCBI NT Bowtie2 index for following up the kmer classification. You should not be worried about the MALT followup step as this should be tuned automatically, i.e. the database is built dynamically for each project. If you run into installation / execution or resource issues, please let me know</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 20:00:09">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-01 20:00:09"><div class="time">2022-11-01 20:00:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Awesome, thanks for the info! This clears thing up - I'll let you know if I bump into anything else as I run it üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 17:04:38">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-07 17:04:38"><div class="time">2022-11-07 17:04:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Quick followup: What is "PathogensFound.tab"? Is this a list of potential pathogens that should be fed into Kraken or the output of Krakenuniq?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 17:07:20">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-07 17:07:20"><div class="time">2022-11-07 17:07:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>MissingInputException in rule Filter_KrakenUniq_Output  in line 25 of /scratch/06909/cbscott/ancient_metagenomics/aMeta/workflow/rules/krakenuniq.smk:
Missing input files for rule Filter_KrakenUniq_Output:
    output: results/KRAKENUNIQ/S30455/krakenuniq.output.filtered, results/KRAKENUNIQ/S30455/krakenuniq.output.pathogens, results/KRAKENUNIQ/S30455/taxID.pathogens
    wildcards: sample=S30455
    affected files:
        resources/pathogensFound.tab</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 17:07:59">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-07 17:07:59"><div class="time">2022-11-07 17:07:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ^is what I'm specifically bumping into - I'm just not sure what input could be missing here, if you have any insight?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 17:11:42">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-07 17:11:42"><div class="time">2022-11-07 17:11:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Carly Scott, yes, this is our very permissive custom list of microbial pathogens that was built based on extensive literature search. This file should be present at the aMeta github repo, I will double check this, could you please also locate this file in your cloned repo?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 18:14:26">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-07 18:14:26"><div class="time">2022-11-07 18:14:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, got it @Nikolay Oskolkov - it is subset in the hidden ".test/resources" folder.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 18:19:59">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-07 18:19:59"><div class="time">2022-11-07 18:19:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Though this looks like a small subset of potential species (just the test file). Should these files have been downloaded through the configuration of the software? In a fresh clone of the git repo in my environment I'm not seeing it.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 19:36:39">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-07 19:36:39"><div class="time">2022-11-07 19:36:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Carly Scott I have checked this and this file is apparently not in the aMeta repo but it is in the PathoGenome <a href="https://doi.org/10.17044/scilifelab.21185887">https://doi.org/10.17044/scilifelab.21185887</a> that we published together with the workflow. You can download the pathogensFound.very_inclusive.tab file, put it to <code>aMeta/resources</code> and provide the path to the file in the config.yaml via <code>pathogenomesFound: resources/pathogenomesFound.tab</code></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">figshare</div>
                            
                            
                                
                                <div class="link-title"><a href="https://doi.org/10.17044/scilifelab.21185887">Bowtie2 index for pathogenic microbial species of NCBI NT from May 2020</a></div>
                                <div class="link-text">
                                    This is a Bowtie2 index built using a subset of NCBI non-redundant NT / NR records from May 2020, &lt;https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nt.gz&gt;, that includes only microbial pathogenic organisms. The Bowtie2 index was built using a custom very permissive list (e.g. opportunistic pathogens were included) pathogensFound.very_inclusive.tab of pathogenic microbes based on extensive literature search. The Bowtie2 index can be used in metagenomic projects for alignment against pathogenic references and detecting pathogenic organisms present in metagenomic samples. The index was built by Nikolay Oskolkov, Lund University, Sweden, within the NBIS SciLifeLab long-term support project, PI Anders G√∂therstr√∂m, Centre for Palaeogenetics, Stockholm, Sweden. The index was build using Bowtie2 version 2.3.5.1 using a computer node with 1 TB of RAM on Uppmax SUPR /SNIC HPC cluster.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://doi.org/10.17044/scilifelab.21185887</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 19:38:04">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-07 19:38:04"><div class="time">2022-11-07 19:38:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* sorry about it, we should have included this file in the resources-directory in the aMeta repo, I will open an issue for that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 19:40:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-11-07 19:40:10"><div class="time">2022-11-07 19:40:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* How the config file in a project folder looks like:
```samplesheet: resources/samples.tsv</p>

<p>analyses:
  mapdamage: true
  authentication: true
  malt: true</p>

<h1>Databases !!!! CHOOSE BETWEEN FULL_NT AND MICROBIAL</h1>

<p>krakenuniq<em>db: /proj/nobackup/metagenomics/databases/DBDIR</em>KrakenUniq<em>MicrobialNT</em>Plus_CompleteGenomes</p>

<h1>krakenuniq<em>db: /proj/nobackup/metagenomics/databases/DBDIR</em>KrakenUniq<em>Full</em>NT</h1>

<p>bowtie2<em>patho</em>db: /proj/nobackup/metagenomics/databases/PathoGenome/library.pathogen.fna
pathogenomesFound: /proj/nobackup/metagenomics/databases/PathoGenome/pathogensFound.very<em>inclusive.tab
pathogenome</em>seqid2taxid<em>db: /proj/nobackup/metagenomics/databases/PathoGenome/seqid2taxid.pathogen.map
malt</em>seqid2taxid<em>db: /proj/nobackup/metagenomics/databases/DBDIR</em>KrakenUniq<em>Full</em>NT/seqid2taxid.map.orig
malt<em>nt</em>fasta: /proj/nobackup/metagenomics/databases/DBDIR<em>KrakenUniq</em>Full<em>NT/library/nt/library.fna
malt</em>accession2taxid: /proj/nobackup/metagenomics/databases/DBDIR<em>KrakenUniq</em>Full<em>NT/taxonomy/nucl</em>gb.accession2taxid
ncbi<em>db: resources/ncbi
krona</em>db: /pfs/data/databases/KronaTools/20220207/</p>

<h1>Additional config</h1>

<p>n<em>unique</em>kmers: 1000
n<em>tax</em>reads: 200```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 19:43:04">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-07 19:43:04"><div class="time">2022-11-07 19:43:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, thank you @Zo√© Pochon! @Carly Scott needs to change this line "pathogenomesFound: /proj/nobackup/metagenomics/databases/PathoGenome/pathogensFound.very_inclusive.tab" after she has downloaded <code>pathogensFound.very_inclusive.tab</code> file from <a href="https://doi.org/10.17044/scilifelab.21185887">https://doi.org/10.17044/scilifelab.21185887</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 21:25:06">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-07 21:25:06"><div class="time">2022-11-07 21:25:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you @Nikolay Oskolkov and @Zo√© Pochon - I missed it by downloading the full nt bowtie index instead of the microbial file. I've got it now - I'll let you know if it works üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 17:39:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-14 17:39:21"><div class="time">2022-11-14 17:39:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Carly Scott, we updated the docs (README) for aMeta <a href="https://github.com/NBISweden/aMeta">https://github.com/NBISweden/aMeta</a>, it should now be much easier to install and quick start running it on your samples. We put references for all databases mentioned in the configuration file, so hopefully you can download them and run aMeta. Please let me know if you still experience difficulties and I will assist you</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/NBISweden/aMeta">GitHub - NBISweden/aMeta: Ancient microbiome snakemake workflow</a></div>
                                <div class="link-text">
                                    Ancient microbiome snakemake workflow. Contribute to NBISweden/aMeta development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/NBISweden/aMeta">
            <img class="preview" src="https://opengraph.githubassets.com/1317dd5db1b9c6915d41ee1ef001779f94bc5aaa34a5283d05e6c6fe9ea64621/NBISweden/aMeta"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/NBISweden/aMeta</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 20:36:20">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-14 20:36:20"><div class="time">2022-11-14 20:36:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for the update @Nikolay Oskolkov. So far things seem to be running on my end - I appreciate the guidance. A quick followup, as I'm working with non-human samples: is the literature-search for PathogensFound based on pathogenic microbes specifically in humans or across taxa?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 21:40:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-14 21:40:40"><div class="time">2022-11-14 21:40:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Glad it is working for you @Carly Scott! The pathogenic microbes are not human-specific, many of them are plant and animal pathogens. The list is very comprehensive and permissive in order not to miss anything interesting, however some manual (common sense) verification is still required. So the fact that they are called "pathogens" in our list does not fully guarantee that they really are, but they rather should be considered as good pathogenic candidates. Do not hesitate to contact me if you have any question or problems with aMeta üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon, Carly Scott
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:08:19">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-18 21:08:19"><div class="time">2022-11-18 21:08:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi again! Is there a way to turn adapter trimming off for aMeta - I've already trimmed my files and removed human contamination,  which I think is causing the following error:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:09:27">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-18 21:09:27"><div class="time">2022-11-18 21:09:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>Job 2: FastQC_BeforeTrimming: RUNNING QUALITY CONTROL WITH FASTQC FOR SAMPLE /work/06909/cbscott/ls6/aDNA_2022/S30455.short.fastq BEFORE TRIMMING ADAPTERS
Reason: Missing output files: results/FASTQC_BEFORE_TRIMMING/S30455_fastqc.zip 
Waiting at most 5 seconds for missing files.
MissingOutputException in rule FastQC_BeforeTrimming in line 1 of /scratch/06909/cbscott/aMeta/workflow/rules/qc.smk:
Job Missing files after 5 seconds.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:21:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-18 21:21:21"><div class="time">2022-11-18 21:21:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Carly Scott could you post me a few lines from the samples .tsv file? It does not seem like an error due to adapter trimming because this is a QC before adapter trimming, i.e. the very first step of the workflow. I believe the workflow can not find the input files, therefore would like to see your samples.tsv</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:22:42">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-18 21:22:42"><div class="time">2022-11-18 21:22:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah! I've got it.  You're right it wasn't a trimming issue, it's a file naming convention issue. Sorry to bug yoU!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:23:10">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-11-18 21:23:10"><div class="time">2022-11-18 21:23:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I named my files NAME.descriptor.descriptor.fastq, and it seems they didn't parse correctly</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:23:15">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-18 21:23:15"><div class="time">2022-11-18 21:23:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Adapter trimming does not take much time and from my experience trimming adapters once might result in some adapter left overs, so trimming an extra time can be only beneficial</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Carly Scott
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-18 21:25:20">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-18 21:25:20"><div class="time">2022-11-18 21:25:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No problems at all, do not hesitate to post your questions here or you can send me a private message as well</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 00:52:47">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 00:52:47"><div class="time">2022-12-02 00:52:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Nikolay Oskolkov, apologies to bother you again! I am trying to run aMeta with a custom database (of marine microbes), and have built my botwtie2 database separately. However, aMeta insists on overwriting these files and recreating the bowtie2 database again, which is really slowing down the software. Do you have any insight to why it might be doing this? I'm fairly certain the database is complete, despite the error message, as the job completes without any issues on my end (using the same script as aMeta, but parallelized).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 00:53:10">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 00:53:10"><div class="time">2022-12-02 00:53:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>bowtie2-build-l resources/bt2db/full.library.fna resources/bt2db/full.library.fna</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 00:53:57">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 00:53:57"><div class="time">2022-12-02 00:53:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The overwrite process:
<code>[Thu Dec 1 17:45:31 2022]</code>
<code>rule Bowtie2_Index:</code>
  <code>input: resources/bt2db/full.library.fna</code>
  <code>output: resources/bt2db/full.library.fna.1.bt2l, resources/bt2db/full.library.fna.2.bt2l, resources/bt2db/full.library.fna.3.bt2l, resources/bt2db/full.library.fna.4.bt2l, resources/bt2db/full.library.fna.rev.1.bt2l, resources/bt2db/full.library.fna.rev.2.bt2l</code> <strong>these outputs already exist in the directory, from my local build</strong>
  <code>log: logs/BOWTIE2_BUILD/resources/bt2db/full.library.fna.log</code>
  <code>jobid: 6</code>
  <code>reason: Forced execution</code>
  <code>resources: tmpdir=/tmp</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 13:12:39">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 13:12:39"><div class="time">2022-12-02 13:12:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Carly Scott, thanks for reporting, please do bother me with your questions, we want to improve aMeta! Regarding your question, did you use <code>--forceall</code> flag? If so, snakemake will try to execute absolutely all rules even if some of them can be skipped like for the case of pre-built custom Bowtie2 index</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 18:21:24">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 18:21:24"><div class="time">2022-12-02 18:21:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks!
I did not use the --forceall flag. I ran it with: <code>snakemake --snakefile workflow/Snakefile -j 10</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 18:34:41">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 18:34:41"><div class="time">2022-12-02 18:34:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Carly Scott it looks like your <code>resources/bt2db/full.library.fna</code> file is newer than the index (**.bt2l files), can it be the case? Only in this case aMeta tries to rebuild the index. I am also puzzled about the <code>reason: Forced execution</code> , this message means that you must have used <code>--forceall</code>, but if you did not, this sounds strange</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 18:39:04">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 18:39:04"><div class="time">2022-12-02 18:39:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Carly Scott would you mind checking the creation dates of your <code>resources/bt2db/full.library.fna</code> and the index (**.bt2l files) in the <code>resources/bt2db</code> folder?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:09:57">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:09:57"><div class="time">2022-12-02 19:09:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah you know what might have happened - I created all of the files in a secondary location to keep them from being overwritten, and then created a copy in resources/bt2db (<code>cp my_backup_dir/full.library.** resources/bt2db</code>), which put the same timestamp on all of the files (see screenshot). Let me redo this by moving first full.library.fna over, followed by the .bt2l files and try rerunning.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F04DC2U3Z0E/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F04DC2U3Z0E/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F04DC2U3Z0E-fd54f9bb8f/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="96" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:17:56">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:17:56"><div class="time">2022-12-02 19:17:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hm, it thinks the files are incomplete:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F04DM5P8PBN/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F04DM5P8PBN/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F04DM5P8PBN-1487bc0c0c/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="103" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F04D45PV4LF/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F04D45PV4LF/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F04D45PV4LF-aedd15badd/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="91" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:18:52">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:18:52"><div class="time">2022-12-02 19:18:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'll create the bt2 index in the directory and see what happens. I'm also not opposed to letting aMeta recreate it, but it's not a memory intensive task for this database, so I've been parallelizing it separately from aMeta</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:19:48">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 19:19:48"><div class="time">2022-12-02 19:19:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Please try to use the <code>--rerun-incomplete</code> flag and see whether aMeta still insists on rebuilding the index</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:21:10">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:21:10"><div class="time">2022-12-02 19:21:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* iT does - this is what I started with during the troubleshooting, but I can send over the message if you would like.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:27:37">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 19:27:37"><div class="time">2022-12-02 19:27:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Carly Scott your custom Bowtie2 database does probably not match the KrakenUniq DB? I mean, those two had presumably different reference fasta files, or did you build a KrakenUniq DB from the same <code>resources/bt2db/full.library.fna</code>?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:31:08">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:31:08"><div class="time">2022-12-02 19:31:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So full.library.fna is a composite of ~1,000 genomes which I catted together. I think I built the KrakenUniqDB based on the individual genomes in a directory, but the bowtieDB on the full.library.fna. I can go back and build them based on the same reference</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:31:20">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:31:20"><div class="time">2022-12-02 19:31:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Generally does aMeta need identical Kraken and Bowtie databases?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:41:38">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 19:41:38"><div class="time">2022-12-02 19:41:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Carly Scott yes, KrakenUniq, Bowtie2 and Malt databases should use exactly the same fasta-file (same headers and seqs). By design, aMeta was not a collection of tools that can be used individually, but they all are supposed to work together and be consistent with each other. Therefore, it is not straightforward to replace a Bowtie2 index without modifying the KrakenUniq DB, unfortunately. In your case, if the concatenated fasta contains exactly the same reference sequences as in the directory with the individual genomes (same headers and seqs), it seems your KrakenUniq DB matches your Bowtie 2 index, so you should not have KrakenUniq vs Bowtie2 incompatibility problems</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:43:16">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 19:43:16"><div class="time">2022-12-02 19:43:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This all however does not seem to explain why aMeta insists on rebuilding the Bowtie2 index, let me think a bit</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:44:42">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 19:44:42"><div class="time">2022-12-02 19:44:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Of course the easiest would be to just let aMeta to rebuild the Bowtie2 index but we need to figures out why aMeta wants it üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôÇ Carly Scott
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:49:27">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-02 19:49:27"><div class="time">2022-12-02 19:49:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'll set up a run this weekend to let aMeta rebuild the index, and let you know what happens. I can then check for discrepancies between the directory/files where I built it separately and aMeta's directory</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-02 19:52:03">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-02 19:52:03"><div class="time">2022-12-02 19:52:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good, thanks! I believe, if you built a KrakenUniq DB using a number of reference fasta-files in a directory, and then just concatenated those reference fasta-files into a single grand fasta-file and built a Bowtie2 index on it, those two DBs are compatible and you should not get troubles due to different seqIDs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-05 18:58:16">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-05 18:58:16"><div class="time">2022-12-05 18:58:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ran just fine over the weekend letting aMeta rebuild the database. As far as I can tell the files are identical to what was there before, but the software no longer needs to rebuild the bowtie2 index at each start up. I'm just going to call this resolved on my end rather than looking into it any further for now.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-05 19:08:27">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-05 19:08:27"><div class="time">2022-12-05 19:08:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Carly Scott, I am glad that it worked fine eventually üôÇ but we (developers) still need to fix this issue. I shared what you told me with my colleagues and we seem to have another user experiencing the same issue, so we will perhaps restrict this rule severely so that it is not activated at all if a pre-built Bowtie2 index is provided. We are currently discussing possible solutions. Thanks a lot for using aMeta and reporting this issue. Please do not hesitate to post here any other issues that you might experience in the future</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-05 22:16:04">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-05 22:16:04"><div class="time">2022-12-05 22:16:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Super simple question - where should I find the final result from aMeta (a table containing microbes per sample sorted by presence + origin)? I'm trying to figure out if I have a software error or am just looking in the wrong place. The abundance tables given in "MALT<em>ABUNDANCE</em>MATRIX_**" are pre-authentication for damage, etc., yes?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-08 15:06:30">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-08 15:06:30"><div class="time">2022-12-08 15:06:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Carly Scott, sorry for the delay! We have recently added a heatmap overview of ancient microbes, please check the ‚ÄúMain results‚Äù section here <a href="https://github.com/NBISweden/aMeta">https://github.com/NBISweden/aMeta</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/NBISweden/aMeta">GitHub - NBISweden/aMeta: Ancient microbiome snakemake workflow</a></div>
                                <div class="link-text">
                                    Ancient microbiome snakemake workflow. Contribute to NBISweden/aMeta development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/NBISweden/aMeta">
            <img class="preview" src="https://opengraph.githubassets.com/d2a5b00911f418b6f0d0ad564aa56ed19534c23a0d0846c66a37dd47d1dd93c2/NBISweden/aMeta"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/NBISweden/aMeta</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-08 15:10:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-08 15:10:14"><div class="time">2022-12-08 15:10:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You have in general 3 abundance matrices delivered by aMeta: KrakenUniq abundance matrix, and 2 Malt abundance matrices (computed from rma6 and sam-alignments). All 3 of them do not tell you anything about authentication, therefore we made the heatmap overview (above) that demonstrates ‚Äúauthentication scores‚Äù (quantification of 7 quality metrics) on the top of detected microbes</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-08 15:16:35">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-08 15:16:35"><div class="time">2022-12-08 15:16:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So if you only want to see what microbes are present in your samples, I would take the KrakenUniq abundance matrix. However, if you want authentication information, you will either have to manually check authentication pdf-figures (per sample per microbe) in results/AUTHENTICATION, or run our script for aggregating authentication scores on a heatmap</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-08 18:18:50">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2022-12-08 18:18:50"><div class="time">2022-12-08 18:18:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov Perfect, thank you! If you're interested, in the meantime I wrote a little shell script that will take the highest ranking microbes for each sample and combine all the pdf's together into one master document you can visually check locally (for lots of samples it's tedious to download them off the server separately). Then, after manual inspection I subset the abundance matrices for my "verified" sample set. Running aMeta with a custom database, I decided it would take too long to inspect all of the pdfs returned (as I've expanded beyond bacteria to other microbes which may be in my samples).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-08 20:17:34">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-12-08 20:17:34"><div class="time">2022-12-08 20:17:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Carly Scott glad to hear you reached the pdf-output step, meaning aMeta seems to have worked correctly üôÇ Indeed, collecting, downloading and inspecting all the numerous pdfs is tedious, therefore we tried to automate it, but have only recently implemented the summarizing rule. I believe when you started running your samples, the summary heatmap was not yet available, so you had to aggregate the pds manually. Hopefully, in the future runs the aggregation will be much easier üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôÇ Carly Scott
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 12:38:18">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-11-01 12:38:18"><div class="time">2022-11-01 12:38:18</div></a>
                <div class="msg">
                    <p>@channel based on @Yuti Gao‚Äôs question above, how useful/interesting do you think an (anonymous) survey of extraction-sequencing costs across the whole field would be ?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëè Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 16:37:53">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-11-01 16:37:53"><div class="time">2022-11-01 16:37:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Blogggy blog blog?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-01 16:41:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-11-01 16:41:49"><div class="time">2022-11-01 16:41:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Blooooog.</p>

<p>But shotgun not doing it (free idea for someone!)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÇ Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 05:25:39">
            
                <img src="https://avatars.slack-edge.com/2022-11-02/4300959989479_befbae4e53ea2f9c1f48_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Laura Woods
                     <span class="print-only user-email">(laura.cyn.woods@gmail.com)</span>
                </div>
                <a href="#2022-11-03 05:25:39"><div class="time">2022-11-03 05:25:39</div></a>
                <div class="msg">
                    <p>Hi all! I'm a newcomer here and, in the spirit of the channel, I'm writing to ask some clumsy, far too general questions that may betray even more ignorance than I'm already anticipating. I've reached out to a few researchers in the ancient metagenomics community and was encouraged to ask a broader audience their thoughts on a few of them. Recently, I've finished up a postdoc where I mostly performed bioinformatic analyses on metagenomic datasets from modern environmental samples, with a little HGT twist given that HGT took centre stage during my PhD in microbial evolution. I liked my work and I got the chance to collaborate on some very diverse topics, but I've always been fascinated by history and archaeology, and believe that ancient DNA work may be a good fit for me. I've been reading up on general methods, challenges in the field, and of course, exciting findings in primary research articles, but I'm worried about approaching potential PIs and being dismissed as an unserious candidate. Has anyone had any experience approaching PIs as someone from outside the field? Relatedly, how did you find your research topics? Did you come in with some very concrete ideas or did you select from projects that already had some momentum behind them? I'm very open to a range of topics but I'm concerned this may simply read as naivety about the field and work against me. Finally, does anyone have any experience with remote (or semi-remote, with periodic travelling) work and the general perception of it among PIs? Though I did wet-lab work for my PhD, my most important skills really are in bioinformatics, and anyway, the idea of being responsible for DNA extraction of some precious limited sample terrifies me (for now--never say never!). And I think we've all demonstrated an ability to work effectively from home in the last few years. The alternative is waiting a couple of years to start, since my partner is doing his postdoc currently, but I'd much rather start sooner rather than later. Any thoughts on these or anything else along this line are very welcome!</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele, Biancamaria Bonucci, Nikolay Oskolkov, Shreya, Sterling Wright
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 08:20:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-11-03 08:20:57"><div class="time">2022-11-03 08:20:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I can't say much about approaching people directly as I've not had to od thi sfo far, but some general thoughts from me:</p>

<ol><li><p>Outside the field is rarely an issue, it's a very small field - very few people 'grow up' in the discipline. Furthermore, often people with modern biology backgrounds are welcome, as many students come from a social sciences background</p></li><li><p>If you have your own ideas/projects, you need to bring your own funding for them, otherwise you will have to apply for existing projects. It's rare that our discipline have money 'hanging around' for 'cool ideas' (so to say)</p></li><li><p>Unless you go to larger labs (which there are relatively few often), often you will have to do everything from A-Z at PhD level anyway. I'm not sure how this is at postdoc; that said the lrager labs are more and more moving into purely analysis (as they have a lab core). I think remote working will always be a lab culture thing though rather than specific to a field</p></li>
</ol>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Laura Woods
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 09:32:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-03 09:32:21"><div class="time">2022-11-03 09:32:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Laura Woods, while James gave you very good field-specific comments, I can briefly share my general impression of migrating across fields as I did my PhD in theoretical physics and switched to biomedicine which was quite shocking experience üôÇ  I have been viewed everyday by many as an unserious candidate üôÇ which is very frustrating, but with time I learnt to overcome this physiological barrier, your different from others background can be your strength. My experience is that the vast majority of PIs prefer talking to somebody who understands their language (terminology), but from time to time I see other PIs who are open for exploring new methods in more risky projects. In your particular case, I would not be worried much because bioinformatic analysis of modern environmental metagenomic samples is super close to what many people in this channel are doing, and you are not an outsider (in contrast to me üôÇ ). Regarding finding your research topics, during my carrier I mostly had to adjust to already established projects, so even if I have my own ideas, they are not straightforward to implement or to be supported (as James very well pointed out it has something to do with own funding üôÇ ). However, my major transition from physics to biomedicine happened when I brought my naive ideas to a group of medical doctors who for some reason found them interesting, but this was a luck and an exception. Generally, I would say, to implement own ideas one needs own funding and a secured position. Otherwise one needs to adjust and find a niche. Most of the time I am learning what other people are doing before suggesting something original üôÇ In summary, I would really encourage you to try. Motivation and interest, in my eyes, are more important than knowledge in a particular area, everything can be learnt providing you have interest and passion</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Biancamaria Bonucci, Maria Lopopolo
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Laura Woods
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 14:53:49">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-11-03 14:53:49"><div class="time">2022-11-03 14:53:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Laura Woods, I would very much agree with what has been said so far. From the perspective of a group leader in a very small group without many resources, I'd say:
‚Ä¢ having people with experience in environmental metagenomics is amazing!  The ancient metagenomics field is very much a conglomerate of people coming from the aDNA branch, the microbiology branch and the bioinfo branch. Then we learn the respective other parts to make sense of our data
‚Ä¢ Approaching people will never hurt, as long as you are fine with getting (hopefully) polite replies stating that in most cases there is no money to employ a person ad hoc. 
‚Ä¢ This is where bringing in own money would be important. If you identify a research group that does what you like, do reach out to them with the prospect to develop a research topic for which you can write up a scholarship application (if they don't have money available). There is a whole set of postdoc grants that would be suitable, from Marie Curie, to EMBO, HFSP, to country-specific ones. Having your own funding, even on a co-developed project, will give you a lot of independence. 
‚Ä¢ In my lab, we tend to do everything from sampling to analyses, but I agree that the bioinformatic analytical expertise is usually the bottleneck.
‚Ä¢ Working remotely will very much depend on a situation, the source of funding and the lab culture. It is possible but I would think that most labs would prefer having the group members physically present. It's not only about doing the work but about having a sense of belonging together, the possibility to chat with each other about science and life. You will never have unexpected spontaneous meetings online... </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Maria Lopopolo
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Laura Woods
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 23:16:00">
            
                <img src="https://avatars.slack-edge.com/2022-11-02/4300959989479_befbae4e53ea2f9c1f48_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Laura Woods
                     <span class="print-only user-email">(laura.cyn.woods@gmail.com)</span>
                </div>
                <a href="#2022-11-03 23:16:00"><div class="time">2022-11-03 23:16:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks a lot for such thoughtful responses, they have been really encouraging! Actually, part of the reason that I even worked up the nerve to write this message was because I suspected that the field was generally more welcoming to outsiders, and it seems that even those with a much greater change in academic focus than mine can find success! As for finding a research project, the idea of coming up with an idea while being so inexperienced is a bit daunting anyway and I certainly haven't developed the kind nose for what projects will have broad interest or get funded properly that one develops after being in a field for long enough. Therefore,  co-developing a project and applying for funding or applying for positions advertising funding are very acceptable options. Regarding remote work, it is always a big ask and to be honest, I do prefer coming into an office--I began and ended my last position in lockdown and had a constant sensation of being a bit adrift. I completely understand that group leaders would share that view. I suppose I've just come up against the old two-body problem: my partner is already working on his postdoc and we'd been a part while I did my last one, so I was hoping we could live together this time round. Ah well, no harm in asking and I can always wait until after he's finished if need be. Thanks again to all of you, I really appreciate your taking the time to answer my questions!</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 10:24:10">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2022-11-04 10:24:10"><div class="time">2022-11-04 10:24:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Totally feel you @Laura Woods, particularly about the 2-b problem. My husband and I fought on this front for years... Happy to chat more about this and any other aspects of changing fields. My moves have not been as drastic as @Nikolay Oskolkov‚Äôs but I have been financing my biggest transitions through the named fellowships, which took the edge off for the group leaders in terms of taking on someone inexperienced in their field.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 19:24:33">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2022-11-03 19:24:33"><div class="time">2022-11-03 19:24:33</div></a>
                <div class="msg">
                    <p>Has anyone ever seen bwa samse act weirdly and randomly inserting duplicate lines (whether reads or headers) in the sam file it makes? I'm working with bwa v. 0.7.17-r1188 either base install on my server or on a conda install, and it is being VERY weird with sam file production. samse/sampe makes these corrupt sam files with repeats (either duplicate headers, or will insert read information and then more headers underneath, then more read data), and bwa mem makes "correct" sam files but all reads are unaligned (no matter the sample, for fastas with 2-15 million reads and reference genomes of 3GB). I've tried googling the errors and so far, the best advice I've seen is with the bwa aln+samse route, try manually removing the duplicate lines if there are few of them (which I don't think is my case), or attempting to use bam utils (this fails with the bam conversion too). I'm very confused on how to proceed. Has anyone ever seen this? Maybe with this version of bwa? Is this a bwa dependency malfunctionning? My preference would be to "fix" the software so no weird things happen rather than dealing with the weird sam files on a case by case basis, but I'm really lost right now...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-03 19:30:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-03 19:30:18"><div class="time">2022-11-03 19:30:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maria, could you post here a few duplicate lines? Also could you extract the corresponding reads from the fastq-file and post for comparison?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 14:36:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-04 14:36:47"><div class="time">2022-11-04 14:36:47</div></a>
                <div class="msg">
                    <p>Hey dear community,
What tools are you using to make your multiple genomes circular comparison maps ?
Anvi‚Äôo looks really nice (<a href="https://merenlab.org/2016/11/08/pangenomics-v2/">https://merenlab.org/2016/11/08/pangenomics-v2/</a>), plus I already have some of their stickers, but I was wondering what‚Äôs your tools of choice</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Meren Lab</div>
                            
                            
                                
                                <div class="link-title"><a href="https://merenlab.org/2016/11/08/pangenomics-v2/">An anvi&#39;o workflow for microbial pangenomics</a></div>
                                <div class="link-text">
                                    The user-friendly interface anvi&#39;o provides to work with pangenomes.
                                </div>
                                
                                
    
        <a href="https://merenlab.org/2016/11/08/pangenomics-v2/">
            <img class="preview" src="https://merenlab.org/images/uchicago-mbl.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://merenlab.org/2016/11/08/pangenomics-v2/</div>
                                
                                
                            
                        </div>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F049BFVQJ14/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F049BFVQJ14/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F049BFVQJ14-682ed4e634/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="335"
                 height="360" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 14:37:07">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-04 14:37:07"><div class="time">2022-11-04 14:37:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Plot is from <a href="https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.13015">https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.13015</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 14:52:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-11-04 14:52:42"><div class="time">2022-11-04 14:52:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* circos</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 15:16:00">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-04 15:16:00"><div class="time">2022-11-04 15:16:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Vanilla circos ? From scratch ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 15:16:40">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-11-04 15:16:40"><div class="time">2022-11-04 15:16:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yup, but mostly just for final results</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 14:56:43">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-07 14:56:43"><div class="time">2022-11-07 14:56:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Adding further softwares for reference:
‚Ä¢ BRIG: <a href="https://brig.sourceforge.net/">https://brig.sourceforge.net/</a>
‚Ä¢ pyCircos: <a href="https://github.com/ponnhide/pyCircos">https://github.com/ponnhide/pyCircos</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/ponnhide/pyCircos">GitHub - ponnhide/pyCircos: python Circos</a></div>
                                <div class="link-text">
                                    python Circos. Contribute to ponnhide/pyCircos development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/ponnhide/pyCircos">
            <img class="preview" src="https://opengraph.githubassets.com/dc9379f59f2977917061dce71c5605fbcf6341a9ae4d832fd24ee708e9b1b318/ponnhide/pyCircos"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/ponnhide/pyCircos</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 15:58:39">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-11-04 15:58:39"><div class="time">2022-11-04 15:58:39</div></a>
                <div class="msg">
                    <p>Does anyone have any information for designing capture probes? I have only captured the human genome and bought a commercial kit. But I guess this is not going to be an option for other species... I am thinking about pathogens (and other animal sp). Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 16:09:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2022-11-04 16:09:35"><div class="time">2022-11-04 16:09:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Depends what pathogen and what animal sp. They all have pretty different genomes etc
Any idea what you want to do yet?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-04 16:34:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-11-04 16:34:28"><div class="time">2022-11-04 16:34:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Megan Michel @aidanva @irinavelsko?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-06 12:29:49">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-11-06 12:29:49"><div class="time">2022-11-06 12:29:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It‚Äôs all very theoretical right now! Just wondered what the process looked like, if there were specific companies to approach and what sort of information they would need?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 09:00:49">
            
                <img src="https://avatars.slack-edge.com/2023-08-11/5727013151811_5866d2031f65a72cd9a6_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ian Light
                     <span class="print-only user-email">(light@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2022-11-07 09:00:49"><div class="time">2022-11-07 09:00:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have some code/pipeline (not yet published and definitely not in a "download and run"-state but we can still share it) that takes in a set of (already downloaded) genomes, and will output the unique probes for this set of genomes with a given step size. These can then be post processed to remove highly similar probes (depending on your definitions for this)</p>

<p>It is not the quickest method imaginable but it is robust and usable in my opinion (depending on how many genomes you start with and how big your genomes are, eg if you try to get probes for microbes it is OK, if for eukaryotes it might be too slow).</p>

<p>let me know if you want to peek at the methods section I have written up!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç s.wasef
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-07 13:53:36">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2022-11-07 13:53:36"><div class="time">2022-11-07 13:53:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh wow Ian!! That sounds great, I would love to have a look if you don't mind! Thank you üòÑ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 10:56:39">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2022-11-14 10:56:39"><div class="time">2022-11-14 10:56:39</div></a>
                <div class="msg">
                    <p>Hello all,
I was wondering what tools people would recommend to explore recombination in viruses?
So far, I have used SplitTree4 on full genomes but the output does not appear to give me details of the possible breakpoints. I also tried to run the same full genome alignments through GARD on datamonkey, but it doesn't load. I was able to make it run going ORF by ORF and it does output possible breakpoints, but I feel I may be making it harder on myself if I go ORF by ORF. Looking in the literature, I found another tool called RDP 5, but it only runs on a PC so as I use Mac, I was wondering if there were any other tools people would recommend (and that are preferably user-friendly)?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 14:16:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-11-14 14:16:14"><div class="time">2022-11-14 14:16:14</div></a>
                <div class="msg">
                    <p>Hello everyone,
Imagine that I have to do a phylogenetic tree within two weeks for 3 pathogenic species. Do you have any suggestion on a realistic way to do it in a short time? Also keep in mind that I have never done a phylogenetic tree before üòÖ. Thanks in advance üôè</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 15:00:46">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2022-11-14 15:00:46"><div class="time">2022-11-14 15:00:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello Zo√©,
If it is a phylogenetic tree with bootstrap values that you're looking for, I would personally recommend IQTree: <a href="http://www.iqtree.org/">http://www.iqtree.org/</a>
You can set IQtree to look for the best-fit model through the option -m TEST and you can also run the tree using bootstrap approximations (UFBoot and SH-aLRT) which will speed up your run - with the options -bb and -alrt. Of course, if you don't want approximations, then you can use the option -b but your analyses will take longer to run. I always start with the approximations to get a good idea before rerunning the analysis with the other option.
The manual is very user-friendly too üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 15:36:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-14 15:36:09"><div class="time">2022-11-14 15:36:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What‚Äôs your starting point ? ie. what kind of data/results do you have right now ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 18:21:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-11-14 18:21:06"><div class="time">2022-11-14 18:21:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you @Oph√©lie Lebrasseur I‚Äôll check that üòä
For @Maxime Borry I‚Äôm not sure exactly sure what kind of information you need so don‚Äôt hesitate to ask for more:
So I have ‚ÄúBacterium 1‚Äù that is found in 4 individuals:
‚Ä¢ KrakenUniq: 160 000 reads, 2 700 000 kmers, after MALT alignment: 93% breadth of coverage
‚Ä¢ KrakenUniq: 11 000 reads, 330 000 kmers, after MALT alignment: 50% breadth of coverage
‚Ä¢ KrakenUniq: 8 500 reads, 140 000 kmers, after MALT alignment: 9036 reads and 19% breadth of coverage
‚Ä¢ KrakenUniq: 5 200, 32 000 kmers after MALT alignment: 6088 reads and 15% breadth of coverage
‚ÄúVirus 1" that is found in 3 individuals:
‚Ä¢ KrakenUniq: 7 100 reads, 31 000 kmers, visualisation with IGV shows that the whole genome is covered
‚Ä¢ KrakenUniq: 630 reads, 4 070 kmers
‚Ä¢ KrakenUniq: 580 reads, 2 500 kmers
A second bacterium is found in 2 individuals:
‚Ä¢ KrakenUniq: 190 000 reads, 81 000 kmers, after MALT alignment: 72% breadth of coverage
‚Ä¢ KrakenUniq: 2 800 reads, 8 500 kmers, after MALT alignment: 41% breadth of coverage
Note: I am rerunning MALT now because I was using the version that had a bug before. But if I understood well, the bug was mostly underestimating the amount of reads assigned to a species so rerunning it should actually improve the result</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 18:25:22">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-14 18:25:22"><div class="time">2022-11-14 18:25:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you already have reconstructed genomes (1), or just reads mapped to a reference genome (2) ?
‚Ä¢ For (1) <a href="https://github.com/maxibor/corephylo">https://github.com/maxibor/corephylo</a>
‚Ä¢ For (2) nf-core/eager to generate VCF + <a href="https://github.com/maxibor/vcfphylo">https://github.com/maxibor/vcfphylo</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/maxibor/corephylo">GitHub - maxibor/corephylo: core genome phylogenies</a></div>
                                <div class="link-text">
                                    core genome phylogenies. Contribute to maxibor/corephylo development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/maxibor/corephylo">
            <img class="preview" src="https://opengraph.githubassets.com/c829ee00ccf89c8827b9d5d75202080d17f0bfdf4b8c4a160eb145be59791fb1/maxibor/corephylo"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/maxibor/corephylo</div>
                                
                                
                            
                        </div>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/maxibor/vcfphylo">GitHub - maxibor/vcfphylo</a></div>
                                <div class="link-text">
                                    Contribute to maxibor/vcfphylo development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/maxibor/vcfphylo">
            <img class="preview" src="https://opengraph.githubassets.com/aea2027fc6556ef9cb0103d7d8e9fdc890890a919b07d6418f69b080ca46054b/maxibor/vcfphylo"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/maxibor/vcfphylo</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 18:59:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-11-14 18:59:20"><div class="time">2022-11-14 18:59:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks! Is one better than the other? I have the second option, reads mapped to a reference genome</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 20:05:27">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-11-14 20:05:27"><div class="time">2022-11-14 20:05:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have done a quick and dirty tree using Snippy to call SNPs and MEGA to build the tree and it works pretty well!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon, Maxime Borry
                        </div>
                    
                        <div class="message-reaction">
                        üòä Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        üëç:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 10:05:05">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2022-11-15 10:05:05"><div class="time">2022-11-15 10:05:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* They're just different. You would do (1) if you have de novo assembled genomes, (2) otherwise</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëå Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-14 23:35:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-11-14 23:35:09"><div class="time">2022-11-14 23:35:09</div></a>
                <div class="msg">
                    <p>Extreeemely dumb question for @Nikolay Oskolkov @Zo√© Pochon or anyone on the team‚Ä¶ is there a command line download link for the KrakenUniq databases on figshare? Trying <code>wget &lt;https://figshare.scilifelab.se/ndownloader/articles/20518251/versions/1&gt;</code> with the link from the ‚Äúdownload all‚Äù button but it doesn‚Äôt get very far before throwing me a ‚ÄúStale file handle‚Äù error, wondering if that‚Äôs at my end or if there‚Äôs a better way to do this!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 07:20:43">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-15 07:20:43"><div class="time">2022-11-15 07:20:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Shreya, you are doing it right, and the ‚ÄúStale file handle‚Äù error is probably an internet connection break, so it is important to be able to resume downloading. A way to download it is to right click on the download button in figshare, copy the link, then use <a href="https://aria2.github.io/">aria2c</a> to download as you would do with wget. It works the same way, and you could use wget as well, but aria2c can resume downloads if they get interrupted. Nevertheless SciLifeLab Figshare recommends to use their API here <a href="https://docs.figshare.com/#file_download">https://docs.figshare.com/#file_download</a>, but I have not tried it so can't advise you, perhaps you could test this way as well?</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">aria2.github.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://aria2.github.io/">aria2</a></div>
                                <div class="link-text">
                                    aria2 is a lightweight multi-protocol &amp;amp; multi-source command-line
download utility. It supports HTTP/HTTPS, FTP, SFTP,
BitTorrent and Metalink. ‚Ä¶
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://aria2.github.io/</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 10:30:27">
            
                <img src="https://secure.gravatar.com/avatar/d7f9ede974c0dacc69882b20d469dbb1.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Piotr Rozwalak
                     <span class="print-only user-email">(pioroz4@amu.edu.pl)</span>
                </div>
                <a href="#2022-11-15 10:30:27"><div class="time">2022-11-15 10:30:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I was also struggling with downloads. Finally, I used aria2 to download separate files. e.g.
<code>aria2c -x 4 "&lt;https://figshare.scilifelab.se/ndownloader/files/37576102&gt;"</code></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 17:39:17">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2022-11-15 17:39:17"><div class="time">2022-11-15 17:39:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you both!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 09:46:30">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2022-11-15 09:46:30"><div class="time">2022-11-15 09:46:30</div></a>
                <div class="msg">
                    <p>Hey everyone, I also have one question which should hopefully be easy to answer. I'm running krakenuniq using the full nt db and I've noticed that it's pretty slow (5h/file even though the files are &lt;1Gb), so I have added the "--preload" flag but I'm not sure it works properly within the for loop. Any ideas why it may run so slow?
<code>for i in $INPUT/merged**; do krakenuniq --db $DB --threads 32 --preload $i --output $i.kraken --report-file $i.report; done</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 09:53:05">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-15 09:53:05"><div class="time">2022-11-15 09:53:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Lennart Schreiber how much available RAM do you have? Second, the speed of KrakenUniq is not really affected by the input fastq-file but the database size. Third, please check the "--preload-size" option which should be much faster. Finally, the sequential for-loop solution did not really work for me either, so I always try to submit different samples to different cluster nodes</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 09:58:07">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2022-11-15 09:58:07"><div class="time">2022-11-15 09:58:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you very much @Nikolay Oskolkov! I allocated a total of 160Gb RAM when I last started it but maybe I should ask for more? I'll try the "--preload-size" option and thanks for the tip about submitting samples to different cluster nodes, I haven't considered that.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 10:01:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2022-11-15 10:01:01"><div class="time">2022-11-15 10:01:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you have 160 GB of RAM and want to fit a full NT krakenuniq DB, I would try "--preload-size 128GB". If it fails (sometimes it does because --preload-size is not very stable yet to my experience), then please try "--preload-size 96GB"</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Lennart Schreiber
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-11-15 10:35:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2022-11-15 10:35:31"><div class="time">2022-11-15 10:35:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes I would also suggest not running it in a for loop but each sample separately</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-16 21:27:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-12-16 21:27:54"><div class="time">2022-12-16 21:27:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* krakenuniq works pretty well for me with the
<code>--preload option</code></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-16 21:29:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-12-16 21:29:28"><div class="time">2022-12-16 21:29:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* what I do is to first preload the DB and then run the for loop for each dataset and it runs very fast per sample. The limiting step is the loading of the DB. I haven‚Äôt tried yet to also use the --preload-size option for the full NCBI DB</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-16 21:33:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-12-16 21:33:06"><div class="time">2022-12-16 21:33:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Here is the code for an sbatch script:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-16 21:33:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2022-12-16 21:33:09"><div class="time">2022-12-16 21:33:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```#!/usr/bin/env bash
#</p>

<h1>SBATCH -o krakenuniq<em>nt.out -e krakenuniq</em>nt.err</h1>

<h1>SBATCH --qos=normal</h1>

<h1>SBATCH --cpus-per-task=30</h1>

<h1>SBATCH --mem=400G</h1>

<h1>SBATCH --qos=normal</h1>

<h1>SBATCH --job-name KrakenUniq</h1>

<p>krakenuniq \
--db DB-DIR-PATH/ \
--preload --threads 30; \
for file in <code>cat samples.ids</code>; do \
krakenuniq \
--db DB-DIR-PATH/ \
--threads 30 \
--report-file ${file}.tax.report.tsv.gz \
--output ${file}.read.report.tsv.gz \
--gzip-compressed \
--paired \
--fastq-input ${file}.trim.R1.fastq.gz ${file}.trim.R2.fastq.gz ; \
done```</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Lennart Schreiber
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-07 12:46:43">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2022-12-07 12:46:43"><div class="time">2022-12-07 12:46:43</div></a>
                <div class="msg">
                    <p>Another not-a-stupid question - for pair-end data, is the universal approach to only use collapsed reads?
we're getting some curious / suspicious alignments when looking non-collapsed reads (this was out of curiosity more than anything else)
thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-07 12:49:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-12-07 12:49:07"><div class="time">2022-12-07 12:49:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Depends on the context. Can be useful for boosting coverage when you've got low endogenous, but you can assume collapsed only reads are more likely to be short and therefore more likely to be really ancient. So in the second context it it maybe a better approach for pathogen screening</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-07 12:51:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2022-12-07 12:51:04"><div class="time">2022-12-07 12:51:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So what you're seeing makes sense if you are doing pathogen screening I suspect</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2022-12-07 13:03:28">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2022-12-07 13:03:28"><div class="time">2022-12-07 13:03:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* exactly, my instinct would be conservative RE the choice of shorter/collapsed reads for ancient pathogens</p>

<p>can see there being contexts of broader inclusion of reads. in this instance the read pairs were often not properly paired (‚ö†Ô∏è) but were individually blasting to the genome of interest (spooky). rather stick with the collapsed reads that are more trustworthy ex ante</p>

<p>thanks for sounding that out!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-08 18:13:28">
            
                <img src="https://avatars.slack-edge.com/2023-01-08/4596140433751_b9717ebfac5bfddaaa23_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Francisco Zorrilla
                     <span class="print-only user-email">(fz274@cam.ac.uk)</span>
                </div>
                <a href="#2023-01-08 18:13:28"><div class="time">2023-01-08 18:13:28</div></a>
                <div class="msg">
                    <p>hi all, I have assembled MAGs from ancient metagenomes and would now like to determine wether they are indeed ancient or modern contamination. Here is the code I used to run pydamage (<a href="https://github.com/franciscozorrilla/metaGEM/issues/119#issuecomment-1374595930">https://github.com/franciscozorrilla/metaGEM/issues/119#issuecomment-1374595930</a>). In short, I concatenate all MAGs assembled from the same sample, then map qfiltered reads to generate a sorted bam file to provide to pydamage.</p>

<p>1) I was wondering if anyone could elaborate on the usage of the <code>-g, --group</code>  parameter for the pydamage tool? I have looked through the documentation and paper, but having a hard time understanding what it does or if it is appropriate for me to use here.</p>

<p>2) If I understand correctly, can I then use the output files to determine ancient provenance of each contig by filtering for <code>predicted_accuracy</code> &gt;= 0.67 &amp; <code>qvalue</code> &lt; 0.05?</p>

<p>any comments, suggestions, and/or advice welcomed!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-08 18:19:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-08 18:19:13"><div class="time">2023-01-08 18:19:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maxime Borry would be the best to answer these</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Francisco Zorrilla
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-08 18:37:45">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-01-08 18:37:45"><div class="time">2023-01-08 18:37:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Francisco Zorrilla ,
1) the group flag is when you want to analyze at the genome/MAG level, ie: treating all the contigs as if they were a single long contig, otherwise PyDamage analyzed contigs individually. For your use case, I don‚Äôt think you‚Äôd want to use it.
2) The 0.67 threshold is the one we used for the dataset of the paper. To determine the threshold for you data, you can let PyDamage set it for you with the kneed method (see doc here: <a href="https://pydamage.readthedocs.io/en/0.7/CLI.html#pydamage-filter">https://pydamage.readthedocs.io/en/0.7/CLI.html#pydamage-filter</a>) or decide that a it‚Äôs anyway a binary classification program and use a threshold of 0.5 (default) üòâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üôå Francisco Zorrilla
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-08 18:43:01">
            
                <img src="https://avatars.slack-edge.com/2023-01-08/4596140433751_b9717ebfac5bfddaaa23_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Francisco Zorrilla
                     <span class="print-only user-email">(fz274@cam.ac.uk)</span>
                </div>
                <a href="#2023-01-08 18:43:01"><div class="time">2023-01-08 18:43:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks for the quick response and clarification!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 13:23:11">
            
                <img src="https://avatars.slack-edge.com/2023-01-08/4596140433751_b9717ebfac5bfddaaa23_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Francisco Zorrilla
                     <span class="print-only user-email">(fz274@cam.ac.uk)</span>
                </div>
                <a href="#2023-01-12 13:23:11"><div class="time">2023-01-12 13:23:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* hey @Maxime Borry I was wondering if its possible to use the output of pydamage to try and roughly estimate the age of a MAG/contig?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 13:28:16">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-01-12 13:28:16"><div class="time">2023-01-12 13:28:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Francisco Zorrilla aDNA damage is not only explained by the age of the sequences, but also by a lot of other factors. So it‚Äôs unfortunately not possible. AFAIK, the only way to estimate the age of DNA from the sequences alone is the use of time trees (Through ML <a href="http://www.iqtree.org/doc/Dating">http://www.iqtree.org/doc/Dating</a> or Bayesian inference <a href="https://github.com/Taming-the-BEAST/Basic-tip-dating">https://github.com/Taming-the-BEAST/Basic-tip-dating</a>)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/Taming-the-BEAST/Basic-tip-dating">GitHub - Taming-the-BEAST/Basic-tip-dating: Basics of tip-dating in BEAST2</a></div>
                                <div class="link-text">
                                    Basics of tip-dating in BEAST2. Contribute to Taming-the-BEAST/Basic-tip-dating development by creating an account on GitHub.
                                </div>
                                
                                
    
        <a href="https://github.com/Taming-the-BEAST/Basic-tip-dating">
            <img class="preview" src="https://opengraph.githubassets.com/857a0141e9eb68c961b5003eeab510c52f0f737ed9892c285d5acf04ec2c8d07/Taming-the-BEAST/Basic-tip-dating"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/Taming-the-BEAST/Basic-tip-dating</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Francisco Zorrilla
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 14:59:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-12 14:59:41"><div class="time">2023-01-12 14:59:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I +1 Maxime's statement. aDNA damage accumulation can be accelerated in warm and wet environments, and slowed down in frozen and dry environments, so it's not a uniform process.</p>

<p>Like Maxime says, the closest you can get is via mutation-rate clock estimates but that requires rather sophisticated analyses in most cases</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 15:01:36">
            
                <img src="https://avatars.slack-edge.com/2023-01-08/4596140433751_b9717ebfac5bfddaaa23_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Francisco Zorrilla
                     <span class="print-only user-email">(fz274@cam.ac.uk)</span>
                </div>
                <a href="#2023-01-12 15:01:36"><div class="time">2023-01-12 15:01:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks for pointing me in the right direction! will read more on those methods üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 15:02:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-12 15:02:11"><div class="time">2023-01-12 15:02:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Orlando, L., Allaby, R., Skoglund, P., Sarkissian, C. D., Stockhammer, P. W., √Åvila-Arcos, M. C., Fu, Q., Krause, J., Willerslev, E., Stone, A. C., &amp; Warinner, C. (2021). Ancient DNA analysis. Nature Reviews Methods Primers, 1(1), 1‚Äì26. <a href="https://doi.org/10.1038/s43586-020-00011-0">https://doi.org/10.1038/s43586-020-00011-0</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Francisco Zorrilla
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 15:02:16">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-12 15:02:16"><div class="time">2023-01-12 15:02:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That might be a good general summary of aDNA</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-12 15:02:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-12 15:02:52"><div class="time">2023-01-12 15:02:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (it's quite broad though)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 11:12:44">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-09 11:12:44"><div class="time">2023-01-09 11:12:44</div></a>
                <div class="msg">
                    <p>Hi there! I have a "has anyone else..." or "are we crazy...??" type of question for you all today.</p>

<p>This is something that we keep running into during our decontamination/filtering pipeline of kraken2 output. In many of our samples (calculus of various mammals, humans), among the most abundant classified taxa in our raw kraken2 output is identified as "human". This is particularly weird because this happens even after we remove all reads mapping to the human genome (and host genome for non-human mammals) during preprocessing. Looking more closely at kraken output, we find that the overwhelming proportion of these "human" reads have a split classification, consisting of: 1) kmers that do indeed hit to the human genome, and 2) kmers that do not have a match in the kraken database (persists with kraken runs on larger databases, i.e. full-nt). Typical kraken output for these reads looks like this:</p>

<p>```C M<em>A00605:444:HMGCKDSX3:3:1102:4119:30671 9606 109 9606:12 0:63
C M</em>A00605:444:HMGCKDSX3:3:1103:2401:4554 9606 73 0:3 9606:6 0
C M<em>A00605:444:HMGCKDSX3:3:1104:12138:27712 9606 84 0:38 9606:7 0:5
C M</em>A00605:444:HMGCKDSX3:3:1104:12943:28886 9606 84 0:38 9606:7 0:5
C M<em>A00605:444:HMGCKDSX3:3:1105:9064:8046 9606 89 0:5 9606:3 0
C M</em>A00605:444:HMGCKDSX3:3:1105:9082:8077 9606 89 0:5 9606:3 0
C M<em>A00605:444:HMGCKDSX3:3:1106:19262:1470 9606 65 0:10 9606:3 0:18
C M</em>A00605:444:HMGCKDSX3:3:1107:28583:10848 9606 83 0:44 9606:5
C M<em>A00605:444:HMGCKDSX3:3:1107:20491:32941 9606 119 0:31 9606:1 0:5 9606:1 0:47
C M</em>A00605:444:HMGCKDSX3:3:1107:16260:28620 9606 119 0:31 9606:1 0:5 9606:1 0:47
``````
So an overall classification classification of "unknown" might be more accurate than "human". I've also done a rudimentary BLAST of some reads with these split classifications and some do hit to records of junk/artificial/sequencing artefacts in NCBI.</p>

<p>I've compared this with reads used to classify other highly-abundant oral taxa. While some reads have kmers split with something not in the kraken database, it's nowhere near as prevalent as it is for the "human" reads.</p>

<p>So.... are we indeed crazy or has anyone else come across this before? Do these reads represent artefacts of the human genome we are using (GRCh37), or possibly artefacts involved with the preparation of samples for sequencing? Any suggestions are most welcome, because we usually just hard-code the removal of any "human" records in kraken data before before downstream analyses.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 11:25:48">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-09 11:25:48"><div class="time">2023-01-09 11:25:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Adrian Forsythe, what database are you using with kraken2, is that NCBI NT?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 11:39:24">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-09 11:39:24"><div class="time">2023-01-09 11:39:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Nikolay, I've run these samples through kraken2 using both the "standard" and nt NCBI databases</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 11:59:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2023-01-09 11:59:00"><div class="time">2023-01-09 11:59:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The presence of human-classified reads in kraken after removing the human-mapped reads is also something we normally observe, but I never invested time to figure out to what they correspond and why they are classified as human. I normally just remove them from the analyses as I consider them confounding‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 12:02:44">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-09 12:02:44"><div class="time">2023-01-09 12:02:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, good to hear that this has also been observed by others, thanks Nico!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 13:00:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-09 13:00:21"><div class="time">2023-01-09 13:00:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Adrian Forsythe I confirm that there is sometimes a mess with seemingly "human" reads. I believe, I observed both types of situations, i.e. 1) when kraken does not report as many human reads as I expect, and 2) when kraken reports a large fraction of "human" reads even after a "human reads removal" procedure. I do not have a clear explanation of what is going on but have a few thoughts so far.</p>

<ol><li>I do not really trust a "human reads removal" procedure which is typically based on mapping a raw fastq to a human reference and filtering away reads mapped with a good quality. Despite this is widely used, this is not a "competative mapping", i.e. the raw metagenomic reads are exposed to only one reference, I have a difficulty with this approach. Exposing metagenomic reads to one reference at a time can be prone to various biases. Other people might think differently @James Fellows Yates?</li><li>NCBI NT has a very poor quality human reference, only small pieces, therefore I would expect a severe underestimating of actual human contamination when using the default NCBI NT unless a good quality human reference has been added manually. In contrast, the "standard" kraken DB should have a good quality human reference.</li><li>Some microbial reference genomes are known to have human contamination <a href="https://pubmed.ncbi.nlm.nih.gov/31064768/">https://pubmed.ncbi.nlm.nih.gov/31064768/</a>, so it is a mess</li>
</ol>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed</div>
                            
                            
                                
                                <div class="link-title"><a href="https://pubmed.ncbi.nlm.nih.gov/31064768/">Human contamination in bacterial genomes has created thousands of spurious proteins - PubMed</a></div>
                                <div class="link-text">
                                    Contaminant sequences that appear in published genomes can cause numerous problems for downstream analyses, particularly for evolutionary studies and metagenomics projects. Our large-scale scan of complete and draft bacterial and archaeal genomes in the NCBI RefSeq database reveals that 2250 genomes ‚Ä¶
                                </div>
                                
                                
    
        <a href="https://pubmed.ncbi.nlm.nih.gov/31064768/">
            <img class="preview" src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://pubmed.ncbi.nlm.nih.gov/31064768/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-09 13:28:06">
            
                <img src="https://avatars.slack-edge.com/2020-09-10/1354433495635_f88cf1f8d3193eeb06d1_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Adrian Forsythe
                     <span class="print-only user-email">(adrian.forsythe@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-09 13:28:06"><div class="time">2023-01-09 13:28:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for the feedback @Nikolay Oskolkov ! If I can add a bit more context:</p>

<ol><li>I agree, and I should have clarified that, in our preprocessing, the removal of host/human reads is a competitive mapping approach, mapping against a combined reference of human and host genomes (in the case of our non-human samples).</li><li>Yes, this is good point. I would assume then that the amount of "human" assignments would differ between standard and NT databases</li><li>In very few cases, we do find some bacterial species that are represented in the kmer assignments that are split with "human" kmers. I believe that some of these bacteria are even specifically mentioned in the paper you attached</li>
</ol>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-11 15:58:05">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2023-01-11 15:58:05"><div class="time">2023-01-11 15:58:05</div></a>
                <div class="msg">
                    <p>Hi, all. I know that it is common to use Herculase II Fusion to index libraries and PfuTurbo Cx to amplify libraries. Is there a reason people don‚Äôt use PfuTurbo Cx for both? Is it because Herculase is cheaper? I have never had dimer issues with PfuTurbo Cx, so I don‚Äôt think it‚Äôs that.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-11 16:03:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-11 16:03:07"><div class="time">2023-01-11 16:03:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I believe it's the other way around: pfu turbo to index and Herculase to amplify (ref: <a href="https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2">https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2</a>)</p>

<p>IIRC, you must at least use pfu turbo (or similar) to ensure you 'incorproate' damage into your reads, when you do initial amplification, after which it doesn't matter which polymerase you use. So you can use a cheaper one like Herculase (or a higher fidelity one)</p>

<p>I normally ping @Christina Warinner as my polymerase expert ;)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">protocols.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2">Illumina double-stranded DNA dual indexing for ancient DNA</a></div>
                                <div class="link-text">
                                    This protocol converts partially completed double-stranded DNA libraries e.g. from: Non-UDG treated double-stranded ancient DNA library preparation for Illumina sequencing (dx....
                                </div>
                                
                                
    
        <a href="https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2">
            <img class="preview" src="https://s3.amazonaws.com/protocols-files/files/dtqk4tje.jpg"
                width="250"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üôå:skin_tone_2: Kelly Blevins
                        </div>
                    
                        <div class="message-reaction">
                        üëÄ Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-11 16:12:47">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2023-01-11 16:12:47"><div class="time">2023-01-11 16:12:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There‚Äôs a paragraph on polymerases under ‚ÄúDNA library amplification‚Äù here: <a href="https://www.nature.com/articles/s43586-020-00011-0">https://www.nature.com/articles/s43586-020-00011-0</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Nature</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.nature.com/articles/s43586-020-00011-0">Ancient DNA analysis</a></div>
                                <div class="link-text">
                                    Nature Reviews Methods Primers - This Primer outlines the best ways to find, handle and analyse ancient DNA from various sources. The authors summarize ethical considerations and the importance of...
                                </div>
                                
                                
    
        <a href="https://www.nature.com/articles/s43586-020-00011-0">
            <img class="preview" src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs43586-020-00011-0/MediaObjects/43586_2020_11_Fig1_HTML.png"
                width="210"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.nature.com/articles/s43586-020-00011-0</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-11 16:51:12">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2023-01-11 16:51:12"><div class="time">2023-01-11 16:51:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you both very much!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-11 21:27:50">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2023-01-11 21:27:50"><div class="time">2023-01-11 21:27:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* mmm so perhaps PfuTurbo Cx is used for indexing because it reads through uracil, but Herculase cannot. Then Herculase is used for subsequent amplifications because the uracils are all back to thymines after indexing and Herculase has the same error rate as PfuTurbo Cx, has higher yields, and is cheaper.</p>

<p><a href="https://www.agilent.com/cs/library/technicaloverviews/public/Technical%20Note_High-Fidelity%20PCR%20Enzymes_5994-0929EN_.pdf">https://www.agilent.com/cs/library/technicaloverviews/public/Technical%20Note_High-Fidelity%20PCR%20Enzymes_5994-0929EN_.pdf</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-11 21:34:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-11 21:34:19"><div class="time">2023-01-11 21:34:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Exactly!</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 13:23:45">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2023-01-13 13:23:45"><div class="time">2023-01-13 13:23:45</div></a>
                <div class="msg">
                    <p>Hi all. I have a question related to hybridization capture: what would be the disadvantages of pooling indexed libraries before capture? I am working with indexed libraries that only contain very small amounts of target DNA (hence the capture) so I'm not that worried about "saturating" the baits. As far as I'm concerned, the double-indexing of the libraries should also prevent any contamination/misassignments.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 14:05:40">
            
                <img src="https://secure.gravatar.com/avatar/4b8f0795e46428bce5fe9524f3434a4f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pooja Swali
                     <span class="print-only user-email">(swalipooja@gmail.com)</span>
                </div>
                <a href="#2023-01-13 14:05:40"><div class="time">2023-01-13 14:05:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I find pooling as little libraries as possible works best for me (also double-indexed) and the protocol we used suggested adjusting the volume according to which library had more endogenous DNA. This paper talks about the issues with index hopping and also suggest avoiding pooling when the endogenous varies too much. Other may be more useful as I've only been using the arbor RNA-seq baits</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F04K6CZ10TT/molecular_ecology_resources_-_2022_-_zavala_-_quantifying_and_reducing_cross___contamination_in_single____and_multiplex.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Molecular Ecology Resources - 2022 - Zavala - Quantifying and reducing cross‚Äêcontamination in single‚Äê and multiplex.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F04K6CZ10TT/molecular_ecology_resources_-_2022_-_zavala_-_quantifying_and_reducing_cross___contamination_in_single____and_multiplex.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F04K6CZ10TT-661c55eb7a/molecular_ecology_resources_-_2022_-_zavala_-_quantifying_and_reducing_cross___contamination_in_single____and_multiplex_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="909"
                 height="1195" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üôè Lennart Schreiber
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 15:10:18">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2023-01-13 15:10:18"><div class="time">2023-01-13 15:10:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you very much for sharing your experience Pooja and also for pointing me to this relevant paper!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 15:52:40">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-13 15:52:40"><div class="time">2023-01-13 15:52:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We have performed capture on pools of double-barcoded extracts (before indexing), which worked rather well (this was done with museum specimens)
<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.12699">https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.12699</a>
Our colleagues working with faecal samples make a strong point about determining the endogenous content first (if capturing the host) and pooling samples by endogenous content:
<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.13300">https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.13300</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Lennart Schreiber
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 15:56:10">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-01-13 15:56:10"><div class="time">2023-01-13 15:56:10</div></a>
                <div class="msg">
                    <p>Hi all! Yesterday at SPAMMTISCH @irinavelsko and I were talking about doing R analysis with Phyloseq and this package offers the opportunity to handle otu tables, taxonomy tables and phylogenetic trees at the same time. However, with Shotgun data we're flying without trees, so we were wondering if anyone has any idea how we could get a phylogenetic tree to integrate into our analysis (for e.g. for phylogenetics-informed diversity studies) with the taxonomy data we get from taxonomic id/classification of shotgun data?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 16:08:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 16:08:45"><div class="time">2023-01-13 16:08:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you use MEGAN you can export nwk trees of output from MALT/MEGAN classfiication</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 16:09:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 16:09:59"><div class="time">2023-01-13 16:09:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (i.e. RMA6 files)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 16:13:33">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-01-13 16:13:33"><div class="time">2023-01-13 16:13:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for your super fast reply! A couple of people tried this with no success. Also, do you know if  there is another way for non-MALT/MEGAN users?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 16:14:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 16:14:20"><div class="time">2023-01-13 16:14:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No success in what way?</p>

<p>Not off the top of my head concretely, but I'm pretty sure you can upload litsts of TaxIDs to services <em>like</em> iTol, and/or possibly build something with NCBI entrez... (but don't hold me to that üò¨ )</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:20:22">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-01-13 17:20:22"><div class="time">2023-01-13 17:20:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I had no success with the MEGAN-exported trees, and wasn't able to figure out why. It looked properly formatted, but no program would read them. The issue I had with trying to build trees in anything related to NCBI was that they were formatted like the big NCBI taxnomy tree, but not nwk, and I couldn't get the ncbi-formatted trees read by any program to convert them to nwk format. If someone knows how do do that, it might help</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:43:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 17:43:30"><div class="time">2023-01-13 17:43:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hm interesting, because that's what I did in our pnss paper (the Megan one)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:44:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 17:44:11"><div class="time">2023-01-13 17:44:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I dunno what the next soaam Tisch will be about but I could try and look what I've done before and report back for them</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:44:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 17:44:14"><div class="time">2023-01-13 17:44:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Then**</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 19:24:28">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-01-13 19:24:28"><div class="time">2023-01-13 19:24:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes we definitely need another spaamtisch for phylogenetic trees bc we discussed them only in the last 15 min. However next month I thought we try another format (e.g. inviting an author to talk about their research)üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:53:24">
            
                <img src="https://avatars.slack-edge.com/2021-09-23/2522174708661_ef8b03f2d80d95a455bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Camila Duitama
                     <span class="print-only user-email">(camiladuitama@gmail.com)</span>
                </div>
                <a href="#2023-01-13 17:53:24"><div class="time">2023-01-13 17:53:24</div></a>
                <div class="msg">
                    <p>Hi all,
Has anybody used <a href="https://sites.google.com/view/recentrifuge">Recentrifuge</a> or <a href="https://deconseq.sourceforge.net/manual.html">Deconseq</a> to remove contaminating samples from your ancient oral metagenomes? If you have, what samples have you used as negative controls in the case of Recentrifuge or which remove/retain database have you used for DeconSeq?
Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:55:58">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-13 17:55:58"><div class="time">2023-01-13 17:55:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I did not use Recentrifuge myself but my colleagues used sequenced blanks as negative controls when running Recentrifuge (as far as I know)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Camila Duitama, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 18:04:01">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2023-01-13 18:04:01"><div class="time">2023-01-13 18:04:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have used it, I would say generally that would be blanks or controls but which type usually depends on the study</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Camila Duitama, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-14 09:08:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-14 09:08:43"><div class="time">2023-01-14 09:08:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've not used the two tools but, as @irinavelsko said at the last <b>#spaamtisch</b>, we used the <code>decontam</code> R package with blanks, but also with archaeological bone samples (femurs) with the package's simple algorithm to additionally remove some environmental contamination too (don't know if that would work for these two tools you mention).</p>

<p>To be honest this would be another good study for someone to do!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-14 09:09:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-14 09:09:06"><div class="time">2023-01-14 09:09:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Try out different decontamination methods and if they work not just on blanks etc</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-14 16:46:31">
            
                <img src="https://avatars.slack-edge.com/2021-09-23/2522174708661_ef8b03f2d80d95a455bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Camila Duitama
                     <span class="print-only user-email">(camiladuitama@gmail.com)</span>
                </div>
                <a href="#2023-01-14 16:46:31"><div class="time">2023-01-14 16:46:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all for the replies :)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-13 17:56:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-13 17:56:42"><div class="time">2023-01-13 17:56:42</div></a>
                <div class="msg">
                    <p>(just want to say it's really nice to see lots of questions and answers(!) on this channel! Keep it up everyone!)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        üòÑ Olivia
                        </div>
                    
                        <div class="message-reaction">
                        üéâ Kelly Blevins, Miriam Bravo, Francisco Zorrilla, Bjorn Bartholdy, Pooja Swali, Felix Key
                        </div>
                    
                        <div class="message-reaction">
                        :headbangingparrot: Maria Lopopolo, Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        üòÅ Meriam van Os
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 14:37:56">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 14:37:56"><div class="time">2023-01-17 14:37:56</div></a>
                <div class="msg">
                    <p>In the spirit of keeping it up, I have a question!</p>

<p>We are trying to deal with removing PCR duplicates in metagenomic mtDNA data. The situation is where you have reads that have been classified with Kraken as e.g. Bovidae mtDNA reads. These can map to many different mtDNA genomes (cow, goat, sheep, etc), and so you can't necessarily dedup by position. We could just dedup based on sequence identity, but we usually do this based on position. Does anyone have any thoughts? It seems like there must be a best practice..</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 15:02:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 15:02:21"><div class="time">2023-01-17 15:02:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* AFAIK generally in metagenomics people <em>don't</em> really do this, as it's assumed duplication is (mostly) equal across all reads, and it's not really worth the additional computational processing/extra steps to remove them.</p>

<p>I believe people who do do it do it like you said based on sequence identity prior classification (IIRC BBTools has a function for this)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 15:03:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 15:03:26"><div class="time">2023-01-17 15:03:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What do you mean by metagenomic mtDNA data though? SG sequenced data against a mtDNA Kraken database?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 15:03:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 15:03:55"><div class="time">2023-01-17 15:03:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>fastp</code> also does deduplication: <a href="https://github.com/OpenGene/fastp#duplication-rate-and-deduplication">https://github.com/OpenGene/fastp#duplication-rate-and-deduplication</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/OpenGene/fastp#duplication-rate-and-deduplication">GitHub - OpenGene/fastp: An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)</a></div>
                                <div class="link-text">
                                    An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...) - GitHub - OpenGene/fastp: An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filter...
                                </div>
                                
                                
    
        <a href="https://github.com/OpenGene/fastp#duplication-rate-and-deduplication">
            <img class="preview" src="https://opengraph.githubassets.com/25fdd8b5fd1589b5cf8d85341f07b5a3434ef093690b005e1b27105944c515b3/OpenGene/fastp"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/OpenGene/fastp#duplication-rate-and-deduplication</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 17:39:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2023-01-17 17:39:33"><div class="time">2023-01-17 17:39:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Not quite sure what you mean but I have used this to dedup in the past: <a href="https://github.com/BioInfoTools/BBMap/blob/master/sh/clumpify.sh">https://github.com/BioInfoTools/BBMap/blob/master/sh/clumpify.sh</a>
But also KrakenUniq is somewhat duplication aware</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/BioInfoTools/BBMap/blob/master/sh/clumpify.sh">BBMap/clumpify.sh at master ¬∑ BioInfoTools/BBMap</a></div>
                                <div class="link-text">
                                    (Not Offical) BBMap short read aligner, and other bioinformatic tools. - BBMap/clumpify.sh at master ¬∑ BioInfoTools/BBMap
                                </div>
                                
                                
    
        <a href="https://github.com/BioInfoTools/BBMap/blob/master/sh/clumpify.sh">
            <img class="preview" src="https://opengraph.githubassets.com/014526b505e6688ca97df85bfbc7e17b2afd6d838b1bf3cbdd0478c9ef8d4919/BioInfoTools/BBMap"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/BioInfoTools/BBMap/blob/master/sh/clumpify.sh</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Betsy Nelson
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 17:40:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 17:40:25"><div class="time">2023-01-17 17:40:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Clumpify that's the one!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 17:40:38">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2023-01-17 17:40:38"><div class="time">2023-01-17 17:40:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I like it üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:39:03">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:39:03"><div class="time">2023-01-17 19:39:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates This is capture data, with probes that tile ~250 mammalian mitochondrial genomes. We use Kraken to bin the reads into Family groups (hence Bovidae), and then map to each reference mtDNA genome within that Family (e.g. cow, sheep, etc).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:39:44">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:39:44"><div class="time">2023-01-17 19:39:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Clumpify looks good - assuming that it still requires reads to be the same length, then allowing distance between reads would help catch sequencing errors.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:40:57">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:40:57"><div class="time">2023-01-17 19:40:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We also thought about just not deduping. How then would you calculate the number of fragments that you have as data? e.g. 100 DNA fragments mapping to a genome is a lot more data than 10 fragments with a duplication rate of 10.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:58:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:58:19"><div class="time">2023-01-17 19:58:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah I see. Well that isn't metagenomics really although you used the tool upstream, you're asking a slightly different question.</p>

<p>You shouldn't really have sequencing error at this stage, you should have QC'd your reads to remove badly called bases by the sequencer.</p>

<p>And really in metagenomics we don't care necessarily about the exact number of reads but rather relative abundance so it's also why we don't tend to need to dedup (with the assumptions etc).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:58:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:58:43"><div class="time">2023-01-17 19:58:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Well by metagenomics I mean stuff like Microbial ecology/microbiome etc</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:59:32">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:59:32"><div class="time">2023-01-17 19:59:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Are you gatekeeping metagenomics, James?? üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò¨ James Fellows Yates, Laura Epp
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:59:36">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:59:36"><div class="time">2023-01-17 19:59:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's a bit different with e.g. pathogen detection, but again the metagenomic part is too detect, afterwards it's standard Genomics usually</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 19:59:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 19:59:58"><div class="time">2023-01-17 19:59:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Blame reviewers for being picky with terminology!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:00:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:00:11"><div class="time">2023-01-17 20:00:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But no, it's more have to think a bit differently</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:00:12">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:00:12"><div class="time">2023-01-17 20:00:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I guess I think of it as metagenomics because our libraries <em>do</em> have multiple species in them.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:00:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:00:34"><div class="time">2023-01-17 20:00:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah but then all biological samples are metagenomic ;)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:00:35">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:00:35"><div class="time">2023-01-17 20:00:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (because they are sediment samples)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:01:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:01:07"><div class="time">2023-01-17 20:01:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But anyway, back to the question: I think clumpify/fastp will help</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:01:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:01:48"><div class="time">2023-01-17 20:01:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But in terms of reporting it depends what your reviewers would expect as their preferred numbers</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:02:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:02:27"><div class="time">2023-01-17 20:02:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would assume at this stage it would be standard Genomics so the 10 reads bit ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:03:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:03:22"><div class="time">2023-01-17 20:03:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My only other suggest that you could try something along the lines of: map two one mtDNA genome, dedup on that, then re'map' back to the 'multi reference'</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:03:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:03:28"><div class="time">2023-01-17 20:03:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Although it's not going to be perfect</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:07:24">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:07:24"><div class="time">2023-01-17 20:07:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Btw: "It is designed for accurate data, meaning Illumina, Ion Torrent, or error-corrected PacBio; it will not work well with high error rate data. Even for Illumina data, quality-trimming or error-correction may be prudent."</p>

<p>From clumpify</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:25:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:25:17"><div class="time">2023-01-17 20:25:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And "For both SE and PE data, fastp supports evaluating its duplication rate and removing duplicated reads/pairs. fastp considers one read as duplicated only if its all base pairs are identical as another one. This meas if there is a sequencing error or an N base, the read will not be treated as duplicated" from fastp</p>

<p>:/</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 20:25:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 20:25:47"><div class="time">2023-01-17 20:25:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So I guess if you're concerned about that, this will remain an issue from then normal ways I'm aware of</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 21:01:28">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 21:01:28"><div class="time">2023-01-17 21:01:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; fastp considers one read as duplicated only if its all base pairs are identical as another one
Yeah, that's what I'm concerned about, but it seemed like clumpify dealt with that? Even if not, there has to be some sort of hacky way to do what I want, I agree.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 21:06:12">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 21:06:12"><div class="time">2023-01-17 21:06:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* e.g. in the clumpify options:</p>

<p>```subs=2              (s) Maximum substitutions allowed between duplicates.</p>

<p>subrate=0.0         (dsr) If set, the number of substitutions allowed will be
                    max(subs, subrate**min(length1, length2)) for 2 sequences.```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 21:06:53">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 21:06:53"><div class="time">2023-01-17 21:06:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Although there does seem to be some hacky-ness even to that solution, given this additional option:</p>

<p><code>scanlimit=5         (scan) Continue for this many reads after encountering a
                    nonduplicate.  Improves detection of inexact duplicates.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-17 22:05:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-17 22:05:29"><div class="time">2023-01-17 22:05:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah exactly...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 09:50:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-18 09:50:18"><div class="time">2023-01-18 09:50:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Benjamin Vernot I agree with @James Fellows Yates that it is not crystal clear whether one should do deduplication in ancient metagenomics or not (I am talking now only about "microbial metagenomics", it might be different if one is after mammalian reads from an environmental or sedaDNA).</p>

<p>My understanding is that the concept of duplicate removal comes from the whole-genome sequencing of  a sample with a clear host (human, animal etc.). There, we know that the host DNA is definitely present in the sample, we often map the reads to the host reference genome alone, and all we want is to do a good variant calling for a popgen analysis, and duplicates can obviously bias the variant calling, therefore they are removed.</p>

<p>In contrast, when working with ancient microbiome, we generally have very little prior knowledge about what microbes are present in a sample. I.e. accurate variant calling is not our first challenge, we are often satisfied and happy if we reliably can conclude presence of a certain microbe in a sample. Therefore we are working simultaneously with thousands (or even millions) reference genomes for an unbiased profiling, and our major first challenge (in my opinion) is the microbial discovery itself (yes or no, present or absent), and not really the genetics of the microbes. And not even quantification of microbial abundance (at the beginning). For doing microbial profiling of ancient metagenomic samples we are not really after good quality alignments immediately. Therefore, I doubt deduplication would be beneficial for microbial discovery (in contrast, it might reduce sensitivity of the discovery), and I also doubt duplicates affect dramatically even microbial abundance quantification. Think about RNAseq, where there is uneven coverage across genes (very similar to metagenomics uneven coverage across organisms), people do not generally do duplicate removal in this field, because variant calling is not their major goal, and duplicates do not really bias the discovery (expressed or not) and gene expression quantification.</p>

<p>However, once a microbe has been reliably detected, one can of course proceed with all the steps from popgen analysis involving good quality mapping to a good quality reference (now there is only one reference!), deduplication, filtering poor sequencing quality or poor mapping quality reads, masking certain regions etc.. This all is needed for a good variant calling.</p>

<p>In summary, there is a difference in motivation behind duplicate removal when working with single and multiple genomes.</p>

<p>P.S. I certainly confirm that removing duplicates based on sequence identity along is not optimal. A definition of a PCR duplicate (to the best of my knowledge üôÇ ) is two reads with identical sequences and identical start and stop positions of mapping. The problem here is that the start and stop positions are very computationally expensive to compare across all reads, therefore some common duplicate removal programs (such as <em>samtools rmdup</em> and <em>Picard</em>) use some heuristics and approximations (for example, they compare only the start but not the stop positions) which result in tones of non-duplicates being removed and tones of duplicates remained. I have seen a lot of artifacts of duplicate removal from those two programs but can't say anything about <em>fastp</em> and <em>clumpify</em>.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Ivany Argueta
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 09:52:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 09:52:43"><div class="time">2023-01-18 09:52:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you @Nikolay Oskolkov - that explains much better than my bad-late-evening rambling üòÖ (looking back now  I do see how it could see I was maybe 'gatekeeping' but that was not at all the intention - sorry @Benjamin Vernot ü§¶  - Nikolay puts it in a much better way)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 09:55:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 09:55:07"><div class="time">2023-01-18 09:55:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maybe another way to go about hte question: what is the precise need to dedup exactly? One could argue given you have similar genomes, does it really matter if they are randomly distributed (they aren't informative for variant calling etc)?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 09:56:32">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 09:56:32"><div class="time">2023-01-18 09:56:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you both for your thoughts!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 09:59:50">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 09:59:50"><div class="time">2023-01-18 09:59:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I guess I'm not fully on the train of "no deduping necessary," even for microbial quantification / detection. For example, your initial ancient DNA fragments will not all be amplified evenly, and so if you don't dedup you'll favor reads with a "better" GC bias, which could in turn bias which microbes you detect. And my understanding of RNAseq is that you don't dedup because you expect many <em>truly independent</em> reads to have the same start and stop, simply because of how the data is generated / exon structure.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:00:27">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:00:27"><div class="time">2023-01-18 10:00:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I don't have a strong analysis to back that up, it's really just my gut, and I trust you when you say that people don't typically do it!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:01:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:01:02"><div class="time">2023-01-18 10:01:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think everyone would agree with you that it's not not <em>necessary</em>, but it's not a priority problem üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:01:14">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:01:14"><div class="time">2023-01-18 10:01:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, that makes sense!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:02:08">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:02:08"><div class="time">2023-01-18 10:02:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For us, the reason we would like to dedup is that we use the number of molecules observed for a particular species as a sort of significance test. i.e., if we observe 10 molecules, maybe we don't really trust that species was present, but if we see 100, we do.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:02:41">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:02:41"><div class="time">2023-01-18 10:02:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So we want to distinguish between 100 truly independent molecules, and 10 independent molecules with a duplication rate of 10.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:02:55">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:02:55"><div class="time">2023-01-18 10:02:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* .. and we want to do this in a systematic way</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:07:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 10:07:30"><div class="time">2023-01-18 10:07:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok! And how does the sequencing QC look (e.g. in fastqc)? Do you have a big concern about base-calling quality resulting in miscalled bases? I ask as after trimming nowadays from my data these look very clean.</p>

<p>Because my <em>feeling</em> then just pulling out the reads IDs Kraken report as bovidae, running a start/stop location with exact sequence match would be sufficient in this case, particularly as you're dealing with very short reads with both ends sequenced (I presume)?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 10:10:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-18 10:10:01"><div class="time">2023-01-18 10:10:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I agree that generally deduplication should be a more conservative way for both detection and variant calling.</p>

<p>I guess a part of the problem is also that we often work with shallow sequencing data in ancient metagenomics and do not want to throw away a large fraction of our reads. When working with a single reference genome, people can afford doing deduplication for clear reasons (i.e. good variant calling). When working with multiple reference genomes, one should probably think twice whether deduplication is not unnecessarily conservative. The risk is that with a thorough deduplication we might discover very few microbes. They will be very, very reliable but most likely modern contamination üòÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:44:58">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:44:58"><div class="time">2023-01-18 15:44:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* TBH I'm not really sure what the FQ looks like for these libraries, but we regularly get duplication rates in the 20s, and yet have very sparse coverage of the mtDNA genome. Even with a low error rate, you'd expect some amount of those duplicates to have mismatches.</p>

<p>I agree that the way to go is to just dedup mapped sequences, we just have this (apparently unique) issue of mapping to multiple genomes. But I will figure out a way around it! üôÇ And that way around it might just be to ignore the issue and have a handful of duplicates kept around. Thank you guys for your help!</p>

<p>@Nikolay Oskolkov I have to admit that I don't follow your argument. If you have very shallow sequencing, then deduping is <em>more</em> robust, and less likely to remove truly independent reads. Keeping more duplicate reads around doesn't give you more signal!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:46:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:46:15"><div class="time">2023-01-18 15:46:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; Keeping more duplicate reads around doesn't give you more signal!
For some tools it does, as they do hard cut offs for minimum reads (e.g., drop evertything under 10 reads, rather than a percentage of e.g. 1%)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:47:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:47:28"><div class="time">2023-01-18 15:47:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But ultimately yes - I think it's a tricky problem! I have had that issue with multi mapping but not at such low such shallow coverage I guess üòï</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:48:22">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:48:22"><div class="time">2023-01-18 15:48:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But then..... why not just duplicate all your reads so they don't get removed, or change the threshold? The thresholds are there for a reason, right?</p>

<p>TBH, this sounds like using a particular statistical test because it gives you p&lt;0.05, and not because it's the right test! I don't mean that is what you're talking about doing, but I"m just saying why my mind is resisting it üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:49:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:49:46"><div class="time">2023-01-18 15:49:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; But then..... why not just duplicate all your reads so they don't get removed, or change the threshold? The thresholds are there for a reason, right?
Some people do in fact do that (Alexander Herbig's group in some contexts!), but the tools I'm referring to are often designed for modern metagneomic samples, not very borderline samples that we deal with (e.g. for pathogen detection), where later you would go capture etc. etc.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:51:56">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:51:56"><div class="time">2023-01-18 15:51:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah, if you're going to capture for further validation, then it makes sense to me to take even very borderline cases. I guess in those cases I would still rather have deduped reads, because then I know how much data I have.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:52:03">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:52:03"><div class="time">2023-01-18 15:52:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But... I think we could go back and forth on this forever üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:52:17">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:52:17"><div class="time">2023-01-18 15:52:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I"m going to go play UNO now üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:52:23">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:52:23"><div class="time">2023-01-18 15:52:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (seriously)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 15:53:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 15:53:05"><div class="time">2023-01-18 15:53:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Tbh much better use of time ü§£</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:14:43">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:14:43"><div class="time">2023-01-18 16:14:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I thought so too, but now my son is pitching a fit because I won! ¬Ø_(„ÉÑ)_/¬Ø</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:15:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:15:42"><div class="time">2023-01-18 16:15:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmmm that sounds very very familiar with my evening two days ago... Have you tried dobble? That can also be... Emotional</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:16:25">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:16:25"><div class="time">2023-01-18 16:16:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We haven't, I'll have to check it out. He's just getting into games a little bit, so we've really only done Memory and UNO.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:16:45">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:16:45"><div class="time">2023-01-18 16:16:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh, I have seen dobble! It seems pretty cool.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:17:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:17:34"><div class="time">2023-01-18 16:17:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah ok! Doubble is between those two in complexity so he might enjoy it!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:18:18">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:18:18"><div class="time">2023-01-18 16:18:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is this one where there is a component of physical prowess / speed?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:18:37">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:18:37"><div class="time">2023-01-18 16:18:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Like you have to recognize you have a match, and be the first?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-18 16:36:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-18 16:36:13"><div class="time">2023-01-18 16:36:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes exactly!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-19 10:14:46">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-19 10:14:46"><div class="time">2023-01-19 10:14:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Benjamin Vernot @James Fellows Yates I agree that a detection threshold can be adjusted to be sensitive enough after duplicate removal has been applied. If I was sure that deduplication programs do a good job, I would probably do that. However, as I mentioned previously, I have seen too many artifacts with <em>Picard</em> and <em>samtools rmdup</em> / <em>samtools markdup</em>, and I know that people, who have seen this as well, write their own deduplication scripts to be on a safe side. This still, however, does not mean that we should not deducplicate provided that we have a good way (accurate and computationally efficient) to do it üôÇ</p>

<p>There is another problem here that I call a "single read limit". Imagine, after deduplication, you found 2 reads in your sedaDNA sample that really look like mammoth reads. I mean, the two reads stand the competitive mapping against a bunch of other mammals (i.e. they have a higher affinity to mammoth compared to other mammals), and they have mammoth-specific alleles that are not present in other mammals. Would you follow up this hit with a capture? Maybe. However, what if, in addition to the 2 mammoth reads, you see 100 Bovidae reads, 5 wolf / dog reads, 20 cat reads, and 1 bison read? I believe you recognize the situation üôÇ What hit would you follow up? Are they all ancient? Well, hard to say for sure for 2 mammoth reads, we can't do any meaningful stats on them ü§î But are they mammoth reads at all? Are we sure that, what we thought was a mammoth-specific allele, is really never encountered in a cow? How confident are we that an accidental damage, sequencing error, misalignment, poor quality reference, glitch of nature and other technical factors, have not affected our conclusion about absence / presence of a mammoth DNA (based on 2 reads only) in our sample? Do we really have multiple mammalian organisms in our sample, or all those reads in reality originate from a single mammal? Now, imagine that you have hundreds samples and hundreds mammals, plants, birds, fish, microbes etc. with only a few reads assigned / mapped to their references after deduplication. You should have some statistical approach for processing / digesting those numerous hits (unless you have time to visually go through all of them in IGV). I am not saying it will be a super-correct statistics to work on a large number of non-independent reads (if you skip deduplication), but at least your statistical algorithms will not fail <em>technically</em>. If you, however, have just a few reads after deducplication, there is no way you can get any meaningful stats from them (unless you are a Bayesian üôÇ ).</p>

<p>This all does not mean that we should not deduplicate üôÇ I just want to say that deduplication is not our <em>major</em> problem üôÇ And therefore, I believe, is not widely emphasized in the metagenomics community. Because it is not a crucial factor that affects our decision a lot; as you said, the threshold can be adjusted. You can deduplicate if you want or you can skip it, I doubt it is going to be decisive (for detection). However, deduplication might bring technical statistical problems with handling too few reads. In contrast, when working with a single genome, PCR duplicates can really bias all the e.g. selection stats, so a good deduplication is one of the major problems there.</p>

<p>I would say, a truly unbiased competitive mapping against numerous good quality references is the major computational challenge in ancient metagenomics that is not really solved (in my opinion). Once it becomes feasible to fit good quality references of hundreds of mammals (and other organisms) into the memory, for an unbiased competitive mapping, we could revisit the deduplication issue üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-19 22:05:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-19 22:05:04"><div class="time">2023-01-19 22:05:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Just came across another tool that may do it: <a href="https://telatin.github.io/seqfu2/tools/derep.html">https://telatin.github.io/seqfu2/tools/derep.html</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">telatin.github.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://telatin.github.io/seqfu2/tools/derep.html">seqfu derep ¬∑ SeqFu docs</a></div>
                                <div class="link-text">
                                    seqfu derep derep is one of the core subprograms of SeqFu, that allows the dereplication of FASTA and FASTQ files. Usage: derep [options] [&amp;lt;...
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://telatin.github.io/seqfu2/tools/derep.html</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-23 21:44:59">
            
                <img src="https://avatars.slack-edge.com/2021-09-23/2522174708661_ef8b03f2d80d95a455bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Camila Duitama
                     <span class="print-only user-email">(camiladuitama@gmail.com)</span>
                </div>
                <a href="#2023-01-23 21:44:59"><div class="time">2023-01-23 21:44:59</div></a>
                <div class="msg">
                    <p>Is anybody aware of a published study of ancient oral DNA in a dilution experiment? Something like what they tested on the <a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0605-2/figures/1">decontam</a> paper but with ancient oral DNA?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-23 21:45:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-23 21:45:48"><div class="time">2023-01-23 21:45:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Nope. I know very few aDNA microbe methodical papers actually</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü•≤ Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-23 23:08:19">
            
                <img src="https://avatars.slack-edge.com/2021-09-23/2522174708661_ef8b03f2d80d95a455bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Camila Duitama
                     <span class="print-only user-email">(camiladuitama@gmail.com)</span>
                </div>
                <a href="#2023-01-23 23:08:19"><div class="time">2023-01-23 23:08:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @James Fellows Yates</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-23 23:29:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2023-01-23 23:29:11"><div class="time">2023-01-23 23:29:11</div></a>
                <div class="msg">
                    <p>Hey all! For people using kraken2 on ancient microbiome samples: what is your go to confidence level? I've read some articles online and they seem to suggest that it might be better to use 0.1 (rather than the 0.0 default). For example see <a href="https://github.com/DerrickWood/kraken2/issues/265">https://github.com/DerrickWood/kraken2/issues/265</a>. But then this is based on modern DNA. I've been trialling some values 0.0 vs 0.05 and 0.1, but I find that if I use 0.1 the classified reads are quite limited, and I thought that it might be a bit too strict for aDNA?</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">GitHub</div>
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/DerrickWood/kraken2/issues/265">Guidance on confidence score ¬∑ Issue #265 ¬∑ DerrickWood/kraken2</a></div>
                                <div class="link-text">
                                    In the manual section on the confidence score, it is stated that we have developed a simple scoring scheme that has yielded good results for us but neither in that section nor in the publication do...
                                </div>
                                
                                
    
        <a href="https://github.com/DerrickWood/kraken2/issues/265">
            <img class="preview" src="https://opengraph.githubassets.com/3dad4f98fe927b8ed4b53580c547986474289941d8b8c7daf43beb1549f2cbb8/DerrickWood/kraken2/issues/265"
                width="500"
                 height="250" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://github.com/DerrickWood/kraken2/issues/265</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 14:21:18">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-01-24 14:21:18"><div class="time">2023-01-24 14:21:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Meriam, I've been asking myself the same, and have been playing around with values up to 0.2</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 08:33:56">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-01-24 08:33:56"><div class="time">2023-01-24 08:33:56</div></a>
                <div class="msg">
                    <p>Good morning all,
I have a Beast2 question - does anyone know if it is possible to get the posterior distribution of the TreeHeight of an internal node? And if so, would anyone have an idea of how to do it? I've been looking online and on the beast.user google group but I haven't yet found how to do. Any help would be greatly appreciated üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 08:34:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 08:34:46"><div class="time">2023-01-24 08:34:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I believe @Arthur Kocher @aidanva used/have used BEAST2, also likely @Meriam Guellil</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Oph√©lie Lebrasseur
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 09:12:38">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 09:12:38"><div class="time">2023-01-24 09:12:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If I remember correctly, you can achieve this by creating a taxon set that contains all the sequences for which this node of interest is the MRCA (see p. 24 of this tutorial: <a href="https://taming-the-beast.org/tutorials/FBD-tutorial/FBD-tutorial.pdf">https://taming-the-beast.org/tutorials/FBD-tutorial/FBD-tutorial.pdf</a>).
Once you have created this taxon set, you can set wide priors to the dates of this taxon sets and let BEAST sample them the posterior.
The tutorial is a bit extensive and there might be a more elegant way, but I remember doing this in  BEAST1 for my master thesis a long time ago.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 12:00:05">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-01-24 12:00:05"><div class="time">2023-01-24 12:00:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much @Alex H√ºbner üòä I'm going to give it a try and report back here!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 15:55:12">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-24 15:55:12"><div class="time">2023-01-24 15:55:12</div></a>
                <div class="msg">
                    <p>Is there a consensus (or personal strong opinions) on estimates of alpha diversity? We'd like to do it through time, so counting on things evening out with many samples is not possible, as we have an evenly spaced temporal dataset with only a few samples for each time point. Have been looking at the breakaway method to correct for undetected taxa (<a href="https://rss.onlinelibrary.wiley.com/doi/full/10.1111/rssc.12206">https://rss.onlinelibrary.wiley.com/doi/full/10.1111/rssc.12206</a>) but could not find a good/complete implementation. Any and all recommendations are welcome!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 17:13:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 17:13:27"><div class="time">2023-01-24 17:13:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* not helpful at all, but I was always a little skeptical with alpha diversity when it comes from ancient samples because contamination will skew that like crazy (and I don't think there is a good environmental decontaminatiom tool).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 17:13:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 17:13:37"><div class="time">2023-01-24 17:13:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @irinavelsko do you have any thoughts?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 17:51:13">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-24 17:51:13"><div class="time">2023-01-24 17:51:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Agree on the issue of contamination but right now more focusing on the effects of different sequencing depth and how to control for it without going through rarefaction (again, for a cross-time comparison, where it is impossible to group samples and hope that things even out)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 08:27:01">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-01-25 08:27:01"><div class="time">2023-01-25 08:27:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My experience with alpha diversity measures leads me to believe we need to do some testing with simulated datasets to be able to understand how this affects different metrics</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 08:27:27">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-01-25 08:27:27"><div class="time">2023-01-25 08:27:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which I know isn't actually helpful, but maybe you have an eager masters student?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 08:29:19">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-01-25 08:29:19"><div class="time">2023-01-25 08:29:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I did some testing in both <a href="https://doi.org/10.1128/msystems.00080-18">10.1128/mSystems.00080-18</a> and the supplement here <a href="https://doi.org/10.1093/pnasnexus/pgac148">https://doi.org/10.1093/pnasnexus/pgac148</a>, but something thorough and systematic wold be really helpful</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 13:37:43">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-25 13:37:43"><div class="time">2023-01-25 13:37:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks a lot, @irinavelsko. Do you or any others have a take on the breakaway method? It sounds quite good and seems to be well-used, just not in ancient metagenomics.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-26 07:57:38">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-01-26 07:57:38"><div class="time">2023-01-26 07:57:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I haven't heard of it before, but I'm interested to know how it works for you if you test it</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-26 16:58:55">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-01-26 16:58:55"><div class="time">2023-01-26 16:58:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Adrian Forsythe is testing it right now, so hopefully we'll be able to report back</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç irinavelsko
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 21:52:56">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-01-24 21:52:56"><div class="time">2023-01-24 21:52:56</div></a>
                <div class="msg">
                    <p>Hello üôÇ can anybody share the ‚Äúsample_age.txt‚Äù we used for Beauti/BEAST during the summer school?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 22:57:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 22:57:33"><div class="time">2023-01-24 22:57:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If it was provided by Aida and Arthur, it <em>should</em> be in here: <a href="https://zenodo.org/record/6983185#.Y9BUM6TTUoA">https://zenodo.org/record/6983185#.Y9BUM6TTUoA</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 22:58:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 22:58:58"><div class="time">2023-01-24 22:58:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* All the material is on the website, under the 'walkthrough' tabs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-24 22:59:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-24 22:59:17"><div class="time">2023-01-24 22:59:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://spaam-community.github.io/wss-summer-school/#/2022/day-5?id=walkthrough">https://spaam-community.github.io/wss-summer-school/#/2022/day-5?id=walkthrough</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">spaam-community.github.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://spaam-community.github.io/wss-summer-school/#/2022/day-5?id=walkthrough">Document</a></div>
                                <div class="link-text">
                                    Description
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://spaam-community.github.io/wss-summer-school/#/2022/day-5?id=walkthrough</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 09:51:52">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-01-25 09:51:52"><div class="time">2023-01-25 09:51:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes, we inclueded within the files. If you have any questions about it let. me know Maria üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 17:21:33">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-01-25 17:21:33"><div class="time">2023-01-25 17:21:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I found them!!! thanks Aida!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 17:23:26">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-01-25 17:23:26"><div class="time">2023-01-25 17:23:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (and James)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 17:48:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-25 17:48:41"><div class="time">2023-01-25 17:48:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see how it is. I don't send you enough cute dog videos, is that what you're saying?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÑ Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 17:48:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-25 17:48:46"><div class="time">2023-01-25 17:48:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* üò¨</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-25 18:49:48">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-01-25 18:49:48"><div class="time">2023-01-25 18:49:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's my  raison d'√™tre üòÅ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 10:52:14">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-01-30 10:52:14"><div class="time">2023-01-30 10:52:14</div></a>
                <div class="msg">
                    <p>Hi all! has anyone had experience using Diamond for taxonomic profiling of ancient datasets with lots of very short reads (&lt;90 bp)?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 10:53:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-30 10:53:14"><div class="time">2023-01-30 10:53:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Guillermo Rangel, I did üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 10:59:42">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-01-30 10:59:42"><div class="time">2023-01-30 10:59:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Nikolay, thanks for replying üôÇ and how did it go for you? I‚Äôm currently trying to run different profilers on my datasets using no-core/taxprofiler‚Ä¶.. At the moment I‚Äôm running Kraken2, Krakenuniq and Diamond, but I‚Äôm currently having issues to finish the Diamond runs, or at least in a decent amount of time (I‚Äôve got a dataset with ~480M reads that crashes after 2 days running, and the log file doesn‚Äôt really tell me much about it, perhaps a memory issue?). I was thinking of setting the --fast option but I‚Äôm afraid it‚Äôll affect the sensitivity?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 10:59:56">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-01-30 10:59:56"><div class="time">2023-01-30 10:59:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* **nf-core</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 11:07:34">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-30 11:07:34"><div class="time">2023-01-30 11:07:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, DIAMOND is much slower than nucleotide-based classifiers (just think about the additional cost of translating the nucleotide sequences to protein sequences), and will probably not classify the vast majority of your reads. On the other hand, DIAMOND can be more conservative / robust in its classification compared to Kraken, i.e. less sensitive to noise. From my experience, DIAMOND results almost always agreed with Kraken results. However, DIAMOND is a modern metagenomics thing which is more suited for long reads, while for ancient metagenomcs DIAMOND is probably not optimal.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 11:09:57">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-01-30 11:09:57"><div class="time">2023-01-30 11:09:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Guillermo Rangel,
I am not familiar with your error message that you get when running DIAMOND, but I second Nikolay about that you should be careful and check that your samples have roughly the same DNA length distribution. Due to the translational step, DIAMOND is not really able to align anything specifically that is shorter than 20 amino acids (or 60 nucleotides). Therefore, an excess of sequences &lt; 60 nucleotides in a sample compared to other samples will give you skewed profile because the short reads won‚Äôt be properly aligned.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Raphael Eisenhofer
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 11:10:37">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-01-30 11:10:37"><div class="time">2023-01-30 11:10:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My rule of thumb is that I consider Kraken as a fast version of blastn while DIAMOND for me is a fast version of blastp üôÇ Regarding the technical limitations, I think I confirm that I had much more headache with DIAMOND than with Kraken. So my suggestion would be to skip DIAMOND at all if you are experiencing technical problems, Kraken results should be good enough, I believe üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 11:13:54">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-01-30 11:13:54"><div class="time">2023-01-30 11:13:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks you so much @Nikolay Oskolkov and @Alex H√ºbner for your answers. Indeed, my samples have lots of reads below 60 bp‚Ä¶. I was actually also thinking about the issue of aligning the rather short peptides that would be translated from these very short reads‚Ä¶ so I think your suggestions make total sense to me‚Ä¶ ok, so I‚Äôll stick to Kraken üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 15:10:43">
            
                <img src="https://avatars.slack-edge.com/2021-11-06/2696551938660_c79c1f6ef245c5ff1040_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christina Warinner
                     <span class="print-only user-email">(warinner@shh.mpg.de)</span>
                </div>
                <a href="#2023-01-30 15:10:43"><div class="time">2023-01-30 15:10:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Agreed, I wouldn‚Äôt recommend using DIAMOND for ancient DNA</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-01-30 15:24:50">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-01-30 15:24:50"><div class="time">2023-01-30 15:24:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks @Christina Warinner üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Christina Warinner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-01 11:24:19">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-02-01 11:24:19"><div class="time">2023-02-01 11:24:19</div></a>
                <div class="msg">
                    <p>Hello all,
I have another BEAST2-related question. I am trying to run a relaxed clock (log normal) on a dataset (full genome, no partition) with tip-dating. I have followed the following tutorial <a href="https://www.beast2.org/2015/06/23/help-beast-acts-weird-or-how-to-set-up-rates.html">https://www.beast2.org/2015/06/23/help-beast-acts-weird-or-how-to-set-up-rates.html</a> in order to understand how to best set the 'substitution rate' in the Site Model panel and 'Clock.rate' in the Clock Model panel. For the former, I have kept it as 1.0, no estimate. For the clock.rate, I have inputted a starting rate based on the values I found in the literature. By clicking the pencil next to the estimate box, I've also been able to add a lower and upper boundary and tick the estimate box.
What I don't understand is why the 'estimate' box for the clock.rate itself is ticked but remains greyed out? Does anyone know what this greying means? When I run Tracer based on the results, I do have the ucld.Mean estimate but I'm unsure if it's taken my parameters into account. Alternatively, I have seen you could click on Mode &gt; Untick Automatic Set Clock Rate, in which case the box becomes ungrey, but I am unsure that's necessary based on what I've read in tutorials and beast user groups. Essentially, I don't know what I am supposed to do üòÖ
I can run two analysis and see if they differ in the results, but if anyone has any clue, I would love the insight üôÇ  Thank you!</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F04MEG6PF7X/screenshot_2023-02-01_at_11.19.43.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-02-01 at 11.19.43.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F04MEG6PF7X/screenshot_2023-02-01_at_11.19.43.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F04MEG6PF7X-21cc9fc02b/screenshot_2023-02-01_at_11.19.43_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="45" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-02 23:24:57">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-02-02 23:24:57"><div class="time">2023-02-02 23:24:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So I found the answer to my own question - I'm posting it here if it's of any use to people - it's essentially because I am using tip-dating, I have strong priors and therefore, Beast automatically detects that it can be estimated. Pretty straightforward but it took me time to get my head around it üòÖ
Also, this blog post may be useful:
<a href="https://www.beast2.org/2020/05/20/estimate-box.html">https://www.beast2.org/2020/05/20/estimate-box.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:19:55">
            
                <img src="https://secure.gravatar.com/avatar/9760142fa7c8ac95c5b97af3dd5a7ec6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Stephanie Dolenz
                     <span class="print-only user-email">(stephanie_dolenz@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 10:19:55"><div class="time">2023-02-15 10:19:55</div></a>
                <div class="msg">
                    <p>Hey everyone! So, to follow up on Ben Vernot‚Äôs previous post we looked into using clumpify, and ran into a problem that maybe someone here has experienced or knows how to remedy.</p>

<p>In short, I am looking at mtDNA of mammals in sediments captured through a mt mammal capture probeset. The sequenced reads were then assigned to specific taxa with kraken ‚Äì here we‚Äôre looking at all reads assigned to a family (i.e. Canidae, Bovidae) or to anything ‚Äúbelow‚Äù that family, i.e. Canis familiaris. We then map those reads to the mtDNA reference genomes for everything in the family. For example for Canidae, I have reads mapped to dogs, coyotes, wolves, etc. In the file, there are PCR duplicate sequences that I want to collapse into one and have been trying to find the best way to do this.</p>

<p>To test, we‚Äôre starting with a bam file of 2399 reads that have been classified as Canidae by kraken.</p>

<p>We have tried a few things so far:</p><ul><li>When we just get unique DNA sequences (with uniq), we get 799 unique sequences.</li><li>When we look at mapped positions (with samtools rmdup), we get 1344 unique sequences. The reason this is larger is that some duplicates map to multiple Canidae reference genomes, and so identical sequences often don‚Äôt have the same position.</li><li>When we use rmdup followed by uniq on the sequences, we get 775 unique sequences.</li><li>When we use clumpify with the dedup parameter, we get 788 unique sequences, which seems pretty good, but some of these are clearly PCR duplicates with 1-2 edits between them.</li>
</ul>

<p>I then tried messing with other options for clumpify (the kmers, substitutions, and border parameters) and s10, k5, b0 seems to be the ‚Äúbest‚Äù option, reducing the number of sequences to 772.</p>

<p>This is the command line: clumpify.sh in=example<em>fasta.fa out=example</em>fasta_clumped.fa dedupe=t scanlimit=2500000 subs=10 k=5 border=0 ignorebadquality</p>

<p>However, in looking at the file there are still two sequence pairs that have only one mismatch between them. I made a new fasta with just those four sequences to see if maybe it was something with the file, but clumpify is still not collapsing them.</p>

<p>This is the file with the sequences I have that are not working:</p>

<p>&gt;test1
TATGGGGTCAAAACCACATTCGTAGGGGCTTGTCTTGCCTGT
&gt;test2
TATGGGGTCAAAACCACATTCGTAGGGGCTTGTCTTGTCTGT
&gt;test3
TTTATGGGTAGCTCGTCTGGTTTCGGGGAGCTTAGTTTAAGTTCTTTGTGTTAAGTTGTTT
&gt;test4
TTTATGGGTAGCTCGTCTGGTTTCGGGGAGCTTAGTTTAAGTTCTTTTTGTTAAGTTGTTT</p>

<p>When running clumpify, the output is odd as it not only does not collapse them but it also rearranges them in a way that moves two of the like sequences away from each other.</p>

<p>&gt;test3
TTTATGGGTAGCTCGTCTGGTTTCGGGGAGCTTAGTTTAAGTTCTTTGTGTTAAGTTGTTT
&gt;test2
TATGGGGTCAAAACCACATTCGTAGGGGCTTGTCTTGTCTGT
&gt;test1
TATGGGGTCAAAACCACATTCGTAGGGGCTTGTCTTGCCTGT
&gt;test4
TTTATGGGTAGCTCGTCTGGTTTCGGGGAGCTTAGTTTAAGTTCTTTTTGTTAAGTTGTTT</p>

<p>Does anyone know what could be causing this or if I am missing a parameter in clumpify that would resolve this? I have only done this for one library that I have and I want to resolve this before running this on the others.</p>

<p>Even if nobody has seen this before, maybe this exploration will be helpful for someone. Thank you for your help!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:25:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 10:25:00"><div class="time">2023-02-15 10:25:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Looking again at the bbtools docs, I wonder if <code>dedupe</code> might actually be a better tool: <a href="https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/dedupe-guide/">https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/dedupe-guide/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">DOE Joint Genome Institute</div>
                            
                            
                                
                                <div class="link-title"><a href="https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/dedupe-guide/">Dedupe Guide - DOE Joint Genome Institute</a></div>
                                <div class="link-text">
                                    Dedupe was written to eliminate duplicate contigs in assemblies, and later expanded to find all contained and overlapping sequences in a dataset, allowing a specified number of substitutions or edit distance. It is now also capable of clustering sequences based on similarity, and printing dot-formatted all-to-all overlap graphs. Kmer-based assemblers do not typically create redundant...
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Est. reading time</div>
                                        8 minutes
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/dedupe-guide/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:25:36">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 10:25:36"><div class="time">2023-02-15 10:25:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>Dedupe has 6 phases, most of which are optional and depend on the processing mode. They are always executed (or skipped) in the same order.
1) Exact Matches.
During this required phase, sequences are loaded into memory, and exact duplicates (including reverse-complements) are detected and discarded. Hashtables are filled with sequence hash codes of sequences. If containments or overlaps will be examined in later stages, kmers from the ends of sequences will also be added to hash tables. After this phase, the input files will not be used again.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:25:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 10:25:59"><div class="time">2023-02-15 10:25:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* clumpify seems to be doing something slightly different with the actual merging of clusters being an 'extra'</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:29:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 10:29:58"><div class="time">2023-02-15 10:29:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The reason why I say that is I see this: <code>. The clusters are not guaranteed to be overlapping; rather, they are guaranteed to share a kmer, meaning they are likely to overlap.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:30:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 10:30:07"><div class="time">2023-02-15 10:30:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So maybe that's why some of your reads are moved away from each other</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 11:24:37">
            
                <img src="https://secure.gravatar.com/avatar/9760142fa7c8ac95c5b97af3dd5a7ec6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Stephanie Dolenz
                     <span class="print-only user-email">(stephanie_dolenz@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 11:24:37"><div class="time">2023-02-15 11:24:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ahhh okay, that‚Äôs interesting. I am going to mess around with BBtools dedupe instead of clumpify and see if that resolves things</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 11:31:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 11:31:12"><div class="time">2023-02-15 11:31:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Wuld be curious to knw if it works üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 13:06:43">
            
                <img src="https://secure.gravatar.com/avatar/9760142fa7c8ac95c5b97af3dd5a7ec6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Stephanie Dolenz
                     <span class="print-only user-email">(stephanie_dolenz@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-15 13:06:43"><div class="time">2023-02-15 13:06:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôll let you know how it goes ‚ò∫Ô∏è Thanks!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 10:31:54">
            
                <img src="https://avatars.slack-edge.com/2022-10-27/4269387323127_52f8b64cd696f1c66ff3_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Magdalena Haller-Caskie
                     <span class="print-only user-email">(m.haller@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2023-02-15 10:31:54"><div class="time">2023-02-15 10:31:54</div></a>
                <div class="msg">
                    <p>Hi everyone,
I am using FigTree to visualize a tree created with beast2. I am trying to understand the section for node labels, but I cannot find a good explanation for the options. What essentially is the difference between "CAheight<em>95%</em>HPD" and "height<em>_</em>95%_HPD"? Anyone used that recently? What options did you choose?
Thanks for the help! üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-15 11:43:39">
            
                <img src="https://avatars.slack-edge.com/2022-02-01/3039453508706_47989b474549d28d3dce_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jasmin Frangenberg
                     <span class="print-only user-email">(jasmin.frangenberg@hki-jena.de)</span>
                </div>
                <a href="#2023-02-15 11:43:39"><div class="time">2023-02-15 11:43:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Magdalena! :headbangingparrot: So the node label annotation depends not on figtree but originates from the tool with which you created the tree. I guess you should check the documentation of that tool. Figtree just displays it as colour/label/whatever. How did you create the trees?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-16 09:04:45">
            
                <img src="https://avatars.slack-edge.com/2022-02-01/3039453508706_47989b474549d28d3dce_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jasmin Frangenberg
                     <span class="print-only user-email">(jasmin.frangenberg@hki-jena.de)</span>
                </div>
                <a href="#2023-02-16 09:04:45"><div class="time">2023-02-16 09:04:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Today at 5pm (German time) there is the informal <b>#spaamtisch</b> meeting, talking also about phylogenetics today. If you have time to join, feel free to discuss the tool there.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-16 15:16:54">
            
                <img src="https://avatars.slack-edge.com/2022-10-27/4269387323127_52f8b64cd696f1c66ff3_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Magdalena Haller-Caskie
                     <span class="print-only user-email">(m.haller@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2023-02-16 15:16:54"><div class="time">2023-02-16 15:16:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Jasmin! Thanks for your reply. I created the tree with beast2 and checked the documentation. I did not find any good explanation on the node labels yet, however, I will endeavor to keep trying üòâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üí™ Jasmin Frangenberg
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-16 15:37:48">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-02-16 15:37:48"><div class="time">2023-02-16 15:37:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've used beast (+figtree) recently, and used CAheight<em>95%</em>HPD for plotting (that's what others in our group use), but am also not clear on the difference between that and height - in the vast majority of cases they're identical, but I have had a couple of cases with slight differences? would also be interested in finding out what the difference is üòÖ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôÇ Magdalena Haller-Caskie
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-17 09:07:56">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-02-17 09:07:56"><div class="time">2023-02-17 09:07:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Heya,
I found the following on the beast user group: "The CA should stand for "Common Ancestor" and it's 95% height should be different from just height<em>95%</em>HPD (unless the clade has pp = 1) because the clade age estimates are based on ALL posterior trees, and not just the subset for which a particular clade was monophyletic. The Common Ancestor Tree (CAT), which is estimated with treeannotator by invoking the "-heights ca" option is an effective way to deal with negative branches, which is a product of nodes in the MCC tree being older on average than the average age of the immediate parent node. You can find more details in Heled and Bouckaert 2013: <a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-13-221">https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-13-221</a>"</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-13-221">Looking for trees in the forest: summary tree from posterior samples - BMC Ecology and Evolution</a></div>
                                <div class="link-text">
                                    Background Bayesian phylogenetic analysis generates a set of trees which are often condensed into a single tree representing the whole set. Many methods exist for selecting a representative topology for a set of unrooted trees, few exist for assigning branch lengths to a fixed topology, and even fewer for simultaneously setting the topology and branch lengths. However, there is very little research into locating a good representative for a set of rooted time trees like the ones obtained from a BEAST analysis. Results We empirically compare new and known methods for generating a summary tree. Some new methods are motivated by mathematical constructions such as tree metrics, while the rest employ tree concepts which work well in practice. These use more of the posterior than existing methods, which discard information not directly mapped to the chosen topology. Using results from a large number of simulations we assess the quality of a summary tree, measuring (a) how well it explains the sequence data under the model and (b) how close it is to the ‚Äútruth‚Äù, i.e to the tree used to generate the sequences. Conclusions Our simulations indicate that no single method is ‚Äúbest‚Äù. Methods producing good divergence time estimates have poor branch lengths and lower model fit, and vice versa. Using the results presented here, a user can choose the appropriate method based on the purpose of the summary tree.
                                </div>
                                
                                
    
        <a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-13-221">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/12862.jpg"
                width="200"
                 height="283" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-13-221</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Jasmin Frangenberg
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-17 09:23:41">
            
                <img src="https://avatars.slack-edge.com/2022-10-27/4269387323127_52f8b64cd696f1c66ff3_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Magdalena Haller-Caskie
                     <span class="print-only user-email">(m.haller@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2023-02-17 09:23:41"><div class="time">2023-02-17 09:23:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Wow, cool thanks a lot! üôå</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Oph√©lie Lebrasseur
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-17 10:13:00">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-02-17 10:13:00"><div class="time">2023-02-17 10:13:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 16:31:08">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-02-22 16:31:08"><div class="time">2023-02-22 16:31:08</div></a>
                <div class="msg">
                    <p>Hi All,</p>

<p>I am interested in using MaAslin2 for performing a differential abundance test. This program has several options for normalization and analysis method. I was curious if anyone has any thoughts on which combination I should apply to my ancient microbiome dataset. By default, the program uses TSS for normalization and a linear model for analysis method. I was thinking of using CLR and LM but not sure if this is the most appropriate. If people have thoughts about this or recommend that I use another DA tool, I am all ears.</p>

<p>Thanks for the help!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 16:47:10">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-02-22 16:47:10"><div class="time">2023-02-22 16:47:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Sterling, I am not familiar with MaAslin2, but your suggestion about CLR + LM looks good to me. I personally, for any differential abundance test (whether it is microbial abundance or e.g. methylation, gene expression, pixel intensities etc.) would go for a non-parametric Mann-Whittney U test, as the most robust to my experience, rather than any other fancy method. However, if you have confounders to correct your differential test for, this would probably be easier to do within a linear model. Nevertheless, a linear model typically has unrealistic assumptions on residual distributions, this is a drawback of this approach. In summary, to be as conservative as possible I would go for a Mann-Whitney U (MWU) test, but if you have clear confounders, they should be regressed out prior to the MWU test üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 17:38:48">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-02-22 17:38:48"><div class="time">2023-02-22 17:38:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Nikolay! And sorry for not mentioning this in the previous post. One of the main reasons why I have turned towards MaAslin2 is because I can incorporate fixed and random effects (i.e. confounders) into a single model, whereas I cannot do this with others like ANCOM-BC or Lefse. Out of curiosity, do you have any experience with CPLM (Compound Poisson Linear Models)? I have been trying to do a literature  search on this but haven't found anything that is useful.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 19:12:02">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-02-22 19:12:02"><div class="time">2023-02-22 19:12:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* CLR transform + linear model is also my recommendation: you get a lot more flexibility than any other solution</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 19:14:22">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-02-22 19:14:22"><div class="time">2023-02-22 19:14:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and if you have random effects, just switch to a linear mixed model</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 19:19:41">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-02-22 19:19:41"><div class="time">2023-02-22 19:19:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* a nice review, with code examples in the linked gh repo
<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6">Comparison study of differential abundance testing methods using two large Parkinson disease gut microbiome datasets derived from 16S amplicon sequencing - BMC Bioinformatics</a></div>
                                <div class="link-text">
                                    Background Testing for differential abundance of microbes in disease is a common practice in microbiome studies. Numerous differential abundance (DA) testing methods exist and range from traditional statistical tests to methods designed for microbiome data. Comparison studies of DA testing methods have been performed, but none performed on microbiome datasets collected for the study of real, complex disease. Due to this, DA testing was performed here using various DA methods in two large, uniformly collected gut microbiome datasets on Parkinson disease (PD), and their results compared. Results Overall, 78‚Äì92% of taxa tested were detected as differentially abundant by at least one method, while 5‚Äì22% were called differentially abundant by the majority of methods (depending on dataset and filtering of taxonomic data prior to testing). Concordances between method results ranged from 1 to 100%. Average concordance for datasets 1 and 2 were 24% and 28% respectively, and 27% for replicated DA signatures. Concordances increased when removing rarer taxa before testing, increasing average concordances by 2‚Äì32%. Certain methods consistently resulted in higher concordances (e.g. ANCOM-BC, LEfSe), while others consistently resulted in lower (e.g. edgeR, fitZIG). Hierarchical clustering revealed three groups of DA signatures that were (1) replicated by the majority of methods on average and included taxa previously associated with PD, (2) replicated by a subset of methods and included taxa largely enriched in PD, and (3) replicated by few to one method(s). Conclusions Differential abundance tests yielded varied concordances, and amounts of detected DA signatures. Some methods were more concordant than others on both filtered and unfiltered data, therefore, if consistency with other study methodology is a key goal, one might choose among these methods. Even still, using one method on one dataset may find true associations, but may also detect false positives. To help lower false positives, one might analyze data with two or more DA methods to gauge concordance, and use a built-in replication dataset. This study will hopefully serve to complement previously reported DA method comparison studies by implementing and coalescing a large number of both previously and yet to be compared methods on two real gut microbiome datasets.
                                </div>
                                
                                
    
        <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/12859.jpg"
                width="200"
                 height="283" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04193-6</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Sterling Wright
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-22 21:51:25">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-02-22 21:51:25"><div class="time">2023-02-22 21:51:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for this! </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-23 09:07:49">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-02-23 09:07:49"><div class="time">2023-02-23 09:07:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Sterling Wright regarding CPLM, I do not know what it is but I know what Poisson linear model is and assume (from the name üôÇ ) that CPLM might be some mixed model (or bayesian hierarchical model) extension of the Poisson linear model. So, no quick help for you, but I am ready to keep discussing this if you have specific questions. Perhaps I can read up on CPLM if you want to dig into details</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-23 15:12:32">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-02-23 15:12:32"><div class="time">2023-02-23 15:12:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would love to discuss it further but only if you‚Äôre interested and there would be some benefit to you. You have a much deeper understanding on this than me. However, I can also try to reach out to my statistics department if you overwhelmed with other commitments. </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-24 18:52:53">
            
                <img src="https://avatars.slack-edge.com/2022-10-18/4240046991811_c1ef3634af338688b0f7_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Miguel Alejandro Navarro
                     <span class="print-only user-email">(miguelnavarro.academic@gmail.com)</span>
                </div>
                <a href="#2023-02-24 18:52:53"><div class="time">2023-02-24 18:52:53</div></a>
                <div class="msg">
                    <p>Hello everyone!</p>

<p>This is a question for all of you who work with hard boney samples!
Currently, in our lab, we process bone and teeth samples using a hand drill type <em>Dremel 3000-N/10</em>. However, after processing some samples (and all the cleaning) the bone dust has apparently gotten into the inner system rendering it useless.
We are debating whether to repair it or buy another one, so I'd like to know what other type/brands of drill you use or if you have any advice on how to deal with this!</p>

<p>Thank you very much for your help!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-24 18:57:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-24 18:57:06"><div class="time">2023-02-24 18:57:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have no experience myself, but according to our protocols we use something from KAVO</p>

<p><a href="https://www.protocols.io/view/tooth-sampling-from-the-inner-pulp-chamber-for-anc-5qpvo5rj9l4o/v2/materials">https://www.protocols.io/view/tooth-sampling-from-the-inner-pulp-chamber-for-anc-5qpvo5rj9l4o/v2/materials</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">protocols.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.protocols.io/view/tooth-sampling-from-the-inner-pulp-chamber-for-anc-5qpvo5rj9l4o/v2/materials">Tooth Sampling from the inner pulp chamber for ancient DNA Extraction protocol materials</a></div>
                                <div class="link-text">
                                    This protocol describes how to obtain powder from the inner pulp chamber of teeth for the extraction of ancient DNA. It is ideal for the simultaneous isolation of both host endo...
                                </div>
                                
                                
    
        <a href="https://www.protocols.io/view/tooth-sampling-from-the-inner-pulp-chamber-for-anc-5qpvo5rj9l4o/v2/materials">
            <img class="preview" src="https://s3.amazonaws.com/protocols-files/public/95ba4215cced856fae314ba2429e3e562674f480c97151a6c3c4eeed7d5b8a80/csq4battx.png"
                width="580"
                 height="580" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.protocols.io/view/tooth-sampling-from-the-inner-pulp-chamber-for-anc-5qpvo5rj9l4o/v2/materials</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-24 18:57:38">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-24 18:57:38"><div class="time">2023-02-24 18:57:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Gunnar Neumann or @Alina Hiss  might know and have advice? I believe they've done a lot of sampling</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-24 19:22:41">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465355512771_d76bd23c72a4c5f61546_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Gunnar Neumann
                     <span class="print-only user-email">(gunnar_neumann@eva.mpg.de)</span>
                </div>
                <a href="#2023-02-24 19:22:41"><div class="time">2023-02-24 19:22:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes. We use the one indicated in the protocol. Good question though about powder getting inside the system. My impression is that these are quite sealed enough to not have this issue. You can take off the tip for easy cleaning inbetween samples and it has this little rubber ring at the tip which should also prevent the powder to get inside the drill. (Very easy to get lost in the drain though so better order some spare ones). 
We still have to send the drills in for repairing sometimes, but I admit I don‚Äôt know the reasons here. All done but our technicians ‚Ä¶ </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Alina Hiss
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-24 19:29:06">
            
                <img src="https://avatars.slack-edge.com/2021-11-06/2696551938660_c79c1f6ef245c5ff1040_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christina Warinner
                     <span class="print-only user-email">(warinner@shh.mpg.de)</span>
                </div>
                <a href="#2023-02-24 19:29:06"><div class="time">2023-02-24 19:29:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôd recommend getting a foot pedal operated dental drill. Much safer, less contamination risk, and no air blowing near the sample because the motor is separate from the cutting tool.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Biancamaria Bonucci
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-25 09:55:36">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2023-02-25 09:55:36"><div class="time">2023-02-25 09:55:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve worked with different tools in the past. If you stick to Dremel I would definitely recommend using the ‚ÄúWelle‚Äù (I don‚Äôt know the english translation, here the link: <a href="https://www.dremel.com/ch/de/p/dremel-biegsame-welle-26150225ja">https://www.dremel.com/ch/de/p/dremel-biegsame-welle-26150225ja</a>) ideally with a foot pedal. I‚Äôve also used an older dremel ‚ÄúWelle‚Äù which is difficult to clean, in this case I wrapped it with aluminum foil after fixing the drill bit.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">dremel.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.dremel.com/ch/de/p/dremel-biegsame-welle-26150225ja">DREMEL¬Æ Biegsame Welle Vorsatzger√§te f√ºr schwer erreichbare Stellen | Dremel</a></div>
                                <div class="link-text">
                                    Was k√∂nnen Sie damit tun?|Die biegsame Welle von Dremel macht dein Multifunktionswerkzeug zu einem kompakten, leichten und komfortablen Pr√§zisionswerkzeug, das
                                </div>
                                
                                
    
        <a href="https://www.dremel.com/ch/de/p/dremel-biegsame-welle-26150225ja">
            <img class="preview" src="https://www.dremel.com/imagestorage/de-ch/3000-1-82-variable-speed-tool-kit-1194-hires-png-rgb-oneux-11338_w_800_h_418.png"
                width="800"
                 height="418" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.dremel.com/ch/de/p/dremel-biegsame-welle-26150225ja</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Brina Zagorc
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-25 10:00:47">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2023-02-25 10:00:47"><div class="time">2023-02-25 10:00:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For the Bern lab I recently ordered the KaVo Ergo-Grip tool (with a pedal as well), after comparing different models together with a colleague who happens to be a trained dentist. We picked this one because it can be used in both directions (forward and reverse): <a href="https://www.kavo.com/dental-lab-equipment/kergogrip-laboratory-handpieces">https://www.kavo.com/dental-lab-equipment/kergogrip-laboratory-handpieces</a>
We didn‚Äôt get it delivered yet, so I haven‚Äôt tested it yet.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">KaVo Dental</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.kavo.com/dental-lab-equipment/kergogrip-laboratory-handpieces">K-ERGOgrip‚Ñ¢</a></div>
                                <div class="link-text">
                                    Despite the intricate nature of the work itself, the long periods of maximum concentration typical of complex laboratory tasks often require a lot of stamina. The angle one&#39;s wrist needs to be set at in order to work with conventional laboratory drives or handpieces can place dental technicians under significant strain.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://www.kavo.com/dental-lab-equipment/kergogrip-laboratory-handpieces</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-02-28 12:45:12">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2023-02-28 12:45:12"><div class="time">2023-02-28 12:45:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I used the Proxxon <em>MICROMOT 60/EF</em> in Copenhagen, with the footswitch/pedal. <a href="https://www.proxxon.com/en/micromot/28500.php">https://www.proxxon.com/en/micromot/28500.php</a>. It‚Äôs very durable</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Miguel Alejandro Navarro
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-03 23:15:58">
            
                <img src="https://avatars.slack-edge.com/2022-10-18/4240046991811_c1ef3634af338688b0f7_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Miguel Alejandro Navarro
                     <span class="print-only user-email">(miguelnavarro.academic@gmail.com)</span>
                </div>
                <a href="#2023-03-03 23:15:58"><div class="time">2023-03-03 23:15:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you a lot for everyone's response. We are looking thoroughly through the options. My supervisor and I are deeply grateful! ‚ù§Ô∏è</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-20 14:45:28">
            
                <img src="https://avatars.slack-edge.com/2022-11-03/4340746039024_2a64519a2f64f5b20ce9_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Diana
                     <span class="print-only user-email">(diana.spurite@uni-jena.de)</span>
                </div>
                <a href="#2023-03-20 14:45:28"><div class="time">2023-03-20 14:45:28</div></a>
                <div class="msg">
                    <p>Good day. Anyone knows if there are any accessible lists of archaeological sites online? I am looking for Tirup village site in Denmark and i‚Äôve gone as far as translating Danish reports, trying my best to not flip over google‚Äôs suggestion to search for it in Sweden and trying to navigate amongst articles about Trump visiting cemeteries and other nonsense. So, even a location for this specific site would be helpful actually, if there are no available.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-20 15:07:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-20 15:07:12"><div class="time">2023-03-20 15:07:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Unfortunately not, you have you do triage based on literature</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-20 15:07:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-20 15:07:30"><div class="time">2023-03-20 15:07:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So follow up citations in the paper for the archaeological report and see if they have maps etc</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-20 15:07:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-20 15:07:46"><div class="time">2023-03-20 15:07:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Normally research gate can help with this</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-20 21:14:07">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2023-03-20 21:14:07"><div class="time">2023-03-20 21:14:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Let me know if you need a better Danish translator than Google!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè:skin_tone_4: Diana
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-21 12:38:04">
            
                <img src="https://avatars.slack-edge.com/2021-05-15/2061871421878_23d6b222ebb4b281dd17_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Liam Lanigan
                     <span class="print-only user-email">(liamlanigan@palaeome.org)</span>
                </div>
                <a href="#2023-03-21 12:38:04"><div class="time">2023-03-21 12:38:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Tirup isn't a village, it's just the name given to the site which was located in west Horsens, Jutland. The site no longer exists as it's now an industrial area. The exact location is below (with some more information, in Danish). Let me know if you need any help with translation.
<a href="https://historiskatlas.dk/@55.8588727,9.8066840,15z">Tirup kirketomt</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Historisk Atlas</div>
                            
                            
                                
                                <div class="link-title"><a href="https://historiskatlas.dk/@55.8588727,9.8066840,15z">Historisk Atlas</a></div>
                                <div class="link-text">
                                    G√• p√• opdagelse i historiske kort og udforsk din lokale kulturarv.
                                </div>
                                
                                
    
        <a href="https://historiskatlas.dk/@55.8588727,9.8066840,15z">
            <img class="preview" src="https://historiskatlas.dk/images/appicons/logo-1000x1000.png"
                width="1000"
                 height="1000" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://historiskatlas.dk/@55.8588727,9.8066840,15z</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëå:skin_tone_4: Diana
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Bjorn Bartholdy
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-21 12:46:15">
            
                <img src="https://avatars.slack-edge.com/2022-11-03/4340746039024_2a64519a2f64f5b20ce9_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Diana
                     <span class="print-only user-email">(diana.spurite@uni-jena.de)</span>
                </div>
                <a href="#2023-03-21 12:46:15"><div class="time">2023-03-21 12:46:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks a lot. I was wondering if Horsens have spread over the location to be fair. Boldsen in 2000 nicely described surrounding areas, but one can‚Äôt help it if it‚Äôs just not there, between the two places he says it should be.  Anyway thanks a bunch for the precise info.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Liam Lanigan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-21 13:01:24">
            
                <img src="https://avatars.slack-edge.com/2022-11-03/4340746039024_2a64519a2f64f5b20ce9_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Diana
                     <span class="print-only user-email">(diana.spurite@uni-jena.de)</span>
                </div>
                <a href="#2023-03-21 13:01:24"><div class="time">2023-03-21 13:01:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Bjorn Bartholdy I‚Äôm using <a href="https://www.deepl.com/translator">https://www.deepl.com/translator</a> which is somewhat better even with a free version. So the text made sense. ü§ì</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">deepl.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.deepl.com/translator">DeepL Translate: The world&#39;s most accurate translator</a></div>
                                <div class="link-text">
                                    Translate texts &amp;amp; full document files instantly. Accurate translations for individuals and Teams. Millions translate with DeepL every day.
                                </div>
                                
                                
    
        <a href="https://www.deepl.com/translator">
            <img class="preview" src="https://www.DeepL.com/img/favicon/deepl_logo_600_300.png"
                width="600"
                 height="300" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.deepl.com/translator</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-21 21:26:05">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2023-03-21 21:26:05"><div class="time">2023-03-21 21:26:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh cool, haven't tried that one!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:40:19">
            
                <img src="https://avatars.slack-edge.com/2023-01-15/4639294999574_85a756ca97035831649b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Freya Steinhagen
                     <span class="print-only user-email">(stu227847@mail.uni-kiel.de)</span>
                </div>
                <a href="#2023-03-23 12:40:19"><div class="time">2023-03-23 12:40:19</div></a>
                <div class="msg">
                    <p>Hi y'all, I am trying to combine two the two SILVA databases for SSU and LSU for a malt screening (inspired by <a href="https://doi.org/10.1038/s41467-022-33494-4">https://doi.org/10.1038/s41467-022-33494-4</a>  "Ancient marine sediment DNA reveals diatom transition in Antarctica") I have concatenated my two reference databases into one but I am uncertain how I can create a mapping file or if there is a malt-build option that allows me to use two mapping files. I cannot fild the answer to that in the paper. I tried a bit naive with <code>./malt-build --input path/SSU_LSU_comb/SILVA_138.1_SSU_LSU_Ref_Nr99_tax_merged.fasta.gz --sequenceType DNA --index path/SSU_LSU_comb/index -s2t path/SSU_LSU/malt/SSURef_Nr99_132_tax_silva_to_NCBI_synonyms.map.gz -s2t path/SSU_LSU/malt/LSURef_132_tax_silva_to_NCBI_synonyms.map.gz -v</code> Does anyone have a suggestion? best, Freya</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:45:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-23 12:45:26"><div class="time">2023-03-23 12:45:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You would need to combine your mapping file together as well</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:46:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-23 12:46:04"><div class="time">2023-03-23 12:46:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Conceptually anyway</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:46:16">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-23 12:46:16"><div class="time">2023-03-23 12:46:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've never tried making my own mappung file.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:47:57">
            
                <img src="https://avatars.slack-edge.com/2023-01-15/4639294999574_85a756ca97035831649b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Freya Steinhagen
                     <span class="print-only user-email">(stu227847@mail.uni-kiel.de)</span>
                </div>
                <a href="#2023-03-23 12:47:57"><div class="time">2023-03-23 12:47:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, I will try that, I just imagined that duplicate sequences might become a problem</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:51:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-03-23 12:51:14"><div class="time">2023-03-23 12:51:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I <em>think</em> it should be ok if the taxonomy IDs are the same for the same species in both</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-23 12:55:35">
            
                <img src="https://avatars.slack-edge.com/2023-01-15/4639294999574_85a756ca97035831649b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Freya Steinhagen
                     <span class="print-only user-email">(stu227847@mail.uni-kiel.de)</span>
                </div>
                <a href="#2023-03-23 12:55:35"><div class="time">2023-03-23 12:55:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* well, something is running. Thanks a lot üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-28 01:30:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-03-28 01:30:42"><div class="time">2023-03-28 01:30:42</div></a>
                <div class="msg">
                    <p>Non-ancient q, is there some sort of database containing nucleotide sequence data for the isoforms of a given gene across different strains of a bacterium? Something that would give me something like RefSeq‚Äôs ‚Äúidentical proteins‚Äù but for non-identical proteins? I think I‚Äôm really leaning into the title of the channel on this one </p>

<p>edit: I can make a pan-genome, but what I have is a list of gene names/locus tags for one strain, and I want to find the ‚Äúequivalents‚Äù from other strains which aren‚Äôt identical</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-28 07:37:11">
            
                <img src="https://avatars.slack-edge.com/2023-08-11/5727013151811_5866d2031f65a72cd9a6_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ian Light
                     <span class="print-only user-email">(light@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2023-03-28 07:37:11"><div class="time">2023-03-28 07:37:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* closest thing i could think of immediately is a blast search and just exclude results that are &gt;99% ANI? not sure if this is anything close to what you  are looking for</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-28 11:16:24">
            
                <img src="https://avatars.slack-edge.com/2023-03-09/4916965675974_53d973be6570cbc0633b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bruno Gonz√°lez
                     <span class="print-only user-email">(gonzlezb@tcd.ie)</span>
                </div>
                <a href="#2023-03-28 11:16:24"><div class="time">2023-03-28 11:16:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi! Something like this? <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000499">https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000499</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">microbiologyresearch.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000499">A comprehensive and high-quality collection of Escherichia coli genomes and their genes | Microbiology Society</a></div>
                                <div class="link-text">
                                    Escherichia coli
is a highly diverse organism that includes a range of commensal and pathogenic variants found across a range of niches and worldwide. In addition to causing severe intestinal and extraintestinal disease, 
E. coli
is considered a priority pathogen due to high levels of observed drug resistance. The diversity in the 
E. coli
population is driven by high genome plasticity and a very large gene pool. All these have made 
E. coli
one of the most well-studied organisms, as well as a commonly used laboratory strain. Today, there are thousands of sequenced 
E. coli
genomes stored in public databases. While data is widely available, accessing the information in order to perform analyses can still be a challenge. Collecting relevant available data requires accessing different sources, where data may be stored in a range of formats, and often requires further manipulation and processing to apply various analyses and extract useful information. In this study, we collated and intensely curated a collection of over 10‚Ää000 
E. coli
and 
Shigella
genomes to provide a single, uniform, high-quality dataset. 
Shigella
were included as they are considered specialized pathovars of 
E. coli
. We provide these data in a number of easily accessible formats that can be used as the foundation for future studies addressing the biological differences between 
E. coli
lineages and the distribution and flow of genes in the 
E. coli
population at a high resolution. The analysis we present emphasizes our lack of understanding of the true diversity of the 
E. coli
species, and the biased nature of our current understanding of the genetic diversity of such a key pathogen.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000499</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-03-28 11:17:22">
            
                <img src="https://avatars.slack-edge.com/2023-03-09/4916965675974_53d973be6570cbc0633b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bruno Gonz√°lez
                     <span class="print-only user-email">(gonzlezb@tcd.ie)</span>
                </div>
                <a href="#2023-03-28 11:17:22"><div class="time">2023-03-28 11:17:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* "(a) A fasta file containing a single representative sequence of each gene of the gene pool (<a href="https://doi.org/10.6084/m9.figshare.13270073">https://doi.org/10.6084/m9.figshare.13270073</a>, File F3)."</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">figshare</div>
                            
                            
                                
                                <div class="link-title"><a href="https://doi.org/10.6084/m9.figshare.13270073">A comprehensive and high-quality collection of E. coli genomes and their genes</a></div>
                                <div class="link-text">
                                    Supplementary Material for &#39;A comprehensive and high-quality collection of E. coli genomes and their genes&#39; published in Microbial Genomics
                                </div>
                                
                                
    
        <a href="https://doi.org/10.6084/m9.figshare.13270073">
            <img class="preview" src="https://s3-eu-west-1.amazonaws.com/ppreviews-sgm-78489651351689498/26113553/thumb.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://doi.org/10.6084/m9.figshare.13270073</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-11 20:16:45">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-04-11 20:16:45"><div class="time">2023-04-11 20:16:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I never came back to thank you both for your suggestions!! At the end some BLASTing and UniProt wound up working for me in this case, but Anvi‚Äôo seems to have a suite of tools that would help with this kind of question!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:03:50">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:03:50"><div class="time">2023-04-13 13:03:50</div></a>
                <div class="msg">
                    <p>Hi All,
I have a phd student who is looking into 16s rRNA analysis of oxalate samples, any recommendations on best aDNA protocol for those samples?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:06:30">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-04-13 13:06:30"><div class="time">2023-04-13 13:06:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @s.wasef ,
If the data are not yet generated, it would be more interesting to look into metagenomics shotgun sequencing rather than 16s.
They elaborate a bit on why in the latest review on aDNA
<a href="https://www.nature.com/articles/s43586-020-00011-0.pdf">https://www.nature.com/articles/s43586-020-00011-0.pdf</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç s.wasef
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:07:28">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:07:28"><div class="time">2023-04-13 13:07:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Perfect! Thanks Maxime.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:12:30">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:12:30"><div class="time">2023-04-13 13:12:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Any suggestions on best extraction protocol to use with those samples? I did some sediment samples before, should I use same protocol?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:17:59">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-04-13 13:17:59"><div class="time">2023-04-13 13:17:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What kind of samples are these ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:28:05">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:28:05"><div class="time">2023-04-13 13:28:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oxalate from rock art</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:32:24">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-04-13 13:32:24"><div class="time">2023-04-13 13:32:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To follow on from Maxime: I think the 'original' paper was Ziesemer 2016(?) sci Rep. For the issue with 16S being too long for aDNA. Also possibly in Warinner 2017 Annual Reviews Human Genetics. @Tina Warinner?</p>

<p>I'm not sure I've heard of any aDNA protocols on that type of samples myself.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç s.wasef
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:36:01">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:36:01"><div class="time">2023-04-13 13:36:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Neither did I. We were brainstorming about best way to do that, and I thought to ask üòâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:37:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-04-13 13:37:41"><div class="time">2023-04-13 13:37:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Might just have to take a couple of boring Samples and try the standard EDTA extraction... I think a lot of people throw that at a lot of stuff and it sort of just always works</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:38:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-04-13 13:38:58"><div class="time">2023-04-13 13:38:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (if it's safe)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:40:05">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:40:05"><div class="time">2023-04-13 13:40:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah good idea! EDTA is always a good start.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-13 13:40:30">
            
                <img src="https://secure.gravatar.com/avatar/492b22763c46947eec1040169bf7f0ce.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">s.wasef
                     <span class="print-only user-email">(s.wasef@griffith.edu.au)</span>
                </div>
                <a href="#2023-04-13 13:40:30"><div class="time">2023-04-13 13:40:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And handy üëå üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-27 11:01:15">
            
                <img src="https://secure.gravatar.com/avatar/4b8f0795e46428bce5fe9524f3434a4f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Pooja Swali
                     <span class="print-only user-email">(swalipooja@gmail.com)</span>
                </div>
                <a href="#2023-04-27 11:01:15"><div class="time">2023-04-27 11:01:15</div></a>
                <div class="msg">
                    <p>non ancient DNA question - has anyone uploaded data to the ENA server and gotten this message - the project has been public for a week but no data is coming up üòï
<code>No public data has been made available in this project yet. Awaiting submission and/or validation of data.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-27 11:05:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-04-27 11:05:13"><div class="time">2023-04-27 11:05:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes.</p>

<p>The Beethoven data has that at the moment (or last week when I was checking it)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòØ Pooja Swali
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-27 11:05:54">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2023-04-27 11:05:54"><div class="time">2023-04-27 11:05:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* as a general rule ENA submission are a mess. sometimes it also takes a bit to show up. I would make sure the datasets show up under the right samples and project in the upload portal first, might be an association/upload issue and they could still be in the "waiting area"</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü•≤ Pooja Swali
                        </div>
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-27 11:07:44">
            
                <img src="https://secure.gravatar.com/avatar/4b8f0795e46428bce5fe9524f3434a4f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pooja Swali
                     <span class="print-only user-email">(swalipooja@gmail.com)</span>
                </div>
                <a href="#2023-04-27 11:07:44"><div class="time">2023-04-27 11:07:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ahh okay! That's good to know - thank you both! yep they look to be under the right project/sample and are linked (they even have a public status next to each file)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-27 11:32:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-04-27 11:32:26"><div class="time">2023-04-27 11:32:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You can also email their helpdesk to find out if something is stuck. My experience is that they normally quite helpful/friendly</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-04-27 11:56:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-04-27 11:56:57"><div class="time">2023-04-27 11:56:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah. I didn't say they were fast ü§£</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò∂ Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:20:06">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-03 14:20:06"><div class="time">2023-05-03 14:20:06</div></a>
                <div class="msg">
                    <p>Hello; I have a KRAKEN2 related question. I was trying to download taxonomy from NCBI to build a database using command "kraken2-build --download-taxonomy --db $database" but it came up with an error message "Downloading nucleotide gb accession to taxon map...rsync: [Receiver] failed to connect to <a href="http://ftp.ncbi.nlm.nih.gov">ftp.ncbi.nlm.nih.gov</a> (130.14.250.12): Connection timed out (110)". I cant figure out what this error is and how to resolve it? Looking online, they say that there is an issue with the script but I don't know how to modify the script and use it. Can someone help me?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:23:41">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-05-03 14:23:41"><div class="time">2023-05-03 14:23:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* this is a super silly question: do you definitely have the ability to connect to the internet on your server? As a beginner, it looks like it's a connection problem rather than a code problem? Am I right in thinking this?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:30:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-03 14:30:25"><div class="time">2023-05-03 14:30:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can be that NCBI has recently changed the path to ref genomes and Kraken2 hasn‚Äôt updated it yet. If you get this error for all ref genomes. It can also be that 1 (or more) particular genome changed its location or was dropped from the database</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:30:58">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-03 14:30:58"><div class="time">2023-05-03 14:30:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah thats the first thought I had. But we definitely have ability to connect to the server as wget works perfectly for downloading zip files from ncbi.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Maria Zicos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:46:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-03 14:46:15"><div class="time">2023-05-03 14:46:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To follow on the super silly question from Maria: for example I've seen people have this error when they've submitted the build command to their cluster, but only the head node has internet - worker nodes don't have access</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:46:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-03 14:46:26"><div class="time">2023-05-03 14:46:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you're using a cluster might be something to crack</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:46:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-03 14:46:28"><div class="time">2023-05-03 14:46:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Check**</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:48:22">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-03 14:48:22"><div class="time">2023-05-03 14:48:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am not using cluster either üòû</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:48:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-03 14:48:28"><div class="time">2023-05-03 14:48:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Check the NCBI address Kraken2 attempts to connect to and try to wget a file from this address</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Kuldeep More, James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:48:55">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-03 14:48:55"><div class="time">2023-05-03 14:48:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thats what I working on now</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 14:49:33">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-03 14:49:33"><div class="time">2023-05-03 14:49:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I also downloaded the old minikraken database, hoping that it has everything thta I need</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 15:31:46">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-03 15:31:46"><div class="time">2023-05-03 15:31:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Kuldeep More,
It‚Äôs a known issue: <a href="https://github.com/DerrickWood/kraken2/issues/412">https://github.com/DerrickWood/kraken2/issues/412</a>
More or less, you have to go to the source code and replace the statements of <code>ftp://</code> to <code>https://</code>. I faced the same issue recently and was afterwards to download everything.</p>
                    <div class="message-attachment"
                            style="border-color: #36a64f">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/DerrickWood/kraken2/issues/412">#412 Error in creating the standard Kraken2 database</a></div>
                                <div class="link-text">
                                    &lt;p&gt;Hello I&#39;m trying to create the standard Kraken 2 database, with the following command:&lt;/p&gt;

&lt;p&gt;kraken2-build --standard --threads 16 --db kraken-Sdb&lt;/p&gt;

&lt;p&gt;I&#39;m getting this:&lt;/p&gt;

&lt;p&gt;Step 1/2: Performing rsync file transfer of requested files&lt;br /&gt;
Rsync file transfer complete.&lt;br /&gt;
Step 2/2: Assigning taxonomic IDs to sequences&lt;br /&gt;
All files processed, cleaning up extra sequence files... done, library complete.&lt;br /&gt;
Masking low-complexity regions of downloaded library... done.&lt;br /&gt;
Step 1/2: Performing rsync file transfer of requested files&lt;br /&gt;
Rsync file transfer complete.&lt;br /&gt;
Step 2/2: Assigning taxonomic IDs to sequences&lt;br /&gt;
All files processed, cleaning up extra sequence files... done, library complete.&lt;br /&gt;
Masking low-complexity regions of downloaded library... done.&lt;br /&gt;
Step 1/2: Performing rsync file transfer of requested files&lt;br /&gt;
Rsync file transfer complete.&lt;br /&gt;
Step 2/2: Assigning taxonomic IDs to sequences&lt;br /&gt;
All files processed, cleaning up extra sequence files... done, library complete.&lt;br /&gt;
Masking low-complexity regions of downloaded library... done.&lt;br /&gt;
Downloading plasmid files from FTP...&lt;br /&gt;
gzip: plasmid.6.1.genomic.fna.gz: invalid compressed data--format violated&lt;/p&gt;

&lt;p&gt;gzip: plasmid.7.1.genomic.fna.gz: invalid compressed data--format violated&lt;/p&gt;

&lt;p&gt;gzip: plasmid.8.1.genomic.fna.gz: invalid compressed data--format violated&lt;/p&gt;

&lt;p&gt;gzip: plasmid.9.1.genomic.fna.gz: invalid compressed data--format violated&lt;/p&gt;

&lt;p&gt;Any thoughts how I can solve this problem?&lt;/p&gt;

&lt;p&gt;Thank you Nathalia&lt;/p&gt;
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Comments</div>
                                        22
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/DerrickWood/kraken2&#34;&gt;DerrickWood/kraken2&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üòµ Jasmin Frangenberg
                        </div>
                    
                        <div class="message-reaction">
                        üëç Kuldeep More, Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        ü§Ø Maria Zicos
                        </div>
                    
                        <div class="message-reaction">
                        üëÜ:skin_tone_2: Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-03 15:45:31">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-03 15:45:31"><div class="time">2023-05-03 15:45:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Alex. I will try to use this and let you know.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container thread_broadcast ">
        <div id="2023-05-04 11:13:46">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:13:46"><div class="time">2023-05-04 11:13:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay now. I have successfully completed two steps for building a database. Kraken manual is not updated and hence its not that all helpful for beginners like me who don't play with the source code. So for people like me, I am posting what I did üòÜ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Bjorn Bartholdy, Maria Zicos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 11:14:37">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:14:37"><div class="time">2023-05-04 11:14:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To download taxonomy I used kraken2-build --download-taxonomy --db trialdb --use-ftp</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 11:15:05">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:15:05"><div class="time">2023-05-04 11:15:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* --use-ftp part is not in the Kraken2 manual</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 11:16:36">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:16:36"><div class="time">2023-05-04 11:16:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* now the next step is to dowload reference library. kraken2-build --download-library viral --db $DBNAME</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 11:17:14">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:17:14"><div class="time">2023-05-04 11:17:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This command doesnt work because of the same ftp issue. So I dowloaded library directly from NCBI and then added that to the database.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 11:17:27">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:17:27"><div class="time">2023-05-04 11:17:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* kraken2-build --add-to-library viral.1.1.genomic.fna --db trialdb</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 11:17:43">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-04 11:17:43"><div class="time">2023-05-04 11:17:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And then building the database as in the manual</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 12:43:26">
            
                <img src="https://avatars.slack-edge.com/2022-07-05/3771568969681_a44610499a0d5b547df0_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bjorn Bartholdy
                     <span class="print-only user-email">(bpbartholdy@pm.me)</span>
                </div>
                <a href="#2023-05-04 12:43:26"><div class="time">2023-05-04 12:43:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for that! I just downloaded the database from <a href="https://benlangmead.github.io/aws-indexes/k2">https://benlangmead.github.io/aws-indexes/k2</a></p>

<p>I wish I had figured out your solution!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-04 13:20:07">
            
                <img src="https://avatars.slack-edge.com/2023-01-24/4687658891222_849c441b3aeb773b4fc7_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Inder J.
                     <span class="print-only user-email">(inderjit837.ij@gmail.com)</span>
                </div>
                <a href="#2023-05-04 13:20:07"><div class="time">2023-05-04 13:20:07</div></a>
                <div class="msg">
                    <p>Hi everyone! Can someone guide me with NCBI Genome submission? Is it essential to create .ASN file or you can submit .fasta files?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 15:32:20">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-05-05 15:32:20"><div class="time">2023-05-05 15:32:20</div></a>
                <div class="msg">
                    <p>Hello all,
I have a HOPS-related question - In the original paper (<a href="https://doi.org/10.1186/s13059-019-1903-0">https://doi.org/10.1186/s13059-019-1903-0</a>), H√ºlner et al mention a HOPS database containing 6249 reference genomes, and state "HOPS database is available upon request."
Does anyone know if this is accessible somewhere online (I'm thinking maybe it was made publicly available recently since), or do I still need to get in touch?
Thank you for your help!</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://doi.org/10.1186/s13059-019-1903-0">HOPS: automated detection and authentication of pathogen DNA in archaeological remains - Genome Biology</a></div>
                                <div class="link-text">
                                    High-throughput DNA sequencing enables large-scale metagenomic analyses of complex biological systems. Such analyses are not restricted to present-day samples and can also be applied to molecular data from archaeological remains. Investigations of ancient microbes can provide valuable information on past bacterial commensals and pathogens, but their molecular detection remains a challenge. Here, we present HOPS (Heuristic Operations for Pathogen Screening), an automated bacterial screening pipeline for ancient DNA sequences that provides detailed information on species identification and authenticity. HOPS is a versatile tool for high-throughput screening of DNA from archaeological material to identify candidates for genome-level analyses.
                                </div>
                                
                                
    
        <a href="https://doi.org/10.1186/s13059-019-1903-0">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/13059.jpg"
                width="200"
                 height="283" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://doi.org/10.1186/s13059-019-1903-0</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:02:54">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-05 16:02:54"><div class="time">2023-05-05 16:02:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Oph√©lie Lebrasseur, I have never sen that specific database but Ron Huebler replies emails and there are many people in this channel who might have it, or who built own Malt DBs with similar content. I built quite a few, they are not easy to share though because of their size. I would say ~6000 ref genomes is not a big DB and hence prone to low sensitivity / specificity of microbial discovery. The smallest Malt / HOPS DB that I have, comprises ~20 000 complete microbial refseq genomes. Please drop me a direct message if you would like me to try to share this (or other) Malt / HOPS DB with you. Alternatively, I can show you how to build your own, this requires a computer (node) with ~1 TB RAM though</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Nora Bergfeldt
                        </div>
                    
                        <div class="message-reaction">
                        üëç:skin_tone_3: Meriam van Os
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:08:53">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-05 16:08:53"><div class="time">2023-05-05 16:08:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I guess HOPS / Malt DBs can be of general interest, right @James Fellows Yates? We recently published a few big KrakenUniq DBs and Bowtie2 indeces:</p>

<p><a href="https://github.com/NBISweden/aMeta">https://github.com/NBISweden/aMeta</a>
<a href="https://www.biorxiv.org/content/10.1101/2022.10.03.510579v1">https://www.biorxiv.org/content/10.1101/2022.10.03.510579v1</a></p>

<p>So potentially I could try to share 2-3 Malt / HOPS DBs via FigShare if there is a general interest in the channel</p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">NBISweden/aMeta</a></div>
                                <div class="link-text">
                                    Ancient microbiome snakemake workflow
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        11
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Python
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Meriam van Os
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:35:23">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-05-05 16:35:23"><div class="time">2023-05-05 16:35:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello @Nikolay Oskolkov,
Thank you very much for the reply and for offering to share your database :)
I think in a first instance, I'll try and get in touch with Ron Huebler - do you know if that is his email address: "<a href="mailto:huebler@shh.mpg.de">huebler@shh.mpg.de</a>" ?</p>

<p>Apart from that, I agree that HOPS/Malt db are commonly-used tools across the community and that maybe it would be worth investigating having a repository with these databases from papers maybe?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:50:38">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-05 16:50:38"><div class="time">2023-05-05 16:50:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No Ron is gone unfortunately and that email is probably dead. You should first try contacting Alexander Herbig. If he doesn't reply let me know, I think I know where the database is on our servers, I could see if I can put it on Zenodo or something</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:51:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-05 16:51:26"><div class="time">2023-05-05 16:51:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'm a bit wary of sharing malt databass though, there have been lots of problems with MALT recently and they are huge in most cases</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:51:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-05 16:51:49"><div class="time">2023-05-05 16:51:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (just saying as other people have asked me the and thing in the past)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:52:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-05 16:52:26"><div class="time">2023-05-05 16:52:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sorry to clarify: Ron does reply, so You could also try contacting Ron</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:52:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-05 16:52:45"><div class="time">2023-05-05 16:52:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I don't know if he ask access to his email nor the mpi-ev servers anymore</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:53:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-05 16:53:19"><div class="time">2023-05-05 16:53:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And what Nikolay says (as usual)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 16:59:24">
            
                <img src="https://secure.gravatar.com/avatar/731faae224d885462e26ab4d3004397a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Oph√©lie Lebrasseur
                     <span class="print-only user-email">(ophelie.lebrasseur@liverpool.ac.uk)</span>
                </div>
                <a href="#2023-05-05 16:59:24"><div class="time">2023-05-05 16:59:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Great, thank you! I'll try to get in touch with both of them, and let you know!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-05 17:21:35">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-05 17:21:35"><div class="time">2023-05-05 17:21:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ron replied my emails to his address <a href="mailto:huebler@shh.mpg.de">huebler@shh.mpg.de</a>, but it was ~2 years ago üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 13:50:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-05-23 13:50:39"><div class="time">2023-05-23 13:50:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates Which tool would you recommend to replace MALT ? On top of the problems, it is also not handy at all to use</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:23:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-23 22:23:52"><div class="time">2023-05-23 22:23:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* all of them at once ü§£</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:24:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-23 22:24:35"><div class="time">2023-05-23 22:24:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (which is why we've developed nf-core/taxprofiler: <a href="https://nf-co.re/taxprofiler#pipeline-summary">https://nf-co.re/taxprofiler#pipeline-summary</a>)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">nf-co.re</div>
                            
                            
                                
                                <div class="link-title"><a href="https://nf-co.re/taxprofiler#pipeline-summary">taxprofiler ¬ª nf-core</a></div>
                                <div class="link-text">
                                    Highly parallelised multi-taxonomic profiling of shotgun and long-read metagenomic data
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://nf-co.re/taxprofiler#pipeline-summary</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:27:24">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-23 22:27:24"><div class="time">2023-05-23 22:27:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or aMeta (üòâ) the approach I really like. You could probably swap out the HOPS bit with sam2lca after alignment with bwa etc and then damageProfiler (or a better stand alone tool that includes more stats)</p>

<p>Tbh the main obstacle is databases rather than the classification algorithm IMO. But the selection of the latter is driven by what you exactly are aiming to do.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:27:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-23 22:27:34"><div class="time">2023-05-23 22:27:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Database design I mean</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:26:22">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:26:22"><div class="time">2023-05-09 15:26:22</div></a>
                <div class="msg">
                    <p>Hello Everyone, back again with a basic question. Does anyone know how can I retrieve the list of refseq accession numbers based on the list of organism names (one refseq entry per organsim)?  Chatgpt generated some code but it didnot work. If anyone here knows any tool, it will be really helpful. Thanks</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:28:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:28:00"><div class="time">2023-05-09 15:28:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maybe entrezcli  or NCBI data?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:36:29">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:36:29"><div class="time">2023-05-09 15:36:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Tried both but not an option for this specific query. Its very easy the other way around</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:36:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:36:44"><div class="time">2023-05-09 15:36:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* üòû</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:37:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:37:09"><div class="time">2023-05-09 15:37:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Not at the top of my head without playing around with that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:37:16">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:37:16"><div class="time">2023-05-09 15:37:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maxime Borry might have an idea but I think he's travelling atm</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:40:58">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:40:58"><div class="time">2023-05-09 15:40:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Going to use biomartr in R</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:41:05">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:41:05"><div class="time">2023-05-09 15:41:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I will update here if it works</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:41:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:41:27"><div class="time">2023-05-09 15:41:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Please do! I think taht would be really helpful</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:41:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:41:53"><div class="time">2023-05-09 15:41:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you're willing maybe you could even write a short blog post on teh problem and the solution for the &lt;#C02D3DJP3MY|spaam-blog>? Then it would be more findable üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Kuldeep More
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:41:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 15:41:58"><div class="time">2023-05-09 15:41:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (outside of slack)(</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:46:31">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:46:31"><div class="time">2023-05-09 15:46:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There: <a href="https://ftp.ncbi.nlm.nih.gov/genomes/refseq/assembly_summary_refseq.txt">https://ftp.ncbi.nlm.nih.gov/genomes/refseq/assembly<em>summary</em>refseq.txt</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:47:26">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:47:26"><div class="time">2023-05-09 15:47:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* what's the "some code" that didn't work from chatGPT ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:50:10">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:50:10"><div class="time">2023-05-09 15:50:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maxime Borry Check this out</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F056GA12SFR/retrieve_refseq_accessions.py?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">retrieve_refseq_accessions.py</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:54:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-05-09 15:54:47"><div class="time">2023-05-09 15:54:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* something similar to what you want to do @Kuldeep More <a href="https://github.com/maxibor/ncbi-genome-download">https://github.com/maxibor/ncbi-genome-download</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">maxibor/ncbi-genome-download</a></div>
                                <div class="link-text">
                                    Python script for downloading bacterial genomes from NCBI ftp
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        1
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Python
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:10:04">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 16:10:04"><div class="time">2023-05-09 16:10:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Maxime. This one requires the input file with taxid</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:15:40">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 16:15:40"><div class="time">2023-05-09 16:15:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So the biomartr package works it seems. I tried only with 2 names, so that my R doesn't crash. Lets see if it works for 50</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:16:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 16:16:28"><div class="time">2023-05-09 16:16:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ramp it up</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ Kuldeep More
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:33:25">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 16:33:25"><div class="time">2023-05-09 16:33:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yaaaaay! it works</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:33:48">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 16:33:48"><div class="time">2023-05-09 16:33:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So the package is available at <a href="https://cran.r-project.org/web/packages/biomartr/vignettes/Sequence_Retrieval.html#genomeset-retrieval">https://cran.r-project.org/web/packages/biomartr/vignettes/Sequence_Retrieval.html#genomeset-retrieval</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:35:09">
            
                <img src="https://avatars.slack-edge.com/2023-05-03/5198646281462_0a29ddad84f6cfd4c051_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kuldeep More
                     <span class="print-only user-email">(kuldeep.aai@gmail.com)</span>
                </div>
                <a href="#2023-05-09 16:35:09"><div class="time">2023-05-09 16:35:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* In multiple retrievals I provided the list of organism names. Fantastic thing is, it automatically downloads only one refseq entry per organism</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates, Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 15:31:21">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-05-09 15:31:21"><div class="time">2023-05-09 15:31:21</div></a>
                <div class="msg">
                    <p>Hi all! I have a question for sedimentary and coprolite metagenomics people! Have you ever thought of comparing damage patterns in sediments/faeces with those in bone at a same locality? Are there differences in DNA taphonomy according to the substrate it is bound to? Has anyone published on this?</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 16:01:49">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-09 16:01:49"><div class="time">2023-05-09 16:01:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I haven‚Äôt heard about anyone publishing this but maybe @irinavelsko knows.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-09 17:32:18">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-05-09 17:32:18"><div class="time">2023-05-09 17:32:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I haven't looked myself, but this is the paper that comes to mind where they might have <a href="https://www.nature.com/articles/s41598-018-28091-9">https://www.nature.com/articles/s41598-018-28091-9</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Nature</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.nature.com/articles/s41598-018-28091-9">Differential preservation of endogenous human and microbial DNA in dental calculus and dentin</a></div>
                                <div class="link-text">
                                    Scientific Reports - Differential preservation of endogenous human and microbial DNA in dental calculus and dentin
                                </div>
                                
                                
    
        <a href="https://www.nature.com/articles/s41598-018-28091-9">
            <img class="preview" src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41598-018-28091-9/MediaObjects/41598_2018_28091_Fig1_HTML.jpg"
                width="1650"
                 height="890" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.nature.com/articles/s41598-018-28091-9</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-11 13:15:33">
            
                <img src="https://secure.gravatar.com/avatar/3f0f432081691561ae770131929daaf6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Markella Moraitou
                     <span class="print-only user-email">(Markella.Moraitou.0437@student.uu.se)</span>
                </div>
                <a href="#2023-05-11 13:15:33"><div class="time">2023-05-11 13:15:33</div></a>
                <div class="msg">
                    <p>Hello everyone :)</p>

<p>I was wondering if anyone has recently (as in post-Brexit) had any experience transferring samples of CITES-listed specimens from the UK to the EU? My group needs to transfer samples from the UK to Sweden ideally by hand-carrying (rather than shipping it there). We cannot find definitive information of what paperwork is needed (other than the loan forms and certificates that the museum will give us), especially relative to airport customs both before the departure and upon arrival.</p>

<p>Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-16 18:28:33">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-05-16 18:28:33"><div class="time">2023-05-16 18:28:33</div></a>
                <div class="msg">
                    <p>Hi all! When you're sequencing for publication and producing say 50-100 M reads for libraries, how much do you sequence controls? I've done screening with controls before, but my lab mostly does endogenous stuff so they'd never think of including controls on a deeper sequencing run. Would love your insight on this to handle controls properly üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-17 08:51:48">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2023-05-17 08:51:48"><div class="time">2023-05-17 08:51:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We sequence controls to 5M reads, since they're usually pretty clonal. Deeper than that just amplifies more duplicates and doesn't add information</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 15:47:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-05-23 15:47:25"><div class="time">2023-05-23 15:47:25</div></a>
                <div class="msg">
                    <p>Hi everyone!
There is this microbe that shows up a little bit too much in my results to be trusted to my taste üßê:
Pseudopropionibacterium propionicum (or Arachnia propionica). Is it only me or did you notice the same? Is there some explanation for it (better preservation or the like) ? Thanks üòä</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 17:50:39">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2023-05-23 17:50:39"><div class="time">2023-05-23 17:50:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Zoe, I had a look into some of my screening data, and I can find this bacteria in my sample too but not in a crazy amount (I'm working with animal material). It is an inhabitant of the animal and human flora so I'm not surprised. What do you mean by " a little bit too much"? Good luckü¶†</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:01:29">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2023-05-23 22:01:29"><div class="time">2023-05-23 22:01:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It is one of the most prevalent and abundant microbes in human dental calculus. It has been renamed multiple times, which may be why you‚Äôre having trouble tracking it. We wrote a bit about it in our 2017 review (see Figure 4):</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F058YKX60KG/annurev-genom-091416-035526.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">annurev-genom-091416-035526.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F058YKX60KG/annurev-genom-091416-035526.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F058YKX60KG-2f689d1543/annurev-genom-091416-035526_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="811"
                 height="1004" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:22:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-23 22:22:47"><div class="time">2023-05-23 22:22:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think it's a highly diverse species/genus with insufficient genome diversity in databases. we also found it in chimps and gorillas (Fig 2), so it's probably a bog standard commensal (I suspect).</p>

<p>If you check for the number of multi-allelic positions when mapping you get a huge amount suggesting lots of strain diversity in samples too (Fig. S10)</p>

<p>So my feeling is that the single good genome attracts a lot of stuff making it seem stronger than it is</p>

<p><a href="https://www.pnas.org/doi/10.1073/pnas.2021655118">https://www.pnas.org/doi/10.1073/pnas.2021655118</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:24:00">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2023-05-23 22:24:00"><div class="time">2023-05-23 22:24:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good point James - probably best to think of it as a group of related strains/species</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-23 22:30:50">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2023-05-23 22:30:50"><div class="time">2023-05-23 22:30:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you really have a lot of reads you could try de novo assembly</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-24 08:32:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-24 08:32:31"><div class="time">2023-05-24 08:32:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ‚òùÔ∏è</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-24 10:05:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-05-24 10:05:35"><div class="time">2023-05-24 10:05:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Louis L'H√¥te @Tina Warinner and @James Fellows Yates for your answers üòä. It‚Äôs the microbe I find the most when screening samples so I thought there was something fishy about it. But what James suggests makes sense, perhaps the lack of a sufficient number of reference genomes for sister taxa in the database could explain why we so often end up finding this one. I checked the links and I think they show well that we don‚Äôt have enough genomes of this genus yet. Interesting üßê!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üíØ James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üëÄ Louis L&#39;H√¥te
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-24 10:07:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-24 10:07:51"><div class="time">2023-05-24 10:07:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We might also discuss this a bit more somewhere in the PNAS paper, I can't remember now</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 05:06:31">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-05-25 05:06:31"><div class="time">2023-05-25 05:06:31</div></a>
                <div class="msg">
                    <p>Hi everyone! new to the community so good to meet you all! I've been working on some samples, and I get the following deamination pattern on a lot of my samples. I was wondering if anyone could help me interpret this. Thank you!</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F059AU80WAZ/screenshot_from_2023-05-25_05-04-16.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot from 2023-05-25 05-04-16.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F059AU80WAZ/screenshot_from_2023-05-25_05-04-16.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F059AU80WAZ-da3801bd07/screenshot_from_2023-05-25_05-04-16_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="233" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 08:22:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 08:22:42"><div class="time">2023-05-25 08:22:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I <em>have</em> seen that before, but I can't remember what caused it... what does your FastQC results look like? Could you maybe have some form of poly-A tails or something still in there?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 08:49:04">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-05-25 08:49:04"><div class="time">2023-05-25 08:49:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The FastQC results look good enough, there are some overrepresented sequences but they don't match the adapters used and no polyA tails.</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 08:49:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 08:49:34"><div class="time">2023-05-25 08:49:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Tina Warinner @Katerina Guschanski any ideas?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 08:58:48">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-25 08:58:48"><div class="time">2023-05-25 08:58:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Looks like there is some deamination on one side but not on the other side of the reads, which I assume might have something to do with the sequencing protocol (if this pattern is consistent across many samples as @Nicole Wagner mentioned), but I have no idea what it is üòû</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 09:26:54">
            
                <img src="https://avatars.slack-edge.com/2022-06-19/3683707887750_d121d54fed43d75a203d_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yoshiki Wakiyama
                     <span class="print-only user-email">(wakiyama-yoshiki278@g.ecc.u-tokyo.ac.jp)</span>
                </div>
                <a href="#2023-05-25 09:26:54"><div class="time">2023-05-25 09:26:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The same situation is mentioned in this paper.
it may be due to the USER enzyme treatment after blunting.
<a href="https://www.sciencedirect.com/science/article/pii/S0960982219307717#mmc4">https://www.sciencedirect.com/science/article/pii/S0960982219307717#mmc4</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">sciencedirect.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.sciencedirect.com/science/article/pii/S0960982219307717#mmc4">Ancient Genomes Reveal Yamnaya-Related Ancestry and a Potential Source of Indo-European Speakers in Iron Age Tianshan</a></div>
                                <div class="link-text">
                                    Recent studies of early Bronze Age human genomes revealed a massive population expansion by individuals-related to the Yamnaya culture, from the Ponti‚Ä¶
                                </div>
                                
                                
    
        <a href="https://www.sciencedirect.com/science/article/pii/S0960982219307717#mmc4">
            <img class="preview" src="https://ars.els-cdn.com/content/image/1-s2.0-S0960982218X00165-cov150h.gif"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.sciencedirect.com/science/article/pii/S0960982219307717#mmc4</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 09:31:23">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-25 09:31:23"><div class="time">2023-05-25 09:31:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Cool, thanks @Yoshiki Wakiyama, do you mean this paragraph:
<code>The specialty of the NEBNext Ultra DNA Library Prep Kit is that the adaptor provided by this kit contains a Uracil base that needs to be cleaved by the USER enzyme before the amplification step of the libraries. However, given that ancient DNA also contains Uracil bases especially at 5‚Ä≤ end, USER enzyme will also remove any Uracil residuals at 5‚Ä≤ end and thus resulted in the absence of deamination damage on 5‚Ä≤ end for the two samples.</code></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Yoshiki Wakiyama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 09:31:51">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-05-25 09:31:51"><div class="time">2023-05-25 09:31:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nicole Wagner did you use USER treatment?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:03:48">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-05-25 10:03:48"><div class="time">2023-05-25 10:03:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov no, the initial illumina library prep was done with NEB Q5 hot start. However, after being sent to us, the illumina libraries were then converted into circular single strand libraries to be used with MGISEQ and i am not certain conversion kit they used.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:04:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:04:32"><div class="time">2023-05-25 10:04:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ooooh that's really out of 'traditional' aDNA libraries, there coudl potentially be a lot of artefacts in there I imagine...</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Nicole Wagner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:04:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:04:45"><div class="time">2023-05-25 10:04:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @√Öshild (Ash) (if you're availble() has worked with MGISeq before though</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:05:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:05:49"><div class="time">2023-05-25 10:05:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (MGI and ancient DNA I mean...)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:07:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:07:19"><div class="time">2023-05-25 10:07:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Possibly related to @Yoshiki Wakiyama‚Äôs paper he posted above, maybe yo ucould contact the lab, they also did benchmarking: <a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.745508/full">https://www.frontiersin.org/articles/10.3389/fgene.2021.745508/full</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Frontiers</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.745508/full">Comparative Performance of the MGISEQ-2000 and Illumina X-Ten Sequencing Platforms for Paleogenomics</a></div>
                                <div class="link-text">
                                    The MGISEQ-2000 sequencer is widely used in various omics studies, but the performance of this platform for paleogenomics has not been evaluated. We here compare the performance of MGISEQ-2000 with the Illumina X-Ten on ancient human DNA using four samples from 1750BCE to 60CE. We found there were only slight differences between the two platforms in most parameters (duplication rate, sequencing bias, Œ∏, Œ¥S, and Œª). MGISEQ-2000 performed well on endogenous rate and library complexity although X-Ten had a higher average base quality and lower error rate. Our results suggest that MGISEQ-2000 and X-Ten have comparable performance, and MGISEQ-2000 can be an alternative platform for paleogenomics sequencing.
                                </div>
                                
                                
    
        <a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.745508/full">
            <img class="preview" src="https://www.frontiersin.org/files/MyHome%20Article%20Library/745508/745508_Thumb_400.jpg"
                width="400"
                 height="378" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.frontiersin.org/articles/10.3389/fgene.2021.745508/full</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:07:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:07:35"><div class="time">2023-05-25 10:07:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (annoyingly without any plots...)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:07:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:07:53"><div class="time">2023-05-25 10:07:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (which I find <em>slightly</em> suspicious but who knows üòâ )</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:08:04">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-05-25 10:08:04"><div class="time">2023-05-25 10:08:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you! I will read through and follow up!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:09:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:09:06"><div class="time">2023-05-25 10:09:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* An earlier BGI paper: <a href="https://academic.oup.com/gigascience/article/6/8/gix049/3888813">https://academic.oup.com/gigascience/article/6/8/gix049/3888813</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:09:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:09:14"><div class="time">2023-05-25 10:09:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (which looks quite similar to the MGI one funnkly enough)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nicole Wagner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 10:10:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-25 10:10:17"><div class="time">2023-05-25 10:10:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Christian Car√∏e Was on the last paper, if he's also still around</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 15:30:26">
            
                <img src="https://avatars.slack-edge.com/2020-08-19/1308459430789_941850639992ceb7ef84_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">√Öshild (Ash)
                     <span class="print-only user-email">(ashild.v@gmail.com)</span>
                </div>
                <a href="#2023-05-25 15:30:26"><div class="time">2023-05-25 15:30:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* All the ancient DNA data that I‚Äôve worked with that was generated by BGI was from double stranded libraries and behaved and looked like I would expect double stranded Illumina libraries to. I suspect it has something to do with the ‚Äúcircular single strand library‚Äù protocol you used, but I am unfamiliar with it</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Biancamaria Bonucci, Nicole Wagner
                        </div>
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Nicole Wagner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 23:34:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2023-05-25 23:34:51"><div class="time">2023-05-25 23:34:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think I remember someone showing me these patters in single strand libraries‚Ä¶ cannot it be something related to that?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Nicole Wagner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-26 13:41:46">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2023-05-26 13:41:46"><div class="time">2023-05-26 13:41:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The jumpiness of the plot suggests there are very few reads going into the analysis. @Nicole Wagner Can you tell us how many reads are shown in this plot? If it is &lt;1000, you can get strange  and irreproducible patterns. Also, was a proofreading enzyme used as the first enzyme to amplify/index the library? If so, you could get artifacts like this that will suppress the damage pattern in ways that could be asymmetrical.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Nicole Wagner, Nikolay Oskolkov, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-26 14:22:02">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-05-26 14:22:02"><div class="time">2023-05-26 14:22:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for the info @Tina Warinner. There are about 4700 reads. There is no mention of a proofreading enzyme in the protocol I was given, but I will need to double check with the person who prepared the libraries.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-26 14:22:56">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-05-26 14:22:56"><div class="time">2023-05-26 14:22:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* 4700 on-target reads should be sufficient in that regard (if evenly distributed across the genome), so I would lean more on polymerase or other artefacts of the library construction protocol</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nicole Wagner, Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-18 18:56:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-18 18:56:20"><div class="time">2023-06-18 18:56:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nicole Wagner did you find the issue in the end?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-18 19:16:06">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-06-18 19:16:06"><div class="time">2023-06-18 19:16:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates we're pretty sure it's because they used the same prep technique as they use for normal metagenome sequencing. Fortunately, I'm going to the meeting in Tartu where i hope to learn more from experts in the field!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-18 19:17:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-18 19:17:08"><div class="time">2023-06-18 19:17:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What do you mean by that exactly? They used a modern kit or something?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-18 19:20:16">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-06-18 19:20:16"><div class="time">2023-06-18 19:20:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'm not certain which kit they used just that they used the same kit they use for sequencing modern samples. I will learn more before the conference.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-18 19:20:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-18 19:20:45"><div class="time">2023-06-18 19:20:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Awesome! I would be curious to hear :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:33:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 16:33:40"><div class="time">2023-07-19 16:33:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nicole Wagner do you mind if I copy and paste that image to a github issue? (sorry to bring up an old thread) it's so I don't forget to follow up in a few months if yo uwork out what the issue is</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:33:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 16:33:45"><div class="time">2023-07-19 16:33:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ...wait actually</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:34:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 16:34:10"><div class="time">2023-07-19 16:34:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://www.spaam-community.org/little-book-of-smiley-plots/proofreading-polymerase.html">https://www.spaam-community.org/little-book-of-smiley-plots/proofreading-polymerase.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:34:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 16:34:19"><div class="time">2023-07-19 16:34:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Isn't it that with few reads?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:53:51">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-07-19 16:53:51"><div class="time">2023-07-19 16:53:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Of course, feel free to share! That is not mine, but looks very much like it.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:23:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 18:23:09"><div class="time">2023-07-19 18:23:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Aha! Then the Q5 is your culprit :D</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:23:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 18:23:51"><div class="time">2023-07-19 18:23:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://elifesciences.org/articles/73346">https://elifesciences.org/articles/73346</a></p>

<p>&gt; This is in line with the DNA nucleotide misincorporation profiles expected for the type of DNA library constructed (Seguin-Orlando et al., 2015), which was caused by the Q5 polymerase being unable to read through 5' uracils, thereby excluding the typical 5' excess of C-to-T. MapDamage profiles were, thus, consistent with Cytosine deamination at 5'-overhanging ends as the most prominent postmortem DNA degradation reactions (J√≥nsson et al., 2013).
I guess you have low number of reads which is why it's spikey</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">eLife</div>
                            
                            
                                
                                <div class="link-title"><a href="https://elifesciences.org/articles/73346">Radiocarbon and genomic evidence for the survival of Equus Sussemionus until the late Holocene</a></div>
                                <div class="link-text">
                                    The first nuclear genomes of the Equus (Sussemionus) ovodovi enable us to reconstruct the speciation and extinction trajectory of this lineage.
                                </div>
                                
                                
    
        <a href="https://elifesciences.org/articles/73346">
            <img class="preview" src="https://elifesciences.org/assets/images/social/icon-600x600@1.52e87ee6.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://elifesciences.org/articles/73346</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:24:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 18:24:10"><div class="time">2023-07-19 18:24:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* More info: <a href="https://international.neb.com/tools-and-resources/feature-articles/polymerase-fidelity-what-is-it-and-what-does-it-mean-for-your-pcr">https://international.neb.com/tools-and-resources/feature-articles/polymerase-fidelity-what-is-it-and-what-does-it-mean-for-your-pcr</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">international.neb.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://international.neb.com/tools-and-resources/feature-articles/polymerase-fidelity-what-is-it-and-what-does-it-mean-for-your-pcr">Polymerase Fidelity: What is it, and what does it mean for your PCR? | NEB</a></div>
                                <div class="link-text">
                                    The discovery and development of high-fidelity polymerases has for many years been a key focus at New England Biolabs (NEB)
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://international.neb.com/tools-and-resources/feature-articles/polymerase-fidelity-what-is-it-and-what-does-it-mean-for-your-pcr</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:55:39">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-07-19 18:55:39"><div class="time">2023-07-19 18:55:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh, wow, thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 20:16:38">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 20:16:38"><div class="time">2023-07-19 20:16:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This is the Q5 horse data downsampled to about 208 reads, it's smoother because the y-axis has more ticks and cut off is at 25bp not 10 in your plot, but I think it looks pretty similar</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05JHUVE4QG/damageplot.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">DamagePlot.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05JHUVE4QG/damageplot.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05JHUVE4QG-eb9c778644/damageplot_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="1286"
                 height="909" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 20:18:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 20:18:03"><div class="time">2023-07-19 20:18:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or even limited to 10bp:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 20:18:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 20:18:09"><div class="time">2023-07-19 20:18:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05HDN6T4KH/damageplot.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">DamagePlot.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05HDN6T4KH/damageplot.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05HDN6T4KH-d127dcbbf0/damageplot_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="1286"
                 height="909" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-20 06:23:21">
            
                <img src="https://secure.gravatar.com/avatar/43d542dac76b8c674764e12c9f3d9ac4.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nicole Wagner
                     <span class="print-only user-email">(nicole.wagner@ki.se)</span>
                </div>
                <a href="#2023-07-20 06:23:21"><div class="time">2023-07-20 06:23:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you! this is extremely helpful. I will discuss this further with our sequencing team.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-05-25 14:10:33">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2023-05-25 14:10:33"><div class="time">2023-05-25 14:10:33</div></a>
                <div class="msg">
                    <p>Hiya, does anyone know of any (or would be willing to share their) pre built database for krakenunique that would be suitable for sourcetracking? Soil, gut, skin, oral things included. Thanks :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:08:25">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-06-06 16:08:25"><div class="time">2023-06-06 16:08:25</div></a>
                <div class="msg">
                    <p>Dear all, I currently have a sample that contains ancient DNA for a specific bacterial pathogen (mean cov 4X). I also have a phylogeny built using modern reference genomes (based on concatenated core genes). What would be a suitable way of determining the closest modern lineage to my ancient pathogen DNA?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:14:03">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:14:03"><div class="time">2023-06-06 16:14:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Did you include your ancient genome in the phylogeny? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:17:22">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:17:22"><div class="time">2023-06-06 16:17:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you haven‚Äôt done that you could extract a consensus of the core genes from the ancient genome if you reconstructed it by mapping, and use them to built the phylogeny</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:17:31">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:17:31"><div class="time">2023-06-06 16:17:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Together with the modern ones </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:23:58">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-06-06 16:23:58"><div class="time">2023-06-06 16:23:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I didn‚Äôt‚Ä¶. to generate the phylogeny I used a pipeline that uses as input the assembled draft genomes, but I couldn‚Äôt assemble a ‚Äúgood‚Äù genome from the ancient sample due to the low coverage</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:24:50">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-06-06 16:24:50"><div class="time">2023-06-06 16:24:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok, so you suggest me to map the ancient reads to the core genome I identified in the previous analysis, and get a consensus out of that, right?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:25:34">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:25:34"><div class="time">2023-06-06 16:25:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, I think that should work </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:26:35">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:26:35"><div class="time">2023-06-06 16:26:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Although you may lose information at the beginning and end of the genes since it is likely that reads don‚Äôt map there </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:27:47">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:27:47"><div class="time">2023-06-06 16:27:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Alternatively, you can use one of the modern genomes as reference to map your ancient reads and extract consensus for the genes in your core genome if it is easy for you to retrieve the coordinates of those </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:31:42">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-06-06 16:31:42"><div class="time">2023-06-06 16:31:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* that sounds good to me üôÇ I‚Äôm going to try to get that consensus sequence‚Ä¶. just one more thing üòä what tool do you usually use to extract the consensus sequence from the mapping file?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:34:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-06 16:34:57"><div class="time">2023-06-06 16:34:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you're using eager, you could use gatk unified genotyper + multivcfanalyzer. The latter will then produce fasta files - you can look at Aida's two stone age plague papers for examples</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:35:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-06 16:35:20"><div class="time">2023-06-06 16:35:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And I think the eager tutorial for pathogens might have that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:36:32">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-06-06 16:36:32"><div class="time">2023-06-06 16:36:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* oh that‚Äôs a great idea actually, and in Eager I can also input the coordinates that I want to focus the mapping on, right?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:37:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-06 16:37:04"><div class="time">2023-06-06 16:37:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://nf-co.re/eager/2.4.7/usage#tutorial-pathogen-genomics---introduction">https://nf-co.re/eager/2.4.7/usage#tutorial-pathogen-genomics---introduction</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">nf-co.re</div>
                            
                            
                                
                                <div class="link-title"><a href="https://nf-co.re/eager/2.4.7/usage#tutorial-pathogen-genomics---introduction">eager ¬ª nf-core</a></div>
                                <div class="link-text">
                                    A fully reproducible and state-of-the-art ancient DNA analysis pipeline
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://nf-co.re/eager/2.4.7/usage#tutorial-pathogen-genomics---introduction</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:38:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-06 16:38:22"><div class="time">2023-06-06 16:38:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Uhh can't remember about the coordinates... You can get basic coverage stats about gene features based on a bedfilw not sure about the mapping step :/</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:40:04">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-06-06 16:40:04"><div class="time">2023-06-06 16:40:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You can use bedtools fasta to extract specific coordinates from a fasta. I can send a link to it later</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:42:04">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-06-06 16:42:04"><div class="time">2023-06-06 16:42:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* awesome, thank you very much @aidanva and @James Fellows Yates</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-06 16:42:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-06 16:42:57"><div class="time">2023-06-06 16:42:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To clarify: I meant not sure you can do that within eager. Aida is right about bedtools manually!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-07 20:07:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-06-07 20:07:29"><div class="time">2023-06-07 20:07:29</div></a>
                <div class="msg">
                    <p>Because it's a question that came back a few times here, here a updated indices for Kraken2 and KrakenUniq</p>

<p><a href="https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;s=19|https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;s=19">https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;s=19|https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;s=19</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Twitter</div>
                            
                            
                                
                                <div class="link-title"><a href="https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;s=19">Ben Langmead on Twitter</a></div>
                                <div class="link-text">
                                    Double kraken2 announcement: (a) new load of indexes at &lt;https://t.co/7RZ7nbJHye&gt;, and (b) new version v2.1.3 with bug fixes and more efficient masker &lt;https://t.co/tyqpwSwvrp&gt;. Importantly, the release also includes ... (1/2)
                                </div>
                                
                                
    
        <a href="https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;s=19">
            <img class="preview" src="https://pbs.twimg.com/profile_images/1591598909243199488/BvI4xpP4_200x200.jpg"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://twitter.com/BenLangmead/status/1666451382457495552?t=wWLtp-0sSwj0VP6xeM2zsQ&amp;amp;s=19</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, James Fellows Yates, Biancamaria Bonucci, Camila Duitama
                        </div>
                    
                        <div class="message-reaction">
                        üëç:skin_tone_5: Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-21 13:27:19">
            
                <img src="https://avatars.slack-edge.com/2022-11-03/4340746039024_2a64519a2f64f5b20ce9_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Diana
                     <span class="print-only user-email">(diana.spurite@uni-jena.de)</span>
                </div>
                <a href="#2023-06-21 13:27:19"><div class="time">2023-06-21 13:27:19</div></a>
                <div class="msg">
                    <p>Hi everyone. Quick question about the AMDirT - do we only enter shotgun sequenced samples or target-captured also? A paper came in with over hundred samples, most of them target-captured several times, but only a little 30 samples are shot gun sequenced according to the Supplement table.  It's Clavel 2023: <a href="https://doi.org/10.1016/j.isci.2023.106787">https://doi.org/10.1016/j.isci.2023.106787</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">sciencedirect.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://doi.org/10.1016/j.isci.2023.106787">Improving the extraction of ancient Yersinia pestis genomes from the dental pulp</a></div>
                                <div class="link-text">
                                    Ancient DNA preserved in the dental pulp offers the opportunity to characterize the genome of some of the deadliest pathogens in human history. Howeve‚Ä¶
                                </div>
                                
                                
    
        <a href="https://doi.org/10.1016/j.isci.2023.106787">
            <img class="preview" src="https://ars.els-cdn.com/content/image/1-s2.0-S2589004223008647-fx1.jpg"
                width="375"
                 height="375" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://doi.org/10.1016/j.isci.2023.106787</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-21 14:16:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-21 14:16:32"><div class="time">2023-06-21 14:16:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Depends on the table, if it's single genome then captured is OK. </p>

<p>Ultimately you want to include whatever was used in the genome reconstruction, in most cases just the targeted is what is used (as in most people don't mix shotgun and capture in their final reconstruction, I believe). </p>

<p>Does that help?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-23 17:13:58">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-06-23 17:13:58"><div class="time">2023-06-23 17:13:58</div></a>
                <div class="msg">
                    <p>Hi all! Has anyone successfully gotten metaDMG to run from its conda installation? Would love to know if you bumped into any dependency/software/compilation issues and how you resolved them (currently getting an unspecified error that I can't seem to track down). Debating building it from scratch - has anyone had better luck with this?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-23 17:21:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-06-23 17:21:27"><div class="time">2023-06-23 17:21:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Antonio Fernandez-Guerra one for you methinks üòÅ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-23 19:05:04">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-06-23 19:05:04"><div class="time">2023-06-23 19:05:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (More info: I'm running this on a remote linux server where I don't have admin permissions)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-23 19:19:07">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-06-23 19:19:07"><div class="time">2023-06-23 19:19:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think that the core metaDMG-cpp file doesn't exit in the conda environment given by the metaDMG-core git page, so I've been wrangling with a separate installation of it from the metaDMG-cpp git page (to no success yet), but let me know if I've just missed it somehow!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-23 21:53:16">
            
                <img src="https://avatars.slack-edge.com/2022-10-08/4217786516640_cfad32b58f07e023019b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kadir Toykan √ñzdoƒüan
                     <span class="print-only user-email">(k.t.ozdogan@uu.nl)</span>
                </div>
                <a href="#2023-06-23 21:53:16"><div class="time">2023-06-23 21:53:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I had been having hard time with it as well. But this worked for me:
<a href="https://github.com/metaDMG-dev/metaDMG-core/issues/11#issuecomment-1546828093">https://github.com/metaDMG-dev/metaDMG-core/issues/11#issuecomment-1546828093</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/metaDMG-dev/metaDMG-core/issues/11#issuecomment-1546828093">Comment on #11 KeyError: &#39;loggers&#39;</a></div>
                                <div class="link-text">
                                    &lt;p&gt;I don&#39;t know if this will help, but you can create a functioning metaDMG environment using this environment.yaml configuration:&lt;/p&gt;

&lt;p&gt;name: metaDMG&lt;br /&gt;
channels:&lt;/p&gt;

&lt;p&gt;‚Ä¢ conda-forge
‚Ä¢ bioconda
‚Ä¢ defaults&lt;br /&gt;
    dependencies:
‚Ä¢ python==3.9
‚Ä¢ pip
‚Ä¢ pip:
    ‚Ä¢ metaDMG[all]
    ‚Ä¢ logger-tt==1.7.0
‚Ä¢ htslib==1.15
‚Ä¢ cxx-compiler
‚Ä¢ c-compiler
‚Ä¢ eigen
‚Ä¢ gcc_linux-64
‚Ä¢ samtools&lt;/p&gt;
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/metaDMG-dev/metaDMG-core&#34;&gt;metaDMG-dev/metaDMG-core&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-23 22:27:34">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-06-23 22:27:34"><div class="time">2023-06-23 22:27:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Gotcha! Thanks for the info. I stumbled across that chunk of code and got metaDMG-core installed (and presumably) running. Did you have to download and compile metaDMG-cpp separately?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-24 18:30:01">
            
                <img src="https://avatars.slack-edge.com/2023-08-03/5687623376547_da635bb16ef0d8c82526_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yi Wang
                     <span class="print-only user-email">(yiwang@uni-konstanz.de)</span>
                </div>
                <a href="#2023-06-24 18:30:01"><div class="time">2023-06-24 18:30:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Carly, I installed and updated metaDMG some while ago and here are the steps I used. Try if that helps! To answer your questions: I installed metaDMG-cpp separately on the server. During the latest update I needed to specify the version of logger_tt to 1.7.2 (see script below) to make it work.
```# install updated metaDMG with dependency requirements
mamba create -n metaDMG2 python=3.9
conda activate metaDMG2</p>

<p>mamba install -c conda-forge libdeflate=1.6
mamba install -c bioconda htslib=1.10
conda install -c omnia eigen3</p>

<p>mamba install -c bioconda -c conda-forge htslib eigen cxx-compiler gsl
pip install git+<a href="https://github.com/metaDMG-dev/metaDMG-core@stopiferrors_branch">https://github.com/metaDMG-dev/metaDMG-core@stopiferrors_branch</a>
pip install iminuit numpyro joblib numba logger_tt==1.7.2 psutil
pip install metaDMG[all] </p>

<h1>now install metaDMG-cpp</h1>

<p>git clone <a href="https://github.com/metaDMG-dev/metaDMG-cpp.git">https://github.com/metaDMG-dev/metaDMG-cpp.git</a>
cd metaDMG-cpp
make</p>

<h1>update metaDMG-cpp</h1>

<p>conda activate metaDMG2
cd ~/metaDMG-cpp
make clean
git pull <a href="https://github.com/metaDMG-dev/metaDMG-cpp.git">https://github.com/metaDMG-dev/metaDMG-cpp.git</a>
make```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-06-26 19:49:58">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-06-26 19:49:58"><div class="time">2023-06-26 19:49:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for the info - this worked but metaDMG-cpp won't compile üòï I'm thinking the issue is a conflict with my cluster's default environment (gsl/gcc) so hopefully our tech support can help resolve the problem.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-05 18:07:32">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-07-05 18:07:32"><div class="time">2023-07-05 18:07:32</div></a>
                <div class="msg">
                    <p>Hello, if you're submitting samples to the ENA where you've got multiple species' genomes to submit from one sample (they're from host-associated metagenome), do you put the host species ID in the tax_id section of the sample checklist? I also don't know what collection date to put down - do people tend to put date of excavation/ date of sampling or date the sample actually came from? I may also be using the wrong checklist for submitting samples - currently I've got the minimal ENA default sample checklist, but if anyone thinks I should use a different one please let me know! Thanks! üòÑ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-05 18:19:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-05 18:19:40"><div class="time">2023-07-05 18:19:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There should be two taxids field I think, one for host and the other for the sequences themselves. I believe there is a dedicated tax I'd for metagenome</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-05 18:19:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-05 18:19:43"><div class="time">2023-07-05 18:19:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* IIRC</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-05 21:14:40">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-07-05 21:14:40"><div class="time">2023-07-05 21:14:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container thread_broadcast ">
        <div id="2023-07-06 10:07:57">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-07-06 10:07:57"><div class="time">2023-07-06 10:07:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* second stupid q- do people put excavation date for collection date or the actual date associated with a sample? or something else (like date the material was actually sampled for DNA analysis?) Thanks üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-06 10:13:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-06 10:13:52"><div class="time">2023-07-06 10:13:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The very latter is my understanding</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-06 10:14:25">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-07-06 10:14:25"><div class="time">2023-07-06 10:14:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* perfecto</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-06 10:14:47">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-07-06 10:14:47"><div class="time">2023-07-06 10:14:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thank you! sorry for all the questions, just don't want to submit the wrong thing üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-06 10:14:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-06 10:14:55"><div class="time">2023-07-06 10:14:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* IT's likely form here: <a href="https://genomicsstandardsconsortium.github.io/mixs/0000011/">https://genomicsstandardsconsortium.github.io/mixs/0000011/</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-06 10:15:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-06 10:15:03"><div class="time">2023-07-06 10:15:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So sampling <em>for</em> DNA I think</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Iseult
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-07 15:02:38">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2023-07-07 15:02:38"><div class="time">2023-07-07 15:02:38</div></a>
                <div class="msg">
                    <p>another (potentially) stupid question: how much difference does it make to use a sequencing strategy of e.g. PE50 compared to PE150? I know my target sequences are short (~50 bp), so from a technical point of view, PE50 is the way to go. However, from a financial point, the same number of reads costs twice as much when using PE50 (compared to PE150). If I understand the theory behind it, PE150 will generate more overlapping and uninformative reads so I would need a larger number to get the same amount of informative output, but I wonder how big that effect is..</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-07 17:04:21">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2023-07-07 17:04:21"><div class="time">2023-07-07 17:04:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No stupid questions! The number of reads generated depends on the flow cell, not the sequencing chemistry, i.e. a library sequenced on a flow cell at 2x50 will have the same number of demultiplexed reads as a library sequenced on the same flow cell at 2x150. The metric you might be seeing increase with increasing sequencing length is Gb/Tb of output, which just refers to the number of bases sequenced, i.e. a lib sequenced at 2x50 will have fewer sequenced bases than a lib sequenced at 2x150 but the same number of reads. Because of this a 2x50 run is generally cheaper than a 2x150 run, but cost also depends on where you get your sequencing done and which flow cells the centre uses the most. Some centers don‚Äôt even offer 2x50 chemistry.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Lennart Schreiber
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-07 17:09:18">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2023-07-07 17:09:18"><div class="time">2023-07-07 17:09:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If your inserts are only 50bp long, 2x50 is plenty if you can find it cheaper than 2x150 lanes. You should prioritize finding the flow cell that will produce the most reads (millions or billions these days) for your budget, not necessarily the most output (Gb, Tb)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Lennart Schreiber
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 11:38:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-07-10 11:38:49"><div class="time">2023-07-10 11:38:49</div></a>
                <div class="msg">
                    <p>Hi everyone! I don't really know where to ask this so don't hesitate to redirect me somewhere more adequate.
@Emrah Kƒ±rd√∂k and me had to move servers recently and we encounter this error when running MALT. We think it is linked to memory somehow but the job doesn't seem to use up all the memory available on the node either. Here is the example for a job. Log file:
```Version   MALT (version 0.6.1, built 25 Oct 2022)
Author(s) Daniel H. Huson
Copyright (C) 2022 Daniel H. Huson. This program comes with ABSOLUTELY NO WARRANTY.
--- LOADING ---:
Reading file: results/MALT<em>DB/maltDB.dat/ref.idx
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (0.7s)
Reading file: results/MALT</em>DB/maltDB.dat/ref.db
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (573.5s)
Number of sequences:     2,165,573
Number of letters:  47,192,673,378
LOADING table (0) ...
Reading file: results/MALT<em>DB/maltDB.dat/index0.idx
Reference sequence type: DNA
100% (0.0s)
Reading file: results/MALT</em>DB/maltDB.dat/table0.idx
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (60.1s)
Reading file: results/MALT<em>DB/maltDB.dat/table0.db
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (1,344.0s)
Table size: 46,493,548,843
Loading ncbi.map: 2,396,736
Loading ncbi.tre: 2,396,740
Reading file: results/MALT</em>DB/maltDB.dat/taxonomy.idx
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (0.2s)
--- ALIGNING ---:
+++++ Aligning file: results/CUTADAPT<em>ADAPTER</em>TRIMMING/ldo058-b1e1l1p1<em>without</em>human.trimmed.fastq.gz
Starting file: results/MALT/ldo058-b1e1l1p1<em>without</em>human.trimmed.rma6
#</p>

<h1>A fatal error has been detected by the Java Runtime Environment:</h1>

<p>#</p>

<h1>SIGBUS (0x7) at pc=0x00007f4e7399d199, pid=70370, tid=70691</h1>

<p>#</p>

<h1>JRE version: OpenJDK Runtime Environment (18.0.2.1+1) (build 18.0.2.1+1)</h1>

<h1>Java VM: OpenJDK 64-Bit Server VM (18.0.2.1+1, mixed mode, tiered, compressed class ptrs, g1 gc, linux-amd64)</h1>

<h1>Problematic frame:</h1>

<h1>V  [libjvm.so+0x813199]  HeapRegionManager::par_iterate(HeapRegionClosure<strong>, HeapRegionClaimer</strong>, unsigned int) const+0x99</h1>

<p>#</p>

<h1>Core dump will be written. Default location: /cfs/klemming/projects/snic/ancient_microbes/pochonz/aMeta/core</h1>

<p>#</p>

<h1>An error report file with more information is saved as:</h1>

<h1>/cfs/klemming/projects/snic/ancient<em>microbes/pochonz/aMeta/hs</em>err_pid70370.log```</h1>

<p>It's not very explicit. A binary "core" file is generated and a very long "hs<em>err</em>pid70370" (attached). And in the slurm log it just notifies that there is an error by adding "/usr/bin/bash: line 1: 70370 Aborted" before the malt command and the usual
<code>unset DISPLAY; malt-run -at SemiGlobal -m BlastN -i results/CUTADAPT_ADAPTER_TRIMMING/ldo058-b1e1l1p1_without_human.trimmed.fastq.gz -o results/MALT/ldo058-b1e1l1p1_without_human.trimmed.rma6 -a results/MALT/ldo058-b1e1l1p1_without_human.trimmed.sam -t 128 -d results/MALT_DB/maltDB.dat -sup 1 -mq 100 -top 1 -mpi 90.0 -id 90.0 -v &amp;amp;&amp;gt; logs/MALT/ldo058-b1e1l1p1_without_human.log
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)</code>
sacct just says "FAILED" and seff shows that it was far from using up all the memory of the node:
<code>Job ID: 1849320
Cluster: dardel
User/Group: pochonz/pochonz
State: FAILED (exit code 1)
Nodes: 1
Cores per node: 256
CPU Utilized: 00:49:19
CPU Efficiency: 0.55% of 6-05:45:36 core-walltime
Job Wall-clock time: 00:35:06
Memory Utilized: 418.67 GB
Memory Efficiency: 23.39% of 1.75 TB</code>
I don't think it is an error in the code because many malt jobs work and other don't. But it's not stochastic because the ones that don't work, won't work when just rerunning. We think it might be that the jobs are not allowed to use up all of the node RAM as memory somehow, but we are not sure if there could be some magic piece of code to prevent this from happening. The server support is not supportive at all, so if it is something that needs to be done on their end, we might as well have to move again üòÖ
Anyone familiar with that error and knows why it happens and maybe even how to fix it?</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05GY2F99EC/hs_err_pid70370.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">hs_err_pid70370.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 11:45:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-10 11:45:22"><div class="time">2023-07-10 11:45:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; We think it might be that the jobs are not allowed to use up all of the node RAM as memory somehow, 
This <em>might</em> be the error because f**ck Java</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 11:46:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-10 11:46:02"><div class="time">2023-07-10 11:46:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Because it does half second peak memory requests</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 11:46:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-10 11:46:12"><div class="time">2023-07-10 11:46:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And the memory loading is stochastic...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 11:46:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-10 11:46:34"><div class="time">2023-07-10 11:46:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I don't know any more than that üòû</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 12:02:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-07-10 12:02:43"><div class="time">2023-07-10 12:02:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah I hate java too üòÇ. Thanks anyway üòä</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 12:31:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-10 12:31:46"><div class="time">2023-07-10 12:31:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sorry it's not more helpful... but it sounds very similar to what we have on our cluster that uses a Java-hating schuldar</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 15:13:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-07-10 15:13:07"><div class="time">2023-07-10 15:13:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I guess if I don't find a fix, I'll have to splice the fastq files in two parts, until we find a new cluster</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 21:42:06">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-07-10 21:42:06"><div class="time">2023-07-10 21:42:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I was having the same issue with java with another software. For me the problem was that in our cluster managed by slurm they set a java memory someplace I could not access to and could not change not even with the -Xmx flag. But if you haven‚Äôt tried yet, and if you have it set up differently in your cluster, maybe you can increase the memory from the default 64m to -Xmx1024m maybe‚Ä¶(sorry very naively answering to your question).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-11 10:20:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-07-11 10:20:47"><div class="time">2023-07-11 10:20:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @Maria Lopopolo, this is a relevant suggestion and sorry that I forgot to mention that I had already adapted the heap space. Thanks anyway üòä</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-11 16:48:26">
            
                <img src="https://secure.gravatar.com/avatar/ae4c854c061452c58742fe62f743a00a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0025-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Julien Fumey
                     <span class="print-only user-email">(julien.fumey@gmail.com)</span>
                </div>
                <a href="#2023-07-11 16:48:26"><div class="time">2023-07-11 16:48:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi,
Did you change the Xmx value ?
Because in the log file you attached, it says <code>-Xmx2000G</code> which is a request of 2TB. However, according to the seff output you have only 1.75TB. Also, in the log file, it says that you program tried to access memory address that are not existing.
So maybe one solution would be to reduce the request of memory that java is doing (to less than 1.75TB because the system need also memory to manage other processes).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-11 16:51:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-07-11 16:51:17"><div class="time">2023-07-11 16:51:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Julien, I think I already tried before with -Xmx1000G but I can try again with -Xmx1500G and see what happens. Thanks üòä</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-11 16:54:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-07-11 16:54:27"><div class="time">2023-07-11 16:54:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I had been told by a colleague that it was no problem to make java believe that it had more memory available than in the truth. He might have been wrong there üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-11 17:00:49">
            
                <img src="https://secure.gravatar.com/avatar/ae4c854c061452c58742fe62f743a00a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0025-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Julien Fumey
                     <span class="print-only user-email">(julien.fumey@gmail.com)</span>
                </div>
                <a href="#2023-07-11 17:00:49"><div class="time">2023-07-11 17:00:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't know, but you clearly have a memory issue as it says your process made a core dump.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-10 16:04:13">
            
                <img src="https://secure.gravatar.com/avatar/5781f2061db06f7b4f846ef353750138.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Christine Ta
                     <span class="print-only user-email">(christine.ta1092@gmail.com)</span>
                </div>
                <a href="#2023-07-10 16:04:13"><div class="time">2023-07-10 16:04:13</div></a>
                <div class="msg">
                    <p>Hi All,</p>

<p>Is there anyone who works with the CARD (comprehensive antibiotic resistance database) database? I‚Äôve aligned it the reads to CARD v.3.2.7 with MALT and now I am trying to open the .rma6 file in MEGAN but I am having trouble. The CARD (C) button on the tool bar vanished in the newest MEGAN version, and even with the older MEGAN v.6.23.4 the CARD button doesn‚Äôt work.
‚Ä¢ Could this be a problem with CARD being recently updated (June 2023) and MEGAN do not have an updated mapping file?
‚Ä¢ Could this be a problem with how the alignment is done with MALT?
Any insights are much appreciated! Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-13 18:41:45">
            
                <img src="https://secure.gravatar.com/avatar/b6d82a8d6c1082eefeea8152795b0bb6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Claudio Ottoni
                     <span class="print-only user-email">(claudio.ottoni@uniroma2.it)</span>
                </div>
                <a href="#2023-07-13 18:41:45"><div class="time">2023-07-13 18:41:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Christine, I worked with the CARD db, I used Blast though and not MALT so I cannot help you with that. I made a Blast DB from Card and run the analysis as described in the study by Brealey et al, 2020 (<a href="https://academic.oup.com/mbe/article/37/10/3003/5848415">https://academic.oup.com/mbe/article/37/10/3003/5848415</a>). I hope this helps!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-14 16:21:58">
            
                <img src="https://secure.gravatar.com/avatar/5781f2061db06f7b4f846ef353750138.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Christine Ta
                     <span class="print-only user-email">(christine.ta1092@gmail.com)</span>
                </div>
                <a href="#2023-07-14 16:21:58"><div class="time">2023-07-14 16:21:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This helps, thank you Claudio!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-13 10:40:52">
            
                <img src="https://avatars.slack-edge.com/2022-10-08/4217786516640_cfad32b58f07e023019b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kadir Toykan √ñzdoƒüan
                     <span class="print-only user-email">(k.t.ozdogan@uu.nl)</span>
                </div>
                <a href="#2023-07-13 10:40:52"><div class="time">2023-07-13 10:40:52</div></a>
                <div class="msg">
                    <p>Hi all, I need to send a pool of sedaDNA libraries from the Netherlands to Germany. The heatwave got me thinking about what would be the best way to secure the DNA quality. I was thinking of a box filled with dry ice (maybe not necessary) or ice packs. What would be your suggestions? Thanks already!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-13 10:43:51">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2023-07-13 10:43:51"><div class="time">2023-07-13 10:43:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would send DNA (extracts, libraries etc.) always on dry ice, regardless of a heatwave.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Kadir Toykan √ñzdoƒüan, Biancamaria Bonucci
                        </div>
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è:skin_tone_2: Kelly Blevins
                        </div>
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è Tina Warinner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 14:42:02">
            
                <img src="https://secure.gravatar.com/avatar/3d3d58f29a8fd2dec25b1ccc543c1e6e.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Luisa Sacristan
                     <span class="print-only user-email">(lf.sacristan2215@uniandes.edu.co)</span>
                </div>
                <a href="#2023-07-18 14:42:02"><div class="time">2023-07-18 14:42:02</div></a>
                <div class="msg">
                    <p>Hello everyone! I am conducting an analysis on an ancient bacterial genome, and I obtained these graphs using mapDamage. I would like to know if any of you know how to interpret this kind of results. Thank you üôÇ</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05HJM51U9H/screenshot_from_2023-07-13_21-17-08.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot from 2023-07-13 21-17-08.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05HJM51U9H/screenshot_from_2023-07-13_21-17-08.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05HJM51U9H-2e11bfd2c1/screenshot_from_2023-07-13_21-17-08_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="359" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 14:59:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-18 14:59:06"><div class="time">2023-07-18 14:59:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think they look OK (although a bit low), the random bump could be maybe somelike like the end of the adapter still there or something like that?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Luisa Sacristan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 14:59:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-18 14:59:12"><div class="time">2023-07-18 14:59:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Checking FastQC results might make sense to doublec hcek</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 15:03:57">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-18 15:03:57"><div class="time">2023-07-18 15:03:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I also thought that the bump at the end might be due to some weird pattern / low-quality bases left. Regarding whether the deamination looks convincing, I would say I doubt so since the frequency of C/T polymorphisms does not exceed (or very marginally) the frequencies of all other polymorphisms (the grey lines). This however may mean that together with truly ancient reads, there is a bunch of mis-aligned modern reads, so the quality of alignments should be checked</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Luisa Sacristan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 15:05:55">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2023-07-18 15:05:55"><div class="time">2023-07-18 15:05:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Its low but apart from the bump, it looks ok.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Luisa Sacristan, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 15:06:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-18 15:06:23"><div class="time">2023-07-18 15:06:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Congrats on the ancient(ish) genome @Luisa Sacristan! üò¨</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Luisa Sacristan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 15:19:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-18 15:19:59"><div class="time">2023-07-18 15:19:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Luisa Sacristan if you work out what it is, please let us know, then we can add it to the little book of smiley plots üòÑ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôÇ Luisa Sacristan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 15:20:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-18 15:20:08"><div class="time">2023-07-18 15:20:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://www.spaam-community.org/little-book-of-smiley-plots/">https://www.spaam-community.org/little-book-of-smiley-plots/</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Luisa Sacristan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-18 17:24:05">
            
                <img src="https://secure.gravatar.com/avatar/3d3d58f29a8fd2dec25b1ccc543c1e6e.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Luisa Sacristan
                     <span class="print-only user-email">(lf.sacristan2215@uniandes.edu.co)</span>
                </div>
                <a href="#2023-07-18 17:24:05"><div class="time">2023-07-18 17:24:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all! I'll check based on your suggestions.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:34:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 16:34:46"><div class="time">2023-07-19 16:34:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Luisa Sacristan do you mind if I copy just the image to the github repo for the little book of smiley plots?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 16:56:33">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-07-19 16:56:33"><div class="time">2023-07-19 16:56:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Just wondering: besides the weird bump on the 5' end, isn't anyone suspicious about the non-G-A grey substitution on the 3' end that is higher in frequency than the G-A substitution? To me that would be a red flag in terms of accepting the reads from this organism as showing damage patterns. It seems like some other processes are taking place: Mixture or organisms, mixture of damaged and non-damaged reads, ligation biases during library prep, etc.</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î Luisa Sacristan, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:22:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 18:22:13"><div class="time">2023-07-19 18:22:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The grey lines are also elevated 5p, but in both cases the grey 'artefact' is consistent -ish. which is why I think it's a lab artefact at least. If it was organism related everything would be elevated.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:22:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-19 18:22:27"><div class="time">2023-07-19 18:22:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (also interesting it's one grey line on 3p, and not 5p)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-19 18:24:07">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-07-19 18:24:07"><div class="time">2023-07-19 18:24:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Exactly, it's the 3' that would worry me more than the 5'</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Luisa Sacristan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-20 08:52:26">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2023-07-20 08:52:26"><div class="time">2023-07-20 08:52:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The depurination pattern is also odd. A should be enriched, not depleted at 5‚Äô -1. @Luisa Sacristan How was the library built? This looks like an enzyme issue to me. Was this built with Nextera or another similar commercial kit? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-20 10:36:51">
            
                <img src="https://secure.gravatar.com/avatar/3d3d58f29a8fd2dec25b1ccc543c1e6e.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Luisa Sacristan
                     <span class="print-only user-email">(lf.sacristan2215@uniandes.edu.co)</span>
                </div>
                <a href="#2023-07-20 10:36:51"><div class="time">2023-07-20 10:36:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates It's ok üôÇ you can copy the image</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-20 10:40:31">
            
                <img src="https://secure.gravatar.com/avatar/3d3d58f29a8fd2dec25b1ccc543c1e6e.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Luisa Sacristan
                     <span class="print-only user-email">(lf.sacristan2215@uniandes.edu.co)</span>
                </div>
                <a href="#2023-07-20 10:40:31"><div class="time">2023-07-20 10:40:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Tina Warinner yes, the A behavior is not normal. I'm waiting the laboratory answer about the lib preparation.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-20 10:44:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-20 10:44:08"><div class="time">2023-07-20 10:44:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Was there anyhting in your FastQC resutls/</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-21 10:29:16">
            
                <img src="https://secure.gravatar.com/avatar/3d3d58f29a8fd2dec25b1ccc543c1e6e.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Luisa Sacristan
                     <span class="print-only user-email">(lf.sacristan2215@uniandes.edu.co)</span>
                </div>
                <a href="#2023-07-21 10:29:16"><div class="time">2023-07-21 10:29:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates there were no adapter content and Per base sequence quality was good.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-21 10:30:04">
            
                <img src="https://secure.gravatar.com/avatar/3d3d58f29a8fd2dec25b1ccc543c1e6e.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Luisa Sacristan
                     <span class="print-only user-email">(lf.sacristan2215@uniandes.edu.co)</span>
                </div>
                <a href="#2023-07-21 10:30:04"><div class="time">2023-07-21 10:30:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05HXU30A8N/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05HXU30A8N/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05HXU30A8N-97d25f6454/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="248" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-21 10:37:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-21 10:37:30"><div class="time">2023-07-21 10:37:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Then it maybe a ligation bias or something like Katja said</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:39:53">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:39:53"><div class="time">2023-07-27 15:39:53</div></a>
                <div class="msg">
                    <p>Hi everyone! quick question‚Ä¶. for programs like MapDamage, is the reference used required to be a complete genome, or is it possible to also analyse your reads with a draft genome available as separate contigs?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:40:44">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 15:40:44"><div class="time">2023-07-27 15:40:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Contigs are fine!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:42:32">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:42:32"><div class="time">2023-07-27 15:42:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @thanks Nikolay, but I guess somewhere in the analysis I‚Äôve got to set a different parameter/setting in comparison with the case a full reference is used?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:43:10">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:43:10"><div class="time">2023-07-27 15:43:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôm asking because I got an ‚Äúodd‚Äù pattern in my plot, let me show you‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:43:44">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:43:44"><div class="time">2023-07-27 15:43:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This is what I got when using a complete reference</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05JMNAL1FZ/fragmisincorporation_plot_nz_cp019659_1.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Fragmisincorporation_plot_NZ_CP019659_1.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05JMNAL1FZ/fragmisincorporation_plot_nz_cp019659_1.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05JMNAL1FZ-2012053000/fragmisincorporation_plot_nz_cp019659_1_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="770"
                 height="770" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:44:12">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:44:12"><div class="time">2023-07-27 15:44:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and I got this when I used a fragmented reference genome</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05K28X1VJ6/fragmisincorporation_plot_pala_0422_00058.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Fragmisincorporation_plot_PALA_0422_00058.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05K28X1VJ6/fragmisincorporation_plot_pala_0422_00058.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05K28X1VJ6-f2fad1c542/fragmisincorporation_plot_pala_0422_00058_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="770"
                 height="770" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:44:26">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 15:44:26"><div class="time">2023-07-27 15:44:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Mmmm, no, the reference you provide with -r flag can be any level of assembly. It just needs to match the reference used for producing the bam-alignment</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:45:04">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 15:45:04"><div class="time">2023-07-27 15:45:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To me, both look similar</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:46:48">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:46:48"><div class="time">2023-07-27 15:46:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* mmm but why do I get all those extra points in the second one and not in the first one?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:47:33">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 15:47:33"><div class="time">2023-07-27 15:47:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You mean the grey lines?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:48:21">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 15:48:21"><div class="time">2023-07-27 15:48:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* no, I mean in the nucleotide frequency plots (the top 4 panels)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:54:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 15:54:28"><div class="time">2023-07-27 15:54:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, not sure, let me think a bit what exactly and how is being plotted in the top 4 panels</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 15:56:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 15:56:25"><div class="time">2023-07-27 15:56:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Intuitively, non-optimal assembly should result in more mis-alignments and more noisy mean allele frequencies. But what exactly all those scattered points mean, I do not know right now</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:07:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-27 16:07:06"><div class="time">2023-07-27 16:07:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Alex H√ºbner?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:08:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-27 16:08:11"><div class="time">2023-07-27 16:08:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (also @Guillermo Rangel this might be one for: <a href="http://www.spaam-community.org/little-book-of-smiley-plots/">http://www.spaam-community.org/little-book-of-smiley-plots/</a>)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:19:12">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 16:19:12"><div class="time">2023-07-27 16:19:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Guillermo Rangel Do you have many contigs in the fragmented reference? How many?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:21:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 16:21:40"><div class="time">2023-07-27 16:21:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I suspect that one point might be one contig</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:28:13">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 16:28:13"><div class="time">2023-07-27 16:28:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* the assembly has 264 contigs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:28:31">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 16:28:31"><div class="time">2023-07-27 16:28:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* it‚Äôd make sense if each point represents a different contig</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:29:26">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-27 16:29:26"><div class="time">2023-07-27 16:29:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and sure @James Fellows Yates , I‚Äôm happy to contribute a figure to the book üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§© James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:35:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-27 16:35:00"><div class="time">2023-07-27 16:35:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Shout when you know the cause!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-27 16:42:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-27 16:42:18"><div class="time">2023-07-27 16:42:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Guillermo Rangel My interpretation is that in the case of complete genome you have one (or a few very similar) reference, therefore one point or a few very close points, while with the fragmented genome you have a lot of heterogeneity in terms of mapping to each contig. If you go to the raw output from mapDamage, there should be 264 points for the top 4 panels, i.e. one of the dimensions of the matrix used for constructing the plots in the top 4 panels should be 264</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-28 10:14:08">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-28 10:14:08"><div class="time">2023-07-28 10:14:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* hey @Nikolay Oskolkov I think you‚Äôre right üôÇ I just checked the files and manual‚Ä¶ so for the misincorporation plot the program combines the results for all the reference sequences used (in my case all the contigs), but for the dnacomp plot it doesn‚Äôt, so I think each point per position represents a different reference sequence</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-28 15:14:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-07-28 15:14:55"><div class="time">2023-07-28 15:14:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So the wierd pattern is alignment against a single contig?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-31 09:09:40">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-31 09:09:40"><div class="time">2023-07-31 09:09:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think that for each position in the dnacomp plots each point represents the nucleotide frequency for each one of the contigs in the target assembly, which in my case had 264 contigs</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-31 09:11:01">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-07-31 09:11:01"><div class="time">2023-07-31 09:11:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and I guess the connected dots are the mean or the median for each position</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-07-31 09:11:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-07-31 09:11:28"><div class="time">2023-07-31 09:11:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, agree!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-01 09:18:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-01 09:18:10"><div class="time">2023-08-01 09:18:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Guillermo Rangel if you're happy with GitHub, you can make a PR:</p>

<p><a href="https://github.com/SPAAM-community/intro-to-ancient-metagenomics-book/">https://github.com/SPAAM-community/intro-to-ancient-metagenomics-book/</a></p>

<p>(instructions are on teh rEADME, but you can probably copy from the other pages üôÇ</p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">SPAAM-community/intro-to-ancient-metagenomics-book</a></div>
                                <div class="link-text">
                                    Introduction to Ancient Metagenomics Book
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Website</div>
                                        &lt;http://www.spaam-community.org/intro-to-ancient-metagenomics-book/&gt;
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        3
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-01 09:19:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-01 09:19:04"><div class="time">2023-08-01 09:19:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* IF you're not, please could you send me the following mapDamage/DamageProfiler files:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05KP2F1Y6P/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05KP2F1Y6P/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05KP2F1Y6P-093a67a694/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="273"
                 height="100" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-01 09:19:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-01 09:19:17"><div class="time">2023-08-01 09:19:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And a short descriptio nof what causes the misincorproation/smiley plots :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:35:56">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 15:35:56"><div class="time">2023-08-02 15:35:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @James Fellows Yates I‚Äôm going to try to send a PR, but where in the repo should I place the files? Do you want me to send you only the .txt files you requested?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:36:12">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 15:36:12"><div class="time">2023-08-02 15:36:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There are instructions on the rEADME üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:36:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 15:36:25"><div class="time">2023-08-02 15:36:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://github.com/SPAAM-community/little-book-of-smiley-plots#technical-guidelines">https://github.com/SPAAM-community/little-book-of-smiley-plots#technical-guidelines</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:40:35">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 15:40:35"><div class="time">2023-08-02 15:40:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh, I think I‚Äôd cloned the wrong repo‚Ä¶ <a href="https://github.com/SPAAM-community/intro-to-ancient-metagenomics-book/">https://github.com/SPAAM-community/intro-to-ancient-metagenomics-book/</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">SPAAM-community/intro-to-ancient-metagenomics-book</a></div>
                                <div class="link-text">
                                    Introduction to Ancient Metagenomics Book
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Website</div>
                                        &lt;http://www.spaam-community.org/intro-to-ancient-metagenomics-book/&gt;
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        3
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:40:49">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 15:40:49"><div class="time">2023-08-02 15:40:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok, I‚Äôll change it and follow the correct guidelines</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:40:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 15:40:59"><div class="time">2023-08-02 15:40:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oops yes</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:41:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 15:41:05"><div class="time">2023-08-02 15:41:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Did I put the wrong link</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:42:18">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 15:42:18"><div class="time">2023-08-02 15:42:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yeah you sent me the one for the intro to metagenomics book, but it should be fine now üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:42:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 15:42:29"><div class="time">2023-08-02 15:42:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* OOOPS my bad</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:42:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 15:42:32"><div class="time">2023-08-02 15:42:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* too many book ideas üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:58:07">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 15:58:07"><div class="time">2023-08-02 15:58:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* haha üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:58:30">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 15:58:30"><div class="time">2023-08-02 15:58:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* James, my data was generated with mapdamage instead of damage profiler, is that still fine?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:06:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:06:57"><div class="time">2023-08-02 16:06:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yup!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:07:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:07:26"><div class="time">2023-08-02 16:07:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The output data should be identical</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:10:28">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 16:10:28"><div class="time">2023-08-02 16:10:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* cool, ok I think I‚Äôm almost done, I‚Äôm going to push changes shortly</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:13:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:13:49"><div class="time">2023-08-02 16:13:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* üéâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:13:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:13:53"><div class="time">2023-08-02 16:13:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Look forward for the PR!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:14:33">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 16:14:33"><div class="time">2023-08-02 16:14:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* cool‚Ä¶ in order to push changes I should fork the repo right? just realised I don‚Äôt have permissions to push directly üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:25:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:25:04"><div class="time">2023-08-02 16:25:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Either I add you a collaborator and you can push to a branch, or you can do a fork</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:25:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:25:05"><div class="time">2023-08-02 16:25:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* up to you</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:54:41">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 16:54:41"><div class="time">2023-08-02 16:54:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve forked the repo üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 16:56:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 16:56:22"><div class="time">2023-08-02 16:56:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ‚ù§Ô∏è</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 17:06:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 17:06:07"><div class="time">2023-08-02 17:06:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Guillermo Rangel could you add a short description to the end of the <code>.qmd</code> file of what causes the damage profile?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 18:21:08">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 18:21:08"><div class="time">2023-08-02 18:21:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* just did üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 18:21:46">
            
                <img src="https://secure.gravatar.com/avatar/7d2821cecf8853fe82207e64e0f4a6be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Guillermo Rangel
                     <span class="print-only user-email">(guillermo.pineros@sund.ku.dk)</span>
                </div>
                <a href="#2023-08-02 18:21:46"><div class="time">2023-08-02 18:21:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* one question though‚Ä¶. the figure in the book will only display the damage plot, or also the dna composition plot?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 18:24:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-02 18:24:19"><div class="time">2023-08-02 18:24:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Technically just the damage plot, but we can include more images below the text if you can recreate the plot!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-03 09:43:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-03 09:43:41"><div class="time">2023-08-03 09:43:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok I realise now that in your specific case we will actually need the bigger file ü§¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-03 09:43:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-03 09:43:45"><div class="time">2023-08-03 09:43:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (with all mutations)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-03 09:44:16">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-03 09:44:16"><div class="time">2023-08-03 09:44:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (the <code>misincorporation table</code> - my bad @Guillermo Rangel!)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:52:42">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-08-02 15:52:42"><div class="time">2023-08-02 15:52:42</div></a>
                <div class="msg">
                    <p>Hi all! Back again with another metaDMG question - is there an easy way to run it on repaired libraries (mine are UDG-half)? Does it account for this already?.... if not, where would I edit the algorithm by hand?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 15:53:14">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-08-02 15:53:14"><div class="time">2023-08-02 15:53:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (though I feel like UDG-half libs already follow the expected pattern somewhat, just in an extreme sense, so maybe will just lower the significance threshold I set)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-02 23:22:05">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-08-02 23:22:05"><div class="time">2023-08-02 23:22:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Actually.... after more exploration I don't think it matters and metaDMG is definitely accurately sorting ancient vs. non ancient reads</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Yi Wang
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-07 10:03:57">
            
                <img src="https://secure.gravatar.com/avatar/133f0246751c777c903e99261e094b2c.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Anna Yssel
                     <span class="print-only user-email">(ansie.yssel@uct.ac.za)</span>
                </div>
                <a href="#2023-08-07 10:03:57"><div class="time">2023-08-07 10:03:57</div></a>
                <div class="msg">
                    <p>Hi all. Can someone recommend a good review paper or a nice resource to learn about the state of the art/best practices of paleo-proteomics?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-07 10:04:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-07 10:04:53"><div class="time">2023-08-07 10:04:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://pubmed.ncbi.nlm.nih.gov/29581591/">https://pubmed.ncbi.nlm.nih.gov/29581591/</a></p>

<p>or the mega (40page!)</p>

<p><a href="https://pubs.acs.org/doi/10.1021/acs.chemrev.1c00703">https://pubs.acs.org/doi/10.1021/acs.chemrev.1c00703</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed</div>
                            
                            
                                
                                <div class="link-title"><a href="https://pubmed.ncbi.nlm.nih.gov/29581591/">A guide to ancient protein studies - PubMed</a></div>
                                <div class="link-text">
                                    Palaeoproteomics is an emerging neologism used to describe the application of mass spectrometry-based approaches to the study of ancient proteomes. As with palaeogenomics (the study of ancient DNA), it intersects evolutionary biology, archaeology and anthropology, with applications ranging from the ‚Ä¶
                                </div>
                                
                                
    
        <a href="https://pubmed.ncbi.nlm.nih.gov/29581591/">
            <img class="preview" src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://pubmed.ncbi.nlm.nih.gov/29581591/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-07 13:04:10">
            
                <img src="https://secure.gravatar.com/avatar/133f0246751c777c903e99261e094b2c.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0021-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anna Yssel
                     <span class="print-only user-email">(ansie.yssel@uct.ac.za)</span>
                </div>
                <a href="#2023-08-07 13:04:10"><div class="time">2023-08-07 13:04:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* much appreciated, thanks</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-07 18:05:58">
            
                <img src="https://avatars.slack-edge.com/2022-10-19/4245228212949_8faeeb72ee0d362b222b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Miriam Bravo
                     <span class="print-only user-email">(bravolomiriam@gmail.com)</span>
                </div>
                <a href="#2023-08-07 18:05:58"><div class="time">2023-08-07 18:05:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* this one too can help: <a href="https://www.science.org/doi/epdf/10.1126/sciadv.abb9314">https://www.science.org/doi/epdf/10.1126/sciadv.abb9314</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Magdalena Haller-Caskie, Anna Yssel
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 06:43:29">
            
                <img src="https://avatars.slack-edge.com/2022-10-27/4269387323127_52f8b64cd696f1c66ff3_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Magdalena Haller-Caskie
                     <span class="print-only user-email">(m.haller@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2023-08-08 06:43:29"><div class="time">2023-08-08 06:43:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Unfortunately, the "guide to ancient protein studies" does not allow access for me (my institution). Does anyone have access and could share it with me?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 08:08:38">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-08 08:08:38"><div class="time">2023-08-08 08:08:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You can try</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 08:08:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-08 08:08:40"><div class="time">2023-08-08 08:08:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05LKDCN7BQ/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05LKDCN7BQ/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05LKDCN7BQ-98243a4f7c/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="50" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 08:08:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-08 08:08:53"><div class="time">2023-08-08 08:08:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Press the Univ. Turin button</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 08:09:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-08 08:09:07"><div class="time">2023-08-08 08:09:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And then the bottom üòâ</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05LBGJ8KTR/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05LBGJ8KTR/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05LBGJ8KTR-e5dd2788dd/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="84" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 08:09:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-08 08:09:29"><div class="time">2023-08-08 08:09:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But in the mean time (on behalf of @Christina Warinner)</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05LUG4SKTN/s41559-018-0510-x.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">s41559-018-0510-x.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05LUG4SKTN/s41559-018-0510-x.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05LUG4SKTN-eaba98d264/s41559-018-0510-x_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="909"
                 height="1208" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 08:24:29">
            
                <img src="https://avatars.slack-edge.com/2022-10-27/4269387323127_52f8b64cd696f1c66ff3_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Magdalena Haller-Caskie
                     <span class="print-only user-email">(m.haller@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2023-08-08 08:24:29"><div class="time">2023-08-08 08:24:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sneaky üòâ @James Fellows Yates
Thank you so much for forwarding the article! this helps a lot @Christina Warinner</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-08 10:43:38">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2023-08-08 10:43:38"><div class="time">2023-08-08 10:43:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This one is the most recent:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05MGGASZB2/acs.chemrev.1c00703.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">acs.chemrev.1c00703.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05MGGASZB2/acs.chemrev.1c00703.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05MGGASZB2-bcc215efa9/acs.chemrev.1c00703_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="928"
                 height="1224" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üôå Magdalena Haller-Caskie
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-09 20:58:25">
            
                <img src="https://secure.gravatar.com/avatar/52b584ad87d77deddf6454dea83718d1.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0009-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Charlotte Shore
                     <span class="print-only user-email">(cdshore@uncg.edu)</span>
                </div>
                <a href="#2023-08-09 20:58:25"><div class="time">2023-08-09 20:58:25</div></a>
                <div class="msg">
                    <p>Hi everyone. I was curious if anyone had ever tried using a kit such as <a href="https://www.qiagen.com/us/products/human-id-and-forensics/nextgeneration-sequencing/generead-size-selection-kit">this</a> to increase the proportion of endogenous DNA to contaminant DNA in a sample? My thinking was that the sample could be run through the column in the supplied buffer, binding fragments above ~150-300 and letting the shorter endogenous DNA through, which could then be extracted from the buffer using something that recovers shorter fragments.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Qiagen.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.qiagen.com/us/products/human-id-and-forensics/nextgeneration-sequencing/generead-size-selection-kit">GeneRead Size Selection Kit</a></div>
                                <div class="link-text">
                                    sequencing
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://www.qiagen.com/us/products/human-id-and-forensics/nextgeneration-sequencing/generead-size-selection-kit</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-10 12:15:59">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-10 12:15:59"><div class="time">2023-08-10 12:15:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think the group of Matthias Meyer at MPI EVA has been using electrophoresis gels for size selection. For this, they ran the library samples on a gel and then cut out the part of the gel they want to keep. However, I am not sure if there is a particular protocol for this that they published.
You should just keep in mind that you loose a lot of DNA in these processes so you need plenty to start from.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 20:55:31">
            
                <img src="https://avatars.slack-edge.com/2022-07-19/3842655941616_52cc6550245d56bdc4b3_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Jaime Zolik
                     <span class="print-only user-email">(zolik006@umn.edu)</span>
                </div>
                <a href="#2023-08-12 20:55:31"><div class="time">2023-08-12 20:55:31</div></a>
                <div class="msg">
                    <p>Hello, I am not sure if this is a question for this channel or for nf-core/eager slack but I had a question about AdapterRemoval, I was wondering when it was appropriate to use <code>--preserve5p</code> ? I am working on shotgun dsDNA library-prep sequences. I am trying to select the parameters for AdapterRemoval and am having trouble figuring out what kind of parameters I should think about with my sequences. If anyone has any suggestions on how I can approach this I would greatly appreciate it. Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 21:23:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-12 21:23:48"><div class="time">2023-08-12 21:23:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05MXJSQ02V/tenor_gif770440044700465630.gif?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">tenor_gif770440044700465630.gif</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05MXJSQ02V/tenor_gif770440044700465630.gif?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05MXJSQ02V-c6de8d7311/tenor_gif770440044700465630_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="220"
                 height="124" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 21:26:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-12 21:26:20"><div class="time">2023-08-12 21:26:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The reason why this was added was in the case of very low coverage data which had a very high duplication factor.</p>

<p>The problem was when trimming was carried out, in some cases of a set of duplicate reads, when doing quality trimming some would be trimmed at 5p but others not. Which led to the deupping tool too incorrect (baby waking up, continue Kater)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 21:32:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-12 21:32:25"><div class="time">2023-08-12 21:32:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ...Incorrectly <em>not</em> dedup some of the duplicates because there 5p was trimmed more than others</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 21:33:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-12 21:33:51"><div class="time">2023-08-12 21:33:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So essentially you left in duplicates, which artificially inflated your coverage and cause false positives in terms of confident variant calling</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 21:34:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-12 21:34:15"><div class="time">2023-08-12 21:34:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So: if you have low coverage data, with a high duplication rate, then it maybe recommended</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-12 22:20:52">
            
                <img src="https://avatars.slack-edge.com/2022-07-19/3842655941616_52cc6550245d56bdc4b3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jaime Zolik
                     <span class="print-only user-email">(zolik006@umn.edu)</span>
                </div>
                <a href="#2023-08-12 22:20:52"><div class="time">2023-08-12 22:20:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ohh that makes sense! Thank you so much for your help and for connecting me to the correct eager channel!</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-17 05:41:40">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2023-08-17 05:41:40"><div class="time">2023-08-17 05:41:40</div></a>
                <div class="msg">
                    <p>Hey all! Lab related question. I'm using a single stranded protocol (Gansauge &amp; Meyer, 2019), and for the ligation of the first adapter they use PEG-8000. We've found that we can only order it in a kit and keep ending up with a lot of other unused reagents. I was wondering if anyone has ever compared PEG-6000 or PEG-4000 vs PEG-8000 for ancient libraries? I used some PEG-4000 for captured samples and that seemed the have worked fine, but trying to figure out what to do for some other samples for shotgun sequencing.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-17 13:22:28">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-17 13:22:28"><div class="time">2023-08-17 13:22:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I asked in our departmant chat and got the following response
<code>Hey James, related to your PEG question - Marie and Matthias have heavily optimised the ssDNA library prep, so I am sure they tested various PEG concentrations (especially since the higher molecular ones are super viscous and pain in the ass to work with). Matthias is currently away until some time next week - but he would be the best person to ask and he's quite responsive to these types of technical questions üôÇ So my recommendation is that the person just shoots an e-mail to him and will get the best advice üòâ</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-17 13:30:58">
            
                <img src="https://avatars.slack-edge.com/2022-03-03/3181209952006_c96e6bba200395e444e2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Giada Ferrari
                     <span class="print-only user-email">(gferrari.rbge@gmail.com)</span>
                </div>
                <a href="#2023-08-17 13:30:58"><div class="time">2023-08-17 13:30:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I usually order PEG8000 as a powder and prepare my own aliquots, in case that is an option for you</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üëç:skin_tone_3: Meriam van Os
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-18 00:11:57">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2023-08-18 00:11:57"><div class="time">2023-08-18 00:11:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks both! I will send Matthias an email, and look into the powder üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 11:40:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-22 11:40:13"><div class="time">2023-08-22 11:40:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05NG5S8B8X/screenshot_20230822-113937-714.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot_20230822-113937-714.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05NG5S8B8X/screenshot_20230822-113937-714.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05NG5S8B8X-7a1a6659df/screenshot_20230822-113937-714_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="360" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 11:40:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-22 11:40:22"><div class="time">2023-08-22 11:40:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Another response!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 11:40:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-22 11:40:41"><div class="time">2023-08-22 11:40:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://www.jenabioscience.com/crystallography-cryo-em/optimization/chemical-environment/crystallization-stock-solutions/polymers/css-256-polyethyleneglycol-8000-50-w-v">https://www.jenabioscience.com/crystallography-cryo-em/optimization/chemical-environment/crystallization-stock-solutions/polymers/css-256-polyethyleneglycol-8000-50-w-v</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Jena Bioscience</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.jenabioscience.com/crystallography-cryo-em/optimization/chemical-environment/crystallization-stock-solutions/polymers/css-256-polyethyleneglycol-8000-50-w-v">Polyethyleneglycol 8000 - 50 % w/v, Crystallization Stock Solutions: Polymers - Jena Bioscience</a></div>
                                <div class="link-text">
                                    CSS-256 / Optimization Chemical Environment Crystallization Stock Solutions Polymers
                                </div>
                                
                                
    
        <a href="https://www.jenabioscience.com/crystallography-cryo-em/optimization/chemical-environment/crystallization-stock-solutions/polymers/css-256-polyethyleneglycol-8000-50-w-v">
            <img class="preview" src="https://www.jenabioscience.com/styles/images/logo-color.png"
                width="963"
                 height="396" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.jenabioscience.com/crystallography-cryo-em/optimization/chemical-environment/crystallization-stock-solutions/polymers/css-256-polyethyleneglycol-8000-50-w-v</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Meriam van Os
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 09:44:52">
            
                <img src="https://avatars.slack-edge.com/2021-12-22/2872808514018_3c9b0a94d8e687729e4f_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nihan D Dagtas
                     <span class="print-only user-email">(dagtas.nd@gmail.com)</span>
                </div>
                <a href="#2023-08-22 09:44:52"><div class="time">2023-08-22 09:44:52</div></a>
                <div class="msg">
                    <p>Hi all! We have a similar issue to Meriam's. We use the Roche extender columns for the Rohland etal 2018 protocol, however most of the buffers from the Viral Kit get piled up, unused. Has anyone found an alternative to buying the whole kit? As far as I know the extenders are not sold separately.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 13:00:15">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2023-08-22 13:00:15"><div class="time">2023-08-22 13:00:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Exactly the same for us, no solution but also keen to hear other ideas</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü•≤ Nihan D Dagtas
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 16:40:07">
            
                <img src="https://avatars.slack-edge.com/2022-06-26/3715307052726_f1fe763ae7c0736660f8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Hannah Moots
                     <span class="print-only user-email">(hmoots@gmail.com)</span>
                </div>
                <a href="#2023-08-22 16:40:07"><div class="time">2023-08-22 16:40:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Same here! I haven‚Äôt found the columns sold separately </p>
                    
                    
                    
                        <div class="message-reaction">
                        ü•≤ Nihan D Dagtas
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 21:04:07">
            
                <img src="https://avatars.slack-edge.com/2023-08-23/5781897083045_3159c2c3a93c996fb348_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anna Chagas
                     <span class="print-only user-email">(anna.jaime-chagas@uni-konstanz.de)</span>
                </div>
                <a href="#2023-08-22 21:04:07"><div class="time">2023-08-22 21:04:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve seen a protocol that was published recently for soil and sediment samples and there they use these EconoSpin¬Æ DNA Only Maxi Spin Columns. <a href="https://epochlifescience.com/products/econospin%C2%AE-dna-only-maxi-spin-column?variant=44335099412771">https://epochlifescience.com/products/econospin%C2%AE-dna-only-maxi-spin-column?variant=44335099412771</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Epoch Life Science</div>
                            
                            
                                
                                <div class="link-title"><a href="https://epochlifescience.com/products/econospin%C2%AE-dna-only-maxi-spin-column?variant=44335099412771">EconoSpin¬Æ DNA Only Maxi Spin Column</a></div>
                                <div class="link-text">
                                    [shortdesc]EconoSpin¬Æ DNA Only Maxi Spin Column is used DNA extraction from various sources. It fits into a standard 15 ml centrifuge tube. [/shortdesc] [els_overview]EconoSpin¬Æ DNA Only Maxi Spin Column is a silica membrane based low cost substitute for DEAE columns. It can be used for DNA extraction from various sour
                                </div>
                                
                                
    
        <a href="https://epochlifescience.com/products/econospin%C2%AE-dna-only-maxi-spin-column?variant=44335099412771">
            <img class="preview" src="http://epochlifescience.com/cdn/shop/products/silimaxi_grande.jpg?v=1677282759"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://epochlifescience.com/products/econospin%C2%AE-dna-only-maxi-spin-column?variant=44335099412771</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 21:04:36">
            
                <img src="https://avatars.slack-edge.com/2023-08-23/5781897083045_3159c2c3a93c996fb348_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anna Chagas
                     <span class="print-only user-email">(anna.jaime-chagas@uni-konstanz.de)</span>
                </div>
                <a href="#2023-08-22 21:04:36"><div class="time">2023-08-22 21:04:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And this is the protocol that was published: <a href="https://www.protocols.io/view/inhibitor-free-dna-extraction-from-soil-and-sedime-bp2l6957zlqe/v1">https://www.protocols.io/view/inhibitor-free-dna-extraction-from-soil-and-sedime-bp2l6957zlqe/v1</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">protocols.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.protocols.io/view/inhibitor-free-dna-extraction-from-soil-and-sedime-bp2l6957zlqe/v1">Inhibitor-free DNA extraction from soil and sediment samples</a></div>
                                <div class="link-text">
                                    This protocol describes how to extract inhibitor-free DNA from soil and sediment samples. of soil or up to of sediment can be processed in one extraction, but there is also...
                                </div>
                                
                                
    
        <a href="https://www.protocols.io/view/inhibitor-free-dna-extraction-from-soil-and-sedime-bp2l6957zlqe/v1">
            <img class="preview" src="https://s3.amazonaws.com/protocols-files/files/mfkhbnf2f.jpg"
                width="753"
                 height="753" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.protocols.io/view/inhibitor-free-dna-extraction-from-soil-and-sedime-bp2l6957zlqe/v1</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 05:55:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 05:55:35"><div class="time">2023-08-23 05:55:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't think we use the viral Kits either:</p>

<p><a href="https://www.protocols.io/view/half-udg-treated-double-stranded-ancient-dna-libra-x54v9j87pg3e/v1/materials">https://www.protocols.io/view/half-udg-treated-double-stranded-ancient-dna-libra-x54v9j87pg3e/v1/materials</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">protocols.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.protocols.io/view/half-udg-treated-double-stranded-ancient-dna-libra-x54v9j87pg3e/v1/materials">Half-UDG treated double-stranded ancient DNA library preparation for Illumina sequencing protocol materials</a></div>
                                <div class="link-text">
                                    Protocol for the preparation of double-stranded genomic libraries for Illumina sequencing, optimised for ancient DNA (aDNA) with partial USER enzyme treatment. This protocol gen...
                                </div>
                                
                                
    
        <a href="https://www.protocols.io/view/half-udg-treated-double-stranded-ancient-dna-libra-x54v9j87pg3e/v1/materials">
            <img class="preview" src="https://s3.amazonaws.com/protocols-files/public/e0ea50c14935d4d2e226abda96f2c60392146fbea15861a37ccc4fae4b68cd89/b3u24tke.jpg"
                width="645"
                 height="645" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.protocols.io/view/half-udg-treated-double-stranded-ancient-dna-libra-x54v9j87pg3e/v1/materials</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 05:55:56">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 05:55:56"><div class="time">2023-08-23 05:55:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2/materials">https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2/materials</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">protocols.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2/materials">Illumina double-stranded DNA dual indexing for ancient DNA protocol materials</a></div>
                                <div class="link-text">
                                    This protocol converts partially completed double-stranded DNA libraries e.g. from: Non-UDG treated double-stranded ancient DNA library preparation for Illumina sequencing (dx....
                                </div>
                                
                                
    
        <a href="https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2/materials">
            <img class="preview" src="https://s3.amazonaws.com/protocols-files/files/dtqk4tje.jpg"
                width="728"
                 height="728" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.protocols.io/view/illumina-double-stranded-dna-dual-indexing-for-anc-4r3l287x3l1y/v2/materials</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 10:09:05">
            
                <img src="https://avatars.slack-edge.com/2021-12-22/2872808514018_3c9b0a94d8e687729e4f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nihan D Dagtas
                     <span class="print-only user-email">(dagtas.nd@gmail.com)</span>
                </div>
                <a href="#2023-08-23 10:09:05"><div class="time">2023-08-23 10:09:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you @Anna Chagas I'll check it out! @James Fellows Yates it's the DNA extraction protocol, forgot to mention üòÅ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Anna Chagas
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 10:18:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 10:18:22"><div class="time">2023-08-23 10:18:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah ok... sorry, I thought Rohland 2018 was the UDG half ü§¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-22 22:32:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-08-22 22:32:32"><div class="time">2023-08-22 22:32:32</div></a>
                <div class="msg">
                    <p>Hi all! We‚Äôve just switched clusters and I‚Äôm trying to run MALT/HOPS and troubleshoot. Seems like on some file sets it‚Äôs working fine and sometimes it throws <code>INFO: java.util.concurrent.ExecutionException: java.lang.NullPointerException</code> ‚Äî this may not be a MALT/HOPS issue per se but if anyone has seen it before I‚Äôd be very grateful!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 03:11:02">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-08-23 03:11:02"><div class="time">2023-08-23 03:11:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Does this pop up when you‚Äôre running malt run or build?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 05:50:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 05:50:52"><div class="time">2023-08-23 05:50:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You'll need to provide more information around the error too - that error is the most generic of all Java errors possible...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 05:57:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-08-23 05:57:51"><div class="time">2023-08-23 05:57:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah, heck. Okay, it seems like at least in my most recent run, the first error that seems to lead to a cascade of disasters is a ‚ÄúINFO: Danger empty keys in File‚Äù at the MaltExtract step. My problem seems very similar to this one. <a href="https://github.com/rhuebler/MaltExtract/issues/2">https://github.com/rhuebler/MaltExtract/issues/2</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 05:58:18">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-08-23 05:58:18"><div class="time">2023-08-23 05:58:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So I suppose I could just hop on the github but was wondering if anyone else has dealt with this!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 06:52:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 06:52:20"><div class="time">2023-08-23 06:52:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can you open your RMA6 file in Megan?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 06:52:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 06:52:39"><div class="time">2023-08-23 06:52:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Empty keys in file would suggest to me it's possibly a broken rma6</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 06:53:27">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-23 06:53:27"><div class="time">2023-08-23 06:53:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But otherwise given the repo is ignored it seems, you should email Alexander Herbig</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 16:31:43">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-08-23 16:31:43"><div class="time">2023-08-23 16:31:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ooh, will check up on the RMA6 files and email Alexander if need be. Thanks so much James!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-23 22:58:17">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-08-23 22:58:17"><div class="time">2023-08-23 22:58:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Aha, something has indeed gone awry with the RMA6. MALT output says out of 32 M reads, ~900K were aligned; MEGAN shows the RMA6 file as having 900K reads but 0 assignments. Thanks James, will send this info to Alexander and hope for the best!</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-24 06:29:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-24 06:29:35"><div class="time">2023-08-24 06:29:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Glad you at least solved the first bit!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-30 16:43:10">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-08-30 16:43:10"><div class="time">2023-08-30 16:43:10</div></a>
                <div class="msg">
                    <p>Is anyone interested in reviewing this paper for PeerJ? You should be able to sign up using the link below.</p>

<h3>Article Id 88906 &lt;<a href="https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpeerj.com%2Fsubmissions%2F88906%2F&amp;data=05%7C01%7Csvw5689%40psu.edu%7Cec3de90afdf84024dafd08dba961df11%7C7cf48d453ddb4389a9c1c115526eb52e%7C0%7C0%7C638290010050724252%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=2YS91HQ1jtRAJN0fJllTkF7R%2FwFkdDvIL2Am6Fw26PM%3D&amp;reserved=0">https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpeerj.com%2Fsubmissions%2F88906%2F&amp;data=05%7C01%7Csvw5689%40psu.edu%7Cec3de90afdf84024dafd08dba961df11%7C7cf48d453ddb4389a9c1c115526eb52e%7C0%7C0%7C638290010050724252%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=2YS91HQ1jtRAJN0fJllTkF7R%2FwFkdDvIL2Am6Fw26PM%3D&amp;reserved=0</a>&gt;</h3>

<h3>Title</h3>

<p>The relationship between leptin and periodontitis: a literature review</p>

<h3>Abstract</h3>

<p>Leptin is a peptide hormone that regulates energy balance, immune inflammatory response, and bone metabolism. On the other hand, periodontitis is a local inflammatory disease that progressively weakens periodontal support tissue, eventually leading to tooth loss. Several studies have demonstrated a relationship between leptin and periodontitis. This article reviews the existing literature and discusses leptin‚Äôs basic characteristics, its relationship with periodontitis, and its effects on periodontal tissue metabolism.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-30 21:05:19">
            
                <img src="https://avatars.slack-edge.com/2023-08-30/5839475950673_431bfaf0a3370226656e_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nev√© Baker
                     <span class="print-only user-email">(dnbaker@ucsc.edu)</span>
                </div>
                <a href="#2023-08-30 21:05:19"><div class="time">2023-08-30 21:05:19</div></a>
                <div class="msg">
                    <p>Hi all, for shotgun sequencing sedaDNA, how do you decide how many reads to target per sample? I‚Äôm trying to put together a budget and there seems to be a big range in the literature.
Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-30 21:33:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-30 21:33:35"><div class="time">2023-08-30 21:33:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Depends on that you want to do!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-30 21:33:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-30 21:33:39"><div class="time">2023-08-30 21:33:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* üò¨</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-30 21:34:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-30 21:34:01"><div class="time">2023-08-30 21:34:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can you provide more info on the goal of the project?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-30 22:22:21">
            
                <img src="https://avatars.slack-edge.com/2023-08-30/5839475950673_431bfaf0a3370226656e_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nev√© Baker
                     <span class="print-only user-email">(dnbaker@ucsc.edu)</span>
                </div>
                <a href="#2023-08-30 22:22:21"><div class="time">2023-08-30 22:22:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ha it always does right? The goal of the project is to understand timing of human arrival and associated environmental change through the late pleistocene and early holocene in currently submerged subarctic coastal sites. We will mainly be metabarcoding core samples to look at plant and animal community change as well as signatures of sea level rise (i.e. shift from terrestrial plants to algae), but my collaborators are interested in doing some shotgun sequencing as well and we‚Äôre just trying to figure out how much sequencing to budget for (approximately)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 11:45:44">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-31 11:45:44"><div class="time">2023-08-31 11:45:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see. I meant more technically - what do you analytically want to do? Just taxonomic profiling? <em>de novo</em> assembly? functional profiling?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 11:47:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-31 11:47:00"><div class="time">2023-08-31 11:47:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But maybe we can crowd source some response (and if you're willing!) maybe yo ucould write a short blog post? @Kevin Daly did this for one of his questions for pathogen screening.</p>

<p><a href="https://www.spaam-community.org/blog/2023/04/07/blog-numreads/">https://www.spaam-community.org/blog/2023/04/07/blog-numreads/</a></p>

<p>A companion piece for sedaDNA would be really nice üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 11:51:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-31 11:51:10"><div class="time">2023-08-31 11:51:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To crowd source ideas, off the top of my head:</p>

<p>@Pete Heintzman @Antonio Fernandez-Guerra @Anan Ibrahim @Kadir Toykan √ñzdoƒüan @Linda Armbrecht @Barbara @Benjamin Vernot @Chenyu Jin (Amend) @Eric Capo @Jamie Wood @Kevin Nota @Laura Epp @Merlin Szymanski @Mike Martin @valentinav</p>

<p>Maybe they could chime in?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 14:12:50">
            
                <img src="https://avatars.slack-edge.com/2022-10-08/4217786516640_cfad32b58f07e023019b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kadir Toykan √ñzdoƒüan
                     <span class="print-only user-email">(k.t.ozdogan@uu.nl)</span>
                </div>
                <a href="#2023-08-31 14:12:50"><div class="time">2023-08-31 14:12:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Just to give one individual example, don't know if it helps.
We tried 8-16-32 million reads per sample for some samples from the same site for screening. What we noticed was that we lost a lot of the diversity with 8 million reads but there was not much difference in that sense between 16 and 32.
But it is just one example and just for screening to see what is there, not for any deep analysis.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 14:13:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-31 14:13:42"><div class="time">2023-08-31 14:13:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What sort of screening/analysis?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 14:14:36">
            
                <img src="https://avatars.slack-edge.com/2022-10-08/4217786516640_cfad32b58f07e023019b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kadir Toykan √ñzdoƒüan
                     <span class="print-only user-email">(k.t.ozdogan@uu.nl)</span>
                </div>
                <a href="#2023-08-31 14:14:36"><div class="time">2023-08-31 14:14:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To see which types of plants and animals are present in family level</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 14:16:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-08-31 14:16:01"><div class="time">2023-08-31 14:16:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (sorry I mean which technique: taxonomic profiling? Mapping? De novo?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-08-31 14:21:20">
            
                <img src="https://avatars.slack-edge.com/2022-10-08/4217786516640_cfad32b58f07e023019b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kadir Toykan √ñzdoƒüan
                     <span class="print-only user-email">(k.t.ozdogan@uu.nl)</span>
                </div>
                <a href="#2023-08-31 14:21:20"><div class="time">2023-08-31 14:21:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* taxonomic profiling by using bowtie2 for alignments</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-01 10:00:41">
            
                <img src="https://avatars.slack-edge.com/2023-10-25/6085479900966_31abc2b5900af9ac7d4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Nota
                     <span class="print-only user-email">(kevin_nota@eva.mpg.de)</span>
                </div>
                <a href="#2023-09-01 10:00:41"><div class="time">2023-09-01 10:00:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I agree with Kadir. It really depends on the type of sample, and how rich your sample is. When we do screening for the preservation of DNA, and have a superficial idea of which plant species were present. We tend to aim between 2-5 million reads and do a mapping approach with an LCA. That usually gives enough data to get some idea about the plants based on nuclear DNA and to see if there is DNA preservation. If we want to get a bit more resolution and have more chance you will pick up an animal signal (in archaeological sediments or lakes), we aim for at least 20-25 million again with a mapping approach with an LCA. Again it will really depend on your samples and how much you will get. In the end, it is a question of which sequencing platform you will use NovaSeq is cheaper per read than HiSeq, and the number of samples you have.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-01 19:51:49">
            
                <img src="https://avatars.slack-edge.com/2023-08-30/5839475950673_431bfaf0a3370226656e_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nev√© Baker
                     <span class="print-only user-email">(dnbaker@ucsc.edu)</span>
                </div>
                <a href="#2023-09-01 19:51:49"><div class="time">2023-09-01 19:51:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for these responses! This is super helpful. We are planning to do taxonomic profiling, but I think my collaborators are not super clear on what specifically they‚Äôre hoping to learn from the shotgun data (this is a sedaDNA add on to a larger project), so this info should help us nail down the details. Thanks so much!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-02 05:35:29">
            
                <img src="https://avatars.slack-edge.com/2022-06-06/3615121700871_7e93c021192bb6956fb5_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Barbara
                     <span class="print-only user-email">(bbmoguel@gmail.com)</span>
                </div>
                <a href="#2023-09-02 05:35:29"><div class="time">2023-09-02 05:35:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also as Kevin mention the prices for secuencing are important, depens of how many samples do you have and how deep do you want to go in. But for taxonomic profiling 5 million reads, cold be enough! Good luck!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-11 12:45:27">
            
                <img src="https://avatars.slack-edge.com/2021-01-13/1617825225959_f2b1daff8c185a5e2b87_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Andrea quagliariello
                     <span class="print-only user-email">(quagliariello.andrea@gmail.com)</span>
                </div>
                <a href="#2023-09-11 12:45:27"><div class="time">2023-09-11 12:45:27</div></a>
                <div class="msg">
                    <p>Hello dear community! Hope you are all fine. I have one question about the analysis of coprolite (very old one), and I hope someone here has more experience than me with these kind of data. Basically, there are a couple of coprolites which have been sequenced using dsDNA library-prep approach. I was wondering if there was a bit of human DNA inside thus I applied both CoproID and nf-core/eager on the same samples to compare the results. I get very discordant results between the two but I am not understanding exactly why this is happening.
I share you here an example from one sample:
From CoproID I obtain a good prediction as being human. I attach both the summary picture than the deamination profile obtained.
Then When I run eager I obtained a completely different result with almost no deamination at all in the same sample.  I have used the same reference for both (GRCh37) and default params for both analysis. Does anyone can help me understanding what is going on there? What am I missing? Thank you people!</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05RSB0A38B/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05RSB0A38B/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05RSB0A38B-9961243291/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="214" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05RSE43C4A/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05RSE43C4A/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05RSE43C4A-03ed9ee8f9/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="219" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F05SFDYB5L0/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F05SFDYB5L0/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F05SFDYB5L0-84d5b13ac1/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="245" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-11 16:00:04">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-11 16:00:04"><div class="time">2023-09-11 16:00:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* coproID damage plots are only from the damage filtered reads, so that might explain why you get this difference.
How many reads do you have aligned to Humans with coproID ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-11 16:57:15">
            
                <img src="https://avatars.slack-edge.com/2021-01-13/1617825225959_f2b1daff8c185a5e2b87_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Andrea quagliariello
                     <span class="print-only user-email">(quagliariello.andrea@gmail.com)</span>
                </div>
                <a href="#2023-09-11 16:57:15"><div class="time">2023-09-11 16:57:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello Maxime, thank you for the explaination. I get 72303 aligned on human, 1870 aligned on dog and 2850 ancient human, 87 ancient on dog.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-11 16:58:08">
            
                <img src="https://avatars.slack-edge.com/2021-01-13/1617825225959_f2b1daff8c185a5e2b87_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Andrea quagliariello
                     <span class="print-only user-email">(quagliariello.andrea@gmail.com)</span>
                </div>
                <a href="#2023-09-11 16:58:08"><div class="time">2023-09-11 16:58:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So you think the problem is that on eager the damage plot is done on all human not just the ancient part, with respect to what CoproID does? is it right?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-12 12:07:35">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-09-12 12:07:35"><div class="time">2023-09-12 12:07:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Given what Maxime says, that makes sense to me. I'm not really sure why you would do a damage plot only showing damaged reads, though?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-12 12:08:13">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-09-12 12:08:13"><div class="time">2023-09-12 12:08:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Seems like that runs the risk of masking quite a bit of contamination?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-12 16:18:37">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-12 16:18:37"><div class="time">2023-09-12 16:18:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* that's what i suspect @Andrea quagliariello</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-12 16:19:13">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-12 16:19:13"><div class="time">2023-09-12 16:19:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* a damage plot for a qualitative and visual confirmation of the damage</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Benjamin Vernot
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-14 16:51:43">
            
                <img src="https://avatars.slack-edge.com/2021-01-13/1617825225959_f2b1daff8c185a5e2b87_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Andrea quagliariello
                     <span class="print-only user-email">(quagliariello.andrea@gmail.com)</span>
                </div>
                <a href="#2023-09-14 16:51:43"><div class="time">2023-09-14 16:51:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok Thank you @Maxime Borry. So the damage plot is draw on the output of pmdtools? because I do not find the same number of ancient reads in the bam of pmd output with respect to the number of ancient aligned presented in the final csv (coproID_result).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 15:52:00">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-18 15:52:00"><div class="time">2023-09-18 15:52:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So to summarize: coproID is just there to help you pinpoint the source/host of your coprolite.
The rest of the classic aDNA authentification steps have to be done with your classic aDNA pipeline (nf-core/eager, or else)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 11:45:02">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-09-18 11:45:02"><div class="time">2023-09-18 11:45:02</div></a>
                <div class="msg">
                    <p>Hello everyone, does anyone have a nice package or script to make circos plots to compare assemblies? They don‚Äôt seem easy! Any suggestion appreciated! Cheers!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 12:35:53">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2023-09-18 12:35:53"><div class="time">2023-09-18 12:35:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Maria,this one is not too bad
<a href="https://github.com/metagenlab/mummer2circos">https://github.com/metagenlab/mummer2circos</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">metagenlab/mummer2circos</a></div>
                                <div class="link-text">
                                    Circular bacterial genome plots based on BLAST or NUCMER/PROMER alignments
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        79
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Python
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Maria Lopopolo
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 15:11:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-09-18 15:11:20"><div class="time">2023-09-18 15:11:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've not used them myself but if I you are a fan of R, I've heard of:</p>

<p><a href="https://r-graph-gallery.com/224-basic-circular-plot.html">https://r-graph-gallery.com/224-basic-circular-plot.html</a>
<a href="https://cran.r-project.org/web/packages/BioCircos/vignettes/BioCircos.html">https://cran.r-project.org/web/packages/BioCircos/vignettes/BioCircos.html</a>
<a href="https://cran.r-project.org/web/packages/RCircos/vignettes/Using_RCircos.pdf">https://cran.r-project.org/web/packages/RCircos/vignettes/Using_RCircos.pdf</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">r-graph-gallery.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://r-graph-gallery.com/224-basic-circular-plot.html">Introduction to the circlize package</a></div>
                                <div class="link-text">
                                    This post is an introduction to the circlize package: the ultimate way to build circular charts with R. It shows how to initialize a circular section and fill it with a scatterplot.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://r-graph-gallery.com/224-basic-circular-plot.html</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 15:11:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-09-18 15:11:42"><div class="time">2023-09-18 15:11:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* but AFAIkj, CIRCOS is never easy)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 16:17:04">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-18 16:17:04"><div class="time">2023-09-18 16:17:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also: <a href="https://moshi4.github.io/pyCirclize/">https://moshi4.github.io/pyCirclize/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">moshi4.github.io</div>
                            
                            
                                
                                <div class="link-title"><a href="https://moshi4.github.io/pyCirclize/">pyCirclize</a></div>
                                <div class="link-text">
                                    Circular visualization in Python
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://moshi4.github.io/pyCirclize/</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 16:31:21">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-09-18 16:31:21"><div class="time">2023-09-18 16:31:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much for the suggestions! I will have a look at all of them!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 16:32:24">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-09-18 16:32:24"><div class="time">2023-09-18 16:32:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* also do you do recommend annotating them all with the same tool beforehand?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 16:40:43">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-18 16:40:43"><div class="time">2023-09-18 16:40:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes ! Different tools will give you different results, so be careful üòâ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 16:44:10">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-18 16:44:10"><div class="time">2023-09-18 16:44:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also, if you have only 2 genomes to compare, I would definitely recommend dotplots (this one is nice and interactive: <a href="https://dgenies.toulouse.inra.fr/">https://dgenies.toulouse.inra.fr/</a>)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 17:10:03">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-09-18 17:10:03"><div class="time">2023-09-18 17:10:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have like 5 but also can do a  closer inspection with 2.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 20:49:49">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-09-18 20:49:49"><div class="time">2023-09-18 20:49:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and if they're too divergent for circos like plot, you can give a try to pangenome graphs (that you can build with a tool such as panaroo)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-18 21:34:58">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2023-09-18 21:34:58"><div class="time">2023-09-18 21:34:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh! I didn't think about that! Thanks a lot üôè yes it's a mixture of closely related and more divergent so maybe I'll try the graph! ‚ú®</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-09-28 17:08:57">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-09-28 17:08:57"><div class="time">2023-09-28 17:08:57</div></a>
                <div class="msg">
                    <p>üéâ New from the SPAAM Blog team (@Ele and me)!
We really mean <strong>no stupid questions</strong> in this channel but we get it, sometimes you‚Äôre just too nervous to ask! Enter the ‚Äú<a href="https://docs.google.com/forms/d/e/1FAIpQLSeW6HM2Ehr7yBkvQBYvDTYmxl-HFFjqbJTyAwnPrLnlOPP5ZA/viewform?usp=sf_link">I‚Äôm too afraid to ask</a>‚Äù form where you can post your anonymous questions! Ele and I will check in periodically and share submitted questions here‚Ä¶ and maybe make a blog post out of the answers. You can also use the form to anonymously submit blog post ideas. It is pinned at the top of the channel whenever you need it!
ü•≥ Have fun!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üíú Maria Lopopolo, Kelly Blevins, Biancamaria Bonucci, James Fellows Yates, Miriam Bravo, Alina Hiss, Pooja Swali
                        </div>
                    
                        <div class="message-reaction">
                        üíö Ele, Kelly Blevins, James Fellows Yates, Alina Hiss
                        </div>
                    
                        <div class="message-reaction">
                        :spaam: Miriam Bravo, Jasmin Frangenberg, Dawn lewis
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-03 22:11:57">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-10-03 22:11:57"><div class="time">2023-10-03 22:11:57</div></a>
                <div class="msg">
                    <p>Hi everyone! Has anyone ever made a Kaiju database including extra NCBI ID's for specific taxa (additionally to the "nr" db which one can specify)? Thanks!! üôè</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:03:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-04 11:03:47"><div class="time">2023-10-04 11:03:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I may know someone who has... but I'm waitnig for their answer. I seem to remember custom Kaiju DBs were complicated</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Aleksandra Laura Pach
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:05:10">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-10-04 11:05:10"><div class="time">2023-10-04 11:05:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Omg thanks! Waiting excitingly! I've been pulling my hair over this</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:05:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-04 11:05:21"><div class="time">2023-10-04 11:05:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't promise anything though üò¨</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:06:12">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-10-04 11:06:12"><div class="time">2023-10-04 11:06:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Nono no worries. A shred of hope that someone has an idea about this is better than nothing though</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:19:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-04 11:19:51"><div class="time">2023-10-04 11:19:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You're in luck... sort of üòÜ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:19:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-04 11:19:52"><div class="time">2023-10-04 11:19:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://hackmd.io/@jfy133/HJ3Pa4Mdo#Kaiju">https://hackmd.io/@jfy133/HJ3Pa4Mdo#Kaiju</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        ü•π Aleksandra Laura Pach
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:20:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-04 11:20:07"><div class="time">2023-10-04 11:20:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I guess you would need to add a few bits of R code to insert your extra specifi ctaxa</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:26:35">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-10-04 11:26:35"><div class="time">2023-10-04 11:26:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Weeee :partyparrot:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 11:26:38">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-10-04 11:26:38"><div class="time">2023-10-04 11:26:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Great! Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-03 22:52:25">
            
                <img src="https://avatars.slack-edge.com/2023-09-27/5956726421685_976081fa5218a8bcf1b9_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ciara Wanket
                     <span class="print-only user-email">(cwanket@ucsc.edu)</span>
                </div>
                <a href="#2023-10-03 22:52:25"><div class="time">2023-10-03 22:52:25</div></a>
                <div class="msg">
                    <p>Hello! Does anyone have advice on long term storage of permafrost once it's been thawed? I have cores at -20C but I'm torn on whether to keep my subsamples at 4C or -20C. Ideally I would like to return to these subsamples in the future instead of resampling the core. Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 10:16:48">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2023-10-04 10:16:48"><div class="time">2023-10-04 10:16:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Best to use -20C for long-term storage of permafrost subsamples ‚Äî they are a great substrate for microbial and fungal growth, which is only slowed at 4C.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-04 21:53:45">
            
                <img src="https://avatars.slack-edge.com/2023-09-27/5956726421685_976081fa5218a8bcf1b9_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ciara Wanket
                     <span class="print-only user-email">(cwanket@ucsc.edu)</span>
                </div>
                <a href="#2023-10-04 21:53:45"><div class="time">2023-10-04 21:53:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Pete!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Pete Heintzman
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 15:59:54">
            
                <img src="https://avatars.slack-edge.com/2022-11-15/4370194090502_89c86a46fd40fa3f6798_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Cameron Ferguson
                     <span class="print-only user-email">(cameron.ferguson.22@ucl.ac.uk)</span>
                </div>
                <a href="#2023-10-09 15:59:54"><div class="time">2023-10-09 15:59:54</div></a>
                <div class="msg">
                    <p>There‚Äôs something I‚Äôve been wondering about lately that I can‚Äôt seem to find a answer for, is there a way to change a value given to a directive in a process in a nf-core eager config with out having to write a whole config?</p>

<p>Say for example if I was using a default docker/singularity profile and I only wanted to change how long adapter removal can run for before it times out or how much memory it can use, and I wanted everything else to remain as default is there a flag I could pass in the command line to change this value?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 16:14:58">
            
                <img src="https://avatars.slack-edge.com/2023-08-11/5727013151811_5866d2031f65a72cd9a6_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ian Light
                     <span class="print-only user-email">(light@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2023-10-09 16:14:58"><div class="time">2023-10-09 16:14:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am almost positive this all needs to be in a config file, rather than anything that could be passed as a  parameter to the actual call of the pipeline. Sorry about that!</p>

<p>the benefit of this is that once you have your config file it will be usable across different runs and help with making sure that when you run it on your cluster you know it will work as you expect!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 16:17:47">
            
                <img src="https://avatars.slack-edge.com/2023-08-11/5727013151811_5866d2031f65a72cd9a6_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ian Light
                     <span class="print-only user-email">(light@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2023-10-09 16:17:47"><div class="time">2023-10-09 16:17:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* here is the CLI info from nextflow itself, from what I can tell there is nothing that can be changed in terms of runtimes/resources for a job, etc. except with a config file <a href="https://www.nextflow.io/docs/latest/cli.html">https://www.nextflow.io/docs/latest/cli.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 16:28:35">
            
                <img src="https://avatars.slack-edge.com/2022-11-15/4370194090502_89c86a46fd40fa3f6798_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Cameron Ferguson
                     <span class="print-only user-email">(cameron.ferguson.22@ucl.ac.uk)</span>
                </div>
                <a href="#2023-10-09 16:28:35"><div class="time">2023-10-09 16:28:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I had a feeling this would be the case based on what I read, oh well it was much more of a curiosity rather than an actual issue. Though I do wonder if you set the values as variables rather then hard coded numbers could you then override the default through passing a flag ü§î</p>

<p>I could definitely see a use case for such a feature</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:17:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:17:21"><div class="time">2023-10-09 17:17:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Based on my understanding your OP, Ian is correct.</p>

<p>You can't change the resource directives via the cli.</p>

<p>You're not <em>meant</em> to need to do this. You should set these appropriately and/or dynamically once, and shouldn't have to touch it again (so least based on the nextflow concept). You don't want to pollute your reproducabile CLI command with extra infrastructure specific flags etc.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:18:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:18:17"><div class="time">2023-10-09 17:18:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So it would suggest if you're hitting cases where you run out of time (for example) your default is suboptimally set or you've not configured the auto retry correctly</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:18:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:18:23"><div class="time">2023-10-09 17:18:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (if this makes sense)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:10:55">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-10-09 17:10:55"><div class="time">2023-10-09 17:10:55</div></a>
                <div class="msg">
                    <p>Happy Monday! I present our first anonymous question for the community! Let‚Äôs help our anonymous friend out üíû</p>

<p>I know everyone is doing shotgun these days. But is there any update in processing 16S metagenomic data? I am currently using qiime2. Everyone in aDNA I asked so far dismissed amplicon sequencing altogether. So if anyone has an idea or works with that, I‚Äôd be infinitely grateful for helpful comments.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:12:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:12:35"><div class="time">2023-10-09 17:12:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oofff this is a hard one... My question for the person would be do they know <em>why</em> we do shotgun?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:12:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:12:47"><div class="time">2023-10-09 17:12:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (maybe if they are watching they can reply on the anon form)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:14:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:14:21"><div class="time">2023-10-09 17:14:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Because the Reason why is amplicon primers are too long for most ancient reads, so the vast majority of reads you amplify will be modern</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:14:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:14:32"><div class="time">2023-10-09 17:14:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (if they didn't know that already)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:30:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-10-09 17:30:02"><div class="time">2023-10-09 17:30:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It can be tough if you‚Äôre a student and the PI made the experimental design. But I think at that stage it would be important to discuss it with them and try to see if there could be any way to complement this analysis with some shotgun sequencing</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:31:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:31:59"><div class="time">2023-10-09 17:31:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes exactly... (thus my initial question..)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:32:38">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:32:38"><div class="time">2023-10-09 17:32:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I would argue there is 'no update' because it shouldn't work (or will be <em>extremely</em> difficult)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:33:08">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-10-09 17:33:08"><div class="time">2023-10-09 17:33:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Anonymous, I‚Äôm keeping an eye on the form so if you want to respond feel free to use the form again!</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 17:33:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 17:33:55"><div class="time">2023-10-09 17:33:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (sorry to be so negative! But Like @Zo√© Pochon says could definitely have a discussion to help come up with a plan to help get around some of the problems it poses [and why people are avoiding it]üôÇ )</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 18:42:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2023-10-09 18:42:25"><div class="time">2023-10-09 18:42:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://www.nature.com/articles/srep16498">https://www.nature.com/articles/srep16498</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Nature</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.nature.com/articles/srep16498">Intrinsic challenges in ancient microbiome reconstruction using 16S rRNA gene amplification</a></div>
                                <div class="link-text">
                                    Scientific Reports - Intrinsic challenges in ancient microbiome reconstruction using 16S rRNA gene amplification
                                </div>
                                
                                
    
        <a href="https://www.nature.com/articles/srep16498">
            <img class="preview" src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fsrep16498/MediaObjects/41598_2015_Article_BFsrep16498_Fig1_HTML.jpg"
                width="1575"
                 height="1553" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.nature.com/articles/srep16498</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-09 18:43:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-09 18:43:11"><div class="time">2023-10-09 18:43:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Meriam Guellil! Was looking for that until üë∂ said no</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòâ Meriam Guellil
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:15:10">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2023-10-10 17:15:10"><div class="time">2023-10-10 17:15:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hold up ‚Äî is anon doing microbial 16S metabarcoding (with long insert lengths and hence problematic) <em><strong>or</strong></em> vertebrate 16S metabarcoding (with short insert lengths and suitable in some contexts)?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:15:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-10 17:15:49"><div class="time">2023-10-10 17:15:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good poinT!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:01:01">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2023-10-10 17:01:01"><div class="time">2023-10-10 17:01:01</div></a>
                <div class="msg">
                    <p>Thanks to everyone who answered our last anon. question. Here we have another for you metagenomics pros:</p>

<p><strong>‚ÄúI want to use SourceTracker but I don't know how to construct the database. Any of you know how? I want to include different sources, not only human-related microorganisms.‚Äù</strong></p>

<p>Questions about SourceTracker have come up before so we would be really keen to run a blog post all about getting the software installed, creating databases and different parameters - let us know if you‚Äôd be interested in volunteering your expertise! Even if you don‚Äôt consider yourself an ‚Äúexpert‚Äù, we‚Äôd love to hear anecdotes about what‚Äôs worked for you and what hasn‚Äôt.</p>
                    
                    
                    
                        <div class="message-reaction">
                        :poop_party: Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:04:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-10 17:04:17"><div class="time">2023-10-10 17:04:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There isn't a specific 'database' for Sourcetracker!</p>

<p>You just pick your 'sources' from whatever is appropriate for your question, and run both those and the sinks through a taxonomic classifier against the same database. The input is literally just an OTU table, and a metadata table saying which sample is a source, and which is a sink üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Sterling Wright, Biancamaria Bonucci, Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:10:53">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:10:53"><div class="time">2023-10-10 17:10:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* In case you want to use the MGnify biomes to retrieve sources from ENA this can be helpful <a href="https://github.com/genomewalker/get-biomes">https://github.com/genomewalker/get-biomes</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">genomewalker/get-biomes</a></div>
                                <div class="link-text">
                                    A simple tool to get biomes related data from MGnify and ENA
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Python
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Last updated</div>
                                        a year ago
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:11:28">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:11:28"><div class="time">2023-10-10 17:11:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This is what I use to gather sources for the sourcetracking analyses</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:11:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-10 17:11:51"><div class="time">2023-10-10 17:11:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You should start publishing or Zenodo'ing these üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:12:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-10 17:12:02"><div class="time">2023-10-10 17:12:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (note Zenodo has auto github archiving üòâ )</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:12:49">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:12:49"><div class="time">2023-10-10 17:12:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is coming a methods preprint with all these tools that make life easier </p>
                    
                    
                    
                        <div class="message-reaction">
                        :partyparrot: James Fellows Yates, Camila Duitama, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:13:43">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:13:43"><div class="time">2023-10-10 17:13:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am compiling some of them here: <a href="https://github.com/aMG-tk">https://github.com/aMG-tk</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">Ancient metaGenomics toolkit</a></div>
                                <div class="link-text">
                                    A collection of tools to analyse ancient metagenomic data
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Repositories</div>
                                        15
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Followers</div>
                                        1
                                    </div>
                                
                                
    
        <a href="None">
            <img class="preview" src="420"
                
                 />
        </a>
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:17:48">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:17:48"><div class="time">2023-10-10 17:17:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also if you use sourcetracker, I recommend using our extension with the diagnostics so you can evaluate how the MCs converge (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100590/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100590/</a>)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed Central (PMC)</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100590/">Meta-SourceTracker: application of Bayesian source tracking to shotgun metagenomics</a></div>
                                <div class="link-text">
                                    Microbial source tracking methods are used to determine the origin of contaminating bacteria and other microorganisms, particularly in contaminated water systems. The Bayesian SourceTracker approach uses deep-sequencing marker gene libraries (16S ribosomal ...
                                </div>
                                
                                
    
        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100590/">
            <img class="preview" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0"
                width="1600"
                 height="836" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100590/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:34:54">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-10-10 17:34:54"><div class="time">2023-10-10 17:34:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I agree with everything James and Antonio have put here. I would just add that another consideration is to try to include sources that were prepared in a similar fashion as your data was created (e.g., shotgun, paired end vs single end,  etc). It may also be useful to do some quality control on your sources before running Sourcetracker. Removing contaminant taxa in the sources. Another consideration may be to do a random forest or PCoA with only the source samples to check their homogeneity. I know that ST2 has an option that checks this but I think it would still be worth doing one of the two options above. Depending on the similarity of your environment sources, you may have several taxa that are present in multiple sources which can impact your downstream results.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:40:17">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:40:17"><div class="time">2023-10-10 17:40:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Exactly, get-biomes was designed with this in mind, to narrow down the potential sources that might fulfill your sink conditions. And as @Sterling Wright says, it is very important to explore and pre-process your abundance tables before ST. We also only use only damaged taxa for the sinks (when possible).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 17:42:05">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 17:42:05"><div class="time">2023-10-10 17:42:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We also used get-biomes to train @Camila Duitama decOM beyond oral microbiomes </p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 18:29:30">
            
                <img src="https://avatars.slack-edge.com/2021-09-23/2522174708661_ef8b03f2d80d95a455bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Camila Duitama
                     <span class="print-only user-email">(camiladuitama@gmail.com)</span>
                </div>
                <a href="#2023-10-10 18:29:30"><div class="time">2023-10-10 18:29:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Was this the Kap K√∏benhavn Formation paper@Antonio Fernandez-Guerra ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 18:33:26">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-10 18:33:26"><div class="time">2023-10-10 18:33:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yep, and other projects we are complementing ST with decOM</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-13 17:13:16">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-10-13 17:13:16"><div class="time">2023-10-13 17:13:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and if you‚Äôre looking at human specific biomes, curatedMetagenomicsData is also a great resource (even with pre-computed metaphlan3 profiles)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 18:36:36">
            
                <img src="https://avatars.slack-edge.com/2021-09-23/2522174708661_ef8b03f2d80d95a455bc_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Camila Duitama
                     <span class="print-only user-email">(camiladuitama@gmail.com)</span>
                </div>
                <a href="#2023-10-10 18:36:36"><div class="time">2023-10-10 18:36:36</div></a>
                <div class="msg">
                    <p>Hi all!</p>

<p>I am interested in getting to know a bit more about human aDNA obtained in Colombia. I'd like to know a bit more about the outcomes of those publications, researchers (foreign and local) working on those samples, ethical issues, etc. Any suggestions?</p>

<p>Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 19:00:18">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-10 19:00:18"><div class="time">2023-10-10 19:00:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* HAAM community for the rescue! @Thiseas C. Lamnidis (but he's sitting next to me atm so will reply later)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 19:00:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-10 19:00:42"><div class="time">2023-10-10 19:00:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://haam-community.github.io/">https://haam-community.github.io/</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 19:12:54">
            
                <img src="https://avatars.slack-edge.com/2022-11-24/4444730834256_55a44f4b7371f8efcdb7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Constanza de la Fuente
                     <span class="print-only user-email">(cdelafuente@uchicago.edu)</span>
                </div>
                <a href="#2023-10-10 19:12:54"><div class="time">2023-10-10 19:12:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* for local researchers, check the work of Andrea Casas-Vargas (mtDNA control region or hg only). Foreign, check this: <a href="https://www.sciencedirect.com/science/article/pii/S1040618220304961">https://www.sciencedirect.com/science/article/pii/S1040618220304961</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">sciencedirect.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.sciencedirect.com/science/article/pii/S1040618220304961">A paleogenetic perspective of the Sabana de Bogot√° (Northern South America) population history over the Holocene (9000‚Äì550¬†cal BP)</a></div>
                                <div class="link-text">
                                    On the basis of distinct lines of evidence, detailed reconstructions of the Holocene population history of the Sabana de Bogot√° (SB) region, Northern ‚Ä¶
                                </div>
                                
                                
    
        <a href="https://www.sciencedirect.com/science/article/pii/S1040618220304961">
            <img class="preview" src="https://ars.els-cdn.com/content/image/1-s2.0-S1040618221X00080-cov150h.gif"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.sciencedirect.com/science/article/pii/S1040618220304961</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Camila Duitama
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-10 19:14:42">
            
                <img src="https://avatars.slack-edge.com/2022-11-24/4444730834256_55a44f4b7371f8efcdb7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Constanza de la Fuente
                     <span class="print-only user-email">(cdelafuente@uchicago.edu)</span>
                </div>
                <a href="#2023-10-10 19:14:42"><div class="time">2023-10-10 19:14:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* i‚Äôm not aware of paleogenomic studies or ethical discussions, but maybe Andrea Casas-Vargas can help?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-13 11:44:04">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2023-10-13 11:44:04"><div class="time">2023-10-13 11:44:04</div></a>
                <div class="msg">
                    <p>Hiya, does anyone have any experience using <a href="https://www.retsch.com/products/milling/mixer-mills/">mixer mills</a> jars to generate powder in the lab? They are expensive so I want to make sure I am ordering something that will last!</p>

<p>Interested in hearing about what size jars people use? What material are they made of? How do you effectively clean the jars between samples?</p>

<p>Thanks! ü¶¥</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-13 13:37:41">
            
                <img src="https://avatars.slack-edge.com/2020-11-25/1540876615025_e4bf4dfa22795dcdb8d3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Marcel Keller
                     <span class="print-only user-email">(marcel.keller@ut.ee)</span>
                </div>
                <a href="#2023-10-13 13:37:41"><div class="time">2023-10-13 13:37:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve used one years ago for ancient DNA samples (Master thesis 2013, PCR-based stuff), my colleagues here use one to prepare bone samples for stable isotope analyses. For the DNA samples back then we used Zirconium oxide jars and a Retsch MM2 mill. We cleaned them with bleach, water and UV irradiated them. In my memory they were quite easy to clean.</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-13 14:16:14">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2023-10-13 14:16:14"><div class="time">2023-10-13 14:16:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We use them and clean with bleach, dna off, ethanol &amp; then UV them. They're pretty easy to clean but we do give them a good scrub in the bleach before the other steps. üôÇ Not sure which ones we actually use, but I think they're about 10cm long-ish??? and maybe 5cm in diameter, but this is from memory (I haven't drilled anything in nearly 2 years at this stage).</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-13 16:38:09">
            
                <img src="https://avatars.slack-edge.com/2021-12-22/2872808514018_3c9b0a94d8e687729e4f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nihan D Dagtas
                     <span class="print-only user-email">(dagtas.nd@gmail.com)</span>
                </div>
                <a href="#2023-10-13 16:38:09"><div class="time">2023-10-13 16:38:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi, I haven't used them in years but one thing I remember is that the polycarbonate cylinders were turning yellow and brittle-ish, possibly due to bleach and UV exposure.</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-16 12:40:50">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2023-10-16 12:40:50"><div class="time">2023-10-16 12:40:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all for the helpful replies üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Iseult, Nihan D Dagtas
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 20:56:48">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-10-14 20:56:48"><div class="time">2023-10-14 20:56:48</div></a>
                <div class="msg">
                    <p>Hi! I've got a memory of someone mentioning a while back that a fish genome had lots of adapters. Was it carp? I'm seeing it in extraction controls for my samples of sloth coprolites and from a layer of dung in the site (15 reads), then also in my samples (up to 130 reads or 0.02% of ids in a sample of soil, and up to 500 reads or 0.13% of ids in a coprolite) - If this is right, do you have a reference that discusses this?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 20:59:08">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2023-10-14 20:59:08"><div class="time">2023-10-14 20:59:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, the carp genome (Cyprinus carpio) is contaminated with adapters. </p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§© Maria Zicos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 21:10:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-14 21:10:07"><div class="time">2023-10-14 21:10:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://web.archive.org/web/20201219022000/http://grahametherington.blogspot.com/2014/09/why-you-should-qc-your-reads-and-your.html?m=1">https://web.archive.org/web/20201219022000/http://grahametherington.blogspot.com/2014/09/why-you-should-qc-your-reads-and-your.html?m=1</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§© Maria Zicos
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 21:10:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-14 21:10:19"><div class="time">2023-10-14 21:10:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It was identified even earlier!!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 21:21:40">
            
                <img src="https://avatars.slack-edge.com/2022-06-05/3617287454902_0acf2ce2d6e78241294a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Zicos
                     <span class="print-only user-email">(m.zicos@qmul.ac.uk)</span>
                </div>
                <a href="#2023-10-14 21:21:40"><div class="time">2023-10-14 21:21:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thank you all!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 21:56:19">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-10-14 21:56:19"><div class="time">2023-10-14 21:56:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <a href="https://twitter.com/baym/status/1712550914077372880">https://twitter.com/baym/status/1712550914077372880</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">X (formerly Twitter)</div>
                            
                            
                                
                                <div class="link-title"><a href="https://twitter.com/baym/status/1712550914077372880">Michael Baym on X</a></div>
                                <div class="link-text">
                                    In genomics we don‚Äôt say your pipeline fails to trim reads properly, we say your data is full of carp and I think that‚Äôs lovely
                                </div>
                                
                                
    
        <a href="https://twitter.com/baym/status/1712550914077372880">
            <img class="preview" src="https://pbs.twimg.com/profile_images/1564276832245088256/EWg_Aypp_200x200.jpg"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://twitter.com/baym/status/1712550914077372880</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates, Aleksandra Laura Pach, Meriam Guellil, Zo√© Pochon, Wenqin Yu
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 10:35:16">
            
                <img src="https://secure.gravatar.com/avatar/bf0027274e456ad2e59e64832c522cc9.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0017-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Bastiaan
                     <span class="print-only user-email">(bastiaan.star@ibv.uio.no)</span>
                </div>
                <a href="#2023-10-18 10:35:16"><div class="time">2023-10-18 10:35:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have used this as a teaching example on the importance of understanding the background of online genomic databases for many years now</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò¨ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-14 20:58:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-10-14 20:58:14"><div class="time">2023-10-14 20:58:14</div></a>
                <div class="msg">
                    <p>Yes, Cyprinos carpio! <a href="https://dgg32.medium.com/carp-in-the-soil-1168818d2191">https://dgg32.medium.com/carp-in-the-soil-1168818d2191</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Medium</div>
                            
                            
                                
                                <div class="link-title"><a href="https://dgg32.medium.com/carp-in-the-soil-1168818d2191">Carp in the Soil</a></div>
                                <div class="link-text">
                                    Ridiculous sequencing results revealed how errors propagated from one research study to a global database
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Reading time</div>
                                        7 min read
                                    </div>
                                
                                
    
        <a href="https://dgg32.medium.com/carp-in-the-soil-1168818d2191">
            <img class="preview" src="https://miro.medium.com/v2/resize:fit:1200/1*WasdCvAsPa_4aFiJLn0pKw.jpeg"
                width="1200"
                 height="900" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://dgg32.medium.com/carp-in-the-soil-1168818d2191</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ü§© Maria Zicos
                        </div>
                    
                        <div class="message-reaction">
                        üëç Pete Heintzman, Eva Chocholov√°
                        </div>
                    
                        <div class="message-reaction">
                        üòÇ Zo√© Pochon, Wenqin Yu
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 10:27:16">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-10-18 10:27:16"><div class="time">2023-10-18 10:27:16</div></a>
                <div class="msg">
                    <p>Hey lovely people!
Does anyone have experience with using Kaiju for ancient data? If yes, how do you authenticate the hits? (Here I'm thinking something corresponding to the ED, ANI, breadth of coverage, ancient damage etc.??)
Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 10:43:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-18 10:43:45"><div class="time">2023-10-18 10:43:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My understanding was it's not recommended as you translate to amino acid sequences, so you loose as unspecific all the Very short actually ancient reads during translation.</p>

<p>And then because you're dealing with amino acids can't do any of our normal validation</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÜ:skin_tone_2: Meriam Guellil
                        </div>
                    
                        <div class="message-reaction">
                        üëå Aleksandra Laura Pach
                        </div>
                    
                        <div class="message-reaction">
                        üëÜ Carly Scott
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:02:14">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-18 11:02:14"><div class="time">2023-10-18 11:02:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, I second this. The translation step would loose almost all reads shorter than 60 bp because there is limited resolution in assigning amino acid sequences with fewer than 20 aa.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Aleksandra Laura Pach
                        </div>
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:40:59">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-18 11:40:59"><div class="time">2023-10-18 11:40:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates you can use my emoji here‚Ä¶ now we are getting good results down to 15 aa, and we will be pushing it down to 10aa in the near future with new sub matrices. We combine read-extension, with a fine tuned version of mmseqs2 and the E-M filtering from <a href="https://github.com/genomewalker/x-filter">https://github.com/genomewalker/x-filter</a> to maximise the information we can use from translated-searches (function and taxonomy)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Alex H√ºbner, Aleksandra Laura Pach
                        </div>
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:45:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-18 11:45:35"><div class="time">2023-10-18 11:45:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But that's not Kaiju is it ;)</p>

<p>Also, publish something then period can use it ü§£</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:45:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-18 11:45:42"><div class="time">2023-10-18 11:45:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But exciting!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:47:00">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-18 11:47:00"><div class="time">2023-10-18 11:47:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It has been used for the 2-million year old stuff, at least is in biorxiv</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:51:06">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-18 11:51:06"><div class="time">2023-10-18 11:51:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt;  But that‚Äôs not Kaiju is it üòâ
No, I don‚Äôt think Kaiju can use other matrix than BLOSUM, which is not recommended for our type of data</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Aleksandra Laura Pach
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-18 11:53:17">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2023-10-18 11:53:17"><div class="time">2023-10-18 11:53:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and also damage introduces artificial non-syn subs and stop codons which are not very suitable for methods like kaiju</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Aleksandra Laura Pach
                        </div>
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-05 12:45:12">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2024-01-05 12:45:12"><div class="time">2024-01-05 12:45:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @James Fellows Yates @Alex H√ºbner @Antonio Fernandez-Guerra @Meriam Guellil @Carly Scott
<strong>Follow up on the question from last time</strong>: does any of you happen to know any publications that mention this issue - especially maybe on the loss of specificity due to homologous peptide regions? But anything really. üôá</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-05 12:48:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-01-05 12:48:23"><div class="time">2024-01-05 12:48:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Internet is bad, so can't Google, but maybe @Raphael Eisenhofer's MALT blastx Vs maltn mode paper might discuss a littke? I think it was in peerj</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Aleksandra Laura Pach
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-05 13:02:36">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2024-01-05 13:02:36"><div class="time">2024-01-05 13:02:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Not aDNA related but this might be useful <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03802-0">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03802-0</a>
This is the approach we reimplemented in xfilter </p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03802-0">In silico benchmarking of metagenomic tools for coding sequence detection reveals the limits of sensitivity and precision - BMC Bioinformatics</a></div>
                                <div class="link-text">
                                    
                                </div>
                                
                                
    
        <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03802-0">
            <img class="preview" src="https://media.springernature.com/w200/springer-static/cover/journal/12859.jpg"
                width="200"
                 height="283" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03802-0</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Aleksandra Laura Pach, Alex H√ºbner
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-23 10:54:12">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2023-10-23 10:54:12"><div class="time">2023-10-23 10:54:12</div></a>
                <div class="msg">
                    <p>Does anyone actually understand how metaDMG creates the output csv files? I have been looking at one example for a while now and can't figure it out. When I check the underlying lca files, every line is a unique read assigned to the lowest taxonomic level but in the metaDMG output file, several rows describe the same reads on different taxonomic levels (see attached). I don't really understand the rationale behind this and it makes the files a bit hard to work with but maybe I am missing something?</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06246V1NCW/metadmg_output.jpg?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">metaDMG_output.JPG</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06246V1NCW/metadmg_output.jpg?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06246V1NCW-70af739768/metadmg_output_360.jpg?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="100" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-23 12:01:43">
            
                <img src="https://avatars.slack-edge.com/2023-08-03/5687623376547_da635bb16ef0d8c82526_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yi Wang
                     <span class="print-only user-email">(yiwang@uni-konstanz.de)</span>
                </div>
                <a href="#2023-10-23 12:01:43"><div class="time">2023-10-23 12:01:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* From my understanding, this is because metaDMG calculates damage at all taxonomic levels. When moving one rank up, it gathers all daughter taxa and recalculate damage. In your example, from Gnathostomata and up there is no new taxon joining, therefore N_reads stays at 85.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-23 15:07:05">
            
                <img src="https://secure.gravatar.com/avatar/0fe03c40515253aa9fba15dab119e286.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Lennart Schreiber
                     <span class="print-only user-email">(lennartschreiber@web.de)</span>
                </div>
                <a href="#2023-10-23 15:07:05"><div class="time">2023-10-23 15:07:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ahh yes, you are right! I got too distracted by the fact that there is no obvious order of the rows and I was also expecting more reads on higher taxonomic levels but yes, when I sum up the reads of e.g. all classes, I get the clade reads etc. Thank you so much üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòä Yi Wang
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-24 16:25:57">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2023-10-24 16:25:57"><div class="time">2023-10-24 16:25:57</div></a>
                <div class="msg">
                    <p>Hey everyone üëã</p>

<p>We have another anonymous question, help a metagenomics pal out:</p>

<p>Hello everyone! We are working with lake sediments and we are wondering what to consider when preparing samples for dating. What do you usually do when macrofossils do not preserve well in your sediments? Have you ever used e.g. pollen extracts for AMS dating?</p>

<p>üëÄ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üçÉ Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-24 16:41:53">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-10-24 16:41:53"><div class="time">2023-10-24 16:41:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sediments ü§î @Stephanie Dolenz perhaps?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-24 17:53:00">
            
                <img src="https://secure.gravatar.com/avatar/9760142fa7c8ac95c5b97af3dd5a7ec6.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0026-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Stephanie Dolenz
                     <span class="print-only user-email">(stephanie_dolenz@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-24 17:53:00"><div class="time">2023-10-24 17:53:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I haven‚Äôt tried to date sediment samples yet, but perhaps @Kevin Nota? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-25 09:02:08">
            
                <img src="https://avatars.slack-edge.com/2023-10-25/6085479900966_31abc2b5900af9ac7d4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Nota
                     <span class="print-only user-email">(kevin_nota@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-25 09:02:08"><div class="time">2023-10-25 09:02:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Mmm, I also never dated samples myself. Maybe @Pete Heintzman can help with this?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-25 11:22:52">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2023-10-25 11:22:52"><div class="time">2023-10-25 11:22:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Have not used pollen extracts for dating but it is possible. Suggest contacting the radiocarbon lab that you use about this.</p>

<p>Do you have any macrofossils at all? Even if they are very small/fragmentary, then these are usually datable (assuming you are looking at a Holocene record -- might not work well if going into MIS2 or MIS3 sediments, closer to the radiocarbon limit).</p>

<p>Depending on the sediment type, you could also try bulk sediment dating (especially of any organic layers). However, this assumes that there is very little to no carbonate present in the sediment (which is radiocarbon-dead and will give too old dates).</p>

<p>OP: feel free to DM me if you want to follow up! Cheers, Pete</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-25 13:54:49">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-25 13:54:49"><div class="time">2023-10-25 13:54:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Was definitely hoping you would just tag someone else, @Pete Heintzman üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-25 13:55:45">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2023-10-25 13:55:45"><div class="time">2023-10-25 13:55:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* As a follow-up, I am sure that @Benjamin Vernot can offer some insight. üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-25 13:56:42">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2023-10-25 13:56:42"><div class="time">2023-10-25 13:56:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* haha, not ME!</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§£ Pete Heintzman, Kevin Nota, Stephanie Dolenz, James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-25 16:48:44">
            
                <img src="https://avatars.slack-edge.com/2023-09-27/5956726421685_976081fa5218a8bcf1b9_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ciara Wanket
                     <span class="print-only user-email">(cwanket@ucsc.edu)</span>
                </div>
                <a href="#2023-10-25 16:48:44"><div class="time">2023-10-25 16:48:44</div></a>
                <div class="msg">
                    <p>Hi everybody! I'm in the middle of extractions with Roche high-volume silica columns and realized I don't have buffer PE, which is what I usually use to wash the membrane. I do have the wash buffer that comes in the Roche kit and Qiagen buffer C5. I'm pretty sure they're all basically the same thing, but thought I'd ask others' opinions before doing anything! Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-26 11:05:16">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2023-10-26 11:05:16"><div class="time">2023-10-26 11:05:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Already talked to Ciara about this but a general FYI to any wet lab folks:</p>

<p>Ready-to-use wash buffer compositions:
‚Ä¢ Qiagen PE buffer: 10 mM Tris (pH 7.5) and 80% ethanol (unconfirmed but suspected composition)
‚Ä¢ Roche HPVNALV kit buffer: 2 mM Tris (pH 7.5), 20 mM NaCl, and 80% ethanol (confirmed composition)
If anyone home-brews, make sure to use the correct Tris-HCl pH, as silica binding/release is highly sensitive to pH:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F062TNZGQ2Z/screenshot_2023-10-25_at_17.35.35.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-10-25 at 17.35.35.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F062TNZGQ2Z/screenshot_2023-10-25_at_17.35.35.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F062TNZGQ2Z-69ab9eb45b/screenshot_2023-10-25_at_17.35.35_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="193" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üëç Liam Lanigan
                        </div>
                    
                        <div class="message-reaction">
                        ü§© Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-26 16:11:49">
            
                <img src="https://avatars.slack-edge.com/2023-09-27/5956726421685_976081fa5218a8bcf1b9_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ciara Wanket
                     <span class="print-only user-email">(cwanket@ucsc.edu)</span>
                </div>
                <a href="#2023-10-26 16:11:49"><div class="time">2023-10-26 16:11:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you again Pete! We ended up getting a pretty good DNA yield out of these. I might do a comparison of the different commercial and home brew wash buffers at some point just to see how different they are. Happy to eventually share the results with anyone who uses the Dabney extraction protocol and is also curious. </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Pete Heintzman
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Ele
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-26 18:37:02">
            
                <img src="https://secure.gravatar.com/avatar/db913ec843c9b9d1de24902beed515f5.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0023-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Abby Gancz
                     <span class="print-only user-email">(agancz@gmail.com)</span>
                </div>
                <a href="#2023-10-26 18:37:02"><div class="time">2023-10-26 18:37:02</div></a>
                <div class="msg">
                    <p>HI all! A researcher I worked with has a question and I was wondering whether anyone has any insights: "I am trying to calculate dN/dS ratios within-species for genes in <em>S. sp.</em> DD04 and <em>S. sanguinis,</em> and I have a few questions and was wondering if you could offer any insight. I have tables with the breadth and depth of coverage for each protein coding sequence in the annotated genome but am not sure how to go about sample/gene selection. What would be considered "sufficient coverage" of a gene to use in this analysis? Right now I have that minimum coverage of 3 is needed to call a variant, so I was thinking a depth of at least that, as well as breadth &gt;95% or 100%.</p>

<p>I've seen that most programs (like CODEML and FUBAR) are guided by a phylogeny built with the genes of interest, which allows for finely characterizing selection pressures, including across different branches. However, given it is ancient DNA, I think this would be limited to genes that have sufficient coverage across all included samples. I also saw the program <a href="https://academic.oup.com/mbe/article/37/8/2450/5804989?login=false">GenomegaMap</a> which is phylogeny free and made for within-species estimation. Do you have any suggestions about which could be the more appropriate approach?"</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-27 14:13:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-10-27 14:13:56"><div class="time">2023-10-27 14:13:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @Abby Gancz ,
From my (short) experimentations, I found HyPhy nice to use <a href="https://hyphy.org/">https://hyphy.org/</a> (also online with <a href="https://datamonkey.org/">https://datamonkey.org/</a></p>

<p>Regarding your gene inclusion thresholds, you can also look at the variant calling quality with bcftools and filter on that instead. (for example Q&gt;=20 and allele support &gt;= 3)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">hyphy.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://hyphy.org/">HyPhy - Hypothesis Testing using Phylogenies</a></div>
                                <div class="link-text">
                                    Information, documentation, and news about the HyPhy software package
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://hyphy.org/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-27 16:57:11">
            
                <img src="https://secure.gravatar.com/avatar/db913ec843c9b9d1de24902beed515f5.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0023-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Abby Gancz
                     <span class="print-only user-email">(agancz@gmail.com)</span>
                </div>
                <a href="#2023-10-27 16:57:11"><div class="time">2023-10-27 16:57:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-10-30 22:07:21">
            
                <img src="https://avatars.slack-edge.com/2021-09-29/2543484980226_3b1e4f62804f72a90b6b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Meriam van Os
                     <span class="print-only user-email">(meriam.vanos@postgrad.otago.ac.nz)</span>
                </div>
                <a href="#2023-10-30 22:07:21"><div class="time">2023-10-30 22:07:21</div></a>
                <div class="msg">
                    <p>Hey all! I'm trying to use blastn on the command line and then visualise the results in megan, and I'm a bit stuck. I tried various things, but both the blast2rma tool and the megan app keep saying something like "Warning: Might not be a BLAST file in XML/SAM/BlastText format: 8511-37.blastn.out", while I do specify a specific output. What am I missing?
Code I use to run blastn (for XML):
blastn -query ${directory}/${sample}.collapsed.fasta -db nt -out ${sample}.blastn.out -outfmt 5 -num<em>threads $SLURM</em>CPUS<em>PER</em>TASK</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:35:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:35:15"><div class="time">2023-12-20 16:35:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Did you ever solve this @Meriam van Os?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-01 12:10:23">
            
                <img src="https://secure.gravatar.com/avatar/0eadc195be2a1e9d6c1bd25a2aa83bfd.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Jan Laine
                     <span class="print-only user-email">(jan.laine@ntnu.no)</span>
                </div>
                <a href="#2023-11-01 12:10:23"><div class="time">2023-11-01 12:10:23</div></a>
                <div class="msg">
                    <p>Hi all, I was wondering if anyone here could help me with my specific issue regarding adding custom genomes into kraken2 üôÇ I have been trying to incorporate two additional genomes of a species that is part of the standard library in kraken2 to the database I would use to analyse my asedDNA. What I‚Äôm trying to achieve is that these two additional reference genomes would show as subspecies in the final taxonomy. I attempted to do this by downloading the names.dmp and nodes.dmp files and adding these genomes to those files with custom taxIDs. However, it seems that the custom genomes are still not present at all in the final database. Would anyone have any idea what went wrong with this approach? Thanks!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-01 19:59:38">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-01 19:59:38"><div class="time">2023-11-01 19:59:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could you provide all the commands you've used?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 10:48:09">
            
                <img src="https://secure.gravatar.com/avatar/0eadc195be2a1e9d6c1bd25a2aa83bfd.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jan Laine
                     <span class="print-only user-email">(jan.laine@ntnu.no)</span>
                </div>
                <a href="#2023-11-02 10:48:09"><div class="time">2023-11-02 10:48:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi James,</p>

<p>Sure, thanks!</p>

<p>So at first I downloaded the taxonomy as follows:</p>

<p>kraken2-build --download-taxonomy --db Jan<em>sedDNA</em>DB</p>

<p>And then the standard libraries, bacteria here as example:</p>

<p>kraken2-build --download-library bacteria --db Jan<em>sedDNA</em>DB --threads 4</p>

<p>I then downloaded the names.dmp and nodes.dmp files and added the following lines there as follows:</p>

<p>9999999 |   69293   |   subspecies  |   GA  |   10  |   1   |   1   |   1   |   2   |   1   |   1   |   0   |       |
9999998 |   69293   |   subspecies  |   GA  |   10  |   1   |   1   |   1   |   2   |   1   |   1   |   0   |       |   </p>

<p>9999999 |   Gasterosteus aculeatus freshwater   |       |   scientific name |
9999998 |   Gasterosteus aculeatus marine   |       |   scientific name |</p>

<p>After uploading these files back to the taxonomy folder of the database I added the custom taxIDs on the fasta files of the custom genomes</p>

<p>&gt;Freshwater|kraken:taxid|9999999
&gt;Marine_fjord|kraken:taxid|9999998</p>

<p>These genomes also had chromosomes indicated with &gt;chr<em>*</em>* so I erased all the lines starting with &gt; except the first one.</p>

<p>sed -i ‚Äò2,${/^&gt;/d}‚Äô gasAcu1.fa
sed -i ‚Äò2,${/^&gt;/d}‚Äô fjord.fa</p>

<p>And then I added the genomes:</p>

<p>kraken2-build --add-to-library gasAcu1.fa --db Jan<em>sedDNA</em>DB
kraken2-build --add-to-library fjord.fa --db Jan<em>sedDNA</em>DB</p>

<p>And finally I just ran the build command:</p>

<p>kraken2-build --build --db Jan<em>sedDNA</em>DB --threads 40</p>

<p>The final database works fine except it just seems that my custom genomes are not there at all. Did I skip some step or do something wrong?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:05:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-02 14:05:45"><div class="time">2023-11-02 14:05:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* How do you validate that the custom genomes are not in there?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:45:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 14:45:51"><div class="time">2023-11-02 14:45:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I noticed the same issue for my custom built db @Jan Laine</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:50:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-02 14:50:08"><div class="time">2023-11-02 14:50:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No solution? Because yeah I agree the commands look ok to me :(</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:52:21">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 14:52:21"><div class="time">2023-11-02 14:52:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôm not sure neither, but that seems to be an issue that has been reported on their github
<a href="https://github.com/DerrickWood/kraken2/issues?q=is%3Aissue+is%3Aopen+custom">https://github.com/DerrickWood/kraken2/issues?q=is%3Aissue+is%3Aopen+custom</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:53:27">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 14:53:27"><div class="time">2023-11-02 14:53:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* From my understanding, it works to add genomes to a completely new database, but if you want to use these pre-selected databases, it doesn‚Äôt work.
But it‚Äôs not clearly documented anywhere</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:55:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-02 14:55:43"><div class="time">2023-11-02 14:55:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ahhh that would make sense, I had no issues with this on a custom database for nf-core/taxprofiler, but that was pure xustom</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:00:51">
            
                <img src="https://secure.gravatar.com/avatar/0eadc195be2a1e9d6c1bd25a2aa83bfd.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jan Laine
                     <span class="print-only user-email">(jan.laine@ntnu.no)</span>
                </div>
                <a href="#2023-11-02 15:00:51"><div class="time">2023-11-02 15:00:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okey thanks a lot for looking into this, I really appreciate it! To validate that they are not in there I just first assumed so since they got no hits even though they should with the samples i tested it. I also downloaded the report and tried to search the custom taxIDs without any hits. But good to know that this seems to be the issue then that it needs to be completely custom, maybe I can work with that info! üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:01:12">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 15:01:12"><div class="time">2023-11-02 15:01:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You can check with the inspect command</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:01:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 15:01:29"><div class="time">2023-11-02 15:01:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But yeah, that‚Äôs not an ideal situation</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:03:13">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-02 15:03:13"><div class="time">2023-11-02 15:03:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah ok. I don't think just not finding hits is necessaily the best way, could be they get moved up the LCA because the genomes are similar? Unless you're looking at the per read hits...</p>

<p>Agree with maxime better to investigate the database itself</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:07:40">
            
                <img src="https://secure.gravatar.com/avatar/0eadc195be2a1e9d6c1bd25a2aa83bfd.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jan Laine
                     <span class="print-only user-email">(jan.laine@ntnu.no)</span>
                </div>
                <a href="#2023-11-02 15:07:40"><div class="time">2023-11-02 15:07:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay thanks, I‚Äôll check that as well!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:50:57">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-02 15:50:57"><div class="time">2023-11-02 15:50:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Seconding that, in my experience I had to start from scratch, downloading all the genomes and running build, when I wanted to just add a couple taxa of interest to the standard database and it was a pain üòï
I wonder if you could avoid the computational hassle by creating a smaller custom database of just that genus/species including your subspecies? And then run just the reads ID‚Äôd as the species of interest from the standard library against the custom database? I need to think through that a bit more though</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 16:07:18">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 16:07:18"><div class="time">2023-11-02 16:07:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do other users of Kraken2 have notice the same behavior ? (@Nikolay Oskolkov maybe ?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 16:09:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 16:09:11"><div class="time">2023-11-02 16:09:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Seems like the behaviour also reported here: <a href="https://github.com/DerrickWood/kraken2/issues/538">https://github.com/DerrickWood/kraken2/issues/538</a></p>
                    <div class="message-attachment"
                            style="border-color: #cb2431">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/DerrickWood/kraken2/issues/538">#538 Adding new genomes to a preexisting db</a></div>
                                <div class="link-text">
                                    &lt;p&gt;Hi, can someone please check if these following steps are correct?&lt;/p&gt;

&lt;p&gt;I am trying to add to my plants kraken2 db (&#34;plant&lt;em&gt;original&#34;) few taxa genomes that I have downloaded from the NCBI website (alnus&lt;/em&gt;glutinosa&lt;em&gt;GCA&lt;/em&gt;003254965.1.fna, carpinus&lt;em&gt;fangiana&lt;/em&gt;GCA_006937295.1.fna etc..).&lt;/p&gt;

&lt;pre&gt;&lt;code&gt;for file in **.fna   
    do
        kraken2-build --add-to-library $file --db PATH/kraken/plant_original
    done    
&lt;/code&gt;&lt;/pre&gt;

&lt;p&gt;Masking low-complexity regions of new file... done. Added &#34;alnus&lt;em&gt;glutinosa&lt;/em&gt;GCA&lt;em&gt;003254965.1.fna&#34; to library (PATH/kraken/plant&lt;/em&gt;original)&lt;br /&gt;
Masking low-complexity regions of new file... done.&lt;br /&gt;
Added &#34;carpinus&lt;em&gt;fangiana&lt;/em&gt;GCA&lt;em&gt;006937295.1.fna&#34; to library (PATH/kraken/plant&lt;/em&gt;original)&lt;br /&gt;
Masking low-complexity regions of new file... done.&lt;/p&gt;

&lt;p&gt;&lt;code&gt;kraken2-build --build --db ~/kraken/plant_original&lt;/code&gt;&lt;/p&gt;

&lt;p&gt;Creating sequence ID to taxonomy ID map (step 1)...&lt;br /&gt;
Sequence ID to taxonomy ID map already present, skipping map creation.&lt;br /&gt;
Estimating required capacity (step 2)...&lt;br /&gt;
Estimated hash table requirement: 73390180936 bytes Capacity estimation complete. [1h53m11.000s]&lt;br /&gt;
Building database files (step 3)... Hash table already present, skipping database file build. Database construction complete. [Total: 1h53m11.000s]&lt;/p&gt;

&lt;p&gt;Then, I run kraken with one of my sample against this new updated plants db (I did not change the name of the db, is still called plant_original):&lt;/p&gt;

&lt;p&gt;&lt;code&gt;kraken2 --db PATH/kraken/plant_original --threads 8 --confidence 0.1 --report PATH/SB0new_report.txt PATH/SB0.fastq.gz --report-zero-counts --output PATH/SB0new_taxa.txt&lt;/code&gt;&lt;/p&gt;

&lt;p&gt;The new kraken2 report is exactly the same as the old one and it did not find any hit to these new added taxa.&lt;/p&gt;

&lt;p&gt;Previously, I did a blastn alignment of these sample‚Äôs reads with these exact same taxa (I created a small db with alnus&lt;em&gt;glutinosa&lt;/em&gt;GCA&lt;em&gt;003254965.1.fna, carpinus&lt;/em&gt;fangiana&lt;em&gt;GCA&lt;/em&gt;006937295.1.fna etc..) and blastn could find some hits. So I was expecting that Kraken2 would have found these matches too.&lt;/p&gt;

&lt;p&gt;Are these steps wrong or am I making some conceptual mistake?&lt;/p&gt;

&lt;p&gt;I appreciate your help.&lt;/p&gt;

&lt;p&gt;Thanks&lt;/p&gt;
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Comments</div>
                                        3
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/DerrickWood/kraken2&#34;&gt;DerrickWood/kraken2&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:41:14">
            
                <img src="https://secure.gravatar.com/avatar/fb0fc2f6be7f8c8559971316afa12bd2.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Louis Kraft
                     <span class="print-only user-email">(lokraf@dtu.dk)</span>
                </div>
                <a href="#2023-11-02 14:41:14"><div class="time">2023-11-02 14:41:14</div></a>
                <div class="msg">
                    <p>Hi all,
I have been thinking about damage profiles and noticed that while the frequency of C-&gt;T substitutions due to deamination is significantly higher at the ends of DNA fragments, one can (often) observe a persistent (very low) frequency of these substitutions observed internally within the fragments, e.g. converging at around 1%.
Now I wonder why that is, because non-typical ancient DNA substitutions often have a baseline of 0%.
Could it be that C-&gt;T substitutions are indeed occurring at a low level throughout the DNA fragments, or might this be influenced by the inherent mutation rates of cytosine and thymine? Or could this pattern be an artifact of the damage estimation methods we use to estimate DNA damage?
Any thoughts?
Thanks! :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 14:49:15">
            
                <img src="https://avatars.slack-edge.com/2023-09-25/5969403070976_12b1e1e90c797643f2c8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Johnson
                     <span class="print-only user-email">(sarahjjohnson@ou.edu)</span>
                </div>
                <a href="#2023-11-02 14:49:15"><div class="time">2023-11-02 14:49:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Are these bacterial fragments or Eukaryotic?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:04:26">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 15:04:26"><div class="time">2023-11-02 15:04:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could also be that you‚Äôre mapping to a further away reference</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:04:57">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 15:04:57"><div class="time">2023-11-02 15:04:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is it only C-&gt;T, or also other substitutions ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:06:02">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2023-11-02 15:06:02"><div class="time">2023-11-02 15:06:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Check the 5' G-->A. If that is at the same level of the mid-molecule C-->T, then suggestive of evolutionary divergence from reference.</p>

<p>Or were these sequenced from single-stranded DNA libraries?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:06:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-02 15:06:29"><div class="time">2023-11-02 15:06:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And you don‚Äôt see it as much on human aDNA damage, because well, usually you know what species/‚Äústrain‚Äù you‚Äôre looking at üòâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:09:16">
            
                <img src="https://secure.gravatar.com/avatar/fb0fc2f6be7f8c8559971316afa12bd2.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Kraft
                     <span class="print-only user-email">(lokraf@dtu.dk)</span>
                </div>
                <a href="#2023-11-02 15:09:16"><div class="time">2023-11-02 15:09:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It was more of an overall observation, so I was not talking about my own data. I found examples of both, human and bacterial samples. So I guess you would agree, that it is a common thing one <em>can</em> observe in samples? üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Pete Heintzman
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-02 15:09:37">
            
                <img src="https://secure.gravatar.com/avatar/fb0fc2f6be7f8c8559971316afa12bd2.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis Kraft
                     <span class="print-only user-email">(lokraf@dtu.dk)</span>
                </div>
                <a href="#2023-11-02 15:09:37"><div class="time">2023-11-02 15:09:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for your fast answers btw!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-03 07:05:32">
            
                <img src="https://avatars.slack-edge.com/2022-10-27/4269387323127_52f8b64cd696f1c66ff3_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Magdalena Haller-Caskie
                     <span class="print-only user-email">(m.haller@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2023-11-03 07:05:32"><div class="time">2023-11-03 07:05:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good question, thank you for asking that üòâ I've come across this also some times and wondered</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-03 10:20:12">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2023-11-03 10:20:12"><div class="time">2023-11-03 10:20:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Wouldn't these be the traces of deaminated CpG positions? They would show up as C--&gt;T as well, wouldn't they?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-03 12:48:50">
            
                <img src="https://avatars.slack-edge.com/2023-09-25/5969403070976_12b1e1e90c797643f2c8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Johnson
                     <span class="print-only user-email">(sarahjjohnson@ou.edu)</span>
                </div>
                <a href="#2023-11-03 12:48:50"><div class="time">2023-11-03 12:48:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That‚Äôs what I was thinking of it was eukaryotic DNA. There could be a variety of reasons depending on the context, library preparation, and sequencing. </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-08 09:19:33">
            
                <img src="https://avatars.slack-edge.com/2023-11-08/6155952966038_ea9a340a7339b0b7a88e_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nathalie Suarez Gonzalez
                     <span class="print-only user-email">(nathalie.suarez.gonzalez@ulb.be)</span>
                </div>
                <a href="#2023-11-08 09:19:33"><div class="time">2023-11-08 09:19:33</div></a>
                <div class="msg">
                    <p>Hi everyone,
I am looking for help regarding the use of KrakenUniq. Does anyone have experience with this software? I am planning to run samples containing between 19.000.000 and 40.000.000 reads (150PE) using the MicrobialDB (384GB). According to a first trial and my estimations, each sample will take between 10 to 20 days to be classify!!! I am copying the DB to the local disk before the run and use the script below. The same samples using Kraken2 only need a few hours to be classify. Does anybody have any suggestion?
I thank you all in advance.
Nathalie</p>

<h1>SBATCH --ntasks=1</h1>

<h1>SBATCH --cpus-per-task=1</h1>

<h1>SBATCH --mem-per-cpu=1G</h1>

<h1>SBATCH --partition=batch</h1>

<p>echo Start job and use local scratch
date
ml releases/2022a
ml KrakenUniq/1.0.3-GCC-11.3.0</p>

<p>data<em>dir=$LOCALSCRATCH/KrakenUniq
upload</em>dir=./Upload
echo Copy data base in local disk $data<em>dir
mkdir -p $data</em>dir
srun cp ./KrakenUniq/kuniq<em>microbialdb.kdb.20200816.tgz ./KrakenUniq/database.kdb $data</em>dir/.
cd $data<em>dir
echo Extract data
srun tar -xzf kuniq</em>microbialdb.kdb.20200816.tgz
cd -</p>

<p>files=( "sample1.pair1.truncated.fq.gz" # input file 1
       "sample1.pair2.truncated.fq.gz" # input file 2
       "sample1krakenuniqClass"        # out file
       "sample2.pair1.truncated.fq.gz"
       "sample2.pair2.truncated.fq.gz"
       "sample2krakenuniqClass"
       )</p>

<p>ArrayLength=${#files[@]}
n=1
for ((i = 0 ; i &lt; ArrayLength ; i++ )); do
   job<em>name="${SLURM</em>JOB<em>NAME}-$n"
   file</em>in1=${files[$i]}
   file<em>in2=${files[(++i)]}
   file</em>out=${files[(++i)]}
   date
   echo Start task $n ...</p>

<p>srun -J "${job<em>name}" krakenuniq --db $data</em>dir --paired --classified-out classified<em>sequences-${job</em>name}.txt ${upload<em>dir}/${file</em>in1} ${upload<em>dir}/${file</em>in2} &gt; ${upload<em>dir}/${file</em>out}-${job_name}</p>

<p>echo "End tasks $n"
   date
   ((n++))
done</p>

<p>echo End all job
echo Remove local data
srun rm -rf $data_dir
date</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-08 15:57:20">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-08 15:57:20"><div class="time">2023-11-08 15:57:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, KrakenUniq definitely takes longer than Kraken2 but not that much! Are you able to increase the memory for the job? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-08 17:51:50">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-11-08 17:51:50"><div class="time">2023-11-08 17:51:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That is what I was going to recommend too. Not sure what capacity of RAM that you have but maybe you can change --mem to a larger number than the default in your current system?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-08 22:41:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-08 22:41:49"><div class="time">2023-11-08 22:41:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Exactly, give it one of the largest nodes and it will run much faster. I‚Äôm using 1TB nodes and it runs normally in less than a day</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-09 12:54:06">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-09 12:54:06"><div class="time">2023-11-09 12:54:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nathalie Suarez Gonzalez given identical RAM available (for example, a 512GB compute node for your 384GB microbial DB), Kraken2 will be (much) faster than KrakenUniq. However, if you have only e.g. 256GB node available, Kraken2 would not be possible to run with your 384GB DB, but KrakenUniq can be run, it will be slow though, i.e. slower than if you had 512GB of RAM available</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-09 12:58:10">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-09 12:58:10"><div class="time">2023-11-09 12:58:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also @Nathalie Suarez Gonzalez, KrakenUniq starts very slow, but accelerates dramatically towards the end (for unclear reasons, I guess is starts with longest reads first, and quickly maps the shortest reads at the end). From my experience, provided that you have enough RAM to accommodate your 384GB microbial DB (for example, if you have a 512GB compute node), and you enable multi-threading in KrakenUniq, 19 mln - 40 mln reads should not take more than 6-8 hours.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-09 15:28:32">
            
                <img src="https://avatars.slack-edge.com/2023-11-08/6155952966038_ea9a340a7339b0b7a88e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nathalie Suarez Gonzalez
                     <span class="print-only user-email">(nathalie.suarez.gonzalez@ulb.be)</span>
                </div>
                <a href="#2023-11-09 15:28:32"><div class="time">2023-11-09 15:28:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you very much for your input üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 14:33:31">
            
                <img src="https://avatars.slack-edge.com/2023-10-11/6008974172311_f11038f168a5e02fad5d_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Anastasia Brativnyk
                     <span class="print-only user-email">(anastasia.brativnyk@crick.ac.uk)</span>
                </div>
                <a href="#2023-11-16 14:33:31"><div class="time">2023-11-16 14:33:31</div></a>
                <div class="msg">
                    <p>Hey everyone, I am new to the topic of metagenomics and have many questions regarding this topic. I would like to filter kraken report based on a number of unique k-mers before deciding what bacteria or virus to focus on. I do not want to miss anything important but also would like to remove potential noise from my kraken report. What would be appropriate lower threshold for number of k-mers to extract? Some viruses have very small genomes so I cannot expect lots of unique k-mers for those...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 14:48:20">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-16 14:48:20"><div class="time">2023-11-16 14:48:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We use n<em>unique</em>kmers=1000 in aMeta, this works well for bactetial genomes, but it is probably too conservative for viruses. I believe, n<em>unique</em>kmers / L<em>genome is a better filter, but I do not have a good threshold in mind. I would plot distributions of n</em>unique<em>kmers / L</em>genome for each sample (which will hopefully be bimodal) before deciding on exact threshold</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 14:52:12">
            
                <img src="https://avatars.slack-edge.com/2023-10-11/6008974172311_f11038f168a5e02fad5d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anastasia Brativnyk
                     <span class="print-only user-email">(anastasia.brativnyk@crick.ac.uk)</span>
                </div>
                <a href="#2023-11-16 14:52:12"><div class="time">2023-11-16 14:52:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, thank you, I read your article on aMeta and that default threshold sounds to hight, so that I why I decided to ask. Ok, I like very much your suggestion, but is there genome ref length stated somewhere in the kraken report? I have kraken2 report so should be similar to KrakenUnique.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 14:59:38">
            
                <img src="https://avatars.slack-edge.com/2023-10-11/6008974172311_f11038f168a5e02fad5d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anastasia Brativnyk
                     <span class="print-only user-email">(anastasia.brativnyk@crick.ac.uk)</span>
                </div>
                <a href="#2023-11-16 14:59:38"><div class="time">2023-11-16 14:59:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Otherwise, it would be a pain to look up every genome in the data base to fetch its length.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 15:03:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-16 15:03:14"><div class="time">2023-11-16 15:03:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Genome lengths you have to compute yourself by something like ‚Äùwc -c ref.fa‚Äù</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 15:19:12">
            
                <img src="https://avatars.slack-edge.com/2023-10-11/6008974172311_f11038f168a5e02fad5d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anastasia Brativnyk
                     <span class="print-only user-email">(anastasia.brativnyk@crick.ac.uk)</span>
                </div>
                <a href="#2023-11-16 15:19:12"><div class="time">2023-11-16 15:19:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you Nikolay, I will give it a try:)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 16:02:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-16 16:02:46"><div class="time">2023-11-16 16:02:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I like to look at the ratio between k-mers and reads for shotgun data.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-16 16:04:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-16 16:04:29"><div class="time">2023-11-16 16:04:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It‚Äôs a good indicator that the reads are distributed randomly over the genome if there are much more k-mers than reads. If there are more reads than k-mers it means the reads are probably stacking (or that you sequenced a loooot or captured)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 13:16:20">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 13:16:20"><div class="time">2023-11-17 13:16:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which kraken ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 13:16:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 13:16:32"><div class="time">2023-11-17 13:16:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* KrakenUniq, or Kraken2 ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 13:19:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 13:19:09"><div class="time">2023-11-17 13:19:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve came up with this for KrakenUniq
<a href="https://maximeborry.com/post/kraken-uniq/">https://maximeborry.com/post/kraken-uniq/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Maxime Borry</div>
                            
                            
                                
                                <div class="link-title"><a href="https://maximeborry.com/post/kraken-uniq/">A new E-score for KrakenUniq | Maxime Borry</a></div>
                                <div class="link-text">
                                    How to distinguish between true and false positive assignations with KrakenUniq
                                </div>
                                
                                
    
        <a href="https://maximeborry.com/post/kraken-uniq/">
            <img class="preview" src="https://maximeborry.com/post/kraken-uniq/featured.png"
                width="731"
                 height="486" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://maximeborry.com/post/kraken-uniq/</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëÜ Zo√© Pochon, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 13:23:41">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 13:23:41"><div class="time">2023-11-17 13:23:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For kraken2 it‚Äôs slightly different, because it doesn‚Äôt compute a ‚Äúcoverage‚Äù</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 14:09:55">
            
                <img src="https://avatars.slack-edge.com/2023-10-11/6008974172311_f11038f168a5e02fad5d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Anastasia Brativnyk
                     <span class="print-only user-email">(anastasia.brativnyk@crick.ac.uk)</span>
                </div>
                <a href="#2023-11-17 14:09:55"><div class="time">2023-11-17 14:09:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am using results from Kraken2, but is has 8 columns, so assuming the same ouput as KrakenUniq...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 14:43:40">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 14:43:40"><div class="time">2023-11-17 14:43:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For Kraken2, I worked out something similar, a score <code>S</code> where a high values S is more likely to be false positive, and low values of <code>S</code> is less likely to be false positive.
With <code>d = nb_minimizer/nb_unique_minimizer</code> and <code>r=nb_reads</code></p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F0661TP5D9U/screenshot_2023-11-17_at_14.40.13.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-11-17 at 14.40.13.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F0661TP5D9U/screenshot_2023-11-17_at_14.40.13.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F0661TP5D9U-6f5437acda/screenshot_2023-11-17_at_14.40.13_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="134" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 15:35:19">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 15:35:19"><div class="time">2023-11-17 15:35:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which gives you something like this (the color scale is the value of S)</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="None"></a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 15:39:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-17 15:39:35"><div class="time">2023-11-17 15:39:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Btw @Maxime Borry, regarding the blog post about the new e-score, do you have a suggestion of threshold for it as well?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 15:39:43">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 15:39:43"><div class="time">2023-11-17 15:39:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066XSRBHCY/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F066XSRBHCY/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F066XSRBHCY-b1a017573f/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="253" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 15:40:00">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 15:40:00"><div class="time">2023-11-17 15:40:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (edit: x axis more readable)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 15:45:17">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2023-11-17 15:45:17"><div class="time">2023-11-17 15:45:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Zo√© Pochon it‚Äôs the tricky balance between precision and recall, but looking towards the end of my blog post, you can see a range of values that would work</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 21:58:52">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-17 21:58:52"><div class="time">2023-11-17 21:58:52</div></a>
                <div class="msg">
                    <p>Hi everyone, fastq files demultiplexed are ready to use the aMeta pipelines, right?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-17 22:18:13">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-17 22:18:13"><div class="time">2023-11-17 22:18:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and can it run in a standard laptop? (or better in university's hpc?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-18 06:40:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-18 06:40:52"><div class="time">2023-11-18 06:40:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Yuti! Yes I think so! However, if it is paired-end data, you still have to merge it first cause we don't have that option.
It's made to be ran on an hpc server. If you search the github page, you'll see a link to a suggested way to set it up on hpc servers.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-18 10:55:46">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-18 10:55:46"><div class="time">2023-11-18 10:55:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Yuti Gao, yes, raw reads (demultiplexed) in fastq-format are the standard input for aMeta. You can install and test aMeta on your laptop, however, for real world applications you will have to use big databases (we published a bunch together with aMeta paper, and you can find the links at the very bottom of README here <a href="https://github.com/NBISweden/aMeta">https://github.com/NBISweden/aMeta</a>). The big databases can efficiently be handled on an HPC. And the size (and RAM requirements) of the databases are not a peculiarity of aMeta itself but the metagenomic analysis in general which I doubt can be run on a laptop (any metagenomic analysis) üôÇ</p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">NBISweden/aMeta</a></div>
                                <div class="link-text">
                                    Ancient microbiome snakemake workflow
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        16
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Python
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 21:06:40">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-20 21:06:40"><div class="time">2023-11-20 21:06:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks @Zo√© Pochon,  yes, it is pair-end data, do you know any written pipelines to do the merge?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 21:08:30">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-20 21:08:30"><div class="time">2023-11-20 21:08:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks @Nikolay Oskolkov, I was following the textbook section about aMeta, <a href="https://www.spaam-community.org/intro-to-ancient-metagenomics-book/ancient-metagenomic-pipelines.html#what-is-ameta">https://www.spaam-community.org/intro-to-ancient-metagenomics-book/ancient-metagenomic-pipelines.html#what-is-ameta</a>, they looks like the same, is it?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 21:15:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-20 21:15:01"><div class="time">2023-11-20 21:15:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao yes, the textbook chapter is good to follow. Regarding merging overlapping PE reads I could tecommend fastp or SeqPrep</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 21:16:07">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-20 21:16:07"><div class="time">2023-11-20 21:16:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Otherwise simply ‚Äùcat R1.fastq.gz R1.fastq.gz &gt; merged.fastq.gz‚Äù would also work as input for aMeta</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 21:17:44">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-20 21:17:44"><div class="time">2023-11-20 21:17:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* cat R1.fastq.gz R2.fastq.gz &gt; merged.fastq.gz?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 21:18:38">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-20 21:18:38"><div class="time">2023-11-20 21:18:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok! that's easy to follow!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-20 22:18:29">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-20 22:18:29"><div class="time">2023-11-20 22:18:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would go with <em>fastp</em> so that you merge potentially overlapping pairs of reads together instead of just concatenating the files in one file.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Yuti Gao
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:19:47">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:19:47"><div class="time">2023-11-21 20:19:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* when I run to test the installation,
<code>cd .test
./runtest.sh -j 4</code>
get an error,
Error in rule KrakenUniq:</p>

<h1>krakenuniq: database ("resources/KrakenUniq_DB") does not contain necessary file database.kdb</h1>

<p>Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-11-21T121632.600005.snakemake.log</p>

<p>do you have any suggestions?</p>

<p>thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:25:52">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:25:52"><div class="time">2023-11-21 20:25:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao could you please post the log-file <code>.snakemake/log/2023-11-21T121632.600005.snakemake.log</code> here?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:32:26">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:32:26"><div class="time">2023-11-21 20:32:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066L02CWQ6/2023-11-21t121632.600005.snakemake.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2023-11-21T121632.600005.snakemake.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:32:46">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:32:46"><div class="time">2023-11-21 20:32:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066V2REXLL/report.html?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">report.html</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:38:16">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:38:16"><div class="time">2023-11-21 20:38:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao could you please do</p>

<ol><li>rm -rf .test/results</li><li>rm .test/resources/KrakenUniq_DB/database**
and restart the testrun via <code>./runtest.sh -j 4</code> ?</li>
</ol>

<p>It looks like you tried to execute it a few times after something has failed in the first run</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:38:47">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:38:47"><div class="time">2023-11-21 20:38:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:40:10">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:40:10"><div class="time">2023-11-21 20:40:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (aMeta) [yuga3894@c3cpu-c11-u3-3 .test]$ rm .test/resources/KrakenUniq<em>DB/database
rm: cannot remove '.test/resources/KrakenUniq</em>DB/database': No such file or directory
(aMeta) [yuga3894@c3cpu-c11-u3-3 .test]$ cd resources/
(aMeta) [yuga3894@c3cpu-c11-u3-3 resources]$ ls
accession2taxid.map  ref.fa.2.bt2l       ref.fa.rev.2.bt2l
KrakenUniq<em>DB     ref.fa.3.bt2l       samples.tsv
pathogenomesFound.tab ref.fa.4.bt2l       seqid2taxid.pathogen.map
ref.fa         ref.fa</em>BOWTIE2<em>BUILD.log
ref.fa.1.bt2l     ref.fa.rev.1.bt2l
(aMeta) [yuga3894@c3cpu-c11-u3-3 resources]$ cd KrakenUniq</em>DB
(aMeta) [yuga3894@c3cpu-c11-u3-3 KrakenUniq_DB]$ ls
library seqid2taxid.map taxonomy</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:40:28">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:40:28"><div class="time">2023-11-21 20:40:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* looks like I don't have database within KrankenUniq</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:43:32">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:43:32"><div class="time">2023-11-21 20:43:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao please note the asterisk ** here: <code>rm .test/resources/KrakenUniq_DB/database**</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:43:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:43:59"><div class="time">2023-11-21 20:43:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao good, please rerun the testrun</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:43:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-21 20:43:59"><div class="time">2023-11-21 20:43:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I'm not in front of my computer, but I would say go into the .test folder, open the script (file finishing with .sh) and run what had to be run within the condition by hand. Sometimes the condition for the first run doesn't seem to activate depending on the environment to my experience.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:55:15">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:55:15"><div class="time">2023-11-21 20:55:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* tried both, get same error when I do ./runtest.sh -j 4,  no red error when I just do bash runtest.sh, but a lot of "No validator" and missing metadata, what should be the output?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:55:17">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 20:55:17"><div class="time">2023-11-21 20:55:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Running workflow...
snakemake --use-conda --conda-frontend mamba --show-failed-logs --conda-cleanup-pkgs cache -s ../workflow/Snakefile
Error: you need to specify the maximum number of CPU cores to be used at the same time. If you want to use N cores, say --cores N or -cN. For all cores on your system (be sure that this is appropriate) use --cores all. For no parallelization use --cores 1 or -c1. &lt;<em>io.TextIOWrapper name='&lt;stderr&gt;' mode='w' encoding='utf-8'&gt;
Generating report...
snakemake -s ../workflow/Snakefile --report --report-stylesheet ../workflow/report/custom.css
No validator found for JSON Schema version identifier '<a href="http://json-schema.org/draft/2020-12/schema#">http://json-schema.org/draft/2020-12/schema#</a>'
Defaulting to validator for JSON Schema version '<a href="https://json-schema.org/draft/2020-12/schema">https://json-schema.org/draft/2020-12/schema</a>'
Note that schema file may not be validated correctly.
Excluding samples 'foobar' from analysis
Restricting analysis to samples 'foo','bar'
No validator found for JSON Schema version identifier '<a href="http://json-schema.org/draft/2020-12/schema#">http://json-schema.org/draft/2020-12/schema#</a>'
Defaulting to validator for JSON Schema version '<a href="https://json-schema.org/draft/2020-12/schema">https://json-schema.org/draft/2020-12/schema</a>'
Note that schema file may not be validated correctly.
Changing directory from /projects/yuga3894/aMeta/.test to /projects/yuga3894/aMeta/workflow
Changing directory back to /projects/yuga3894/aMeta/.test
Building DAG of jobs...
Creating report...
Missing metadata for file results/MULTIQC/multiqc</em>report.html. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/BOWTIE2/foo/AlignedToBowtie2DB.bam. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/BOWTIE2/foo/AlignedToBowtie2DB.bam.bai. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/BOWTIE2/bar/AlignedToBowtie2DB.bam. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/BOWTIE2/bar/AlignedToBowtie2DB.bam.bai. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MAPDAMAGE/foo. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/krakenuniq.output.filtered. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/krakenuniq.output.pathogens. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/taxID.pathogens. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/taxID.species. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/krakenuniq.output. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/sequences.krakenuniq. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MAPDAMAGE/bar. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/krakenuniq.output.filtered. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/krakenuniq.output.pathogens. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/taxID.pathogens. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/taxID.species. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/krakenuniq.output. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/sequences.krakenuniq. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/krakenuniq.output.filtered<em>taxIDs</em>kmers1000.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/sequences.krakenuniq<em>kmers1000.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/sequences.krakenuniq</em>kmers1000.krona. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/foo/taxonomy.krona.html. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/krakenuniq.output.filtered<em>taxIDs</em>kmers1000.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/sequences.krakenuniq<em>kmers1000.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/sequences.krakenuniq</em>kmers1000.krona. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ/bar/taxonomy.krona.html. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>ABUNDANCE</em>MATRIX<em>SAM. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT</em>ABUNDANCE<em>MATRIX</em>SAM/malt<em>abundance</em>matrix<em>sam.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT</em>QUANTIFY<em>ABUNDANCE/foo. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT</em>QUANTIFY<em>ABUNDANCE/foo/sam</em>counts.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT/foo.trimmed.rma6. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT/foo.trimmed.sam.gz. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>DB/seqid2taxid.project.map. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT</em>DB/seqids.project. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>DB/project.headers. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT</em>DB/library.project.fna. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>DB/maltDB.dat. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ</em>ABUNDANCE<em>MATRIX. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ</em>ABUNDANCE<em>MATRIX/unique</em>species<em>taxid</em>list.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ<em>ABUNDANCE</em>MATRIX/unique<em>species</em>names<em>list.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ</em>ABUNDANCE<em>MATRIX/krakenuniq</em>abundance<em>matrix.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/KRAKENUNIQ</em>ABUNDANCE<em>MATRIX/krakenuniq</em>absolute<em>abundance</em>heatmap.pdf. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>QUANTIFY</em>ABUNDANCE/bar. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>QUANTIFY</em>ABUNDANCE/bar/sam<em>counts.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT/bar.trimmed.rma6. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT/bar.trimmed.sam.gz. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT</em>ABUNDANCE<em>MATRIX</em>RMA6. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/MALT<em>ABUNDANCE</em>MATRIX<em>RMA6/malt</em>abundance<em>matrix</em>rma6.txt. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/AUTHENTICATION/foo/.extract<em>taxids</em>done. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/AUTHENTICATION/bar/.extract<em>taxids</em>done. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/overview<em>heatmap</em>scores.pdf. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/AUTHENTICATION/.foo<em>done. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Missing metadata for file results/AUTHENTICATION/.bar</em>done. Maybe metadata was deleted or it was created using an older version of Snakemake. This is a non critical warning.
Downloading resources and rendering HTML.
Report created: report.html.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:00:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 21:00:59"><div class="time">2023-11-21 21:00:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao did you remove the .test/results prior to rerunning the testrun?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:01:14">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 21:01:14"><div class="time">2023-11-21 21:01:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* also, after I installed aMeta in hpc i just see there is a section "Environment module configuration" in the github, should I do this part before I run the test</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:01:30">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 21:01:30"><div class="time">2023-11-21 21:01:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes @Nikolay Oskolkov</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:04:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 21:04:18"><div class="time">2023-11-21 21:04:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao no, wait a bit with the environment module configuration. Can you please show me the content of .test/results?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:09:33">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 21:09:33"><div class="time">2023-11-21 21:09:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (aMeta) [yuga3894@c3cpu-c11-u19-3 results]$ ls
CUTADAPT<em>ADAPTER</em>TRIMMING FASTQC<em>AFTER</em>TRIMMING FASTQC<em>BEFORE</em>TRIMMING</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:11:43">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 21:11:43"><div class="time">2023-11-21 21:11:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao I am afraid it still did not work üòû Could you please delete aMeta installation like this:</p>

<ol><li>rm -rf aMeta <b>#remove</b> the whole aMeta folder</li><li>conda remove -n aMeta --all <b>#remove</b> aMeta environment
and just redo everything from scratch, i.e. cloning of aMeta repo, creating aMeta environment and running the testrun. And please post here the very first issue you encounter (the log-file would be ideal)</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 22:42:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-21 22:42:10"><div class="time">2023-11-21 22:42:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao sorry that you encountered this problem.
When running the test file for the first time, keep an eye open to see if it says something like :"This looks like the first run". If not it means it is somehow not gonna create the necessary test files before running the test.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 23:18:35">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 23:18:35"><div class="time">2023-11-21 23:18:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I reinstalled it, notice this message:</p>

<p>Krona installed. You still need to manually update the taxonomy
databases before Krona can generate taxonomic reports. The update
script is ktUpdateTaxonomy.sh. The default location for storing
taxonomic databases is /projects/yuga3894/software/anaconda/envs/aMeta/opt/krona/taxonomy</p>

<p>should I run ktUpdateTaxonomy.sh before I run the test?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 23:47:27">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 23:47:27"><div class="time">2023-11-21 23:47:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I did get the first run message,</p>

<p>(aMeta) [yuga3894@c3cpu-c15-u9-1 .test]$ ./runtest.sh -j 4
This looks like the first test run... Installing bioconda packages...</p>

<p>...
Your conda installation is not configured to use strict channel priorities. This is however crucial for having robust and correct environments (for details, see <a href="https://conda-forge.org/docs/user/tipsandtricks.html">https://conda-forge.org/docs/user/tipsandtricks.html</a>). Please consider to configure strict priorities by executing 'conda config --set channel_priority strict'.</p>

<p>and looks like eveything goes well until</p>

<p>MissingOutputException in rule KrakenUniq in file /projects/yuga3894/aMeta/workflow/rules/krakenuniq.smk, line 1:
Job 16 completed successfully, but some output files are missing. Missing files after 5 seconds. This might be due to filesystem latency. If that is the case, consider to increase the wait time with --latency-wait:
results/KRAKENUNIQ/bar/sequences.krakenuniq</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 23:47:33">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-21 23:47:33"><div class="time">2023-11-21 23:47:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066TDZ38JX/2023-11-21t153832.206006.snakemake.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2023-11-21T153832.206006.snakemake.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 09:22:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 09:22:40"><div class="time">2023-11-22 09:22:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Yuti Gao, thank for the very detailed reporting, a few comments:</p>

<ol><li>The "no validator found" warning is fine, please ignore it</li><li>The "Krona installed. You still need to manually update the taxonomy" message is also not very relevant for you at this stage (it will become important later), so please ignore it as well.</li><li>The message "Your conda installation is not configured to use strict channel priorities" should also be ignored. Snakemake is ultra-verbose üôÇ </li><li>The log-file from the testrun you sent does not seem to be the log from the very first run. I would like to see the yellow messages regarding installing environments, something like:
<code>Creating conda environment ../workflow/envs/malt.yaml...
Downloading and installing remote packages.</code></li><li>Immediately after the yellow messages about installing environments, there will be some white text concerning building a KrakenUniq_DB on the fly, something like:</li>
</ol>

<p><code>Building krakenuniq data
Kraken build set to minimize disk writes.
Finding all library files
Found 1 sequence files (**.{fna,fa,ffn,fasta,fsa}) in the library directory.
Creating k-mer set (step 1 of 6)...</code>
This step seems to fail for you and I need to see the log-file to understand why.</p>

<p>In summary, could you please do again:</p>

<ol><li>rm -rf aMeta <b>#remove</b> the whole aMeta folder</li><li>conda remove -n aMeta --all <b>#remove</b> aMeta environment</li><li>git clone <a href="https://github.com/NBISweden/aMeta">https://github.com/NBISweden/aMeta</a></li><li>cd aMeta</li><li>mamba env create -f workflow/envs/environment.yaml</li><li>conda activate aMeta</li><li>cd .test</li><li>./runtest.sh -j 4
and send me the log-file? Please do not restart the testrun if you suspect that something is not going right, I need to see the log-file from the very first run (failed or successful). The log-file can be found following the message at the very end, something like this "Complete log: .snakemake/log/2023-11-22T091420.873756.snakemake.log"</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 10:40:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-22 10:40:40"><div class="time">2023-11-22 10:40:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is there an .initdb file in your .test folder after running the test?
It would mean that it really went through the first run condition all the way. If not, I would take the code from the condition and run it on the terminal, from within the .test folder :</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 10:42:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-22 10:42:06"><div class="time">2023-11-22 10:42:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```echo "This looks like the first test run... Installing bioconda packages..."
        snakemake --use-conda --show-failed-logs -j 2 --conda-cleanup-pkgs cache --conda-create-envs-only -s ../workflow/Snakefile</p>

<pre><code>    source $(dirname $(dirname $CONDA_EXE))/etc/profile.d/conda.sh

    ##############################
    # Krakenuniq database
    ##############################
    echo Building krakenuniq data
    env=$(grep krakenuniq .snakemake/conda/**yaml | awk '{print $1}' | sed -e "s/.&amp;lt;yaml://g&amp;gt;")
    conda activate $env
    krakenuniq-build --db resources/KrakenUniq_DB --kmer-len 21 --minimizer-len 11 --jellyfish-bin $(pwd)/$env/bin/jellyfish
    conda deactivate

    ##############################
    # Krona taxonomy
    ##############################
    echo Building krona taxonomy
    env=$(grep krona .snakemake/conda/**yaml | awk '{print $1}' | sed -e "s/.&amp;lt;yaml://g&amp;gt;" | head -1)
    conda activate $env
    cd $env/opt/krona
    ./updateTaxonomy.sh taxonomy
    cd -
    conda deactivate

    ##############################
    # Adjust malt max memory usage
    ##############################
    echo Adjusting malt max memory usage
    env=$(grep hops .snakemake/conda/**yaml | awk '{print $1}' | sed -e "s/.&amp;lt;yaml://g&amp;gt;" | head -1)
    conda activate $env
    version=$(conda list malt --json | grep version | sed -e "s/\"//g" | awk '{print $2}')
    cd $env/opt/malt-$version
    sed -i -e "s/-Xmx64G/-Xmx3G/" malt-build.vmoptions
    sed -i -e "s/-Xmx64G/-Xmx3G/" malt-run.vmoptions
    cd -
    conda deactivate

    touch .initdb```
</code></pre>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:42:15">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 17:42:15"><div class="time">2023-11-22 17:42:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066Y00129Z/log.txt?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">log.txt</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:44:11">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 17:44:11"><div class="time">2023-11-22 17:44:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov I found my terminal record after this:
<code>Creating conda environment ../workflow/envs/malt.yaml...
Downloading and installing remote packages.</code>
if this doesn'r work, I will rerun the installation</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:47:50">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 17:47:50"><div class="time">2023-11-22 17:47:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thanks for you help!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:51:28">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 17:51:28"><div class="time">2023-11-22 17:51:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Zo√© Pochon thanks, is the .initdb file directly under the /test folder, looks I don't have it then
[yuga3894@login13 .test]$ ls
benchmarks config data logs report.html resources results runtest.sh</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 18:03:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2023-11-22 18:03:20"><div class="time">2023-11-22 18:03:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ls -a
To see the hidden files that begin with a dot</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 18:04:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 18:04:40"><div class="time">2023-11-22 18:04:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuti Gao from the the log-file you posted it seems your KrakenUniq_DB was correctly built, so it seems to me that you should have had the database-files, which you previously reported missing. I suggest we have a clean run, so could you please do these steps and post the log-file here:</p>

<p>In summary, could you please do again:</p>

<ol><li>rm -rf aMeta <b>#remove</b> the whole aMeta folder</li><li>conda remove -n aMeta --all <b>#remove</b> aMeta environment</li><li>git clone <a href="https://github.com/NBISweden/aMeta">https://github.com/NBISweden/aMeta</a></li><li>cd aMeta</li><li>mamba env create -f workflow/envs/environment.yaml</li><li>conda activate aMeta</li><li>cd .test</li><li>./runtest.sh -j 4</li>
</ol>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">NBISweden/aMeta</a></div>
                                <div class="link-text">
                                    Ancient microbiome snakemake workflow
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        16
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Python
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Yuti Gao
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 19:26:41">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 19:26:41"><div class="time">2023-11-22 19:26:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066YH6N4KD/log_11.25am?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">log 11.25am</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 19:27:07">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 19:27:07"><div class="time">2023-11-22 19:27:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (aMeta) [yuga3894@c3cpu-a5-u28-1 .test]$ ls
benchmarks config data logs report.html resources results runtest.sh</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 19:27:33">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-22 19:27:33"><div class="time">2023-11-22 19:27:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* just remove everythign and reinstall, looks like I got the same error</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 08:58:16">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 08:58:16"><div class="time">2023-11-23 08:58:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you @Yuti Gao, this looks good, please do not tweak anything in aMeta right now, let me investigate it a bit. Your KrakenUniq_DB building process looks fine, so there must be something wrong with the KrakenUniq run itself. Could you please post here the log-file located in aMeta/.test/logs/KRAKENUNIQ/bar.log?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:34:50">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:34:50"><div class="time">2023-11-23 17:34:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:34:56">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:34:56"><div class="time">2023-11-23 17:34:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F067FSH8E8H/bar.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">bar.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:36:56">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:36:56"><div class="time">2023-11-23 17:36:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, thank you @Yuti Gao, this explains it!</p>

<p><code>NOTE: No need to use --gzip-compressed or --bzip2-compressed anymore, format is detected automatically.
NOTE: No need to use --fasta-input or --fastq-input anymore, format is detected automatically.
/projects/yuga3894/aMeta/.test/.snakemake/conda/74c9aeceba45102acd1e47590c34ff6d_/share/krakenuniq-1.0.4-1/libexec/classify -d resources/KrakenUniq_DB/database.kdb -i resources/KrakenUniq_DB/database.idx -t 4 -o results/KRAKENUNIQ/bar/sequences.krakenuniq -c -M -r results/KRAKENUNIQ/bar/krakenuniq.output -a resources/KrakenUniq_DB/taxDB -p 12
classify: thread count exceeds number of processors</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:38:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:38:11"><div class="time">2023-11-23 17:38:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I believe you do not have 4 processors available, so you should run it as
<code>./runtest.sh -j 1</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:38:26">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:38:26"><div class="time">2023-11-23 17:38:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ohh!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:38:52">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:38:52"><div class="time">2023-11-23 17:38:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* should I talk to hpc staff to get 4?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:39:45">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:39:45"><div class="time">2023-11-23 17:39:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Are you using login node of your HPC or did you reserve a node (with several CPUs) for running aMeta?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:40:54">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:40:54"><div class="time">2023-11-23 17:40:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* right now you should just try <code>./runtest.sh -j 1</code>. If this solves the issue, we can talk how to run aMeta on real data, then you certainly need nodes with many CPUs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:40:55">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:40:55"><div class="time">2023-11-23 17:40:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think it's login node now, I did get an account to reserve a node, although I don't know how to do it now</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:41:09">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:41:09"><div class="time">2023-11-23 17:41:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ok!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:48:20">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:48:20"><div class="time">2023-11-23 17:48:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F067S8TR4FJ/1123.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">1123.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:48:53">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:48:53"><div class="time">2023-11-23 17:48:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Error in rule Build<em>Malt</em>DB:
  jobid: 22
  input: results/KRAKENUNIQ<em>ABUNDANCE</em>MATRIX/unique<em>species</em>taxid<em>list.txt
  output: results/MALT</em>DB/seqid2taxid.project.map, results/MALT<em>DB/seqids.project, results/MALT</em>DB/project.headers, results/MALT<em>DB/library.project.fna, results/MALT</em>DB/maltDB.dat
  log: logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log (check log file(s) for error details)
  conda-env: /projects/yuga3894/aMeta/.test/.snakemake/conda/dcf57682cc9d8eaf05305773b2b3d994_
Logfile logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:52:47">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:52:47"><div class="time">2023-11-23 17:52:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, it seems that unfortunately your login node provides only very little RAM, something like 4-8 GB, which is not enough even for building this very small Malt database. @Yuti Gao would it be possible to to book a node with more resources?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:53:43">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:53:43"><div class="time">2023-11-23 17:53:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* alternatively, your laptop should have more resources compared to this HPC login node üôÇ so you might want to try installing aMeta on your laptop and learn the installation process</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:56:37">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:56:37"><div class="time">2023-11-23 17:56:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ohhh, ok then I would talk to hpc after people come back from this thanksgiving holiday, yes, I can book a node with more RAM.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:57:07">
            
                <img src="https://avatars.slack-edge.com/2022-08-05/3904450283828_0be1ac4078bb36a33df2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuti Gao
                     <span class="print-only user-email">(yuga3894@colorado.edu)</span>
                </div>
                <a href="#2023-11-23 17:57:07"><div class="time">2023-11-23 17:57:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thank you Nikolay for the help these days!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 18:02:34">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 18:02:34"><div class="time">2023-11-23 18:02:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You are welcome @Yuti Gao, and I am very sorry for all the troubles! Eventually, it looks like the problem with KrakenUniq was due to the lack of available threads on the login node (so we should put <code>./runtest.sh -j 1</code> to aMeta's README in order to avoid it in the future), and the problem with Malt was due to the lack of RAM on the login node.</p>

<p>Strategically, I would encourage you to learn how to book nodes with a lot of RAM in your HPC system, since aMeta will require much more RAM to use real databases on real-world data. Once you have booked a node with some reasonable RAM (at least 128-256GB, preferably more), please DM me and I will do my best to help you</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon, Yuti Gao
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 19:43:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 19:43:09"><div class="time">2023-11-21 19:43:09</div></a>
                <div class="msg">
                    <p>Hello! More of a conda issue than an aMeta one so I am not going to bother Zo√© and Nikolay directly in case anyone else has dealt with this error:
<code>ModuleNotFoundError in file /scratch/shreya23/aMeta/workflow/rules/common.smk, line 6:</code>
<code>No module named 'pytz.tzinfo'</code>
‚Ä¶but when I do <code>conda list</code> it seems like pytz is installed:
<code>pytz           2023.3.post1    pyhd8ed1ab_0  conda-forge</code>
and removing/uninstalling python and pytz doesn‚Äôt seem to fix it!</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:27:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:27:40"><div class="time">2023-11-21 20:27:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya strange, I have never seen this error, and it is puzzling that the line 6 in <code>common.smk</code> does not seem to directly deal with <code>pytz</code></p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F067578DUAV/screenshot_from_2023-11-21_20-17-29.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot from 2023-11-21 20-17-29.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F067578DUAV/screenshot_from_2023-11-21_20-17-29.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F067578DUAV-383d070cde/screenshot_from_2023-11-21_20-17-29_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="203" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:28:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:28:59"><div class="time">2023-11-21 20:28:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* When did this error popup? During installation or testrun or later?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:30:05">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:30:05"><div class="time">2023-11-21 20:30:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* During the test run! No issues popped up at installation.   </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:41:31">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:41:31"><div class="time">2023-11-21 20:41:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya are you sure you activate aMeta conda environment? If yes, what if you <code>pip install pytz</code> ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:42:53">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:42:53"><div class="time">2023-11-21 20:42:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôm pretty sure I activated it, but let me try again!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:47:39">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:47:39"><div class="time">2023-11-21 20:47:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Unfortunately <code>pip install pytz</code> gives me the same error. I‚Äôve got 2 separate conda installations I‚Äôm trying, one I did myself and one our admin installed on the cluster, and I get the same error for both. Here‚Äôs the full error:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066L2CA3JS/screenshot_2023-11-21_at_1.46.52_pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-11-21 at 1.46.52 PM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F066L2CA3JS/screenshot_2023-11-21_at_1.46.52_pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F066L2CA3JS-e4034666d3/screenshot_2023-11-21_at_1.46.52_pm_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="216" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:47:55">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:47:55"><div class="time">2023-11-21 20:47:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Perhaps this is a question for cluster admin then?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:49:59">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:49:59"><div class="time">2023-11-21 20:49:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, the line 6 in common.smk is importing pandas, and the pytz error has something to do with pandas from the screenshot you posted. Can you at all run <code>import pandas</code> if you start Python? Looks like something is wrong with pandas installation</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:51:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:51:56"><div class="time">2023-11-21 20:51:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>(aMeta) [shreya23@cri22cn094 .test]$ python
Python 3.10.13 | packaged by conda-forge | (main, Oct 26 2023, 18:07:37) [GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
&amp;gt;&amp;gt;&amp;gt; import pandas
Traceback (most recent call last):
  File "&amp;lt;stdin&amp;gt;", line 1, in &amp;lt;module&amp;gt;
  File "/home/shreya23/.conda/envs/aMeta/lib/python3.10/site-packages/pandas/__init__.py", line 46, in &amp;lt;module&amp;gt;
    from pandas.core.api import (
  File "/home/shreya23/.conda/envs/aMeta/lib/python3.10/site-packages/pandas/core/api.py", line 1, in &amp;lt;module&amp;gt;
    from pandas._libs import (
  File "/home/shreya23/.conda/envs/aMeta/lib/python3.10/site-packages/pandas/_libs/__init__.py", line 18, in &amp;lt;module&amp;gt;
    from pandas._libs.interval import Interval
  File "interval.pyx", line 1, in init pandas._libs.interval
  File "hashtable.pyx", line 1, in init pandas._libs.hashtable
  File "missing.pyx", line 1, in init pandas._libs.missing
  File "/home/shreya23/.conda/envs/aMeta/lib/python3.10/site-packages/pandas/_libs/tslibs/__init__.py", line 39, in &amp;lt;module&amp;gt;
    from pandas._libs.tslibs.conversion import localize_pydatetime
  File "conversion.pyx", line 1, in init pandas._libs.tslibs.conversion
  File "offsets.pyx", line 1, in init pandas._libs.tslibs.offsets
  File "timestamps.pyx", line 1, in init pandas._libs.tslibs.timestamps
  File "timedeltas.pyx", line 1, in init pandas._libs.tslibs.timedeltas
  File "timezones.pyx", line 32, in init pandas._libs.tslibs.timezones
ModuleNotFoundError: No module named 'pytz.tzinfo'</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:52:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:52:35"><div class="time">2023-11-21 20:52:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And then if I deactivate aMeta and just try on base:</p>

<p><code>(base) [shreya23@cri22cn094 .test]$ python
Python 3.10.11 | packaged by conda-forge | (main, May 10 2023, 18:58:44) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
&amp;gt;&amp;gt;&amp;gt; import pandas
Traceback (most recent call last):
 File "&amp;lt;stdin&amp;gt;", line 1, in &amp;lt;module&amp;gt;
ModuleNotFoundError: No module named 'pandas'</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:52:57">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 20:52:57"><div class="time">2023-11-21 20:52:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oho‚Ä¶ seems like something is up here! Thank you so much for helping me debug</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 20:57:51">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 20:57:51"><div class="time">2023-11-21 20:57:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya I would first try to install pandas in your base-environment with <code>pip install pandas</code> and then just delete aMeta installation like this:</p>

<ol><li>rm -rf aMeta <b>#remove</b> the whole aMeta folder</li><li>conda remove -n aMeta --all <b>#remove</b> aMeta environment
and just redo everething from scratch, i.e. cloning of aMeta repo, creating aMeta environment and running the testrun. This is just to be on a safe side</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:00:30">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 21:00:30"><div class="time">2023-11-21 21:00:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay!! Thank you so much Nikolay!! Will give this a shot and keep you posted!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:02:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-21 21:02:01"><div class="time">2023-11-21 21:02:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes @Shreya, we can have a zoom meeting with screen sharing if needed, and I will try to do my best to help you</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-21 21:02:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-21 21:02:33"><div class="time">2023-11-21 21:02:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you!! I really appreciate it! And I know it is rather late in Sweden right now!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 01:29:27">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 01:29:27"><div class="time">2023-11-22 01:29:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Nikolay!! Great news!! Installing pandas in base worked and now the workflow is starting!!</p>

<p>There is a new issue that I‚Äôm hoping is an easier fix? In running the test script I now get this error:
<code>MissingOutputException in rule Bowtie2_Index in file /scratch/shreya23/aMeta/workflow/rules/align.smk, line 1:
Job 9  completed successfully, but some output files are missing. Missing files after 5 seconds. This might be due to filesystem latency. If that is the case, consider to increase the wait time with --latency-wait:
resources/ref.fa.1.bt2l
resources/ref.fa.2.bt2l
resources/ref.fa.3.bt2l
resources/ref.fa.4.bt2l
resources/ref.fa.rev.1.bt2l
resources/ref.fa.rev.2.bt2l
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message</code>
I did edit the run script to add --latency-wait 60 (is that not long enough?) and got the same error. The files in <code>resources</code> are:
<code>accession2taxid.map
KrakenUniq_DB
pathogenomesFound.tab
ref.fa
ref.fa.1.bt2
ref.fa.2.bt2
ref.fa.3.bt2
ref.fa.4.bt2
ref.fa_BOWTIE2_BUILD.log
ref.fa.rev.1.bt2
ref.fa.rev.2.bt2
samples.tsv
seqid2taxid.pathogen.map</code>
so I am wondering if I need to rename the .bt2 files. to .bt2l? Thank you!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 09:32:47">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 09:32:47"><div class="time">2023-11-22 09:32:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Shreya, great that it worked with pandas and, yes, the Bowtie2_Index seems to be easier to fix. Could you please send me the file <code>ref.fa_BOWTIE2_BUILD.log</code> from <code>./test/resources</code>? Please do not modify the --latency-wait options, this is not the cause of the problem. You seem to have correctly built the Bowtie2 index for the <code>ref.fa</code>  reference but they remarkably (I have never seen this) have extensions <code>bt2</code> instead of <code>bt2l</code> . Even if, indeed, as you suggest, renaming the <em>*.bt2 files to *</em>.bt2l could help, you of course are not expected to do so in aMeta, and to give you a better advice I need to check the <code>ref.fa_BOWTIE2_BUILD.log</code>  file üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:42:04">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 16:42:04"><div class="time">2023-11-22 16:42:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay, I just purged all modules and ran from scratch and now I have a different bowtie error! Here it is:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:42:07">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 16:42:07"><div class="time">2023-11-22 16:42:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```Error in rule Bowtie2<em>Index:
    jobid: 9
    input: resources/ref.fa
    output: resources/ref.fa.1.bt2l, resources/ref.fa.2.bt2l, resources/ref.fa.3.bt2l, resources/ref.fa.4.bt2l, resources/ref.fa.rev.1.bt2l, resources/ref.fa.rev.2.bt2l
    log: resources/ref.fa</em>BOWTIE2<em>BUILD.log (check log file(s) for error details)
    conda-env: /scratch/shreya23/aMeta</em>test2/aMeta/.test/.snakemake/conda/6254db56e34ae5808d40831fc7518aca_
    shell:
        bowtie2-build-l --threads 1 resources/ref.fa resources/ref.fa &gt; resources/ref.fa<em>BOWTIE2</em>BUILD.log 2&gt;&amp;1
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)</p>

<h1>Logfile resources/ref.fa<em>BOWTIE2</em>BUILD.log:</h1>

<p>bowtie2<em>build</em>l: unrecognized option '--threads'
Bowtie 2 version 2.2.3 by Ben Langmead (<a href="mailto:langmea@cs.jhu.edu">langmea@cs.jhu.edu</a>, <a href="http://www.cs.jhu.edu/~langmea|www.cs.jhu.edu/~langmea">http://www.cs.jhu.edu/~langmea|www.cs.jhu.edu/~langmea</a>)
Usage: bowtie2-build-l [options]<em>* &lt;reference_in&gt; &lt;bt2_index_base&gt;
    reference_in            comma-separated list of files with ref sequences
    bt2_index_base          write bt2l data to files with this dir/basename
<strong></em><em></strong> Bowtie 2 indexes work only with v2 (not v1).  Likewise for v1 indexes. <strong></em>*</strong>
Options:
    -f                      reference files are Fasta (default)
    -c                      reference sequences given on cmd line (as
                            &lt;reference_in&gt;)
    -a/--noauto             disable automatic -p/--bmax/--dcv memory-fitting
    -p/--packed             use packed strings internally; slower, less memory
    --bmax &lt;int&gt;            max bucket sz for blockwise suffix-array builder
    --bmaxdivn &lt;int&gt;        max bucket sz as divisor of ref len (default: 4)
    --dcv &lt;int&gt;             diff-cover period for blockwise (default: 1024)
    --nodc                  disable diff-cover (algorithm becomes quadratic)
    -r/--noref              don't build .3/.4 index files
    -3/--justref            just build .3/.4 index files
    -o/--offrate &lt;int&gt;      SA is sampled every 2^&lt;int&gt; BWT chars (default: 5)
    -t/--ftabchars &lt;int&gt;    # of chars consumed in initial lookup (default: 10)
    --seed &lt;int&gt;            seed for random number generator
    -q/--quiet              verbose output (for debugging)
    -h/--help               print detailed description of tool and its options
    --usage                 print this usage message
    --version               print version information and quit</p>

<p><strong><em>*</strong> Warning <strong></em>*</strong>
'bowtie2-build-l' was run directly.  It is recommended that you run the wrapper script 'bowtie2-build' instead.</p>

<p>Error: Encountered internal Bowtie 2 exception (#1)
Command: bowtie2-build-l --threads 1 resources/ref.fa resources/ref.fa ```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:42:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 16:42:42"><div class="time">2023-11-22 16:42:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think I  had bowtie loaded as a module separately when I got the last error, but here‚Äôs the log file for that run in case:</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F067LLNL796/ref.fa_bowtie2_build.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">ref.fa_BOWTIE2_BUILD.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:48:31">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 16:48:31"><div class="time">2023-11-22 16:48:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya I think I understand the error. Strange that we did not notice it previously. Could you please open aMeta/workflow/rules/align.smk and add <code>--large-index</code> to this line:</p>

<p><code>bowtie2-build-l --threads {threads} {input.ref} {input.ref} &amp;gt; {log} 2&amp;gt;&amp;amp;1</code>
so it should become something like this:</p>

<p><code>bowtie2-build-l --large-index --threads {threads} {input.ref} {input.ref} &amp;gt; {log} 2&amp;gt;&amp;amp;1</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:50:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 16:50:22"><div class="time">2023-11-22 16:50:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If this fixes the error, we should quickly fix it in the main aMeta github repo. Apparently, as for now, the Bowtie2_Index rule builds small index, therefore produces <em>*.bt2 files, while it should be a large index (for metagenomic projects) by default, hence *</em>.bt2l files are expected by aMeta</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:51:56">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 16:51:56"><div class="time">2023-11-22 16:51:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* From the log-file you sent me, it did build the index correctly, just a small index, while it should build a large index (a large index is expected by aMeta), therefore you get this error. Strange that we never discovered this bug</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 16:52:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 16:52:56"><div class="time">2023-11-22 16:52:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ooh, let me give that a shot right now!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:15:25">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:15:25"><div class="time">2023-11-22 17:15:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, now I‚Äôm getting this:</p>

<p>```Error in rule Bowtie2<em>Index:
    jobid: 9
    input: resources/ref.fa
    output: resources/ref.fa.1.bt2l, resources/ref.fa.2.bt2l, resources/ref.fa.3.bt2l, resources/ref.fa.4.bt2l, resources/ref.fa.rev.1.bt2l, resources/ref.fa.rev.2.bt2l
    log: resources/ref.fa</em>BOWTIE2<em>BUILD.log (check log file(s) for error details)
    conda-env: /scratch/shreya23/aMeta</em>test2/aMeta/.test/.snakemake/conda/6254db56e34ae5808d40831fc7518aca_
    shell:
        bowtie2-build-l --large-index --threads 1 resources/ref.fa resources/ref.fa &gt; resources/ref.fa<em>BOWTIE2</em>BUILD.log 2&gt;&amp;1
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)</p>

<h1>Logfile resources/ref.fa<em>BOWTIE2</em>BUILD.log:</h1>

<p>bowtie2<em>build</em>l: unrecognized option '--large-index'```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:16:53">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:16:53"><div class="time">2023-11-22 17:16:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, sorry, this line should be correct:</p>

<p><code>bowtie2-build --large-index --threads {threads} {input.ref} {input.ref} &amp;gt; {log} 2&amp;gt;&amp;amp;1</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:17:39">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:17:39"><div class="time">2023-11-22 17:17:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* let me see!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:18:06">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:18:06"><div class="time">2023-11-22 17:18:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôm deleting and re-cloning aMeta each time so it takes a few min</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:20:37">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:20:37"><div class="time">2023-11-22 17:20:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* as far as I understand <code>bowtie2-build --large-index</code> should be equivalent to <code>bowtie2-build-l</code> but for some reason it still builds a small index, which is strange, so hopefully <code>bowtie2-build --large-index</code>  should be more stable</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:21:40">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:21:40"><div class="time">2023-11-22 17:21:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya thank you for your patience, and yes, cleaning and re-installing is a good strategy in one encounters errors</p>
                    
                    
                    
                        <div class="message-reaction">
                        :partyparrot: Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:23:14">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:23:14"><div class="time">2023-11-22 17:23:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* thank YOU for all your help!! Super excited to get this up and running üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:29:28">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:29:28"><div class="time">2023-11-22 17:29:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>bowtie2-build: unrecognized option '--threads'
Bowtie 2 version 2.2.3 by Ben Langmead (&lt;a href="mailto:langmea@cs.jhu.edu"&gt;langmea@cs.jhu.edu&lt;/a&gt;, &lt;http://www.cs.jhu.edu/~langmea|www.cs.jhu.edu/~langmea&gt;)</code>
Is the threads option required?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:33:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:33:14"><div class="time">2023-11-22 17:33:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* wait, Shreya, this is super-weird. --threads is a typical flag, I can't believe that bowtie2-build does not recognize it. Let me think a bit. Can you post the whole Bowtie2_Index rule from aMeta/workflow/rules/align.smk?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:34:49">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:34:49"><div class="time">2023-11-22 17:34:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh no I‚Äôm sorry! I‚Äôm certain I must have messed something up with my conda installing</p>

<p><code>rule Bowtie2_Index:
  output:
    expand(
      f"{config['bowtie2_db']}{{ext}}",
      ext=[
        ".1.bt2l",
        ".2.bt2l",
        ".3.bt2l",
        ".4.bt2l",
        ".rev.1.bt2l",
        ".rev.2.bt2l",
      ],
    ),
  input:
    ref=ancient(config["bowtie2_db"]),
  conda:
    "../envs/bowtie2.yaml"
  envmodules:
    **config["envmodules"]["bowtie2"],
  threads: 1
  log:
    f"{config['bowtie2_db']}_BOWTIE2_BUILD.log",
  shell:
    "bowtie2-build --large-index --threads {threads} {input.ref} {input.ref} &amp;gt; {log} 2&amp;gt;&amp;amp;1"</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:35:54">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:35:54"><div class="time">2023-11-22 17:35:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also, I noticed that when I do <code>conda list,</code> I get this version of bowtie: <code>bowtie2          2.5.1      py310ha0a81b8_2  bioconda</code> , but the log file says 2.2.3</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:36:32">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:36:32"><div class="time">2023-11-22 17:36:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya I think you are using some local version of Bowtie2, i.e. not the one installed togeter with aMeta. The latest Bowtie2 version is 2.5.2, this is what should be installed, while you seem to be using 2.2.3</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:37:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:37:47"><div class="time">2023-11-22 17:37:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* bizarre! It doesn‚Äôt appear that I have bowtie2 loaded as a module. Should I try installing and uninstalling it from base?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:38:32">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:38:32"><div class="time">2023-11-22 17:38:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya would you mind taking a screenshot of the very beginning of your testrun, I want to see a few first lines after you do:</p>

<p><code>cd .test
./runtest.sh -j 4</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:39:14">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:39:14"><div class="time">2023-11-22 17:39:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F0670FNFMQU/screenshot_2023-11-22_at_10.38.44_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-11-22 at 10.38.44 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F0670FNFMQU/screenshot_2023-11-22_at_10.38.44_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F0670FNFMQU-3700a22693/screenshot_2023-11-22_at_10.38.44_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="219" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:40:44">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:40:44"><div class="time">2023-11-22 17:40:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you, what do you see if you run now <code>bowtie2-build -h</code> ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:41:11">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:41:11"><div class="time">2023-11-22 17:41:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066Y2S8JFL/screenshot_2023-11-22_at_10.40.56_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-11-22 at 10.40.56 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F066Y2S8JFL/screenshot_2023-11-22_at_10.40.56_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F066Y2S8JFL-818545126c/screenshot_2023-11-22_at_10.40.56_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="255" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:41:28">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:41:28"><div class="time">2023-11-22 17:41:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ü§î so it seems like a bowtie version issue</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:41:44">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:41:44"><div class="time">2023-11-22 17:41:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much for helping with this!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:43:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:43:01"><div class="time">2023-11-22 17:43:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also could you send me the full log-file located at .snakemake/log?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:45:21">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:45:21"><div class="time">2023-11-22 17:45:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F066HH833DM/2023-11-22t102630.618706.snakemake.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2023-11-22T102630.618706.snakemake.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:47:38">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:47:38"><div class="time">2023-11-22 17:47:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya could you please do:</p>

<ol><li>conda activate /scratch/shreya23/aMeta<em>test2/aMeta/.test/.snakemake/conda/6254db56e34ae5808d40831fc7518aca</em></li><li>bowtie2-build -h</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:49:21">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:49:21"><div class="time">2023-11-22 17:49:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Still Bowtie 2 version 2.2.3!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:50:39">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:50:39"><div class="time">2023-11-22 17:50:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I can see that the installation script is installing Bowtie2, but when you execute aMtea, an old version of Bowtie2 is run. So you either have this old Bowtie2 version somewhere in the base-environment (which still should not be a problem), or (more likely) your conda insists on installing the oldest Bowtie2 version</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:51:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:51:51"><div class="time">2023-11-22 17:51:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh this is very strange! Yes, when I activate # packages in environment at /scratch/shreya23/aMeta<em>test2/aMeta/.test/.snakemake/conda/6254db56e34ae5808d40831fc7518aca</em>:, and then  conda list, the installed Bowtie2 is 2.5.2</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:52:15">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:52:15"><div class="time">2023-11-22 17:52:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Maybe I can go to base and try and upgrade the bowtie there and start from scratch</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:53:44">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:53:44"><div class="time">2023-11-22 17:53:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, please try <code>conda install bowtie2=2.5.2</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:54:16">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:54:16"><div class="time">2023-11-22 17:54:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay! I will do that and clean/reinstall aMeta and keep you posted‚Ä¶ thank you!!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:54:39">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:54:39"><div class="time">2023-11-22 17:54:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or at least perhaps bowtie2=2.5.2 should be added instead of just bowtie2 aMeta/workflow/envs/bowtie2.yaml</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:55:18">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:55:18"><div class="time">2023-11-22 17:55:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ooh, I will also try that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:55:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 17:55:22"><div class="time">2023-11-22 17:55:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* but on the other hand, you do seem to have the right bowtie2 version in your local bowtie2-environment /scratch/shreya23/aMeta<em>test2/aMeta/.test/.snakemake/conda/6254db56e34ae5808d40831fc7518aca</em></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 17:59:16">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 17:59:16"><div class="time">2023-11-22 17:59:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay so maybe bowtie2=2.5.2 will force the right version? I‚Äôll try that next</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 18:01:09">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-22 18:01:09"><div class="time">2023-11-22 18:01:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I would do both, do conda install bowtie2=2.5.2 in the base-environment (it seems to be heavily affecting aMeta for some reason, while it should not) and add bowtie2=2.5.2 to your aMeta/workflow/envs/bowtie2.yaml</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 18:02:46">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 18:02:46"><div class="time">2023-11-22 18:02:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* sounds good, thank you!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 18:02:53">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 18:02:53"><div class="time">2023-11-22 18:02:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* stay tuned üôÇ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 23:12:12">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 23:12:12"><div class="time">2023-11-22 23:12:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay, I have figured out a bit more!
If I don‚Äôt <code>module load gcc</code>and<code>module load python</code> before running the test script, I get this error:
<code>./runtest.sh: /home/shreya23/.local/bin/snakemake: /apps/software/gcc-6.2.0/python/3.6.0/bin/python3.6: bad interpreter: No such file or directory</code> </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-22 23:15:53">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-22 23:15:53"><div class="time">2023-11-22 23:15:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But if I load python beforehand, the pipeline starts running and breaks at the bowtie step. When I do ‚Äúwhich bowtie2‚Äù it defaults to <code>/apps/software/gcc-12.1.0/python/3.10.5/bin/bowtie2</code> , not the one packaged in aMeta. But if I unload python and do ‚Äúwhich bowtie2" it gives me the aMeta version--but then I get the ‚Äúbad interpreter‚Äù error. I‚Äôve asked our sysadmin to update the installed bowtie2 to 2.5.2 so we‚Äôll see if that helps!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 08:52:00">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 08:52:00"><div class="time">2023-11-23 08:52:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Shreya, thanks, I believe you should not load any modules at your HPC. Everything necessary should be delivered together with the aMeta installation. Including python. Also, this is a weird path <code>/apps/software/gcc-12.1.0/python/3.10.5/bin/bowtie2</code>. I do not understand how Bowtie2 can be related to python ü§î</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:05:19">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:05:19"><div class="time">2023-11-23 17:05:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmmmm, okay. ü§îindeed! Thanks so much for the debugging ‚Äî I will take this to the HPC admins and see if they can help sort things out. Will have to wait for after the Thanksgiving holiday but excited to get back to it soon! Thank you again Nikolay, I will keep you posted!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:07:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:07:25"><div class="time">2023-11-23 17:07:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya while you are waiting for the reply from the admins, I would recommend you to set aMeta up on your laptop. The testrun works fine on a laptop, and you will at least learn how a proper testrun output should looks like</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç:skin_tone_5: Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:08:48">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:08:48"><div class="time">2023-11-23 17:08:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Your HPC seems to be special, all the errors you posted are new for me, although I have experience helping quite a few people working on different HPCs, but your base-env and module system interfering the aMeta environment seem extreme (therefore I even asked you in the beginning if you are sure you activated the aMeta environment, because everything looked like you did not üôÇ )</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:10:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:10:56"><div class="time">2023-11-23 17:10:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh dear! I wonder why our HPC is strange. I did try to install it on my laptop but I got a segmentation fault after this bit <code>290724 packages in &lt;https://conda.anaconda.org/conda-forge/osx-64&gt;</code> so I gave up on that and headed to the cluster!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üò± Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:12:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:12:11"><div class="time">2023-11-23 17:12:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* actually, installing aMeta on a laptop and running the testrun should not take longer than 5-10 min, I do it regularly (and on different laptops). Installing on an HPC can be generally more difficult compared to a laptop. So if you have time now, please go ahead, I will be available here for ~1 hour to assist you</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:14:10">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:14:10"><div class="time">2023-11-23 17:14:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* hmm, @Shreya is your laptop running Windows?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:14:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:14:32"><div class="time">2023-11-23 17:14:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* nope, macbook air!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:15:02">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:15:02"><div class="time">2023-11-23 17:15:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you have a miniconda installed?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:15:52">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:15:52"><div class="time">2023-11-23 17:15:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes, I have miniconda3 and have been able to install other packages before!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:17:05">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:17:05"><div class="time">2023-11-23 17:17:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, do you want to try again on your laptop now? Or you are busy with something else?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:17:44">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:17:44"><div class="time">2023-11-23 17:17:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I just tried it and got the same seg fault! I‚Äôm wondering if I need to update my conda installation or something</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:18:31">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:18:31"><div class="time">2023-11-23 17:18:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôm trying again with the conda option instead of mamba, just in case</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:19:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:19:11"><div class="time">2023-11-23 17:19:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya what do you get if you do:</p>

<ol><li>mamba -h</li><li>conda -h
in your terminal (base-environment)</li>
</ol>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:19:48">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:19:48"><div class="time">2023-11-23 17:19:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```(base) Shreyas-MacBook-Air:~ shreya$ mamba -h
usage: mamba [-h] [-V] command ...</p>

<p>conda is a tool for managing and deploying applications, environments and packages.</p>

<p>Options:</p>

<p>positional arguments:
 command
  clean    Remove unused packages and caches.
  config    Modify configuration values in .condarc. This is modeled
         after the git config command. Writes to the user .condarc
         file (/Users/shreya/.condarc) by default.
  create    Create a new conda environment from a list of specified
         packages.
  help     Displays a list of available conda commands and their help
         strings.
  info     Display information about current conda install.
  init     Initialize conda for shell interaction. [Experimental]
  install   Installs a list of packages into a specified conda
         environment.
  list     List linked packages in a conda environment.
  package   Low-level conda package utility. (EXPERIMENTAL)
  remove    Remove a list of packages from a specified conda environment.
  uninstall  Alias for conda remove.
  run     Run an executable in a conda environment. [Experimental]
  search    Search for packages and display associated information. The
         input is a MatchSpec, a query language for conda packages.
         See examples below.
  update    Updates conda packages to the latest compatible version.
  upgrade   Alias for conda update.</p>

<p>optional arguments:
 -h, --help   Show this help message and exit.
 -V, --version Show the conda version number and exit.</p>

<p>conda commands available from other packages:
 env```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:19:59">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:19:59"><div class="time">2023-11-23 17:19:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```(base) Shreyas-MacBook-Air:~ shreya$ conda -h
usage: conda [-h] [-V] command ...</p>

<p>conda is a tool for managing and deploying applications, environments and packages.</p>

<p>Options:</p>

<p>positional arguments:
  command
    clean        Remove unused packages and caches.
    config       Modify configuration values in .condarc. This is modeled
                 after the git config command. Writes to the user .condarc
                 file (/Users/shreya/.condarc) by default.
    create       Create a new conda environment from a list of specified
                 packages.
    help         Displays a list of available conda commands and their help
                 strings.
    info         Display information about current conda install.
    init         Initialize conda for shell interaction. [Experimental]
    install      Installs a list of packages into a specified conda
                 environment.
    list         List linked packages in a conda environment.
    package      Low-level conda package utility. (EXPERIMENTAL)
    remove       Remove a list of packages from a specified conda environment.
    uninstall    Alias for conda remove.
    run          Run an executable in a conda environment. [Experimental]
    search       Search for packages and display associated information. The
                 input is a MatchSpec, a query language for conda packages.
                 See examples below.
    update       Updates conda packages to the latest compatible version.
    upgrade      Alias for conda update.</p>

<p>optional arguments:
  -h, --help     Show this help message and exit.
  -V, --version  Show the conda version number and exit.</p>

<p>conda commands available from other packages:
  env```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:21:09">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:21:09"><div class="time">2023-11-23 17:21:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Looks correct, your mamba are conda do not seem to be broken</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè:skin_tone_5: Shreya
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:21:47">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:21:47"><div class="time">2023-11-23 17:21:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So instead of a segmentation fault when I try with conda, I get this now:
```(base) Shreyas-MacBook-Air:aMeta shreya$ conda env create -f workflow/envs/environment.yaml
Collecting package metadata (repodata.json): done
Solving environment: failed</p>

<p>ResolvePackageNotFound: </p><ul><li>megan```</li>
</ul>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:22:51">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:22:51"><div class="time">2023-11-23 17:22:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* vs this for mamba:</p>

<p>```(base) Shreyas-MacBook-Air:aMeta shreya$ mamba env create -f workflow/envs/environment.yaml
Getting  conda-forge osx-64
Getting  conda-forge noarch
Getting  bioconda osx-64
Getting  bioconda noarch
Getting  pkgs/main osx-64
Getting  pkgs/main noarch
Getting  pkgs/r osx-64
Getting  pkgs/r noarch
Getting  r osx-64
Getting  r noarch</p>

<p>Looking for: ['python &gt;=3.7', 'snakemake-minimal &gt;=5.18', 'mamba', 'pandas', 'biopython', 'fastqc', 'mapdamage2', 'multiqc', 'bowtie2', 'samtools', 'cutadapt', 'krakenuniq', 'krona', 'r-base', 'hops', 'seqtk', 'parallel', 'megan', 'r-pheatmap', 'pmdtools', 'pysam', 'pygments', 'jinja2', 'networkx', 'pygraphviz', 'imagemagick', 'graphviz', 'pandoc']</p>

<p>290724 packages in <a href="https://conda.anaconda.org/conda-forge/osx-64">https://conda.anaconda.org/conda-forge/osx-64</a>
9493 packages in <a href="https://conda.anaconda.org/r/noarch">https://conda.anaconda.org/r/noarch</a>
5301 packages in <a href="https://conda.anaconda.org/r/osx-64">https://conda.anaconda.org/r/osx-64</a>
44225 packages in <a href="https://conda.anaconda.org/bioconda/osx-64">https://conda.anaconda.org/bioconda/osx-64</a>
115641 packages in <a href="https://conda.anaconda.org/conda-forge/noarch">https://conda.anaconda.org/conda-forge/noarch</a>
43793 packages in <a href="https://conda.anaconda.org/bioconda/noarch">https://conda.anaconda.org/bioconda/noarch</a>
31318 packages in <a href="https://repo.anaconda.com/pkgs/main/osx-64">https://repo.anaconda.com/pkgs/main/osx-64</a>
9493 packages in <a href="https://repo.anaconda.com/pkgs/r/noarch">https://repo.anaconda.com/pkgs/r/noarch</a>
4910 packages in <a href="https://repo.anaconda.com/pkgs/main/noarch">https://repo.anaconda.com/pkgs/main/noarch</a>
5301 packages in <a href="https://repo.anaconda.com/pkgs/r/osx-64">https://repo.anaconda.com/pkgs/r/osx-64</a>
Segmentation fault: 11```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:23:35">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:23:35"><div class="time">2023-11-23 17:23:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya do you have enough disk space on your laptop?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:24:37">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:24:37"><div class="time">2023-11-23 17:24:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôd think so?</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06708QMNBX/screenshot_2023-11-23_at_10.24.25_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2023-11-23 at 10.24.25 AM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06708QMNBX/screenshot_2023-11-23_at_10.24.25_am.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06708QMNBX-285e257517/screenshot_2023-11-23_at_10.24.25_am_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="92" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:28:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:28:28"><div class="time">2023-11-23 17:28:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Shreya what do you get if you do:</p>

<p><code>conda install -c bioconda megan</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:29:13">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:29:13"><div class="time">2023-11-23 17:29:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* let‚Äôs see!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:36:18">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:36:18"><div class="time">2023-11-23 17:36:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```PackagesNotFoundError: The following packages are not available from current channels:</p><ul><li>megan```</li>
</ul>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:42:17">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:42:17"><div class="time">2023-11-23 17:42:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* looks like megan is not available for mac in bioconda channel. It is certainly available for unix. Strange!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:42:44">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:42:44"><div class="time">2023-11-23 17:42:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh!! Bizarre!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:43:03">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:43:03"><div class="time">2023-11-23 17:43:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I can only recommend you to install megain manually from here <a href="https://software-ab.cs.uni-tuebingen.de/download/megan6/welcome.html">https://software-ab.cs.uni-tuebingen.de/download/megan6/welcome.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:43:26">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:43:26"><div class="time">2023-11-23 17:43:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Unfortunately, I am going to have to head out soon ‚Äî but I do have Megan installed already on my computer, so I will see what I can do to point the installation there.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:43:29">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:43:29"><div class="time">2023-11-23 17:43:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I believe you should use this file <a href="https://software-ab.cs.uni-tuebingen.de/download/megan6/MEGAN_Community_macos_6_25_6.dmg">MEGAN<em>Community</em>macos<em>6</em>25_6.dmg</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:45:02">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:45:02"><div class="time">2023-11-23 17:45:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see. I am very sorry about all the troubles. Installing software can be a pain. Please do not give up, we can have a zoom session and try to do the installation. Once it is installed, things will be much more stable</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:46:08">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:46:08"><div class="time">2023-11-23 17:46:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No worries at all Nikolay, I‚Äôm sure the problem is somewhere at my end! I really appreciate your help with installing. I am very excited to get it up and running and have already promised a labmate I will screen her samples so I am certainly committed to getting it working!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:47:41">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-11-23 17:47:41"><div class="time">2023-11-23 17:47:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good, thank you, and please do not hesitate to DM me to keep talking installation. I am available and will do my best to help you</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-23 17:47:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2023-11-23 17:47:58"><div class="time">2023-11-23 17:47:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much, I will certainly keep you posted!!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-29 17:01:43">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-11-29 17:01:43"><div class="time">2023-11-29 17:01:43</div></a>
                <div class="msg">
                    <p>Hey all! Weird lab question - We're designing a homebrew capture to enrich our endogenous DNA content. We tested it on modern DNA and had no problems recovering the target sequences or weird behavior.  However, when trying to capture our ancient libraries (which do have a different primer setup) we're getting some weird stuff: we are seeing amplification of our captured libraries (good), <em>no</em> amplification in our PCR negative (also good), but amplification in a "no probe control", where no baits were added to the libraries. The beads are streptavidin beads and the baits are biotinilated, so in theory only the baits should stick to the beads(?) Has anyone ran into something like this before? Perhaps an affinity of the beads for certain sequences, etc?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-29 17:14:16">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-29 17:14:16"><div class="time">2023-11-29 17:14:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This might be a naive question but have you tested whether your probes share a sequence similarity with spurious sequences that are not your intentional target? Blasting some of them against NCBI GenBank might give you an idea whether they might be able to anneal to DNA from other species, too.
For me such a scenario seems more likely than that the beads themselves attract certain DNA molecules.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-29 17:17:34">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-11-29 17:17:34"><div class="time">2023-11-29 17:17:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Alex! We would blast the probes, but we've constructed them from random RAD fragments (so don't know what the sequences are). We do get some off target capture using this method (but it's negligible). Thanks for the suggestion - we were thinking the same think about the beads, but couldn't find much info on it.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-29 17:19:53">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-29 17:19:53"><div class="time">2023-11-29 17:19:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you have a taxonomic profile of your samples prior to enrichment?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-29 17:21:52">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2023-11-29 17:21:52"><div class="time">2023-11-29 17:21:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Without knowing the rough origin of your sequences that were used to design the probes, this is very hard to debug.
But maybe there is someone here in the community who is more knowledgable about capture probes and binding specificity. Sorry!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-30 04:12:19">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-11-30 04:12:19"><div class="time">2023-11-30 04:12:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What temperatures were you using for the hybridization? Have you tested it at different temps? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-30 16:19:08">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-11-30 16:19:08"><div class="time">2023-11-30 16:19:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* 60degC - We haven't played with the temps much, but writing this now that definitely feels like something we should do</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-30 16:19:40">
            
                <img src="https://avatars.slack-edge.com/2023-11-29/6289961831792_a5ae73690edd5251acde_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Carly Scott
                     <span class="print-only user-email">(cbscott@utexas.edu)</span>
                </div>
                <a href="#2023-11-30 16:19:40"><div class="time">2023-11-30 16:19:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* and @Alex H√ºbnerwe do! Appreciate the insights on things so far.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-11-30 21:58:19">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2023-11-30 21:58:19"><div class="time">2023-11-30 21:58:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* We saw slight improvements at 65 degrees with our enrichments. Not sure if this will fix your issue but may be worth a try.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Carly Scott
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-11 11:32:48">
            
                <img src="https://secure.gravatar.com/avatar/4aaf54e57b049bfdc27125430fdd776a.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0019-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Charlotte Greenall
                     <span class="print-only user-email">(charlotte.greenall@nhm.ac.uk)</span>
                </div>
                <a href="#2023-12-11 11:32:48"><div class="time">2023-12-11 11:32:48</div></a>
                <div class="msg">
                    <p>Hi! Has anyone has any problems when they try to build a BLAST database and it generates multiple header/index and option files e.g library<em>blast.fna.00.nhr
library</em>blast.fna.00.nin
library<em>blast.fna.00.nog
library</em>blast.fna.00.nsq
library<em>blast.fna.01.nhr
library</em>blast.fna.01.nin
library<em>blast.fna.01.nog
library</em>blast.fna.01.nsq
library<em>blast.fna.02.nhr
library</em>blast.fna.02.nin
library<em>blast.fna.02.nog
library</em>blast.fna.02.nsq
library<em>blast.fna.03.nhr
library</em>blast.fna.03.nin
library_blast.fna.03.nog</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-12 00:17:01">
            
                <img src="https://avatars.slack-edge.com/2023-10-18/6047876445718_86d2be60a3803d298b6c_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jamie Wood
                     <span class="print-only user-email">(jamie.wood@adelaide.edu.au)</span>
                </div>
                <a href="#2023-12-12 00:17:01"><div class="time">2023-12-12 00:17:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think this is normal for large databases, was there a particular problem you were encountering?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:10:14">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-12-20 16:10:14"><div class="time">2023-12-20 16:10:14</div></a>
                <div class="msg">
                    <p>Hola!
Does anyone have any sort of estimate or reference for the minimum amount of reads needed for proper ancient damage estimation? üìâ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:19:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:19:30"><div class="time">2023-12-20 16:19:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hola!</p>

<p>I did a mini simulation here: Mann, Allison E., James A. Fellows Yates, Zandra Fagern√§s, Rita M. Austin, Elizabeth A. Nelson, and Courtney A. Hofman. 2020. ‚ÄúDo I Have Something in My Teeth? The Trouble with Genetic Analyses of Diet from Archaeological Dental Calculus.‚Äù Quaternary International: The Journal of the International Union for Quaternary Research, November. <a href="https://doi.org/10.1016/j.quaint.2020.11.019">https://doi.org/10.1016/j.quaint.2020.11.019</a>.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">sciencedirect.com</div>
                            
                            
                                
                                <div class="link-title"><a href="https://doi.org/10.1016/j.quaint.2020.11.019">Do I have something in my teeth? The trouble with genetic analyses of diet from archaeological dental calculus</a></div>
                                <div class="link-text">
                                    Dental calculus and other preserved microbiome substrates are an attractive target for dietary reconstruction in past populations through a variety of‚Ä¶
                                </div>
                                
                                
    
        <a href="https://doi.org/10.1016/j.quaint.2020.11.019">
            <img class="preview" src="https://ars.els-cdn.com/content/image/1-s2.0-S1040618223X00081-cov150h.gif"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://doi.org/10.1016/j.quaint.2020.11.019</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:19:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:19:46"><div class="time">2023-12-20 16:19:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Figure.... 4 or 5? Can't remember from my head</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Aleksandra Laura Pach
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:20:33">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:20:33"><div class="time">2023-12-20 16:20:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Figure 2 (now it loads) so I was completely wrong ü§£</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:20:41">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-12-20 16:20:41"><div class="time">2023-12-20 16:20:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Fig 2? Right!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:21:08">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:21:08"><div class="time">2023-12-20 16:21:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But yes roughly what Nikolay said, but you can just about go down to 50</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:21:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:21:19"><div class="time">2023-12-20 16:21:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But not reliabiy</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:21:41">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-12-20 16:21:41"><div class="time">2023-12-20 16:21:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've read this paper at least 20 times and I forgot that it was in there. I'm getting more and more thankful for this channel as my memory declines...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:21:58">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-12-20 16:21:58"><div class="time">2023-12-20 16:21:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But that's perfect. These references are exactly what I needed!</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:30:17">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2023-12-20 16:30:17"><div class="time">2023-12-20 16:30:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; I've read this paper at least 20 times and I forgot that it was in there. I'm getting more and more thankful for this channel as my memory declines...</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06B0RK6MMY/tenor_gif519520171098513221.gif?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">tenor_gif519520171098513221.gif</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06B0RK6MMY/tenor_gif519520171098513221.gif?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06B0RK6MMY-4e8eee87f0/tenor_gif519520171098513221_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="220"
                 height="124" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:33:45">
            
                <img src="https://avatars.slack-edge.com/2023-09-12/5883654875490_10a038e90753e4e4f8a7_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Aleksandra Laura Pach
                     <span class="print-only user-email">(alp@science.ku.dk)</span>
                </div>
                <a href="#2023-12-20 16:33:45"><div class="time">2023-12-20 16:33:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sowwy! Probably won't forget again ü•π
I keep using the potato and tomato example though! Can't even look at these vegetables in the same way anymore. üçÖü•î</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ James Fellows Yates, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2023-12-20 16:13:26">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2023-12-20 16:13:26"><div class="time">2023-12-20 16:13:26</div></a>
                <div class="msg">
                    <p>@Aleksandra Laura Pach We recommend a minimum threshold of 100-200 in aMeta exactly for that reason, i.e. a convincing damage pattern is hard to get from fewer reads</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëå Aleksandra Laura Pach, Dawn lewis
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-12 21:35:43">
            
                <img src="https://avatars.slack-edge.com/2022-07-19/3842655941616_52cc6550245d56bdc4b3_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Jaime Zolik
                     <span class="print-only user-email">(zolik006@umn.edu)</span>
                </div>
                <a href="#2024-01-12 21:35:43"><div class="time">2024-01-12 21:35:43</div></a>
                <div class="msg">
                    <p>Hello! I am performing metagenomic analysis on ancient DNA. I am currently following the eMeta but I am coming up to the point where I need to build a project-specific MALT database. Does anyone have any advice on this next step? I am trying to anticipate the amount of computing power and need and how to keep all of my files organized/backed up.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-13 16:35:48">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2024-01-13 16:35:48"><div class="time">2024-01-13 16:35:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Jaime,</p>

<p>I would say it would be good to have at least 1 TB of RAM if not more. I would say try to make your database as comprehensive (i.e. large) as possible to improve your sensitivity. If you can get access to 3-4 TB of RAM, that would be ideal. But if not, you may have to reduce your database size.</p>

<p>You can use the krakenuniq-build to include the database or the NCBI Datasets command-line tools. I know aMeta uses krakenuniq so that may be the best approach.</p>

<p>I am not sure if you come across these papers below. But I think they may also help with what to think about when constructing a database and the amount of memory it would require.
<a href="https://journals.asm.org/doi/full/10.1128/msystems.00080-18">https://journals.asm.org/doi/full/10.1128/msystems.00080-18</a></p>

<p><a href="https://peerj.com/articles/6594/">https://peerj.com/articles/6594/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PeerJ</div>
                            
                            
                                
                                <div class="link-title"><a href="https://peerj.com/articles/6594/">Assessing alignment-based taxonomic classification of ancient microbial DNA</a></div>
                                <div class="link-text">
                                    
                                </div>
                                
                                
    
        <a href="https://peerj.com/articles/6594/">
            <img class="preview" src="https://dfzljdn9uc3pi.cloudfront.net/2019/6594/1/fig-1-1x.jpg"
                width="600"
                 height="473" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://peerj.com/articles/6594/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-13 22:22:19">
            
                <img src="https://avatars.slack-edge.com/2022-07-19/3842655941616_52cc6550245d56bdc4b3_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Jaime Zolik
                     <span class="print-only user-email">(zolik006@umn.edu)</span>
                </div>
                <a href="#2024-01-13 22:22:19"><div class="time">2024-01-13 22:22:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much, this is very helpful!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Sterling Wright
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-14 09:41:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-01-14 09:41:07"><div class="time">2024-01-14 09:41:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Jaime! Sterling is right, for the MALT database build step of aMeta, I would advise to use a node with 1TB RAM. And for the alignment step as well, by the way üòä</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-16 09:25:17">
            
                <img src="https://avatars.slack-edge.com/2023-12-15/6357190656532_bb8da8eebccd842bd4a2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Liam Fitzstevens
                     <span class="print-only user-email">(liam.fitzstevens@tuebingen.mpg.de)</span>
                </div>
                <a href="#2024-01-16 09:25:17"><div class="time">2024-01-16 09:25:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You might be interested in taking a look at Struo2 (which works in conjunction with Kraken2/Bracken)!
<a href="https://pubmed.ncbi.nlm.nih.gov/34616633/">https://pubmed.ncbi.nlm.nih.gov/34616633/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed</div>
                            
                            
                                
                                <div class="link-title"><a href="https://pubmed.ncbi.nlm.nih.gov/34616633/">Struo2: efficient metagenome profiling database construction for ever-expanding microbial genome datasets - PubMed</a></div>
                                <div class="link-text">
                                    Mapping metagenome reads to reference databases is the standard approach for assessing microbial taxonomic and functional diversity from metagenomic data. However, public reference databases often lack recently generated genomic data such as metagenome-assembled genomes (MAGs), which can limit the s ‚Ä¶
                                </div>
                                
                                
    
        <a href="https://pubmed.ncbi.nlm.nih.gov/34616633/">
            <img class="preview" src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg"
                width="1600"
                 height="830" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://pubmed.ncbi.nlm.nih.gov/34616633/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-21 09:43:08">
            
                <img src="https://avatars.slack-edge.com/2023-01-15/4639294999574_85a756ca97035831649b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Freya Steinhagen
                     <span class="print-only user-email">(stu227847@mail.uni-kiel.de)</span>
                </div>
                <a href="#2024-01-21 09:43:08"><div class="time">2024-01-21 09:43:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If your computational capacities are really limited I would recommend working (screening) target oriented on specific databases. For the detection of Metazoans I had really good results with the refsec <a href="https://ftp.ncbi.nlm.nih.gov/refseq/release/mitochondrion/mitochondrion.1.1.genomic.fna.gz">mitochondrion database</a>. It can also be used for plant detection. But if plants are the main target, it can also be beneficial to try screening with the refsec plastid database (also ncbi). When screening for microorganisms I had really good results implementing the <a href="https://www.arb-silva.de/documentation/release-138/">SILVA SSU and LSU databases</a>. It is quite easy to implement in the MALT workflow. One last thing that I can recommend when working with limited computational capacities is screening on sample at a time instead of a bulked operations. Much more work but it can prevent craching. Hope this can help :)</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">arb-silva.de</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.arb-silva.de/documentation/release-138/">Release 138</a></div>
                                <div class="link-text">
                                    Release information for the SILVA 138 ribosomal RNA (rRNA) databases
                                </div>
                                
                                
    
        <a href="https://www.arb-silva.de/documentation/release-138/">
            <img class="preview" src="https://www.arb-silva.de/favicons/apple-touch-icon.png?v=2"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.arb-silva.de/documentation/release-138/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 18:54:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-01-25 18:54:21"><div class="time">2024-01-25 18:54:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Jaime Zolik, apologies for my very late reply! If you send me a DM with some details on the output of the KrakenUniq step (the one prior to the building of project-specific MALT DB in aMeta) I can advice you in details about recommended resources. So far it is hard to guess since I do not know how many microbes were detected in the first (KrakenUniq ) step</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 18:19:21">
            
                <img src="https://avatars.slack-edge.com/2023-09-25/5969403070976_12b1e1e90c797643f2c8_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Sarah Johnson
                     <span class="print-only user-email">(sarahjjohnson@ou.edu)</span>
                </div>
                <a href="#2024-01-25 18:19:21"><div class="time">2024-01-25 18:19:21</div></a>
                <div class="msg">
                    <p>Hey everyone! I have a Kraken database question. Does anyone have experience reducing the built database to some max<em>db</em>size and if so, did you notice some loss of sensitivity?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 18:29:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-01-25 18:29:31"><div class="time">2024-01-25 18:29:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I've not directly experienced it myself, but it makes sense -IIRC you're reducing the number of kmers by down sampling to try and keep the most informative ones, but the most informative ones aren't the only ones that would also be valid.</p>

<p>If you remove any amountreference k-msrs you will reduce the chances of getting a hit </p>

<p>@Maxime Borry probably knows better</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 18:46:43">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2024-01-25 18:46:43"><div class="time">2024-01-25 18:46:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I can attest to what James is saying. With Kraken uniq and kmer set at 31, we were able to still run malt build. We noticed a considerable reduction in the number of species. It is my understanding that a lot of this is discussed in the aMeta paper and they have a lot of the data to show how sensitivity and specificity is impacted by changing the database size. </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 19:05:48">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-01-25 19:05:48"><div class="time">2024-01-25 19:05:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Sarah Johnson, reducing database size will certainly lead to reduced sensitivity (and indirectly specificity). We discuss this here <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9</a>, please see the section "Effect of database size". Regarding your particular question about reducing Kraken DB size with max<em>db</em>size, if I recall correctly (perhaps it was in KrakenUniq and not Kraken2) this option was broken long time ago. I remember I tried to use this option, and was still running out of memory when building a database. But perhaps they have already fixed it, or I remember incorrectly, so I would encourage you to try and let us know how it went, I would be curious to know your experience. Still, I guess, you would not like to reduce you DB size because you do not want to sacrifice analysis accuracy, but you probably have to do it because you do not have enough RAM. This is unfortunately a typical problem in this field: the balance between the accuracy and resources, there is no straightforward solution here üòû</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 19:38:25">
            
                <img src="https://avatars.slack-edge.com/2023-09-25/5969403070976_12b1e1e90c797643f2c8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Johnson
                     <span class="print-only user-email">(sarahjjohnson@ou.edu)</span>
                </div>
                <a href="#2024-01-25 19:38:25"><div class="time">2024-01-25 19:38:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all for your responses. Yes, I‚Äôm more concerned with resources. The HPC at my university has the resources but it is shared across the university so I‚Äôm trying to sacrifice as little specificity and sensitivity as possible while optimizing resource utilization. I think I might try a small experiment looking at that if I have time and will update as I have more information. Kraken1 has the ability to shrink databases but I‚Äôm not sure if one built with kraken2 would be shrink-able. </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 19:45:33">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-01-25 19:45:33"><div class="time">2024-01-25 19:45:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's definitely working for kraken2, that's how they do it for <a href="https://benlangmead.github.io/aws-indexes/k2">https://benlangmead.github.io/aws-indexes/k2</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 19:53:46">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-01-25 19:53:46"><div class="time">2024-01-25 19:53:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Maxime Borry! @Sarah Johnson I would suggest that perhaps an intelligent way of reducing database size would be excluding redundancy and keeping representative organisms per e.g. genus or family. This however needs a lot of time and manual work. But if you have time and energy, this way could result in a dramatic database reduction without (presumably) loosing much accuracy</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon, Biancamaria Bonucci
                        </div>
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-25 20:04:53">
            
                <img src="https://avatars.slack-edge.com/2023-09-25/5969403070976_12b1e1e90c797643f2c8_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Johnson
                     <span class="print-only user-email">(sarahjjohnson@ou.edu)</span>
                </div>
                <a href="#2024-01-25 20:04:53"><div class="time">2024-01-25 20:04:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov I had the same thought. I wrote a little script to do just that. We‚Äôll see. Im going to try shrinking and/or building a reduced db and will let you know. </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Biancamaria Bonucci, Wenqin Yu
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 15:44:18">
            
                <img src="https://secure.gravatar.com/avatar/d1a9503a47879969d6b213f9c0e9cf7f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nora Bergfeldt
                     <span class="print-only user-email">(nora.bergfeldt@gmail.com)</span>
                </div>
                <a href="#2024-01-31 15:44:18"><div class="time">2024-01-31 15:44:18</div></a>
                <div class="msg">
                    <p>Hello! I have a question re: phylogenies. I found an interesting bacterial species when running MALT (or actually the aMeta workflow). I want to find out what strain of the species the reads belong to but the coverage is low. My idea was to create a phylogeny using a few known strains and see where my reads end up in the tree. However, I've failed massively so far so I'm turning to you for some input üôÇ  Has anyone here done something similar, or read about something similar in a paper somewhere?
What I have done so far is downloading different strains, cat them together with the reference genome used in MALT and my sequence data and tried to create a consensus alignment. I think the fact that the sequence data is so fragmented makes it difficult to create the consensus because I keep running out of memory on our cluster.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 15:47:12">
            
                <img src="https://secure.gravatar.com/avatar/d1a9503a47879969d6b213f9c0e9cf7f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nora Bergfeldt
                     <span class="print-only user-email">(nora.bergfeldt@gmail.com)</span>
                </div>
                <a href="#2024-01-31 15:47:12"><div class="time">2024-01-31 15:47:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I can add that I tried a run with fewer samples (meaning I forgot the outgroup...) and then it worked fine, creating a consensus with muscle and then the tree with BEAST. So I think I'm not way out of line with this idea but obviously something is not right</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 15:49:28">
            
                <img src="https://secure.gravatar.com/avatar/d1a9503a47879969d6b213f9c0e9cf7f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nora Bergfeldt
                     <span class="print-only user-email">(nora.bergfeldt@gmail.com)</span>
                </div>
                <a href="#2024-01-31 15:49:28"><div class="time">2024-01-31 15:49:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh also, I realise selecting a few genes to focus on could be an idea but I'm afraid I won't have coverage of full genes in my sequence data</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 16:30:42">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465355512771_d76bd23c72a4c5f61546_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Gunnar Neumann
                     <span class="print-only user-email">(gunnar_neumann@eva.mpg.de)</span>
                </div>
                <a href="#2024-01-31 16:30:42"><div class="time">2024-01-31 16:30:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Nora,
i once used pathPhynder to place low coverage genomes into a given phylogeny.
<a href="https://doi.org/10.1093/molbev/msac017">https://doi.org/10.1093/molbev/msac017</a>
not sure what the minimum coverage would though (always wanted to test that).
it‚Äôs quite easy to use.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 18:10:32">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2024-01-31 18:10:32"><div class="time">2024-01-31 18:10:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I second pathPhynder for this. It has also been applied to very low coverage situations (ie. ancient environmental DNA)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 19:01:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-08/2606090924960_c0c01b22080e523492a2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nico Rascovan
                     <span class="print-only user-email">(nicorasco@gmail.com)</span>
                </div>
                <a href="#2024-01-31 19:01:40"><div class="time">2024-01-31 19:01:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or EPA-ng, non? Just include your low-coverage genome in your alignment (it will be full of N‚Äôs), build the tree with the good genomes only, and then place the low-coverage genome in that tree</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Pete Heintzman
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-01-31 20:03:51">
            
                <img src="https://secure.gravatar.com/avatar/d1a9503a47879969d6b213f9c0e9cf7f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nora Bergfeldt
                     <span class="print-only user-email">(nora.bergfeldt@gmail.com)</span>
                </div>
                <a href="#2024-01-31 20:03:51"><div class="time">2024-01-31 20:03:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-01 07:10:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-01 07:10:09"><div class="time">2024-02-01 07:10:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* To do a @Antonio Fernandez-Guerra ;):</p>

<p>nf-core has a pipeline for EPA, maybe that would be helpful?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-01 07:16:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-01 07:16:10"><div class="time">2024-02-01 07:16:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06GFG0FV0E/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06GFG0FV0E/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06GFG0FV0E-f605a88bd1/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="298" />
        </a>
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        üëÄ Pete Heintzman
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 13:43:58">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 13:43:58"><div class="time">2024-02-07 13:43:58</div></a>
                <div class="msg">
                    <p>Hey everyone,
What‚Äôs your experience with haplotype phasing/strain deconvolution for your short metagenomic read mapping to bacterial genomes ? (ie: getting the consensus sequence/haplotype of each strain when you have multiple strains mapping to a species reference genome).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 13:51:41">
            
                <img src="https://avatars.slack-edge.com/2023-12-15/6357190656532_bb8da8eebccd842bd4a2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Liam Fitzstevens
                     <span class="print-only user-email">(liam.fitzstevens@tuebingen.mpg.de)</span>
                </div>
                <a href="#2024-02-07 13:51:41"><div class="time">2024-02-07 13:51:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Maxime, inStrain is currently the best SNP-based approach to do this:
<a href="https://www.nature.com/articles/s41587-020-00797-0">https://www.nature.com/articles/s41587-020-00797-0</a>
SynTracker is a newer tool that does the same thing, but based on synteny (making it insensitive to SNPs):
<a href="https://github.com/leylabmpi/SynTracker">https://github.com/leylabmpi/SynTracker</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 13:54:03">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 13:54:03"><div class="time">2024-02-07 13:54:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But aDNA with our very short reads is kind of excluded for inStrain
&gt;  For each read pair aligned to the reference genome (de novo assembled from the same sample or a genome from another source) the mapQ score, average nucleotide identity (ANI) of the pair to the reference genome and the insert size between aligned reads are calculated. Read pairs that do not pass adjustable quality cutoffs are removed, as are all unpaired reads. The exclusive use of pairs doubles the number of bases used to calculate the read ANI and mapQ score, increasing their accuracy and substantially increasing the span of genome analyzed.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:03:55">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 14:03:55"><div class="time">2024-02-07 14:03:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Most of the time, the aDNA fragments are shorter than the reads, so paired reads are merged (‚Äúnegative‚Äù insert size) because the forward and reverse reads actually overlap</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:14:02">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 14:14:02"><div class="time">2024-02-07 14:14:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Syntracker requires metagenomic assemblies üòû</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:15:19">
            
                <img src="https://avatars.slack-edge.com/2023-12-15/6357190656532_bb8da8eebccd842bd4a2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Liam Fitzstevens
                     <span class="print-only user-email">(liam.fitzstevens@tuebingen.mpg.de)</span>
                </div>
                <a href="#2024-02-07 14:15:19"><div class="time">2024-02-07 14:15:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh, right, sorry I am new here üòÖ 
If you haven‚Äôt already, I‚Äôd recommend reaching out to Matt Olm on inStrain‚Äôs Git page! He‚Äôs super responsive; would likely have ideas </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:16:07">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 14:16:07"><div class="time">2024-02-07 14:16:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good idea, thanks @Liam Fitzstevens üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 17:42:26">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2024-02-07 17:42:26"><div class="time">2024-02-07 17:42:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Assuming you have a mix of two haplotypes, you could try the method presented in <a href="https://www.nature.com/articles/s42003-021-01710-4">https://www.nature.com/articles/s42003-021-01710-4</a> to deconvolute the two sequences.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Nature</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.nature.com/articles/s42003-021-01710-4">Environmental palaeogenomic reconstruction of an Ice Age algal population</a></div>
                                <div class="link-text">
                                    Communications Biology - Lammers et al. use sedimentary ancient DNA to reconstruct palaeogenomes of Nannochloropsis. This study demonstrates the value of sedaDNA for palaeogenomic reconstructions...
                                </div>
                                
                                
    
        <a href="https://www.nature.com/articles/s42003-021-01710-4">
            <img class="preview" src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs42003-021-01710-4/MediaObjects/42003_2021_1710_Fig1_HTML.png"
                width="2000"
                 height="1935" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.nature.com/articles/s42003-021-01710-4</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 17:45:03">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2024-02-07 17:45:03"><div class="time">2024-02-07 17:45:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Alternatively there is Kallisto, developed by @Benjamin Vernot‚Äôs group. This does not pull out the consensus sequences but can give an indication of the relative abundance of different haplotypes (and will work with &gt;2 haplotypes).</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 18:11:27">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 18:11:27"><div class="time">2024-02-07 18:11:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Pete Heintzman
You mean dividing your variants into high and low frequency set ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 18:20:37">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2024-02-07 18:20:37"><div class="time">2024-02-07 18:20:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yep!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Maxime Borry
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-08 18:11:45">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-02-08 18:11:45"><div class="time">2024-02-08 18:11:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi! Regarding this question I also tried to use Instrain profile with ancient DNA (short reads). I collapsed the reads before mapping it against the genome reference and I have a bam file that I give as input to instrain. I used the flag --pairing<em>filter non</em>discordant (non_discordant = Keep all paired reads and singleton reads that map to a single scaffold) to keep all reads in the analysis. I am new to this kind of analysis, so maybe it is not the best approach to bypass the problem with the short fragments.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-09 14:54:47">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-09 14:54:47"><div class="time">2024-02-09 14:54:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Kallisto wouldn't give you the consensus sequences, but if you had a collection of strain references it might be pretty good for picking out the best mixture</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-09 14:55:02">
            
                <img src="https://secure.gravatar.com/avatar/71a57b146a134b162437ac0f5e77f753.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0004-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Benjamin Vernot
                     <span class="print-only user-email">(benjamin_vernot@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-09 14:55:02"><div class="time">2024-02-09 14:55:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Lots of caveats, though. Happy to chat when I'm back.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:40:39">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 14:40:39"><div class="time">2024-02-07 14:40:39</div></a>
                <div class="msg">
                    <p>And related question: do you have any recommendation for ploidy estimation (ie: number of strains) for aDNA short reads ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:43:39">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 14:43:39"><div class="time">2024-02-07 14:43:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Variation graph would be one solution, but only for well studied species, where there are already all the variants in the reference genomes variation graph.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 14:48:56">
            
                <img src="https://avatars.slack-edge.com/2020-09-14/1361802137683_94343aa8740b8605afa2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Iseult
                     <span class="print-only user-email">(ijackson@tcd.ie)</span>
                </div>
                <a href="#2024-02-07 14:48:56"><div class="time">2024-02-07 14:48:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* depending on what genomes you're working with, you could try treating strain-specific regions as a "chromosome" and look at coverage variation between chromosomes? this only really works if you have variable gene content though. you could also try using heterozygosity in different regions as a proxy for this? I'm not sure how this would be impacted by e.g. contamination though, but might be worth trying</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 17:58:10">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-02-07 17:58:10"><div class="time">2024-02-07 17:58:10</div></a>
                <div class="msg">
                    <p>Hi all!
Has anyone tried to use the consensus_aDNA.py script from cmseq? I have a rescaled bam file (I used mapdamage2 to rescale it) that contains reads from an ancient bacteria aligned against the genome reference.
This is the command that I used:</p>

<p>consensus<em>aDNA.py --mincov 5 --minqual 30 -r genome</em>ref.fasta --pos<em>specific</em>prob<em>tab Stats</em>out<em>MCMC</em>correct<em>prob.csv --pos</em>damage<em>prob</em>thrsh 0.95 bam<em>rmdup</em>sort.rescaled.bam</p>

<p>And this is the error:</p>

<p>Traceback (most recent call last):
  File "/softwares/condaenvs/metagenomics3/bin/consensus<em>aDNA.py", line 10, in &lt;module&gt;
    sys.exit(consensus</em>from<em>file())
  File "/softwares/condaenvs/metagenomics3/lib/python3.7/site-packages/cmseq/consensus</em>aDNA.py", line 265, in consensus<em>from</em>file
    trimReads=None,post<em>damage</em>prob=pos<em>prob</em>thrsh,pos<em>prob</em>db=pos<em>stats</em>db, refseq<em>idx=RefSeq</em>idx)
  File "/softwares/condaenvs/metagenomics3/lib/python3.7/site-packages/cmseq/consensus<em>aDNA.py", line 98, in reference</em>free<em>consensus
    consensus</em>positions[pileupcolumn] = consensus<em>rule(dict((k,v) for k,v in position</em>data['base<em>freq'].items() if k != 'N'))
  File "/softwares/condaenvs/metagenomics3/lib/python3.7/site-packages/cmseq/cmseq.py", line 206, in majority</em>rule
    freq<em>array= data</em>array['base<em>freq']
KeyError: 'base</em>freq'</p>

<p>Has anyone dealt with this error?</p>

<p>Thank you in advance!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 18:24:35">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 18:24:35"><div class="time">2024-02-07 18:24:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You‚Äôre in luck, the author of this code is in this workspace üôÇ @Kun Huang</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 18:28:10">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 18:28:10"><div class="time">2024-02-07 18:28:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you decide to use something else than cmseq, I‚Äôd recommend calling your variants (using freebayes for example) from your mapdamage rescaled bam file, and then filter on variant support and/or quality (bcftools)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 18:28:32">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 18:28:32"><div class="time">2024-02-07 18:28:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* bam -&gt; freebayes -&gt; bcftools view -&gt; bcftools consensus</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 19:35:20">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-02-07 19:35:20"><div class="time">2024-02-07 19:35:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you very much for your help! üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 19:40:42">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-02-07 19:40:42"><div class="time">2024-02-07 19:40:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And what if I use angsd to create the consensus sequence (using as input the mapdamage rescaled bam) and then I merged all the fasta from the different individuals and use snp-sites to have a multifasta only with the snps? Is it a good idea or it is not a good approach for ancient data?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-07 20:15:19">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-07 20:15:19"><div class="time">2024-02-07 20:15:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Any variant caller should work (if using sensible parameters). My go to caller is freebayes üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-08 10:21:18">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-02-08 10:21:18"><div class="time">2024-02-08 10:21:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:10:56">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:10:56"><div class="time">2024-02-13 12:10:56</div></a>
                <div class="msg">
                    <p>Hi all, I have a question about MultiVcfanalyzer: <a href="https://github.com/alexherbig/MultiVCFAnalyzer">https://github.com/alexherbig/MultiVCFAnalyzer</a>. I am basically trying to call SNPs and to get a full alignment from several bams. For that, I used Eager 2.5 for all the mapping, filtering, and genotyping steps. Unfortunately at the snp table generation step I always get an error message from MultiVCFAnalyzer. I replicate the issue with versions 0.85.1 and 0.85.2 of MultiVCFAnalyzer. I always get this error message which I really do not understand:     MultiVCFAnalyzer - 0.85.2
by Alexander Herbig</p>

<p>No positions to exclude provided! All positions will be used!
Now processing 1/182: 0ad1bd59442cf28fa247368a0a9603
Illegal arguments in function getAmbiguousBase: R, G
Illegal arguments in function getAmbiguousBase: M, A
Illegal arguments in function getAmbiguousBase: Y, T
Illegal arguments in function getAmbiguousBase: S, C
Illegal arguments in function getAmbiguousBase: W, A
Illegal arguments in function getAmbiguousBase: K, G
Illegal arguments in function getAmbiguousBase: K, T
Illegal arguments in function getAmbiguousBase: S, C
Illegal arguments in function getAmbiguousBase: K, G
Illegal arguments in function getAmbiguousBase: Y, T
500000 positions processed.
Illegal arguments in function getAmbiguousBase: W, A
Illegal arguments in function getAmbiguousBase: K, T
Illegal arguments in function getAmbiguousBase: W, A
Illegal arguments in function getAmbiguousBase: R, A
Illegal arguments in function getAmbiguousBase: W, T
1000000 positions processed.
Illegal arguments in function getAmbiguousBase: S, C
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 1178035
        at MultiVCFAnalyzer.main(MultiVCFAnalyzer.java:337) .                                                                                                                                                                                                                                                                                                                                                    I was wondering if some other people got the same errors and how did you manage to fix them? And if not, what are you using instead of MultiVCFAnalyzer (especially a tool which have "Minimal allele frequency for homozygous call" and "Minimal allele frequency for heterozygous call"  options). Thanks in advance ü¶† !!</p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">alexherbig/MultiVCFAnalyzer</a></div>
                                <div class="link-text">
                                    
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Stars</div>
                                        8
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Java
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:13:38">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-02-13 12:13:38"><div class="time">2024-02-13 12:13:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi, I have seen similar errors but not that specific one. Which genotyper are you using? </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:14:33">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:14:33"><div class="time">2024-02-13 12:14:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* GATK UnifiedGenotyper</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:17:03">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-02-13 12:17:03"><div class="time">2024-02-13 12:17:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which parameters are you using for the genotyping step? I think I‚Äôve seen something like this when the vcfs are not exactly how multivcfanalyzer expects them</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:17:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-13 12:17:46"><div class="time">2024-02-13 12:17:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Which reference genome are you using (if you can say)?</p>

<p>MultiVCFAnalyzer doesn't work with multi-chromosome references...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:18:24">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-13 12:18:24"><div class="time">2024-02-13 12:18:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But honestly the best way will be to email Alex Herbig... üòÖ the error messages are renowned for being useless</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:19:57">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:19:57"><div class="time">2024-02-13 12:19:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* --run<em>genotyping --genotyping</em>tool 'ug' --gatk<em>ploidy 2 --gatk</em>ug<em>out</em>mode 'EMIT<em>ALL</em>SITES' --gatk<em>ug</em>genotype_model 'SNP'</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:20:39">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:20:39"><div class="time">2024-02-13 12:20:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It is a bacteria reference genome with 2 chromosomes, so it could because of that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:21:07">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-02-13 12:21:07"><div class="time">2024-02-13 12:21:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, that looks right. Yeah, the multiple chromosome is a thing </p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§î Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:22:07">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:22:07"><div class="time">2024-02-13 12:22:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, i might try again with only one chromosome to see if the issue is coming from that</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:22:11">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:22:11"><div class="time">2024-02-13 12:22:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks a lot</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 12:22:17">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 12:22:17"><div class="time">2024-02-13 12:22:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I let you know</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 17:25:43">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 17:25:43"><div class="time">2024-02-13 17:25:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Update, it is working when i divided the work by chromosome, thnaks a lot!</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 17:26:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-13 17:26:58"><div class="time">2024-02-13 17:26:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* &gt; Ok, that looks right. Yeah, the multiple chromosome is a thing
@Zo√© Pochon (saw your reaction), MultiVCFAnalyzer was apparently never meant to be proper software, it was sort of a 'throwaway script' which kept being used üòÖ</p>

<p>Aida and I were talking about it the other day and realised probably just can do the same thing (more reliability) with a wrapper around <code>bcftools consensus</code> if we come up with the correct filtering conditions...</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòä Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 17:27:07">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-13 17:27:07"><div class="time">2024-02-13 17:27:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (although won't get the SNP table etc, but still)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 17:44:13">
            
                <img src="https://avatars.slack-edge.com/2022-10-25/4275640433060_c233df9e8212ec3fb238_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Louis L&#39;H√¥te
                     <span class="print-only user-email">(lhtel@tcd.ie)</span>
                </div>
                <a href="#2024-02-13 17:44:13"><div class="time">2024-02-13 17:44:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, i was thinking about using a handmade script, but I do not think that any other software allows you to play with the heterozygosity, is it?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-13 17:48:18">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-13 17:48:18"><div class="time">2024-02-13 17:48:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* As long as you use a variant caller that allows reporting of multiple alleles, bcftools should allow (if you work out the filtering conditions,). You run that on each of your genomes and then cat the resulting fasta, you've pretty much got it... In theory...</p>

<p>I wouldn't make a whole script to parse the vcf etc when bcftools probably does it really well already</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 00:52:55">
            
                <img src="https://secure.gravatar.com/avatar/237033a57bbaa764381ab3b02c376e2b.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Karissa Hughes
                     <span class="print-only user-email">(karissa.s.hughes-1@ou.edu)</span>
                </div>
                <a href="#2024-02-16 00:52:55"><div class="time">2024-02-16 00:52:55</div></a>
                <div class="msg">
                    <p>Hi everyone! This is my first post in this group so hello and nice to meet you all virtually! I have many no-stupid-questions questions!
I‚Äôm looking for some clarification when it comes to using Kraken2 and Ganon. I‚Äôm going to ask my Ganon questions later. Today is about the beast, kraken.</p>

<p>The PDF will have most of this along with the tables and stuff that didn‚Äôt want to copy over well.</p>

<p>Just as some background, we are trying to use read classification tools for multiple aDNA projects - so keep in mind, everything is short and damaged. We‚Äôve built one database that has fish species (along with human and Unive_Core) in the hopes of being able to use it to identify samples to the species level for fish bones that are morphologically hard to distinguish in archeological contexts. We‚Äôve built a much larger database with ALL the RefSeq genomes (limited to 3 assemblies per species). We were hoping to use this RefSeq db for sediment and coprolite projects - some examples of what we are trying to find include environment/host microbial communities, host diet, and for some of the sediment stuff we want to see if we can identify animal community too.</p>

<p>For Kraken2, does anyone have experience building Kraken2 databases with different kmer and minimizer parameters? We noticed with some of our stuff, shorter reads weren‚Äôt being classified when using the default parameters, so we wanted to try shorter kmer lengths than the default, and they just don‚Äôt seem to be classifying reads as well, especially to lower taxonomic ranks like genus and species.</p>

<p>Here is some information on the build parameters used for 2 builds with the same references as input:
--kmer-len 25 --minimizer-len 21 --minimizer-spaces 4
‚Ä¢ 2,552,952,570 total kmers
‚Ä¢ 14.5 GB Estimated hash table requirement
--kmer-len 35 --minimizer-len 31 --minimizer-spaces 7
‚Ä¢ 9,600,970,913 total kmers
‚Ä¢ 54 GB Estimated hash table requirement
Question1: Why does the 35 kmer db have so many more total kmers? I‚Äôve always thought that if you decrease kmer length, you‚Äôd have more kmers and larger database files, but I‚Äôm seeing the opposite, please help me understand!</p>

<p>Moving on, I tested the two dbs by running one sample through each db with the same classification parameters (minimum hit 4, confidence 0.10):</p>

<p>Question2: Does anyone have recommendations for minimum hit groups and confidence thresholds for aDNA reads? And can you tell me why those are your recs? I‚Äôve done some tests on simulated data (with damage) using different parameters, but I‚Äôd love your input too.
‚Ä¢ I know confidence thresholds can help balance between filtering out spurious matches and allowing for potential errors due to ancient damage. So what confidence thresholds are aDNA ppl using (Q2.a1)? And would you all recommend using different confidence thresholds when trying to identify microbial community vs animal species (like the classifying reads to fish species/host diet stuff)? (Q2.a2)
‚Ä¢ I‚Äôve been thinking lately that maybe a minimum hit of 4 might be too high for shorter reads and lowering it could increase the likelihood of classifying shorter reads. Are people just using the default (i think its 2)? (Q2.b1)
    ‚ó¶ Bonus points if someone can reassure me that I‚Äôm thinking of minimum hit groups correctly (Q2.b2):
        ‚ñ™Ô∏é see PDF for options/example
            ‚Ä¢ I thought it was option1 when I was first testing these parameters, but after reading the brief description of it in the kraken2 readme a few months back, I think it is option2 now (love that I probably misinterpreted this initially).
See PDF for tables with results from the test of the two dbs. I ran one sample through each db with the same classification parameters (minimum hit 4, confidence 0.10):
‚Ä¢ It initially seems like the 25 kmer db is classifying more reads, which is great. But it seems like a lot of those reads are being classified to higher taxonomic levels that aren‚Äôt as informative for our research qs like order, suborder, and family. And when you get to the species level, it seems like the 25kmer db decreases in precision and sensitivity (still don‚Äôt fully understand how these 2 benchmarking terms are different in total honesty). It‚Äôs top 2 species hits were to species that aren‚Äôt even in the correct genus!
‚Ä¢ Q3: For people who have been using shorter kmer lengths, are you noticing this pattern too? Have you found a way to mitigate it whether that is using different build or classification parameters? Because theoretically it would be nice to have a shorter kmer length to be able to classify shorter reads, but at the same time I don‚Äôt really want to sacrifice precision and sensitivity.
Looking forward to your input and expertise!!!</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06JWURP2CW/kraken2-qs.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Kraken2-Qs.pdf</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06JWURP2CW/kraken2-qs.pdf?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06JWURP2CW-3ee3e37c32/kraken2-qs_thumb_pdf.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="935"
                 height="1210" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 09:31:24">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-02-16 09:31:24"><div class="time">2024-02-16 09:31:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Karissa Hughes, a lot of questions, ha-ha üôÇ I will come to some of them a bit later (do not have much time right now), but your question 1 is more or less obvious, so I will start with this one. The numbers of k-mers that you mention in your post are related to the numbers of possible combinations of constructing 25- and 35-character words out of 4 letters (nucleotides). For example, you can construct 4^2=16 possible 2-character words out of 4 letters (A, C, T, G), i.e. you can pick 16 different pairs of letters out of 4 possible letters. If you want to construct 5-character long words using only 4 letters, you can build 4^5=1024 such words. And so on. So the longer words you would like to build out of a fixed number of letters, the larger "dictionary" you will get, so the database size will increase if one wants to achieve higher analysis specificity and use longer k-mers. Now, in your case 4^25 and 4^35 are much greater numbers than the ones you reported in your post. I believe this is because not all <strong>possible</strong> k-mers are used in the database in the end. Probably some theoretically possible k-mers were never detected in your reference genomes, this is one reason. Another reason is that, for example in text analysis, people usually severely prune their dictionaries and keep only most common words, so ultra-rare k-mers (e.g. seen in just one reference) were probably dropped from your final database. Hope this helps for the beginning, and as I said I will try to come back to your other questions a bit later when I have more time</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 14:03:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-16 14:03:29"><div class="time">2024-02-16 14:03:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Q2: we need to find an alternative to Slack and its limitation to 90 days of message archiving on a free plan, because this has been answered multiple times already üòû But TLDR: using a combination of number of reads, and duplication rate is a good starting point. In Kraken2 it‚Äôs slightly different , but as food for thoughts, this is a starting point <a href="https://maximeborry.com/post/kraken-uniq/">https://maximeborry.com/post/kraken-uniq/</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Maxime Borry</div>
                            
                            
                                
                                <div class="link-title"><a href="https://maximeborry.com/post/kraken-uniq/">A new E-score for KrakenUniq | Maxime Borry</a></div>
                                <div class="link-text">
                                    How to distinguish between true and false positive assignations with KrakenUniq
                                </div>
                                
                                
    
        <a href="https://maximeborry.com/post/kraken-uniq/">
            <img class="preview" src="https://maximeborry.com/post/kraken-uniq/featured.png"
                width="731"
                 height="486" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://maximeborry.com/post/kraken-uniq/</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 14:07:38">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-16 14:07:38"><div class="time">2024-02-16 14:07:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Q3: This behaviour is expected.
Shorter k-mers are less specific, meaning that they will be found more often than longer ones.
So you would get more classified reads, especially if you have sequences in your dataset that are not represented directly in your reference database.
However, because these are less specific, they will be found in more reference sequences than longer k-mer. And the more you find a given k-mer in different refs, the more the LCA will send it back to less precise taxonomic ranks.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 14:09:45">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-02-16 14:09:45"><div class="time">2024-02-16 14:09:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Given a 4 base alphabet (A,T,G,C), the probability of finding a given k-mer of k is (1/4)^k</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 15:04:51">
            
                <img src="https://secure.gravatar.com/avatar/003433d8ddbe540ee811de3c25aaaace.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Antonio Fernandez-Guerra
                     <span class="print-only user-email">(antonio@metagenomics.eu)</span>
                </div>
                <a href="#2024-02-16 15:04:51"><div class="time">2024-02-16 15:04:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This one also seems to be a nice way of post processing kraken2 results <a href="https://www.biorxiv.org/content/10.1101/2024.02.02.578701">https://www.biorxiv.org/content/10.1101/2024.02.02.578701</a> haven‚Äôt tried it though 
<a href="https://github.com/cdiener/architeuthis">https://github.com/cdiener/architeuthis</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">bioRxiv</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.biorxiv.org/content/10.1101/2024.02.02.578701">Metagenomic estimation of dietary intake from human stool</a></div>
                                <div class="link-text">
                                    Dietary intake is tightly coupled to gut microbiota composition, human metabolism, and to the incidence of virtually all major chronic diseases. Dietary and nutrient intake are usually quantified using dietary questionnaires, which tend to focus on broad food categories, suffer from self-reporting biases, and require strong compliance from study participants. Here, we present MEDI (Metagenomic Estimation of Dietary Intake): a method for quantifying dietary intake using food-derived DNA in stool metagenomes. We show that food items can be accurately detected in metagenomic shotgun sequencing data, even when present at low abundances (&amp;gt;10 reads). Furthermore, we show how dietary intake, in terms of DNA abundance from specific organisms, can be converted into a detailed metabolic representation of nutrient intake. MEDI could identify the onset of solid food consumption in infants and it accurately predicted food questionnaire responses in an adult population. Additionally, we were able to identify specific dietary features associated with metabolic syndrome in a large clinical cohort, providing a proof-of-concept for detailed quantification of individual-specific dietary patterns without the need for questionnaires.

### Competing Interest Statement

The authors have declared no competing interest.
                                </div>
                                
                                
    
        <a href="https://www.biorxiv.org/content/10.1101/2024.02.02.578701">
            <img class="preview" src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.biorxiv.org/content/10.1101/2024.02.02.578701</div>
                                
                                
                            
                        </div>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">cdiener/architeuthis</a></div>
                                <div class="link-text">
                                    Tools to analyze and summarize data for Kraken2.
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Language</div>
                                        Go
                                    </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Last updated</div>
                                        5 months ago
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-16 20:02:54">
            
                <img src="https://secure.gravatar.com/avatar/237033a57bbaa764381ab3b02c376e2b.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0016-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Karissa Hughes
                     <span class="print-only user-email">(karissa.s.hughes-1@ou.edu)</span>
                </div>
                <a href="#2024-02-16 20:02:54"><div class="time">2024-02-16 20:02:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you all for your replies so far! I appreciate it! I'll take a look at all the material you've shared before I ask any follow up qs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:08:09">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:08:09"><div class="time">2024-02-20 11:08:09</div></a>
                <div class="msg">
                    <p>Hello everyone! It's my first post here, as I'm new to the community, and I have questions about the worflow eager. I'm currently a MSc student doing an internship and I'm working on ancient data of human dental calculus.
I built my tsv file like this :
Sample<em>Name     Library</em>ID      Lane    Colour<em>Chemistry        SeqType Organism        Strandedness    UDG</em>Treatment   R1      R2      BAM
ROU2014 lr41    1       2       SE      Human   double  none    /work/project/GenIn/ROUSILLE/data/concat<em>samples/ROU2014</em>lr41.fastq.gz  NA      NA
ROU2030A        lr46    1       2       SE      Human   double  none    /work/project/GenIn/ROUSILLE/data/concat<em>samples/ROU2030A</em>lr46.fastq.gz NA     NA
ROU2031 lr48    1       2       SE      Human   double  none    /work/project/GenIn/ROUSILLE/data/concat<em>samples/ROU2031</em>lr48.fastq.gz  NA      NA</p>

<p>I specified the SeqType as SE because my reads have been trimmed and collapsed, even though they are originally paired-end. I skipped adapterRemoval and followed the tutorial on metagenomics. For the MALT database I used the script "099-refseq<em>genomes</em>bacteria<em>archaea</em>homo<em>complete</em>chromosome<em>scaffold</em>walkthrough_20181122.Rmd" from the article "The evolution and changing ecology of the African hominid oral microbiome" from 2021.</p>

<p>I'm wondering if I specified correctly the SeqType and if I have to put that they were originally paired end what should I put in the R1 and R2 columns?</p>

<p>I ran the workflow on the tsv file and it processes correctly until the maltextract step where I get the error "Danger empty keys in File" when eager try to read my rma6 files. I tried to visualize one of them in MEGAN but I'm not used to this software so I can't see any issues with my rma file.</p>

<p>If you could help me with those steps, I would be very glad. Thank you for the opportunity to exchange with the community.</p>

<p>Best regards,
Mathias CAIRE</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëã James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:10:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:10:57"><div class="time">2024-02-20 11:10:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Mathias Caire! For nf-core/eager specific questions we have a dedicated pipeline slack channel you can access via the nf-core slack: <a href="https://nf-co.re/join">https://nf-co.re/join</a></p>

<p>However I'm in both so can answer hereüôÇ</p>

<p>For the SeqType, you're correct there to set as SE if they are already collapsed üëç</p>

<p>For the maltextract error, can you supply your full eager command? If possible please put in a code block (put triple backticks befroe and after the command)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:11:23">
            
                <img src="https://avatars.slack-edge.com/2023-08-11/5727013151811_5866d2031f65a72cd9a6_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ian Light
                     <span class="print-only user-email">(light@mpiib-berlin.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:11:23"><div class="time">2024-02-20 11:11:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* maltextract can be pretty wonky -- if you get the rma6 files to your local machine and open them with MEGAN you could see if the issue is with the general MALT step or with maltextract specifically</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è James Fellows Yates, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:12:36">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:12:36"><div class="time">2024-02-20 11:12:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And for the code block: <a href="https://commonmark.org/help/tutorial/09-code.html#:~:text=To%20create%20a%20code%20block,and%20below%20the%20code%20block.&amp;text=A%20code%20block%20or%20span,4%20spaces%20or%20one%20tab">https://commonmark.org/help/tutorial/09-code.html#:~:text=To%20create%20a%20code%20block,and%20below%20the%20code%20block.&amp;text=A%20code%20block%20or%20span,4%20spaces%20or%20one%20tab</a>.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:26:45">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:26:45"><div class="time">2024-02-20 11:26:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you, I'm relieved that I specified correctly the SeqType.
The eager command of maltExtract is:
```#!/bin/bash -euo pipefail
MaltExtract   -Xmx1024g   -t core<em>genera-anthropoids</em>hominids<em>panhomo-20180131.txt   -i lr41.unmapped.rma6 lr46.unmapped.rma6 lr48.unmapped.rma6   -o results/   -r hops   -p 64   -f def</em>anc   -a 0.01   --minPI 85.0   --destackingOff</p>

<p>postprocessing.AMPS.r -r results/ -m def<em>anc -t 64 -n core</em>genera-anthropoids<em>hominids</em>panhomo-20180131.txt -j```
Do you need more informations about how I ran the workflow?
I also registered on the nf-core slack, do we continue the conversation there?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:32:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:32:22"><div class="time">2024-02-20 11:32:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Let's keep it in one place for now, and stay here but in the future we can switch</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:32:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:32:49"><div class="time">2024-02-20 11:32:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can you <code>head</code> the <code>core_genera_.....</code>  file?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:34:14">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:34:14"><div class="time">2024-02-20 11:34:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I simply took the one from the tutorial and the whole file is :</p>

<pre><code>Streptococcus
Tannerella
Porphyromona
</code></pre>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:34:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:34:53"><div class="time">2024-02-20 11:34:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What MALT database did you use/</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:37:50">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:37:50"><div class="time">2024-02-20 11:37:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I built it thanks to your script :
"099-refseq<em>genomes</em>bacteria<em>archaea</em>homo<em>complete</em>chromosome<em>scaffold</em>walkthrough_20181122.Rmd"
So it's a RefSeq database of bacteria archaea and homo sapiens with 56.104 genomes in it, I indexed it with malt-buid</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:38:04">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:38:04"><div class="time">2024-02-20 11:38:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What version of MALT?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:38:27">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:38:27"><div class="time">2024-02-20 11:38:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I used MALT 0.6.2</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:38:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:38:31"><div class="time">2024-02-20 11:38:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* (NGL I'm extremely impressed that you've been able to understand that horrific github mess üòÜ )</p>
                    
                    
                    
                        <div class="message-reaction">
                        üòÜ Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:38:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:38:35"><div class="time">2024-02-20 11:38:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, that could be the problem</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:38:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:38:49"><div class="time">2024-02-20 11:38:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What was the <code>malt-build</code> command?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:39:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:39:05"><div class="time">2024-02-20 11:39:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You likely will need to re-build the database but with the same or older verison of MALT as I used...</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:39:48">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:39:48"><div class="time">2024-02-20 11:39:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I work on the genotoul cluster so it's a SLURM script :
```#!/bin/bash</p>

<h1>SBATCH -p workq</h1>

<h1>SBATCH -t 01-00:00:00</h1>

<h1>SBATCH --cpus-per-task 64</h1>

<h1>SBATCH --mem=2000G</h1>

<h1>SBATCH -J "malt_indexing"</h1>

<h1>SBATCH --output=slurm<em>output/indexing</em>%A.out</h1>

<h1>Warning! Default threads number:64 (don't forget to adjust -t options of malt command and --cpus-per-task for Slurm).</h1>

<h1>Load modules</h1>

<p>module load bioinfo/MALT/0.6.2</p>

<p>malt-build -t 64 -i references/<strong>.</strong> -d index --sequenceType DNA -J-Xmx2000g```</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:40:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:40:23"><div class="time">2024-02-20 11:40:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No <code>--mapdb</code>? or <code>--acc2tax</code>?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëÄ Ian Light
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:40:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-20 11:40:48"><div class="time">2024-02-20 11:40:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That may be your problem in this case</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 11:43:24">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 11:43:24"><div class="time">2024-02-20 11:43:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If I'm honnest, I am not familiar with those options, I will investigate this thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-20 16:11:22">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-20 16:11:22"><div class="time">2024-02-20 16:11:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey, me again! I do need to specify a file mapping RefSeq identifiers to taxon ids. Upon searching, I found tables that link accession numbers to taxon ids at
<code>&lt;https://ftp.ncbi.nih.gov/pub/taxonomy/accession2taxid/&gt;</code>
However, it seems they are using GenBank accession numbers. Do you happen to know where I could find the equivalent for RefSeq accessions?
I have the taxid of the assemblies I used to build the MALT database, so I'm currently creating a custom table, but I'm unsure about which format to use.
Thank you very much for your assistance.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:06:52">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-21 09:06:52"><div class="time">2024-02-21 09:06:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I <em>think</em> the genbank and refseq accessions are in the same file</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:08:50">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-21 09:08:50"><div class="time">2024-02-21 09:08:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* nucl_wgs.accession2taxid.gz
TaxID mapping for live nucleotide sequence records of type WGS or TSA.</p>

<p>nucl_gb.accession2taxid.gz
TaxID mapping for live nucleotide sequence records that are not WGS or TSA</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:09:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-21 09:09:01"><div class="time">2024-02-21 09:09:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* From the acc2tax README</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:09:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-21 09:09:40"><div class="time">2024-02-21 09:09:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's not raw reads (WGS) as refseq should be assemblies so presumably that counts as the second?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:10:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-21 09:10:14"><div class="time">2024-02-21 09:10:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Refseq is just a 'cleaned up' genbank</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:18:03">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-02-21 09:18:03"><div class="time">2024-02-21 09:18:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for the time you're dedicating to me, it helps me a lot. I will try this.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-21 09:23:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-21 09:23:01"><div class="time">2024-02-21 09:23:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You're welcome! I'll be AFK from tonight for a couple of days as we're flying to Australia, but if you have more questions just make a new post on this channel and someone else can help you :)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-22 14:50:37">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2024-02-22 14:50:37"><div class="time">2024-02-22 14:50:37</div></a>
                <div class="msg">
                    <p>I goofed real bad today and started a double-stranded partial-UDG lib prep (<a href="https://pubmed.ncbi.nlm.nih.gov/25487342/">https://pubmed.ncbi.nlm.nih.gov/25487342/</a>) with only enough T4 PNK for half the samples. After the UGI incubation, I put half in the freezer. Does anyone know if they are stable at this point? Can I continue with blunt end repair in a week or so when the T4 PNK arrives?</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">PubMed</div>
                            
                            
                                
                                <div class="link-title"><a href="https://pubmed.ncbi.nlm.nih.gov/25487342/">Partial uracil-DNA-glycosylase treatment for screening of ancient DNA - PubMed</a></div>
                                <div class="link-text">
                                    The challenge of sequencing ancient DNA has led to the development of specialized laboratory protocols that have focused on reducing contamination and maximizing the number of molecules that are extracted from ancient remains. Despite the fact that success in ancient DNA studies is typically obtaine ‚Ä¶
                                </div>
                                
                                
    
        <a href="https://pubmed.ncbi.nlm.nih.gov/25487342/">
            <img class="preview" src="https://cdn.ncbi.nlm.nih.gov/pubmed/persistent/pubmed-meta-image-v2.jpg"
                width="1600"
                 height="830" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://pubmed.ncbi.nlm.nih.gov/25487342/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-22 17:26:14">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2024-02-22 17:26:14"><div class="time">2024-02-22 17:26:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* In theory, this should be fine as there is no immediate need to end repair the partial-UDG treated DNA. Good luck!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå:skin_tone_2: Kelly Blevins
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-23 12:00:56">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2024-02-23 12:00:56"><div class="time">2024-02-23 12:00:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Awesome. This is what I was hoping! Thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 17:28:57">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-26 17:28:57"><div class="time">2024-02-26 17:28:57</div></a>
                <div class="msg">
                    <p>Hi there, I am trying to index my bacterial database with MALT and I consistently get java errors. I increased the malt memory in malt-build.vmoptions and also ran ‚Äúset JAVA<em>OPTS=%JAVA</em>OPTS% -Xms1024m -Xmx1024m‚Äù
I am still getting ‚Äú<strong>java.lang.OutOfMemoryError: Java heap space</strong>‚Äù Does anyone know how to solve it or where the problem may be?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:13:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:13:41"><div class="time">2024-02-26 23:13:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For the VM options there you've only set a gigabyte of memory, is that correct?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:14:18">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:14:18"><div class="time">2024-02-26 23:14:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also I think it's not good to set both of those parameters in Java (I can't remember exactly, Java memory management is a mystery for many people)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:14:25">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:14:25"><div class="time">2024-02-26 23:14:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* so in the past we'vce set it like:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:14:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:14:47"><div class="time">2024-02-26 23:14:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* <code>malt-run J-Xmx1800G &amp;lt;....&amp;gt;</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:14:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:14:57"><div class="time">2024-02-26 23:14:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That sets for to 1.8TB</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:19:36">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-26 23:19:36"><div class="time">2024-02-26 23:19:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I set it to 64GB and a few other values just to see what works, if anything, and since it did not work I also changed the other parameter. I will try to run it again using your suggestion</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:20:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:20:40"><div class="time">2024-02-26 23:20:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* How many bacterial genomes</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:20:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:20:41"><div class="time">2024-02-26 23:20:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:30:40">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-26 23:30:40"><div class="time">2024-02-26 23:30:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I will know tomorrow and get back to u!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:31:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:31:11"><div class="time">2024-02-26 23:31:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok! Because 64GB seems quite small, we normally run for e.g. &gt;1000 genomes a couple of hundred of GB</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:31:40">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-26 23:31:40"><div class="time">2024-02-26 23:31:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* okay then I guess it was too small. I am using all complete genomes from the ncbi</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:37:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:37:42"><div class="time">2024-02-26 23:37:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ooooooh yeah you'll need more than that üòÖ you can also reduce the step size a bit to reduce the memory requirements. In 2017 we needed 300GB for a similar database</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:38:15">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-26 23:38:15"><div class="time">2024-02-26 23:38:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* what do you mean by step size</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:38:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:38:48"><div class="time">2024-02-26 23:38:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* From the malt-build documentation:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:38:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:38:51"><div class="time">2024-02-26 23:38:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06M3GATB5X/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06M3GATB5X/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06M3GATB5X-ac9a58a5a4/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="32" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:39:46">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-26 23:39:46"><div class="time">2024-02-26 23:39:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's the size of the seed offset. You lose detection/alignment sensitivity slightly, but it's minor if you set to a step size of 4 or 8 (any more of that you get a big drop off)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-26 23:40:49">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-26 23:40:49"><div class="time">2024-02-26 23:40:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I will play around with these on Thursday when I try to rerun the indexing.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-28 21:09:48">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-28 21:09:48"><div class="time">2024-02-28 21:09:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I increased the memory to 1200GB and I get this.. ‚ÄúOpening file: table0.db
Allocating: 1010 GB
java.lang.OutOfMemoryError: Java heap space‚Äù and ‚ÄúException in thread ‚ÄúSync Timer Thread‚Äù java.lang.SecurityException: Could not lock User prefs. Lock file access denied.‚Äù ‚Ä¶ T.T</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-28 21:16:37">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-28 21:16:37"><div class="time">2024-02-28 21:16:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There is some progress tho because the table0.db file is not empty. But that‚Äôs it</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-28 21:53:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-28 21:53:03"><div class="time">2024-02-28 21:53:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* With the -J parameters?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-28 21:54:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-28 21:54:40"><div class="time">2024-02-28 21:54:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But your might need to keep going higher... Did you try dropping the step size too?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-28 23:15:13">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-02-28 23:15:13"><div class="time">2024-02-28 23:15:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It complained about the -J command, maybe I wrote something wrong. I went higher and it is running. Complained ‚ÄúException in thread ‚ÄúTimer-0‚Äù but despite that it keeps running.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-02-29 01:09:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-02-29 01:09:10"><div class="time">2024-02-29 01:09:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok I <em>think</em> you can ignore the time-0... but let's see if it completes...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-10 19:40:34">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-03-10 19:40:34"><div class="time">2024-03-10 19:40:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Update: increasing memory worked and I screened one sample. Now I am trying other samples and it does not work.. It just stops midway. I get the output file, but I think it is incomplete. I get no error at all. I increased the memory further and nothing, I will try to play around with the step size as you suggested</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-10 19:47:27">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-03-10 19:47:27"><div class="time">2024-03-10 19:47:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah step option is just for indexing xD I do not know why the screening does not work for all samples. Maybe not enough memory..</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 14:52:05">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-11 14:52:05"><div class="time">2024-03-11 14:52:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Are you submitting to a HPC?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-01 18:43:17">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-01 18:43:17"><div class="time">2024-03-01 18:43:17</div></a>
                <div class="msg">
                    <p>Hi!  I need a little orientation with MALT anyone here had been working with it?       The  issue is:  I am using the version 0.6.2 of MALT and in my command I need to use "--classify Taxonomy", but it is not  working because it said it does not exist, in the terminal, but it exist in the manual for that version. So I can run it with a lower version as 0.4.0 but not with the 5 or the last one as 6.2. This variable "--classify Taxonomy" is important in my command because with out it does not make the correct assignation. Also I have change it just putting -c Taxonomy and in the complete way. I'll be glad to hear some advice.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-01 23:11:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-01 23:11:23"><div class="time">2024-03-01 23:11:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Don't trust the manual, it's very weirdly out of date - check the help message. Also don't use the mapdb it mentions, it's broken. Use the deprecated acc2tax flag (it still works) with the ncbi acc2tax map</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 21:43:46">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-02 21:43:46"><div class="time">2024-03-02 21:43:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I hava the same error with acc2tax.  And my command is malt-build -i Virus<em>all</em>seq.fasta.gz -d index<em>prueba</em>tax -s DNA --acc2tax NCBI.txt       Also I read the samen problem in MEGAN community but no one answer, I just replay it to know if the person could fix it.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06MJCTB493/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06MJCTB493/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06MJCTB493-821dd267b5/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="90" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 21:44:11">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-02 21:44:11"><div class="time">2024-03-02 21:44:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Was this one, dated last year <a href="https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2">https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">MEGAN Community</div>
                            
                            
                                
                                <div class="link-title"><a href="https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2">Malt-build assign wrong taxonID on some reference genomes</a></div>
                                <div class="link-text">
                                    Could you resolve it? I have the same problem and even using acc2tax is not working in the version 0.6.2 says it does not exist. üôÅ
                                </div>
                                
                                
    
        <a href="https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2">
            <img class="preview" src="https://megan.cs.uni-tuebingen.de/uploads/default/original/1X/225133463d361fb3d7dd183b3f3f2da17bef5813.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 22:36:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-02 22:36:20"><div class="time">2024-03-02 22:36:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm ok, I definitely used it a couple of weeks ago, when I get to my laptop I'll share with you the command I used...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 22:36:40">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-02 22:36:40"><div class="time">2024-03-02 22:36:40</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ahh wait, try the short version of the parameter!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 22:36:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-02 22:36:48"><div class="time">2024-03-02 22:36:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Like -a2t or something!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 22:36:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-02 22:36:54"><div class="time">2024-03-02 22:36:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think that was another bug...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-02 22:39:06">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-02 22:39:06"><div class="time">2024-03-02 22:39:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I use -a2t and know is running I'll wait to the result</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§û James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-03 20:30:14">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-03 20:30:14"><div class="time">2024-03-03 20:30:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It works! thanks üòÑ</p>
                    
                    
                    
                        <div class="message-reaction">
                        :mask_parrot: James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 16:57:21">
            
                <img src="https://secure.gravatar.com/avatar/63b92c9d75dc9b86b64c65e4304c4413.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Yuejiao Huang
                     <span class="print-only user-email">(hsf378@alumni.ku.dk)</span>
                </div>
                <a href="#2024-03-04 16:57:21"><div class="time">2024-03-04 16:57:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi James, I have the exact same problem as this post (some phage such as S. mitis phage (AY007505.3) mis-assigned into S.mitis (taxid 28037)): <a href="https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2">https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2</a>
 I am using MALT/0.6.2, any idea to solve it? Thanks !</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">MEGAN Community</div>
                            
                            
                                
                                <div class="link-title"><a href="https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2">Malt-build assign wrong taxonID on some reference genomes</a></div>
                                <div class="link-text">
                                    Could you resolve it? I have the same problem and even using acc2tax is not working in the version 0.6.2 says it does not exist. üôÅ
                                </div>
                                
                                
    
        <a href="https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2">
            <img class="preview" src="https://megan.cs.uni-tuebingen.de/uploads/default/original/1X/225133463d361fb3d7dd183b3f3f2da17bef5813.png"
                
                 />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://megan.cs.uni-tuebingen.de/t/malt-build-assign-wrong-taxonid-on-some-reference-genomes/2169/2</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 17:21:34">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-04 17:21:34"><div class="time">2024-03-04 17:21:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That was my question too üòÖ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:02:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:02:51"><div class="time">2024-03-04 21:02:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Unfortunately, no I don't know. I realise now I forgot to say in my first reply: I don't really recommend MALT anymore because there seem to be a lot of bugs... (Sorry about that @Ania TSL</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:03:39">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:03:39"><div class="time">2024-03-04 21:03:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It might be worth also replying saying it's a problem, the more people saying they have this issue, also in later versions of malt, the more likely it would be fixed</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:18:39">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-04 21:18:39"><div class="time">2024-03-04 21:18:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Some recommended HOPS as a kind of substitution of MALT but I need to learn a bit more of it, also right now it is working so maybe if I have more problems I'll check it with more details.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:19:21">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:19:21"><div class="time">2024-03-04 21:19:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* HOPS is a pipeline that has MALT in it</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:19:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:19:42"><div class="time">2024-03-04 21:19:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You can try using the last reported version of the malt which I think was either 0.38 or 0.40</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:20:18">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-04 21:20:18"><div class="time">2024-03-04 21:20:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Yuejiao Huang use -a2t with the accession numbers for the NCBI and it works</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:46:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:46:22"><div class="time">2024-03-04 21:46:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh interesting! @Ania TSL maybe you can post that on the forum thread?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-05 23:11:59">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-05 23:11:59"><div class="time">2024-03-05 23:11:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes I did it.</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-07 04:12:57">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-03-07 04:12:57"><div class="time">2024-03-07 04:12:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could you share your final version of command  if you solved this problem?:) I am also trying to build a malt custom databaseü•π.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-26 00:28:39">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-03-26 00:28:39"><div class="time">2024-03-26 00:28:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi, sorry for my late response I did not notice the message until now, are you still working on it?  The final version is: <strong>malt-build -i</strong> ~Virus<em>all</em>seq.fasta.gz~ <strong>-d index<em>prueba</em>tax -s DNA -a2t ~NCBI.txt~</strong>                                                                              Let me know if it works for you @Wenqin Yu</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 05:02:21">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-01 05:02:21"><div class="time">2024-04-01 05:02:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for your reply, Alitery! I‚Äôve tried MALT for a long time, but it always returned a memory space error unless I provided a database that only includes viruses. Then I tried HAYSTAC, and it seems to be working so far. Thanks you again!!! :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 19:59:11">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-04-01 19:59:11"><div class="time">2024-04-01 19:59:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I use #$ -l vf=10G</p>

<h1>$ -pe openmp 20</h1>

<p>export omp<em>num</em>threads=24                       You need to be sure that you have enough space, If you want we could have a meet by zoom to check specific points.  I am working with a db of virus too.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-02 13:23:24">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-02 13:23:24"><div class="time">2024-04-02 13:23:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It doesn't seem too resource-intensive! Thanks for your sharing, Ania! I'd like to hear more about your settings at your convenience.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-03 18:08:53">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-03-03 18:08:53"><div class="time">2024-03-03 18:08:53</div></a>
                <div class="msg">
                    <p>Hi all! I am trying to run pydamage on my assembled ancient metagenomic data, however, when I do the filter step (for pydamage analyse I used the flag -w 30 and for the filter -t 0.67) I end up with few contigs (62 out of 173459). I already analyzed this sample to be sure that it contains ancient sequences, so I was not expecting this result. Does anyone have this same problem with assembled ancient metagenomic data? I am attaching a plot with the result for one of the contigs that was removed after the filter step.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06N1A8N7A5/node_104_length_37285_cov_23.743557.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">NODE_104_length_37285_cov_23.743557.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06N1A8N7A5/node_104_length_37285_cov_23.743557.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06N1A8N7A5-6c7d0b0531/node_104_length_37285_cov_23.743557_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="270" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-03 20:42:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-03 20:42:03"><div class="time">2024-03-03 20:42:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The pale pink line of the actual frequency looks a bit funky, it implies none of your reads have a C-T on the first position, it's that to be expected? We often see that when there is a artefact left on the read</p>

<p>Otherwise @Alex H√ºbner @Maxime Borry can hopefully advise</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-03 21:51:45">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-03 21:51:45"><div class="time">2024-03-03 21:51:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, this looks rather odd. Would you expect this from building your libraries? How did you assemble the contigs and align the data?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-03 23:22:17">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-03-03 23:22:17"><div class="time">2024-03-03 23:22:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I used metaSPAdes with --meta flag and default options (read length for this sample are on-average 51 bp, so I kept the default k-mer length used in metaSPAdes), and bowtie2 with --sensitive-local option.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 08:59:16">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 08:59:16"><div class="time">2024-03-04 08:59:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* This might be an issue of using <code>bowtie2</code> with <code>--sensitive-local</code>. The <code>-local</code> option allows for soft-clipping of reads and therefore, might clip off read ends that have damage because it is alignment-wise better than have three consecutive bases that have damage. You could try to use the global variant <code>--sensitive</code> and my guess you would have a proper damage signal.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 11:42:22">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-03-04 11:42:22"><div class="time">2024-03-04 11:42:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much! It worked. So, if I understood correctly, for ancient metagenomic data assembly it is better to use the --sensitive flag instead of the --sensitive-local flag? It is a doubt that I had since I started this analysis, because I read some bibliography and it seemed that the --sensitive-local was widely used. When I used metaQuast to compare the completeness of the genomes (--sensitive-local vs --sensitive), the pipeline in which I used the --sensitive-local flag seemed to perform better.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 11:46:06">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 11:46:06"><div class="time">2024-03-04 11:46:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* "--sensitive-local" is commonly used in most assembly pipelines for modern data. It usually doesn't matter for a lot of analyses, e.g. metaQUAST, but it has a particular effect when measuring the ancient DNA damage. @Maxime Borry had observed this before when he used nf-core/mag with default settings. It uses "--sensitive-local" and then the damage signal was gone.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 15:58:44">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-03-04 15:58:44"><div class="time">2024-03-04 15:58:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for your help! üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 17:30:29">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-03-04 17:30:29"><div class="time">2024-03-04 17:30:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* My 2 cents from Nepal: @Alex H√ºbner said everything, most likely soft clipping üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 20:59:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 20:59:54"><div class="time">2024-03-04 20:59:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I wonder if this could be a little blog post... A few people have tripped up on this...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:20:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:20:42"><div class="time">2024-03-04 21:20:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No wait, add it to the little book of smiley plots!</p>

<p><a href="https://www.spaam-community.org/little-book-of-smiley-plots/">https://www.spaam-community.org/little-book-of-smiley-plots/</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-04 21:23:18">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-04 21:23:18"><div class="time">2024-03-04 21:23:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Patr√≠cia Santos would you mind sharing the <code>analyze</code> output table for just that contig/bin you have in the plot? Then someone can rerender it for the book and we can use Alex's description :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-06 10:05:33">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-03-06 10:05:33"><div class="time">2024-03-06 10:05:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, sure! It will be really helpful for people that are doing <em>de-novo</em> genome assembly for the first time :)</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06N590K1UJ/pydamage_results_node104.txt?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">pydamage_results_node104.txt</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-06 10:22:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-06 10:22:09"><div class="time">2024-03-06 10:22:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Perfect thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-02 19:09:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-02 19:09:06"><div class="time">2024-04-02 19:09:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Alex H√ºbner , @Patr√≠cia Santos, and @Maxime Borry!</p>

<p>I finally got to adding this to the little book of smiley plots!</p>

<p>The PR is here if you can check if it looks OK: <a href="https://github.com/SPAAM-community/little-book-of-smiley-plots/pull/25">https://github.com/SPAAM-community/little-book-of-smiley-plots/pull/25</a></p>

<p>The page rendering are attached if you don't want to look at HTML and R code</p>

<p>If you're OK just say here or please leave a comment on Github (@Alex H√ºbner feel free to entirely re-write the description for phrasing and/or accuracy üò¨, @Maxime Borry is the pyDamage extremely-simplified representation OK?)</p>
                    <div class="message-attachment"
                            style="border-color: #36a64f">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/SPAAM-community/little-book-of-smiley-plots/pull/25">#25 Add softclipping example</a></div>
                                <div class="link-text">
                                    &lt;p&gt;Based on SPAAM discussion between &lt;a href=&#34;https://github.com/alexhbnr&#34;&gt;@alexhbnr&lt;/a&gt; and Patr√≠cia Samtools&lt;/p&gt;
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/SPAAM-community/little-book-of-smiley-plots&#34;&gt;SPAAM-community/little-book-of-smiley-plots&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06S8MH1ZT8/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06S8MH1ZT8/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06S8MH1ZT8-49fd8cbde2/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="336" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06SU0TNSKT/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06SU0TNSKT/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06SU0TNSKT-b04c9f823b/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="336" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-04 11:12:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-04 11:12:53"><div class="time">2024-04-04 11:12:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* OK I've got OKs from Alex and Patricia, @Patr√≠cia Santos I will merge now, and share the link to the pages on the website. If you want any changes/suggestions let me know üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-04 11:16:34">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-04 11:16:34"><div class="time">2024-04-04 11:16:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Softclipping page: <a href="https://www.spaam-community.org/little-book-of-smiley-plots/softclipping.html">https://www.spaam-community.org/little-book-of-smiley-plots/softclipping.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-04 11:16:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-04 11:16:42"><div class="time">2024-04-04 11:16:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Contributors page: <a href="https://www.spaam-community.org/little-book-of-smiley-plots/contributors.html">https://www.spaam-community.org/little-book-of-smiley-plots/contributors.html</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-04 11:24:37">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-04-04 11:24:37"><div class="time">2024-04-04 11:24:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok @James Fellows Yates, thank you üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-04 11:28:20">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-04 11:28:20"><div class="time">2024-04-04 11:28:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for sharing the plot and data üòÑ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-04 11:28:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-04 11:28:35"><div class="time">2024-04-04 11:28:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Now to find a artist üòÑ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-05 21:54:21">
            
                <img src="https://avatars.slack-edge.com/2023-08-02/5677577781653_54d94decedb70a28356c_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Michelle Cheng
                     <span class="print-only user-email">(yuanyu.cheng@mail.mcgill.ca)</span>
                </div>
                <a href="#2024-03-05 21:54:21"><div class="time">2024-03-05 21:54:21</div></a>
                <div class="msg">
                    <p>Hi all! Do you know if there's a way to quantitatively assess the performance of taxonomic assignment produced by different algorithms based on phylogenetic clustering? Thanks!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-05 22:54:54">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-05 22:54:54"><div class="time">2024-03-05 22:54:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Can you describe in more detail as to what you mean?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-05 22:54:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-05 22:54:55"><div class="time">2024-03-05 22:54:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you just mean compare accuracy of taxonomic assignment?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-05 22:54:55">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-05 22:54:55"><div class="time">2024-03-05 22:54:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Like comparing the topology?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-06 01:06:28">
            
                <img src="https://avatars.slack-edge.com/2023-08-02/5677577781653_54d94decedb70a28356c_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Michelle Cheng
                     <span class="print-only user-email">(yuanyu.cheng@mail.mcgill.ca)</span>
                </div>
                <a href="#2024-03-06 01:06:28"><div class="time">2024-03-06 01:06:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @James Fellows Yates - Sorry I was not clear &gt;.&lt; For example, here I colored the nodes based on phylum-level identification. Amplicons in the same cluster share genetic similarity and so should have the same color. I would like to demonstrate that classification in the 2nd image, by showing a more cohesive color scheme, is likely doing a better job in taxonomic assignment compared to the one in the 1st image. Here it is only a visual representation, and I wonder if there is a quantitative measurement to support my statement.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06N0DLSRL5/screenshot_2024-03-05_at_6.58.00___pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2024-03-05 at 6.58.00‚ÄØPM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06N0DLSRL5/screenshot_2024-03-05_at_6.58.00___pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06N0DLSRL5-b2f4f8e31d/screenshot_2024-03-05_at_6.58.00___pm_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="327" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06MNQ4U91D/screenshot_2024-03-05_at_6.58.41___pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2024-03-05 at 6.58.41‚ÄØPM.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06MNQ4U91D/screenshot_2024-03-05_at_6.58.41___pm.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06MNQ4U91D-ea813fd8d7/screenshot_2024-03-05_at_6.58.41___pm_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="272" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container thread_broadcast ">
        <div id="2024-03-06 21:08:56">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-06 21:08:56"><div class="time">2024-03-06 21:08:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I see... I'm not sure myself... Maybe someone else has an idea?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-08 16:09:16">
            
                <img src="https://secure.gravatar.com/avatar/d1a9503a47879969d6b213f9c0e9cf7f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Nora Bergfeldt
                     <span class="print-only user-email">(nora.bergfeldt@gmail.com)</span>
                </div>
                <a href="#2024-03-08 16:09:16"><div class="time">2024-03-08 16:09:16</div></a>
                <div class="msg">
                    <p>Hi! I'm going to upload data to ENA and was wondering what "checklist" spreadsheet people have used the most? My samples are human and I will upload fastq files in most cases, but for some of the samples only bam files with only the reads mapping to certain microorganisms/bacteria. Any experiences?
EDIT: The question is for "Register samples", I see that for uploading reads it's more straightforward but I assume I should start with registering samples?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-08 20:57:43">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-08 20:57:43"><div class="time">2024-03-08 20:57:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's a little tricky for aDNA because of all of our samples are by default 'metagenomes'. But ENA doesn't seem to have the specific metagenome checklists versus mags etc)</p>

<p>I would thus go with the MIxS host- and/or human- associated, if they are from skeletons. But worst comes to worst can use the ENA default checklist</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 13:49:35">
            
                <img src="https://secure.gravatar.com/avatar/d1a9503a47879969d6b213f9c0e9cf7f.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0014-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nora Bergfeldt
                     <span class="print-only user-email">(nora.bergfeldt@gmail.com)</span>
                </div>
                <a href="#2024-03-11 13:49:35"><div class="time">2024-03-11 13:49:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @James Fellows Yates!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 07:28:02">
            
                <img src="https://secure.gravatar.com/avatar/f07b63212c8a31d7178538b47a88d3c8.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0006-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Elisa Davis
                     <span class="print-only user-email">(emily.davis@utas.edu.au)</span>
                </div>
                <a href="#2024-03-11 07:28:02"><div class="time">2024-03-11 07:28:02</div></a>
                <div class="msg">
                    <p>Hi all! I have a bioconda//docker question that I couldn't quite figure out from the online forums. Basically, I am wondering if is this a problem with package versions I'm using, or that I need to get in touch with my cluster to reconfigure my usr/local/src memory allocation (??).</p>

<p>I am trying to pull a docker image (<a href="https://hub.docker.com/r/vanearranz/mares/tags">https://hub.docker.com/r/vanearranz/mares/tags</a>) via apptainer within a newly constructed environment on my local HPC cluster:
```### miniconda2 </p>

<h3>set up channel priorities</h3>

<p>conda config --add channels defaults
conda config --add channels bioconda
conda config --add channels conda-forge</p>

<h2>create environment with apptainer to pull docker image</h2>

<p>conda create -n MARES_test -c conda-forge apptainer</p>

<h1>enter environment</h1>

<p>conda activate MARES_test</p>

<h1>trying to pull MARES docker image</h1>

<p>apptainer pull <docker://vanearranz/mares:latest></p>

<h1>and I got this error (there is more, but I wanted to just put up the fatal error bit)</h1>

<p>FATAL:   While making image from oci registry: error fetching image to cache: while building SIF from layers: packer failed to pack: while unpacking tmpfs: error unpacking rootfs: unpack layer: unpack entry: usr/local/src/taxdump/names.dmp: unpack to regular file: short write: write /tmp/build-temp-1152360637/rootfs/usr/local/src/taxdump/names.dmp: no space left on device```
If anyone has any something similar or a good direction I could look I would really appreciate it. Cheers!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 08:19:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-11 08:19:23"><div class="time">2024-03-11 08:19:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You've run out of space on your /tmp/ , normally that means your hpc doesn't use the default Linux /tmp/ space</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 08:19:31">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-11 08:19:31"><div class="time">2024-03-11 08:19:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You can see that with the very last line</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 10:15:35">
            
                <img src="https://secure.gravatar.com/avatar/f07b63212c8a31d7178538b47a88d3c8.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0006-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Elisa Davis
                     <span class="print-only user-email">(emily.davis@utas.edu.au)</span>
                </div>
                <a href="#2024-03-11 10:15:35"><div class="time">2024-03-11 10:15:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ah yeah, thanks. I was a bit confused, so I wasn't sure if it was hpc issue or a package version issue. Thanks James! I hope you're having a great time at womad :headbangingparrot:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-11 10:33:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-11 10:33:15"><div class="time">2024-03-11 10:33:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Heh yeah, one of those cases you really need to read the entire error log and sort through all the cruft</p>

<p>Actually in the 'Jewel' at Singapore airport at the moment (left this morning)</p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§Ø Elisa Davis
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:07:17">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2024-03-13 19:07:17"><div class="time">2024-03-13 19:07:17</div></a>
                <div class="msg">
                    <p>Likely a very stupid "no-stupid-questions": We are asked to deposit our sequencing data to enable manuscript review and to provide "reviewer tokens". We would ideally fully release the data upon publication. However, ENA doesn't allow partial access, afaik. Do you know of other databases that could accomplish it or have any recommendations for how to navigate the request?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:11:01">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-13 19:11:01"><div class="time">2024-03-13 19:11:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It's NCBI related databases only that has that stuff...</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:12:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-13 19:12:19"><div class="time">2024-03-13 19:12:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I think you could argue that a) the ENA doesn't support this but it is a well established and commkn database for upload, b) if it is explicitly requested by reviewer your happy to share via a FTP server or something</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:12:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-13 19:12:57"><div class="time">2024-03-13 19:12:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But it doesn't make sense to upload it twice, and nor upload to the wrong side of the world (internet speed reasons, and also submitting to ENA is easier)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:16:56">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2024-03-13 19:16:56"><div class="time">2024-03-13 19:16:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, James! That's very helpful (and speedy üöÄ)!!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:18:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-13 19:18:15"><div class="time">2024-03-13 19:18:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Or say you will release the data publicly if it's minor revisions üò¨</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:18:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-13 19:18:19"><div class="time">2024-03-13 19:18:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So likely to be accepted</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:18:56">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2024-03-13 19:18:56"><div class="time">2024-03-13 19:18:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hahahaha, way to say "want the data? just publish us!"</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:19:30">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-13 19:19:30"><div class="time">2024-03-13 19:19:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* </p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06PS4MFD33/tenor_gif8348304872380716363.gif?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">tenor_gif8348304872380716363.gif</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06PS4MFD33/tenor_gif8348304872380716363.gif?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06PS4MFD33-e272108ab4/tenor_gif8348304872380716363_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="220"
                 height="148" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-14 08:04:21">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2024-03-14 08:04:21"><div class="time">2024-03-14 08:04:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Katerina Guschanski I got the same reviewer request and simply gave them the link to ENA saying they don't do that: <a href="https://ena-docs.readthedocs.io/en/latest/faq/release.html#can-i-make-my-study-partially-available">https://ena-docs.readthedocs.io/en/latest/faq/release.html#can-i-make-my-study-partially-available</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        ü§© James Fellows Yates, Benjamin Vernot
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-14 08:04:37">
            
                <img src="https://avatars.slack-edge.com/2022-02-28/3180714720545_69cc1bf5a8bc016a21df_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">irinavelsko
                     <span class="print-only user-email">(irinavelsko@gmail.com)</span>
                </div>
                <a href="#2024-03-14 08:04:37"><div class="time">2024-03-14 08:04:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Instead I provided the submission confirmation files, and that was enough for them</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-14 09:31:16">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2024-03-14 09:31:16"><div class="time">2024-03-14 09:31:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Brilliant! Thank you @irinavelsko!!! That's a huge help. And I love the collective experience of SPAAM üòç</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-14 09:31:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-03-14 09:31:51"><div class="time">2024-03-14 09:31:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* hivemind</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 19:53:00">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2024-03-13 19:53:00"><div class="time">2024-03-13 19:53:00</div></a>
                <div class="msg">
                    <p>Hiya, lab related question. Has <em>anyone</em> managed to find nice storage boxes that properly fit eppendorf tubes into them? We usually use white cardboard boxes (which are also a poor fit for the tubes), but as I am at the start of a project, I had this lovely idea that I will colour code my freezer shelf with pink boxes for pellets, green for extracts and orange for libraries. So I ordered the boxes in the photo (<a href="https://www.starlabgroup.com/p/PF-SL-186914/product%2Fstarstore-81-storagebox-pf-sl-186914.html/I2381-5041">https://www.starlabgroup.com/p/PF-SL-186914/product%2Fstarstore-81-storagebox-pf-sl-186914.html/I2381-5041</a>). But the eppendorfs tubes don't fit properly in these boxes either because of the caps! It's such a stupid thing but it's driving me mad!!! Pls help üôÉ</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06PDHBA683/screenshot_2024-03-13_at_18.49.44.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2024-03-13 at 18.49.44.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06PDHBA683/screenshot_2024-03-13_at_18.49.44.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06PDHBA683-7d5f32643e/screenshot_2024-03-13_at_18.49.44_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="220" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-13 21:17:29">
            
                <img src="https://secure.gravatar.com/avatar/b2ca1a2a5eabe81f5aa950b434e78600.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0010-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Helja Niinem√§e
                     <span class="print-only user-email">(sinihiir@gmail.com)</span>
                </div>
                <a href="#2024-03-13 21:17:29"><div class="time">2024-03-13 21:17:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi, we are also using these, leaving the last row and column empty.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-14 12:26:59">
            
                <img src="https://avatars.slack-edge.com/2020-08-17/1329280434768_bcb74c947c2bfe107d34_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ele
                     <span class="print-only user-email">(eg715@york.ac.uk)</span>
                </div>
                <a href="#2024-03-14 12:26:59"><div class="time">2024-03-14 12:26:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for the tip @Helja Niinem√§e I'll try some rearranging next time I'm in the lab üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-18 20:00:00">
            
                <img src="https://avatars.slack-edge.com/2023-03-30/5022504259159_700a1c6b7380ab0565ef_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Tina Warinner
                     <span class="print-only user-email">(twarinner@gmail.com)</span>
                </div>
                <a href="#2024-03-18 20:00:00"><div class="time">2024-03-18 20:00:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I‚Äôve also used those. They‚Äôre not too bad - you just have to angle the caps diagonally and leave some empties along the edges</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Helja Niinem√§e, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-19 16:48:55">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-19 16:48:55"><div class="time">2024-03-19 16:48:55</div></a>
                <div class="msg">
                    <p>Hi!
I am trying to run aMeta and I am getting this error at the Build<em>Malt</em>DB, that I am not sure how to fix (shortened the paths to make it more readable):
```Job 14: Build<em>Malt</em>DB: BUILDING MALT DATABASE USING SPECIES DETECTED BY KRAKENUNIQ
Reason: Missing output files: results/MALT_DB/maltDB.dat</p>

<p>Activating conda environment: .snakemake/conda/214b9a1bcfb604a1ea11b90cad2330f0_
Activating conda environment: .snakemake/conda/214b9a1bcfb604a1ea11b90cad2330f0_
.snakemake/scripts/tmp1yl44jjf.malt-build.py:17: SyntaxWarning: invalid escape sequence '\d'
  regex = re.compile("version (?P<major>\d+).(?P<minor>\d+)")
Traceback (most recent call last):
  File ".snakemake/scripts/tmp1yl44jjf.malt-build.py", line 37, in <module>
    shell(
  File "aMeta<em>env/lib/python3.10/site-packages/snakemake/shell.py", line 294, in <strong>new</strong>
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  grep -wFf results/KRAKENUNIQ</em>ABUNDANCE<em>MATRIX/unique</em>species<em>taxid</em>list.txt references/aMeta<em>data/MALT/seqid2taxid.map.orig &gt; results/MALT</em>DB/seqid2taxid.project.map; cut -f1 results/MALT<em>DB/seqid2taxid.project.map &gt; results/MALT</em>DB/seqids.project; grep -Ff results/MALT<em>DB/seqids.project references/aMeta</em>data/MALT/library.fna.gz | sed 's/>//g' &gt; results/MALT<em>DB/project.headers; seqtk subseq references/aMeta</em>data/MALT/library.fna.gz results/MALT<em>DB/project.headers &gt; results/MALT</em>DB/library.project.fna  2>&gt; logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log; unset DISPLAY; malt-build -i results/MALT<em>DB/library.project.fna -a2t references/aMeta</em>data/MALT/nucl<em>gb.accession2taxid -s DNA -t 20 -d results/MALT</em>DB/maltDB.dat  2>&gt; logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log' returned non-zero exit status 1.
[Tue Mar 19 16:24:53 2024]
Error in rule Build<em>Malt</em>DB:
    jobid: 14
    input: results/KRAKENUNIQ<em>ABUNDANCE</em>MATRIX/unique<em>species</em>taxid<em>list.txt
    output: results/MALT</em>DB/seqid2taxid.project.map, results/MALT<em>DB/seqids.project, results/MALT</em>DB/project.headers, results/MALT<em>DB/library.project.fna, results/MALT</em>DB/maltDB.dat
    log: logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log (check log file(s) for error details)
    conda-env: .snakemake/conda/214b9a1bcfb604a1ea11b90cad2330f0_</p>

<p>RuleException:
CalledProcessError in file workflow/rules/malt.smk, line 26:
Command 'source /sw/apps/conda/latest/rackham<em>stage/bin/activate '.snakemake/conda/214b9a1bcfb604a1ea11b90cad2330f0</em>'; set -euo pipefail;  python .snakemake/scripts/tmp1yl44jjf.malt-build.py' returned non-zero exit status 1.
  File "workflow/rules/malt.smk", line 26, in <em>_rule</em>Build<em>Malt</em>DB
  File "aMeta_env/lib/python3.10/concurrent/futures/thread.py", line 58, in run``<code>
It seems to start from one of the aMeta scripts (</code>.snakemake/scripts/tmp1yl44jjf.malt-build.py`). Has anyone got the same error? How would you deal with it?</p>

<p>Thank you!!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-19 19:57:20">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-19 19:57:20"><div class="time">2024-03-19 19:57:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Markella  Moraitou, thanks for using aMeta and I am ready to assist you with possible issues, do not hesitate post your questions here or direct message me. For now, could you post the log file from logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 10:58:04">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-20 10:58:04"><div class="time">2024-03-20 10:58:04</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Nikolay, thanks for the quick response! The log file doesn't exist and neither does the BUILD<em>MALT</em>DB directory</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:03:42">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-03-20 11:03:42"><div class="time">2024-03-20 11:03:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @Markella  Moraitou! This error is due to a problem with the new python versions and some syntax in the malt-build.py script.
@Nikolay Oskolkov There is already an issue on Github about this error. <a href="https://github.com/NBISweden/aMeta/issues/155">https://github.com/NBISweden/aMeta/issues/155</a>
I didn‚Äôt have time to work on a fix, but trying an older python version should work</p>
                    <div class="message-attachment"
                            style="border-color: #36a64f">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/NBISweden/aMeta/issues/155">#155 Python syntax of Malt-build script doesn&#39;t work with Python 3.12.1</a></div>
                                <div class="link-text">
                                    &lt;p&gt;I was helping debug with &lt;a href=&#34;https://github.com/npsonis&#34;&gt;@npsonis&lt;/a&gt; and it seems that the new python version 3.12.1 fails to read the syntax of one of the command in the malt-build.py script.&lt;/p&gt;

&lt;p&gt;This is the command:&lt;br /&gt;
regex = re.compile(&#34;version (?P\d+).(?P\d+)&#34;)&lt;br /&gt;
And this is the warning but the malt-build eventually fails:&lt;br /&gt;
:1: SyntaxWarning: invalid escape sequence &#39;\d&#39;&lt;/p&gt;

&lt;p&gt;We tested it on the command line when activating python 3.12.1 and got the same error but we didn&#39;t get the error when running it with python 3.11.4. So it might be some python update that makes this syntax not readable anymore?&lt;/p&gt;
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Comments</div>
                                        1
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/NBISweden/aMeta&#34;&gt;NBISweden/aMeta&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:05:02">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-20 11:05:02"><div class="time">2024-03-20 11:05:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much! Apologies, I didn't think of checking the issues in GitHub!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:06:11">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-03-20 11:06:11"><div class="time">2024-03-20 11:06:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No worries! We are available on this channel and we should definitely  fix that as soon as possible. Sorry about that! üòÖ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:08:41">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-20 11:08:41"><div class="time">2024-03-20 11:08:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Zo√© Pochon! So looks like we need to fix python3.10, right?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:12:03">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-20 11:12:03"><div class="time">2024-03-20 11:12:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Markella  Moraitou please try Zoe's suggestion and let us know it worked out</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:13:47">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-03-20 11:13:47"><div class="time">2024-03-20 11:13:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Well, I guess we would need to adapt the syntax of the script so that python 3.10 and later versions don‚Äôt complain</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-20 11:14:58">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-20 11:14:58"><div class="time">2024-03-20 11:14:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, git it, let me check the syntax</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôå Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 17:53:01">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-25 17:53:01"><div class="time">2024-03-25 17:53:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Not sure if I am doing something wrong, but I tried this with Python 3.11.4 and 3.9.5 and it fails again! I think the error message is different this time:
<code>Activating conda environment: .snakemake/conda/214b9a1bcfb604a1ea11b90cad2330f0_
Activating conda environment: .snakemake/conda/214b9a1bcfb604a1ea11b90cad2330f0_
Traceback (most recent call last):
  File ".snakemake/scripts/tmpocxh1tb0.malt-build.py", line 37, in &amp;lt;module&amp;gt;
    shell(
  File "aMeta_env/lib/python3.10/site-packages/snakemake/shell.py", line 294, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  grep -wFf results/KRAKENUNIQ_ABUNDANCE_MATRIX/unique_species_taxid_list.txt aMeta_data/MALT/seqid2taxid.map.orig &amp;gt; results/MALT_DB/seqid2taxid.project.map; cut -f1 results/MALT_DB/seqid2taxid.project.map &amp;gt; results/MALT_DB/seqids.project; grep -Ff results/MALT_DB/seqids.project aMeta_data/MALT/library.fna.gz | sed 's/&amp;gt;//g' &amp;gt; results/MALT_DB/project.headers; seqtk subseq aMeta_data/MALT/library.fna.gz results/MALT_DB/project.headers &amp;gt; results/MALT_DB/library.project.fna  2&amp;gt;&amp;gt; logs/BUILD_MALT_DB/BUILD_MALT_DB.log; unset DISPLAY; malt-build -i results/MALT_DB/library.project.fna -a2t aMeta_data/MALT/nucl_gb.accession2taxid -s DNA -t 3 -d results/MALT_DB/maltDB.dat  2&amp;gt;&amp;gt; logs/BUILD_MALT_DB/BUILD_MALT_DB.log' returned non-zero exit status 1.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 17:57:17">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-25 17:57:17"><div class="time">2024-03-25 17:57:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Markella  Moraitou please post the log-file. If there is none or its empty, could you delete the whole folder ‚Äùresults/MALT_DB‚Äù and restart aMeta?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:02:22">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-25 18:02:22"><div class="time">2024-03-25 18:02:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for responding so fast Nikolay! The log is indeed empty. The <code>results/MALT_DB</code> is not there either because the pipeline automatically deleted it after the error
<code>Removing output files of failed job Build_Malt_DB since they might be corrupted:
results/MALT_DB/seqid2taxid.project.map, results/MALT_DB/seqids.project, results/MALT_DB/project.headers
Shutting down, this might take some time.</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:08:12">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-25 18:08:12"><div class="time">2024-03-25 18:08:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have run and re-run the pipeline quite a few times because I ran into different errors before (or had the job time out). I wonder if there would be any benefit in deleting all results and starting from zero.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:09:16">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-25 18:09:16"><div class="time">2024-03-25 18:09:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No, please do not delete the results so far</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Markella  Moraitou
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:09:35">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-03-25 18:09:35"><div class="time">2024-03-25 18:09:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Do you have a system log file maybe for the MALT_DB rule? I generally get a log in the log folder but also a log for the job from the system</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:13:14">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-25 18:13:14"><div class="time">2024-03-25 18:13:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* There is this log file<code>.snakemake/log/2024-03-25T172216.737883.snakemake.log</code>
I have been running it interactively using SLURM so I do not have a log file from the SLURM job, but this seems to include what I have been seeing in the terminal.</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06RL6HTKLH/2024-03-25t172216.737883.snakemake.log?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">2024-03-25T172216.737883.snakemake.log</a></div>
                            
                                
    
        <!-- no preview available -->
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:13:42">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-25 18:13:42"><div class="time">2024-03-25 18:13:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Markella  Moraitou is your krakenuniq<em>abundance</em>matrix.txt non-empty?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 18:16:21">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-25 18:16:21"><div class="time">2024-03-25 18:16:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov No, it is not empty</p>

<pre><code>results/KRAKENUNIQ_ABUNDANCE_MATRIX/krakenuniq_abundance_matrix.txt 
x
Acidovorax sp. YS12     218
Acinetobacter baumannii 507
Acinetobacter bouvetii  284
Acinetobacter cumulans  253
Acinetobacter defluvii  771
Acinetobacter indicus   1283
Acinetobacter johnsonii 850
Acinetobacter piscicola 18846
Acinetobacter schindleri        46
</code></pre>

<p>By the way, I am running this on a single sample as a test! I assume this shouldn't matter, but just in case!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 20:48:59">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-03-25 20:48:59"><div class="time">2024-03-25 20:48:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think this shows that it is still using python 3.10:
<code>/crex/proj/sllstore2017021/nobackup/MARKELLA/mammal_om_evolution/software/aMeta_env/lib/python3.10/concurrent/futures/thread.py", line 58, in run</code></p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 21:08:15">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-03-25 21:08:15"><div class="time">2024-03-25 21:08:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Markella  Moraitou especially if you are still in the first trials with the pipeline, I would indeed begin from 0 again and remove the conda aMeta environment using the conda command <code>conda env remove -n aMeta</code> and remove the aMeta folder. But keep your samples.tsv and config.yaml files preciously as well as your data files.
Then clone a new aMeta directory and open the file aMeta/workflow/envs/environment.yaml and change the line about python with that line: <code>- python&amp;gt;=3.7,&amp;lt;3.10</code> . Then do the first step of the readme where you create the aMeta environment using mamba and replace your samples.tsv and config.yaml files where they should be. Then your aMeta environment will be using the python version on which aMeta should run bugfree ü§û. Then you can run again the lines from the README to recreate the rules environments and update the conda taxonomy. If you need help with this or something goes wrong, we can try to zoom next week. I'm sorry about that version problem. I also see that you are working from uppmax so I can provide you with a slurm config file to use the slurm queue with the pipeline.
Otherwise, if you can afford to wait a bit, I'll try to find a fix as soon as I find time (trying to wrap up a manuscript right now).</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-25 21:12:02">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-03-25 21:12:02"><div class="time">2024-03-25 21:12:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Zo√© Pochon, I agree, a correct python version should be specified from the very beginning, i.e. when installing aMeta, so @Markella  Moraitou I would try what Zoe suggests but would still copy the results-folder to a safe place just in case</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-03-26 18:58:26">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-03-26 18:58:26"><div class="time">2024-03-26 18:58:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks! I will try that! üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 21:03:25">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-15 21:03:25"><div class="time">2024-05-15 21:03:25</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Markella  Moraitou I am working on the fix for the problem with building Malt database which you reported. So far I can't reproduce the error but we suspect it has something to do with the changed python syntax in the new python versions which makes malt-build.py fail. Do you by any chance have the log-file logs/BUILD<em>MALT</em>DB/BUILD<em>MALT</em>DB.log (apologies if I already asked for it, can't remember)? Or perhaps you have already managed to solve this issue by using an older python version?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-17 10:39:41">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-17 10:39:41"><div class="time">2024-05-17 10:39:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Markella  Moraitou we have a fix for your issue. This will be implemented centrally in aMeta soon, but in the meantime if you simply replace this line</p>

<p><code>regex = re.compile("version (?P&amp;lt;major&amp;gt;\d+)\.(?P&amp;lt;minor&amp;gt;\d+)")</code>
with this line</p>

<p><code>regex =  re.compile(r'version (?P&amp;lt;major&amp;gt;\d+)\.(?P&amp;lt;minor&amp;gt;\d+)')</code>
in the <code>aMeta/workflow/scripts/malt-build.py</code>, this should fix the issue.  Could you please try it and let me know whether it worked? Apologies for the issue, it came with some syntax changes in the recent Python versions and we had to adjust aMeta codes accordingly</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-17 17:57:52">
            
                <img src="https://secure.gravatar.com/avatar/da05885fc71fdc3eb4da4e45b05bdf57.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0020-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Markella  Moraitou
                     <span class="print-only user-email">(markella.mor@gmail.com)</span>
                </div>
                <a href="#2024-05-17 17:57:52"><div class="time">2024-05-17 17:57:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi Nikolay! Thank you so much, I will test this as soon as I can! :D
Apologies for the silence after your last suggestion, I tried it but then I started running into other (possibly unrelated) issues so I left it aside for a bit!
Unfortunately it seems like I deleted the previous output including the log files, but I will send the new one, if I get to the database building step without problems!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-17 18:00:36">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-17 18:00:36"><div class="time">2024-05-17 18:00:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks @Markella  Moraitou! We will certainly assist you with other issues as well, let us know here or DM me or Zoe</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 04:55:16">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-01 04:55:16"><div class="time">2024-04-01 04:55:16</div></a>
                <div class="msg">
                    <p>Hello everyone! Our lab has obtained several dental calculus samples, but we have no experience handling calculus. Since there are methods available that can recover both DNA and proteins, I would like to ask if you would extract both or only DNA?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 08:06:51">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-01 08:06:51"><div class="time">2024-04-01 08:06:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That's hard to say, depends on your question/what you want to do!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 08:10:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-01 08:10:06"><div class="time">2024-04-01 08:10:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I think the general takeaway is by doing both you will in one way or another get a lower DNA yield, which might not be great if you have bad preservation and your focus is on DNA</p>

<p>@Zandra Fagern√§s @irinavelsko can correct me if I'm wrong as authors of one such protocol...</p>

<p>But calculs is generally well preserved so might not be an issue if your question you want to ask wants protein</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 09:13:36">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-01 09:13:36"><div class="time">2024-04-01 09:13:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks for your advice, James! If proteins are generally well preserved, then we need to pay more attention to the importance of protein extraction, even though we don't currently have the capacity for protein analysis. We will keep this in mind as we move forward with our research.üôè</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-01 11:36:52">
            
                <img src="https://avatars.slack-edge.com/2020-02-05/937654706020_e9e8576298b89a1ec13c_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zandra Fagern√§s
                     <span class="print-only user-email">(fagernaes@shh.mpg.de)</span>
                </div>
                <a href="#2024-04-01 11:36:52"><div class="time">2024-04-01 11:36:52</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* James is indeed correct! With well-preserved calculus, you will not see much loss in yield through a dual extraction protocol, but for poorly preserved samples you might. It's still better than splitting a sample in two and doing separate extractions though. But, as James said, the first question is what your research aim is, as the two biomolecules have slightly different strengths! üòä</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è James Fellows Yates, Wenqin Yu
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-02 12:48:51">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-02 12:48:51"><div class="time">2024-04-02 12:48:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, Zandra! This is really helpful!  Having both of your perspectives clarifies the trade-offs between protein and DNA yield depending on preservation quality and research focus.  We'll definitely consider this when finalizing our protocol.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-02 11:45:11">
            
                <img src="https://avatars.slack-edge.com/2023-11-15/6201650067012_3de502d1d10e39f7598c_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Alicia Muriel
                     <span class="print-only user-email">(aliciamurielg@gmail.com)</span>
                </div>
                <a href="#2024-04-02 11:45:11"><div class="time">2024-04-02 11:45:11</div></a>
                <div class="msg">
                    <p>Hi! I was wondering if anyone has any recommendations for parasite specific databases. I've been using aMeta for my sample analysis but I'd like to look a bit more into parasite genomes and I don't really know where I should start</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-02 13:50:10">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-02 13:50:10"><div class="time">2024-04-02 13:50:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Alicia Muriel, together with aMeta we provide a database called Microbial NT which includes 1.4 mln parasitic reference sequences:</p>

<blockquote>
  <p>Microbial NCBI NT database included 11,840,243 reference sequences (2,465,945 viral, 17,519 archaeal, 1,737,968 bacterial, 4,530,716 fungal, 1,689,877 protozoa, and 1,398,218 parasitic worms sequences)</p>
</blockquote>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Alicia Muriel
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-03 15:57:36">
            
                <img src="https://avatars.slack-edge.com/2023-11-15/6201650067012_3de502d1d10e39f7598c_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alicia Muriel
                     <span class="print-only user-email">(aliciamurielg@gmail.com)</span>
                </div>
                <a href="#2024-04-03 15:57:36"><div class="time">2024-04-03 15:57:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you! I was wondering if I should use a more specific database but I‚Äôll stick to that one then</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-03 17:36:06">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2024-04-03 17:36:06"><div class="time">2024-04-03 17:36:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It may worth exploring whether you can use NCBI Datasets command line tools. You may be able to download parasites genomes but I‚Äôm not sure as I only used it to download arcahaea and bacterial genomes. </p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Alicia Muriel
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 10:01:56">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-17 10:01:56"><div class="time">2024-04-17 10:01:56</div></a>
                <div class="msg">
                    <p>Hello everyone,
I'm an M.Sc. student currently in an internship, and I'm working on ancient metagenomics data from dental calculus. I wanted to try aMeta for its resources' efficiency, but I am not familiar with SnakeMake and I struggle with the basic first steps. Could I ask you some questions ?</p>

<p>I'm trying to run the test provided with the workflow before running it on my data. I'm working on an HPC cluster that operates with SLURM. So I've filled the <code>config/envmodules.yaml</code> file like so :
<code>envmodules:
  fastqc:
    - bioinfo/FastQC/0.12.1
  cutadapt:
    - bioinfo/Cutadapt/4.3
  multiqc:
    - bioinfo/MultiQC/1.19
  bowtie2:
    - bioinfo/bowtie/2.5.1
[...]</code>
I didn't touch the <code>config/config.yaml</code>, and I used a profile for SLURM submissions wich is called <em>smk-simple-slurm</em> and added the <code>--use-envmodules</code> option . It looks like this :
<code>cluster:
  mkdir -p logs/{rule} &amp;amp;&amp;amp;
  sbatch
    --partition={resources.partition}
    --qos={resources.qos}
    --cpus-per-task={threads}
    --mem={resources.mem_mb}
    --job-name=smk-{rule}-{wildcards}
    --output=logs/{rule}/{rule}-{wildcards}-%j.out
default-resources:
  - partition=workq
  - qos=inraeregion
  - mem_mb=1000
restart-times: 3
max-jobs-per-second: 10
max-status-checks-per-second: 1
local-cores: 1
latency-wait: 60
jobs: 500
keep-going: True
rerun-incomplete: True
printshellcmds: True
scheduler: greedy
use-envmodules: True</code>
So I'm firstly unsure if I have configured the workflow correctly.</p>

<p>I still tried to run the workflow from the <code>.test/</code> directory with <em>snakemake-v7.20.0</em> like so : <code>snakemake --snakefile ../workflow/Snakefile -j 1 --profile simple-slurm/</code></p>

<p>I think it loads the module properly, but I have this error with the first FastQC step : <code>Memory value 10240 MB was outside the allowed range (100 - 10000) at /usr/local/bioinfo/src/FastQC/FastQC-v0.12.1/fastqc line 203.</code></p>

<p>Should I adjust the memory allocated to FastQC, or is there a way to modify the allowed range?</p>

<p>Thank you in advance for taking the time to assist me. I am relatively new to workflow in general, especially SnakeMake ones, so I hope my questions aren't too trivial !</p>

<p>Best regards</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 12:52:01">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-17 12:52:01"><div class="time">2024-04-17 12:52:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Mathias Caire, me and @Zo√© Pochon will help you with aMeta. A quick fix for the FastQC "outside the allowed range" error is to change the default <code>mem_mb</code>  in the <code>aMeta/workflow/rules/qc.smk</code> from its default 10240  to e.g. 1024 in the rules <code>FastQC_BeforeTrimming</code> and <code>FastQC_AfterTrimming</code>. Apologies for this silly bug, we will fix it centrally in aMeta asap.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 12:53:36">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-17 12:53:36"><div class="time">2024-04-17 12:53:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Regarding the configuration, I need to take a closer look, and I am at a meeting right now, so I will get back to you a bit later. In the meantime perhaps @Zo√© Pochon could comment on your configuration?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 14:20:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-17 14:20:41"><div class="time">2024-04-17 14:20:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @Mathias Caire! First, good job on putting that together.
If it is running and it fails at the fastqc step it means that at least your files seems to work up to that step and there is no major error in them.
Indeed, the fastqc rule has been changed recently in aMeta and since then it complains about the default range.
You can change that in your <em>smk-simple-slurm</em> file like this:</p>

<p>```# Amount of threads per rule
set-threads:</p><ul><li>FastQC_AfterTrimming=2</li><li>FastQC_BeforeTrimming=2</li>
</ul>

<h1>Amount of resources per rule</h1>

<p>set-resources:</p><ul><li>FastQC<em>AfterTrimming:mem</em>mb=6000</li><li>FastQC<em>BeforeTrimming:mem</em>mb=6000```
Note that those are not tabs but spaces everywhere</li>
</ul>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-18 09:48:48">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-18 09:48:48"><div class="time">2024-04-18 09:48:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you very much, the fastQC step now works ! I hope I will get the all thing to run soon with my data</p>
                    
                    
                    
                        <div class="message-reaction">
                        üéâ Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:02:51">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:02:51"><div class="time">2024-04-22 16:02:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello ! Me again, I encounter an error with the filterign of krakenuniq outputs. Here is the message :
```Error in rule Filter<em>KrakenUniq</em>Output:
    jobid: 15
    input: results/KRAKENUNIQ/bar/krakenuniq.output, resources/pathogenomesFound.tab
    output: results/KRAKENUNIQ/bar/krakenuniq.output.filtered, results/KRAKENUNIQ/bar/krakenuniq.output.pathogens, results/KRAKENUNIQ/bar/taxID.pathogens, results/KRAKENUNIQ/bar/taxID.species
    log: logs/FILTER<em>KRAKENUNIQ</em>OUTPUT/bar.log (check log file(s) for error details)
    conda-env: /work/project/GenIn/ROUSSILLE/aMeta/.test/.snakemake/conda/6e63d658e3ebb26afeed2f78cbd5f25d_
    shell:
        /work/project/GenIn/ROUSSILLE/aMeta/workflow/scripts/filter<em>krakenuniq.py results/KRAKENUNIQ/bar/krakenuniq.output 1000 200 resources/pathogenomesFound.tab &amp;&gt; logs/FILTER</em>KRAKENUNIQ<em>OUTPUT/bar.log; cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.pathogens | tail -n +2 &gt; results/KRAKENUNIQ/bar/taxID.pathogens;cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.filtered | tail -n +2 &gt; results/KRAKENUNIQ/bar/taxID.species
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)
    cluster</em>jobid: Submitted batch job 7020974</p>

<p>Error executing rule Filter<em>KrakenUniq</em>Output on cluster (jobid: 15, external: Submitted batch job 7020974, jobscript: /work/project/GenIn/ROUSSILLE/aMeta/.test/.snakemake/tmp.ff2hgy7s/snakejob.Filter<em>KrakenUniq</em>Output.15.sh). For error details see the cluster log and the log files of the involved rule(s).``<code>
The weird thing is that there is no log file in the</code>FILTER<em>KRAKENUNIQ</em>OUTPUT` folder so I can't identify where the error comes from... I was thinking that you may have an idea about this issue. Thank you in advance for taking the time to assist me.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:10:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:10:22"><div class="time">2024-04-22 16:10:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Mathias!
Let's rule out a time-limit or memory issue first.
Can you write on your terminal
<code>sacct -j 7020974</code>
and
<code>seff 7020974</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:12:48">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:12:48"><div class="time">2024-04-22 16:12:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ```JobID           JobName  Partition    Account  AllocCPUS      State ExitCode</p>

<hr />

<p>7020974      smk-Filte+      workq     mcaire          1     FAILED      1:0
7020974.bat+      batch                mcaire          1     FAILED      1:0
7020974.ext+     extern                mcaire          1  COMPLETED      0:0```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:13:22">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:13:22"><div class="time">2024-04-22 16:13:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Okay, then something went wrong. What do you have in your <code>aMeta/results/KrakenUniq/**/**</code>folder?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:16:17">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:16:17"><div class="time">2024-04-22 16:16:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I just threw the output in the chat sorry, I sent it by accident haha
In my KrakenUniq output I think I have the right files, here they are :
```==&gt; bar/krakenuniq.output &lt;==</p>

<h1>KrakenUniq v1.0.4 DATE:2024<em>04</em>22T13:44:43Z DB:resources/KrakenUniq<em>DB DB</em>SIZE:99800462 WD:/work/project/GenIn/ROUSSILLE/aMeta/.test</h1>

<h1>CL:/usr/local/bioinfo/src/KrakenUniq/krakenuniq-v1.0.4/krakenuniq --preload --db resources/KrakenUniq<em>DB --fastq-input results/CUTADAPT</em>ADAPTER_TRIMMING/bar.trimmed.fastq.gz --threads 10 --output results/KRAKENUNIQ/bar/sequences.krakenuniq --report-file results/KRAKENUNIQ/bar/krakenuniq.output --gzip-compressed --only-classified-out</h1>

<p>%       reads   taxReads        kmers   dup     cov     taxID   rank    taxName
55.64   710     710     33190   1.04    NA      0       no rank unclassified
44.36   566     0       17686   1       0.001772        1       no rank root
44.36   566     0       17686   1       0.001772        131567  no rank
40.13   512     0       15898   1       0.00177 2       superkingdom        Bacteria
40.13   512     0       15898   1       0.00177 1224    phylum
40.13   512     0       15898   1       0.00177 1236    class
40.13   512     0       15898   1       0.00177 91347   order</p>

<p>==&gt; bar/sequences.krakenuniq &lt;==
C       endo<em>542:+:3651:3702:51e1</em>0-2/1 632     51      632:16 0:15
C       endo<em>830:-:5112:5155:43e1</em>0-2/1 632     43      0:14 632:9
C       endo<em>113:+:3286:3332:46e1</em>0-2/1 632     46      632:1 0:21 632:4
C       endo<em>410:+:4001:4061:60e1</em>0-2/1 632     60      0:15 632:25
C       endo<em>492:+:697:738:41e1</em>0-2/1   632     41      0:8 632:13
C       endo<em>130:-:805:849:44e1</em>0-2/1   632     44      632:5 0:19
C       endo<em>156:+:2707:2745:38e1</em>0-2/1 632     38      632:18
C       endo<em>916:-:1531:1587:56e1</em>0-2/1 9605    56      9605:17 0:19
C       endo<em>863:+:7062:7132:70e1</em>0-2/1 632     70      632:15 0:21 632:14
C       endo<em>445:+:8065:8147:82e1</em>0-2/1 632     75      632:16 0:21 632:18</p>

<p>==&gt; foo/krakenuniq.output &lt;==</p>

<h1>KrakenUniq v1.0.4 DATE:2024<em>04</em>22T13:43:57Z DB:resources/KrakenUniq<em>DB DB</em>SIZE:99800462 WD:/work/project/GenIn/ROUSSILLE/aMeta/.test</h1>

<h1>CL:/usr/local/bioinfo/src/KrakenUniq/krakenuniq-v1.0.4/krakenuniq --preload --db resources/KrakenUniq<em>DB --fastq-input results/CUTADAPT</em>ADAPTER_TRIMMING/foo.trimmed.fastq.gz --threads 10 --output results/KRAKENUNIQ/foo/sequences.krakenuniq --report-file results/KRAKENUNIQ/foo/krakenuniq.output --gzip-compressed --only-classified-out</h1>

<p>%       reads   taxReads        kmers   dup     cov     taxID   rank    taxName
44.57   608     608     31393   1.05    NA      0       no rank unclassified
55.43   756     0       23399   1       0.002345        1       no rank root
55.43   756     0       23399   1       0.002345        131567  no rank
49.63   677     0       20950   1       0.002332        2       superkingdom        Bacteria
49.63   677     0       20950   1       0.002332        1224    phylum
49.63   677     0       20950   1       0.002332        1236    class
49.63   677     0       20950   1       0.002332        91347   order</p>

<p>==&gt; foo/sequences.krakenuniq &lt;==
C       endo<em>735:-:9954:9993:39e1</em>0-2/1 632     39      632:5 0:14
C       endo<em>752:+:2727:2828:101e1</em>0-2/1        632     75      632:55
C       endo<em>516:+:8375:8423:48e1</em>0-2/1 632     48      632:28
C       endo<em>46:-:4794:4997:203e1</em>0-2/1 632     75      632:45 0:10
C       endo<em>54:+:7210:7299:89e1</em>0-2/1  632     75      632:14 0:21 632:20
C       endo<em>795:+:1290:1415:125e1</em>0-2/1        632     75      632:55
C       endo<em>417:+:867:920:53e1</em>0-2/1   632     53      0:16 632:17
C       endo<em>72:-:7405:7476:71e1</em>0-2/1  632     71      632:51
C       endo<em>88:-:3241:3280:39e1</em>0-2/1  632     39      632:19
C       endo<em>986:-:5062:5107:45e1</em>0-2/1 9605    45      9605:23 0:2 ```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:17:53">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:17:53"><div class="time">2024-04-22 16:17:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* And you said that there is no log file there?
logs/FILTER<em>KRAKENUNIQ</em>OUTPUT/bar.log</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:18:42">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:18:42"><div class="time">2024-04-22 16:18:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes FILTER<em>KRAKENUNIQ</em>OUTPUT is an empty directory, that's strange no ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:21:02">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:21:02"><div class="time">2024-04-22 16:21:02</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm. There should be at least a log from slurm. I see that in your config file, you tell slurm to save the logs this way:
--output=logs/{rule}/{rule}-{wildcards}-%j.out
Can you try to look if that one is there?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:22:14">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:22:14"><div class="time">2024-04-22 16:22:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Should be logs/Filter<em>KrakenUniq</em>Output/Filter<em>KrakenUniq</em>Output**</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:23:30">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:23:30"><div class="time">2024-04-22 16:23:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have some files but they simply contain the standard output of the workflow :
```[Mon Apr 22 15:51:50 2024]
Job 0: Filter<em>KrakenUniq</em>Output: APPLYING DEPTH AND BREADTH OF COVERAGE FILTERS TO KRAKENUNIQ OUTPUT FOR SAMPLE results/KRAKENUNIQ/bar/krakenuniq.output resources/pathogenomesFound.tab
Reason: Missing output files: results/KRAKENUNIQ/bar/krakenuniq.output.pathogens, results/KRAKENUNIQ/bar/taxID.pathogens, results/KRAKENUNIQ/bar/krakenuniq.output.filtered, benchmarks/FILTER<em>KRAKENUNIQ</em>OUTPUT/bar.benchmark.txt, results/KRAKENUNIQ/bar/taxID.species</p>

<p>/work/project/GenIn/ROUSSILLE/aMeta/workflow/scripts/filter<em>krakenuniq.py results/KRAKENUNIQ/bar/krakenuniq.output 1000 200 resources/pathogenomesFound.tab &amp;&gt; logs/FILTER</em>KRAKENUNIQ<em>OUTPUT/bar.log; cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.pathogens | tail -n +2 &gt; results/KRAKENUNIQ/bar/taxID.pathogens;cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.filtered | tail -n +2 &gt; results/KRAKENUNIQ/bar/taxID.species
Activating environment modules: bioinfo/KrakenUniq/1.0.4
/tools/tools/modules/4.6.1/init/bash: line 88: deactivate: command not found
[Mon Apr 22 15:51:51 2024]
Error in rule Filter</em>KrakenUniq<em>Output:
    jobid: 0
    input: results/KRAKENUNIQ/bar/krakenuniq.output, resources/pathogenomesFound.tab
    output: results/KRAKENUNIQ/bar/krakenuniq.output.filtered, results/KRAKENUNIQ/bar/krakenuniq.output.pathogens, results/KRAKENUNIQ/bar/taxID.pathogens, results/KRAKENUNIQ/bar/taxID.species
    log: logs/FILTER</em>KRAKENUNIQ<em>OUTPUT/bar.log (check log file(s) for error details)
    conda-env: /work/project/GenIn/ROUSSILLE/aMeta/.test/.snakemake/conda/6e63d658e3ebb26afeed2f78cbd5f25d</em>
    shell:
        /work/project/GenIn/ROUSSILLE/aMeta/workflow/scripts/filter<em>krakenuniq.py results/KRAKENUNIQ/bar/krakenuniq.output 1000 200 resources/pathogenomesFound.tab &amp;&gt; logs/FILTER</em>KRAKENUNIQ_OUTPUT/bar.log; cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.pathogens | tail -n +2 &gt; results/KRAKENUNIQ/bar/taxID.pathogens;cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.filtered | tail -n +2 &gt; results/KRAKENUNIQ/bar/taxID.species
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)</p>

<p>Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:26:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:26:26"><div class="time">2024-04-22 16:26:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, it is tricky to find the error without the log üßê</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:27:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:27:57"><div class="time">2024-04-22 16:27:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is there maybe a problem with one of your modules?
<code>/tools/tools/modules/4.6.1/init/bash: line 88: deactivate: command not found</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:29:06">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:29:06"><div class="time">2024-04-22 16:29:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't normally get that warning/error</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:29:41">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:29:41"><div class="time">2024-04-22 16:29:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't know, since I use the <code>--use-env-modules</code> option I thought it was normal, what do you think ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:30:42">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:30:42"><div class="time">2024-04-22 16:30:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I just found another strange thing, when I simply try to run :
<code>/work/project/GenIn/ROUSSILLE/aMeta/workflow/scripts/filter_krakenuniq.py results/KRAKENUNIQ/bar/krakenuniq.output 1000 200 resources/pathogenomesFound.tab &amp;amp;&amp;gt; logs/FILTER_KRAKENUNIQ_OUTPUT/bar.log; cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.pathogens | tail -n +2 &amp;gt; results/KRAKENUNIQ/bar/taxID.pathogens;cut -f7 results/KRAKENUNIQ/bar/krakenuniq.output.filtered | tail -n +2 &amp;gt; results/KRAKENUNIQ/bar/taxID.species</code>
There is no error and I have the output files like this :
```==&gt; results/KRAKENUNIQ/bar/krakenuniq.output &lt;==</p>

<h1>KrakenUniq v1.0.4 DATE:2024<em>04</em>22T13:44:43Z DB:resources/KrakenUniq<em>DB DB</em>SIZE:99800462 WD:/work/project/GenIn/ROUSSILLE/aMeta/.test</h1>

<h1>CL:/usr/local/bioinfo/src/KrakenUniq/krakenuniq-v1.0.4/krakenuniq --preload --db resources/KrakenUniq<em>DB --fastq-input results/CUTADAPT</em>ADAPTER_TRIMMING/bar.trimmed.fastq.gz --threads 10 --output results/KRAKENUNIQ/bar/sequences.krakenuniq --report-file results/KRAKENUNIQ/bar/krakenuniq.output --gzip-compressed --only-classified-out</h1>

<p>%       reads   taxReads        kmers   dup     cov     taxID   rank    taxName
55.64   710     710     33190   1.04    NA      0       no rank unclassified
44.36   566     0       17686   1       0.001772        1       no rank root
44.36   566     0       17686   1       0.001772        131567  no rank
40.13   512     0       15898   1       0.00177 2       superkingdom        Bacteria
40.13   512     0       15898   1       0.00177 1224    phylum
40.13   512     0       15898   1       0.00177 1236    class
40.13   512     0       15898   1       0.00177 91347   order</p>

<p>==&gt; results/KRAKENUNIQ/bar/krakenuniq.output.filtered &lt;==
%       reads   taxReads        kmers   dup     cov     taxID   rank    taxName
40.13   512     512     15898   1.0     0.00177 632     species                   Yersinia pestis</p>

<p>==&gt; results/KRAKENUNIQ/bar/krakenuniq.output.pathogens &lt;==
%       reads   taxReads        kmers   dup     cov     taxID   rank    taxName
40.13   512     512     15898   1.0     0.00177 632     species                   Yersinia pestis</p>

<p>==&gt; results/KRAKENUNIQ/bar/sequences.krakenuniq &lt;==
C       endo<em>542:+:3651:3702:51e1</em>0-2/1 632     51      632:16 0:15
C       endo<em>830:-:5112:5155:43e1</em>0-2/1 632     43      0:14 632:9
C       endo<em>113:+:3286:3332:46e1</em>0-2/1 632     46      632:1 0:21 632:4
C       endo<em>410:+:4001:4061:60e1</em>0-2/1 632     60      0:15 632:25
C       endo<em>492:+:697:738:41e1</em>0-2/1   632     41      0:8 632:13
C       endo<em>130:-:805:849:44e1</em>0-2/1   632     44      632:5 0:19
C       endo<em>156:+:2707:2745:38e1</em>0-2/1 632     38      632:18
C       endo<em>916:-:1531:1587:56e1</em>0-2/1 9605    56      9605:17 0:19
C       endo<em>863:+:7062:7132:70e1</em>0-2/1 632     70      632:15 0:21 632:14
C       endo<em>445:+:8065:8147:82e1</em>0-2/1 632     75      632:16 0:21 632:18</p>

<p>==&gt; results/KRAKENUNIQ/bar/taxID.pathogens &lt;==
632</p>

<p>==&gt; results/KRAKENUNIQ/bar/taxID.species &lt;==
632```</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:33:41">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:33:41"><div class="time">2024-04-22 16:33:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Cool! I'm honestly not sure what is causing that problem. Sometimes snakemake understands as a "fail" when there is a warning message produced. Did you get any warning when running it manually?
And do you get that warning:
<code>/tools/tools/modules/4.6.1/init/bash: line 88: deactivate: command not found</code>
In other slurm log files of completed jobs?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:36:12">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:36:12"><div class="time">2024-04-22 16:36:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I didn't get any warnings but I'm glad that it's working, now if I rerun the workflow the following steps should start right ?
I do get the <code>deactivate: command not found</code> everywhere, even the completed jobs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:37:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:37:10"><div class="time">2024-04-22 16:37:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes I think it will start from there. You can do a dry run by adding -n to your snakemake command before running it for real and you will see which jobs it is planning on running</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:37:56">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-04-22 16:37:56"><div class="time">2024-04-22 16:37:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for the help !</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:38:23">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-04-22 16:38:23"><div class="time">2024-04-22 16:38:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* No worries! Fingers crossed üòäü§û</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 15:14:36">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-15 15:14:36"><div class="time">2024-05-15 15:14:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello !
I come back to ask questions about aMeta because I'm I'm struggling with the resources that I need. The workflow works really well with my data when I use the <em>microbial part of NCBI RefSeq</em> database that you provided on scilifelab. However, I would like to use a more comprehensive one.
I downloaded the Refeq archaea, bacteria, viral, plasmid, human1, UniVec_Core, protozoa &amp; fungi from the Kraken 2 indexes repository here <a href="https://benlangmead.github.io/aws-indexes/k2">https://benlangmead.github.io/aws-indexes/k2</a>. But the index is in this format :
<code>3.9M    database100mers.kmer_distrib
3.6M    database150mers.kmer_distrib
3.3M    database200mers.kmer_distrib
3.0M    database250mers.kmer_distrib
2.8M    database300mers.kmer_distrib
4.5M    database50mers.kmer_distrib
4.2M    database75mers.kmer_distrib
77G     hash.k2d
3.2M    inspect.txt
59G     k2_pluspf_20240112.tar.gz
2.4M    ktaxonomy.tsv
44M     library_report.tsv
512     opts.k2d
3.7M    taxo.k2d
8.0K    unmapped_accessions.txt</code>
Which doesn't work when I just use the folder containing it in aMeta.
Do I need to take additional steps for KrakenUniq to utilize it?</p>

<p>Additionally, I utilized the option to screen for pathological taxa with Bowtie2 but I cannot find where the results are after running the workflow.</p>

<p>Thank you in advance for your help !</p>

<p>Best regards Mathias CAIRE</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 15:40:07">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-15 15:40:07"><div class="time">2024-05-15 15:40:07</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Mathias Caire, aMeta is very KrakenUniq-centric and does not work with Kraken2 database. We did not want to use Kraken2 for two reasons. First, higher RAM demands of Kraken2 compared to KrakenUniq. Second, the breadth of coverage filter (number of unique kmers) was initially developed in KrakenUniq and imported later to Kraken2, so at the moment of releasing first versions of aMeta, Kraken2 did not have that filter. In summary you will need a Kraken1 or KrakenUniq database to make aMeta work. I would recommend you to download the Microbial NT database which is actually more comprehensive than the standard Kraken2 database, this was shown in the aMeta paper</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 15:45:38">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-15 15:45:38"><div class="time">2024-05-15 15:45:38</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for the response, I will follow your advice and run aMeta on the Microbial NT Database.
Could you also assist me with pathogen screening using Bowtie2? What output is expected for this step of the workflow?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 16:01:39">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-15 16:01:39"><div class="time">2024-05-15 16:01:39</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Oh, sorry @Mathias Caire, forgot to answer. In the results/KRAKENUNIQ folder for each sample you have a ‚Äú.pathogens‚Äù files which represent an overlap of krakenuniq.filtered with a custom very permissive list of known pathogens. So you can check which of your detected microbes can be potential pathogens. In earlier versions of aMeta, only reads corresponding to those pathogenic microbes were aligned to their references with Bowtie2, and their ancient status was checked with mapDamage. Now this is done for all detected microbes, including the pathogenic microbes. The Bowtie2 branch of aMeta served as a quick and dirty way to get alignments+deamination. Ideally one should do it via Malt which has an LCA, i.e. a more clever and metagenomics-specific way of alignment. However, since Malt can be a headache, while waiting for Malt results, one might want to get quick preliminary results from the Bowtie2 analysis</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Mathias Caire
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 16:04:51">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-15 16:04:51"><div class="time">2024-05-15 16:04:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you very much! I do indeed have the file you mentioned. I must say that I really like the workflow, and I'm impressed with how the resources are handled. Have a nice day!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        :meow_party: Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-15 16:05:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-15 16:05:21"><div class="time">2024-05-15 16:05:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Great to hear @Mathias Caire, thank you!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 12:50:08">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-23 12:50:08"><div class="time">2024-05-23 12:50:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello, I have some more questions if you have the time to answer them. I need to run some analysis on the results, but for that, I require the OTU table containing the raw abundance of each taxa in my samples. I believe I should use the MALT abundance matrix, but I'm unclear about the differences between the SAM and RMA6 versions.</p>

<p>Before transitioning to aMeta, I had already conducted MALT analysis on my samples and I was processing the RMA6 files with MEGAN. I could compare them using MEGAN options and export an OTU table along with a tree in Newick format (both extremely helpful for my subsequent analysis). I've attempted to replicate this process with the outputs in the MALT directory, but I'm struggling to obtain the same matrix as the malt<em>abundance</em>matrix_rma6.</p>

<p>What are your thoughts on this?</p>

<p>Best regards,
Mathias CAIRE</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 13:45:58">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-23 13:45:58"><div class="time">2024-05-23 13:45:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Mathias Caire, you most likely need to use the RMA6 abundance matrix which should be equivalent to the MEGAN abundance matrix which you used previously. The SAM abundance matrix was provided mainly for an "upper limit estimate" of the microbial abundance (i.e. likely an overestimate) as a possible way to quantify SAM-alignments provided by MALT.</p>

<p>Since MALT outputs two types of alignments: RMA6 (bioinformatically unfriendly) and SAM (bioinformatically friendly), we wanted to look inside both of them. The difference between the RMA6 and SAM abundance matrices is that the former is computed from RMA6 alignments (with LCA, more appropriate for metagenomic samples), and the latter is computed from SAM-alignments (without LCA, aka regular Bowtie2 alignments). The lack of LCA will most likely result in accounting for multi-mappers and therefore an overestimate of the true abundance, while the RMA6 (with LCA) abundance may be too conservative, i.e. ignoring too many valuable reads (especially for low-coverage samples) which it treats as multi-mappers. Still I would perhaps recommend to stick to the RMA6 abundance and use the SAM as a way to double-check the microbes which you found interesting from the RMA6 table</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 13:54:28">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-23 13:54:28"><div class="time">2024-05-23 13:54:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you for answering so quickly, that helps a lot. I also need a tree in Newick format but I think I can manage to extract it from the ncbi.tre file and the taxon list.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 13:57:03">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-23 13:57:03"><div class="time">2024-05-23 13:57:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm, to get a Newick tree you should probably input the RMA6-alignments from a Malt job to MEGAN, i.e. outside of aMeta</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 14:00:30">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-23 14:00:30"><div class="time">2024-05-23 14:00:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I cannot mannage to get the same species as in the RMA6 abundance matrix. For now I simply loaded all the RMA6 in MEGAN and used the optioin "compare" of MEGAN and selected all of them. I uncollapsed the tree and I selected the species to extract the tree. Should I do something differently ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 14:47:50">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-23 14:47:50"><div class="time">2024-05-23 14:47:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hmm @Mathias Caire, for quantifying microbial abundance on species level from rma6-files we use rma-tabuliser tool from @James Fellows Yates. The species count table from rma-tabuliser should be more or less equivalent to the rma6-files which you can visualize in MEGAN, i.e. the species names at least should be the same, right James?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 15:39:09">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-23 15:39:09"><div class="time">2024-05-23 15:39:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I may have caused some confusion. I initially thought that the species represented in the heatmap were the same as those in the RMA6 matrix. When I mentioned that I couldn't find the same species, I meant that some species present in the heatmap are missing from the table I generated using MEGAN, and vice versa.</p>

<p>I think I got a little bit confused because the RMA6 matrix looks like this :
<code>node SP1 SP2 SP3
1    ... ... ...
2    ... ... ...
29   ... ... ...
31   ... ... ...</code>
Are those nodes the taxIDs? If so, how can I obtain their taxonomical names? Currently, I'm running a script that searches through a taxDB file to find the taxa names, but I am finding various taxonomical ranks (genus, order, etc.) rather than just species as i would expect in the RMA6 matrix.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 18:00:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-23 18:00:18"><div class="time">2024-05-23 18:00:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Mathias Caire the species in the heatmap are usually a subset of those in RMA6. This is because not all detected species (i.e. the ones in the RMA6 file) can be successfully authenticated. So the heatmap represents the most robust species which passed multiple validation and authentication steps and got the authentication score (from 0 to 10), which is displayed in the heatmap.</p>

<p>Yes, the first column in the malt<em>abundance</em>matrix<em>rma6.txt are taxIDs. You can get their corresponding scientific names scientific names if you navigate to aMeta/results/KRAKENUNIQ</em>ABUNDANCE_MATRIX and type</p>

<p><code>paste unique_species_taxid_list.txt unique_species_names_list.txt</code></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-23 18:01:18">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-23 18:01:18"><div class="time">2024-05-23 18:01:18</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Btw thank you for reminding, it has been a long-standing plan to make it more clear for the users. I will fix it now asap in the next release</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-24 09:39:32">
            
                <img src="https://secure.gravatar.com/avatar/56dea1b7d94169b028e4b5ccff804d77.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Mathias Caire
                     <span class="print-only user-email">(mathias.caire@univ-tlse3.fr)</span>
                </div>
                <a href="#2024-05-24 09:39:32"><div class="time">2024-05-24 09:39:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov thank you very much for your help, I'm glad I can contribute to make aMeta more user-friendly. Have a great day!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        :meow_party: Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 12:07:33">
            
                <img src="https://avatars.slack-edge.com/2023-01-15/4639294999574_85a756ca97035831649b_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Freya Steinhagen
                     <span class="print-only user-email">(stu227847@mail.uni-kiel.de)</span>
                </div>
                <a href="#2024-04-17 12:07:33"><div class="time">2024-04-17 12:07:33</div></a>
                <div class="msg">
                    <p>Hi y'all,</p>

<p>a while ago somebody held a super interesting talk about detection of false positivehits in  (in metagenomic screenings), I believe to remember that it was on the example of parasites and/or pathogens. It was also reassiesing some older published material. I would be very glad if someone remembers more closely and can give me the the link to the paper.</p>

<p>If someone has generally more literature on identification methods for false positive hits to share, I would be interested as this topic seems to pop up more and more recently.</p>

<p>All the best :)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 12:38:32">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-17 12:38:32"><div class="time">2024-04-17 12:38:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Freya Steinhagen, I have have my private collection of false-positive hits in metagenomic analyses, and if you want we could talk about it in more details (e.g. via zoom). I do not think there are many publications on this topic, but I could immediately recommend this very good review <a href="https://www.annualreviews.org/content/journals/10.1146/annurev-genom-091416-035526">https://www.annualreviews.org/content/journals/10.1146/annurev-genom-091416-035526</a> and especially the fantastic supplementary figure 1 from that paper <a href="https://www.annualreviews.org/docserver/fulltext/genom/18/1/gg18_warinner_supfigures.pdf?expires=1713350923&id=id&accname=ar-269936&checksum=6CEFB66D33511B2F24A760D418E20F29|https://www.annualreviews.org/docserver/fulltext/genom/18/1/gg18_warinner_supfigures.pd[‚Ä¶]ccname=ar-269936&checksum=6CEFB66D33511B2F24A760D418E20F29">https://www.annualreviews.org/docserver/fulltext/genom/18/1/gg18<em>warinner</em>supfigures.pdf?expires=1713350923&amp;id=id&amp;accname=ar-269936&amp;checksum=6CEFB66D33511B2F24A760D418E20F29|https://www.annualreviews.org/docserver/fulltext/genom/18/1/gg18<em>warinner</em>supfigures.pd[‚Ä¶]ccname=ar-269936&amp;checksum=6CEFB66D33511B2F24A760D418E20F29</a>, which shows how easy it is to "discover" a pathogen in a soil or ocean if you are doing a naive analysis.  Also, a large chunk of our aMeta paper <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9</a> (see Figure 9) was about false-positive discoveries and how aMeta attempts to combat them</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">annualreviews.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.annualreviews.org/content/journals/10.1146/annurev-genom-091416-035526">A Robust Framework for Microbial Archaeology | Annual Reviews</a></div>
                                <div class="link-text">
                                    Microbial archaeology is flourishing in the era of high-throughput sequencing, revealing the agents behind devastating historical plagues, identifying the cryptic movements of pathogens in prehistory, and reconstructing the ancestral microbiota of humans. Here, we introduce the fundamental concepts and theoretical framework of the discipline, then discuss applied methodologies for pathogen identification and microbiome characterization from archaeological samples. We give special attention to the process of identifying, validating, and authenticating ancient microbes using high-throughput DNA sequencing data. Finally, we outline standards and precautions to guide future research in the field.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://www.annualreviews.org/content/journals/10.1146/annurev-genom-091416-035526</div>
                                
                                
                            
                        </div>
                    <div class="message-attachment"
                            >
                            <div class="service-name">BioMed Central</div>
                            
                            
                                
                                <div class="link-title"><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9">aMeta: an accurate and memory-efficient ancient metagenomic profiling workflow - Genome Biology</a></div>
                                <div class="link-text">
                                    Analysis of microbial data from archaeological samples is a growing field with great potential for understanding ancient environments, lifestyles, and diseases. However, high error rates have been a challenge in ancient metagenomics, and the availability of computational frameworks that meet the demands of the field is limited. Here, we propose aMeta, an accurate metagenomic profiling workflow for ancient DNA designed to minimize the amount of false discoveries and computer memory requirements. Using simulated data, we benchmark aMeta against a current state-of-the-art workflow and demonstrate its superiority in microbial detection and authentication, as well as substantially lower usage of computer memory.
                                </div>
                                
                                
    
        <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9">
            <img class="preview" src="https://static-content.springer.com/image/art%3A10.1186%2Fs13059-023-03083-9/MediaObjects/13059_2023_3083_Fig1_HTML.png"
                width="1430"
                 height="751" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03083-9</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Freya Steinhagen
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-17 12:41:13">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-17 12:41:13"><div class="time">2024-04-17 12:41:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also, you might want to check this <a href="https://dgg32.medium.com/carp-in-the-soil-1168818d2191">https://dgg32.medium.com/carp-in-the-soil-1168818d2191</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Medium</div>
                            
                            
                                
                                <div class="link-title"><a href="https://dgg32.medium.com/carp-in-the-soil-1168818d2191">Carp in the Soil</a></div>
                                <div class="link-text">
                                    Ridiculous sequencing results revealed how errors propagated from one research study to a global database
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Reading time</div>
                                        7 min read
                                    </div>
                                
                                
    
        <a href="https://dgg32.medium.com/carp-in-the-soil-1168818d2191">
            <img class="preview" src="https://miro.medium.com/v2/resize:fit:1200/1*WasdCvAsPa_4aFiJLn0pKw.jpeg"
                width="1200"
                 height="900" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://dgg32.medium.com/carp-in-the-soil-1168818d2191</div>
                                
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ü´† Freya Steinhagen
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 14:33:49">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-22 14:33:49"><div class="time">2024-04-22 14:33:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Besides Nicholay 's, is the speech "Parasit detection in ancient and modern metagenomic datasets" from Jonas Niemann?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Liam Lanigan
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:12:42">
            
                <img src="https://avatars.slack-edge.com/2023-01-15/4639294999574_85a756ca97035831649b_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Freya Steinhagen
                     <span class="print-only user-email">(stu227847@mail.uni-kiel.de)</span>
                </div>
                <a href="#2024-04-22 16:12:42"><div class="time">2024-04-22 16:12:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, thanks! that might be the one I was thinking of</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 16:32:46">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-22 16:32:46"><div class="time">2024-04-22 16:32:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Wenqin Yu and @Freya Steinhagen do you have the reference for the "Parasite detection..."?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-24 10:55:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-24 10:55:21"><div class="time">2024-04-24 10:55:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Also a good read from @Aleksandra Laura Pach <a href="https://www.biorxiv.org/content/10.1101/2024.02.27.581519v1">https://www.biorxiv.org/content/10.1101/2024.02.27.581519v1</a></p>
                    
                    
                    
                        <div class="message-reaction">
                        üôá Aleksandra Laura Pach
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-24 13:29:20">
            
                <img src="https://avatars.slack-edge.com/2021-05-15/2061871421878_23d6b222ebb4b281dd17_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Liam Lanigan
                     <span class="print-only user-email">(liamlanigan@palaeome.org)</span>
                </div>
                <a href="#2024-04-24 13:29:20"><div class="time">2024-04-24 13:29:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Jonas Niemann hopefully something soon?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üôè Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 13:56:10">
            
                <img src="https://avatars.slack-edge.com/2023-02-15/4806714050115_ea8d91b2c95c6dad1aa2_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Wenqin Yu
                     <span class="print-only user-email">(1132895879@qq.com)</span>
                </div>
                <a href="#2024-04-25 13:56:10"><div class="time">2024-04-25 13:56:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov Sorry I can't find it as well, it seems an unpublished work.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 18:43:13">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-25 18:43:13"><div class="time">2024-04-25 18:43:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, I also realized that it was (apparently) a SPAAMtisch talk that I probably missed. @Maria Lopopolo are all the SPAAMtisch talks recorded and available? Could you post a link? It was my long-term plan to watch some talks that I missed</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 19:39:31">
            
                <img src="https://avatars.slack-edge.com/2023-10-22/6073844927141_0012868a261ce055e14a_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maria Lopopolo
                     <span class="print-only user-email">(maria.lopopolo1989@gmail.com)</span>
                </div>
                <a href="#2024-04-25 19:39:31"><div class="time">2024-04-25 19:39:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hello, yes all the talks from speakers who gave me consent are recorded. Most of them are on the SPAAM you tube channel. Unfortunately the parasite one was one of the few I could not record for the reason that I did not receive consent from the speaker. But I can ask the speaker if we can open a Google doc with questions you may have for them. Do you think it can help?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
                        <div class="message-reaction">
                        ‚ù§Ô∏è Wenqin Yu, Freya Steinhagen
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-26 07:16:28">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-04-26 07:16:28"><div class="time">2024-04-26 07:16:28</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Maria for the great job you and others are doing within the SPAAMtisch initiative! I could see that very few SPAAMtisch talks were present at the SPAAM youtube channel and thought there was another channel. But if the speaker did not want the talk to be recorded, it is fine, no need to bother üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:38:08">
            
                <img src="https://avatars.slack-edge.com/2023-06-12/5406037305413_f59ff1ab794dee282c37_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Chenyu Jin (Amend)
                     <span class="print-only user-email">(amend.jin@gmail.com)</span>
                </div>
                <a href="#2024-04-22 10:38:08"><div class="time">2024-04-22 10:38:08</div></a>
                <div class="msg">
                    <p>Hi all, I have gotten 0.1 and 0.05 as 2 damage frequencies at the read end for different species from the same ancient metagenomic sample. The screenshots are 2 species for example. There has been a discussion for whether the one with lower damage frequency (0.05) should be considered as authenticated or with a clear damage pattern. Do you think the lower one is a result of the probability of DNA damage and  stochasticity of reads, or it shall not be considered authenticated?</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06VAA82F7X/screenshot_2024-04-22_at_10.31.40.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot 2024-04-22 at 10.31.40.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06VAA82F7X/screenshot_2024-04-22_at_10.31.40.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06VAA82F7X-a5c5c603ff/screenshot_2024-04-22_at_10.31.40_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="360"
                 height="223" />
        </a>
    
                            
                        </div>
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F070N1G4W9W/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">image.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F070N1G4W9W/image.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F070N1G4W9W-a699b3657f/image_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="333"
                 height="191" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:43:44">
            
                <img src="https://avatars.slack-edge.com/2024-01-23/6500116916647_a8a5d85fa916acb13fc4_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Pete Heintzman
                     <span class="print-only user-email">(peteheintzman@gmail.com)</span>
                </div>
                <a href="#2024-04-22 10:43:44"><div class="time">2024-04-22 10:43:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The extent of a recovered aDNA damage signal is impacted by divergence from the reference sequence. In this case, the species on the left is more diverged (non-damage substitution rates are higher), which might be limiting the damage signal.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Chenyu Jin (Amend), Dawn lewis
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:47:49">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-22 10:47:49"><div class="time">2024-04-22 10:47:49</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Depends on the level of degradation too (e.g. age,other preservation factors). To me the lower damage one would still be valid, the curve is relatively smooth rather than spiky and inconsistent </p>

<p>You could try running it against @Maxime Borry's PyDamage if you want a statistical check</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Pete Heintzman, Chenyu Jin (Amend)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:47:57">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-22 10:47:57"><div class="time">2024-04-22 10:47:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Against a model</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:49:32">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-22 10:49:32"><div class="time">2024-04-22 10:49:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you look at @Katerina Guschanski and @Jaelle Brealey's 200y old bear calculus microbial genomes, iirc they look quite similar to your lower ok nes</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Chenyu Jin (Amend)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:52:03">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-22 10:52:03"><div class="time">2024-04-22 10:52:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you were to fix the Y axis it would also help evaluate the difference from reference: you would expect to see the base line of all other substitutions to be quite high if you have the wrong or diverted reference (right @Pete Heintzman?)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 10:56:14">
            
                <img src="https://avatars.slack-edge.com/2023-06-12/5406037305413_f59ff1ab794dee282c37_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Chenyu Jin (Amend)
                     <span class="print-only user-email">(amend.jin@gmail.com)</span>
                </div>
                <a href="#2024-04-22 10:56:14"><div class="time">2024-04-22 10:56:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you Pete and James.</p>

<p>I ran Pydamage with default setting and it passed all the species (even the spiky ones). I am looking for the original script right now.</p>

<p>I will read the paper of Katja and Jaelle ‚ù§Ô∏è</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 13:28:44">
            
                <img src="https://avatars.slack-edge.com/2020-09-21/1370018779238_c6652e9e36f750f71bba_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Katerina Guschanski
                     <span class="print-only user-email">(katerina.guschanski@ebc.uu.se)</span>
                </div>
                <a href="#2024-04-22 13:28:44"><div class="time">2024-04-22 13:28:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The level of damage would be consistent with younger (museum-age, a few hundred years) samples. The baseline, which as @Pete Heintzman suggests would be indicative to distance to the reference genomes, actually looks quite similar to me for both examples, so I'm not sure if it would explain the difference in damaged proportion. I would tend to agree that the left-hand example looks genuine</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates, Pete Heintzman, Chenyu Jin (Amend)
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-22 23:01:36">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-04-22 23:01:36"><div class="time">2024-04-22 23:01:36</div></a>
                <div class="msg">
                    <p>Where can I find the fasta sequences from the AMDirT? I want them (just the viral ones) for my DB and I can not find them in the NCBI neither on EMBI-EBI, I have the ids as DA66 for ERS2484289
DA337 ERS2484292... any advice?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-23 06:14:37">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-23 06:14:37"><div class="time">2024-04-23 06:14:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* AMDirT only supports FASTQ files at the moment, as per the warning, as otherwise the way to download the data are too heterogeneous so its not easy to implement.</p>

<p>I guess though the FASTA sequences can be found via the ENA so the warning isn't very precise and we should improve that.</p>

<p>Also add that caveat to the documentation now I think about it (ping @Maxime Borry)</p>
                    
                    <div class="message-upload">
                            <div class="link-title"><a href="https://files.slack.com/files-pri/TPYE6400N-F06V4HEGZ5M/screenshot_20240423-061155.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">Screenshot_20240423-061155.png</a></div>
                            
                                
    
        <a href="https://files.slack.com/files-pri/TPYE6400N-F06V4HEGZ5M/screenshot_20240423-061155.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e">
            <img class="preview" src="https://files.slack.com/files-tmb/TPYE6400N-F06V4HEGZ5M-ee59828ffe/screenshot_20240423-061155_360.png?t=xoxe-814482136022-7387777877203-7381259671766-35a2e6d45fbe7af4c322830e8bb2364e"
                width="162"
                 height="360" />
        </a>
    
                            
                        </div>
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-23 06:15:09">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-23 06:15:09"><div class="time">2024-04-23 06:15:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* You'll have to inspect the tables manually to get to get the FASTAs... Sorry @Ania TSL :(</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-23 09:26:43">
            
                <img src="https://avatars.slack-edge.com/2023-09-28/5958614551893_afabbec739414b4f4f4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Alex H√ºbner
                     <span class="print-only user-email">(alexander_huebner@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-23 09:26:43"><div class="time">2024-04-23 09:26:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Ania TSL if you check the sample table <code>ancientsinglegenome-hostassociated</code> and filter for the sample name <code>DA66</code>, you see at the far end of the table that the <code>archive</code> is <code>GitHub</code>. The authors never uploaded their data to either NCBI or ENA but just kept them on GitHub.
I guess the sequences you search for are here: <a href="https://github.com/acorg/parvo-2018/blob/master/consensuses/consensuses.fasta">https://github.com/acorg/parvo-2018/blob/master/consensuses/consensuses.fasta</a></p>
                    <div class="message-attachment"
                            style="border-color: #24292f">
                            
                            
                            
                                
                                <div class="link-title"><a href="">&lt;https://github.com/acorg/parvo-2018/blob/master/consensuses/consensuses.fasta | consensuses.fasta&gt;</a></div>
                                <div class="link-text">
                                    &lt;p&gt;```&lt;/p&gt;

&lt;blockquote&gt;
  &lt;p&gt;NEO&lt;em&gt;DA251-user&lt;/em&gt;6862&lt;em&gt;RUS&lt;/em&gt;EasternAsia&lt;em&gt;Human
  ATGGAGTTATTTAGAGGGGTGCTTCAAGTTTYTTCCACCATTCTTGACTG?GCTAACGATAACTGGTGGTGTTCTATGCTGGATTTAGACACTTCTGACTGGGAACCATTAACTCATTCTAACAGACTAATGGCAATATACTTAAGCAGTGTGGCCTCTAAACTTGACTTTACAGGGGGGCC??TAGC?GGGTGCTTGTACTTTTTTCAAGTAGAATGTAACAAATTTGAGGAAGGCTATCATATTCATGTGGTTATTGGGGG???AGGGTTAAACCCCAGAAACCTAACAGTGTGTGTAGAGGGATTATTTAATAATGTACTTTATCACCTTGTAACTGAAAATGTAAAACTTAAATTTTTGCCAGGAATGACTACAAAAGGCAAATATTTTAGAGATGGAGAACAATTTATAGAAAACTATTTAATGAAAAAAATACCTTTAAATGTTGTGTGGTGTGTAACTAATATTGATGGGTACATAGATACCTGTATTTCTGCTTCTKTTAGACGGGGAGCTTGTCATGCCAAAAAACCCCGCATTAGTACAAGCATTGATGGTGCT??TA?TGA??C?GGGGAGTCTAG????????????????????????????????GGAAGGGGACTAAAG?T?GCATAAAGTTTCAAACTATGGTAAATTGGTTGTGTGAAAATAG?GT?TTTACAGAGGATAAGTGGAAACTAGTTGACTTTAACCAGTACACTTTACTTAGCAGTAGTCATAGTGGAAGTTTTCAAATTCAAAGTGCATTAAAACTAGCTATTTATAAGGCTACTAATTTAGTGCCTACTAGTACATTTTTATTACACACAGACTTTGAGC?????????????????????????????????????????????????????????????????TTGGTGGGGCAGCATGTTTTAAAGTGGATTGATAAAAAATGTGGCAAAAAAAATACACTGTGGTTTTATGGCCCGCCAAGTACAGGAAAAACAAATTTGGCAATGGCCATTGCTAAAACTGTTCCAATGTATGGTATGGTTAATTGGAATAATGAAAATTTTCCATTTAATGATGTAGCAGGAAAAAGCTTGGTGGTCTGGGATGAAGGTATTATTAAGTCTACCATTGTAGAAGCTGCAAAAGCCATTTTAGGCGGGCAACCAACCAGGGTGGATCAAAAAATGCGTGGAAGTGTAGCAGTGCCTGGAGTACCAGTGGTAATAACCAGCAATGGTGACATTACTTTTGTTGTTAGTGGGAACACTACAACAACTGTACATGCTAAAGCCTTAAAAGAGCAAATGGTAAAGTTAAACTTTAC?G???????????????????????????????????????????????????????????????????????CAAAGCTGGGACCACTATAAAAACTGGGCAATAAACTACACTTTTGATTTCCCTGGAATAAATGCAGATGCCCTTCACCCAGACCTCCAAACCGCCCCAATTGTCACAGACACCAGTGTCAGCAGCAGTRGTGGTGAAAGC????????????????AAAGCAGCTTTTTTAACCTCATCACCCCAGGCGCCTGGAACACTGAAACCCCGCGCTCTAGTACGCCAGTCCCC?GGACCA?TTCAGGAGAATCATTTGTCGGAAGCTCAGTTTCCTCCGAAGCTGTAGCTGCATCGTGGGAAGAAGCCTTCTACACACCTTTGGCAGATCAGTTTCGTGAACTGTTAGTTGGGGTTGACTATGTATGGGACGGTGTAAGGGGTTTACCTGTTTGTTGTGTGCAACATATTAATAACAGTGGGGGAGGCTTGGGACTTTGTCCCCATTGCATTAATGTGGGAGCTTGGTATAATGGATGGAAGTTTCGAGAATTTACCCCAGATTTG??????????????????????????????AATCCCTTTTCTGT??TAACCTGCAAAAAATGTGCTTACCTGTCGGGATTGCAAAGTTTTGTAGATTATGAGTAAAGAAAGTGGCAAATGGTGGGAAAGTAATGATAAATTTGCTAAAGACGTRTAT?AGCAATTTGTAGAATTTTATGAAAAAGTTACTGAAACAGACTTAGAGCTTATTCAAATATTAAAAGATCATTACAATATTTCTTTAGATAATCCCTTAGAAAACCCATCTTCTTTGTTTGACTTAGTTGCTCGTATTAAAA????????????????????????TATATAGTCATCATTTTCAAAGTCATGGACAGTTATCTGACCACCCCCATGCCTTATCATCCAGTAACAGTCATGCAGAACCTAGAGGAGAAAATGCAGTATTATCTAGTGAAGACTTACACAAGCCTGGGCAAGTTA?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????AGCTGCCCCTGTGGCCCATTTTCAAGGAAGTTTGCCGGAAGTTCCCGCTTACAACGCCTCAGAAAAATACCCAAGCATGACTTCAGTTAATTCTGCAGAAGCCAGCACTGGTGCAGGA????????????????????CAAAAGCATGTGGAGTGAGGGGGCCACTTTTACTGCCAACTCTGTAACTTGTACATTTTCCAGACAGTTTTTAATTCCATATGACCCAGAGCACCATTATAA????????????????????????????????????????????????????????????CACCATTAGTCCCATAATGGGATACTCAACGCCATGGAGATACTTAGATTTTAATGCTTTAAATTTATTTTTTTCACCTTTAGAGTTTCAGCATTTAATTGAAAATTATGGAAGTATAGCTCCTGATG?TTTAACTGTGACCATATCAGAAATTGCTGTTAAAGATGTTACAGACAAAACTGGAGGAGGGGTACAGGTTACTGACAGCACTACAGGGCGTTTATGCATATTAGTAGACCATGAATACAAGTACCC??????????????????TCAGGATACCTTARCCCCAGAACTTCCTATTTGGGTATA?TT?CC?CCTCAATATGCTTATTTAACAGTAGGAGATGTAAACACACAAGGAATTTCAGGAGACAGTAAAAAATTAGCTAGTGAAGAATCAGCATTTTATGTCCTGGAACACAGTTCTTTT?????????????????????????????TATGTCTTATAAGTTTCCTGCAGTACCCCCAGAAAATTTAGAAGGCTGCAGTCAACACTTTTATGAAATGTACAACCCGTTATACGGATCCCGTTTAGGAGTCCCTGACACATTAGGAGGARACCCCAAATTTAGATCTTTAACACATGAAGAC????????????????????????????????????????????????????????????????????????????????????????GAAAAGCCCTAACAGGCCTTAGTACAGGAACCAGTCAAAGCACCAGAATATCATTACGCCCTGGGCCAGTGTCTCAGCCATATCATCACTGGGACACAGATAAATATGTGACAGG?ATAAATGCCATTTCTCATGGTCAAACCACTTATGGAAACGCTGAAGACAAAGAGTATCAGCAAGGAGTAGGTAGATTCCCAAATGAAAAAGAACAACTAAAACAGTTACA?GGTTTAAA?????ACACCTATTTTCCCAATAAAGGTACCCAGCAATATACAGATCAAATTGAGCGCCCCCTAATGGTAGGTT?TGTGTGGAATAGAAGAGCCCTTCACTATGAAAGCCAGCTGTGGAGTAAAATCCCAAATTTAGATGACAGTTTTAAAACTCAATTTGCAGCCCTAGGAGGATGGG????????CAGCCACCCCCTCAAATATTTTTAAAAATATTACCACAAAGTGGGCCAATTGGGGGTATTAAGTCAATGGGAATTACAACCTTAGTTCAATACGCTGTGGGTATTATGACAGTAACCATGACATTTAAATT?GGGCCCCGCAAAGCTACGGGACGGTGGAACCCTCAACCTGGAGTGTACCCCCCTCACGCAGCAGGCCATTTACCATATGTACTATATGACCCTACAGCTACAGATGCAAAGCAACACCACA?ACATG?ATATGAAAA?CCTGAAGAATTGTGGAC?GCCAAAAGCCGTGT?CACCC???????
  NEO&lt;/em&gt;DA337&lt;em&gt;3967&lt;/em&gt;RUS&lt;em&gt;EasternAsia&lt;/em&gt;Human
  ATGGAGTTATTTAGAGGGGTGCTTCAAGTTTCTTCCAGTATTCTTGACTGTGCTAACGATAACTGGTGGTGCTCTATGCTGGATTTAGACACTTCTGACTGGGAACCATTAACTCATTCTAACAGACTAATGGCAATATACTTAAGCAGTGTGGCCTCTAAACTTGACTTTACGGGGGGGCCCTTAGCTGGGTGCTTGTACTTTTTTCAAGTGGAATGTAACAAATTTGAAGAAGGCTATCATATTCATGTGGTTATTGGGGGGCCAGGGCTAAACCCCAGAAACCTAACAGTGTGTGTAGAGGGGTTATTTAATAATGTACTTTATCACCTTGTAACTGAAAATGTAAAACTTAAATTTTTGCCAGGAATGACTACAAAAGGCAAATATTTTAGAGATGGAGAACAATTTATAGAAAACTATTTAATGAAAAAAATACCTTTAAATGTTGTGTGGTGTGTAACTAATATTGATGGGTACATAGATACCTGTATTTCTGCTTCTTTTAGACGGGGAGCTTGTCATGCCAAAAAACCCCGCATTAGTAC?AGCAT?????????TTGTA?TGAACCGGGGGAGTCTAGCGCTACAGGGGGAGATGTTGTGCCATTTGCTGGGAAGGGGACTAAAGCTGGCATAAAGTTTCAAACTATGGTAAATTGGTTGTGTGAAAATAGGGTGTTTACAGAGGATAAGTGGAAACTAGTTGACTTTAACCAGTACACTTTACTAAGTAGCAGTCATAGTGGAAGTTTTCAAATTCAAAGTGCATTAAAACTAGCAATTTATAAGGCAACTAATTTAGTGCCTACTAGTACATTTTTATTACATACAGACTTTGAGCAAGTTACCTGTATTAAAGAAAATAAAATAGTTAAACTGTTACTATGTCAAAATTATGACCCCTTGTTGGTGGGGCAGCATGTTTTAAAGTGGATTGATAAAAAATGTGGCAAAAAAAATACACTGTGGTTTTATGGCCCGCCAAGTACAGGCAAAACAAATTTGGCAATGGCCATTGCTAAAGCTGTTCCAGTGTATGGTATGGTTAATTGGAATAATGAAAATTTCCCATTTAATGATGTAGCAGGAAAAAGCTTGGTGGTCTGGGATGAAGGTATTATTAAGTCTACAATTGTAGAAGCTGCAAAAGCCATTTTAGGCGGGCAACCAACCAGGGTAGATCAAAAAATGCGTGGAAGTGTAGCAGTGCCTGGAGTACCAGTGGTAATAACCAGCAATGGTGACATTACTTTTGTTGTTAGTGGGAACACTAYAACAACTGTACATGCTAAAGCCTTAAAAGAGCGAATRGTAAAGTTAAACTTTACCGTAAGATGCAGCCCTGACATGG????????????????????????????????CTCACATGGTGTAATGCACAAAGCTGGGACCACTATGAAAACTGGGCAATAAACTACACTTTTGATTTCCCTGGAATAAATGCAGATGCCCTCCACCCAGACCTCCAAACCGCCCCAATTGTCACAGACACCAGTGTCAGCAGCAGTGGTGGTGAAAGCTCTGAAGAACTCAGTGAAAGCAGCTTTTTCAACCTCATCACCCCAGGCACCTGGAACACTGAAACCCCGCGCTCTAGTACGCCCATCCCCGGGACCAGTTCAGGAGAATCATTTGTCGGAAGCTCAGTTTCCTCCGAAGCTGTAGCTGCATCGTGGGAAGAAGCCTTCTACACACCTTTGGCAGATCAGTTTCGTGAACTGTTAGTTGGGGTTGACTATGTATGGGACGGTGTGAGGGGCTTACCTGTTTGTTGTGTGCAACATATTAATAATAGTGGGGGAGGCTTGGGACTTTGTCCCCACTGCATTAATGTAGGGGCTTGGTATAATGGATGGAAGTTTCGAGAATTTACCCCA?ATTT?GTGCGGTGTAGCT??CATGT??GAG?TTCTAATYCCTTTTCTGTGTTAACCTGCAAAAAATGTGCTTACCTGTCTGGATTGCAAAGTTTTGTAGATTATGAGTAAAGAAAGTGGCAAATGGTGGGAAAGTAATGATAAATTTGCTAAAGCCGTGTATCAGCAATTTGTAGAATTTTATGAAAAAATTACTGGAACAGACTTAGAGCTTATTCAAATATTAAAAGATCATTACAATATTTCTTTAGATAATYC??TAGAAAACCCAT?TTCTTTGTTTGACTTAGTTGCTCGTATTAAAAATAACCTTAAAAACTCTCCA?AC?TATATAGTCATCATTTTCAAAGTCATGGACAGTTATCTGACCACCCCCATGCCTTATCATCCAGTAACAGTCATGCAGAACCTAGAGGAGAAAATGCAGTATTATCTAGTGAAGACTTACACAAGCCTGGGCAAGTTAGC??????CTACCCGGTACTAACTATGTTRGGCCTRGCAATDAGCTACAAGCTGGGCCCCCGCAAAGTGCTGTGGACAGTGCTGCAAGGATTCATGACTTTAGGTATAGCCAACTGGCTAAGTTGGGAATAAATCCATATACTCATTGGACTGTAGCAGATGAGGAGCTGTTAAAAAATATAAAAAATGAAACTGAGTTTCAARCACAAGCAGTAAAAGATTACTTTACTTTAAAAGGTGCAGCTGCCCCTGTGGCCCATTTTCAAGGAAGTTTGCCGGAAGTTCCCGCTTACAACGCCTCAGAAAAATACCCAAGCATGACTTCAGTTAATTCTGCAGAAGCCAGCACTGGCGCAGGAGGGGGCGGCAGTAATCCTGCCAAAAGCATGTGGAGTGAGGGGGCCACTTTTACTAGCAACTCTGTAACTTGTACATTTTCCAGACAGTTTTTAATTCCATATGACCCAGAGCACCATTATAAAGTGTTTTCTCCCGCA????GTAGCTGTCACAATGCCAGTGG?AAAGAGGCAAAGGTTTG?ACCATTAGTCY?ATAATGGGATA?TCAACACCATGGAGATACTTAGATTTTAATGCTTTAAATTTATTTTTTTCACCTTTAGAGTTTCAGCATTTAATTGAAAATTATGGAAGTATAGCTCCTGATGCTTTAACTGTGACTATATCAGAAATTGCTGTTAAGGATGTTACAGACAAAACTGGAGGAGGGGTACAGGTTACTGACAGTACTACAGGGCGTTTATGCATGTTGGTAGACCATGAATACAAGTACCCATATGTGTTAGGTCAGGGTCAGGATACCTTAGCCCCAGAACTTCCTATTTGGGTATATTTTCCCCCTCAATATGCTTATTTAACAGTAGGAGATGTAAACACACAAGGAATTTCAGGAGACAGTAAAAAATTAGCCAGTGAAGAATCAGCATTTTATGTTCTGGAGCACAGTTCTTTTGAACTTTTAGGTACAGGAGGCTCTGCTAGTATGTCTTATAAGTTTCCCGCAGTACCCCCAGAAAATTTAGAAGGCTGCAGTCAACACTTTTATGAAATGTACAACCCCTTATACGGATCCCGTTTAGGAGTTCCTGACACATTAGGAGGTGACCCCAAATTTAGATCTTTAACACATGAAGACCATGCAATTCAGCCACAAAACTTTATGCCGG‚Ä¶&lt;/p&gt;
&lt;/blockquote&gt;
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/acorg/parvo-2018&#34;&gt;acorg/parvo-2018&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
                        <div class="message-reaction">
                        ‚òùÔ∏è James Fellows Yates
                        </div>
                    
                        <div class="message-reaction">
                        üôå Ania TSL
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-23 09:32:58">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2559525262037_b6997ae3d531931ed9bc_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">James Fellows Yates
                     <span class="print-only user-email">(james_fellows_yates@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-23 09:32:58"><div class="time">2024-04-23 09:32:58</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks Alex!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-24 00:11:56">
            
                <img src="https://avatars.slack-edge.com/2023-07-31/5676552869649_e9442ec6e5e038847d33_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Ania TSL
                     <span class="print-only user-email">(leon.atny.7@gmail.com)</span>
                </div>
                <a href="#2024-04-24 00:11:56"><div class="time">2024-04-24 00:11:56</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thank you so much for your replies.  Also @Miriam Bravo show me how to download them from the ENA with the project number and is working.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç James Fellows Yates
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 12:02:58">
            
                <img src="https://avatars.slack-edge.com/2022-11-15/4370194090502_89c86a46fd40fa3f6798_72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Cameron Ferguson
                     <span class="print-only user-email">(cameron.ferguson.22@ucl.ac.uk)</span>
                </div>
                <a href="#2024-04-25 12:02:58"><div class="time">2024-04-25 12:02:58</div></a>
                <div class="msg">
                    <p>Hi I had a quick question that I hoped someone here could help me with, does anyone know where to find the SNP Evaluation documentation? I've checked the github repo but can't seem to find it. There is a powerpoint but it details a problem and a fix for the program but they don't seem to have any information on usage or how the program actually works.</p>

<p>only thing I could find was an active issue opened in 2019 saying the link to the pdf manual is broken:</p>

<p><a href="https://github.com/andreasKroepelin/SNP_Evaluation/issues/1">https://github.com/andreasKroepelin/SNP_Evaluation/issues/1</a></p>

<p>If anyone knows where to find the documentation or could point me in the right direction it would be greatly appreciated. üôÇ</p>
                    <div class="message-attachment"
                            style="border-color: #36a64f">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/andreasKroepelin/SNP_Evaluation/issues/1">#1 Link to PDF manual is broken</a></div>
                                <div class="link-text">
                                    &lt;p&gt;Hi Andreas,&lt;br /&gt;
unfortunately, the link to the PDF manual does not work. Would it be possible to provide the manual in the repository? That would be great :)&lt;/p&gt;

&lt;p&gt;Best, Judith&lt;/p&gt;
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/andreasKroepelin/SNP_Evaluation&#34;&gt;andreasKroepelin/SNP_Evaluation&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 12:46:59">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-04-25 12:46:59"><div class="time">2024-04-25 12:46:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Last time I asked, it didn't exist.
But if I remember correctly from the last (and only) time I used it, there were help pages available in the help menu. Maybe @aidanva or @Alina Hiss knows more details</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 12:57:11">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-04-25 12:57:11"><div class="time">2024-04-25 12:57:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* If you go to the ‚Äú?‚Äù in the menu, there is a bit of information on how to use it. But it is not a lot. I will suggest you get into contact with Alexander Herbig (<a href="mailto:alexander_herbig@eva.mpg.de">alexander_herbig@eva.mpg.de</a>), to see if he has a copy of this mysterious PDF.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 14:43:16">
            
                <img src="https://avatars.slack-edge.com/2022-11-15/4370194090502_89c86a46fd40fa3f6798_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Cameron Ferguson
                     <span class="print-only user-email">(cameron.ferguson.22@ucl.ac.uk)</span>
                </div>
                <a href="#2024-04-25 14:43:16"><div class="time">2024-04-25 14:43:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks, its a shame that this is turning into a much more difficult task then I'd hoped. I was mostly interested in how the program decided what is and ins‚Äôt a false positive SNP.</p>

<p>A paper I was looking at used it and based off the output table in the supplementary material it wasn‚Äôt clear to me how and why SNPs had or hadn‚Äôt been classified as false positives by the program so I was curious where this score was coming from üòÖ.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 15:31:48">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-04-25 15:31:48"><div class="time">2024-04-25 15:31:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* may I ask which paper?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 15:33:32">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-04-25 15:33:32"><div class="time">2024-04-25 15:33:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The score is based on the different metrics you can calculate, one of them being able to compare more strict mapping vs less strict mapping and seeing how the coverage is affected, you can also calculate the number of heterozygous calls around the SNP and also the number of bases missing in a a window around the SNP</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 15:59:20">
            
                <img src="https://avatars.slack-edge.com/2022-11-15/4370194090502_89c86a46fd40fa3f6798_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Cameron Ferguson
                     <span class="print-only user-email">(cameron.ferguson.22@ucl.ac.uk)</span>
                </div>
                <a href="#2024-04-25 15:59:20"><div class="time">2024-04-25 15:59:20</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Thanks after going back to the table and carefully looking through I reached the same conclusion. I noticed that only SNPs with a score of 1 were deemed to be not due to contamination and the score equaled the sum of the mapping ratio (50bp window), heterozygous positions (50bp window) and positions not covered (50bp window).</p>

<p>Meaning only SNPs that didn‚Äôt have any heterozygous calls in a 50 bp window, had all positions in a 50bp window covered and the mean coverage of the lenient mapping and strict mapping in a 50bp window was identical were considered not to be due to contamination.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 15:59:32">
            
                <img src="https://avatars.slack-edge.com/2022-11-15/4370194090502_89c86a46fd40fa3f6798_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Cameron Ferguson
                     <span class="print-only user-email">(cameron.ferguson.22@ucl.ac.uk)</span>
                </div>
                <a href="#2024-04-25 15:59:32"><div class="time">2024-04-25 15:59:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* The paper was Spyrou et al 2019 so not the most recent paper:
<a href="https://www.nature.com/articles/s41467-019-12154-0">https://www.nature.com/articles/s41467-019-12154-0</a></p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">Nature</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.nature.com/articles/s41467-019-12154-0">Phylogeography of the second plague pandemic revealed through analysis of historical Yersinia pestis genomes</a></div>
                                <div class="link-text">
                                    Nature Communications - The second plague pandemic, caused by Yersinia pestis, started in the 14th century and recurred in Europe until the 18th century. Here, the authors reconstruct ancient Y....
                                </div>
                                
                                
    
        <a href="https://www.nature.com/articles/s41467-019-12154-0">
            <img class="preview" src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-019-12154-0/MediaObjects/41467_2019_12154_Fig1_HTML.png"
                width="2000"
                 height="1258" />
        </a>
    
                                
                                    <div class="print-only">Original URL: https://www.nature.com/articles/s41467-019-12154-0</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 16:01:00">
            
                <img src="https://avatars.slack-edge.com/2020-03-04/981575467157_4d7ec14dc34e3091fa3b_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">aidanva
                     <span class="print-only user-email">(aida.andrades@gmail.com)</span>
                </div>
                <a href="#2024-04-25 16:01:00"><div class="time">2024-04-25 16:01:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yeah, exactly, that‚Äôs how you should interpret the score üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-25 18:03:30">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465355512771_d76bd23c72a4c5f61546_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Gunnar Neumann
                     <span class="print-only user-email">(gunnar_neumann@eva.mpg.de)</span>
                </div>
                <a href="#2024-04-25 18:03:30"><div class="time">2024-04-25 18:03:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* you might also want to check Keller et al. 2019: Ancient Yersinia pestis genomes from across Western Europe reveal early diversification during the First Pandemic
<a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1820447116">www.pnas.org/cgi/doi/10.1073/pnas.1820447116</a>
although i am not sure if it gives more info than what has been said here before</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-04-26 13:20:34">
            
                <img src="https://secure.gravatar.com/avatar/ca68fe1a380a485e095f6f13ef8730ee.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0005-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Johnny Richards
                     <span class="print-only user-email">(s2052280@ed.ac.uk)</span>
                </div>
                <a href="#2024-04-26 13:20:34"><div class="time">2024-04-26 13:20:34</div></a>
                <div class="msg">
                    <p>Hi All! Have a quick wet lab question: we‚Äôre looking at Quantas Quantiflor as an alternative to Qubit HS for quantification. Unsure if they‚Äôre comparable in performance for low concentration. Anyone used Quantas/found it as good as Qubit?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:38:18">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 19:38:18"><div class="time">2024-05-06 19:38:18</div></a>
                <div class="msg">
                    <p>Hi there, I am trying to establish analytical pipelines on a new cluster. I am doing it from scratch for the first time and I am running into some issues. Currently, I am trying to install aMeta @Zo√© Pochon and when I run the test I do not get the full set of results with plots etc. What is supposed to be the output from the test run?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:41:29">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 19:41:29"><div class="time">2024-05-06 19:41:29</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Joanna H. Bonczarowska please check this section in the aMeta github to learn about the main output files <a href="https://github.com/NBISweden/aMeta?tab=readme-ov-file#main-results-of-the-workflow-and-their-interpretation">https://github.com/NBISweden/aMeta?tab=readme-ov-file#main-results-of-the-workflow-and-their-interpretation</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:48:31">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 19:48:31"><div class="time">2024-05-06 19:48:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I only have "CUTADAPT<em>ADAPTER</em>TRIMMING" directory with fastq files foo and bar in the ./test/results So I guess it means that the test run failed and did not generate all the output?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:50:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 19:50:21"><div class="time">2024-05-06 19:50:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, possibly. Snakemake usually throws a lot of red text when an error occurs. Also at the very final lines of snakemake output it gives a link to a overall log-file, could you please post it here?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:55:41">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 19:55:41"><div class="time">2024-05-06 19:55:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I submitted the runtest.sh as a batch job in a cluster and I did not get any information printed out into a log file anywhere</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:58:22">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 19:58:22"><div class="time">2024-05-06 19:58:22</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* A batch job should typically produce a log-file of everything it would write on a screen if it was executed locally.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 19:59:45">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 19:59:45"><div class="time">2024-05-06 19:59:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Perhaps for installation purposes I would recommend to book a node via an interactive job. This way (when you control all the messages) the installation will be more efficient</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:13:19">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:13:19"><div class="time">2024-05-06 20:13:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey Joanna! Indeed, I think it might be best to run the test jobs directly on the login node or if this is strictly forbidden to book an interactive session of some cores to run the test and directly see the error messages. Maybe you could add the option - - keep-going in the Snakemake command within the runtest script so that it doesn't stop at the first small error</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:18:59">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:18:59"><div class="time">2024-05-06 20:18:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am now running it as you suggested, in an interactive session, and I got this error:</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:19:27">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:19:27"><div class="time">2024-05-06 20:19:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Error in rule FastQC<em>BeforeTrimming:
   jobid: 3
   input: data/bar.fq.gz
   output: results/FASTQC</em>BEFORE<em>TRIMMING/bar</em>fastqc.html, results/FASTQC<em>BEFORE</em>TRIMMING/bar<em>fastqc.zip
   log: logs/FASTQC</em>BEFORE<em>TRIMMING/bar.log (check log file(s) for error details)
   conda-env: /gpfs/gibbs.../.test/.snakemake/conda/c6ec736af08d79289f3602ae7da070b7</em>
   shell:
       fastqc data/bar.fq.gz --memory 10240 --threads 1 --nogroup --outdir results/FASTQC<em>BEFORE</em>TRIMMING &amp;&gt; logs/FASTQC<em>BEFORE</em>TRIMMING/bar.log
       (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)</p>

<h1>Logfile logs/FASTQC<em>BEFORE</em>TRIMMING/bar.log:</h1>

<p>Memory value 10240 MB was outside the allowed range (100 - 10000) at /gpfs/gibbs/project/.../.test/.snakemake/conda/c6ec736af08d79289f3602ae7da070b7_/bin/fastqc line 203.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:19:48">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:19:48"><div class="time">2024-05-06 20:19:48</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah, I thought it was that üôà</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:20:01">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:20:01"><div class="time">2024-05-06 20:20:01</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I tried to both increase and decrease the memory for the job but it did not help</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:20:12">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:20:12"><div class="time">2024-05-06 20:20:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, a known bug. To be fixed very soon</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:20:45">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:20:45"><div class="time">2024-05-06 20:20:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Is there a way to go around it?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:21:10">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:21:10"><div class="time">2024-05-06 20:21:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* For the moment I only specify the amount of cores and memory in my slurm config file like this:</p>

<h1>Amount of threads per rule</h1>

<p>set-threads:</p><ul><li>FastQC_AfterTrimming=2</li><li>FastQC_BeforeTrimming=2</li>
</ul>

<h1>Amount of resources per rule</h1>

<p>set-resources:</p><ul><li>FastQC<em>AfterTrimming:mem</em>mb=6000</li><li>FastQC<em>BeforeTrimming:mem</em>mb=6000</li>
</ul>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:22:13">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:22:13"><div class="time">2024-05-06 20:22:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes, please try Zoe‚Äôs suggestion, and also please check here <a href="https://github.com/NBISweden/aMeta/issues/156">https://github.com/NBISweden/aMeta/issues/156</a></p>
                    <div class="message-attachment"
                            style="border-color: #36a64f">
                            
                            
                            
                                
                                <div class="link-title"><a href="https://github.com/NBISweden/aMeta/issues/156">#156 mem_mb in FastQC rule should be changed</a></div>
                                <div class="link-text">
                                    &lt;p&gt;Right now FasQC&lt;em&gt;Before/After&lt;/em&gt;Trimming, the memory parameter is set to 1024 MB, which leads to this error message:&lt;/p&gt;

&lt;pre&gt;&lt;code&gt;Memory value 10240 MB was outside the allowed range (100 - 10000) at /aMeta/.snakemake/conda/29408ff88243ec3671290557f41f37da_/bin/fastqc line 203.
logs/FASTQC_AFTER_TRIMMING/NW-17.log (END)
&lt;/code&gt;&lt;/pre&gt;
                                </div>
                                
                                    <div class="attachment-field">
                                        <div class="field-title">Comments</div>
                                        2
                                    </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                
                                    <div class="attachment-footer">
                                        <img src="https://slack.github.com/static/img/favicon-neutral.png" class="icon" />
                                        &lt;a href=&#34;https://github.com/NBISweden/aMeta&#34;&gt;NBISweden/aMeta&lt;/a&gt;
                                    </div>
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:23:36">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:23:36"><div class="time">2024-05-06 20:23:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* But I don't really know how to implement that within the test run ü§î</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:25:33">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:25:33"><div class="time">2024-05-06 20:25:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* In the ./tes/config.yaml?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:25:36">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:25:36"><div class="time">2024-05-06 20:25:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* test**</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:26:03">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:26:03"><div class="time">2024-05-06 20:26:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok that did not work üíÄ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:26:05">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:26:05"><div class="time">2024-05-06 20:26:05</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Please open aMeta/workflow/rules/qc.smk and change one line in the rules FastQC<em>AfterTrimming and FastQC</em>BevoreTrimming</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:26:36">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:26:36"><div class="time">2024-05-06 20:26:36</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* mem_mb=10240</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:26:43">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:26:43"><div class="time">2024-05-06 20:26:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* please remove 0 at the end</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:27:00">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:27:00"><div class="time">2024-05-06 20:27:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yeah, what Nikolay is suggesting is probably the easiest</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:29:55">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:29:55"><div class="time">2024-05-06 20:29:55</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* there were two lines with mem_mb=10240, I changed them both (when I changed just the first one it did not work). Now it seems to be running!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:30:27">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:30:27"><div class="time">2024-05-06 20:30:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Please modify both lines</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:31:27">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:31:27"><div class="time">2024-05-06 20:31:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Apologies for this stupid bug, everyone in the team is currently busy but we will fix it very soon</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:32:45">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:32:45"><div class="time">2024-05-06 20:32:45</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Joanna H. Bonczarowska Another known bug that we haven't fixed yet is that one python script is not working with some of the newest python versions. So if you want to run the pipeline soon it would be good to check which python is used in your aMeta conda environment and in case it is a version of python.
If it is a version higher than 3.10, I would remove the conda aMeta environment using the conda command <code>conda env remove -n aMeta</code>
Then open the file aMeta/workflow/envs/environment.yaml and change the line about python with that line: <code>- python&amp;gt;=3.7,&amp;lt;3.10</code> and recreate the aMeta environment.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:33:26">
            
                <img src="https://avatars.slack-edge.com/2021-10-04/2556573805990_6b8e4ed271cbbb868f1f_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Zo√© Pochon
                     <span class="print-only user-email">(zoe.pochon@gmail.com)</span>
                </div>
                <a href="#2024-05-06 20:33:26"><div class="time">2024-05-06 20:33:26</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sorry about that, we've been busy indeed and haven't fixed it yet</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:34:24">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 20:34:24"><div class="time">2024-05-06 20:34:24</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It now failed at the loading the ncbi.tree, so I will look into the python version</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 20:49:51">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 20:49:51"><div class="time">2024-05-06 20:49:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Joanna H. Bonczarowska does your cluster have internet connection?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:00:21">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:00:21"><div class="time">2024-05-06 21:00:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes it does</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:00:42">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:00:42"><div class="time">2024-05-06 21:00:42</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I got a new error now üôà</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:01:19">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:01:19"><div class="time">2024-05-06 21:01:19</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Please post it here</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:10:41">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:10:41"><div class="time">2024-05-06 21:10:41</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* RuleException:
CalledProcessError in file /gpfs/gibbs.../aMeta/workflow/rules/malt.smk, line 26:
Command 'source /home/jhb94/.conda/envs/aMeta/bin/activate '/gpfs/gibbs/.../aMeta/.test/.snakemake/conda/45303af74a9961a9c754f37c4bed599c<em>'; set -euo pipefail; python /gpfs/gibbs/.../aMeta/.test/.snakemake/scripts/tmp5bxrmrnn.malt-build.py' returned non-zero exit status 1.
 File "/gpfs/gibbs/.../aMeta/workflow/rules/malt.smk", line 26, in _</em>rule<em>Build</em>Malt_DB
 File "/home/jhb94/.conda/envs/aMeta/lib/python3.9/concurrent/futures/thread.py", line 58, in run</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:12:32">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:12:32"><div class="time">2024-05-06 21:12:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Joanna H. Bonczarowska this looks like the wrong python version error. Did you reinstall aMeta from scratch as Zoe suggested?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:13:27">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:13:27"><div class="time">2024-05-06 21:13:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Could you check (being within aMeta environment) what <code>python --version</code> gives to you?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:14:09">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:14:09"><div class="time">2024-05-06 21:14:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I did reinstall aMeta and the version 3.9.19</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:16:27">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:16:27"><div class="time">2024-05-06 21:16:27</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, could you please post the log-file from .test/logs/BUILD<em>MALT</em>DB?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:18:08">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:18:08"><div class="time">2024-05-06 21:18:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* "Output files of filed job Build<em>Malt</em>DB were removed since they might be corrupted"</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:18:46">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:18:46"><div class="time">2024-05-06 21:18:46</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* no, log-files are not removed, please check the .test/logs/BUILD<em>MALT</em>DB folder. There should be a BUILD<em>MALT</em>DB.log files inside, I would like to look at this file</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:20:14">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:20:14"><div class="time">2024-05-06 21:20:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Version  MALT (version 0.6.2, built 12 Sep 2023)
Author(s) Daniel H. Huson
Copyright (C) 2023 Daniel H. Huson. This program comes with ABSOLUTELY NO WARRANTY.
Java version: 20.0.2; max memory: 2.9G
Classifications to use: Taxonomy
Reference sequence type set to: DNA
Seed shape(s): 111110111011110110111111
Number input files:           1
Loading FastA files:
10% 100% (0.3s)
Number of sequences:      1,000
Number of letters:   10,000,000
BUILDING table (0)...
Seeds found:     9,977,000
tableSize=      16,777,216
hashMask.length=24
maxHitsPerHash set to: 1000
Initializing arrays...
100% (0.0s)
Analysing seeds...
10% 20% 30% 40% 100% (2.0s)
Number of low-complexity seeds skipped: 43
Allocating hash table...
10% 100% (0.5s)
Total keys used:        7,519,628
Total seeds matched:    9,976,957
Total seeds dropped:            0
Opening file: results/MALT<em>DB/maltDB.dat/table0.db
Allocating: 41.8 MB
Filling hash table...
10% 20% 30% 40% 50% 100% (2.5s)
Randomizing rows...
10% 20% 30% 100% (0.3s)
Writing file: results/MALT</em>DB/maltDB.dat/table0.idx
10% 20% 30% 40% 50% 60% 70% 80% 90% 100% (2.4s)
Writing file: results/MALT<em>DB/maltDB.dat/table0.db
Size: 41.8 MB
10% 100% (0.8s)
Writing file: results/MALT</em>DB/maltDB.dat/index0.idx
100% (0.0s)
Loading ncbi.map: 2,396,736
Loading ncbi.tre: 2,396,740
Loading file: resources/accession2taxid.map</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:21:21">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:21:21"><div class="time">2024-05-06 21:21:21</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* is that all? No more lines in that files?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:21:51">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:21:51"><div class="time">2024-05-06 21:21:51</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* yes thats all the content of the log file</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:23:30">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:23:30"><div class="time">2024-05-06 21:23:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* are you running it on the login-node or on a booked compute node as an interactive job? I mean some login nodes have very little RAM so that even a test-run can't finish successfully üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:24:23">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:24:23"><div class="time">2024-05-06 21:24:23</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* on a compute node</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:24:47">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:24:47"><div class="time">2024-05-06 21:24:47</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I will request more memory and try again</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:25:06">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:25:06"><div class="time">2024-05-06 21:25:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* to get a clean experiment, could you please just delete the whole results folder as "rm -rf .test/results" and re-run the testrun?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:26:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:26:11"><div class="time">2024-05-06 21:26:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* if you booked just one CPU, it might happen it had very little to no RAM. The error you posted above looks like very little RAM (no more than ~50MB) was available at that compute node</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:27:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:27:11"><div class="time">2024-05-06 21:27:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* booking a lot of RAM is essential for production runs of aMeta, so better to learn from the very beginning how to book more RAM</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:27:59">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:27:59"><div class="time">2024-05-06 21:27:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am reruning it from scratch with more memory. If it fails again I will request more CPUs.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:31:10">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:31:10"><div class="time">2024-05-06 21:31:10</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok the error remained, but I got more information on the malt build error.. "java.lang.Out.Of.MemoryError: Java Heap space"</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:32:33">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:32:33"><div class="time">2024-05-06 21:32:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ok, the solution for Java heapspace error is described here <a href="https://github.com/NBISweden/aMeta/tree/main?tab=readme-ov-file#i-get-java-heap-space-error-on-the-malt-step-what-should-i-do">https://github.com/NBISweden/aMeta/tree/main?tab=readme-ov-file#i-get-java-heap-space-error-on-the-malt-step-what-should-i-do</a></p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:32:54">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:32:54"><div class="time">2024-05-06 21:32:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I have seen it before when using malt and could fix it in the "vmoptions" file</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:32:57">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:32:57"><div class="time">2024-05-06 21:32:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* ah yes!</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:33:00">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:33:00"><div class="time">2024-05-06 21:33:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Perhaps the easiest solution is to book a few CPUs</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:34:30">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:34:30"><div class="time">2024-05-06 21:34:30</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* fixing the vmoptions should also help but I believe booking more CPU, which automatically should result in more RAM, is an even easier way</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:46:32">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:46:32"><div class="time">2024-05-06 21:46:32</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* It seems like it is running but I dont want to jinx it</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:54:57">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:54:57"><div class="time">2024-05-06 21:54:57</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I got pdfs generated! It just shows Y. pestis in both samples. Was that the expected output?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:55:14">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:55:14"><div class="time">2024-05-06 21:55:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes! üôÇ </p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:56:15">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-06 21:56:15"><div class="time">2024-05-06 21:56:15</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Congratulations! You are done with the testrun. Let us try to run a real sample tomorrow</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-06 21:56:43">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-06 21:56:43"><div class="time">2024-05-06 21:56:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* That is great! I needed a "win" today üôÇ Thank you so much for your help. I will try a real sample next</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Zo√© Pochon
                        </div>
                    
                        <div class="message-reaction">
                        üôå Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-10 20:06:03">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-10 20:06:03"><div class="time">2024-05-10 20:06:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov I am now trying to download the datasets to run my samples with the pipeline and I have a question concerning the helping files for building malt repository. When I follow the link
```# Helping files for building Malt database</p>

<h1>can be downloaded from <a href="https://doi.org/10.17044/scilifelab.21070063">https://doi.org/10.17044/scilifelab.21070063</a></h1>

<p>malt<em>nt</em>fasta: resources/library.fna
malt<em>seqid2taxid</em>db: resources/seqid2taxid.map.orig
malt<em>accession2taxid: resources/nucl</em>gb.accession2taxid```
It takes me to bowtie2 index for full ncbi nt dataset. Is that correct?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-10 20:08:33">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-10 20:08:33"><div class="time">2024-05-10 20:08:33</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Yes @Joanna H. Bonczarowska, that is correct. Please download only the files that are mentioned above, i.e. you do not have to download the other files from the bowtie2 index</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-10 20:53:00">
            
                <img src="https://avatars.slack-edge.com/2022-09-19/4123699280720_045b27cb2029fd260b2d_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna H. Bonczarowska
                     <span class="print-only user-email">(j.bonczarowska@ikmb.uni-kiel.de)</span>
                </div>
                <a href="#2024-05-10 20:53:00"><div class="time">2024-05-10 20:53:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ah, I see! Thank you</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-24 09:51:08">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-24 09:51:08"><div class="time">2024-05-24 09:51:08</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hi @Joanna H. Bonczarowska, we have recently fixed a few issues with aMeta, it should be more stable now. So if <code>git pull</code> the recent changes, I hope you should not encounter any problems with testrun or real data run. As I said previously, I am ready to assist you to properly install it once so that you can use aMeta in the future for your projects</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-25 19:49:13">
            
                <img src="https://secure.gravatar.com/avatar/a2113cd3072096ab48dfd32d6c99bcee.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna Bonczarowska
                     <span class="print-only user-email">(joanna.bonczarowska@yale.edu)</span>
                </div>
                <a href="#2024-06-25 19:49:13"><div class="time">2024-06-25 19:49:13</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov I finally managed to get the databases downloaded onto the cluster! I also have my first data ready and I will now try to run the pipeline on my real data. For the installation of the job-specific environments part, do I just copy the commands provided on github?</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Zo√© Pochon
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-25 19:58:35">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-06-25 19:58:35"><div class="time">2024-06-25 19:58:35</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Joanna Bonczarowska yes, you will need to prepare the config.yaml and samples.tsv files following the instructions from here <a href="https://github.com/NBISweden/aMeta?tab=readme-ov-file#quick-start">https://github.com/NBISweden/aMeta?tab=readme-ov-file#quick-start</a>. Please carefully specify the paths to the databases on your disk, and do not forget to unzip the library.fna.gz file (it is a common mistake to keep it zipped). To be on a safe size, you can send me both config.yaml and samples.tsv via a direct message (in order to not overload this thread) so that I could have a look and correct them if necessary. Then you should be ready to start with installation of job specific environments like this:
```cd aMeta</p>

<h1>install job-specific environments</h1>

<p>snakemake --snakefile workflow/Snakefile --use-conda --conda-create-envs-only -j 20```</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Joanna Bonczarowska
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-25 20:30:12">
            
                <img src="https://secure.gravatar.com/avatar/a2113cd3072096ab48dfd32d6c99bcee.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0000-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Joanna Bonczarowska
                     <span class="print-only user-email">(joanna.bonczarowska@yale.edu)</span>
                </div>
                <a href="#2024-06-25 20:30:12"><div class="time">2024-06-25 20:30:12</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Great, thank you very much! I will be in touch with you via private message üòÑ</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-13 11:36:34">
            
                <img src="https://secure.gravatar.com/avatar/ba6d03066ef230c5975ebf616f7428be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Sarah Martin
                     <span class="print-only user-email">(sarah.martin@ntnu.no)</span>
                </div>
                <a href="#2024-05-13 11:36:34"><div class="time">2024-05-13 11:36:34</div></a>
                <div class="msg">
                    <p>Hello! I‚Äôm doing sedaDNA metagenomic sequencing from archaeological sites. My focus is eukaryotes (for now, until I can get PhD student to dive further into the data). How much data do you recommend? I‚Äôm hoping to go for 10GB for a preliminary look‚Ä¶is that too low though?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-13 20:09:59">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-05-13 20:09:59"><div class="time">2024-05-13 20:09:59</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Hey @Sarah Martin,
Could you give a bit more details on what you want to do ?
Non targeted shotgun sequencing of sedaDNA samples won't yield a lot of aDNA, but I'm not sure what you mean by then 10GB. 10 billion reads ?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-13 20:13:17">
            
                <img src="https://secure.gravatar.com/avatar/ba6d03066ef230c5975ebf616f7428be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Martin
                     <span class="print-only user-email">(sarah.martin@ntnu.no)</span>
                </div>
                <a href="#2024-05-13 20:13:17"><div class="time">2024-05-13 20:13:17</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Maxime Borry  We are taking soil samples down a stratigraphic wall and want to compare the contents and look at the introduction of domesticates, that sort of thing.
I‚Äôve done this before with 20Gbases of data, but want to know if there‚Äôs a ‚Äústandard‚Äù number of bases, or reads, people use for this sort of thing. I‚Äôd ideally like to reduce my sequencing to 10 billion bases (I‚Äôm counting in bases because that‚Äôs what I order from Novogene, rather than reads). Does that help?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-13 23:34:53">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2476010422849_6a5c72f7f68ad99874b8_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Maxime Borry
                     <span class="print-only user-email">(maxime.borry@gmail.com)</span>
                </div>
                <a href="#2024-05-13 23:34:53"><div class="time">2024-05-13 23:34:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Sounds like a question for @Kevin Nota @Niall Cooke @Merlin Szymanski üôÇ</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-14 17:17:31">
            
                <img src="https://avatars.slack-edge.com/2023-10-25/6085479900966_31abc2b5900af9ac7d4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Nota
                     <span class="print-only user-email">(kevin_nota@eva.mpg.de)</span>
                </div>
                <a href="#2024-05-14 17:17:31"><div class="time">2024-05-14 17:17:31</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I don't think there is really a standard. We usually aim for 5-10M reads per sample to get an idea of the taxonomic composition. This number of reads tends to give us mainly hits to nuclear genomes and only some mtDNA in the very rich samples. In most cases, I would say doubling or even getting 10 times more data will not help you that much for getting extra taxa. You would get 10 times more reads for the taxa you already have, and some low abundant species that have something like 10 reads will then give you 100 reads, which in most cases is not enough to do things with. In the end, it depends on the sample and budget, if you sequence more, you are likely to get more.
I am not sure how to translate this to the number of bases, this would depend on the read length.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-14 17:19:37">
            
                <img src="https://secure.gravatar.com/avatar/ba6d03066ef230c5975ebf616f7428be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Martin
                     <span class="print-only user-email">(sarah.martin@ntnu.no)</span>
                </div>
                <a href="#2024-05-14 17:19:37"><div class="time">2024-05-14 17:19:37</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Kevin Nota thanks! This is very helpful.
I have another question for you. For my purposes (simply assigning eukaryote taxa) would you suggest Holi? We‚Äôve tried aMeta, but I get the feeling it‚Äôs not particularly suited for eukaryotes‚Ä¶</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-14 17:38:54">
            
                <img src="https://avatars.slack-edge.com/2023-10-25/6085479900966_31abc2b5900af9ac7d4e_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Nota
                     <span class="print-only user-email">(kevin_nota@eva.mpg.de)</span>
                </div>
                <a href="#2024-05-14 17:38:54"><div class="time">2024-05-14 17:38:54</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* As a start, I think the Holi pipeline is a good choice. We are working on our own pipeline for nuclear DNA that is somewhat inspired by the Holi pipeline. It is hard to recommend a pipeline, that will work well "straight out of the box" and the taxonomic list that these pipelines produce needs validation per site and taxonomic groups. We have been kind of stuck on this part, that comes after mapping and classification - the validation, like how much you can really trust mappings, are the reference genomes that attract mapped reads are good enough? and to what extent is taxonomic bleedthrough a problem etc. 'simply assigning eukaryote taxa' for nuclear data is hard:)</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Sarah Martin, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-14 17:39:50">
            
                <img src="https://secure.gravatar.com/avatar/ba6d03066ef230c5975ebf616f7428be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Martin
                     <span class="print-only user-email">(sarah.martin@ntnu.no)</span>
                </div>
                <a href="#2024-05-14 17:39:50"><div class="time">2024-05-14 17:39:50</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Kevin Nota thanks! Your input is most appreciated!</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Kevin Nota
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-24 09:58:11">
            
                <img src="https://avatars.slack-edge.com/2021-06-03/2119668201959_d232efbdd302cd3f8174_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Nikolay Oskolkov
                     <span class="print-only user-email">(nikolay.oskolkov@scilifelab.se)</span>
                </div>
                <a href="#2024-05-24 09:58:11"><div class="time">2024-05-24 09:58:11</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Sarah Martin I agree, while aMeta can technically be used for detecting eukaryotes if you use the full NCBI NT database (prebuilt and provided together with aMeta), it may not be optimal on the authentication analysis step as aMeta's coverage metrics do not currently assume multiple-chromosome reference genomes. I do use aMeta from time to time for environmental / sedimentary aDNA projects for figuring out what organisms are in my samples, however authentication requires some extra work</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-24 11:18:14">
            
                <img src="https://secure.gravatar.com/avatar/ba6d03066ef230c5975ebf616f7428be.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Sarah Martin
                     <span class="print-only user-email">(sarah.martin@ntnu.no)</span>
                </div>
                <a href="#2024-05-24 11:18:14"><div class="time">2024-05-24 11:18:14</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Nikolay Oskolkov thanks for your input! We decided to rerun our analyses with HOLI and so far it seems more suitable for our needs. We‚Äôll authenticate the major species separately.</p>
                    
                    
                    
                        <div class="message-reaction">
                        üëç Nikolay Oskolkov, Biancamaria Bonucci
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-24 15:56:00">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2024-05-24 15:56:00"><div class="time">2024-05-24 15:56:00</div></a>
                <div class="msg">
                    <p>Have folks experience with sending (pathogen) genomic dna for rna bait production e.g. Arbor Bioscience / daicel?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-05-24 15:57:23">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2024-05-24 15:57:23"><div class="time">2024-05-24 15:57:23</div></a>
                <div class="msg">
                    <p>I heard this was cheap way for bait production but there's something about ordering dna of a zoonotic organism that doesn't sound right üò∞  or if it was possible to order (from a dna bank..?) and simply deliver straight to bait producer</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ùì Iseult, Louis L&#39;H√¥te
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-26 11:32:43">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2449837579815_a720cf75a64421bc99b2_72.png" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kelly Blevins
                     <span class="print-only user-email">(blevinske1@gmail.com)</span>
                </div>
                <a href="#2024-06-26 11:32:43"><div class="time">2024-06-26 11:32:43</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* @Kevin Daly sorry for the late reply just seeing this. I got genomic DNA from several <em>M. tuberculosis</em> strains from <a href="https://www.beiresources.org/">https://www.beiresources.org/</a> I sent the gDNA to Arbor and they created the RNA baits. It‚Äôs much cheaper than synthesising baits. The DNA I sent was enough for over 100rxns I believe. The captures worked well. Pretty even and high coverage for good performing samples.</p>
                    <div class="message-attachment"
                            >
                            <div class="service-name">beiresources.org</div>
                            
                            
                                
                                <div class="link-title"><a href="https://www.beiresources.org/">BEI Resources Web Portal &amp;gt; Home</a></div>
                                <div class="link-text">
                                    Biodefense and Emerging Infections Research Resources Repository (BEI Resources) is the leading source for reagents, tools and information for studying NIAID&#39;s Category A, B, and C priority pathogens and emerging infectious disease agents.
                                </div>
                                
                                
    
        <!-- no preview available -->
    
                                
                                    <div class="print-only">Original URL: https://www.beiresources.org/</div>
                                
                                
                            
                        </div>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-03 18:36:49">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-06-03 18:36:49"><div class="time">2024-06-03 18:36:49</div></a>
                <div class="msg">
                    <p>Hi all! I am trying to run gtdbtk classify<em>wf in an assembled dataset (30 MAGs). However, when the analysis arrives to the step "Identifying TIGRFAM protein families" the job fails but I do not have any error in my log file. My guess is that the analysis needs more memory than requested. I am giving 100 Gb and 64 CPUs. Does anyone have experience running this kind of analysis? Do you think the problem could be the memory? This is my command: gtdbtk classify</em>wf --cpus 64 --extension fa --genome<em>dir /data/input --out</em>dir /data/output --skip<em>ani</em>screen (I also tried to run the analysis without the flag --skip<em>ani</em>screen but it gave me an error: gtdbtk classify<em>wf: error: one of the arguments --skip</em>ani<em>screen --mash</em>db is required)</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-03 18:42:44">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2024-06-03 18:42:44"><div class="time">2024-06-03 18:42:44</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Ohhh, I just dealt with this and it was a weird conda dependency issue</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-03 18:43:06">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2024-06-03 18:43:06"><div class="time">2024-06-03 18:43:06</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* What version of gtdbtk are you using?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-04 10:20:03">
            
                <img src="https://avatars.slack-edge.com/2023-08-28/5795636690391_6ce28c761ab7c1227f75_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Patr√≠cia Santos
                     <span class="print-only user-email">(sntprc1@unife.it)</span>
                </div>
                <a href="#2024-06-04 10:20:03"><div class="time">2024-06-04 10:20:03</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* I am using in docker version 2.4.0. Did you solve the problem with the conda dependency?</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-13 05:48:00">
            
                <img src="https://avatars.slack-edge.com/2021-09-07/2465439210194_f020299b9cdea4d5d503_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Shreya
                     <span class="print-only user-email">(shreya23@uchicago.edu)</span>
                </div>
                <a href="#2024-06-13 05:48:00"><div class="time">2024-06-13 05:48:00</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* So it was stupid, it turned out that I also had python (which had gtdbtk) loaded independently of the conda environment I had loaded, so the versions were clashing</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-12 13:53:14">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2024-06-12 13:53:14"><div class="time">2024-06-12 13:53:14</div></a>
                <div class="msg">
                    <p>another not-stupid-question: for those who have designed/had designed for them in-solution probes for specific genomes, how much tradeoff was given to specificity i.e. excluding probes which may also match environmental relatives?</p>
                    
                    
                    
                        <div class="message-reaction">
                        ‚ùì Iseult
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-12 16:06:09">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2024-06-12 16:06:09"><div class="time">2024-06-12 16:06:09</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* honestly depends on the species and how many closely related species you are targeting.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-12 16:07:53">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2024-06-12 16:07:53"><div class="time">2024-06-12 16:07:53</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* but its not just about excluding but also playing with  % of allowed mismatches/hybridization temperature/hybridization time for example</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-12 16:56:16">
            
                <img src="https://avatars.slack-edge.com/2022-10-26/4280634576436_ca15e84edcbc8e7b7f95_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Kevin Daly
                     <span class="print-only user-email">(dalyk1@tcd.ie)</span>
                </div>
                <a href="#2024-06-12 16:56:16"><div class="time">2024-06-12 16:56:16</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* Good points. I had been hoping to go along with arbor's recommendation and probe design heuristics but they probably don't take any of these variables into account</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-12 16:58:34">
            
                <img src="https://avatars.slack-edge.com/2021-09-06/2459639456802_3577bc5843aa2d622b50_72.jpg" class="user_icon_reply" />
            
            
                <div class="reply">
            
                <div class="username">Meriam Guellil
                     <span class="print-only user-email">(meriam.guellil.ac@gmail.com)</span>
                </div>
                <a href="#2024-06-12 16:58:34"><div class="time">2024-06-12 16:58:34</div></a>
                <div class="msg">
                    <p><strong><em>*Thread Reply:</em></strong>* They do to some degree if you ask them. Plus they have some set thresholds but some of it doesn't depend on them either since it is dependent on your hybridisation setup during enrichment. Sequence divergence within the species will also be a deciding factor.</p>
                    
                    
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
                
                    <div class="message-container">
        <div id="2024-06-13 05:45:57">
            
                <img src="https://secure.gravatar.com/avatar/0c3a4922bc49d07ad19c1a4733e64e0d.jpg?s=72&amp;d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png" class="user_icon" />
            
            
                <div class="message">
            
                <div class="username">Sterling Wright
                     <span class="print-only user-email">(sterlingwright2016@utexas.edu)</span>
                </div>
                <a href="#2024-06-13 05:45:57"><div class="time">2024-06-13 05:45:57</div></a>
                <div class="msg">
                    <p>ls</p>
                    
                    
                    
                        <div class="message-reaction">
                        :headbangingparrot: Jasmin Frangenberg, Nikolay Oskolkov
                        </div>
                    
		</div>
            </div>
        </div>
    </div>
                
            
        
            
        
    </div>
</div>

<script>
(function() {
  var sidebar = document.querySelector('#sidebar');
  var selected = document.querySelector('.active');
  sidebar.scrollTop = selected.offsetTop - 200;

  // make dropdown from channel title
  var channel_title = document.querySelector("#channel-title");
  var channel_dropdown = document.querySelector("#channel-list");
  channel_title.addEventListener('click', function() {
    channel_title.classList.toggle('arrow');
    channel_dropdown.classList.toggle('close');
  });

  // make dropdown from group title
  var group_title = document.querySelector("#group-title");
  var group_dropdown = document.querySelector("#group-list");
  group_title.addEventListener('click', function() {
    group_title.classList.toggle('arrow');
    group_dropdown.classList.toggle('close');
  });

  // make dropdown from dm title
  var dm_title = document.querySelector("#dm-title");
  var dm_dropdown = document.querySelector("#dms-list");
  dm_title.addEventListener('click', function() {
    dm_title.classList.toggle('arrow');
    dm_dropdown.classList.toggle('close');
  });

  // make dromdown from group dm title
  var mpim_title = document.querySelector("#mpim-title");
  var mpim_dropdown = document.querySelector("#mpims-list");
  mpim_title.addEventListener('click', function() {
    mpim_title.classList.toggle('arrow');
    mpim_dropdown.classList.toggle('close');
  });
})()
</script>
</body>
</html>